{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pyseqrna import pyseqrna_utils as pu\n",
    "from pyseqrna import quality_check as qc\n",
    "from pyseqrna import quality_trimming as qt\n",
    "from pyseqrna import  aligners as al\n",
    "from pyseqrna import pyseqrna_stats as ps\n",
    "from pyseqrna import quants\n",
    "from pyseqrna import differential_expression as de\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pyseqrna.pyseqrna_utils import PyseqrnaLogger\n",
    "\n",
    "log = PyseqrnaLogger(mode='w', log='pp')\n",
    "\n",
    "log.info(\"Analysis started\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "data = pu.read_input_file(\"pyseqrna/example/input_Sample_PE.txt\", \"pyseqrna/example/data/\" , paired=True)\n",
    "samples= data['samples']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pu.make_directory(\"pySeqRNA_results\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "a = qt.trim_galoreRun(sampleDict=samples, paired=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "aligner = al.STAR_Aligner(genome=\"pyseqrna/example/data/arabidopsis10.fasta\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "j = aligner.build_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "aligner.check_index()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "outalign, job = aligner.run_Alignment(target=a[0], pairedEND=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df= ps.align_stats(sampleDict=samples,trimDict=a[0],bamDict=outalign, pairedEND=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df.to_excel(\"pySeqRNA_results/alignment_stats.xlsx\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import matplotlib.pyplot as plt\n",
    "import numpy as np\n",
    "import matplotlib\n",
    "import seaborn as sns"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "sample = list(df['Sample'])\n",
    "creads = list(df['Input_reads2x'])\n",
    "iread = list(df['%_Cleaned2x'])\n",
    "tc= list(100-df['%_Cleaned2x'])\n",
    "\n",
    "plt.barh(sample,iread,0.6,label=\"cleaned Reads\", color='skyblue')\n",
    "plt.barh(sample, tc, 0.6,left=iread, label=\"discarded\", color='tomato')\n",
    "plt.margins(y=0.01)\n",
    "plt.xticks(fontsize=15)\n",
    "plt.yticks(fontsize=15)\n",
    "plt.xlabel(\"% Number of reads\", fontsize=18)\n",
    "plt.ylabel(\"Samples\", fontsize=18)\n",
    "plt.legend(['Cleaned', 'Discarded'], bbox_to_anchor=(1.0, 1.0), loc='upper left', fontsize=15)\n",
    "fig = matplotlib.pyplot.gcf()\n",
    "fig.set_size_inches(18.5, 10.5)\n",
    "\n",
    "# plt.legend(frameon=False, fontsize=17)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "tc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "a= quants.featureCount(bamDict=outalign, gff=\"pyseqrna/example/data/arabidopsis10.gff\", outDir=\"pySeqRNA_results\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "counts= pd.read_csv(\"pySeqRNA_results/Counts_final.txt\", sep=\"\\t\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "counts"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "targets= data['targets']\n",
    "comb= data['combinations']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "dd= de.runDESeq2(countDF=counts,targetFile=targets,design='sample', combination=comb)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os \n",
    "dd.to_excel(os.path.join(\"pySeqRNA_results\",\"Raw_DEGs_all.xlsx\"), index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "dd= pd.read_excel(\"pySeqRNA_results/Raw_DEGs_all.xlsx\")\n",
    "filtered_DEG = de.degFilter(degDF=dd, CompareList=comb, FDR=0.05, FOLD=1)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "summary= filtered_DEG['summary']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "wd= pd.ExcelWriter(os.path.join(\"pySeqRNA_results\",\"filtered_DEGs.xlsx\"))\n",
    "for key, value in filtered_DEG['filtered'].items():\n",
    "    value.to_excel(wd,sheet_name=key)\n",
    "    wd.save()\n",
    "wd.close()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "pu.getGenes(os.path.join(\"pySeqRNA_results\",\"filtered_DEGs.xlsx\"), combinations=comb)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pyseqrna import normalize_counts as nc"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rpkm = nc.Normalization(countFile=\"pySeqRNA_results/Counts_final.txt\", featureFile=\"pyseqrna/example/data/arabidopsis10.gff\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rpk = rpkm.meanRatioCount()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rpk.to_excel(\"pySeqRNA_results/rpkm.xlsx\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rcount= pd.read_excel(\"pySeqRNA_results/rpkm.xlsx\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "rcount"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pyseqrna import pyseqrna_plots as pp"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "heatmap, ax = pp.plotHeatmap(dd,combinations=comb,num=50, type='deg')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pyseqrna import gene_ontology as go"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "bdata= go.query('athaliana')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gdata = go.preprocessBioMart(bdata)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "results = go.enrichGO(gdata,\"pySeqRNA_results.1/diff_genes/M1-V6.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "results.to_excel(\"pySeqRNA_results.1/M1-V6_go.xlsx\", index=False)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "cd= pd.read_excel(\"pySeqRNA_results.1/M1-V6_deg.xlsx\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "gdata"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_request(url,  **params):\n",
    "\n",
    "    if params:\n",
    "        r = requests.get(url, params=params, stream=True)\n",
    "    else:\n",
    "        r = requests.get(url)\n",
    "    r.raise_for_status()\n",
    "\n",
    "    return r\n",
    "\n",
    "\n",
    "def _add_attr_node(root, attr):\n",
    "    attr_el = ElementTree.SubElement(root, 'Attribute')\n",
    "    attr_el.set('name', attr)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "\n",
    "\n",
    "root = ElementTree.Element('Query')\n",
    "root.set('virtualSchemaName', 'plants_mart')\n",
    "root.set('formatter', 'TSV')\n",
    "root.set('header', '1')\n",
    "root.set('uniqueRows', native_str(int(True)))\n",
    "root.set('datasetConfigVersion', '0.6')\n",
    "\n",
    "dataset = ElementTree.SubElement(root, 'Dataset')\n",
    "dataset.set('name', species+\"_eg_gene\")\n",
    "dataset.set('interface', 'default')\n",
    "attributes = [\"ensembl_gene_id\", \"ensembl_transcript_id\",\n",
    "                \"go_id\", \"name_1006\", \"namespace_1003\", \"definition_1006\"]\n",
    "for attr in attributes:\n",
    "    _add_attr_node(dataset, attr)\n",
    "\n",
    "response = get_request(\n",
    "    \"https://plants.ensembl.org/biomart/martservice\", query=ElementTree.tostring(root))\n",
    "result = pd.read_csv(StringIO(response.text), sep='\\t')\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pyseqrna import pathway\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[12:40:48]  pathway :: INFO : Reading pathways from KEGG\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ENTRY       ath00010                    Pathway\n",
      "NAME        Glycolysis / Gluconeogenesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Glycolysis is the process of converting glucose into pyruvate and generating small amounts of ATP (energy) and NADH (reducing power). It is a central pathway that produces important precursor metabolites: six-carbon compounds of glucose-6P and fructose-6P and three-carbon compounds of glycerone-P, glyceraldehyde-3P, glycerate-3P, phosphoenolpyruvate, and pyruvate [MD:M00001]. Acetyl-CoA, another important precursor metabolite, is produced by oxidative decarboxylation of pyruvate [MD:M00307]. When the enzyme genes of this pathway are examined in completely sequenced genomes, the reaction steps of three-carbon compounds from glycerone-P to pyruvate form a conserved core module [MD:M00002], which is found in almost all organisms and which sometimes contains operon structures in bacterial genomes. Gluconeogenesis is a synthesis pathway of glucose from noncarbohydrate precursors. It is essentially a reversal of glycolysis with minor variations of alternative paths [MD:M00003].\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00010  Glycolysis / Gluconeogenesis\n",
      "MODULE      ath_M00001  Glycolysis (Embden-Meyerhof pathway), glucose => pyruvate [PATH:ath00010]\n",
      "ath_M00002  Glycolysis, core module involving three-carbon compounds [PATH:ath00010]\n",
      "ath_M00003  Gluconeogenesis, oxaloacetate => fructose-6P [PATH:ath00010]\n",
      "ath_M00307  Pyruvate oxidation, pyruvate => acetyl-CoA [PATH:ath00010]\n",
      "DBLINKS     GO: 0006096 0006094\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G37840  HKL3; hexokinase-like 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G47840  HXK3; hexokinase 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT2G19860  HXK2; hexokinase 2 [KO:K00844] [EC:2.7.1.1]\n",
      "AT4G29130  HXK1; hexokinase 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G50460  HKL1; hexokinase-like 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT3G20040  ATHXK4; Hexokinase [KO:K00844] [EC:2.7.1.1]\n",
      "AT4G24620  PGI1; phosphoglucose isomerase 1 [KO:K01810] [EC:5.3.1.9]\n",
      "AT5G42740  Sugar isomerase (SIS) family protein [KO:K01810] [EC:5.3.1.9]\n",
      "AT4G32840  PFK6; phosphofructokinase 6 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G56630  PFK7; phosphofructokinase 7 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G61580  PFK4; phosphofructokinase 4 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G29220  PFK1; phosphofructokinase 1 [KO:K00850] [EC:2.7.1.11]\n",
      "AT2G22480  PFK5; phosphofructokinase 5 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G26270  PFK3; phosphofructokinase 3 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G47810  PFK2; phosphofructokinase 2 [KO:K00850] [EC:2.7.1.11]\n",
      "AT1G76550  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT4G04040  MEE51; Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT1G12000  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT1G20950  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT1G43670  FBP; Inositol monophosphatase family protein [KO:K03841] [EC:3.1.3.11]\n",
      "AT3G54050  HCEF1; high cyclic electron flow 1 [KO:K03841] [EC:3.1.3.11]\n",
      "AT5G64380  Inositol monophosphatase family protein [KO:K03841] [EC:3.1.3.11]\n",
      "AT4G26520  FBA7; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G36460  FBA6; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT3G52930  FBA8; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G01140  PDE345; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT4G38970  FBA2; fructose-bisphosphate aldolase 2 [KO:K01623] [EC:4.1.2.13]\n",
      "AT4G26530  FBA5; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT5G03690  FBA4; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G21330  FBA1; fructose-bisphosphate aldolase 1 [KO:K01623] [EC:4.1.2.13]\n",
      "AT3G55440  TPI; triosephosphate isomerase [KO:K01803] [EC:5.3.1.1]\n",
      "AT2G21170  TIM; triosephosphate isomerase [KO:K01803] [EC:5.3.1.1]\n",
      "AT1G13440  GAPC2; glyceraldehyde-3-phosphate dehydrogenase C2 [KO:K00134] [EC:1.2.1.12]\n",
      "AT1G16300  GAPCP-2; glyceraldehyde-3-phosphate dehydrogenase of plastid 2 [KO:K00134] [EC:1.2.1.12]\n",
      "AT1G79530  GAPCP-1; glyceraldehyde-3-phosphate dehydrogenase of plastid 1 [KO:K00134] [EC:1.2.1.12]\n",
      "AT3G04120  GAPC1; glyceraldehyde-3-phosphate dehydrogenase C subunit 1 [KO:K00134] [EC:1.2.1.12]\n",
      "AT3G12780  PGK1; phosphoglycerate kinase 1 [KO:K00927] [EC:2.7.2.3]\n",
      "AT1G56190  Phosphoglycerate kinase family protein [KO:K00927] [EC:2.7.2.3]\n",
      "AT1G79550  PGK; phosphoglycerate kinase [KO:K00927] [EC:2.7.2.3]\n",
      "AT1G22170  Phosphoglycerate mutase family protein [KO:K01834] [EC:5.4.2.11]\n",
      "AT1G78050  PGM; phosphoglycerate/bisphosphoglycerate mutase [KO:K01834] [EC:5.4.2.11]\n",
      "AT3G50520  Phosphoglycerate mutase family protein [KO:K15634] [EC:5.4.2.11]\n",
      "AT1G09780  iPGAM1; Phosphoglycerate mutase, 2,3-bisphosphoglycerate-independent [KO:K15633] [EC:5.4.2.12]\n",
      "AT3G08590  iPGAM2; Phosphoglycerate mutase, 2,3-bisphosphoglycerate-independent [KO:K15633] [EC:5.4.2.12]\n",
      "AT1G74030  ENO1; enolase 1 [KO:K01689] [EC:4.2.1.11]\n",
      "AT2G36530  LOS2; Enolase [KO:K01689] [EC:4.2.1.11]\n",
      "AT2G29560  ENOC; cytosolic enolase [KO:K01689] [EC:4.2.1.11]\n",
      "AT3G04050  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT5G52920  PKP-BETA1; plastidic pyruvate kinase beta subunit 1 [KO:K00873] [EC:2.7.1.40]\n",
      "AT5G56350  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G25960  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G52990  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G55650  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G55810  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT5G08570  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT5G63680  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT4G26390  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT1G32440  PKp3; plastidial pyruvate kinase 3 [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G49160  pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G22960  PKP-ALPHA; Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT2G36580  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT4G15530  PPDK; pyruvate orthophosphate dikinase [KO:K01006] [EC:2.7.9.1]\n",
      "AT1G59900  E1_ALPHA; pyruvate dehydrogenase complex E1 alpha subunit [KO:K00161] [EC:1.2.4.1]\n",
      "AT1G01090  PDH-E1_ALPHA; pyruvate dehydrogenase E1 alpha [KO:K00161] [EC:1.2.4.1]\n",
      "AT1G24180  IAR4; Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00161] [EC:1.2.4.1]\n",
      "AT5G50850  MAB1; Transketolase family protein [KO:K00162] [EC:1.2.4.1]\n",
      "AT2G34590  Transketolase family protein [KO:K00162] [EC:1.2.4.1]\n",
      "AT1G30120  PDH-E1_BETA; pyruvate dehydrogenase E1 beta [KO:K00162] [EC:1.2.4.1]\n",
      "AT1G34430  EMB3003; 2-oxoacid dehydrogenases acyltransferase family protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G52200  LTA3; Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT1G54220  Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G13930  Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G25860  LTA2; 2-oxoacid dehydrogenases acyltransferase family protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G17260  Lactate/malate dehydrogenase family protein [KO:K00016] [EC:1.1.1.27]\n",
      "AT4G33070  Thiamine pyrophosphate dependent pyruvate decarboxylase family protein [KO:K01568] [EC:4.1.1.1]\n",
      "AT5G01320  Thiamine pyrophosphate dependent pyruvate decarboxylase family protein [KO:K01568] [EC:4.1.1.1]\n",
      "AT5G01330  PDC3; pyruvate decarboxylase-3 [KO:K01568] [EC:4.1.1.1]\n",
      "AT5G54960  PDC2; pyruvate decarboxylase-2 [KO:K01568] [EC:4.1.1.1]\n",
      "AT1G22440  Zinc-binding alcohol dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT5G42250  Zinc-binding alcohol dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT5G43940  HOT5; GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT4G22110  GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT1G22430  GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT1G77120  ADH1; alcohol dehydrogenase 1 [KO:K18857] [EC:1.1.1.1]\n",
      "AT1G32780  GroES-like zinc-binding dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT5G24760  GroES-like zinc-binding dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT1G64710  GroES-like zinc-binding alcohol dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT2G37790  AKR4C10; NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT5G01670  NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT5G36880  ACS; acetyl-CoA synthetase [KO:K01895] [EC:6.2.1.1]\n",
      "AT3G16910  AAE7; acyl-activating enzyme 7 [KO:K01913] [EC:6.2.1.1 6.2.1.2]\n",
      "AT3G17940  Galactose mutarotase-like superfamily protein [KO:K01785] [EC:5.1.3.3]\n",
      "AT3G47800  Galactose mutarotase-like superfamily protein [KO:K01785] [EC:5.1.3.3]\n",
      "AT5G15140  Galactose mutarotase-like superfamily protein [KO:K01785] [EC:5.1.3.3]\n",
      "AT3G01260  Galactose mutarotase-like superfamily protein [KO:K01785] [EC:5.1.3.3]\n",
      "AT5G51820  PGM; phosphoglucomutase [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G23190  PGM3; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G70730  PGM2; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT4G23730  Galactose mutarotase-like superfamily protein [KO:K01792] [EC:5.1.3.15]\n",
      "AT3G61610  Galactose mutarotase-like superfamily protein [KO:K01792] [EC:5.1.3.15]\n",
      "AT3G01590  Galactose mutarotase-like superfamily protein [KO:K01792] [EC:5.1.3.15]\n",
      "AT5G57330  Galactose mutarotase-like superfamily protein [KO:K01792] [EC:5.1.3.15]\n",
      "AT5G14500  aldose 1-epimerase family protein [KO:K01792] [EC:5.1.3.15]\n",
      "AT4G25900  Galactose mutarotase-like superfamily protein [KO:K01792] [EC:5.1.3.15]\n",
      "AT5G66530  Galactose mutarotase-like superfamily protein [KO:K01792] [EC:5.1.3.15]\n",
      "AT1G09870  histidine acid phosphatase family protein [KO:K03103] [EC:3.1.3.62 3.1.3.80]\n",
      "AT2G24270  ALDH11A3; aldehyde dehydrogenase 11A3 [KO:K00131] [EC:1.2.1.9]\n",
      "AT5G65690  PCK2; phosphoenolpyruvate carboxykinase 2 [KO:K01610] [EC:4.1.1.49]\n",
      "AT4G37870  PCK1; phosphoenolpyruvate carboxykinase 1 [KO:K01610] [EC:4.1.1.49]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00031  D-Glucose\n",
      "C00033  Acetate\n",
      "C00036  Oxaloacetate\n",
      "C00068  Thiamin diphosphate\n",
      "C00074  Phosphoenolpyruvate\n",
      "C00084  Acetaldehyde\n",
      "C00103  D-Glucose 1-phosphate\n",
      "C00111  Glycerone phosphate\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00186  (S)-Lactate\n",
      "C00197  3-Phospho-D-glycerate\n",
      "C00221  beta-D-Glucose\n",
      "C00236  3-Phospho-D-glyceroyl phosphate\n",
      "C00267  alpha-D-Glucose\n",
      "C00469  Ethanol\n",
      "C00631  2-Phospho-D-glycerate\n",
      "C00668  alpha-D-Glucose 6-phosphate\n",
      "C01159  2,3-Bisphospho-D-glycerate\n",
      "C01172  beta-D-Glucose 6-phosphate\n",
      "C01451  Salicin\n",
      "C05125  2-(alpha-Hydroxyethyl)thiamine diphosphate\n",
      "C05345  beta-D-Fructose 6-phosphate\n",
      "C05378  beta-D-Fructose 1,6-bisphosphate\n",
      "C06186  Arbutin\n",
      "C06187  Arbutin 6-phosphate\n",
      "C06188  Salicin 6-phosphate\n",
      "C15972  Enzyme N6-(lipoyl)lysine\n",
      "C15973  Enzyme N6-(dihydrolipoyl)lysine\n",
      "C16255  [Dihydrolipoyllysine-residue acetyltransferase] S-acetyldihydrolipoyllysine\n",
      "REFERENCE   (map 1)\n",
      "AUTHORS   Nishizuka Y (ed).\n",
      "TITLE     [Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1980)\n",
      "REFERENCE   (map 1)\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REFERENCE\n",
      "AUTHORS   Michal G.\n",
      "TITLE     Biochemical Pathways\n",
      "JOURNAL   Wiley (1999)\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00500  Starch and sucrose metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00640  Propanoate metabolism\n",
      "ath00710  Carbon fixation in photosynthetic organisms\n",
      "KO_PATHWAY  ko00010\n",
      "///\n",
      "ENTRY       ath00020                    Pathway\n",
      "NAME        Citrate cycle (TCA cycle) - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The citrate cycle (TCA cycle, Krebs cycle) is an important aerobic pathway for the final steps of the oxidation of carbohydrates and fatty acids. The cycle starts with acetyl-CoA, the activated form of acetate, derived from glycolysis and pyruvate oxidation for carbohydrates and from beta oxidation of fatty acids. The two-carbon acetyl group in acetyl-CoA is transferred to the four-carbon compound of oxaloacetate to form the six-carbon compound of citrate. In a series of reactions two carbons in citrate are oxidized to CO2 and the reaction pathway supplies NADH for use in the oxidative phosphorylation and other metabolic processes. The pathway also supplies important precursor metabolites including 2-oxoglutarate. At the end of the cycle the remaining four-carbon part is transformed back to oxaloacetate. According to the genome sequence data, many organisms seem to lack genes for the full cycle [MD:M00009], but contain genes for specific segments [MD:M00010 M00011].\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00020  Citrate cycle (TCA cycle)\n",
      "MODULE      ath_M00003  Gluconeogenesis, oxaloacetate => fructose-6P [PATH:ath00020]\n",
      "ath_M00010  Citrate cycle, first carbon oxidation, oxaloacetate => 2-oxoglutarate [PATH:ath00020]\n",
      "ath_M00307  Pyruvate oxidation, pyruvate => acetyl-CoA [PATH:ath00020]\n",
      "DBLINKS     GO: 0006099\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G42790  CSY3; citrate synthase 3 [KO:K01647] [EC:2.3.3.1]\n",
      "AT3G58740  CSY1; citrate synthase 1 [KO:K01647] [EC:2.3.3.1]\n",
      "AT3G58750  CSY2; citrate synthase 2 [KO:K01647] [EC:2.3.3.1]\n",
      "AT3G60100  CSY5; citrate synthase 5 [KO:K01647] [EC:2.3.3.1]\n",
      "AT2G44350  ATCS; Citrate synthase family protein [KO:K01647] [EC:2.3.3.1]\n",
      "AT3G06650  ACLB-1; ATP-citrate lyase B-1 [KO:K01648] [EC:2.3.3.8]\n",
      "AT5G49460  ACLB-2; ATP citrate lyase subunit B 2 [KO:K01648] [EC:2.3.3.8]\n",
      "AT1G60810  ACLA-2; ATP-citrate lyase A-2 [KO:K01648] [EC:2.3.3.8]\n",
      "AT1G10670  ACLA-1; ATP-citrate lyase A-1 [KO:K01648] [EC:2.3.3.8]\n",
      "AT1G09430  ACLA-3; ATP-citrate lyase A-3 [KO:K01648] [EC:2.3.3.8]\n",
      "AT4G35830  ACO1; aconitase 1 [KO:K01681] [EC:4.2.1.3]\n",
      "AT2G05710  ACO3; aconitase 3 [KO:K01681] [EC:4.2.1.3]\n",
      "AT4G26970  ACO2; aconitase 2 [KO:K01681] [EC:4.2.1.3]\n",
      "AT1G65930  cICDH; cytosolic NADP+-dependent isocitrate dehydrogenase [KO:K00031] [EC:1.1.1.42]\n",
      "AT1G54340  ICDH; isocitrate dehydrogenase [KO:K00031] [EC:1.1.1.42]\n",
      "AT5G14590  Isocitrate/isopropylmalate dehydrogenase family protein [KO:K00031] [EC:1.1.1.42]\n",
      "AT4G35650  IDH-III; isocitrate dehydrogenase III [KO:K00030] [EC:1.1.1.41]\n",
      "AT5G03290  IDH-V; isocitrate dehydrogenase V [KO:K00030] [EC:1.1.1.41]\n",
      "AT2G17130  IDH2; isocitrate dehydrogenase subunit 2 [KO:K00030] [EC:1.1.1.41]\n",
      "AT4G35260  IDH1; isocitrate dehydrogenase 1 [KO:K00030] [EC:1.1.1.41]\n",
      "AT3G09810  IDH-VI; isocitrate dehydrogenase VI [KO:K00030] [EC:1.1.1.41]\n",
      "AT1G32480  IDH-IV; isocitrate dehydrogenase IV [KO:K00030] [EC:1.1.1.41]\n",
      "AT3G55410  2-oxoglutarate dehydrogenase, E1 component [KO:K00164] [EC:1.2.4.2]\n",
      "AT5G65750  2-oxoglutarate dehydrogenase, E1 component [KO:K00164] [EC:1.2.4.2]\n",
      "AT5G55070  Dihydrolipoamide succinyltransferase [KO:K00658] [EC:2.3.1.61]\n",
      "AT4G26910  Dihydrolipoamide succinyltransferase [KO:K00658] [EC:2.3.1.61]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT5G08300  Succinyl-CoA ligase, alpha subunit [KO:K01899] [EC:6.2.1.4 6.2.1.5]\n",
      "AT5G23250  Succinyl-CoA ligase, alpha subunit [KO:K01899] [EC:6.2.1.4 6.2.1.5]\n",
      "AT2G20420  ATP citrate lyase (ACL) family protein [KO:K01900] [EC:6.2.1.4 6.2.1.5]\n",
      "AT5G66760  SDH1-1; succinate dehydrogenase 1-1 [KO:K00234] [EC:1.3.5.1]\n",
      "AT2G18450  SDH1-2; succinate dehydrogenase 1-2 [KO:K00234] [EC:1.3.5.1]\n",
      "AT5G40650  SDH2-2; succinate dehydrogenase 2-2 [KO:K00235] [EC:1.3.5.1]\n",
      "AT5G65165  SDH2-3; succinate dehydrogenase 2-3 [KO:K00235] [EC:1.3.5.1]\n",
      "AT3G27380  SDH2-1; succinate dehydrogenase 2-1 [KO:K00235] [EC:1.3.5.1]\n",
      "AT4G32210  SDH3-2; succinate dehydrogenase 3-2 [KO:K00236]\n",
      "AT5G09600  SDH3-1; succinate dehydrogenase 3-1 [KO:K00236]\n",
      "AT5G50950  FUM2; FUMARASE 2 [KO:K01679] [EC:4.2.1.2]\n",
      "AT2G47510  FUM1; fumarase 1 [KO:K01679] [EC:4.2.1.2]\n",
      "AT1G04410  c-NAD-MDH1; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G56720  c-NAD-MDH3; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G43330  c-NAD-MDH2; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT3G47520  MDH; malate dehydrogenase [KO:K00026] [EC:1.1.1.37]\n",
      "AT1G53240  mMDH1; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT2G22780  PMDH1; peroxisomal NAD-malate dehydrogenase 1 [KO:K00026] [EC:1.1.1.37]\n",
      "AT3G15020  mMDH2; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT5G09660  PMDH2; peroxisomal NAD-malate dehydrogenase 2 [KO:K00026] [EC:1.1.1.37]\n",
      "AT5G65690  PCK2; phosphoenolpyruvate carboxykinase 2 [KO:K01610] [EC:4.1.1.49]\n",
      "AT4G37870  PCK1; phosphoenolpyruvate carboxykinase 1 [KO:K01610] [EC:4.1.1.49]\n",
      "AT1G59900  E1_ALPHA; pyruvate dehydrogenase complex E1 alpha subunit [KO:K00161] [EC:1.2.4.1]\n",
      "AT1G01090  PDH-E1_ALPHA; pyruvate dehydrogenase E1 alpha [KO:K00161] [EC:1.2.4.1]\n",
      "AT1G24180  IAR4; Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00161] [EC:1.2.4.1]\n",
      "AT5G50850  MAB1; Transketolase family protein [KO:K00162] [EC:1.2.4.1]\n",
      "AT2G34590  Transketolase family protein [KO:K00162] [EC:1.2.4.1]\n",
      "AT1G30120  PDH-E1_BETA; pyruvate dehydrogenase E1 beta [KO:K00162] [EC:1.2.4.1]\n",
      "AT1G34430  EMB3003; 2-oxoacid dehydrogenases acyltransferase family protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G52200  LTA3; Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT1G54220  Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G13930  Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G25860  LTA2; 2-oxoacid dehydrogenases acyltransferase family protein [KO:K00627] [EC:2.3.1.12]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00026  2-Oxoglutarate\n",
      "C00036  Oxaloacetate\n",
      "C00042  Succinate\n",
      "C00068  Thiamin diphosphate\n",
      "C00074  Phosphoenolpyruvate\n",
      "C00091  Succinyl-CoA\n",
      "C00122  Fumarate\n",
      "C00149  (S)-Malate\n",
      "C00158  Citrate\n",
      "C00311  Isocitrate\n",
      "C00417  cis-Aconitate\n",
      "C05125  2-(alpha-Hydroxyethyl)thiamine diphosphate\n",
      "C05379  Oxalosuccinate\n",
      "C05381  3-Carboxy-1-hydroxypropyl-ThPP\n",
      "C15972  Enzyme N6-(lipoyl)lysine\n",
      "C15973  Enzyme N6-(dihydrolipoyl)lysine\n",
      "C16254  [Dihydrolipoyllysine-residue succinyltransferase] S-succinyldihydrolipoyllysine\n",
      "C16255  [Dihydrolipoyllysine-residue acetyltransferase] S-acetyldihydrolipoyllysine\n",
      "REFERENCE   (map 2)\n",
      "AUTHORS   Nishizuka Y (ed).\n",
      "TITLE     [Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1980)\n",
      "REFERENCE   (map 3)\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REFERENCE\n",
      "AUTHORS   Michal G.\n",
      "TITLE     Biochemical Pathways\n",
      "JOURNAL   Wiley (1999)\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00053  Ascorbate and aldarate metabolism\n",
      "ath00061  Fatty acid biosynthesis\n",
      "ath00062  Fatty acid elongation\n",
      "ath00071  Fatty acid degradation\n",
      "ath00190  Oxidative phosphorylation\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00280  Valine, leucine and isoleucine degradation\n",
      "ath00350  Tyrosine metabolism\n",
      "ath00470  D-Amino acid metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "KO_PATHWAY  ko00020\n",
      "///\n",
      "ENTRY       ath00030                    Pathway\n",
      "NAME        Pentose phosphate pathway - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The pentose phosphate pathway is a process of glucose turnover that produces NADPH as reducing equivalents and pentoses as essential parts of nucleotides. There are two different phases in the pathway. One is irreversible oxidative phase in which glucose-6P is converted to ribulose-5P by oxidative decarboxylation, and NADPH is generated [MD:M00006]. The other is reversible non-oxidative phase in which phosphorylated sugars are interconverted to generate xylulose-5P, ribulose-5P, and ribose-5P [MD:M00007]. Phosphoribosyl pyrophosphate (PRPP) formed from ribose-5P [MD:M00005] is an activated compound used in the biosynthesis of histidine and purine/pyrimidine nucleotides. This pathway map also shows the Entner-Doudoroff pathway where 6-P-gluconate is dehydrated and then cleaved into pyruvate and glyceraldehyde-3P [MD:M00008].\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00030  Pentose phosphate pathway\n",
      "MODULE      ath_M00004  Pentose phosphate pathway (Pentose phosphate cycle) [PATH:ath00030]\n",
      "ath_M00005  PRPP biosynthesis, ribose 5P => PRPP [PATH:ath00030]\n",
      "ath_M00006  Pentose phosphate pathway, oxidative phase, glucose 6P => ribulose 5P [PATH:ath00030]\n",
      "ath_M00007  Pentose phosphate pathway, non-oxidative phase, fructose 6P => ribose 5P [PATH:ath00030]\n",
      "DBLINKS     GO: 0006098\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G24620  PGI1; phosphoglucose isomerase 1 [KO:K01810] [EC:5.3.1.9]\n",
      "AT5G42740  Sugar isomerase (SIS) family protein [KO:K01810] [EC:5.3.1.9]\n",
      "AT3G27300  G6PD5; glucose-6-phosphate dehydrogenase 5 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT5G13110  G6PD2; glucose-6-phosphate dehydrogenase 2 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT1G09420  G6PD4; glucose-6-phosphate dehydrogenase 4 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT5G35790  G6PD1; glucose-6-phosphate dehydrogenase 1 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT1G24280  G6PD3; glucose-6-phosphate dehydrogenase 3 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT5G40760  G6PD6; glucose-6-phosphate dehydrogenase 6 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT5G24410  PGL4; 6-phosphogluconolactonase 4 [KO:K01057] [EC:3.1.1.31]\n",
      "AT5G24420  PGL5; 6-phosphogluconolactonase 5 [KO:K01057] [EC:3.1.1.31]\n",
      "AT3G49360  PGL2; 6-phosphogluconolactonase 2 [KO:K01057] [EC:3.1.1.31]\n",
      "AT1G13700  PGL1; 6-phosphogluconolactonase 1 [KO:K01057] [EC:3.1.1.31]\n",
      "AT5G24400  EMB2024; NagB/RpiA/CoA transferase-like superfamily protein [KO:K01057] [EC:3.1.1.31]\n",
      "AT1G64190  6-phosphogluconate dehydrogenase family protein [KO:K00033] [EC:1.1.1.44 1.1.1.343]\n",
      "AT5G41670  6-phosphogluconate dehydrogenase family protein [KO:K00033] [EC:1.1.1.44 1.1.1.343]\n",
      "AT3G02360  6-phosphogluconate dehydrogenase family protein [KO:K00033] [EC:1.1.1.44 1.1.1.343]\n",
      "AT1G63290  Aldolase-type TIM barrel family protein [KO:K01783] [EC:5.1.3.1]\n",
      "AT5G61410  RPE; D-ribulose-5-phosphate-3-epimerase [KO:K01783] [EC:5.1.3.1]\n",
      "AT3G01850  Aldolase-type TIM barrel family protein [KO:K01783] [EC:5.1.3.1]\n",
      "AT3G60750  Transketolase [KO:K00615] [EC:2.2.1.1]\n",
      "AT2G45290  Transketolase [KO:K00615] [EC:2.2.1.1]\n",
      "AT5G13420  TRA2; Aldolase-type TIM barrel family protein [KO:K00616] [EC:2.2.1.2]\n",
      "AT1G12230  Aldolase superfamily protein [KO:K00616] [EC:2.2.1.2]\n",
      "AT1G71100  RSW10; Ribose 5-phosphate isomerase, type A protein [KO:K01807] [EC:5.3.1.6]\n",
      "AT2G01290  RPI2; ribose-5-phosphate isomerase 2 [KO:K01807] [EC:5.3.1.6]\n",
      "AT3G04790  EMB3119; Ribose 5-phosphate isomerase, type A protein [KO:K01807] [EC:5.3.1.6]\n",
      "AT5G44520  NagB/RpiA/CoA transferase-like superfamily protein [KO:K01807] [EC:5.3.1.6]\n",
      "AT1G17160  pfkB-like carbohydrate kinase family protein [KO:K00852] [EC:2.7.1.15]\n",
      "AT5G51820  PGM; phosphoglucomutase [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G23190  PGM3; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G70730  PGM2; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT2G44530  Phosphoribosyltransferase family protein [KO:K00948] [EC:2.7.6.1]\n",
      "AT1G32380  PRS2; phosphoribosyl pyrophosphate (PRPP) synthase 2 [KO:K00948] [EC:2.7.6.1]\n",
      "AT2G35390  Phosphoribosyltransferase family protein [KO:K00948] [EC:2.7.6.1]\n",
      "AT2G24270  ALDH11A3; aldehyde dehydrogenase 11A3 [KO:K00131] [EC:1.2.1.9]\n",
      "AT2G16790  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K00851] [EC:2.7.1.12]\n",
      "AT4G26520  FBA7; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G36460  FBA6; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT3G52930  FBA8; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G01140  PDE345; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT4G38970  FBA2; fructose-bisphosphate aldolase 2 [KO:K01623] [EC:4.1.2.13]\n",
      "AT4G26530  FBA5; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT5G03690  FBA4; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G21330  FBA1; fructose-bisphosphate aldolase 1 [KO:K01623] [EC:4.1.2.13]\n",
      "AT1G43670  FBP; Inositol monophosphatase family protein [KO:K03841] [EC:3.1.3.11]\n",
      "AT3G54050  HCEF1; high cyclic electron flow 1 [KO:K03841] [EC:3.1.3.11]\n",
      "AT5G64380  Inositol monophosphatase family protein [KO:K03841] [EC:3.1.3.11]\n",
      "AT4G32840  PFK6; phosphofructokinase 6 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G56630  PFK7; phosphofructokinase 7 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G61580  PFK4; phosphofructokinase 4 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G29220  PFK1; phosphofructokinase 1 [KO:K00850] [EC:2.7.1.11]\n",
      "AT2G22480  PFK5; phosphofructokinase 5 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G26270  PFK3; phosphofructokinase 3 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G47810  PFK2; phosphofructokinase 2 [KO:K00850] [EC:2.7.1.11]\n",
      "AT1G76550  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT4G04040  MEE51; Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT1G12000  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT1G20950  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00031  D-Glucose\n",
      "C00117  D-Ribose 5-phosphate\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00119  5-Phospho-alpha-D-ribose 1-diphosphate\n",
      "C00121  D-Ribose\n",
      "C00197  3-Phospho-D-glycerate\n",
      "C00198  D-Glucono-1,5-lactone\n",
      "C00199  D-Ribulose 5-phosphate\n",
      "C00204  2-Dehydro-3-deoxy-D-gluconate\n",
      "C00221  beta-D-Glucose\n",
      "C00231  D-Xylulose 5-phosphate\n",
      "C00257  D-Gluconic acid\n",
      "C00258  D-Glycerate\n",
      "C00279  D-Erythrose 4-phosphate\n",
      "C00345  6-Phospho-D-gluconate\n",
      "C00577  D-Glyceraldehyde\n",
      "C00620  alpha-D-Ribose 1-phosphate\n",
      "C00631  2-Phospho-D-glycerate\n",
      "C00668  alpha-D-Glucose 6-phosphate\n",
      "C00672  2-Deoxy-D-ribose 1-phosphate\n",
      "C00673  2-Deoxy-D-ribose 5-phosphate\n",
      "C01151  D-Ribose 1,5-bisphosphate\n",
      "C01172  beta-D-Glucose 6-phosphate\n",
      "C01218  6-Phospho-2-dehydro-D-gluconate\n",
      "C01236  D-Glucono-1,5-lactone 6-phosphate\n",
      "C01801  Deoxyribose\n",
      "C03752  2-Amino-2-deoxy-D-gluconate\n",
      "C04442  2-Dehydro-3-deoxy-6-phospho-D-gluconate\n",
      "C05345  beta-D-Fructose 6-phosphate\n",
      "C05378  beta-D-Fructose 1,6-bisphosphate\n",
      "C05382  Sedoheptulose 7-phosphate\n",
      "C06019  D-arabino-Hex-3-ulose 6-phosphate\n",
      "C06473  2-Keto-D-gluconic acid\n",
      "C20589  D-Glucosaminate-6-phosphate\n",
      "REFERENCE   (map 3)\n",
      "AUTHORS   Nishizuka Y (ed).\n",
      "TITLE     [Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1980)\n",
      "REFERENCE   (map 4)\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REFERENCE\n",
      "AUTHORS   Michal G.\n",
      "TITLE     Biochemical Pathways\n",
      "JOURNAL   Wiley (1999)\n",
      "REFERENCE   PMID:12700258\n",
      "AUTHORS   Hove-Jensen B, Rosenkrantz TJ, Haldimann A, Wanner BL.\n",
      "TITLE     Escherichia coli phnN, encoding ribose 1,5-bisphosphokinase activity (phosphoribosyl diphosphate forming): dual role in phosphonate degradation and NAD biosynthesis pathways.\n",
      "JOURNAL   J Bacteriol 185:2793-801 (2003)\n",
      "DOI:10.1128/JB.185.9.2793-2801.2003\n",
      "REFERENCE   PMID:16788179\n",
      "AUTHORS   Orita I, Sato T, Yurimoto H, Kato N, Atomi H, Imanaka T, Sakai Y\n",
      "TITLE     The ribulose monophosphate pathway substitutes for the missing pentose phosphate pathway in the archaeon Thermococcus kodakaraensis.\n",
      "JOURNAL   J Bacteriol 188:4698-704 (2006)\n",
      "DOI:10.1128/JB.00492-06\n",
      "REFERENCE   PMID:16428816\n",
      "AUTHORS   Kato N, Yurimoto H, Thauer RK\n",
      "TITLE     The physiological role of the ribulose monophosphate pathway in bacteria and archaea.\n",
      "JOURNAL   Biosci Biotechnol Biochem 70:10-21 (2006)\n",
      "DOI:10.1271/bbb.70.10\n",
      "REFERENCE   PMID:23279921\n",
      "AUTHORS   Kouril T, Wieloch P, Reimann J, Wagner M, Zaparty M, Albers SV, Schomburg D, Ruoff P, Siebers B\n",
      "TITLE     Unraveling the function of the two Entner-Doudoroff branches in the thermoacidophilic Crenarchaeon Sulfolobus solfataricus P2.\n",
      "JOURNAL   FEBS J 280:1126-38 (2013)\n",
      "DOI:10.1111/febs.12106\n",
      "REFERENCE   PMID:16458304\n",
      "AUTHORS   Reher M, Schonheit P\n",
      "TITLE     Glyceraldehyde dehydrogenases from the thermoacidophilic euryarchaeota Picrophilus torridus and Thermoplasma acidophilum, key enzymes of the non-phosphorylative Entner-Doudoroff pathway, constitute a novel enzyme family within the aldehyde dehydrogenase superfamily.\n",
      "JOURNAL   FEBS Lett 580:1198-204 (2006)\n",
      "DOI:10.1016/j.febslet.2006.01.029\n",
      "REFERENCE   PMID:20023024\n",
      "AUTHORS   Reher M, Fuhrer T, Bott M, Schonheit P\n",
      "TITLE     The nonphosphorylative Entner-Doudoroff pathway in the thermoacidophilic euryarchaeon Picrophilus torridus involves a novel 2-keto-3-deoxygluconate- specific aldolase.\n",
      "JOURNAL   J Bacteriol 192:964-74 (2010)\n",
      "DOI:10.1128/JB.01281-09\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00040  Pentose and glucuronate interconversions\n",
      "ath00052  Galactose metabolism\n",
      "ath00230  Purine metabolism\n",
      "ath00240  Pyrimidine metabolism\n",
      "ath00340  Histidine metabolism\n",
      "ath00750  Vitamin B6 metabolism\n",
      "KO_PATHWAY  ko00030\n",
      "///\n",
      "ENTRY       ath00040                    Pathway\n",
      "NAME        Pentose and glucuronate interconversions - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00040  Pentose and glucuronate interconversions\n",
      "MODULE      ath_M00081  Pectin degradation [PATH:ath00040]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G43270  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT1G53830  PME2; pectin methylesterase 2 [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G04970  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT2G45220  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G19730  Pectin lyase-like superfamily protein [KO:K01051] [EC:3.1.1.11]\n",
      "AT1G11590  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G49180  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT2G47030  VGDH1; Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT1G69940  PPME1; Pectin lyase-like superfamily protein [KO:K01051] [EC:3.1.1.11]\n",
      "AT4G33220  PME44; pectin methylesterase 44 [KO:K01051] [EC:3.1.1.11]\n",
      "AT3G29090  PME31; pectin methylesterase 31 [KO:K01051] [EC:3.1.1.11]\n",
      "AT4G03930  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT2G21610  PE11; pectinesterase 11 [KO:K01051] [EC:3.1.1.11]\n",
      "AT1G11580  PMEPCRA; methylesterase PCR A [KO:K01051] [EC:3.1.1.11]\n",
      "AT3G05620  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G27870  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT3G14310  PME3; pectin methylesterase 3 [KO:K01051] [EC:3.1.1.11]\n",
      "AT4G02320  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT3G47400  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G51500  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G51490  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT1G02810  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT3G10720  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT2G47550  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT4G02330  ATPMEPCRB; Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT3G05610  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G55590  QRT1; Pectin lyase-like superfamily protein [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G07410  Pectin lyase-like superfamily protein [KO:K01051] [EC:3.1.1.11]\n",
      "AT3G60730  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT3G06830  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT4G02300  Plant invertase/pectin methylesterase inhibitor superfamily [KO:K01051] [EC:3.1.1.11]\n",
      "AT4G00190  PME38; pectin methylesterase 38 [KO:K01051] [EC:3.1.1.11]\n",
      "AT5G61680  Pectin lyase-like superfamily protein [KO:K01051] [EC:3.1.1.11]\n",
      "AT4G13710  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G55140  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G09540  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G27400  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT5G55720  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT5G09280  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G24670  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT1G30350  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT2G02720  Pectate lyase family protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G01270  Pectate lyase family protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G53190  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT1G14420  AT59; Pectate lyase family protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT5G04310  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT5G15110  Pectate lyase family protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT1G11920  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT5G63180  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT4G22090  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT5G48900  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT4G22080  RHS14; root hair specific 14 [KO:K01728] [EC:4.2.2.2]\n",
      "AT1G67750  Pectate lyase family protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT4G24780  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G07010  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT4G13210  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G24230  Pectate lyase family protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT1G04680  Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G54920  PMR6; Pectin lyase-like superfamily protein [KO:K01728] [EC:4.2.2.2]\n",
      "AT3G59850  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT5G14650  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT3G15720  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT5G17200  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT3G57510  ADPG1; Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT1G05650  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT2G43880  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT2G43870  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT2G43860  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT2G43890  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT1G05660  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT4G32370  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT4G32375  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT4G32380  pectin lyase superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT4G35670  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT5G27530  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT5G39910  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT5G44830  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT5G44840  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT2G41850  PGAZAT; polygalacturonase ADPG2-like protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT1G65570  Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT3G07970  QRT2; Pectin lyase-like superfamily protein [KO:K01184] [EC:3.2.1.15]\n",
      "AT3G07850  Pectin lyase-like superfamily protein [KO:K01213] [EC:3.2.1.67]\n",
      "AT1G02790  PGA4; polygalacturonase 4 [KO:K01213] [EC:3.2.1.67]\n",
      "AT3G07820  Pectin lyase-like superfamily protein [KO:K01213] [EC:3.2.1.67]\n",
      "AT3G14040  Pectin lyase-like superfamily protein [KO:K01213] [EC:3.2.1.67]\n",
      "AT4G18180  Pectin lyase-like superfamily protein [KO:K01213] [EC:3.2.1.67]\n",
      "AT3G07830  Pectin lyase-like superfamily protein [KO:K01213] [EC:3.2.1.67]\n",
      "AT5G48140  Pectin lyase-like superfamily protein [KO:K01213] [EC:3.2.1.67]\n",
      "AT5G14470  GHMP kinase family protein [KO:K16190] [EC:2.7.1.43]\n",
      "AT3G01640  GLCAK; glucuronokinase G [KO:K16190] [EC:2.7.1.43]\n",
      "AT5G52560  USP; UDP-sugar pyrophosphorylase [KO:K12447] [EC:2.7.7.64]\n",
      "AT5G39320  UDG4; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT1G26570  UGD1; UDP-glucose dehydrogenase 1 [KO:K00012] [EC:1.1.1.22]\n",
      "AT5G15490  UGD3; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT3G29360  UGD2; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT3G01010  UDP-glucose/GDP-mannose dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT5G17310  UGP2; UDP-glucose pyrophosphorylase 2 [KO:K00963] [EC:2.7.7.9]\n",
      "AT3G03250  UGP1; UDP-GLUCOSE PYROPHOSPHORYLASE 1 [KO:K00963] [EC:2.7.7.9]\n",
      "AT2G37790  AKR4C10; NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT5G01670  NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT1G63290  Aldolase-type TIM barrel family protein [KO:K01783] [EC:5.1.3.1]\n",
      "AT5G61410  RPE; D-ribulose-5-phosphate-3-epimerase [KO:K01783] [EC:5.1.3.1]\n",
      "AT3G01850  Aldolase-type TIM barrel family protein [KO:K01783] [EC:5.1.3.1]\n",
      "AT5G49650  XK-2; xylulose kinase-2 [KO:K00854] [EC:2.7.1.17]\n",
      "AT5G57655  xylose isomerase family protein [KO:K01805] [EC:5.3.1.5]\n",
      "AT5G51970  GroES-like zinc-binding alcohol dehydrogenase family protein [KO:K00008] [EC:1.1.1.14]\n",
      "AT2G21370  XK-1; xylulose kinase-1 [KO:K22935] [EC:2.7.1.47]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00026  2-Oxoglutarate\n",
      "C00029  UDP-glucose\n",
      "C00103  D-Glucose 1-phosphate\n",
      "C00111  Glycerone phosphate\n",
      "C00116  Glycerol\n",
      "C00167  UDP-glucuronate\n",
      "C00181  D-Xylose\n",
      "C00191  D-Glucuronate\n",
      "C00199  D-Ribulose 5-phosphate\n",
      "C00204  2-Dehydro-3-deoxy-D-gluconate\n",
      "C00216  D-Arabinose\n",
      "C00231  D-Xylulose 5-phosphate\n",
      "C00259  L-Arabinose\n",
      "C00266  Glycolaldehyde\n",
      "C00309  D-Ribulose\n",
      "C00310  D-Xylulose\n",
      "C00312  L-Xylulose\n",
      "C00333  D-Galacturonate\n",
      "C00379  Xylitol\n",
      "C00433  2,5-Dioxopentanoate\n",
      "C00470  Pectate\n",
      "C00474  Ribitol\n",
      "C00476  D-Lyxose\n",
      "C00502  D-Xylonate\n",
      "C00508  L-Ribulose\n",
      "C00514  D-Mannonate\n",
      "C00532  L-Arabitol\n",
      "C00558  D-Tagaturonate\n",
      "C00618  3-Dehydro-L-gulonate\n",
      "C00714  Pectin\n",
      "C00789  CDP-ribitol\n",
      "C00800  L-Gulonate\n",
      "C00817  D-Altronate\n",
      "C00905  D-Fructuronate\n",
      "C01068  D-Ribitol 5-phosphate\n",
      "C01101  L-Ribulose 5-phosphate\n",
      "C01508  L-Lyxose\n",
      "C01904  D-Arabitol\n",
      "C02266  D-Xylonolactone\n",
      "C02273  Digalacturonate\n",
      "C02426  L-Glyceraldehyde\n",
      "C02753  D-Xylono-1,4-lactone\n",
      "C03033  beta-D-Glucuronoside\n",
      "C03291  L-Xylulose 5-phosphate\n",
      "C03826  2-Dehydro-3-deoxy-D-xylonate\n",
      "C04053  5-Dehydro-4-deoxy-D-glucuronate\n",
      "C04349  (4S)-4,6-Dihydroxy-2,5-dioxohexanoate\n",
      "C04575  (4R,5S)-4,5,6-Trihydroxy-2,3-dioxohexanoate\n",
      "C05385  D-Glucuronate 1-phosphate\n",
      "C05411  L-Xylonate\n",
      "C05412  L-Lyxonate\n",
      "C06118  4-(4-Deoxy-alpha-D-gluc-4-enuronosyl)-D-galacturonate\n",
      "C06441  L-Xylulose 1-phosphate\n",
      "C14899  3-Dehydro-L-gulonate 6-phosphate\n",
      "C15930  L-Galactonate\n",
      "C20680  2-Dehydro-3-deoxy-L-galactonate\n",
      "C22337  D-Ribulose 1-phosphate\n",
      "REFERENCE   PMID:11741871\n",
      "AUTHORS   Yew WS, Gerlt JA.\n",
      "TITLE     Utilization of L-ascorbate by Escherichia coli K-12: assignments of functions to products of the yjf-sga and yia-sgb operons.\n",
      "JOURNAL   J Bacteriol 184:302-6 (2002)\n",
      "DOI:10.1128/JB.184.1.302-306.2002\n",
      "REFERENCE   PMID:15697206\n",
      "AUTHORS   Yew WS, Akana J, Wise EL, Rayment I, Gerlt JA\n",
      "TITLE     Evolution of enzymatic activities in the orotidine 5'-monophosphate decarboxylase suprafamily: enhancing the promiscuous D-arabino-hex-3-ulose 6-phosphate synthase reaction catalyzed by 3-keto-L-gulonate 6-phosphate decarboxylase.\n",
      "JOURNAL   Biochemistry 44:1807-15 (2005)\n",
      "DOI:10.1021/bi047815v\n",
      "REFERENCE   PMID:10572115\n",
      "AUTHORS   Yasueda H, Kawahara Y, Sugimoto S.\n",
      "TITLE     Bacillus subtilis yckG and yckF encode two key enzymes of the ribulose monophosphate pathway used by methylotrophs, and yckH is required for their expression.\n",
      "JOURNAL   J Bacteriol 181:7154-60 (1999)\n",
      "DOI:10.1128/JB.181.23.7154-7160.1999\n",
      "REFERENCE   PMID:10913097\n",
      "AUTHORS   Ibanez E, Gimenez R, Pedraza T, Baldoma L, Aguilar J, Badia J\n",
      "TITLE     Role of the yiaR and yiaS genes of Escherichia coli in metabolism of endogenously formed L-xylulose.\n",
      "JOURNAL   J Bacteriol 182:4625-7 (2000)\n",
      "DOI:10.1128/JB.182.16.4625-4627.2000\n",
      "REFERENCE   PMID:15901685\n",
      "AUTHORS   Orita I, Yurimoto H, Hirai R, Kawarabayasi Y, Sakai Y, Kato N\n",
      "TITLE     The archaeon Pyrococcus horikoshii possesses a bifunctional enzyme for formaldehyde fixation via the ribulose monophosphate pathway.\n",
      "JOURNAL   J Bacteriol 187:3636-42 (2005)\n",
      "DOI:10.1128/JB.187.11.3636-3642.2005\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00052  Galactose metabolism\n",
      "ath00053  Ascorbate and aldarate metabolism\n",
      "ath00500  Starch and sucrose metabolism\n",
      "ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "ath00561  Glycerolipid metabolism\n",
      "ath00562  Inositol phosphate metabolism\n",
      "ath00740  Riboflavin metabolism\n",
      "KO_PATHWAY  ko00040\n",
      "///\n",
      "ENTRY       ath00051                    Pathway\n",
      "NAME        Fructose and mannose metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00051  Fructose and mannose metabolism\n",
      "MODULE      ath_M00114  Ascorbate biosynthesis, plants, fructose-6P => ascorbate [PATH:ath00051]\n",
      "DBLINKS     GO: 0006000 0006013\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G10900  Glycosyl hydrolase superfamily protein [KO:K19355] [EC:3.2.1.78]\n",
      "AT2G20680  MAN2; Glycosyl hydrolase superfamily protein [KO:K19355] [EC:3.2.1.78]\n",
      "AT5G01930  MAN6; Glycosyl hydrolase superfamily protein [KO:K19355] [EC:3.2.1.78]\n",
      "AT5G66460  MAN7; Glycosyl hydrolase superfamily protein [KO:K19355] [EC:3.2.1.78]\n",
      "AT1G02310  MAN1; Glycosyl hydrolase superfamily protein [KO:K19355] [EC:3.2.1.78]\n",
      "AT4G28320  MAN5; Glycosyl hydrolase superfamily protein [KO:K19355] [EC:3.2.1.78]\n",
      "AT3G30540  Glycosyl hydrolase superfamily protein [KO:K19355] [EC:3.2.1.78]\n",
      "AT3G10890  Glycosyl hydrolase superfamily protein [KO:K19355] [EC:3.2.1.78]\n",
      "AT5G51830  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT2G31390  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT4G10260  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G06030  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT3G59480  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G66430  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G06020  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G50390  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT3G02570  MEE31; Mannose-6-phosphate isomerase, type I [KO:K01809] [EC:5.3.1.8]\n",
      "AT1G67070  DIN9; Mannose-6-phosphate isomerase, type I [KO:K01809] [EC:5.3.1.8]\n",
      "AT2G45790  PMM; phosphomannomutase [KO:K17497] [EC:5.4.2.8]\n",
      "AT1G74910  ADP-glucose pyrophosphorylase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT2G39770  CYT1; Glucose-1-phosphate adenylyltransferase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT3G55590  Glucose-1-phosphate adenylyltransferase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT4G30570  Glucose-1-phosphate adenylyltransferase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT2G04650  ADP-glucose pyrophosphorylase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT5G66280  GMD1; GDP-D-mannose 4,6-dehydratase 1 [KO:K01711] [EC:4.2.1.47]\n",
      "AT3G51160  MUR1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K01711] [EC:4.2.1.47]\n",
      "AT1G17890  GER2; NAD(P)-binding Rossmann-fold superfamily protein [KO:K02377] [EC:1.1.1.271]\n",
      "AT1G73250  GER1; GDP-4-keto-6-deoxymannose-3, 5-epimerase-4-reductase 1 [KO:K02377] [EC:1.1.1.271]\n",
      "AT4G11980  NUDX14; nudix hydrolase homolog 14 [KO:K18447] [EC:3.6.1.21]\n",
      "AT4G37840  HKL3; hexokinase-like 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G47840  HXK3; hexokinase 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT2G19860  HXK2; hexokinase 2 [KO:K00844] [EC:2.7.1.1]\n",
      "AT4G29130  HXK1; hexokinase 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G50460  HKL1; hexokinase-like 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT3G20040  ATHXK4; Hexokinase [KO:K00844] [EC:2.7.1.1]\n",
      "AT4G32840  PFK6; phosphofructokinase 6 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G56630  PFK7; phosphofructokinase 7 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G61580  PFK4; phosphofructokinase 4 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G29220  PFK1; phosphofructokinase 1 [KO:K00850] [EC:2.7.1.11]\n",
      "AT2G22480  PFK5; phosphofructokinase 5 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G26270  PFK3; phosphofructokinase 3 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G47810  PFK2; phosphofructokinase 2 [KO:K00850] [EC:2.7.1.11]\n",
      "AT1G76550  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT4G04040  MEE51; Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT1G12000  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT1G20950  Phosphofructokinase family protein [KO:K00895] [EC:2.7.1.90]\n",
      "AT1G43670  FBP; Inositol monophosphatase family protein [KO:K03841] [EC:3.1.3.11]\n",
      "AT3G54050  HCEF1; high cyclic electron flow 1 [KO:K03841] [EC:3.1.3.11]\n",
      "AT5G64380  Inositol monophosphatase family protein [KO:K03841] [EC:3.1.3.11]\n",
      "AT1G07110  F2KP; fructose-2,6-bisphosphatase [KO:K01103] [EC:2.7.1.105 3.1.3.46]\n",
      "AT5G51970  GroES-like zinc-binding alcohol dehydrogenase family protein [KO:K00008] [EC:1.1.1.14]\n",
      "AT5G57655  xylose isomerase family protein [KO:K01805] [EC:5.3.1.5]\n",
      "AT4G26520  FBA7; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G36460  FBA6; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT3G52930  FBA8; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G01140  PDE345; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT4G38970  FBA2; fructose-bisphosphate aldolase 2 [KO:K01623] [EC:4.1.2.13]\n",
      "AT4G26530  FBA5; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT5G03690  FBA4; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G21330  FBA1; fructose-bisphosphate aldolase 1 [KO:K01623] [EC:4.1.2.13]\n",
      "AT3G55440  TPI; triosephosphate isomerase [KO:K01803] [EC:5.3.1.1]\n",
      "AT2G21170  TIM; triosephosphate isomerase [KO:K01803] [EC:5.3.1.1]\n",
      "AT1G48430  Dihydroxyacetone kinase [KO:K00863] [EC:2.7.1.28 2.7.1.29 4.6.1.15]\n",
      "COMPOUND    C00095  D-Fructose\n",
      "C00096  GDP-mannose\n",
      "C00111  Glycerone phosphate\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00159  D-Mannose\n",
      "C00186  (S)-Lactate\n",
      "C00247  L-Sorbose\n",
      "C00267  alpha-D-Glucose\n",
      "C00275  D-Mannose 6-phosphate\n",
      "C00325  GDP-L-fucose\n",
      "C00392  Mannitol\n",
      "C00424  (S)-Lactaldehyde\n",
      "C00464  Mannan\n",
      "C00507  L-Rhamnose\n",
      "C00577  D-Glyceraldehyde\n",
      "C00636  D-Mannose 1-phosphate\n",
      "C00644  D-Mannitol 1-phosphate\n",
      "C00665  beta-D-Fructose 2,6-bisphosphate\n",
      "C00794  D-Sorbitol\n",
      "C00861  L-Rhamnulose\n",
      "C00976  GDP-D-mannuronate\n",
      "C01019  6-Deoxy-L-galactose\n",
      "C01094  D-Fructose 1-phosphate\n",
      "C01096  Sorbitol 6-phosphate\n",
      "C01099  L-Fuculose 1-phosphate\n",
      "C01131  L-Rhamnulose 1-phosphate\n",
      "C01222  GDP-4-dehydro-6-deoxy-D-mannose\n",
      "C01355  Fructan\n",
      "C01487  D-Allose\n",
      "C01720  L-Fuconate\n",
      "C01721  L-Fuculose\n",
      "C01768  (Alginate)n\n",
      "C01934  L-Rhamnonate\n",
      "C02431  L-Rhamnofuranose\n",
      "C02492  1,4-beta-D-Mannan\n",
      "C02888  Sorbose 1-phosphate\n",
      "C02962  D-Allose 6-phosphate\n",
      "C02977  GDP-6-deoxy-D-talose\n",
      "C02985  L-Fucose 1-phosphate\n",
      "C02991  L-Rhamnono-1,4-lactone\n",
      "C03117  GDP-6-deoxy-D-mannose\n",
      "C03267  beta-D-Fructose 2-phosphate\n",
      "C03827  2-Dehydro-3-deoxy-L-fuconate\n",
      "C03979  2-Dehydro-3-deoxy-L-rhamnonate\n",
      "C05144  (beta-D-Mannuronate)n\n",
      "C05345  beta-D-Fructose 6-phosphate\n",
      "C05378  beta-D-Fructose 1,6-bisphosphate\n",
      "C05392  Oligouronide with 4-deoxy-alpha-L-erythro-hex-4-enopyranuronosyl group\n",
      "C06192  ADP-mannose\n",
      "C11516  2-(alpha-D-Mannosyl)-3-phosphoglycerate\n",
      "C11544  2-O-(alpha-D-Mannosyl)-D-glycerate\n",
      "C18028  L-Fucono-1,5-lactone\n",
      "C18096  D-Allulose 6-phosphate\n",
      "C20781  2,4-Diketo-3-deoxy-L-fuconate\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00052  Galactose metabolism\n",
      "ath00053  Ascorbate and aldarate metabolism\n",
      "ath00510  N-Glycan biosynthesis\n",
      "ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "KO_PATHWAY  ko00051\n",
      "///\n",
      "ENTRY       ath00052                    Pathway\n",
      "NAME        Galactose metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00052  Galactose metabolism\n",
      "DBLINKS     GO: 0006012\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G17940  Galactose mutarotase-like superfamily protein [KO:K01785] [EC:5.1.3.3]\n",
      "AT3G47800  Galactose mutarotase-like superfamily protein [KO:K01785] [EC:5.1.3.3]\n",
      "AT5G15140  Galactose mutarotase-like superfamily protein [KO:K01785] [EC:5.1.3.3]\n",
      "AT3G01260  Galactose mutarotase-like superfamily protein [KO:K01785] [EC:5.1.3.3]\n",
      "AT5G18200  putative galactose-1-phosphate uridyltransferase [KO:K00965] [EC:2.7.7.12]\n",
      "AT1G63180  UGE3; UDP-D-glucose/UDP-D-galactose 4-epimerase 3 [KO:K01784] [EC:5.1.3.2]\n",
      "AT1G64440  RHD1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K01784] [EC:5.1.3.2]\n",
      "AT1G12780  UGE1; UDP-D-glucose/UDP-D-galactose 4-epimerase 1 [KO:K01784] [EC:5.1.3.2]\n",
      "AT4G23920  UGE2; UDP-D-glucose/UDP-D-galactose 4-epimerase 2 [KO:K01784] [EC:5.1.3.2]\n",
      "AT4G10960  UGE5; UDP-D-glucose/UDP-D-galactose 4-epimerase 5 [KO:K01784] [EC:5.1.3.2]\n",
      "AT5G17310  UGP2; UDP-glucose pyrophosphorylase 2 [KO:K00963] [EC:2.7.7.9]\n",
      "AT3G03250  UGP1; UDP-GLUCOSE PYROPHOSPHORYLASE 1 [KO:K00963] [EC:2.7.7.9]\n",
      "AT5G51820  PGM; phosphoglucomutase [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G23190  PGM3; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G70730  PGM2; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT4G37840  HKL3; hexokinase-like 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G47840  HXK3; hexokinase 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT2G19860  HXK2; hexokinase 2 [KO:K00844] [EC:2.7.1.1]\n",
      "AT4G29130  HXK1; hexokinase 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G50460  HKL1; hexokinase-like 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT3G20040  ATHXK4; Hexokinase [KO:K00844] [EC:2.7.1.1]\n",
      "AT3G54440  glycoside hydrolase family 2 protein [KO:K01190] [EC:3.2.1.23]\n",
      "AT1G72990  BGAL17; beta-galactosidase 17 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G52840  BGAL2; beta-galactosidase 2 [KO:K12309] [EC:3.2.1.23]\n",
      "AT5G52560  USP; UDP-sugar pyrophosphorylase [KO:K12447] [EC:2.7.7.64]\n",
      "AT2G47180  GolS1; galactinol synthase 1 [KO:K18819] [EC:2.4.1.123]\n",
      "AT1G56600  GolS2; galactinol synthase 2 [KO:K18819] [EC:2.4.1.123]\n",
      "AT1G09350  GolS3; galactinol synthase 3 [KO:K18819] [EC:2.4.1.123]\n",
      "AT1G60470  GolS4; galactinol synthase 4 [KO:K18819] [EC:2.4.1.123]\n",
      "AT5G23790  GolS5; galactinol synthase 5 [KO:K18819] [EC:2.4.1.123]\n",
      "AT4G26250  GolS6; galactinol synthase 6 [KO:K18819] [EC:2.4.1.123]\n",
      "AT1G60450  GolS7; galactinol synthase 7 [KO:K18819] [EC:2.4.1.123]\n",
      "AT5G30500  GolS10; Nucleotide-diphospho-sugar transferases superfamily protein [KO:K18819] [EC:2.4.1.123]\n",
      "AT1G55740  SIP1; seed imbibition 1 [KO:K06617] [EC:2.4.1.82]\n",
      "AT3G57520  SIP2; seed imbibition 2 [KO:K06617] [EC:2.4.1.82]\n",
      "AT5G20250  DIN10; Raffinose synthase family protein [KO:K06617] [EC:2.4.1.82]\n",
      "AT5G40390  SIP1; Raffinose synthase family protein [KO:K06617] [EC:2.4.1.82]\n",
      "AT3G56310  Melibiase family protein [KO:K07407] [EC:3.2.1.22]\n",
      "AT5G08370  AGAL2; alpha-galactosidase 2 [KO:K07407] [EC:3.2.1.22]\n",
      "AT5G08380  AGAL1; alpha-galactosidase 1 [KO:K07407] [EC:3.2.1.22]\n",
      "AT4G32840  PFK6; phosphofructokinase 6 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G56630  PFK7; phosphofructokinase 7 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G61580  PFK4; phosphofructokinase 4 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G29220  PFK1; phosphofructokinase 1 [KO:K00850] [EC:2.7.1.11]\n",
      "AT2G22480  PFK5; phosphofructokinase 5 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G26270  PFK3; phosphofructokinase 3 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G47810  PFK2; phosphofructokinase 2 [KO:K00850] [EC:2.7.1.11]\n",
      "AT2G36190  cwINV4; cell wall invertase 4 [KO:K01193] [EC:3.2.1.26]\n",
      "AT3G13790  ATBFRUCT1; Glycosyl hydrolases family 32 protein [KO:K01193] [EC:3.2.1.26]\n",
      "AT1G12240  ATBETAFRUCT4; Glycosyl hydrolases family 32 protein [KO:K01193] [EC:3.2.1.26]\n",
      "AT1G62660  Glycosyl hydrolases family 32 protein [KO:K01193] [EC:3.2.1.26]\n",
      "AT3G13784  CWINV5; cell wall invertase 5 [KO:K01193] [EC:3.2.1.26]\n",
      "AT5G11920  cwINV6; 6-&1-fructan exohydrolase [KO:K01193] [EC:3.2.1.26]\n",
      "AT3G52600  CWINV2; cell wall invertase 2 [KO:K01193] [EC:3.2.1.26]\n",
      "AT1G55120  FRUCT5; beta-fructofuranosidase 5 [KO:K01193] [EC:3.2.1.26]\n",
      "AT4G01970  STS; stachyose synthase [KO:K06611] [EC:2.4.1.67]\n",
      "AT5G11720  Glycosyl hydrolases family 31 protein [KO:K01187] [EC:3.2.1.20]\n",
      "COMPOUND    C00029  UDP-glucose\n",
      "C00031  D-Glucose\n",
      "C00052  UDP-alpha-D-galactose\n",
      "C00085  D-Fructose 6-phosphate\n",
      "C00089  Sucrose\n",
      "C00095  D-Fructose\n",
      "C00103  D-Glucose 1-phosphate\n",
      "C00111  Glycerone phosphate\n",
      "C00116  Glycerol\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00124  D-Galactose\n",
      "C00137  myo-Inositol\n",
      "C00159  D-Mannose\n",
      "C00243  Lactose\n",
      "C00267  alpha-D-Glucose\n",
      "C00446  alpha-D-Galactose 1-phosphate\n",
      "C00492  Raffinose\n",
      "C00577  D-Glyceraldehyde\n",
      "C00668  alpha-D-Glucose 6-phosphate\n",
      "C00794  D-Sorbitol\n",
      "C00795  D-Tagatose\n",
      "C00880  D-Galactonate\n",
      "C00984  alpha-D-Galactose\n",
      "C01097  D-Tagatose 6-phosphate\n",
      "C01113  D-Galactose 6-phosphate\n",
      "C01132  N-Acetyl-D-galactosamine\n",
      "C01216  2-Dehydro-3-deoxy-D-galactonate\n",
      "C01235  alpha-D-Galactosyl-(1->3)-1D-myo-inositol\n",
      "C01286  2-Dehydro-3-deoxy-6-phospho-D-galactonate\n",
      "C01613  Stachyose\n",
      "C01697  Galactitol\n",
      "C02262  D-Galactosamine\n",
      "C02669  D-Galactono-1,5-lactone\n",
      "C03383  D-Galactono-1,4-lactone\n",
      "C03733  UDP-alpha-D-galactofuranose\n",
      "C03785  D-Tagatose 1,6-bisphosphate\n",
      "C05396  Lactose 6'-phosphate\n",
      "C05399  Melibiitol\n",
      "C05400  Epimelibiose\n",
      "C05401  3-beta-D-Galactosyl-sn-glycerol\n",
      "C05402  Melibiose\n",
      "C05404  D-Gal alpha 1->6D-Gal alpha 1->6D-Glucose\n",
      "C05796  Galactan\n",
      "C06311  Galactitol 1-phosphate\n",
      "C06376  N-Acetyl-D-galactosamine 6-phosphate\n",
      "C06377  D-Galactosamine 6-phosphate\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00040  Pentose and glucuronate interconversions\n",
      "ath00051  Fructose and mannose metabolism\n",
      "ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "KO_PATHWAY  ko00052\n",
      "///\n",
      "ENTRY       ath00053                    Pathway\n",
      "NAME        Ascorbate and aldarate metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00053  Ascorbate and aldarate metabolism\n",
      "MODULE      ath_M00114  Ascorbate biosynthesis, plants, fructose-6P => ascorbate [PATH:ath00053]\n",
      "DBLINKS     GO: 0019852 0019572\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G39320  UDG4; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT1G26570  UGD1; UDP-glucose dehydrogenase 1 [KO:K00012] [EC:1.1.1.22]\n",
      "AT5G15490  UGD3; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT3G29360  UGD2; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT3G01010  UDP-glucose/GDP-mannose dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT5G52560  USP; UDP-sugar pyrophosphorylase [KO:K12447] [EC:2.7.7.64]\n",
      "AT5G14470  GHMP kinase family protein [KO:K16190] [EC:2.7.1.43]\n",
      "AT3G01640  GLCAK; glucuronokinase G [KO:K16190] [EC:2.7.1.43]\n",
      "AT2G19800  MIOX2; myo-inositol oxygenase 2 [KO:K00469] [EC:1.13.99.1]\n",
      "AT1G14520  MIOX1; myo-inositol oxygenase 1 [KO:K00469] [EC:1.13.99.1]\n",
      "AT4G26260  MIOX4; myo-inositol oxygenase 4 [KO:K00469] [EC:1.13.99.1]\n",
      "AT5G56640  MIOX5; myo-inositol oxygenase 5 [KO:K00469] [EC:1.13.99.1]\n",
      "AT2G37790  AKR4C10; NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT5G01670  NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT5G11540  GulLO3; D-arabinono-1,4-lactone oxidase family protein [KO:K00103] [EC:1.1.3.8]\n",
      "AT2G46750  GulLO2; D-arabinono-1,4-lactone oxidase family protein [KO:K00103] [EC:1.1.3.8]\n",
      "AT2G46740  GulLO5; D-arabinono-1,4-lactone oxidase family protein [KO:K00103] [EC:1.1.3.8]\n",
      "AT1G32300  GulLO1; D-arabinono-1,4-lactone oxidase family protein [KO:K00103] [EC:1.1.3.8]\n",
      "AT2G46760  GulLO6; D-arabinono-1,4-lactone oxidase family protein [KO:K00103] [EC:1.1.3.8]\n",
      "AT5G56490  GulLO4; D-arabinono-1,4-lactone oxidase family protein [KO:K00103] [EC:1.1.3.8]\n",
      "AT5G56470  GulLO7; FAD-dependent oxidoreductase family protein [KO:K00103] [EC:1.1.3.8]\n",
      "AT5G28840  GME; GDP-D-mannose 3',5'-epimerase [KO:K10046] [EC:5.1.3.18 5.1.3.-]\n",
      "AT5G55120  VTC5; GDP-L-galactose phosphorylase VITAMIN C DEFECTIVE 5 (VTC5) [KO:K14190] [EC:2.7.7.69]\n",
      "AT4G26850  VTC2; GDP-L-galactose phosphorylase 1 [KO:K14190] [EC:2.7.7.69]\n",
      "AT3G02870  VTC4; Inositol monophosphatase family protein [KO:K10047] [EC:3.1.3.25 3.1.3.93]\n",
      "AT4G39120  IMPL2; inositol monophosphatase family protein [KO:K18649] [EC:3.1.3.25 3.1.3.93 3.1.3.15]\n",
      "AT4G33670  NAD(P)-linked oxidoreductase superfamily protein [KO:K17744] [EC:1.1.1.316]\n",
      "AT3G47930  GLDH; L-galactono-1,4-lactone dehydrogenase [KO:K00225] [EC:1.3.2.3]\n",
      "AT4G00110  GAE3; UDP-D-glucuronate 4-epimerase 3 [KO:K08679] [EC:5.1.3.6]\n",
      "AT4G30440  GAE1; UDP-D-glucuronate 4-epimerase 1 [KO:K08679] [EC:5.1.3.6]\n",
      "AT4G12250  GAE5; UDP-D-glucuronate 4-epimerase 5 [KO:K08679] [EC:5.1.3.6]\n",
      "AT2G45310  GAE4; UDP-D-glucuronate 4-epimerase 4 [KO:K08679] [EC:5.1.3.6]\n",
      "AT1G02000  GAE2; UDP-D-glucuronate 4-epimerase 2 [KO:K08679] [EC:5.1.3.6]\n",
      "AT3G23820  GAE6; UDP-D-glucuronate 4-epimerase 6 [KO:K08679] [EC:5.1.3.6]\n",
      "AT5G03630  ATMDAR2; Pyridine nucleotide-disulfide oxidoreductase family protein [KO:K08232] [EC:1.6.5.4]\n",
      "AT3G52880  MDAR1; monodehydroascorbate reductase 1 [KO:K08232] [EC:1.6.5.4]\n",
      "AT3G27820  MDAR4; monodehydroascorbate reductase 4 [KO:K08232] [EC:1.6.5.4]\n",
      "AT3G09940  MDHAR; monodehydroascorbate reductase [KO:K08232] [EC:1.6.5.4]\n",
      "AT1G63940  MDAR6; monodehydroascorbate reductase 6 [KO:K08232] [EC:1.6.5.4]\n",
      "AT1G07890  APX1; ascorbate peroxidase 1 [KO:K00434] [EC:1.11.1.11]\n",
      "AT3G09640  APX2; ascorbate peroxidase 2 [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G35000  APX3; ascorbate peroxidase 3 [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G35970  APX5; ascorbate peroxidase 5 [KO:K00434] [EC:1.11.1.11]\n",
      "AT1G77490  TAPX; thylakoidal ascorbate peroxidase [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G08390  SAPX; stromal ascorbate peroxidase [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G09010  TL29; ascorbate peroxidase 4 [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G32320  APX6; ascorbate peroxidase 6 [KO:K00434] [EC:1.11.1.11]\n",
      "AT1G19550  Glutathione S-transferase family protein [KO:K21888] [EC:1.8.5.1 2.5.1.18]\n",
      "AT1G19570  DHAR1; dehydroascorbate reductase [KO:K21888] [EC:1.8.5.1 2.5.1.18]\n",
      "AT1G75270  DHAR2; dehydroascorbate reductase 2 [KO:K21888] [EC:1.8.5.1 2.5.1.18]\n",
      "AT5G16710  DHAR3; dehydroascorbate reductase 1 [KO:K21888] [EC:1.8.5.1 2.5.1.18]\n",
      "AT5G21100  Plant L-ascorbate oxidase [KO:K00423] [EC:1.10.3.3]\n",
      "AT1G55570  sks12; SKU5 similar 12 [KO:K00423] [EC:1.10.3.3]\n",
      "AT5G21105  Plant L-ascorbate oxidase [KO:K00423] [EC:1.10.3.3]\n",
      "AT3G13390  sks11; SKU5 similar 11 [KO:K00423] [EC:1.10.3.3]\n",
      "AT1G55560  sks14; SKU5 similar 14 [KO:K00423] [EC:1.10.3.3]\n",
      "AT3G13400  sks13; SKU5 similar 13 [KO:K00423] [EC:1.10.3.3]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00026  2-Oxoglutarate\n",
      "C00029  UDP-glucose\n",
      "C00072  Ascorbate\n",
      "C00096  GDP-mannose\n",
      "C00137  myo-Inositol\n",
      "C00167  UDP-glucuronate\n",
      "C00191  D-Glucuronate\n",
      "C00216  D-Arabinose\n",
      "C00231  D-Xylulose 5-phosphate\n",
      "C00259  L-Arabinose\n",
      "C00333  D-Galacturonate\n",
      "C00433  2,5-Dioxopentanoate\n",
      "C00545  L-Arabinonate\n",
      "C00558  D-Tagaturonate\n",
      "C00617  UDP-D-galacturonate\n",
      "C00618  3-Dehydro-L-gulonate\n",
      "C00652  D-Arabinono-1,4-lactone\n",
      "C00679  5-Dehydro-4-deoxy-D-glucarate\n",
      "C00684  2-Dehydro-3-deoxy-L-arabinonate\n",
      "C00800  L-Gulonate\n",
      "C00818  D-Glucarate\n",
      "C00878  D-Arabinonate\n",
      "C00879  D-Galactarate\n",
      "C01040  L-Gulono-1,4-lactone\n",
      "C01041  Monodehydroascorbate\n",
      "C01101  L-Ribulose 5-phosphate\n",
      "C01114  L-Arabinono-1,4-lactone\n",
      "C01115  L-Galactono-1,4-lactone\n",
      "C01146  2-Hydroxy-3-oxopropanoate\n",
      "C01620  Threonate\n",
      "C01825  L-Galactose\n",
      "C02280  GDP-L-galactose\n",
      "C02670  D-Glucuronolactone\n",
      "C03033  beta-D-Glucuronoside\n",
      "C03064  3-Dehydro-L-threonate\n",
      "C03289  L-xylo-Hexulonolactone\n",
      "C03291  L-Xylulose 5-phosphate\n",
      "C03826  2-Dehydro-3-deoxy-D-xylonate\n",
      "C03921  2-Dehydro-3-deoxy-D-glucarate\n",
      "C04037  1-Phospho-alpha-D-galacturonate\n",
      "C04575  (4R,5S)-4,5,6-Trihydroxy-2,3-dioxohexanoate\n",
      "C05385  D-Glucuronate 1-phosphate\n",
      "C05411  L-Xylonate\n",
      "C05412  L-Lyxonate\n",
      "C05422  Dehydroascorbate\n",
      "C06316  Dehydro-D-arabinono-1,4-lactone\n",
      "C14899  3-Dehydro-L-gulonate 6-phosphate\n",
      "C15923  L-Gulose\n",
      "C15924  L-Gulose 1-phosphate\n",
      "C15925  GDP-L-gulose\n",
      "C15926  beta-L-Galactose 1-phosphate\n",
      "C15930  L-Galactonate\n",
      "C16186  L-Ascorbate 6-phosphate\n",
      "C20889  D-Galactaro-1,5-lactone\n",
      "C20896  D-Galactaro-1,4-lactone\n",
      "C21955  L-Galactono-1,5-lactone\n",
      "REFERENCE   PMID:17222174\n",
      "AUTHORS   Linster CL, Van Schaftingen E\n",
      "TITLE     Vitamin C. Biosynthesis, recycling and degradation in mammals.\n",
      "JOURNAL   FEBS J 274:1-22 (2007)\n",
      "DOI:10.1111/j.1742-4658.2006.05607.x\n",
      "REFERENCE   PMID:15564123\n",
      "AUTHORS   Valpuesta V, Botella MA.\n",
      "TITLE     Biosynthesis of L-ascorbic acid in plants: new pathways for an old antioxidant.\n",
      "JOURNAL   Trends Plant Sci 9:573-7 (2004)\n",
      "DOI:10.1016/j.tplants.2004.10.002\n",
      "REFERENCE   PMID:10517845\n",
      "AUTHORS   Davey MW, Gilot C, Persiau G, Ostergaard J, Han Y, Bauw GC, Van Montagu MC.\n",
      "TITLE     Ascorbate biosynthesis in Arabidopsis cell suspension culture.\n",
      "JOURNAL   Plant Physiol 121:535-43 (1999)\n",
      "DOI:10.1104/pp.121.2.535\n",
      "REFERENCE   PMID:16595667\n",
      "AUTHORS   Conklin PL, Gatzek S, Wheeler GL, Dowdle J, Raymond MJ, Rolinski S, Isupov M, Littlechild JA, Smirnoff N.\n",
      "TITLE     Arabidopsis thaliana VTC4 encodes L-galactose-1-P phosphatase, a plant ascorbic acid biosynthetic enzyme.\n",
      "JOURNAL   J Biol Chem 281:15662-70 (2006)\n",
      "DOI:10.1074/jbc.M601409200\n",
      "REFERENCE   PMID:17462988\n",
      "AUTHORS   Linster CL, Gomez TA, Christensen KC, Adler LN, Young BD, Brenner C, Clarke SG.\n",
      "TITLE     Arabidopsis VTC2 Encodes a GDP-L-Galactose Phosphorylase, the Last Unknown Enzyme in the Smirnoff-Wheeler Pathway to Ascorbic Acid in Plants.\n",
      "JOURNAL   J Biol Chem 282:18879-85 (2007)\n",
      "DOI:10.1074/jbc.M702094200\n",
      "REFERENCE   PMID:14996803\n",
      "AUTHORS   Campos E, Baldoma L, Aguilar J, Badia J.\n",
      "TITLE     Regulation of expression of the divergent ulaG and ulaABCDEF operons involved in LaAscorbate dissimilation in Escherichia coli.\n",
      "JOURNAL   J Bacteriol 186:1720-8 (2004)\n",
      "DOI:10.1128/JB.186.6.1720-1728.2004\n",
      "REFERENCE   PMID:12644495\n",
      "AUTHORS   Zhang Z, Aboulwafa M, Smith MH, Saier MH Jr.\n",
      "TITLE     The ascorbate transporter of Escherichia coli.\n",
      "JOURNAL   J Bacteriol 185:2243-50 (2003)\n",
      "DOI:10.1128/JB.185.7.2243-2250.2003\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00040  Pentose and glucuronate interconversions\n",
      "ath00051  Fructose and mannose metabolism\n",
      "ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "KO_PATHWAY  ko00053\n",
      "///\n",
      "ENTRY       ath00061                    Pathway\n",
      "NAME        Fatty acid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00061  Fatty acid biosynthesis\n",
      "MODULE      ath_M00082  Fatty acid biosynthesis, initiation [PATH:ath00061]\n",
      "ath_M00083  Fatty acid biosynthesis, elongation [PATH:ath00061]\n",
      "ath_M00086  beta-Oxidation, acyl-CoA synthesis [PATH:ath00061]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G36160  ACC1; acetyl-CoA carboxylase 1 [KO:K11262] [EC:6.4.1.2 6.3.4.14 2.1.3.15]\n",
      "AT1G36180  ACC2; acetyl-CoA carboxylase 2 [KO:K11262] [EC:6.4.1.2 6.3.4.14 2.1.3.15]\n",
      "AT2G38040  CAC3; acetyl Co-enzyme a carboxylase carboxyltransferase alpha subunit [KO:K01962] [EC:6.4.1.2 2.1.3.15]\n",
      "AT5G16390  CAC1; chloroplastic acetylcoenzyme A carboxylase 1 [KO:K02160]\n",
      "AT5G15530  BCCP2; biotin carboxyl carrier protein 2 [KO:K02160]\n",
      "AT5G35360  CAC2; acetyl Co-enzyme a carboxylase biotin carboxylase subunit [KO:K01961] [EC:6.4.1.2 6.3.4.14]\n",
      "ArthCp031  accD; acetyl-CoA carboxylase beta subunit [KO:K01963] [EC:6.4.1.2 2.1.3.15]\n",
      "AT3G16170  AAE13; AMP-dependent synthetase and ligase family protein [KO:K18660] [EC:6.2.1.-]\n",
      "AT2G30200  EMB3147; EMBRYO DEFECTIVE 3147 [KO:K00645] [EC:2.3.1.39]\n",
      "AT1G62640  KAS_III; 3-ketoacyl-acyl carrier protein synthase III [KO:K00648] [EC:2.3.1.180]\n",
      "AT2G04540  Beta-ketoacyl synthase [KO:K09458] [EC:2.3.1.179]\n",
      "AT1G74960  FAB1; fatty acid biosynthesis 1 [KO:K09458] [EC:2.3.1.179]\n",
      "AT5G46290  KASI; 3-ketoacyl-acyl carrier protein synthase I [KO:K09458] [EC:2.3.1.179]\n",
      "AT4G13180  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT3G03980  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT1G24360  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT3G04000  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT5G18210  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT5G10160  Thioesterase superfamily protein [KO:K02372] [EC:4.2.1.59]\n",
      "AT2G22230  Thioesterase superfamily protein [KO:K02372] [EC:4.2.1.59]\n",
      "AT2G05990  MOD1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K00208] [EC:1.3.1.9 1.3.1.10]\n",
      "AT3G45770  Polyketide synthase, enoylreductase family protein [KO:K07512] [EC:1.3.1.- 1.3.1.38]\n",
      "AT1G08510  FATB; fatty acyl-ACP thioesterases B [KO:K10781] [EC:3.1.2.14 3.1.2.21]\n",
      "AT3G25110  FaTA; fatA acyl-ACP thioesterase [KO:K10782] [EC:3.1.2.14]\n",
      "AT4G13050  Acyl-ACP thioesterase [KO:K10782] [EC:3.1.2.14]\n",
      "AT1G43800  FTM1; Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT3G02610  plant stearoyl-acyl-carrier desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT3G02620  Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT3G02630  Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT5G16230  Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT5G16240  Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT2G43710  SSI2; Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT4G14070  AAE15; acyl-activating enzyme 15 [KO:K01897] [EC:6.2.1.3]\n",
      "AT5G27600  LACS7; long-chain acyl-CoA synthetase 7 [KO:K01897] [EC:6.2.1.3]\n",
      "AT3G05970  LACS6; long-chain acyl-CoA synthetase 6 [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G64400  LACS3; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G49430  LACS2; long-chain acyl-CoA synthetase 2 [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G77590  LACS9; long chain acyl-CoA synthetase 9 [KO:K01897] [EC:6.2.1.3]\n",
      "AT4G11030  AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT2G04350  LACS8; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT3G23790  AAE16; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT2G47240  LACS1; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT4G23850  LACS4; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00083  Malonyl-CoA\n",
      "C00154  Palmitoyl-CoA\n",
      "C00173  Acyl-[acyl-carrier protein]\n",
      "C00229  Acyl-carrier protein\n",
      "C00249  Hexadecanoic acid\n",
      "C00383  Malonate\n",
      "C00638  Long-chain fatty acid\n",
      "C00685  3-Oxoacyl-[acyl-carrier protein]\n",
      "C00693  trans-2,3-Dehydroacyl-[acyl-carrier protein]\n",
      "C00712  (9Z)-Octadecenoic acid\n",
      "C01203  Oleoyl-[acyl-carrier protein]\n",
      "C01209  Malonyl-[acyl-carrier protein]\n",
      "C01271  (3R)-3-Hydroxyacyl-[acyl-carrier protein]\n",
      "C01530  Octadecanoic acid\n",
      "C01571  Decanoic acid\n",
      "C02679  Dodecanoic acid\n",
      "C02843  Long-chain acyl-CoA\n",
      "C03939  Acetyl-[acyl-carrier protein]\n",
      "C04088  Octadecanoyl-[acyl-carrier protein]\n",
      "C04180  cis-Dec-3-enoyl-[acp]\n",
      "C04246  But-2-enoyl-[acyl-carrier protein]\n",
      "C04618  (3R)-3-Hydroxybutanoyl-[acyl-carrier protein]\n",
      "C04619  (3R)-3-Hydroxydecanoyl-[acyl-carrier protein]\n",
      "C04620  (3R)-3-Hydroxyoctanoyl-[acyl-carrier protein]\n",
      "C04633  (3R)-3-Hydroxypalmitoyl-[acyl-carrier protein]\n",
      "C04688  (3R)-3-Hydroxytetradecanoyl-[acyl-carrier protein]\n",
      "C05223  Dodecanoyl-[acyl-carrier protein]\n",
      "C05744  Acetoacetyl-[acp]\n",
      "C05745  Butyryl-[acp]\n",
      "C05746  3-Oxohexanoyl-[acp]\n",
      "C05747  (R)-3-Hydroxyhexanoyl-[acp]\n",
      "C05748  trans-Hex-2-enoyl-[acp]\n",
      "C05749  Hexanoyl-[acp]\n",
      "C05750  3-Oxooctanoyl-[acp]\n",
      "C05751  trans-Oct-2-enoyl-[acp]\n",
      "C05752  Octanoyl-[acp]\n",
      "C05753  3-Oxodecanoyl-[acp]\n",
      "C05754  trans-Dec-2-enoyl-[acp]\n",
      "C05755  Decanoyl-[acp]\n",
      "C05756  3-Oxododecanoyl-[acp]\n",
      "C05757  (R)-3-Hydroxydodecanoyl-[acp]\n",
      "C05758  trans-Dodec-2-enoyl-[acp]\n",
      "C05759  3-Oxotetradecanoyl-[acp]\n",
      "C05760  trans-Tetradec-2-enoyl-[acp]\n",
      "C05761  Tetradecanoyl-[acp]\n",
      "C05762  3-Oxohexadecanoyl-[acp]\n",
      "C05763  trans-Hexadec-2-enoyl-[acp]\n",
      "C05764  Hexadecanoyl-[acp]\n",
      "C06423  Octanoic acid\n",
      "C06424  Tetradecanoic acid\n",
      "C08362  (9Z)-Hexadecenoic acid\n",
      "C16219  3-Oxostearoyl-[acp]\n",
      "C16220  (R)-3-Hydroxyoctadecanoyl-[acp]\n",
      "C16221  (2E)-Octadecenoyl-[acp]\n",
      "C16520  Hexadecenoyl-[acyl-carrier protein]\n",
      "C20683  Long-chain acyl-[acyl-carrier protein]\n",
      "C20794  n-7 Unsaturated acyl-[acyl-carrier protein]\n",
      "REFERENCE   PMID:12061798\n",
      "AUTHORS   Salas JJ, Ohlrogge JB.\n",
      "TITLE     Characterization of substrate specificity of plant FatA and FatB acyl-ACP thioesterases.\n",
      "JOURNAL   Arch Biochem Biophys 403:25-34 (2002)\n",
      "DOI:10.1016/S0003-9861(02)00017-6\n",
      "REFERENCE   PMID:12518017\n",
      "AUTHORS   Zhang YM, Marrakchi H, White SW, Rock CO.\n",
      "TITLE     The application of computational methods to explore the diversity and structure of bacterial fatty acid synthase.\n",
      "JOURNAL   J Lipid Res 44:1-10 (2003)\n",
      "DOI:10.1194/jlr.R200016-JLR200\n",
      "REFERENCE   PMID:11337402\n",
      "AUTHORS   Voelker T, Kinney AJ.\n",
      "TITLE     VARIATIONS IN THE BIOSYNTHESIS OF SEED-STORAGE LIPIDS.\n",
      "JOURNAL   Annu Rev Plant Physiol Plant Mol Biol 52:335-361 (2001)\n",
      "DOI:10.1146/annurev.arplant.52.1.335\n",
      "REFERENCE   PMID:17573542\n",
      "AUTHORS   Barker GC, Larson TR, Graham IA, Lynn JR, King GJ.\n",
      "TITLE     Novel insights into seed fatty acid synthesis and modification pathways from genetic diversity and quantitative trait Loci analysis of the Brassica C genome.\n",
      "JOURNAL   Plant Physiol 144:1827-42 (2007)\n",
      "DOI:10.1104/pp.107.096172\n",
      "REFERENCE   PMID:30201289\n",
      "AUTHORS   Bowman CE, Wolfgang MJ\n",
      "TITLE     Role of the malonyl-CoA synthetase ACSF3 in mitochondrial metabolism.\n",
      "JOURNAL   Adv Biol Regul 71:34-40 (2019)\n",
      "DOI:10.1016/j.jbior.2018.09.002\n",
      "REFERENCE   PMID:25203508\n",
      "AUTHORS   Venkatesan R, Sah-Teli SK, Awoniyi LO, Jiang G, Prus P, Kastaniotis AJ, Hiltunen JK, Wierenga RK, Chen Z\n",
      "TITLE     Insights into mitochondrial fatty acid synthesis from the structure of heterotetrameric 3-ketoacyl-ACP reductase/3R-hydroxyacyl-CoA dehydrogenase.\n",
      "JOURNAL   Nat Commun 5:4805 (2014)\n",
      "DOI:10.1038/ncomms5805\n",
      "REFERENCE   PMID:19686777\n",
      "AUTHORS   Hiltunen JK, Chen Z, Haapalainen AM, Wierenga RK, Kastaniotis AJ\n",
      "TITLE     Mitochondrial fatty acid synthesis--an adopted set of enzymes making a pathway of major importance for the cellular metabolism.\n",
      "JOURNAL   Prog Lipid Res 49:27-45 (2010)\n",
      "DOI:10.1016/j.plipres.2009.08.001\n",
      "REFERENCE   PMID:27553474\n",
      "AUTHORS   Kastaniotis AJ, Autio KJ, Keratar JM, Monteuuis G, Makela AM, Nair RR, Pietikainen LP, Shvetsova A, Chen Z, Hiltunen JK\n",
      "TITLE     Mitochondrial fatty acid synthesis, fatty acids and mitochondrial physiology.\n",
      "JOURNAL   Biochim Biophys Acta Mol Cell Biol Lipids 1862:39-48 (2017)\n",
      "DOI:10.1016/j.bbalip.2016.08.011\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00062  Fatty acid elongation\n",
      "ath00071  Fatty acid degradation\n",
      "ath00410  beta-Alanine metabolism\n",
      "ath00561  Glycerolipid metabolism\n",
      "ath00564  Glycerophospholipid metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00785  Lipoic acid metabolism\n",
      "KO_PATHWAY  ko00061\n",
      "///\n",
      "ENTRY       ath00062                    Pathway\n",
      "NAME        Fatty acid elongation - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00062  Fatty acid elongation\n",
      "MODULE      ath_M00415  Fatty acid elongation in endoplasmic reticulum [PATH:ath00062]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G45770  Polyketide synthase, enoylreductase family protein [KO:K07512] [EC:1.3.1.- 1.3.1.38]\n",
      "AT4G17470  alpha/beta-Hydrolases superfamily protein [KO:K01074] [EC:3.1.2.22]\n",
      "AT4G17480  alpha/beta-Hydrolases superfamily protein [KO:K01074] [EC:3.1.2.22]\n",
      "AT4G17483  alpha/beta-Hydrolases superfamily protein [KO:K01074] [EC:3.1.2.22]\n",
      "AT5G47340  alpha/beta-Hydrolases superfamily protein [KO:K01074] [EC:3.1.2.22]\n",
      "AT5G47330  alpha/beta-Hydrolases superfamily protein [KO:K01074] [EC:3.1.2.22]\n",
      "AT3G60340  alpha/beta-Hydrolases superfamily protein [KO:K01074] [EC:3.1.2.22]\n",
      "AT5G47350  alpha/beta-Hydrolases superfamily protein [KO:K01074] [EC:3.1.2.22]\n",
      "AT2G46720  KCS13; 3-ketoacyl-CoA synthase 13 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G01120  KCS1; 3-ketoacyl-CoA synthase 1 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G26640  KCS11; 3-ketoacyl-CoA synthase 11 [KO:K15397] [EC:2.3.1.199]\n",
      "AT3G10280  KCS14; 3-ketoacyl-CoA synthase 14 [KO:K15397] [EC:2.3.1.199]\n",
      "AT4G34510  KCS17; 3-ketoacyl-CoA synthase 17 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G04220  KCS2; 3-ketoacyl-CoA synthase 2 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G19440  KCS4; 3-ketoacyl-CoA synthase 4 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G25450  KCS5; 3-ketoacyl-CoA synthase 5 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G68530  KCS6; 3-ketoacyl-CoA synthase 6 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G71160  KCS7; 3-ketoacyl-CoA synthase 7 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G15090  KCS8; 3-ketoacyl-CoA synthase 8 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G16280  KCS9; 3-ketoacyl-CoA synthase 9 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G26250  KCS10; 3-ketoacyl-CoA synthase 10 [KO:K15397] [EC:2.3.1.199]\n",
      "AT4G34250  KCS16; 3-ketoacyl-CoA synthase 16 [KO:K15397] [EC:2.3.1.199]\n",
      "AT4G34520  KCS18; 3-ketoacyl-CoA synthase 18 [KO:K15397] [EC:2.3.1.199]\n",
      "AT5G43760  KCS20; 3-ketoacyl-CoA synthase 20 [KO:K15397] [EC:2.3.1.199]\n",
      "AT3G52160  KCS15; 3-ketoacyl-CoA synthase 15 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G28630  KCS12; 3-ketoacyl-CoA synthase 12 [KO:K15397] [EC:2.3.1.199]\n",
      "AT5G04530  KCS19; 3-ketoacyl-CoA synthase 19 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G07720  KCS3; 3-ketoacyl-CoA synthase 3 [KO:K15397] [EC:2.3.1.199]\n",
      "AT5G49070  KCS21; 3-ketoacyl-CoA synthase 21 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G67730  KCR1; beta-ketoacyl reductase 1 [KO:K10251] [EC:1.1.1.62 1.1.1.330]\n",
      "AT1G24470  KCR2; beta-ketoacyl reductase 2 [KO:K10251] [EC:1.1.1.62 1.1.1.330]\n",
      "AT5G10480  PAS2; Protein-tyrosine phosphatase-like, PTPLA [KO:K10703] [EC:4.2.1.134]\n",
      "AT5G59770  Protein-tyrosine phosphatase-like, PTPLA [KO:K10703] [EC:4.2.1.134]\n",
      "AT3G55360  CER10; 3-oxo-5-alpha-steroid 4-dehydrogenase family protein [KO:K10258] [EC:1.3.1.93]\n",
      "AT1G01710  acyl-CoA thioesterase II [KO:K01068] [EC:3.1.2.2]\n",
      "AT4G00520  Acyl-CoA thioesterase family protein [KO:K01068] [EC:3.1.2.2]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00040  Acyl-CoA\n",
      "C00083  Malonyl-CoA\n",
      "C00136  Butanoyl-CoA\n",
      "C00154  Palmitoyl-CoA\n",
      "C00162  Fatty acid\n",
      "C00249  Hexadecanoic acid\n",
      "C00264  3-Oxoacyl-CoA\n",
      "C00605  2,3-Dehydroacyl-CoA\n",
      "C00638  Long-chain fatty acid\n",
      "C00640  (3S)-3-Hydroxyacyl-CoA\n",
      "C00658  trans-2,3-Dehydroacyl-CoA\n",
      "C01832  Lauroyl-CoA\n",
      "C01944  Octanoyl-CoA\n",
      "C02593  Tetradecanoyl-CoA\n",
      "C02843  Long-chain acyl-CoA\n",
      "C03221  2-trans-Dodecenoyl-CoA\n",
      "C05258  (S)-3-Hydroxyhexadecanoyl-CoA\n",
      "C05259  3-Oxopalmitoyl-CoA\n",
      "C05260  (S)-3-Hydroxytetradecanoyl-CoA\n",
      "C05261  3-Oxotetradecanoyl-CoA\n",
      "C05262  (S)-3-Hydroxydodecanoyl-CoA\n",
      "C05263  3-Oxododecanoyl-CoA\n",
      "C05264  (S)-Hydroxydecanoyl-CoA\n",
      "C05265  3-Oxodecanoyl-CoA\n",
      "C05266  (S)-3-Hydroxyoctanoyl-CoA\n",
      "C05267  3-Oxooctanoyl-CoA\n",
      "C05268  (S)-Hydroxyhexanoyl-CoA\n",
      "C05269  3-Oxohexanoyl-CoA\n",
      "C05270  Hexanoyl-CoA\n",
      "C05271  trans-Hex-2-enoyl-CoA\n",
      "C05272  trans-Hexadec-2-enoyl-CoA\n",
      "C05273  trans-Tetradec-2-enoyl-CoA\n",
      "C05274  Decanoyl-CoA\n",
      "C05275  trans-Dec-2-enoyl-CoA\n",
      "C05276  trans-Oct-2-enoyl-CoA\n",
      "C20876  Very-long-chain acyl-CoA\n",
      "C20877  Very-long-chain 3-oxoacyl-CoA\n",
      "C20878  Very-long-chain (3R)-3-hydroxyacyl-CoA\n",
      "C20879  Very-long-chain trans-2,3-dehydroacyl-CoA\n",
      "REFERENCE   PMID:4387390\n",
      "AUTHORS   Seubert W, Lamberts I, Kramer R, Ohly B\n",
      "TITLE     On the mechanism of malonyl-CoA-independent fatty acid synthesis. I. The mechanism of elongation of long-chain fatty acids by acetyl-CoA.\n",
      "JOURNAL   Biochim Biophys Acta 164:498-517 (1968)\n",
      "DOI:10.1016/0005-2760(68)90180-X\n",
      "REFERENCE   PMID:7150615\n",
      "AUTHORS   El-Fakhri M, Middleton B\n",
      "TITLE     The existence of an inner-membrane-bound, long acyl-chain-specific 3-hydroxyacyl-CoA dehydrogenase in mammalian mitochondria.\n",
      "JOURNAL   Biochim Biophys Acta 713:270-9 (1982)\n",
      "DOI:10.1016/0005-2760(82)90244-2\n",
      "REFERENCE   PMID:8190743\n",
      "AUTHORS   Cassagne C, Lessire R, Bessoule JJ, Moreau P, Creach A, Schneider F, Sturbois B\n",
      "TITLE     Biosynthesis of very long chain fatty acids in higher plants.\n",
      "JOURNAL   Prog Lipid Res 33:55-69 (1994)\n",
      "DOI:10.1016/0163-7827(94)90009-4\n",
      "REFERENCE   PMID:16564093\n",
      "AUTHORS   Jakobsson A, Westerberg R, Jacobsson A.\n",
      "TITLE     Fatty acid elongases in mammals: their regulation and roles in metabolism.\n",
      "JOURNAL   Prog Lipid Res 45:237-49 (2006)\n",
      "DOI:10.1016/j.plipres.2006.01.004\n",
      "REFERENCE   PMID:16765910\n",
      "AUTHORS   Blacklock BJ, Jaworski JG\n",
      "TITLE     Substrate specificity of Arabidopsis 3-ketoacyl-CoA synthases.\n",
      "JOURNAL   Biochem Biophys Res Commun 346:583-90 (2006)\n",
      "DOI:10.1016/j.bbrc.2006.05.162\n",
      "REFERENCE   PMID:16866954\n",
      "AUTHORS   Shepherd T, Wynne Griffiths D\n",
      "TITLE     The effects of stress on plant cuticular waxes.\n",
      "JOURNAL   New Phytol 171:469-99 (2006)\n",
      "DOI:10.1111/j.1469-8137.2006.01826.x\n",
      "REFERENCE   PMID:17719534\n",
      "AUTHORS   Riezman H\n",
      "TITLE     The long and short of fatty acid synthesis.\n",
      "JOURNAL   Cell 130:587-8 (2007)\n",
      "DOI:10.1016/j.cell.2007.08.008\n",
      "REFERENCE   PMID:19649180\n",
      "AUTHORS   Raffaele S, Leger A, Roby D\n",
      "TITLE     Very long chain fatty acid and lipid signaling in the response of plants to pathogens.\n",
      "JOURNAL   Plant Signal Behav 4:94-9 (2009)\n",
      "DOI:10.4161/psb.4.2.7580\n",
      "REL_PATHWAY ath00071  Fatty acid degradation\n",
      "ath00073  Cutin, suberine and wax biosynthesis\n",
      "KO_PATHWAY  ko00062\n",
      "///\n",
      "ENTRY       ath00071                    Pathway\n",
      "NAME        Fatty acid degradation - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00071  Fatty acid degradation\n",
      "MODULE      ath_M00086  beta-Oxidation, acyl-CoA synthesis [PATH:ath00071]\n",
      "ath_M00087  beta-Oxidation [PATH:ath00071]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G48230  ACAT2; acetoacetyl-CoA thiolase 2 [KO:K00626] [EC:2.3.1.9]\n",
      "AT5G47720  AACT1; Thiolase family protein [KO:K00626] [EC:2.3.1.9]\n",
      "AT1G04710  PKT4; peroxisomal 3-ketoacyl-CoA thiolase 4 [KO:K07513] [EC:2.3.1.16]\n",
      "AT2G33150  PKT3; peroxisomal 3-ketoacyl-CoA thiolase 3 [KO:K07513] [EC:2.3.1.16]\n",
      "AT5G48880  KAT5; peroxisomal 3-keto-acyl-CoA thiolase 2 [KO:K07513] [EC:2.3.1.16]\n",
      "AT3G06860  MFP2; multifunctional protein 2 [KO:K10527] [EC:4.2.1.17 1.1.1.35 1.1.1.211]\n",
      "AT4G29010  AIM1; Enoyl-CoA hydratase/isomerase family [KO:K10527] [EC:4.2.1.17 1.1.1.35 1.1.1.211]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "AT2G35690  ACX5; acyl-CoA oxidase 5 [KO:K00232] [EC:1.3.3.6]\n",
      "AT4G16760  ACX1; acyl-CoA oxidase 1 [KO:K00232] [EC:1.3.3.6]\n",
      "AT5G65110  ACX2; acyl-CoA oxidase 2 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G51840  ACX4; acyl-CoA oxidase 4 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06290  ACX3; acyl-CoA oxidase 3 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06310  ACX6; acyl-CoA oxidase 6 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G06690  putative acyl-coenzyme A oxidase [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G06810  IBR3; acyl-CoA dehydrogenase-like protein [KO:K00249] [EC:1.3.8.7]\n",
      "AT4G14070  AAE15; acyl-activating enzyme 15 [KO:K01897] [EC:6.2.1.3]\n",
      "AT5G27600  LACS7; long-chain acyl-CoA synthetase 7 [KO:K01897] [EC:6.2.1.3]\n",
      "AT3G05970  LACS6; long-chain acyl-CoA synthetase 6 [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G64400  LACS3; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G49430  LACS2; long-chain acyl-CoA synthetase 2 [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G77590  LACS9; long chain acyl-CoA synthetase 9 [KO:K01897] [EC:6.2.1.3]\n",
      "AT4G11030  AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT2G04350  LACS8; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT3G23790  AAE16; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT2G47240  LACS1; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT4G23850  LACS4; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G65520  ECI1; delta(3), delta(2)-enoyl CoA isomerase 1 [KO:K07517] [EC:5.3.3.8]\n",
      "AT4G14430  IBR10; indole-3-butyric acid response 10 [KO:K07517] [EC:5.3.3.8]\n",
      "AT4G14440  HCD1; 3-hydroxyacyl-CoA dehydratase 1 [KO:K18880] [EC:5.3.3.8]\n",
      "AT5G58860  CYP86A1; cytochrome P450, family 86, subfamily A, polypeptide 1 [KO:K15401] [EC:1.14.14.80]\n",
      "AT1G69500  CYP704B1; cytochrome P450, family 704, subfamily B, polypeptide 1 [KO:K20495] [EC:1.14.14.80]\n",
      "AT1G22440  Zinc-binding alcohol dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT5G42250  Zinc-binding alcohol dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT5G43940  HOT5; GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT4G22110  GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT1G22430  GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT1G77120  ADH1; alcohol dehydrogenase 1 [KO:K18857] [EC:1.1.1.1]\n",
      "AT1G32780  GroES-like zinc-binding dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT5G24760  GroES-like zinc-binding dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT1G64710  GroES-like zinc-binding alcohol dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "COMPOUND    C00010  CoA\n",
      "C00024  Acetyl-CoA\n",
      "C00071  Aldehyde\n",
      "C00136  Butanoyl-CoA\n",
      "C00154  Palmitoyl-CoA\n",
      "C00162  Fatty acid\n",
      "C00226  Primary alcohol\n",
      "C00229  Acyl-carrier protein\n",
      "C00249  Hexadecanoic acid\n",
      "C00332  Acetoacetyl-CoA\n",
      "C00340  Reduced rubredoxin\n",
      "C00435  Oxidized rubredoxin\n",
      "C00489  Glutarate\n",
      "C00517  Hexadecanal\n",
      "C00527  Glutaryl-CoA\n",
      "C00638  Long-chain fatty acid\n",
      "C00823  1-Hexadecanol\n",
      "C00877  Crotonoyl-CoA\n",
      "C01144  (S)-3-Hydroxybutanoyl-CoA\n",
      "C01371  Alkane\n",
      "C01832  Lauroyl-CoA\n",
      "C01944  Octanoyl-CoA\n",
      "C02593  Tetradecanoyl-CoA\n",
      "C02990  L-Palmitoylcarnitine\n",
      "C03221  2-trans-Dodecenoyl-CoA\n",
      "C03547  omega-Hydroxy fatty acid\n",
      "C03561  (R)-3-Hydroxybutanoyl-CoA\n",
      "C05102  alpha-Hydroxy fatty acid\n",
      "C05258  (S)-3-Hydroxyhexadecanoyl-CoA\n",
      "C05259  3-Oxopalmitoyl-CoA\n",
      "C05260  (S)-3-Hydroxytetradecanoyl-CoA\n",
      "C05261  3-Oxotetradecanoyl-CoA\n",
      "C05262  (S)-3-Hydroxydodecanoyl-CoA\n",
      "C05263  3-Oxododecanoyl-CoA\n",
      "C05264  (S)-Hydroxydecanoyl-CoA\n",
      "C05265  3-Oxodecanoyl-CoA\n",
      "C05266  (S)-3-Hydroxyoctanoyl-CoA\n",
      "C05267  3-Oxooctanoyl-CoA\n",
      "C05268  (S)-Hydroxyhexanoyl-CoA\n",
      "C05269  3-Oxohexanoyl-CoA\n",
      "C05270  Hexanoyl-CoA\n",
      "C05271  trans-Hex-2-enoyl-CoA\n",
      "C05272  trans-Hexadec-2-enoyl-CoA\n",
      "C05273  trans-Tetradec-2-enoyl-CoA\n",
      "C05274  Decanoyl-CoA\n",
      "C05275  trans-Dec-2-enoyl-CoA\n",
      "C05276  trans-Oct-2-enoyl-CoA\n",
      "C05279  trans,cis-Lauro-2,6-dienoyl-CoA\n",
      "C05280  cis,cis-3,6-Dodecadienoyl-CoA\n",
      "C20683  Long-chain acyl-[acyl-carrier protein]\n",
      "REFERENCE   PMID:869535\n",
      "AUTHORS   Parekh VR, Traxler RW, Sobek JM\n",
      "TITLE     N-Alkane oxidation enzymes of a pseudomonad.\n",
      "JOURNAL   Appl Environ Microbiol 33:881-4 (1977)\n",
      "DOI:10.1128/AEM.33.4.881-884.1977\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00061  Fatty acid biosynthesis\n",
      "ath00062  Fatty acid elongation\n",
      "ath00561  Glycerolipid metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00650  Butanoate metabolism\n",
      "KO_PATHWAY  ko00071\n",
      "///\n",
      "ENTRY       ath00073                    Pathway\n",
      "NAME        Cutin, suberine and wax biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00073  Cutin, suberine and wax biosynthesis\n",
      "DBLINKS     GO: 0010143 0010345 0010025\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G00360  CYP86A2; cytochrome P450, family 86, subfamily A, polypeptide 2 [KO:K15398] [EC:1.14.-.-]\n",
      "AT1G63710  CYP86A7; cytochrome P450, family 86, subfamily A, polypeptide 7 [KO:K15398] [EC:1.14.-.-]\n",
      "AT1G01600  CYP86A4; cytochrome P450, family 86, subfamily A, polypeptide 4 [KO:K15398] [EC:1.14.-.-]\n",
      "AT2G45970  CYP86A8; cytochrome P450, family 86, subfamily A, polypeptide 8 [KO:K15398] [EC:1.14.-.-]\n",
      "AT1G69500  CYP704B1; cytochrome P450, family 704, subfamily B, polypeptide 1 [KO:K20495] [EC:1.14.14.80]\n",
      "AT5G04660  CYP77A4; cytochrome P450, family 77, subfamily A, polypeptide 4 [KO:K21995] [EC:1.14.-.-]\n",
      "AT3G10560  UNE9; Cytochrome P450 superfamily protein [KO:K21995] [EC:1.14.-.-]\n",
      "AT5G04630  CYP77A9; cytochrome P450, family 77, subfamily A, polypeptide 9 [KO:K21995] [EC:1.14.-.-]\n",
      "AT1G11600  CYP77B1; cytochrome P450, family 77, subfamily B, polypeptide 1 [KO:K21995] [EC:1.14.-.-]\n",
      "AT3G10570  CYP77A6; cytochrome P450, family 77, subfamily A, polypeptide 6 [KO:K21995] [EC:1.14.-.-]\n",
      "AT5G41040  RWP1; HXXXD-type acyl-transferase family protein [KO:K15400] [EC:2.3.1.188]\n",
      "AT5G63560  FACT; HXXXD-type acyl-transferase family protein [KO:K15400] [EC:2.3.1.188]\n",
      "AT1G03390  HXXXD-type acyl-transferase family protein [KO:K15400] [EC:2.3.1.188]\n",
      "AT5G58860  CYP86A1; cytochrome P450, family 86, subfamily A, polypeptide 1 [KO:K15401] [EC:1.14.14.80]\n",
      "AT2G33380  RD20; Caleosin-related family protein [KO:K17991] [EC:1.11.2.3]\n",
      "AT1G70670  CLO4; Caleosin-related family protein [KO:K17991] [EC:1.11.2.3]\n",
      "AT4G26740  ATS1; peroxygenase 1 [KO:K17991] [EC:1.11.2.3]\n",
      "AT5G55240  ATPXG2; PEROXYGENASE 2 [KO:K17991] [EC:1.11.2.3]\n",
      "AT1G70680  Caleosin-related family protein [KO:K17991] [EC:1.11.2.3]\n",
      "AT1G23240  Caleosin-related family protein [KO:K17991] [EC:1.11.2.3]\n",
      "AT5G29560  caleosin-related family protein [KO:K17991] [EC:1.11.2.3]\n",
      "AT5G23190  CYP86B1; cytochrome P450, family 86, subfamily B, polypeptide 1 [KO:K15402] [EC:1.14.-.-]\n",
      "AT5G08250  Cytochrome P450 superfamily protein [KO:K15402] [EC:1.14.-.-]\n",
      "AT1G72970  HTH; Glucose-methanol-choline (GMC) oxidoreductase family protein [KO:K15403] [EC:1.1.-.-]\n",
      "AT1G02205  CER1; Fatty acid hydroxylase superfamily [KO:K15404] [EC:4.1.99.5]\n",
      "AT1G02190  Fatty acid hydroxylase superfamily [KO:K15404] [EC:4.1.99.5]\n",
      "AT2G37700  Fatty acid hydroxylase superfamily [KO:K15404] [EC:4.1.99.5]\n",
      "AT1G57750  CYP96A15; cytochrome P450, family 96, subfamily A, polypeptide 15 [KO:K15405]\n",
      "AT4G33790  CER4; Jojoba acyl CoA reductase-related male sterility protein [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G11980  MS2; Jojoba acyl CoA reductase-related male sterility protein [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G44540  FAR4; fatty acid reductase 4 [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G44550  FAR5; fatty acid reductase 5 [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G44560  FAR8; fatty acid reductase 8 [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G56700  FAR6; fatty acid reductase 6 [KO:K13356] [EC:1.2.1.84]\n",
      "AT5G22500  FAR1; fatty acid reductase 1 [KO:K13356] [EC:1.2.1.84]\n",
      "AT5G22420  FAR7; fatty acid reductase 7 [KO:K13356] [EC:1.2.1.84]\n",
      "AT5G37300  WSD1; O-acyltransferase (WSD1-like) family protein [KO:K15406] [EC:2.3.1.75 2.3.1.20]\n",
      "COMPOUND    C00226  Primary alcohol\n",
      "C00249  Hexadecanoic acid\n",
      "C00406  Feruloyl-CoA\n",
      "C00609  Long-chain aldehyde\n",
      "C00712  (9Z)-Octadecenoic acid\n",
      "C01371  Alkane\n",
      "C01450  Ketone\n",
      "C01612  Secondary alcohol\n",
      "C01629  Wax ester\n",
      "C02843  Long-chain acyl-CoA\n",
      "C08281  Docosanoic acid\n",
      "C08285  10,16-Dihydroxyhexadecanoic acid\n",
      "C18217  16-Feruloyloxypalmitate\n",
      "C18218  16-Hydroxypalmitate\n",
      "C19418  cis-9,10-Epoxystearic acid\n",
      "C19614  16-Oxopalmitate\n",
      "C19615  Hexadecanedioate\n",
      "C19616  18-Hydroxyoleate\n",
      "C19617  18-Oxooleate\n",
      "C19618  Octadec-9-ene-1,18-dioic-acid\n",
      "C19620  9,10-Epoxy-18-hydroxystearate\n",
      "C19621  9,10,18-Trihydroxystearate\n",
      "C19622  9,10-Dihydroxystearate\n",
      "C19623  22-Hydroxydocosanoate\n",
      "C19624  22-Oxodocosanoate\n",
      "C19625  Docosanedioate\n",
      "C19630  Diketone\n",
      "REFERENCE   PMID:18621978\n",
      "AUTHORS   Li F, Wu X, Lam P, Bird D, Zheng H, Samuels L, Jetter R, Kunst L\n",
      "TITLE     Identification of the wax ester synthase/acyl-coenzyme A: diacylglycerol acyltransferase WSD1 required for stem wax ester biosynthesis in Arabidopsis.\n",
      "JOURNAL   Plant Physiol 148:97-107 (2008)\n",
      "DOI:10.1104/pp.108.123471\n",
      "REFERENCE   PMID:16866954\n",
      "AUTHORS   Shepherd T, Wynne Griffiths D\n",
      "TITLE     The effects of stress on plant cuticular waxes.\n",
      "JOURNAL   New Phytol 171:469-99 (2006)\n",
      "DOI:10.1111/j.1469-8137.2006.01826.x\n",
      "REFERENCE   PMID:19649180\n",
      "AUTHORS   Raffaele S, Leger A, Roby D\n",
      "TITLE     Very long chain fatty acid and lipid signaling in the response of plants to pathogens.\n",
      "JOURNAL   Plant Signal Behav 4:94-9 (2009)\n",
      "DOI:10.4161/psb.4.2.7580\n",
      "REFERENCE   PMID:16404574\n",
      "AUTHORS   Kurdyukov S, Faust A, Trenkamp S, Bar S, Franke R, Efremova N, Tietjen K, Schreiber L, Saedler H, Yephremov A\n",
      "TITLE     Genetic and biochemical evidence for involvement of HOTHEAD in the biosynthesis of long-chain alpha-,omega-dicarboxylic fatty acids and formation of extracellular matrix.\n",
      "JOURNAL   Planta 224:315-29 (2006)\n",
      "DOI:10.1007/s00425-005-0215-7\n",
      "REFERENCE   PMID:17323080\n",
      "AUTHORS   King A, Nam JW, Han J, Hilliard J, Jaworski JG\n",
      "TITLE     Cuticular wax biosynthesis in petunia petals: cloning and characterization of an alcohol-acyltransferase that synthesizes wax-esters.\n",
      "JOURNAL   Planta 226:381-94 (2007)\n",
      "DOI:10.1007/s00425-007-0489-z\n",
      "REFERENCE   PMID:18440267\n",
      "AUTHORS   Pollard M, Beisson F, Li Y, Ohlrogge JB\n",
      "TITLE     Building lipid barriers: biosynthesis of cutin and suberin.\n",
      "JOURNAL   Trends Plant Sci 13:236-46 (2008)\n",
      "DOI:10.1016/j.tplants.2008.03.003\n",
      "REFERENCE   PMID:19344744\n",
      "AUTHORS   Li Y, Beisson F\n",
      "TITLE     The biosynthesis of cutin and suberin as an alternative source of enzymes for the production of bio-based chemicals and materials.\n",
      "JOURNAL   Biochimie 91:685-91 (2009)\n",
      "DOI:10.1016/j.biochi.2009.03.016\n",
      "REL_PATHWAY ath00062  Fatty acid elongation\n",
      "ath00940  Phenylpropanoid biosynthesis\n",
      "ath01040  Biosynthesis of unsaturated fatty acids\n",
      "KO_PATHWAY  ko00073\n",
      "///\n",
      "ENTRY       ath00100                    Pathway\n",
      "NAME        Steroid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00100  Steroid biosynthesis\n",
      "MODULE      ath_M00917  Phytosterol biosynthesis, squalene 2,3-epoxide => campesterol/sitosterol [PATH:ath00100]\n",
      "DBLINKS     GO: 0016126\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G34650  SQS2; squalene synthase 2 [KO:K00801] [EC:2.5.1.21]\n",
      "AT4G34640  SQS1; squalene synthase 1 [KO:K00801] [EC:2.5.1.21]\n",
      "AT1G58440  XF1; FAD/NAD(P)-binding oxidoreductase family protein [KO:K00511] [EC:1.14.14.17]\n",
      "AT4G37760  SQE3; squalene epoxidase 3 [KO:K00511] [EC:1.14.14.17]\n",
      "AT5G24140  SQP2; squalene monooxygenase 2 [KO:K00511] [EC:1.14.14.17]\n",
      "AT5G24160  SQE6; squalene monooxygenase 6 [KO:K00511] [EC:1.14.14.17]\n",
      "AT2G22830  SQE2; squalene epoxidase 2 [KO:K00511] [EC:1.14.14.17]\n",
      "AT5G24150  SQP1; FAD/NAD(P)-binding oxidoreductase family protein [KO:K00511] [EC:1.14.14.17]\n",
      "AT3G45130  LAS1; lanosterol synthase 1 [KO:K01852] [EC:5.4.99.7]\n",
      "AT1G11680  CYP51G1; CYTOCHROME P450 51G1 [KO:K05917] [EC:1.14.14.154 1.14.15.36]\n",
      "AT3G52940  FK; Ergosterol biosynthesis ERG4/ERG24 family [KO:K00222] [EC:1.3.1.70]\n",
      "AT1G20050  HYD1; C-8,7 sterol isomerase [KO:K01824] [EC:5.3.3.5]\n",
      "AT3G19820  DWF1; cell elongation protein / DWARF1 / DIMINUTO (DIM) [KO:K09828] [EC:1.3.1.72 1.3.1.-]\n",
      "AT3G02580  STE1; sterol 1 [KO:K00227] [EC:1.14.19.20]\n",
      "AT3G02590  Fatty acid hydroxylase superfamily protein [KO:K00227] [EC:1.14.19.20]\n",
      "AT1G50430  DWF5; Ergosterol biosynthesis ERG4/ERG24 family [KO:K00213] [EC:1.3.1.21]\n",
      "AT2G15230  LIP1; lipase 1 [KO:K01052] [EC:3.1.1.13]\n",
      "AT3G57140  SDP1-LIKE; sugar-dependent 1-like protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G04040  SDP1; Patatin-like phospholipase family protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G13710  SMT1; sterol methyltransferase 1 [KO:K00559] [EC:2.1.1.41]\n",
      "AT2G07050  CAS1; cycloartenol synthase 1 [KO:K01853] [EC:5.4.99.8]\n",
      "AT4G12110  SMO1-1; sterol-4alpha-methyl oxidase 1-1 [KO:K14423] [EC:1.14.18.10]\n",
      "AT4G22756  SMO1-2; sterol C4-methyl oxidase 1-2 [KO:K14423] [EC:1.14.18.10]\n",
      "AT4G22755  methylsterol monooxygenase 1-3 [KO:K14423] [EC:1.14.18.10]\n",
      "AT2G26260  3BETAHSD/D2; 3beta-hydroxysteroid-dehydrogenase/decarboxylase isoform 2 [KO:K23558] [EC:1.1.1.418]\n",
      "AT2G43420  3-beta hydroxysteroid dehydrogenase/isomerase family protein [KO:K23558] [EC:1.1.1.418]\n",
      "AT1G47290  3BETAHSD/D1; 3beta-hydroxysteroid-dehydrogenase/decarboxylase isoform 1 [KO:K23558] [EC:1.1.1.418]\n",
      "AT5G50375  CPI1; cyclopropyl isomerase [KO:K08246] [EC:5.5.1.9]\n",
      "AT1G20330  SMT2; sterol methyltransferase 2 [KO:K08242] [EC:2.1.1.143]\n",
      "AT1G76090  SMT3; sterol methyltransferase 3 [KO:K08242] [EC:2.1.1.143]\n",
      "AT1G07420  SMO2-1; sterol 4-alpha-methyl-oxidase 2-1 [KO:K14424] [EC:1.14.18.11]\n",
      "AT2G29390  SMO2-2; sterol 4-alpha-methyl-oxidase 2-2 [KO:K14424] [EC:1.14.18.11]\n",
      "AT2G28850  CYP710A3; cytochrome P450, family 710, subfamily A, polypeptide 3 [KO:K09832] [EC:1.14.19.41]\n",
      "AT2G28860  CYP710A4; cytochrome P450, family 710, subfamily A, polypeptide 4 [KO:K09832] [EC:1.14.19.41]\n",
      "AT2G34490  CYP710A2; cytochrome P450, family 710, subfamily A, polypeptide 2 [KO:K09832] [EC:1.14.19.41]\n",
      "AT2G34500  CYP710A1; cytochrome P450, family 710, subfamily A, polypeptide 1 [KO:K09832] [EC:1.14.19.41]\n",
      "COMPOUND    C00187  Cholesterol\n",
      "C00448  trans,trans-Farnesyl diphosphate\n",
      "C00751  Squalene\n",
      "C01054  (S)-2,3-Epoxysqualene\n",
      "C01164  7-Dehydrocholesterol\n",
      "C01189  Lathosterol\n",
      "C01561  Calcidiol\n",
      "C01673  Calcitriol\n",
      "C01694  Ergosterol\n",
      "C01724  Lanosterol\n",
      "C01753  beta-Sitosterol\n",
      "C01789  Campesterol\n",
      "C01802  Desmosterol\n",
      "C01902  Cycloartenol\n",
      "C01943  Obtusifoliol\n",
      "C02141  Cycloeucalenol\n",
      "C02530  Cholesterol ester\n",
      "C03428  Presqualene diphosphate\n",
      "C03845  5alpha-Cholest-8-en-3beta-ol\n",
      "C04525  Fecosterol\n",
      "C05103  4alpha-Methylzymosterol\n",
      "C05107  7-Dehydrodesmosterol\n",
      "C05108  14-Demethyllanosterol\n",
      "C05109  24,25-Dihydrolanosterol\n",
      "C05437  Zymosterol\n",
      "C05439  5alpha-Cholesta-7,24-dien-3beta-ol\n",
      "C05440  Ergosta-5,7,22,24(28)-tetraen-3beta-ol\n",
      "C05441  Vitamin D2\n",
      "C05442  Stigmasterol\n",
      "C05443  Vitamin D3\n",
      "C07712  Secalciferol\n",
      "C08813  Brassicasterol\n",
      "C08821  Isofucosterol\n",
      "C08830  24-Methylidenecycloartanol\n",
      "C11455  4,4-Dimethyl-5alpha-cholesta-8,14,24-trien-3beta-ol\n",
      "C11508  4alpha-Methyl-5alpha-ergosta-8,14,24(28)-trien-3beta-ol\n",
      "C11522  24-Methylenelophenol\n",
      "C11523  24-Ethylidenelophenol\n",
      "C15776  4alpha-Methylfecosterol\n",
      "C15777  Episterol\n",
      "C15778  5,7,24(28)-Ergostatrienol\n",
      "C15780  5-Dehydroepisterol\n",
      "C15781  24-Methylenecholesterol\n",
      "C15782  Delta7-Avenasterol\n",
      "C15783  5-Dehydroavenasterol\n",
      "C15808  4alpha-Methylzymosterol-4-carboxylate\n",
      "C15816  3-Keto-4-methylzymosterol\n",
      "C15915  4,4-Dimethyl-5alpha-cholesta-8-en-3beta-ol\n",
      "C18231  Calcitetrol\n",
      "C21106  24-epi-Campesterol\n",
      "C22112  4alpha-Carboxy-5alpha-cholesta-8,24-dien-3beta-ol\n",
      "C22116  3beta-Hydroxy-4beta,14alpha-dimethyl-9beta,19-cyclo-5alpha-ergost-24(24(1))-en-4alpha-carboxylate\n",
      "C22119  3beta-Hydroxyergosta-7,24(24(1))-dien-4alpha-carboxylate\n",
      "C22120  4alpha-Carboxy-stigmasta-7,24(24(1))-dien-3beta-ol\n",
      "C22121  Cycloeucalenone\n",
      "C22122  Episterone\n",
      "C22123  Avenastenone\n",
      "C22136  Zymosterone\n",
      "REFERENCE\n",
      "AUTHORS   Koshiba K, Kamiya Y.\n",
      "TITLE     [Atarasii Syokubutuhorumon no Kagaku] (In Japanese)\n",
      "JOURNAL   Koudansya saientific:199-138 (2002)\n",
      "REFERENCE   PMID:16531502\n",
      "AUTHORS   Morikawa T, Mizutani M, Aoki N, Watanabe B, Saga H, Saito S, Oikawa A, Suzuki H, Sakurai N, Shibata D, Wadano A, Sakata K, Ohta D.\n",
      "TITLE     Cytochrome P450 CYP710A encodes the sterol C-22 desaturase in Arabidopsis and tomato.\n",
      "JOURNAL   Plant Cell 18:1008-22 (2006)\n",
      "DOI:10.1105/tpc.105.036012\n",
      "REFERENCE   PMID:12777052\n",
      "AUTHORS   Lange BM, Ghassemian M.\n",
      "TITLE     Genome organization in Arabidopsis thaliana: a survey for genes involved in isoprenoid and chlorophyll metabolism.\n",
      "JOURNAL   Plant Mol Biol 51:925-48 (2003)\n",
      "DOI:10.1023/A:1023005504702\n",
      "REL_PATHWAY ath00900  Terpenoid backbone biosynthesis\n",
      "ath00905  Brassinosteroid biosynthesis\n",
      "KO_PATHWAY  ko00100\n",
      "///\n",
      "ENTRY       ath00130                    Pathway\n",
      "NAME        Ubiquinone and other terpenoid-quinone biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Ubiquinone (UQ), also called coenzyme Q, and plastoquinone (PQ) are electron carriers in oxidative phosphorylation and photosynthesis, respectively. The quinoid nucleus of ubiquinone is derived from the shikimate pathway; 4-hydroxybenzoate is directly formed from chorismate in bacteria, while it can be formed from either chorismate or tyrosine in yeast. The following biosynthesis of terpenoid moiety involves reactions of prenylation, decarboxylation, and three hydroxylations alternating with three methylations. The order of these reactions are somewhat different between bacteria and yeast. Phylloquinone (vitamin K1), menaquinone (vitamin K2), and tocopherol (vitamin E) are fat-soluble vitamins. Phylloquinone is a compound present in all photosynthetic plants serving as a cofactor for photosystem I-mediated electron transport. Menaquinone is an obligatory component of the electron-transfer pathway in bacteria.\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00130  Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "MODULE      ath_M00112  Tocopherol/tocotorienol biosynthesis, homogentisate + phytyl/geranylgeranyl-PP => tocopherol/tocotorienol [PATH:ath00130]\n",
      "ath_M00932  Phylloquinone biosynthesis, chorismate (+ phytyl-PP) => phylloquinol [PATH:ath00130]\n",
      "ath_M00933  Plastoquinone biosynthesis, homogentisate + solanesyl-PP => plastoquinol [PATH:ath00130]\n",
      "DBLINKS     GO: 0006744 0009234 0042372\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G53970  TAT7; Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT5G36160  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT2G24850  TAT3; tyrosine aminotransferase 3 [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28410  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28420  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G23590  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT2G30490  C4H; cinnamate-4-hydroxylase [KO:K00487] [EC:1.14.14.91]\n",
      "AT4G05160  AMP-dependent synthetase and ligase family protein [KO:K01904] [EC:6.2.1.12]\n",
      "AT1G65060  4CL3; 4-coumarate:CoA ligase 3 [KO:K01904] [EC:6.2.1.12]\n",
      "AT1G62940  ACOS5; acyl-CoA synthetase 5 [KO:K01904] [EC:6.2.1.12]\n",
      "AT3G21240  4CL2; 4-coumarate:CoA ligase 2 [KO:K01904] [EC:6.2.1.12]\n",
      "AT1G51680  4CL1; 4-coumarate:CoA ligase 1 [KO:K01904] [EC:6.2.1.12]\n",
      "AT3G21230  4CL5; 4-coumarate:CoA ligase 5 [KO:K01904] [EC:6.2.1.12]\n",
      "AT4G19010  AMP-dependent synthetase and ligase family protein [KO:K01904] [EC:6.2.1.12]\n",
      "AT5G63380  AMP-dependent synthetase and ligase family protein [KO:K01904] [EC:6.2.1.12]\n",
      "AT4G23660  PPT1; polyprenyltransferase 1 [KO:K06125] [EC:2.5.1.39]\n",
      "AT3G24200  FAD/NAD(P)-binding oxidoreductase family protein [KO:K06126] [EC:1.14.13.-]\n",
      "AT2G30920  COQ3; coenzyme Q 3 [KO:K00591] [EC:2.1.1.114 2.1.1.64]\n",
      "AT5G57300  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K06127] [EC:2.1.1.201]\n",
      "AT1G74710  EDS16; ADC synthase superfamily protein [KO:K02552] [EC:5.4.4.2]\n",
      "AT1G18870  ICS2; isochorismate synthase 2 [KO:K02552] [EC:5.4.4.2]\n",
      "AT1G68890  PHYLLO; 2-oxoglutarate decarboxylase/hydro-lyase/magnesium ion-binding protein [KO:K14759] [EC:5.4.4.2 2.2.1.9 4.2.99.20 4.2.1.113]\n",
      "AT1G30520  AAE14; acyl-activating enzyme 14 [KO:K14760] [EC:6.2.1.26]\n",
      "AT1G60550  ECHID; enoyl-CoA hydratase/isomerase D [KO:K01661] [EC:4.1.3.36]\n",
      "AT1G48320  DHNAT1; Thioesterase superfamily protein [KO:K19222] [EC:3.1.2.28]\n",
      "AT5G48950  DHNAT2; Thioesterase superfamily protein [KO:K19222] [EC:3.1.2.28]\n",
      "AT1G60600  ABC4; UbiA prenyltransferase family protein [KO:K23094] [EC:2.5.1.130]\n",
      "AT5G08740  NDC1; NAD(P)H dehydrogenase C1 [KO:K17872] [EC:1.6.5.12]\n",
      "AT1G23360  MENG; S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K23095] [EC:2.1.1.329]\n",
      "AT4G36750  Quinone reductase family protein [KO:K03809] [EC:1.6.5.2]\n",
      "AT4G27270  Quinone reductase family protein [KO:K03809] [EC:1.6.5.2]\n",
      "AT5G58800  Quinone reductase family protein [KO:K03809] [EC:1.6.5.2]\n",
      "AT5G54500  FQR1; flavodoxin-like quinone reductase 1 [KO:K03809] [EC:1.6.5.2]\n",
      "AT1G06570  PDS1; 4-hydroxyphenylpyruvate dioxygenase [KO:K00457] [EC:1.13.11.27]\n",
      "AT3G11945  HST; homogentisate prenyltransferase [KO:K12501] [EC:2.5.1.117]\n",
      "AT3G63410  APG1; S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K12502] [EC:2.1.1.295]\n",
      "AT2G18950  HPT1; homogentisate phytyltransferase 1 [KO:K09833] [EC:2.5.1.115 2.5.1.116]\n",
      "AT4G32770  VTE1; tocopherol cyclase, chloroplast / vitamin E deficient 1 (VTE1) / sucrose export defective 1 (SXD1) [KO:K09834] [EC:5.5.1.24]\n",
      "AT1G64970  G-TMT; gamma-tocopherol methyltransferase [KO:K05928] [EC:2.1.1.95]\n",
      "COMPOUND    C00082  L-Tyrosine\n",
      "C00156  4-Hydroxybenzoate\n",
      "C00223  p-Coumaroyl-CoA\n",
      "C00251  Chorismate\n",
      "C00341  Geranyl diphosphate\n",
      "C00353  Geranylgeranyl diphosphate\n",
      "C00390  Ubiquinol\n",
      "C00423  trans-Cinnamate\n",
      "C00544  Homogentisate\n",
      "C00811  4-Coumarate\n",
      "C00828  Menaquinone\n",
      "C00885  Isochorismate\n",
      "C01179  3-(4-Hydroxyphenyl)pyruvate\n",
      "C02059  Phylloquinone\n",
      "C02477  alpha-Tocopherol\n",
      "C02483  gamma-Tocopherol\n",
      "C02730  2-Succinylbenzoate\n",
      "C02949  4-Hydroxybenzoyl-CoA\n",
      "C03160  2-Succinylbenzoyl-CoA\n",
      "C03313  Phylloquinol\n",
      "C03657  1,4-Dihydroxy-2-naphthoate\n",
      "C03964  (R)-3-(4-Hydroxyphenyl)lactate\n",
      "C03993  4-(beta-D-Glucosyloxy)benzoate\n",
      "C04145  all-trans-Nonaprenyl diphosphate\n",
      "C05427  Phytyl diphosphate\n",
      "C05807  2-Polyprenylphenol\n",
      "C05817  (1R,6R)-6-Hydroxy-2-succinylcyclohexa-2,4-diene-1-carboxylate\n",
      "C05819  Menaquinol\n",
      "C05847  all-trans-Polyprenyl diphosphate\n",
      "C05848  4-Hydroxy-3-polyprenylbenzoate\n",
      "C05849  Vitamin K1 epoxide\n",
      "C06986  Spirodilactone\n",
      "C10793  Geranylhydroquinone\n",
      "C13309  2-Phytyl-1,4-naphthoquinone\n",
      "C14151  delta-Tocopherol\n",
      "C14152  beta-Tocopherol\n",
      "C14153  alpha-Tocotrienol\n",
      "C14154  beta-Tocotrienol\n",
      "C14155  gamma-Tocotrienol\n",
      "C14156  delta-Tocotrienol\n",
      "C15547  1,4-Dihydroxy-2-naphthoyl-CoA\n",
      "C15882  2-Methyl-6-phytylquinol\n",
      "C15883  2,3-Dimethyl-5-phytylquinol\n",
      "C16519  2-Succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylate\n",
      "C16695  Plastoquinol-9\n",
      "C16999  Futalosine\n",
      "C17010  Dehypoxanthine futalosine\n",
      "C17017  Cyclic dehypoxanthine futalosine\n",
      "C17018  1,4-Dihydroxy-6-naphthoate\n",
      "C17412  Shikonin\n",
      "C17551  2-Polyprenyl-6-hydroxyphenol\n",
      "C17552  2-Polyprenyl-6-methoxyphenol\n",
      "C17554  3,4-Dihydroxy-5-polyprenylbenzoate\n",
      "C17559  3-Methoxy-4-hydroxy-5-polyprenylbenzoate\n",
      "C17570  2-Methyl-6-solanyl-1,4-benzoquinol\n",
      "C18131  Geranyl-hydroxybenzoate\n",
      "C18132  3''-Hydroxy-geranylhydroquinone\n",
      "C18133  Deoxyshikonin\n",
      "C18134  Dihydroechinofuran\n",
      "C18135  Dihydroshikonofuran\n",
      "C19847  Demethylmenaquinol\n",
      "C19858  2-Methoxy-6-all-trans-polyprenyl-1,4-benzoquinol\n",
      "C19859  6-Methoxy-3-methyl-2-all-trans-polyprenyl-1,4-benzoquinol\n",
      "C20265  2,3-Epoxymenaquinone\n",
      "C20737  6-Geranylgeranyl-2-methylbenzene-1,4-diol\n",
      "C20738  6-Geranylgeranyl-2,3-dimethylbenzene-1,4-diol\n",
      "C20772  3-[(1-Carboxyvinyl)oxy]benzoate\n",
      "C20773  6-Amino-6-deoxyfutalosine\n",
      "C21084  Demethylphylloquinol\n",
      "C21860  3-Demethylubiquinol\n",
      "C22039  2-Carboxy-1,4-naphthoquinone\n",
      "REFERENCE   PMID:12908086\n",
      "AUTHORS   Ward MJ, Fu QS, Rhoads KR, Yeung CH, Spormann AM, Criddle CS.\n",
      "TITLE     A derivative of the menaquinone precursor 1,4-dihydroxy-2-naphthoate is involved in the reductive transformation of carbon tetrachloride by aerobically grown Shewanella oneidensis MR-1.\n",
      "JOURNAL   Appl Microbiol Biotechnol 63:571-7 (2004)\n",
      "DOI:10.1007/s00253-003-1407-3\n",
      "REFERENCE   PMID:14499605\n",
      "AUTHORS   Monzingo AF, Gao J, Qiu J, Georgiou G, Robertus JD.\n",
      "TITLE     The X-ray structure of Escherichia coli RraA (MenG), A protein inhibitor of RNA processing.\n",
      "JOURNAL   J Mol Biol 332:1015-24 (2003)\n",
      "DOI:10.1016/S0022-2836(03)00970-7\n",
      "REFERENCE   PMID:11470786\n",
      "AUTHORS   Johnson TW, Zybailov B, Jones AD, Bittl R, Zech S, Stehlik D, Golbeck JH, Chitnis PR.\n",
      "TITLE     Recruitment of a foreign quinone into the A1 site of photosystem I. In vivo replacement of plastoquinone-9 by media-supplemented naphthoquinones in phylloquinone biosynthetic pathway mutants of Synechocystis sp. PCC 6803.\n",
      "JOURNAL   J Biol Chem 276:39512-21 (2001)\n",
      "DOI:10.1074/jbc.M104040200\n",
      "REFERENCE   PMID:12615349\n",
      "AUTHORS   Wade Johnson T, Naithani S, Stewart C Jr, Zybailov B, Daniel Jones A, Golbeck JH, Chitnis PR.\n",
      "TITLE     The menD and menE homologs code for 2-succinyl-6-hydroxyl-2,4-cyclohexadiene-1-carboxylate synthase and O-succinylbenzoic acid-CoA synthase in the phylloquinone biosynthetic pathway of Synechocystis sp. PCC 6803.\n",
      "JOURNAL   Biochim Biophys Acta 1557:67-76 (2003)\n",
      "DOI:10.1016/S0005-2728(02)00396-1\n",
      "REFERENCE   PMID:12721307\n",
      "AUTHORS   Gin P, Hsu AY, Rothman SC, Jonassen T, Lee PT, Tzagoloff A, Clarke CF.\n",
      "TITLE     The Saccharomyces cerevisiae COQ6 gene encodes a mitochondrial flavin-dependent monooxygenase required for coenzyme Q biosynthesis.\n",
      "JOURNAL   J Biol Chem 278:25308-16 (2003)\n",
      "DOI:10.1074/jbc.M303234200\n",
      "REFERENCE   PMID:18284213\n",
      "AUTHORS   Jiang M, Chen X, Guo ZF, Cao Y, Chen M, Guo Z.\n",
      "TITLE     Identification and Characterization of (1R,6R)-2-Succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate Synthase in the Menaquinone Biosynthesis of Escherichia coli.\n",
      "JOURNAL   Biochemistry 47:3426-34 (2008)\n",
      "DOI:10.1021/bi7023755\n",
      "REFERENCE   PMID:17760421\n",
      "AUTHORS   Jiang M, Cao Y, Guo ZF, Chen M, Chen X, Guo Z.\n",
      "TITLE     Menaquinone biosynthesis in Escherichia coli: identification of 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylate as a novel intermediate and re-evaluation of MenD activity.\n",
      "JOURNAL   Biochemistry 46:10979-89 (2007)\n",
      "DOI:10.1021/bi700810x\n",
      "REFERENCE   PMID:17956107\n",
      "AUTHORS   Jiang M, Chen M, Cao Y, Yang Y, Sze KH, Chen X, Guo Z.\n",
      "TITLE     Determination of the stereochemistry of 2-succinyl-5-enolpyruvyl-6-hydroxy-3- cyclohexene-1-carboxylate, a key intermediate in menaquinone biosynthesis.\n",
      "JOURNAL   Org Lett 9:4765-7 (2007)\n",
      "DOI:10.1021/ol702126m\n",
      "REFERENCE   PMID:18801996\n",
      "AUTHORS   Hiratsuka T, Furihata K, Ishikawa J, Yamashita H, Itoh N, Seto H, Dairi T\n",
      "TITLE     An alternative menaquinone biosynthetic pathway operating in microorganisms.\n",
      "JOURNAL   Science 321:1670-3 (2008)\n",
      "DOI:10.1126/science.1160446\n",
      "REFERENCE   PMID:12232327\n",
      "AUTHORS   Loscher R, Heide L\n",
      "TITLE     Biosynthesis of p-Hydroxybenzoate from p-Coumarate and p-Coumaroyl-Coenzyme A in Cell-Free Extracts of Lithospermum erythrorhizon Cell Cultures.\n",
      "JOURNAL   Plant Physiol 106:271-279 (1994)\n",
      "DOI:10.1104/pp.106.1.271\n",
      "REFERENCE   PMID:11583838\n",
      "AUTHORS   Meganathan R\n",
      "TITLE     Ubiquinone biosynthesis in microorganisms.\n",
      "JOURNAL   FEMS Microbiol Lett 203:131-9 (2001)\n",
      "DOI:10.1111/j.1574-6968.2001.tb10831.x\n",
      "REFERENCE   PMID:14508009\n",
      "AUTHORS   Cheng Z, Sattler S, Maeda H, Sakuragi Y, Bryant DA, DellaPenna D\n",
      "TITLE     Highly divergent methyltransferases catalyze a conserved reaction in tocopherol and plastoquinone synthesis in cyanobacteria and photosynthetic eukaryotes.\n",
      "JOURNAL   Plant Cell 15:2343-56 (2003)\n",
      "DOI:10.1105/tpc.013656\n",
      "REFERENCE   PMID:16989822\n",
      "AUTHORS   Sadre R, Gruber J, Frentzen M\n",
      "TITLE     Characterization of homogentisate prenyltransferases involved in plastoquinone-9 and tocochromanol biosynthesis.\n",
      "JOURNAL   FEBS Lett 580:5357-62 (2006)\n",
      "DOI:10.1016/j.febslet.2006.09.002\n",
      "REFERENCE   PMID:12523993\n",
      "AUTHORS   Hofius D, Sonnewald U.\n",
      "TITLE     Vitamin E biosynthesis: biochemistry meets cell biology.\n",
      "JOURNAL   Trends Plant Sci 8:6-8 (2003)\n",
      "DOI:10.1016/S1360-1385(02)00002-X\n",
      "REFERENCE   PMID:16233343\n",
      "AUTHORS   Kawamukai M\n",
      "TITLE     Biosynthesis, bioproduction and novel roles of ubiquinone.\n",
      "JOURNAL   J Biosci Bioeng 94:511-7 (2002)\n",
      "DOI:10.1016/S1389-1723(02)80188-8\n",
      "REFERENCE   PMID:21239697\n",
      "AUTHORS   Tie JK, Jin DY, Straight DL, Stafford DW\n",
      "TITLE     Functional study of the vitamin K cycle in mammalian cells.\n",
      "JOURNAL   Blood 117:2967-74 (2011)\n",
      "DOI:10.1182/blood-2010-08-304303\n",
      "REFERENCE   PMID:23113660\n",
      "AUTHORS   Zhou L, Wang JY, Wu J, Wang J, Poplawsky A, Lin S, Zhu B, Chang C, Zhou T, Zhang LH, He YW\n",
      "TITLE     The diffusible factor synthase XanB2 is a bifunctional chorismatase that links the shikimate pathway to ubiquinone and xanthomonadins biosynthetic pathways.\n",
      "JOURNAL   Mol Microbiol 87:80-93 (2013)\n",
      "DOI:10.1111/mmi.12084\n",
      "REL_PATHWAY ath00190  Oxidative phosphorylation\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00900  Terpenoid backbone biosynthesis\n",
      "ath00940  Phenylpropanoid biosynthesis\n",
      "KO_PATHWAY  ko00130\n",
      "///\n",
      "ENTRY       ath00190                    Pathway\n",
      "NAME        Oxidative phosphorylation - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Energy metabolism\n",
      "PATHWAY_MAP ath00190  Oxidative phosphorylation\n",
      "MODULE      ath_M00142  NADH:ubiquinone oxidoreductase, mitochondria [PATH:ath00190]\n",
      "ath_M00143  NADH dehydrogenase (ubiquinone) Fe-S protein/flavoprotein complex, mitochondria [PATH:ath00190]\n",
      "ath_M00145  NAD(P)H:quinone oxidoreductase, chloroplasts and cyanobacteria [PATH:ath00190]\n",
      "ath_M00151  Cytochrome bc1 complex respiratory unit [PATH:ath00190]\n",
      "ath_M00157  F-type ATPase, prokaryotes and chloroplasts [PATH:ath00190]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        ArthCp079  ndhA; NADH dehydrogenase subunit 1 [KO:K05572] [EC:7.1.1.2]\n",
      "ArthCp068  ndhB; NADH dehydrogenase subunit 2 [KO:K05573] [EC:7.1.1.2]\n",
      "ArthCp086  ndhB; NADH dehydrogenase subunit 2 [KO:K05573] [EC:7.1.1.2]\n",
      "ArthCp027  ndhC; NADH dehydrogenase subunit 3 [KO:K05574] [EC:7.1.1.2]\n",
      "ArthCp074  ndhD; NADH dehydrogenase subunit 4 [KO:K05575] [EC:7.1.1.2]\n",
      "ArthCp076  ndhE; NADH dehydrogenase subunit 4L [KO:K05576] [EC:7.1.1.2]\n",
      "ArthCp071  ndhF; NADH dehydrogenase subunit 5 [KO:K05577] [EC:7.1.1.2]\n",
      "ArthCp077  ndhG; NADH dehydrogenase subunit 6 [KO:K05578] [EC:7.1.1.2]\n",
      "ArthCp080  ndhH; NADH dehydrogenase subunit 7 [KO:K05579] [EC:7.1.1.2]\n",
      "ArthCp078  ndhI; NADH dehydrogenase subunit I [KO:K05580] [EC:7.1.1.2]\n",
      "ArthCp025  ndhJ; NADH dehydrogenase subunit J [KO:K05581] [EC:7.1.1.2]\n",
      "ArthCp026  ndhK; NADH dehydrogenase subunit K [KO:K05582] [EC:7.1.1.2]\n",
      "AT2G07785  NADH dehydrogenase family protein [KO:K03878] [EC:7.1.1.2]\n",
      "DA397_mgp34  nad1; NADH dehydrogenase subunit 1 [KO:K03878] [EC:7.1.1.2]\n",
      "AT2G07689  NADH-Ubiquinone/plastoquinone (complex I) protein [KO:K03879] [EC:7.1.1.2]\n",
      "DA397_mgp31  nad2; NADH dehydrogenase subunit 2 [KO:K03879] [EC:7.1.1.2]\n",
      "AT2G07751  NADH:ubiquinone/plastoquinone oxidoreductase, chain 3 protein [KO:K03880] [EC:7.1.1.2]\n",
      "DA397_mgp14  nad3; NADH dehydrogenase subunit 3 [KO:K03880] [EC:7.1.1.2]\n",
      "DA397_mgp23  nad4; NADH dehydrogenase subunit 4 [KO:K03881] [EC:7.1.1.2]\n",
      "DA397_mgp04  nad4L; NADH dehydrogenase subunit 4L [KO:K03882] [EC:7.1.1.2]\n",
      "DA397_mgp37  nad5; NADH dehydrogenase subunit 5 [KO:K03883] [EC:7.1.1.2]\n",
      "DA397_mgp28  nad6; NADH dehydrogenase subunit 6 [KO:K03884] [EC:7.1.1.2]\n",
      "AT5G37510  EMB1467; NADH-ubiquinone dehydrogenase [KO:K03934] [EC:7.1.1.2]\n",
      "DA397_mgp19  nad7; NADH dehydrogenase subunit 7 [KO:K03935] [EC:7.1.1.2]\n",
      "DA397_mgp38  nad9; NADH dehydrogenase subunit 9 [KO:K03936] [EC:7.1.1.2]\n",
      "AT5G67590  FRO1; NADH-ubiquinone oxidoreductase-like protein [KO:K03937]\n",
      "AT2G47690  NADH-ubiquinone oxidoreductase-like protein [KO:K03938]\n",
      "AT3G62790  NADH-ubiquinone oxidoreductase-like protein [KO:K03938]\n",
      "AT3G03070  NADH-ubiquinone oxidoreductase-like protein [KO:K03939]\n",
      "AT5G11770  NADH-ubiquinone oxidoreductase 20 kDa subunit [KO:K03940] [EC:7.1.1.2]\n",
      "AT1G79010  Alpha-helical ferredoxin [KO:K03941] [EC:7.1.1.2]\n",
      "AT1G16700  Alpha-helical ferredoxin [KO:K03941] [EC:7.1.1.2]\n",
      "AT5G08530  CI51; 51 kDa subunit of complex I [KO:K03942] [EC:7.1.1.2]\n",
      "AT4G02580  NADH-ubiquinone oxidoreductase 24 kDa subunit [KO:K03943] [EC:7.1.1.2]\n",
      "AT3G08610  NADH dehydrogenase ubiquinone 1 alpha subcomplex subunit [KO:K03945]\n",
      "AT5G47890  NADH-ubiquinone oxidoreductase B8 subunit [KO:K03946]\n",
      "AT5G52840  NADH-ubiquinone oxidoreductase-like protein [KO:K03949]\n",
      "AT3G12260  LYR family of Fe/S cluster biogenesis protein [KO:K03950]\n",
      "AT3G06310  Cox19-like CHCH family protein [KO:K03952]\n",
      "AT5G18800  Cox19-like CHCH family protein [KO:K03952]\n",
      "AT2G20360  NAD(P)-binding Rossmann-fold superfamily protein [KO:K03953]\n",
      "AT1G65290  mtACP2; mitochondrial acyl carrier protein 2 [KO:K03955]\n",
      "AT2G44620  MTACP-1; mitochondrial acyl carrier protein 1 [KO:K03955]\n",
      "AT5G47630  mtACP3; mitochondrial acyl carrier protein 3 [KO:K03955]\n",
      "AT3G03100  NADH:ubiquinone oxidoreductase, 17.2kDa subunit [KO:K11352]\n",
      "AT2G33220  GRIM-19 protein [KO:K11353]\n",
      "AT1G04630  MEE4; GRIM-19 protein [KO:K11353]\n",
      "AT2G02050  NADH-ubiquinone oxidoreductase B18 subunit [KO:K03963]\n",
      "AT4G34700  CIB22; LYR family of Fe/S cluster biogenesis protein [KO:K03965]\n",
      "AT1G49140  NADH dehydrogenase ubiquinone 1 beta subcomplex subunit 10-B-like protein (Complex I subunit NDUFS6) [KO:K03966]\n",
      "AT3G18410  NADH dehydrogenase ubiquinone 1 beta subcomplex subunit 10-B-like protein (Complex I subunit NDUFS6) [KO:K03966]\n",
      "AT5G66760  SDH1-1; succinate dehydrogenase 1-1 [KO:K00234] [EC:1.3.5.1]\n",
      "AT2G18450  SDH1-2; succinate dehydrogenase 1-2 [KO:K00234] [EC:1.3.5.1]\n",
      "AT5G40650  SDH2-2; succinate dehydrogenase 2-2 [KO:K00235] [EC:1.3.5.1]\n",
      "AT5G65165  SDH2-3; succinate dehydrogenase 2-3 [KO:K00235] [EC:1.3.5.1]\n",
      "AT3G27380  SDH2-1; succinate dehydrogenase 2-1 [KO:K00235] [EC:1.3.5.1]\n",
      "AT4G32210  SDH3-2; succinate dehydrogenase 3-2 [KO:K00236]\n",
      "AT5G09600  SDH3-1; succinate dehydrogenase 3-1 [KO:K00236]\n",
      "AT5G13430  Ubiquinol-cytochrome C reductase iron-sulfur subunit [KO:K00411] [EC:7.1.1.8]\n",
      "AT5G13440  Ubiquinol-cytochrome C reductase iron-sulfur subunit [KO:K00411] [EC:7.1.1.8]\n",
      "AT2G07727  cytochrome b [KO:K00412]\n",
      "DA397_mgp27  cob; apocytochrome b [KO:K00412]\n",
      "AT3G27240  Cytochrome C1 family [KO:K00413]\n",
      "AT5G40810  Cytochrome C1 family [KO:K00413]\n",
      "AT1G15120  Ubiquinol-cytochrome C reductase hinge protein [KO:K00416]\n",
      "AT2G01090  Ubiquinol-cytochrome C reductase hinge protein [KO:K00416]\n",
      "AT4G32470  Cytochrome bd ubiquinol oxidase, 14kDa subunit [KO:K00417]\n",
      "AT5G25450  Cytochrome bd ubiquinol oxidase, 14kDa subunit [KO:K00417]\n",
      "AT3G52730  ubiquinol-cytochrome C reductase UQCRX/QCR9-like family protein [KO:K00419]\n",
      "AT2G44520  COX10; cytochrome c oxidase 10 [KO:K02257] [EC:2.5.1.141]\n",
      "AT2G07687  Cytochrome c oxidase, subunit III [KO:K02262]\n",
      "DA397_mgp07  cox3; cytochrome c oxidase subunit 3 [KO:K02262]\n",
      "DA397_mgp30  cox1; cytochrome c oxidase subunit 1 [KO:K02256] [EC:7.1.1.9]\n",
      "DA397_mgp01  cox2; cytochrome c oxidase subunit 2 [KO:K02261]\n",
      "AT1G80230  Rubredoxin-like superfamily protein [KO:K02265]\n",
      "AT3G15640  Rubredoxin-like superfamily protein [KO:K02265]\n",
      "AT4G37830  cytochrome c oxidase-like protein [KO:K02266]\n",
      "AT4G28060  Cytochrome c oxidase, subunit Vib family protein [KO:K02267]\n",
      "AT1G22450  COX6B; cytochrome C oxidase 6B [KO:K02267]\n",
      "AT1G32710  Cytochrome c oxidase, subunit Vib family protein [KO:K02267]\n",
      "AT5G57815  Cytochrome c oxidase, subunit Vib family protein [KO:K02267]\n",
      "AT1G02410  cytochrome c oxidase assembly protein CtaG / Cox11 family [KO:K02258]\n",
      "AT5G56090  COX15; cytochrome c oxidase 15 [KO:K02259] [EC:1.17.99.9]\n",
      "AT3G15352  COX17; cytochrome c oxidase 17 [KO:K02260]\n",
      "AT1G53030  Cytochrome C oxidase copper chaperone (COX17) [KO:K02260]\n",
      "AT4G10040  CYTC-2; cytochrome c-2 [KO:K08738]\n",
      "AT1G22840  CYTC-1; CYTOCHROME C-1 [KO:K08738]\n",
      "ArthCp007  atpA; ATP synthase CF1 alpha subunit [KO:K02111] [EC:7.1.2.2 7.2.2.1]\n",
      "ArthCp029  atpB; ATP synthase CF1 beta subunit [KO:K02112] [EC:7.1.2.2 7.2.2.1]\n",
      "AT1G15700  ATPC2; ATPase, F1 complex, gamma subunit protein [KO:K02115]\n",
      "AT4G04640  ATPC1; ATPase, F1 complex, gamma subunit protein [KO:K02115]\n",
      "AT4G09650  ATPD; F-type H+-transporting ATPase subunit delta [KO:K02113]\n",
      "ArthCp028  atpE; ATP synthase CF1 epsilon subunit [KO:K02114]\n",
      "ArthCp010  atpI; ATP synthase CF0 A subunit [KO:K02108]\n",
      "ArthCp008  atpF; ATP synthase CF0 B subunit [KO:K02109]\n",
      "AT4G32260  PDE334; ATPase, F0 complex, subunit B/B', bacterial/chloroplast [KO:K02109]\n",
      "AT2G07707  Plant mitochondrial ATPase, F0 complex, subunit 8 protein [KO:K02109]\n",
      "DA397_mgp18  atp8; ATPase subunit 8 [KO:K02109]\n",
      "ArthCp009  atpH; ATP synthase CF0 C subunit [KO:K02110]\n",
      "AT2G07698  ATPase, F1 complex, alpha subunit protein [KO:K02132]\n",
      "DA397_mgp33  atp1; ATPase subunit 1 [KO:K02132]\n",
      "AT5G08670  ATP synthase alpha/beta family protein [KO:K02133] [EC:7.1.2.2]\n",
      "AT5G08680  ATP synthase alpha/beta family protein [KO:K02133] [EC:7.1.2.2]\n",
      "AT5G08690  ATP synthase alpha/beta family protein [KO:K02133] [EC:7.1.2.2]\n",
      "AT2G33040  ATP3; gamma subunit of Mt ATP synthase [KO:K02136]\n",
      "AT5G47030  ATPase, F1 complex, delta/epsilon subunit [KO:K02134]\n",
      "AT1G51650  ATP synthase epsilon chain [KO:K02135]\n",
      "AT5G13450  ATP5; delta subunit of Mt ATP synthase [KO:K02137]\n",
      "AT2G07741  ATPase, F0 complex, subunit A protein [KO:K02126]\n",
      "DA397_mgp35  atp6; ATPase subunit 6 [KO:K02126]\n",
      "DA397_mgp17  atp6; ATPase subunit 6 [KO:K02126]\n",
      "AT2G07671  ATP synthase subunit C family protein [KO:K02128]\n",
      "DA397_mgp16  atp9; ATPase subunit 9 [KO:K02128]\n",
      "AT3G52300  ATPQ; ATP synthase D chain [KO:K02138]\n",
      "AT2G19680  Mitochondrial ATP synthase subunit G protein [KO:K02140]\n",
      "AT4G29480  Mitochondrial ATP synthase subunit G protein [KO:K02140]\n",
      "AT4G26210  Mitochondrial ATP synthase subunit G protein [KO:K02140]\n",
      "AT1G78900  VHA-A; vacuolar ATP synthase subunit A [KO:K02145] [EC:7.1.2.2]\n",
      "AT1G76030  VAB1; ATPase, V1 complex, subunit B protein [KO:K02147]\n",
      "AT1G20260  VAB3; ATPase, V1 complex, subunit B protein [KO:K02147]\n",
      "AT4G38510  VAB2; ATPase, V1 complex, subunit B protein [KO:K02147]\n",
      "AT1G12840  DET3; vacuolar ATP synthase subunit C (VATC) / V-ATPase C subunit / vacuolar proton pump C subunit (DET3) [KO:K02148]\n",
      "AT3G58730  vacuolar ATP synthase subunit D (VATD) / V-ATPase D subunit / vacuolar proton pump D subunit (VATPD) [KO:K02149]\n",
      "AT1G64200  VHA-E3; vacuolar H+-ATPase subunit E isoform 3 [KO:K02150]\n",
      "AT3G08560  VHA-E2; vacuolar H+-ATPase subunit E isoform 2 [KO:K02150]\n",
      "AT4G11150  TUF; vacuolar ATP synthase subunit E1 [KO:K02150]\n",
      "AT4G02620  vacuolar ATPase subunit F family protein [KO:K02151]\n",
      "AT4G23710  VAG2; vacuolar ATP synthase subunit G2 [KO:K02152]\n",
      "AT4G25950  VATG3; vacuolar ATP synthase G3 [KO:K02152]\n",
      "AT3G01390  VMA10; vacuolar membrane ATPase 10 [KO:K02152]\n",
      "AT3G42050  vacuolar ATP synthase subunit H family protein [KO:K02144]\n",
      "AT2G21410  VHA-A2; vacuolar proton ATPase A2 [KO:K02154]\n",
      "AT2G28520  VHA-A1; vacuolar proton ATPase A1 [KO:K02154]\n",
      "AT4G39080  VHA-A3; vacuolar proton ATPase A3 [KO:K02154]\n",
      "AT1G19910  AVA-P2; ATPase, F0/V0 complex, subunit C protein [KO:K02155]\n",
      "AT1G75630  AVA-P4; vacuolar H+-pumping ATPase 16 kDa proteolipid subunit 4 [KO:K02155]\n",
      "AT2G16510  ATPase, F0/V0 complex, subunit C protein [KO:K02155]\n",
      "AT4G34720  AVA-P1; ATPase, F0/V0 complex, subunit C protein [KO:K02155]\n",
      "AT4G38920  VHA-C3; vacuolar-type H[+]-ATPase C3 [KO:K02155]\n",
      "AT2G25610  ATPase, F0/V0 complex, subunit C protein [KO:K03661]\n",
      "AT4G32530  ATPase, F0/V0 complex, subunit C protein [KO:K03661]\n",
      "AT3G28710  ATPase, V0/A0 complex, subunit C/D [KO:K02146]\n",
      "AT3G28715  ATPase, V0/A0 complex, subunit C/D [KO:K02146]\n",
      "AT5G55290  ATPase, V0 complex, subunit E [KO:K02153]\n",
      "AT4G26710  ATPase, V0 complex, subunit E [KO:K02153]\n",
      "AT2G24520  HA5; plasma membrane H+-ATPase [KO:K01535] [EC:7.1.2.1]\n",
      "AT5G57350  HA3; H[+]-ATPase 3 [KO:K01535] [EC:7.1.2.1]\n",
      "AT3G47950  HA4; H[+]-ATPase 4 [KO:K01535] [EC:7.1.2.1]\n",
      "AT3G60330  HA7; H[+]-ATPase 7 [KO:K01535] [EC:7.1.2.1]\n",
      "AT1G80660  HA9; H[+]-ATPase 9 [KO:K01535] [EC:7.1.2.1]\n",
      "AT2G07560  HA6; H[+]-ATPase 6 [KO:K01535] [EC:7.1.2.1]\n",
      "AT2G18960  HA1; H[+]-ATPase 1 [KO:K01535] [EC:7.1.2.1]\n",
      "AT4G30190  HA2; H[+]-ATPase 2 [KO:K01535] [EC:7.1.2.1]\n",
      "AT1G17260  AHA10; autoinhibited H[+]-ATPase isoform 10 [KO:K01535] [EC:7.1.2.1]\n",
      "AT5G62670  HA11; H[+]-ATPase 11 [KO:K01535] [EC:7.1.2.1]\n",
      "AT3G42640  HA8; H[+]-ATPase 8 [KO:K01535] [EC:7.1.2.1]\n",
      "AT4G11730  Cation transporter/ E1-E2 ATPase family protein [KO:K01535] [EC:7.1.2.1]\n",
      "AT2G46860  PPa3; pyrophosphorylase 3 [KO:K01507] [EC:3.6.1.1]\n",
      "AT3G53620  PPa4; pyrophosphorylase 4 [KO:K01507] [EC:3.6.1.1]\n",
      "AT4G01480  PPa5; pyrophosphorylase 5 [KO:K01507] [EC:3.6.1.1]\n",
      "AT1G01050  PPa1; pyrophosphorylase 1 [KO:K01507] [EC:3.6.1.1]\n",
      "AT2G18230  PPa2; pyrophosphorylase 2 [KO:K01507] [EC:3.6.1.1]\n",
      "AT5G09650  PPa6; pyrophosphorylase 6 [KO:K01507] [EC:3.6.1.1]\n",
      "COMPOUND    C00001  H2O\n",
      "C00002  ATP\n",
      "C00003  NAD+\n",
      "C00004  NADH\n",
      "C00007  Oxygen\n",
      "C00008  ADP\n",
      "C00009  Orthophosphate\n",
      "C00013  Diphosphate\n",
      "C00042  Succinate\n",
      "C00061  FMN\n",
      "C00080  H+\n",
      "C00122  Fumarate\n",
      "C00390  Ubiquinol\n",
      "C00399  Ubiquinone\n",
      "C00524  Cytochrome c\n",
      "C00536  Triphosphate\n",
      "REFERENCE   PMID:16469879\n",
      "AUTHORS   Sazanov LA, Hinchliffe P.\n",
      "TITLE     Structure of the hydrophilic domain of respiratory complex I from Thermus thermophilus.\n",
      "JOURNAL   Science 311:1430-6 (2006)\n",
      "DOI:10.1126/science.1123809\n",
      "REFERENCE   PMID:16584177\n",
      "AUTHORS   Hinchliffe P, Carroll J, Sazanov LA.\n",
      "TITLE     Identification of a novel subunit of respiratory complex I from Thermus thermophilus.\n",
      "JOURNAL   Biochemistry 45:4413-20 (2006)\n",
      "DOI:10.1021/bi0600998\n",
      "KO_PATHWAY  ko00190\n",
      "///\n",
      "ENTRY       ath00195                    Pathway\n",
      "NAME        Photosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Photosynthesis in green plants and specialized bacteria is the process of utilizing light energy to synthesize organic compounds from carbon dioxide and water. It consists of the light dependent part (light reaction) and the light independent part (dark reaction, carbon fixation). The light reaction takes place in thylakoid, a membrane-bound compartment inside chloroplasts and cyanobacteria. The light energy is used by photosystems I and II to generate proton motive force and reducing power (NADPH or NADH). The proton motive force is used by ATP synthase to generate ATP, essentially in the same way as the mitochondrial respiratory chain. The supplies of ATP and NAD(P)H are then used to fix carbon dioxide.\n",
      "CLASS       Metabolism; Energy metabolism\n",
      "PATHWAY_MAP ath00195  Photosynthesis\n",
      "MODULE      ath_M00157  F-type ATPase, prokaryotes and chloroplasts [PATH:ath00195]\n",
      "ath_M00161  Photosystem II [PATH:ath00195]\n",
      "ath_M00163  Photosystem I [PATH:ath00195]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        ArthCp002  psbA; photosystem II protein D1 [KO:K02703] [EC:1.10.3.9]\n",
      "ArthCp049  psbB; photosystem II 47 kDa protein [KO:K02704]\n",
      "ArthCp018  psbC; photosystem II 44 kDa protein [KO:K02705]\n",
      "ArthCp017  psbD; photosystem II protein D2 [KO:K02706] [EC:1.10.3.9]\n",
      "ArthCp039  psbE; photosystem II protein V [KO:K02707]\n",
      "ArthCp038  psbF; photosystem II protein VI [KO:K02708]\n",
      "ArthCp052  psbH; photosystem II protein H [KO:K02709]\n",
      "ArthCp006  psbI; photosystem II protein I [KO:K02710]\n",
      "ArthCp036  psbJ; photosystem II protein J [KO:K02711]\n",
      "ArthCp005  psbK; photosystem II protein K [KO:K02712]\n",
      "ArthCp037  psbL; photosystem II protein L [KO:K02713]\n",
      "ArthCp016  psbM; photosystem II protein M [KO:K02714]\n",
      "AT5G66570  PSBO1; PS II oxygen-evolving complex 1 [KO:K02716]\n",
      "AT3G50820  PSBO2; photosystem II subunit O-2 [KO:K02716]\n",
      "AT1G06680  PSBP-1; photosystem II subunit P-1 [KO:K02717]\n",
      "AT2G30790  PSBP-2; photosystem II subunit P-2 [KO:K02717]\n",
      "AT3G55330  PPL1; PsbP-like protein 1 [KO:K02717]\n",
      "AT1G14150  PnsL2; PsbQ-like 2 [KO:K08901]\n",
      "AT3G01440  PnsL3; PsbQ-like 1 [KO:K08901]\n",
      "AT4G05180  PSBQ-2; photosystem II subunit Q-2 [KO:K08901]\n",
      "AT4G21280  PSBQA; photosystem II subunit QA [KO:K08901]\n",
      "AT1G79040  PSBR; photosystem II subunit R [KO:K03541]\n",
      "AT1G44575  NPQ4; Chlorophyll A-B binding family protein [KO:K03542]\n",
      "ArthCp050  psbT; photosystem II protein T [KO:K02718]\n",
      "AT2G30570  PSBW; photosystem II reaction center W [KO:K02721]\n",
      "AT1G67740  PSBY; photosystem II BY [KO:K02723]\n",
      "ArthCp019  psbZ; photosystem II protein Z [KO:K02724]\n",
      "AT1G03600  PSB27; photosystem II family protein [KO:K08902]\n",
      "AT4G28660  PSB28; photosystem II reaction center PSB28 protein [KO:K08903]\n",
      "ArthCp022  psaA; photosystem I P700 chlorophyll a apoprotein A1 [KO:K02689]\n",
      "ArthCp021  psaB; photosystem I P700 chlorophyll a apoprotein A2 [KO:K02690]\n",
      "ArthCp075  psaC; photosystem I subunit VII [KO:K02691]\n",
      "AT4G02770  PSAD-1; photosystem I subunit D-1 [KO:K02692]\n",
      "AT1G03130  PSAD-2; photosystem I subunit D-2 [KO:K02692]\n",
      "AT4G28750  PSAE-1; Photosystem I reaction centre subunit IV / PsaE protein [KO:K02693]\n",
      "AT2G20260  PSAE-2; photosystem I subunit E-2 [KO:K02693]\n",
      "AT1G31330  PSAF; photosystem I subunit F [KO:K02694]\n",
      "AT1G55670  PSAG; photosystem I subunit G [KO:K08905]\n",
      "AT1G52230  PSAH2; photosystem I subunit H2 [KO:K02695]\n",
      "AT3G16140  PSAH-1; photosystem I subunit H-1 [KO:K02695]\n",
      "ArthCp032  psaI; photosystem I subunit VIII [KO:K02696]\n",
      "ArthCp042  psaJ; photosystem I subunit IX [KO:K02697]\n",
      "AT1G30380  PSAK; photosystem I subunit K [KO:K02698]\n",
      "AT4G12800  PSAL; photosystem I subunit l [KO:K02699]\n",
      "AT5G64040  PSAN; photosystem I reaction center subunit PSI-N, chloroplast, putative / PSI-N, putative (PSAN) [KO:K02701]\n",
      "AT1G08380  PSAO; photosystem I subunit O [KO:K14332]\n",
      "ArthCp035  petA; cytochrome f [KO:K02634]\n",
      "ArthCp053  petB; cytochrome b6 [KO:K02635]\n",
      "AT4G03280  PETC; photosynthetic electron transfer C [KO:K02636] [EC:7.1.1.6]\n",
      "ArthCp054  petD; cytochrome b6/f complex subunit IV [KO:K02637]\n",
      "ArthCp041  petG; cytochrome b6/f complex subunit V [KO:K02640]\n",
      "ArthCp015  petN; cytochrome b6/f complex subunit VIII [KO:K03689]\n",
      "AT1G20340  DRT112; Cupredoxin superfamily protein [KO:K02638]\n",
      "AT1G76100  PETE1; plastocyanin 1 [KO:K02638]\n",
      "AT1G10960  FD1; ferredoxin 1 [KO:K02639]\n",
      "AT1G60950  FED_A; 2Fe-2S ferredoxin-like superfamily protein [KO:K02639]\n",
      "AT2G27510  FD3; ferredoxin 3 [KO:K02639]\n",
      "AT4G14890  FdC1; 2Fe-2S ferredoxin-like superfamily protein [KO:K02639]\n",
      "AT5G10000  FD4; ferredoxin 4 [KO:K02639]\n",
      "AT1G32550  FdC2; 2Fe-2S ferredoxin-like superfamily protein [KO:K02639]\n",
      "AT1G20020  FNR2; ferredoxin-NADP[+]-oxidoreductase 2 [KO:K02641] [EC:1.18.1.2]\n",
      "AT1G30510  RFNR2; root FNR 2 [KO:K02641] [EC:1.18.1.2]\n",
      "AT5G66190  FNR1; ferredoxin-NADP[+]-oxidoreductase 1 [KO:K02641] [EC:1.18.1.2]\n",
      "AT4G05390  RFNR1; root FNR 1 [KO:K02641] [EC:1.18.1.2]\n",
      "AT5G45040  CYTC6A; Cytochrome c [KO:K08906]\n",
      "ArthCp028  atpE; ATP synthase CF1 epsilon subunit [KO:K02114]\n",
      "ArthCp029  atpB; ATP synthase CF1 beta subunit [KO:K02112] [EC:7.1.2.2 7.2.2.1]\n",
      "AT1G15700  ATPC2; ATPase, F1 complex, gamma subunit protein [KO:K02115]\n",
      "AT4G04640  ATPC1; ATPase, F1 complex, gamma subunit protein [KO:K02115]\n",
      "ArthCp007  atpA; ATP synthase CF1 alpha subunit [KO:K02111] [EC:7.1.2.2 7.2.2.1]\n",
      "AT4G09650  ATPD; F-type H+-transporting ATPase subunit delta [KO:K02113]\n",
      "ArthCp008  atpF; ATP synthase CF0 B subunit [KO:K02109]\n",
      "AT4G32260  PDE334; ATPase, F0 complex, subunit B/B', bacterial/chloroplast [KO:K02109]\n",
      "AT2G07707  Plant mitochondrial ATPase, F0 complex, subunit 8 protein [KO:K02109]\n",
      "DA397_mgp18  atp8; ATPase subunit 8 [KO:K02109]\n",
      "ArthCp009  atpH; ATP synthase CF0 C subunit [KO:K02110]\n",
      "ArthCp010  atpI; ATP synthase CF0 A subunit [KO:K02108]\n",
      "COMPOUND    C00001  H2O\n",
      "C00002  ATP\n",
      "C00005  NADPH\n",
      "C00006  NADP+\n",
      "C00007  Oxygen\n",
      "C00008  ADP\n",
      "C00009  Orthophosphate\n",
      "C00034  Manganese\n",
      "C00080  H+\n",
      "C02061  Plastoquinone\n",
      "C02185  Plastoquinol-1\n",
      "REFERENCE   PMID:14764885\n",
      "AUTHORS   Ferreira KN, Iverson TM, Maghlaoui K, Barber J, Iwata S.\n",
      "TITLE     Architecture of the photosynthetic oxygen-evolving center.\n",
      "JOURNAL   Science 303:1831-8 (2004)\n",
      "DOI:10.1126/science.1093087\n",
      "REFERENCE   PMID:12518057\n",
      "AUTHORS   Kamiya N, Shen JR.\n",
      "TITLE     Crystal structure of oxygen-evolving photosystem II from Thermosynechococcus vulcanus at 3.7-A resolution.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 100:98-103 (2003)\n",
      "DOI:10.1073/pnas.0135651100\n",
      "REFERENCE   PMID:11217865\n",
      "AUTHORS   Zouni A, Witt HT, Kern J, Fromme P, Krauss N, Saenger W, Orth P.\n",
      "TITLE     Crystal structure of photosystem II from Synechococcus elongatus at 3.8 A resolution.\n",
      "JOURNAL   Nature 409:739-743 (2001)\n",
      "DOI:10.1038/35055589\n",
      "REFERENCE   PMID:11418848\n",
      "AUTHORS   Jordan P, Fromme P, Witt HT, Klukas O, Saenger W, Krauss N.\n",
      "TITLE     Three-dimensional structure of cyanobacterial photosystem I at 2.5 A resolution.\n",
      "JOURNAL   Nature 411:909-917 (2001)\n",
      "DOI:10.1038/35082000\n",
      "REFERENCE   PMID:11369223\n",
      "AUTHORS   Jones MR, Fyfe PK.\n",
      "TITLE     Photosynthesis: new light on biological oxygen production.\n",
      "JOURNAL   Curr Biol 17:R318-R321 (2001)\n",
      "DOI:10.1016/S0960-9822(01)00174-9\n",
      "REFERENCE   PMID:11914373\n",
      "AUTHORS   Shen G, Zhao J, Reimer SK, Antonkine ML, Cai Q, Weiland SM, Golbeck JH, Bryant DA.\n",
      "TITLE     Assembly of photosystem I. I. Inactivation of the rubA gene encoding a membrane-associated rubredoxin in the cyanobacterium Synechococcus sp. PCC 7002 causes a loss of photosystem I activity.\n",
      "JOURNAL   J Biol Chem 277:20343-54 (2002)\n",
      "DOI:10.1074/jbc.M201103200\n",
      "REL_PATHWAY ath00196  Photosynthesis - antenna proteins\n",
      "ath00710  Carbon fixation in photosynthetic organisms\n",
      "KO_PATHWAY  ko00195\n",
      "///\n",
      "ENTRY       ath00196                    Pathway\n",
      "NAME        Photosynthesis - antenna proteins - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The chlorophyll-binding subunits of photosystems I and II are internal antenna light-harvesting proteins of oxygenic photosynthesis. The antenna proteins that exist in phycobilisomes in cyanobacteria and light-harvesting chlorophyll protein complexes in green plants act as peripheral antenna systems, enabling more efficient absorption of light energy.\n",
      "CLASS       Metabolism; Energy metabolism\n",
      "PATHWAY_MAP ath00196  Photosynthesis - antenna proteins\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G54890  LHCA1; chlorophyll a-b binding protein 6 [KO:K08907]\n",
      "AT5G28450  Chlorophyll A-B binding family protein [KO:K08908]\n",
      "AT1G19150  Lhca6; PSI type II chlorophyll a/b-binding protein (Lhca2*1) [KO:K08908]\n",
      "AT3G61470  LHCA2; photosystem I light harvesting complex protein [KO:K08908]\n",
      "AT1G61520  LHCA3; PSI type III chlorophyll a/b-binding protein [KO:K08909]\n",
      "AT3G47470  LHCA4; light-harvesting chlorophyll-protein complex I subunit A4 [KO:K08910]\n",
      "AT1G45474  Lhca5; photosystem I light harvesting complex protein 5 [KO:K08911]\n",
      "AT1G29920  CAB2; chlorophyll A/B-binding protein 2 [KO:K08912]\n",
      "AT2G34420  LHB1B2; photosystem II light harvesting complex protein B1B2 [KO:K08912]\n",
      "AT2G34430  LHB1B1; light-harvesting chlorophyll-protein complex II subunit B1 [KO:K08912]\n",
      "AT1G29930  CAB1; chlorophyll A/B binding protein 1 [KO:K08912]\n",
      "AT1G29910  CAB3; chlorophyll A/B binding protein 3 [KO:K08912]\n",
      "AT3G27690  LHCB2.3; photosystem II light harvesting complex protein 2.3 [KO:K08913]\n",
      "AT2G05070  LHCB2.2; photosystem II light harvesting complex protein 2.2 [KO:K08913]\n",
      "AT2G05100  LHCB2.1; photosystem II light harvesting complex protein 2.1 [KO:K08913]\n",
      "AT5G54270  LHCB3; light-harvesting chlorophyll B-binding protein 3 [KO:K08914]\n",
      "AT5G01530  LHCB4.1; light harvesting complex photosystem II [KO:K08915]\n",
      "AT2G40100  LHCB4.3; light harvesting complex photosystem II [KO:K08915]\n",
      "AT3G08940  LHCB4.2; light harvesting complex photosystem II [KO:K08915]\n",
      "AT4G10340  LHCB5; light harvesting complex of photosystem II 5 [KO:K08916]\n",
      "AT1G15820  LHCB6; light harvesting complex photosystem II subunit 6 [KO:K08917]\n",
      "AT1G76570  Chlorophyll A-B binding family protein [KO:K14172]\n",
      "REFERENCE   PMID:10625426\n",
      "AUTHORS   Nield J, Orlova EV, Morris EP, Gowen B, van Heel M, Barber J.\n",
      "TITLE     3D map of the plant photosystem II supercomplex obtained by cryoelectron microscopy and single particle analysis.\n",
      "JOURNAL   Nat Struct Biol 7:44-7 (2000)\n",
      "DOI:10.1038/71242\n",
      "REFERENCE   PMID:15994896\n",
      "AUTHORS   Sener MK, Jolley C, Ben-Shem A, Fromme P, Nelson N, Croce R, Schulten K.\n",
      "TITLE     Comparison of the light-harvesting networks of plant and cyanobacterial photosystem I.\n",
      "JOURNAL   Biophys J 89:1630-42 (2005)\n",
      "DOI:10.1529/biophysj.105.066464\n",
      "REFERENCE   PMID:14668855\n",
      "AUTHORS   Ben-Shem A, Frolow F, Nelson N.\n",
      "TITLE     Crystal structure of plant photosystem I.\n",
      "JOURNAL   Nature 426:630-5 (2003)\n",
      "DOI:10.1038/nature02200\n",
      "REFERENCE   PMID:15955818\n",
      "AUTHORS   Jolley C, Ben-Shem A, Nelson N, Fromme P.\n",
      "TITLE     Structure of plant photosystem I revealed by theoretical modeling.\n",
      "JOURNAL   J Biol Chem 280:33627-36 (2005)\n",
      "DOI:10.1074/jbc.M500937200\n",
      "REFERENCE   PMID:16098971\n",
      "AUTHORS   Ihalainen JA, Klimmek F, Ganeteg U, van Stokkum IH, van Grondelle R, Jansson S, Dekker JP.\n",
      "TITLE     Excitation energy trapping in photosystem I complexes depleted in Lhca1 and Lhca4.\n",
      "JOURNAL   FEBS Lett 579:4787-91 (2005)\n",
      "DOI:10.1016/j.febslet.2005.06.091\n",
      "REFERENCE   PMID:16401064\n",
      "AUTHORS   Melkozernov AN, Barber J, Blankenship RE.\n",
      "TITLE     Light harvesting in photosystem I supercomplexes.\n",
      "JOURNAL   Biochemistry 45:331-45 (2006)\n",
      "DOI:10.1021/bi051932o\n",
      "REL_PATHWAY ath00195  Photosynthesis\n",
      "KO_PATHWAY  ko00196\n",
      "///\n",
      "ENTRY       ath00220                    Pathway\n",
      "NAME        Arginine biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00220  Arginine biosynthesis\n",
      "MODULE      ath_M00028  Ornithine biosynthesis, glutamate => ornithine [PATH:ath00220]\n",
      "ath_M00844  Arginine biosynthesis, ornithine => arginine [PATH:ath00220]\n",
      "DBLINKS     GO: 0006525\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G75330  OTC; ornithine carbamoyltransferase [KO:K00611] [EC:2.1.3.3]\n",
      "AT4G24830  arginosuccinate synthase family [KO:K01940] [EC:6.3.4.5]\n",
      "AT5G10920  L-Aspartase-like family protein [KO:K01755] [EC:4.3.2.1]\n",
      "AT4G08900  ARGAH1; arginase [KO:K01476] [EC:3.5.3.1]\n",
      "AT4G08870  ARGAH2; Arginase/deacetylase superfamily protein [KO:K01476] [EC:3.5.3.1]\n",
      "AT1G67550  URE; urease [KO:K01427] [EC:3.5.1.5]\n",
      "AT3G47450  NOA1; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K13427] [EC:1.14.13.39]\n",
      "AT5G37600  GSR_1; hypothetical protein [KO:K01915] [EC:6.3.1.2]\n",
      "AT3G17820  GLN1.3; glutamine synthetase 1.3 [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G48470  GLN1;5; glutamine synthetase 1;5 [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G35630  GS2; glutamine synthetase 2 [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G16570  GLN1;4; glutamine synthetase 1;4 [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G66200  GSR2; hypothetical protein [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G07440  GDH2; glutamate dehydrogenase 2 [KO:K00261] [EC:1.4.1.3]\n",
      "AT5G18170  GDH1; glutamate dehydrogenase 1 [KO:K00261] [EC:1.4.1.3]\n",
      "AT3G03910  GDH3; glutamate dehydrogenase 3 [KO:K00261] [EC:1.4.1.3]\n",
      "AT1G51720  Amino acid dehydrogenase family protein [KO:K00262] [EC:1.4.1.4]\n",
      "AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "AT1G17290  AlaAT1; alanine aminotransferas [KO:K00814] [EC:2.6.1.2]\n",
      "AT1G72330  ALAAT2; alanine aminotransferase 2 [KO:K00814] [EC:2.6.1.2]\n",
      "AT1G70580  AOAT2; alanine-2-oxoglutarate aminotransferase 2 [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT1G23310  GGT1; glutamate:glyoxylate aminotransferase [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT2G22910  NAGS1; N-acetyl-l-glutamate synthase 1 [KO:K14682] [EC:2.3.1.1]\n",
      "AT4G37670  NAGS2; N-acetyl-l-glutamate synthase 2 [KO:K14682] [EC:2.3.1.1]\n",
      "AT2G37500  arginine biosynthesis protein ArgJ family [KO:K00620] [EC:2.3.1.35 2.3.1.1]\n",
      "AT3G57560  NAGK; N-acetyl-l-glutamate kinase [KO:K00930] [EC:2.7.2.8]\n",
      "AT2G19940  Putative N-acetyl-gamma-glutamyl-phosphate reductase [KO:K00145] [EC:1.2.1.38]\n",
      "AT1G80600  WIN1; HOPW1-1-interacting 1 [KO:K00818] [EC:2.6.1.11]\n",
      "AT4G38220  AQI; Peptidase M20/M25/M40 family protein [KO:K14677] [EC:3.5.1.14]\n",
      "AT1G44820  Peptidase M20/M25/M40 family protein [KO:K14677] [EC:3.5.1.14]\n",
      "AT1G44180  Peptidase M20/M25/M40 family protein [KO:K14677] [EC:3.5.1.14]\n",
      "AT4G17830  Peptidase M20/M25/M40 family protein [KO:K01438] [EC:3.5.1.16]\n",
      "COMPOUND    C00011  CO2\n",
      "C00014  Ammonia\n",
      "C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00049  L-Aspartate\n",
      "C00062  L-Arginine\n",
      "C00064  L-Glutamine\n",
      "C00077  L-Ornithine\n",
      "C00086  Urea\n",
      "C00122  Fumarate\n",
      "C00169  Carbamoyl phosphate\n",
      "C00327  L-Citrulline\n",
      "C00437  N-Acetylornithine\n",
      "C00624  N-Acetyl-L-glutamate\n",
      "C01010  Urea-1-carboxylate\n",
      "C01250  N-Acetyl-L-glutamate 5-semialdehyde\n",
      "C03406  N-(L-Arginino)succinate\n",
      "C04133  N-Acetyl-L-glutamate 5-phosphate\n",
      "C15532  N-Acetyl-L-citrulline\n",
      "C20948  LysW-L-glutamate\n",
      "C20949  LysW-L-glutamyl 5-phosphate\n",
      "C20950  LysW-L-glutamate 5-semialdehyde\n",
      "C20951  LysW-L-ornithine\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00240  Pyrimidine metabolism\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00470  D-Amino acid metabolism\n",
      "ath00910  Nitrogen metabolism\n",
      "KO_PATHWAY  ko00220\n",
      "///\n",
      "ENTRY       ath00230                    Pathway\n",
      "NAME        Purine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Nucleotide metabolism\n",
      "PATHWAY_MAP ath00230  Purine metabolism\n",
      "MODULE      ath_M00005  PRPP biosynthesis, ribose 5P => PRPP [PATH:ath00230]\n",
      "ath_M00048  Inosine monophosphate biosynthesis, PRPP + glutamine => IMP [PATH:ath00230]\n",
      "ath_M00049  Adenine ribonucleotide biosynthesis, IMP => ADP,ATP [PATH:ath00230]\n",
      "ath_M00050  Guanine ribonucleotide biosynthesis, IMP => GDP,GTP [PATH:ath00230]\n",
      "DBLINKS     GO: 0042278\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G24730  manganese-dependent ADP-ribose/CDP-alcohol diphosphatase-like protein [KO:K01517] [EC:3.6.1.13 3.6.1.16 3.6.1.53]\n",
      "AT2G42070  NUDX23; nudix hydrolase homolog 23 [KO:K18453] [EC:3.6.1.13 3.6.1.18]\n",
      "AT4G11980  NUDX14; nudix hydrolase homolog 14 [KO:K18447] [EC:3.6.1.21]\n",
      "AT5G51820  PGM; phosphoglucomutase [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G23190  PGM3; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G70730  PGM2; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT2G44530  Phosphoribosyltransferase family protein [KO:K00948] [EC:2.7.6.1]\n",
      "AT1G32380  PRS2; phosphoribosyl pyrophosphate (PRPP) synthase 2 [KO:K00948] [EC:2.7.6.1]\n",
      "AT2G35390  Phosphoribosyltransferase family protein [KO:K00948] [EC:2.7.6.1]\n",
      "AT4G34740  ASE2; GLN phosphoribosyl pyrophosphate amidotransferase 2 [KO:K00764] [EC:2.4.2.14]\n",
      "AT2G16570  ASE1; GLN phosphoribosyl pyrophosphate amidotransferase 1 [KO:K00764] [EC:2.4.2.14]\n",
      "AT4G38880  ASE3; GLN phosphoribosyl pyrophosphate amidotransferase 3 [KO:K00764] [EC:2.4.2.14]\n",
      "AT1G09830  PUR2; Glycinamide ribonucleotide (GAR) synthetase [KO:K01945] [EC:6.3.4.13]\n",
      "AT1G31220  Formyl transferase [KO:K00601] [EC:2.1.2.2]\n",
      "AT1G74260  PUR4; purine biosynthesis 4 [KO:K01952] [EC:6.3.5.3]\n",
      "AT3G55010  PUR5; phosphoribosylformylglycinamidine cyclo-ligase, chloroplast / phosphoribosyl-aminoimidazole synthetase / AIR synthase (PUR5) [KO:K01933] [EC:6.3.3.1]\n",
      "AT2G37690  phosphoribosylaminoimidazole carboxylase, putative / AIR carboxylase [KO:K11808] [EC:4.1.1.21]\n",
      "AT3G21110  PUR7; purin 7 [KO:K01923] [EC:6.3.2.6]\n",
      "AT1G36280  L-Aspartase-like family protein [KO:K01756] [EC:4.3.2.2]\n",
      "AT4G18440  L-Aspartase-like family protein [KO:K01756] [EC:4.3.2.2]\n",
      "AT2G35040  AICARFT/IMPCHase bienzyme family protein [KO:K00602] [EC:2.1.2.3 3.5.4.10]\n",
      "AT4G12440  APT4; adenine phosphoribosyl transferase 4 [KO:K00759] [EC:2.4.2.7]\n",
      "AT5G11160  APT5; adenine phosphoribosyltransferase 5 [KO:K00759] [EC:2.4.2.7]\n",
      "AT4G22570  APT3; adenine phosphoribosyl transferase 3 [KO:K00759] [EC:2.4.2.7]\n",
      "AT1G80050  APT2; adenine phosphoribosyl transferase 2 [KO:K00759] [EC:2.4.2.7]\n",
      "AT1G27450  APT1; adenine phosphoribosyl transferase 1 [KO:K00759] [EC:2.4.2.7]\n",
      "AT1G72880  Survival protein SurE-like phosphatase/nucleotidase [KO:K03787] [EC:3.1.3.5]\n",
      "AT4G14930  Survival protein SurE-like phosphatase/nucleotidase [KO:K03787] [EC:3.1.3.5]\n",
      "AT1G71750  HGPT; Hypoxanthine-guanine phosphoribosyltransferase [KO:K00760] [EC:2.4.2.8]\n",
      "AT1G16350  Aldolase-type TIM barrel family protein [KO:K00088] [EC:1.1.1.205]\n",
      "AT1G79470  Aldolase-type TIM barrel family protein [KO:K00088] [EC:1.1.1.205]\n",
      "AT4G23900  Nucleoside diphosphate kinase family protein [KO:K00940] [EC:2.7.4.6]\n",
      "AT5G63310  NDPK2; nucleoside diphosphate kinase 2 [KO:K00940] [EC:2.7.4.6]\n",
      "AT4G11010  NDPK3; nucleoside diphosphate kinase 3 [KO:K00940] [EC:2.7.4.6]\n",
      "AT4G09320  NDPK1; nucleoside diphosphate kinase [KO:K00940] [EC:2.7.4.6]\n",
      "AT1G14250  GDA1/CD39 nucleoside phosphatase family protein [KO:K01510] [EC:3.6.1.5]\n",
      "AT1G14230  GDA1/CD39 nucleoside phosphatase family protein [KO:K01510] [EC:3.6.1.5]\n",
      "AT1G14240  GDA1/CD39 nucleoside phosphatase family protein [KO:K01510] [EC:3.6.1.5]\n",
      "AT2G02970  GDA1/CD39 nucleoside phosphatase family protein [KO:K01510] [EC:3.6.1.5]\n",
      "AT3G04080  APY1; apyrase 1 [KO:K14641] [EC:3.6.1.5]\n",
      "AT5G18280  APY2; apyrase 2 [KO:K14641] [EC:3.6.1.5]\n",
      "AT4G13720  Inosine triphosphate pyrophosphatase family protein [KO:K01519] [EC:3.6.1.-]\n",
      "AT4G34890  XDH1; xanthine dehydrogenase 1 [KO:K00106] [EC:1.17.1.4 1.17.3.2]\n",
      "AT4G34900  XDH2; xanthine dehydrogenase 2 [KO:K00106] [EC:1.17.1.4 1.17.3.2]\n",
      "AT1G63660  GMP synthase (glutamine-hydrolyzing), putative / glutamine amidotransferase [KO:K01951] [EC:6.3.5.2]\n",
      "AT3G06200  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K00942] [EC:2.7.4.8]\n",
      "AT2G41880  GK-1; guanylate kinase 1 [KO:K00942] [EC:2.7.4.8]\n",
      "AT3G57550  AGK2; guanylate kinase [KO:K00942] [EC:2.7.4.8]\n",
      "AT2G21790  RNR1; ribonucleotide reductase 1 [KO:K10807] [EC:1.17.4.1]\n",
      "AT3G27060  TSO2; Ferritin/ribonucleotide reductase-like family protein [KO:K10808] [EC:1.17.4.1]\n",
      "AT3G23580  RNR2A; ribonucleotide reductase 2A [KO:K10808] [EC:1.17.4.1]\n",
      "AT1G26160  Metal-dependent phosphohydrolase [KO:K07023] [EC:3.1.3.89]\n",
      "AT2G23820  Metal-dependent phosphohydrolase [KO:K07023] [EC:3.1.3.89]\n",
      "AT1G54130  RSH3; RELA/SPOT homolog 3 [KO:K00951] [EC:2.7.6.5]\n",
      "AT3G17470  CRSH; Ca2+-activated RelA/spot-like protein [KO:K00951] [EC:2.7.6.5]\n",
      "AT3G14050  RSH2; RELA/SPOT homolog 2 [KO:K00951] [EC:2.7.6.5]\n",
      "AT4G02260  RSH1; RELA/SPOT homolog 1 [KO:K00951] [EC:2.7.6.5]\n",
      "AT3G57610  ADSS; adenylosuccinate synthase [KO:K01939] [EC:6.3.4.4]\n",
      "AT2G38280  FAC1; AMP deaminase, putative / myoadenylate deaminase [KO:K01490] [EC:3.5.4.6]\n",
      "AT5G03300  ADK2; adenosine kinase 2 [KO:K00856] [EC:2.7.1.20]\n",
      "AT3G09820  ADK1; adenosine kinase 1 [KO:K00856] [EC:2.7.1.20]\n",
      "AT4G04880  adenosine/AMP deaminase family protein [KO:K01488] [EC:3.5.4.4]\n",
      "AT5G47840  AMK2; adenosine monophosphate kinase [KO:K00939] [EC:2.7.4.3]\n",
      "AT5G50370  Adenylate kinase family protein [KO:K00939] [EC:2.7.4.3]\n",
      "AT5G63400  ADK1; adenylate kinase 1 [KO:K00939] [EC:2.7.4.3]\n",
      "AT5G35170  adenylate kinase family protein [KO:K00939] [EC:2.7.4.3]\n",
      "AT2G39270  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K00939] [EC:2.7.4.3]\n",
      "AT2G37250  ADK; adenosine kinase [KO:K00939] [EC:2.7.4.3]\n",
      "AT3G01820  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K00939] [EC:2.7.4.3]\n",
      "AT5G60340  AAK6; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K18532] [EC:2.7.4.3]\n",
      "AT5G66005  Expressed protein [KO:K06928] [EC:3.6.1.15]\n",
      "AT5G58240  FHIT; FRAGILE HISTIDINE TRIAD [KO:K01522] [EC:3.6.1.29]\n",
      "AT4G14680  APS3; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT3G22890  APS1; ATP sulfurylase 1 [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT1G19920  APS2; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT5G43780  APS4; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT5G67520  APK4; adenosine-5'-phosphosulfate (APS) kinase 4 [KO:K00860] [EC:2.7.1.25]\n",
      "AT3G03900  APK3; adenosine-5'-phosphosulfate (APS) kinase 3 [KO:K00860] [EC:2.7.1.25]\n",
      "AT2G14750  APK; APS kinase [KO:K00860] [EC:2.7.1.25]\n",
      "AT4G39940  AKN2; APS-kinase 2 [KO:K00860] [EC:2.7.1.25]\n",
      "AT4G29680  Alkaline-phosphatase-like family protein [KO:K01513] [EC:3.1.4.1 3.6.1.9]\n",
      "AT2G26230  uricase / urate oxidase / nodulin 35 [KO:K00365] [EC:1.7.3.3]\n",
      "AT5G58220  TTL; transthyretin-like protein [KO:K13484] [EC:3.5.2.17 4.1.1.97]\n",
      "AT4G04955  ALN; allantoinase [KO:K01466] [EC:3.5.2.5]\n",
      "AT4G20070  AAH; allantoate amidohydrolase [KO:K02083] [EC:3.5.3.9]\n",
      "AT4G17050  UGLYAH; ureidoglycine aminohydrolase [KO:K14977] [EC:3.5.3.26]\n",
      "AT5G43600  UAH; ureidoglycolate amidohydrolase [KO:K18151] [EC:3.5.1.116]\n",
      "AT1G67550  URE; urease [KO:K01427] [EC:3.5.1.5]\n",
      "COMPOUND    C00002  ATP\n",
      "C00008  ADP\n",
      "C00011  CO2\n",
      "C00014  Ammonia\n",
      "C00020  AMP\n",
      "C00035  GDP\n",
      "C00037  Glycine\n",
      "C00039  DNA\n",
      "C00044  GTP\n",
      "C00046  RNA\n",
      "C00048  Glyoxylate\n",
      "C00053  3'-Phosphoadenylyl sulfate\n",
      "C00054  Adenosine 3',5'-bisphosphate\n",
      "C00059  Sulfate\n",
      "C00064  L-Glutamine\n",
      "C00081  ITP\n",
      "C00086  Urea\n",
      "C00104  IDP\n",
      "C00117  D-Ribose 5-phosphate\n",
      "C00119  5-Phospho-alpha-D-ribose 1-diphosphate\n",
      "C00130  IMP\n",
      "C00131  dATP\n",
      "C00144  GMP\n",
      "C00147  Adenine\n",
      "C00169  Carbamoyl phosphate\n",
      "C00206  dADP\n",
      "C00209  Oxalate\n",
      "C00212  Adenosine\n",
      "C00224  Adenylyl sulfate\n",
      "C00242  Guanine\n",
      "C00262  Hypoxanthine\n",
      "C00286  dGTP\n",
      "C00288  HCO3-\n",
      "C00294  Inosine\n",
      "C00301  ADP-ribose\n",
      "C00330  Deoxyguanosine\n",
      "C00360  dAMP\n",
      "C00361  dGDP\n",
      "C00362  dGMP\n",
      "C00366  Urate\n",
      "C00385  Xanthine\n",
      "C00387  Guanosine\n",
      "C00499  Allantoate\n",
      "C00559  Deoxyadenosine\n",
      "C00575  3',5'-Cyclic AMP\n",
      "C00603  (S)-Ureidoglycolate\n",
      "C00620  alpha-D-Ribose 1-phosphate\n",
      "C00655  Xanthosine 5'-phosphate\n",
      "C00700  XTP\n",
      "C00802  Oxalureate\n",
      "C00942  3',5'-Cyclic GMP\n",
      "C01228  Guanosine 3',5'-bis(diphosphate)\n",
      "C01260  P1,P4-Bis(5'-adenosyl)tetraphosphate\n",
      "C01261  P1,P4-Bis(5'-guanosyl) tetraphosphate\n",
      "C01344  dIDP\n",
      "C01345  dITP\n",
      "C01367  3'-AMP\n",
      "C01444  Oxamate\n",
      "C01762  Xanthosine\n",
      "C02091  (S)-Ureidoglycine\n",
      "C02348  (R)(-)-Allantoin\n",
      "C02350  (S)-Allantoin\n",
      "C02353  2',3'-Cyclic AMP\n",
      "C02718  N-Formiminoglycine\n",
      "C03090  5-Phosphoribosylamine\n",
      "C03373  Aminoimidazole ribotide\n",
      "C03483  Adenosine tetraphosphate\n",
      "C03614  Inosine 5'-tetraphosphate\n",
      "C03794  N6-(1,2-Dicarboxyethyl)-AMP\n",
      "C03838  5'-Phosphoribosylglycinamide\n",
      "C04051  5-Amino-4-imidazolecarboxyamide\n",
      "C04376  5'-Phosphoribosyl-N-formylglycinamide\n",
      "C04392  P1,P4-Bis(5'-xanthosyl) tetraphosphate\n",
      "C04494  Guanosine 3'-diphosphate 5'-triphosphate\n",
      "C04640  2-(Formamido)-N1-(5'-phosphoribosyl)acetamidine\n",
      "C04677  1-(5'-Phosphoribosyl)-5-amino-4-imidazolecarboxamide\n",
      "C04734  1-(5'-Phosphoribosyl)-5-formamido-4-imidazolecarboxamide\n",
      "C04751  1-(5-Phospho-D-ribosyl)-5-amino-4-imidazolecarboxylate\n",
      "C04823  1-(5'-Phosphoribosyl)-5-amino-4-(N-succinocarboxamide)-imidazole\n",
      "C05239  5-Aminoimidazole\n",
      "C05512  Deoxyinosine\n",
      "C05513  Urate-3-ribonucleoside\n",
      "C05515  5-Ureido-4-imidazole carboxylate\n",
      "C05516  5-Amino-4-imidazole carboxylate\n",
      "C05993  Acetyl adenylate\n",
      "C06193  Guanosine 3'-phosphate\n",
      "C06194  2',3'-Cyclic GMP\n",
      "C06195  Imidazolone\n",
      "C06196  2'-Deoxyinosine 5'-phosphate\n",
      "C06197  P1,P3-Bis(5'-adenosyl) triphosphate\n",
      "C06433  5'-Benzoylphosphoadenosine\n",
      "C06435  5'-Butyrylphosphoinosine\n",
      "C11821  5-Hydroxyisourate\n",
      "C12248  5-Hydroxy-2-oxo-4-ureido-2,5-dihydro-1H-imidazole-5-carboxylate\n",
      "C15667  5-Carboxyamino-1-(5-phospho-D-ribosyl)imidazole\n",
      "REFERENCE   PMID:10601204\n",
      "AUTHORS   Cusa E, Obradors N, Baldoma L, Badia J, Aguilar J.\n",
      "TITLE     Genetic analysis of a chromosomal region containing genes required for assimilation of allantoin nitrogen and linked glyoxylate metabolism in Escherichia coli.\n",
      "JOURNAL   J Bacteriol 181:7479-84 (1999)\n",
      "DOI:10.1128/JB.181.24.7479-7484.1999\n",
      "REFERENCE   PMID:10986234\n",
      "AUTHORS   Xi H, Schneider BL, Reitzer L.\n",
      "TITLE     Purine catabolism in Escherichia coli and function of xanthine dehydrogenase in purine salvage.\n",
      "JOURNAL   J Bacteriol 182:5332-41 (2000)\n",
      "DOI:10.1128/JB.182.19.5332-5341.2000\n",
      "REL_PATHWAY ath00030  Pentose phosphate pathway\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00340  Histidine metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00730  Thiamine metabolism\n",
      "ath00740  Riboflavin metabolism\n",
      "ath00790  Folate biosynthesis\n",
      "KO_PATHWAY  ko00230\n",
      "///\n",
      "ENTRY       ath00232                    Pathway\n",
      "NAME        Caffeine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00232  Caffeine metabolism\n",
      "DBLINKS     UMBBD: caf caf2\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G34890  XDH1; xanthine dehydrogenase 1 [KO:K00106] [EC:1.17.1.4 1.17.3.2]\n",
      "AT4G34900  XDH2; xanthine dehydrogenase 2 [KO:K00106] [EC:1.17.1.4 1.17.3.2]\n",
      "AT2G26230  uricase / urate oxidase / nodulin 35 [KO:K00365] [EC:1.7.3.3]\n",
      "COMPOUND    C00048  Glyoxylate\n",
      "C00385  Xanthine\n",
      "C01762  Xanthosine\n",
      "C07130  Theophylline\n",
      "C07480  Theobromine\n",
      "C07481  Caffeine\n",
      "C13747  1,7-Dimethylxanthine\n",
      "C16352  7-Methylxanthosine\n",
      "C16353  7-Methylxanthine\n",
      "C16355  7-Methyluric acid\n",
      "C16356  1,7-Dimethyluric acid\n",
      "C16357  3-Methylxanthine\n",
      "C16358  1-Methylxanthine\n",
      "C16359  1-Methyluric acid\n",
      "C16360  3,7-Dimethyluric acid\n",
      "C16361  1,3,7-Trimethyluric acid\n",
      "C16362  3,6,8-Trimethylallantoin\n",
      "C16363  N-Methylurea\n",
      "C16364  N,N'-Dimethylurea\n",
      "C16365  5-Acetylamino-6-formylamino-3-methyluracil\n",
      "C16366  5-Acetylamino-6-amino-3-methyluracil\n",
      "C21144  1,3,7-Trimethyl-5-hydroxyisourate\n",
      "REFERENCE   PMID:9705852\n",
      "AUTHORS   Madyastha KM, Sridhar GR.\n",
      "TITLE     A novel pathway for the metabolism of caffeine by a mixed culture consortium.\n",
      "JOURNAL   Biochem Biophys Res Commun 249:178-81 (1998)\n",
      "DOI:10.1006/bbrc.1998.9102\n",
      "REFERENCE   PMID:14977550\n",
      "AUTHORS   Mazzafera P.\n",
      "TITLE     Catabolism of caffeine in plants and microorganisms.\n",
      "JOURNAL   Front Biosci 9:1348-59 (2004)\n",
      "DOI:10.2741/1339\n",
      "REFERENCE   PMID:11393735\n",
      "AUTHORS   Nyeki A, Biollaz J, Kesselring UW, Decosterd LA.\n",
      "TITLE     Extractionless method for the simultaneous high-performance liquid chromatographic determination of urinary caffeine metabolites for N-acetyltransferase 2, cytochrome P450 1A2 and xanthine oxidase activity assessment.\n",
      "JOURNAL   J Chromatogr B Biomed Sci Appl 755:73-84 (2001)\n",
      "DOI:10.1016/S0378-4347(00)00616-2\n",
      "REFERENCE   PMID:11544129\n",
      "AUTHORS   Ashihara H, Crozier A.\n",
      "TITLE     Caffeine: a well known but little mentioned compound in plant science.\n",
      "JOURNAL   Trends Plant Sci 6:407-13 (2001)\n",
      "DOI:10.1016/S1360-1385(01)02055-6\n",
      "REFERENCE   PMID:16333668\n",
      "AUTHORS   Yoneyama N, Morimoto H, Ye CX, Ashihara H, Mizuno K, Kato M.\n",
      "TITLE     Substrate specificity of N-methyltransferase involved in purine alkaloids synthesis is dependent upon one amino acid residue of the enzyme.\n",
      "JOURNAL   Mol Genet Genomics 275:125-35 (2006)\n",
      "DOI:10.1007/s00438-005-0070-z\n",
      "REL_PATHWAY ath00230  Purine metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "KO_PATHWAY  ko00232\n",
      "///\n",
      "ENTRY       ath00240                    Pathway\n",
      "NAME        Pyrimidine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Nucleotide metabolism\n",
      "PATHWAY_MAP ath00240  Pyrimidine metabolism\n",
      "MODULE      ath_M00046  Pyrimidine degradation, uracil => beta-alanine, thymine => 3-aminoisobutanoate [PATH:ath00240]\n",
      "ath_M00052  Pyrimidine ribonucleotide biosynthesis, UMP => UDP/UTP,CDP/CTP [PATH:ath00240]\n",
      "ath_M00053  Pyrimidine deoxyribonuleotide biosynthesis, CDP => dCTP [PATH:ath00240]\n",
      "ath_M00938  Pyrimidine deoxyribonuleotide biosynthesis, UDP => dTTP [PATH:ath00240]\n",
      "DBLINKS     GO: 0006220\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G29900  CARB; carbamoyl phosphate synthetase B [KO:K01955] [EC:6.3.5.5]\n",
      "AT3G27740  CARA; carbamoyl phosphate synthetase A [KO:K01956] [EC:6.3.5.5]\n",
      "AT3G20330  PYRB; PYRIMIDINE B [KO:K00609] [EC:2.1.3.2]\n",
      "AT4G22930  PYR4; pyrimidin 4 [KO:K01465] [EC:3.5.2.3]\n",
      "AT5G23300  PYRD; pyrimidine d [KO:K00254] [EC:1.3.5.2]\n",
      "AT3G54470  uridine 5'-monophosphate synthase / UMP synthase (PYRE-F) (UMPS) [KO:K13421] [EC:2.4.2.10 4.1.1.23]\n",
      "AT5G26667  PYR6; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K13800] [EC:2.7.4.14]\n",
      "AT4G25280  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K13800] [EC:2.7.4.14]\n",
      "AT3G60180  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K13800] [EC:2.7.4.14]\n",
      "AT3G18680  Amino acid kinase family protein [KO:K09903] [EC:2.7.4.22]\n",
      "AT4G23900  Nucleoside diphosphate kinase family protein [KO:K00940] [EC:2.7.4.6]\n",
      "AT5G63310  NDPK2; nucleoside diphosphate kinase 2 [KO:K00940] [EC:2.7.4.6]\n",
      "AT4G11010  NDPK3; nucleoside diphosphate kinase 3 [KO:K00940] [EC:2.7.4.6]\n",
      "AT4G09320  NDPK1; nucleoside diphosphate kinase [KO:K00940] [EC:2.7.4.6]\n",
      "AT1G14250  GDA1/CD39 nucleoside phosphatase family protein [KO:K01510] [EC:3.6.1.5]\n",
      "AT1G14230  GDA1/CD39 nucleoside phosphatase family protein [KO:K01510] [EC:3.6.1.5]\n",
      "AT1G14240  GDA1/CD39 nucleoside phosphatase family protein [KO:K01510] [EC:3.6.1.5]\n",
      "AT2G02970  GDA1/CD39 nucleoside phosphatase family protein [KO:K01510] [EC:3.6.1.5]\n",
      "AT3G04080  APY1; apyrase 1 [KO:K14641] [EC:3.6.1.5]\n",
      "AT5G18280  APY2; apyrase 2 [KO:K14641] [EC:3.6.1.5]\n",
      "AT4G29680  Alkaline-phosphatase-like family protein [KO:K01513] [EC:3.1.4.1 3.6.1.9]\n",
      "AT2G34890  CTP synthase family protein [KO:K01937] [EC:6.3.4.2]\n",
      "AT3G12670  emb2742; CTP synthase family protein [KO:K01937] [EC:6.3.4.2]\n",
      "AT1G30820  CTP synthase family protein [KO:K01937] [EC:6.3.4.2]\n",
      "AT4G20320  CTP synthase family protein [KO:K01937] [EC:6.3.4.2]\n",
      "AT4G02120  CTP synthase family protein [KO:K01937] [EC:6.3.4.2]\n",
      "AT5G40870  UK/UPRT1; uridine kinase/uracil phosphoribosyltransferase 1 [KO:K00876] [EC:2.7.1.48]\n",
      "AT3G27190  UKL2; uridine kinase-like 2 [KO:K00876] [EC:2.7.1.48]\n",
      "AT4G26510  UKL4; uridine kinase-like 4 [KO:K00876] [EC:2.7.1.48]\n",
      "AT1G55810  UKL3; uridine kinase-like 3 [KO:K00876] [EC:2.7.1.48]\n",
      "AT3G27440  UKL5; uridine kinase-like 5 [KO:K00876] [EC:2.7.1.48]\n",
      "AT1G26190  Phosphoribulokinase / Uridine kinase family [KO:K00876] [EC:2.7.1.48]\n",
      "AT1G73980  Phosphoribulokinase / Uridine kinase family [KO:K00876] [EC:2.7.1.48]\n",
      "AT1G72880  Survival protein SurE-like phosphatase/nucleotidase [KO:K03787] [EC:3.1.3.5]\n",
      "AT4G14930  Survival protein SurE-like phosphatase/nucleotidase [KO:K03787] [EC:3.1.3.5]\n",
      "AT2G36310  URH1; uridine-ribohydrolase 1 [KO:K01240] [EC:3.2.2.3]\n",
      "AT3G53900  UPP; uracil phosphoribosyltransferase [KO:K00761] [EC:2.4.2.9]\n",
      "AT3G17810  PYD1; pyrimidine 1 [KO:K00207] [EC:1.3.1.2]\n",
      "AT5G12200  PYD2; pyrimidine 2 [KO:K01464] [EC:3.5.2.2]\n",
      "AT5G64370  BETA-UP; beta-ureidopropionase [KO:K01431] [EC:3.5.1.6]\n",
      "AT1G50510  indigoidine synthase A family protein [KO:K16329] [EC:4.2.1.70]\n",
      "AT2G38680  5'-nucleotidase / magnesium ion binding protein [KO:K24242] [EC:3.1.3.91]\n",
      "AT2G19570  CDA1; cytidine deaminase 1 [KO:K01489] [EC:3.5.4.5]\n",
      "AT4G29570  Cytidine/deoxycytidylate deaminase family protein [KO:K01489] [EC:3.5.4.5]\n",
      "AT4G29640  Cytidine/deoxycytidylate deaminase family protein [KO:K01489] [EC:3.5.4.5]\n",
      "AT4G29610  Cytidine/deoxycytidylate deaminase family protein [KO:K01489] [EC:3.5.4.5]\n",
      "AT4G29600  Cytidine/deoxycytidylate deaminase family protein [KO:K01489] [EC:3.5.4.5]\n",
      "AT4G29620  Cytidine/deoxycytidylate deaminase family protein [KO:K01489] [EC:3.5.4.5]\n",
      "AT4G29630  Cytidine/deoxycytidylate deaminase family protein [KO:K01489] [EC:3.5.4.5]\n",
      "AT2G21790  RNR1; ribonucleotide reductase 1 [KO:K10807] [EC:1.17.4.1]\n",
      "AT3G27060  TSO2; Ferritin/ribonucleotide reductase-like family protein [KO:K10808] [EC:1.17.4.1]\n",
      "AT3G23580  RNR2A; ribonucleotide reductase 2A [KO:K10808] [EC:1.17.4.1]\n",
      "AT3G25400  dCTP pyrophosphatase-like protein [KO:K16904] [EC:3.6.1.12]\n",
      "AT1G26160  Metal-dependent phosphohydrolase [KO:K07023] [EC:3.1.3.89]\n",
      "AT2G23820  Metal-dependent phosphohydrolase [KO:K07023] [EC:3.1.3.89]\n",
      "AT3G48540  Cytidine/deoxycytidylate deaminase family protein [KO:K01493] [EC:3.5.4.12]\n",
      "AT5G59440  ZEU1; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K00943] [EC:2.7.4.9]\n",
      "AT3G46940  DUT1; DUTP-PYROPHOSPHATASE-LIKE 1 [KO:K01520] [EC:3.6.1.23]\n",
      "AT3G07800  TK1a; Thymidine kinase [KO:K00857] [EC:2.7.1.21]\n",
      "AT5G23070  TK1b; Thymidine kinase [KO:K00857] [EC:2.7.1.21]\n",
      "AT4G34570  THY-2; thymidylate synthase 2 [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "AT2G16370  THY-1; thymidylate synthase 1 [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "AT2G21550  Bifunctional dihydrofolate reductase/thymidylate synthase [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "COMPOUND    C00015  UDP\n",
      "C00029  UDP-glucose\n",
      "C00039  DNA\n",
      "C00046  RNA\n",
      "C00055  CMP\n",
      "C00063  CTP\n",
      "C00064  L-Glutamine\n",
      "C00075  UTP\n",
      "C00086  Urea\n",
      "C00099  beta-Alanine\n",
      "C00105  UMP\n",
      "C00106  Uracil\n",
      "C00112  CDP\n",
      "C00119  5-Phospho-alpha-D-ribose 1-diphosphate\n",
      "C00169  Carbamoyl phosphate\n",
      "C00178  Thymine\n",
      "C00214  Thymidine\n",
      "C00222  3-Oxopropanoate\n",
      "C00239  dCMP\n",
      "C00295  Orotate\n",
      "C00299  Uridine\n",
      "C00337  (S)-Dihydroorotate\n",
      "C00363  dTDP\n",
      "C00364  dTMP\n",
      "C00365  dUMP\n",
      "C00380  Cytosine\n",
      "C00383  Malonate\n",
      "C00429  5,6-Dihydrouracil\n",
      "C00438  N-Carbamoyl-L-aspartate\n",
      "C00458  dCTP\n",
      "C00459  dTTP\n",
      "C00460  dUTP\n",
      "C00475  Cytidine\n",
      "C00526  Deoxyuridine\n",
      "C00672  2-Deoxy-D-ribose 1-phosphate\n",
      "C00705  dCDP\n",
      "C00813  Barbiturate\n",
      "C00881  Deoxycytidine\n",
      "C01013  3-Hydroxypropanoate\n",
      "C01103  Orotidine 5'-phosphate\n",
      "C01168  Pseudouridine 5'-phosphate\n",
      "C01205  (R)-3-Amino-2-methylpropanoate\n",
      "C01346  dUDP\n",
      "C01368  3'-UMP\n",
      "C01563  Carbamate\n",
      "C02067  Pseudouridine\n",
      "C02170  Methylmalonate\n",
      "C02354  2',3'-Cyclic CMP\n",
      "C02355  2',3'-Cyclic UMP\n",
      "C02376  5-Methylcytosine\n",
      "C02642  3-Ureidopropionate\n",
      "C03997  5-Hydroxymethyldeoxycytidylate\n",
      "C05281  5-Methylbarbiturate\n",
      "C05822  3'-CMP\n",
      "C06198  P1,P4-Bis(5'-uridyl) tetraphosphate\n",
      "C11038  2'-Deoxy-5-hydroxymethylcytidine-5'-diphosphate\n",
      "C11039  2'-Deoxy-5-hydroxymethylcytidine-5'-triphosphate\n",
      "C15607  3-Oxo-3-ureidopropanoate\n",
      "C20231  (Z)-3-Ureidoacrylate peracid\n",
      "C20249  (Z)-3-Peroxyaminoacrylate\n",
      "C20253  Aminoacrylate\n",
      "C20254  Ureidoacrylate\n",
      "C20846  3-Oxo-3-ureidoisobutyrate\n",
      "C21028  (R)-5,6-Dihydrothymine\n",
      "C21029  (R)-3-Ureidoisobutyrate\n",
      "REL_PATHWAY ath00030  Pentose phosphate pathway\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00280  Valine, leucine and isoleucine degradation\n",
      "ath00410  beta-Alanine metabolism\n",
      "KO_PATHWAY  ko00240\n",
      "///\n",
      "ENTRY       ath00250                    Pathway\n",
      "NAME        Alanine, aspartate and glutamate metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00250  Alanine, aspartate and glutamate metabolism\n",
      "MODULE      ath_M00027  GABA (gamma-Aminobutyrate) shunt [PATH:ath00250]\n",
      "DBLINKS     GO: 0006522 0006531 0006536\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "AT5G14760  AO; L-aspartate oxidase [KO:K00278] [EC:1.4.3.16]\n",
      "AT5G08100  ASPGA1; N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K13051] [EC:3.5.1.1 3.4.19.5]\n",
      "AT3G16150  ASPGB1; N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K13051] [EC:3.5.1.1 3.4.19.5]\n",
      "AT5G65010  ASN2; asparagine synthetase 2 [KO:K01953] [EC:6.3.5.4]\n",
      "AT3G47340  ASN1; glutamine-dependent asparagine synthase 1 [KO:K01953] [EC:6.3.5.4]\n",
      "AT5G10240  ASN3; asparagine synthetase 3 [KO:K01953] [EC:6.3.5.4]\n",
      "AT5G12040  Nitrilase/cyanide hydratase and apolipoprotein N-acyltransferase family protein [KO:K13566] [EC:3.5.1.3]\n",
      "AT1G17290  AlaAT1; alanine aminotransferas [KO:K00814] [EC:2.6.1.2]\n",
      "AT1G72330  ALAAT2; alanine aminotransferase 2 [KO:K00814] [EC:2.6.1.2]\n",
      "AT2G13360  AGT; alanine:glyoxylate aminotransferase [KO:K00830] [EC:2.6.1.44 2.6.1.45 2.6.1.51]\n",
      "AT4G39660  AGT2; alanine:glyoxylate aminotransferase 2 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT3G08860  PYD4; PYRIMIDINE 4 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT2G38400  AGT3; alanine:glyoxylate aminotransferase 3 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT1G70580  AOAT2; alanine-2-oxoglutarate aminotransferase 2 [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT1G23310  GGT1; glutamate:glyoxylate aminotransferase [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT4G24830  arginosuccinate synthase family [KO:K01940] [EC:6.3.4.5]\n",
      "AT5G10920  L-Aspartase-like family protein [KO:K01755] [EC:4.3.2.1]\n",
      "AT3G57610  ADSS; adenylosuccinate synthase [KO:K01939] [EC:6.3.4.4]\n",
      "AT1G36280  L-Aspartase-like family protein [KO:K01756] [EC:4.3.2.2]\n",
      "AT4G18440  L-Aspartase-like family protein [KO:K01756] [EC:4.3.2.2]\n",
      "AT3G20330  PYRB; PYRIMIDINE B [KO:K00609] [EC:2.1.3.2]\n",
      "AT2G02000  GAD3; glutamate decarboxylase 3 [KO:K01580] [EC:4.1.1.15]\n",
      "AT3G17760  GAD5; glutamate decarboxylase 5 [KO:K01580] [EC:4.1.1.15]\n",
      "AT5G17330  GAD; glutamate decarboxylase [KO:K01580] [EC:4.1.1.15]\n",
      "AT2G02010  GAD4; glutamate decarboxylase 4 [KO:K01580] [EC:4.1.1.15]\n",
      "AT1G65960  GAD2; glutamate decarboxylase 2 [KO:K01580] [EC:4.1.1.15]\n",
      "AT3G22200  POP2; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K16871] [EC:2.6.1.96]\n",
      "AT1G79440  ALDH5F1; aldehyde dehydrogenase 5F1 [KO:K17761] [EC:1.2.1.24]\n",
      "AT5G53460  GLT1; NADH-dependent glutamate synthase 1 [KO:K00264] [EC:1.4.1.14]\n",
      "AT5G07440  GDH2; glutamate dehydrogenase 2 [KO:K00261] [EC:1.4.1.3]\n",
      "AT5G18170  GDH1; glutamate dehydrogenase 1 [KO:K00261] [EC:1.4.1.3]\n",
      "AT3G03910  GDH3; glutamate dehydrogenase 3 [KO:K00261] [EC:1.4.1.3]\n",
      "AT1G51720  Amino acid dehydrogenase family protein [KO:K00262] [EC:1.4.1.4]\n",
      "AT5G62530  ALDH12A1; aldehyde dehydrogenase 12A1 [KO:K00294] [EC:1.2.1.88]\n",
      "AT5G37600  GSR_1; hypothetical protein [KO:K01915] [EC:6.3.1.2]\n",
      "AT3G17820  GLN1.3; glutamine synthetase 1.3 [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G48470  GLN1;5; glutamine synthetase 1;5 [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G35630  GS2; glutamine synthetase 2 [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G16570  GLN1;4; glutamine synthetase 1;4 [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G66200  GSR2; hypothetical protein [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G29900  CARB; carbamoyl phosphate synthetase B [KO:K01955] [EC:6.3.5.5]\n",
      "AT3G27740  CARA; carbamoyl phosphate synthetase A [KO:K01956] [EC:6.3.5.5]\n",
      "AT3G24090  GFAT; putative glucosamine-fructose-6-phosphate aminotransferase [KO:K00820] [EC:2.6.1.16]\n",
      "AT4G34740  ASE2; GLN phosphoribosyl pyrophosphate amidotransferase 2 [KO:K00764] [EC:2.4.2.14]\n",
      "AT2G16570  ASE1; GLN phosphoribosyl pyrophosphate amidotransferase 1 [KO:K00764] [EC:2.4.2.14]\n",
      "AT4G38880  ASE3; GLN phosphoribosyl pyrophosphate amidotransferase 3 [KO:K00764] [EC:2.4.2.14]\n",
      "COMPOUND    C00014  Ammonia\n",
      "C00022  Pyruvate\n",
      "C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00036  Oxaloacetate\n",
      "C00041  L-Alanine\n",
      "C00042  Succinate\n",
      "C00049  L-Aspartate\n",
      "C00064  L-Glutamine\n",
      "C00122  Fumarate\n",
      "C00152  L-Asparagine\n",
      "C00158  Citrate\n",
      "C00169  Carbamoyl phosphate\n",
      "C00232  Succinate semialdehyde\n",
      "C00334  4-Aminobutanoate\n",
      "C00352  D-Glucosamine 6-phosphate\n",
      "C00402  D-Aspartate\n",
      "C00438  N-Carbamoyl-L-aspartate\n",
      "C00940  2-Oxoglutaramate\n",
      "C01042  N-Acetyl-L-aspartate\n",
      "C02362  2-Oxosuccinamate\n",
      "C03090  5-Phosphoribosylamine\n",
      "C03406  N-(L-Arginino)succinate\n",
      "C03794  N6-(1,2-Dicarboxyethyl)-AMP\n",
      "C03912  (S)-1-Pyrroline-5-carboxylate\n",
      "C12270  N-Acetylaspartylglutamate\n",
      "C20775  beta-Citryl-L-glutamate\n",
      "C20776  N-Acetylaspartylglutamylglutamate\n",
      "REFERENCE\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REFERENCE   PMID:9687539\n",
      "AUTHORS   Wu G\n",
      "TITLE     Intestinal mucosal amino acid catabolism.\n",
      "JOURNAL   J Nutr 128:1249-52 (1998)\n",
      "DOI:10.1093/jn/128.8.1249\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00230  Purine metabolism\n",
      "ath00240  Pyrimidine metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00261  Monobactam biosynthesis\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00340  Histidine metabolism\n",
      "ath00410  beta-Alanine metabolism\n",
      "ath00460  Cyanoamino acid metabolism\n",
      "ath00470  D-Amino acid metabolism\n",
      "ath00480  Glutathione metabolism\n",
      "ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00650  Butanoate metabolism\n",
      "ath00660  C5-Branched dibasic acid metabolism\n",
      "ath00760  Nicotinate and nicotinamide metabolism\n",
      "ath00770  Pantothenate and CoA biosynthesis\n",
      "ath00860  Porphyrin and chlorophyll metabolism\n",
      "ath00910  Nitrogen metabolism\n",
      "KO_PATHWAY  ko00250\n",
      "///\n",
      "ENTRY       ath00260                    Pathway\n",
      "NAME        Glycine, serine and threonine metabolism - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Serine is derived from 3-phospho-D-glycerate, an intermediate of glycolysis [MD:M00020], and glycine is derived from serine. Threonine is an essential amino acid, which animals cannot synthesize. In bacteria and plants, threonine is derived from aspartate [MD:M00018].\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00260  Glycine, serine and threonine metabolism\n",
      "MODULE      ath_M00018  Threonine biosynthesis, aspartate => homoserine => threonine [PATH:ath00260]\n",
      "ath_M00020  Serine biosynthesis, glycerate-3P => serine [PATH:ath00260]\n",
      "ath_M00555  Betaine biosynthesis, choline => betaine [PATH:ath00260]\n",
      "DBLINKS     GO: 0006544 0006563 0070178 0006566\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G13280  AK-LYS1; aspartate kinase 1 [KO:K00928] [EC:2.7.2.4]\n",
      "AT5G14060  CARAB-AK-LYS; Aspartate kinase family protein [KO:K00928] [EC:2.7.2.4]\n",
      "AT3G02020  AK3; aspartate kinase 3 [KO:K00928] [EC:2.7.2.4]\n",
      "AT1G31230  AK-HSDH_I; aspartate kinase-homoserine dehydrogenase i [KO:K12524] [EC:2.7.2.4 1.1.1.3]\n",
      "AT4G19710  AK-HSDH_II; aspartate kinase-homoserine dehydrogenase ii [KO:K12524] [EC:2.7.2.4 1.1.1.3]\n",
      "AT1G14810  semialdehyde dehydrogenase family protein [KO:K00133] [EC:1.2.1.11]\n",
      "AT2G17265  HSK; homoserine kinase [KO:K00872] [EC:2.7.1.39]\n",
      "AT1G72810  Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K01733] [EC:4.2.3.1]\n",
      "AT4G29840  MTO2; Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K01733] [EC:4.2.3.1]\n",
      "AT1G08630  THA1; threonine aldolase 1 [KO:K01620] [EC:4.1.2.48]\n",
      "AT3G04520  THA2; threonine aldolase 2 [KO:K01620] [EC:4.1.2.48]\n",
      "AT4G13890  EDA36; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G13930  SHM4; serine hydroxymethyltransferase 4 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G22020  SHM6; serine hydroxymethyltransferase 6 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G32520  SHM3; serine hydroxymethyltransferase 3 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G37930  SHM1; serine transhydroxymethyltransferase 1 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G36370  SHM7; serine hydroxymethyltransferase 7 [KO:K00600] [EC:2.1.2.1]\n",
      "AT5G26780  SHM2; serine hydroxymethyltransferase 2 [KO:K00600] [EC:2.1.2.1]\n",
      "AT2G13360  AGT; alanine:glyoxylate aminotransferase [KO:K00830] [EC:2.6.1.44 2.6.1.45 2.6.1.51]\n",
      "AT1G68010  HPR; hydroxypyruvate reductase [KO:K15893] [EC:1.1.1.29]\n",
      "AT1G79870  D-isomer specific 2-hydroxyacid dehydrogenase family protein [KO:K15919] [EC:1.1.1.79 1.1.1.81]\n",
      "AT1G12550  HPR3; D-isomer specific 2-hydroxyacid dehydrogenase family protein [KO:K15919] [EC:1.1.1.79 1.1.1.81]\n",
      "AT1G22170  Phosphoglycerate mutase family protein [KO:K01834] [EC:5.4.2.11]\n",
      "AT1G78050  PGM; phosphoglycerate/bisphosphoglycerate mutase [KO:K01834] [EC:5.4.2.11]\n",
      "AT3G50520  Phosphoglycerate mutase family protein [KO:K15634] [EC:5.4.2.11]\n",
      "AT1G09780  iPGAM1; Phosphoglycerate mutase, 2,3-bisphosphoglycerate-independent [KO:K15633] [EC:5.4.2.12]\n",
      "AT3G08590  iPGAM2; Phosphoglycerate mutase, 2,3-bisphosphoglycerate-independent [KO:K15633] [EC:5.4.2.12]\n",
      "AT1G80380  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K15918] [EC:2.7.1.31]\n",
      "AT3G19480  D-3-phosphoglycerate dehydrogenase [KO:K00058] [EC:1.1.1.95 1.1.1.399]\n",
      "AT4G34200  EDA9; D-3-phosphoglycerate dehydrogenase [KO:K00058] [EC:1.1.1.95 1.1.1.399]\n",
      "AT1G17745  PGDH; D-3-phosphoglycerate dehydrogenase [KO:K00058] [EC:1.1.1.95 1.1.1.399]\n",
      "AT2G17630  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00831] [EC:2.6.1.52]\n",
      "AT4G35630  PSAT; phosphoserine aminotransferase [KO:K00831] [EC:2.6.1.52]\n",
      "AT1G18640  PSP; 3-phosphoserine phosphatase [KO:K01079] [EC:3.1.3.3]\n",
      "AT2G42490  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31710  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31690  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G14940  AO1; amine oxidase 1 [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G62810  CuAO1; Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT3G43670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G12290  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT2G26080  GLDP2; glycine decarboxylase P-protein 2 [KO:K00281] [EC:1.4.4.2]\n",
      "AT4G33010  GLDP1; glycine decarboxylase P-protein 1 [KO:K00281] [EC:1.4.4.2]\n",
      "AT1G11860  Glycine cleavage T-protein family [KO:K00605] [EC:2.1.2.10]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G32470  Single hybrid motif superfamily protein [KO:K02437]\n",
      "AT2G35120  Single hybrid motif superfamily protein [KO:K02437]\n",
      "AT2G35370  GDCH; glycine decarboxylase complex H [KO:K02437]\n",
      "AT4G39660  AGT2; alanine:glyoxylate aminotransferase 2 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT3G08860  PYD4; PYRIMIDINE 4 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT2G38400  AGT3; alanine:glyoxylate aminotransferase 3 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT1G70580  AOAT2; alanine-2-oxoglutarate aminotransferase 2 [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT1G23310  GGT1; glutamate:glyoxylate aminotransferase [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT4G29890  choline monooxygenase, putative (CMO-like) [KO:K00499] [EC:1.14.15.7]\n",
      "AT3G48170  ALDH10A9; aldehyde dehydrogenase 10A9 [KO:K00130] [EC:1.2.1.8]\n",
      "AT1G74920  ALDH10A8; aldehyde dehydrogenase 10A8 [KO:K00130] [EC:1.2.1.8]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT2G24580  FAD-dependent oxidoreductase family protein [KO:K00306] [EC:1.5.3.1 1.5.3.7]\n",
      "AT3G10050  OMR1; L-O-methylthreonine resistant 1 [KO:K01754] [EC:4.3.1.19]\n",
      "AT4G11640  SR; serine racemase [KO:K12235] [EC:5.1.1.18]\n",
      "AT4G02610  Aldolase-type TIM barrel family protein [KO:K01695] [EC:4.2.1.20]\n",
      "AT3G54640  TSA1; tryptophan synthase alpha chain [KO:K01695] [EC:4.2.1.20]\n",
      "AT5G54810  TSB1; tryptophan synthase beta-subunit 1 [KO:K01696] [EC:4.2.1.20]\n",
      "AT4G27070  TSB2; tryptophan synthase beta-subunit 2 [KO:K01696] [EC:4.2.1.20]\n",
      "AT5G28237  Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K01696] [EC:4.2.1.20]\n",
      "AT5G38530  TSBtype2; tryptophan synthase beta type 2 [KO:K06001] [EC:4.2.1.20]\n",
      "COMPOUND    C00011  CO2\n",
      "C00014  Ammonia\n",
      "C00022  Pyruvate\n",
      "C00037  Glycine\n",
      "C00048  Glyoxylate\n",
      "C00049  L-Aspartate\n",
      "C00065  L-Serine\n",
      "C00078  L-Tryptophan\n",
      "C00097  L-Cysteine\n",
      "C00101  Tetrahydrofolate\n",
      "C00109  2-Oxobutanoate\n",
      "C00114  Choline\n",
      "C00143  5,10-Methylenetetrahydrofolate\n",
      "C00168  Hydroxypyruvate\n",
      "C00188  L-Threonine\n",
      "C00197  3-Phospho-D-glycerate\n",
      "C00213  Sarcosine\n",
      "C00258  D-Glycerate\n",
      "C00263  L-Homoserine\n",
      "C00300  Creatine\n",
      "C00430  5-Aminolevulinate\n",
      "C00441  L-Aspartate 4-semialdehyde\n",
      "C00546  Methylglyoxal\n",
      "C00576  Betaine aldehyde\n",
      "C00581  Guanidinoacetate\n",
      "C00631  2-Phospho-D-glycerate\n",
      "C00719  Betaine\n",
      "C00740  D-Serine\n",
      "C00986  1,3-Diaminopropane\n",
      "C01005  O-Phospho-L-serine\n",
      "C01026  N,N-Dimethylglycine\n",
      "C01102  O-Phospho-L-homoserine\n",
      "C01242  [Protein]-S8-aminomethyldihydrolipoyllysine\n",
      "C01888  Aminoacetone\n",
      "C02051  Lipoylprotein\n",
      "C02291  L-Cystathionine\n",
      "C02737  Phosphatidylserine\n",
      "C02972  Dihydrolipoylprotein\n",
      "C03082  4-Phospho-L-aspartate\n",
      "C03194  (R)-1-Aminopropan-2-ol\n",
      "C03232  3-Phosphonooxypyruvate\n",
      "C03283  L-2,4-Diaminobutanoate\n",
      "C03508  L-2-Amino-3-oxobutanoic acid\n",
      "C05519  L-Allothreonine\n",
      "C06231  Ectoine\n",
      "C06442  N(gamma)-Acetyldiaminobutyrate\n",
      "C16432  5-Hydroxyectoine\n",
      "C19929  N(alpha)-Acetyl-L-2,4-diaminobutyrate\n",
      "REFERENCE\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REFERENCE   PMID:11823218\n",
      "AUTHORS   Kuhlmann AU, Bremer E.\n",
      "TITLE     Osmotically regulated synthesis of the compatible solute ectoine in Bacillus pasteurii and related Bacillus spp.\n",
      "JOURNAL   Appl Environ Microbiol 68:772-83 (2002)\n",
      "DOI:10.1128/AEM.68.2.772-783.2002\n",
      "REFERENCE   PMID:10473374\n",
      "AUTHORS   Canovas D, Borges N, Vargas C, Ventosa A, Nieto JJ, Santos H.\n",
      "TITLE     Role of Ngamma-acetyldiaminobutyrate as an enzyme stabilizer and an intermediate in the biosynthesis of hydroxyectoine.\n",
      "JOURNAL   Appl Environ Microbiol 65:3774-9 (1999)\n",
      "DOI:10.1128/AEM.65.9.3774-3779.1999\n",
      "REFERENCE   PMID:17636255\n",
      "AUTHORS   Bursy J, Pierik AJ, Pica N, Bremer E.\n",
      "TITLE     Osmotically induced synthesis of the compatible solute hydroxyectoine is mediated by an evolutionarily conserved ectoine hydroxylase.\n",
      "JOURNAL   J Biol Chem 282:31147-55 (2007)\n",
      "DOI:10.1074/jbc.M704023200\n",
      "REFERENCE   PMID:16707670\n",
      "AUTHORS   Garcia-Estepa R, Argandona M, Reina-Bueno M, Capote N, Iglesias-Guerra F, Nieto JJ, Vargas C.\n",
      "TITLE     The ectD gene, which is involved in the synthesis of the compatible solute hydroxyectoine, is essential for thermoprotection of the halophilic bacterium Chromohalobacter salexigens.\n",
      "JOURNAL   J Bacteriol 188:3774-84 (2006)\n",
      "DOI:10.1128/JB.00136-06\n",
      "REFERENCE   PMID:15470257\n",
      "AUTHORS   Arevalo-Rodriguez M, Pan X, Boeke JD, Heitman J\n",
      "TITLE     FKBP12 controls aspartate pathway flux in Saccharomyces cerevisiae to prevent toxic intermediate accumulation.\n",
      "JOURNAL   Eukaryot Cell 3:1287-96 (2004)\n",
      "DOI:10.1128/EC.3.5.1287-1296.2004\n",
      "REFERENCE   PMID:7639721\n",
      "AUTHORS   Rees WD, Hay SM\n",
      "TITLE     The biosynthesis of threonine by mammalian cells: expression of a complete bacterial biosynthetic pathway in an animal cell.\n",
      "JOURNAL   Biochem J 309 ( Pt 3):999-1007 (1995)\n",
      "DOI:10.1042/bj3090999\n",
      "REFERENCE   PMID:11341914\n",
      "AUTHORS   Jacques SL, Nieman C, Bareich D, Broadhead G, Kinach R, Honek JF, Wright GD\n",
      "TITLE     Characterization of yeast homoserine dehydrogenase, an antifungal target: the invariant histidine 309 is important for enzyme integrity.\n",
      "JOURNAL   Biochim Biophys Acta 1544:28-41 (2001)\n",
      "DOI:10.1016/S0167-4838(00)00203-X\n",
      "REFERENCE   PMID:15520012\n",
      "AUTHORS   Chen X, Jhee KH, Kruger WD\n",
      "TITLE     Production of the neuromodulator H2S by cystathionine beta-synthase via the condensation of cysteine and homocysteine.\n",
      "JOURNAL   J Biol Chem 279:52082-6 (2004)\n",
      "DOI:10.1074/jbc.C400481200\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00230  Purine metabolism\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00460  Cyanoamino acid metabolism\n",
      "ath00470  D-Amino acid metabolism\n",
      "ath00564  Glycerophospholipid metabolism\n",
      "ath00600  Sphingolipid metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00640  Propanoate metabolism\n",
      "ath00860  Porphyrin and chlorophyll metabolism\n",
      "ath00920  Sulfur metabolism\n",
      "KO_PATHWAY  ko00260\n",
      "///\n",
      "ENTRY       ath00261                    Pathway\n",
      "NAME        Monobactam biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Monobactams are beta-lactam antibiotics containing a monocyclic beta-lactam nucleus, which is structurally different from penicillin and cephalosporin core structures with another fused ring. This diagram shows biosynthesis of nocardicin A, a naturally occurring monobactam, via the pentapeptide formed by condensation of L-4-hydroxyphenylglycine (L-pHPG), L-arginine and L-serine [MD:M00736]. Other naturally occurring monobactams are also shown but the biosynthetic pathway is not yet fully characterized. Sulfazecin and other 3-aminomonobactamic acid derivatives are derived from serine or threonine, and tabtoxinine-beta-lactam is a phytotoxin. Aztreonam, a synthetic monobactam originally isolated as SQ 26,180 from Chromobacterium violaceum, is the first clinically used monobactam.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00261  Monobactam biosynthesis\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G14680  APS3; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT3G22890  APS1; ATP sulfurylase 1 [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT1G19920  APS2; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT5G43780  APS4; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT5G13280  AK-LYS1; aspartate kinase 1 [KO:K00928] [EC:2.7.2.4]\n",
      "AT5G14060  CARAB-AK-LYS; Aspartate kinase family protein [KO:K00928] [EC:2.7.2.4]\n",
      "AT3G02020  AK3; aspartate kinase 3 [KO:K00928] [EC:2.7.2.4]\n",
      "AT1G31230  AK-HSDH_I; aspartate kinase-homoserine dehydrogenase i [KO:K12524] [EC:2.7.2.4 1.1.1.3]\n",
      "AT4G19710  AK-HSDH_II; aspartate kinase-homoserine dehydrogenase ii [KO:K12524] [EC:2.7.2.4 1.1.1.3]\n",
      "AT1G14810  semialdehyde dehydrogenase family protein [KO:K00133] [EC:1.2.1.11]\n",
      "AT2G45440  DHDPS2; dihydrodipicolinate synthase [KO:K01714] [EC:4.3.3.7]\n",
      "AT3G60880  DHDPS1; dihydrodipicolinate synthase 1 [KO:K01714] [EC:4.3.3.7]\n",
      "AT3G59890  Dihydrodipicolinate reductase, bacterial/plant [KO:K00215] [EC:1.17.1.8]\n",
      "AT2G44040  Dihydrodipicolinate reductase, bacterial/plant [KO:K00215] [EC:1.17.1.8]\n",
      "COMPOUND    C00019  S-Adenosyl-L-methionine\n",
      "C00022  Pyruvate\n",
      "C00049  L-Aspartate\n",
      "C00059  Sulfate\n",
      "C00062  L-Arginine\n",
      "C00065  L-Serine\n",
      "C00082  L-Tyrosine\n",
      "C00188  L-Threonine\n",
      "C00224  Adenylyl sulfate\n",
      "C00441  L-Aspartate 4-semialdehyde\n",
      "C00927  Isonocardicin A\n",
      "C01179  3-(4-Hydroxyphenyl)pyruvate\n",
      "C01739  Nocardicin E\n",
      "C01941  Nocardicin A\n",
      "C03082  4-Phospho-L-aspartate\n",
      "C03198  (S)-4-Hydroxymandelate\n",
      "C03590  4-Hydroxyphenylglyoxylate\n",
      "C03972  2,3,4,5-Tetrahydrodipicolinate\n",
      "C05539  N-Acetyl-L-2-amino-6-oxopimelate\n",
      "C12323  L-4-Hydroxyphenylglycine\n",
      "C16842  SQ 26180\n",
      "C17351  Nocardicin C\n",
      "C17352  Isonocardicin C\n",
      "C17355  Nocardicin G\n",
      "C20258  (2S,4S)-4-Hydroxy-2,3,4,5-tetrahydrodipicolinate\n",
      "C20911  C20911\n",
      "C20912  C20912\n",
      "C20913  C20913\n",
      "C20914  C20914\n",
      "C20915  C20915\n",
      "C20916  C20916\n",
      "C20917  Tabtoxin\n",
      "C20918  Tabtoxinine-beta-lactam\n",
      "C20919  Isotabtoxin\n",
      "C20920  Tabtoxinine-delta-lactam\n",
      "C20921  N1-Acetyl-tabtoxinine-beta-lactam\n",
      "C20927  Sulfazecin\n",
      "C20928  MM 42842\n",
      "C20929  3alpha-Hydroxy-3-aminoacylmonobactamic acid\n",
      "REFERENCE   PMID:23135477\n",
      "AUTHORS   Hamed RB, Gomez-Castellanos JR, Henry L, Ducho C, McDonough MA, Schofield CJ\n",
      "TITLE     The enzymes of beta-lactam biosynthesis.\n",
      "JOURNAL   Nat Prod Rep 30:21-107 (2013)\n",
      "DOI:10.1039/c2np20065a\n",
      "REFERENCE   PMID:15271351\n",
      "AUTHORS   Gunsior M, Breazeale SD, Lind AJ, Ravel J, Janc JW, Townsend CA\n",
      "TITLE     The biosynthetic gene cluster for a monocyclic beta-lactam antibiotic, nocardicin A.\n",
      "JOURNAL   Chem Biol 11:927-38 (2004)\n",
      "DOI:10.1016/j.chembiol.2004.04.012\n",
      "REFERENCE   PMID:15252031\n",
      "AUTHORS   Kelly WL, Townsend CA\n",
      "TITLE     Mutational analysis and characterization of nocardicin C-9' epimerase.\n",
      "JOURNAL   J Biol Chem 279:38220-7 (2004)\n",
      "DOI:10.1074/jbc.M405450200\n",
      "REFERENCE   PMID:15629944\n",
      "AUTHORS   Kelly WL, Townsend CA\n",
      "TITLE     Mutational analysis of nocK and nocL in the nocardicin a producer Nocardia uniformis.\n",
      "JOURNAL   J Bacteriol 187:739-46 (2005)\n",
      "DOI:10.1128/JB.187.2.739-746.2005\n",
      "REFERENCE   PMID:9804844\n",
      "AUTHORS   Reeve AM, Breazeale SD, Townsend CA\n",
      "TITLE     Purification, characterization, and cloning of an S-adenosylmethionine-dependent 3-amino-3-carboxypropyltransferase in nocardicin biosynthesis.\n",
      "JOURNAL   J Biol Chem 273:30695-703 (1998)\n",
      "DOI:10.1074/jbc.273.46.30695\n",
      "REFERENCE   PMID:12105888\n",
      "AUTHORS   Kelly WL, Townsend CA\n",
      "TITLE     Role of the cytochrome P450 NocL in nocardicin A biosynthesis.\n",
      "JOURNAL   J Am Chem Soc 124:8186-7 (2002)\n",
      "DOI:10.1021/ja025926g\n",
      "REFERENCE   PMID:11137816\n",
      "AUTHORS   Hubbard BK, Thomas MG, Walsh CT\n",
      "TITLE     Biosynthesis of L-p-hydroxyphenylglycine, a non-proteinogenic amino acid constituent of peptide antibiotics.\n",
      "JOURNAL   Chem Biol 7:931-42 (2000)\n",
      "DOI:10.1016/S1074-5521(00)00043-0\n",
      "REFERENCE   PMID:25624104\n",
      "AUTHORS   Gaudelli NM, Long DH, Townsend CA\n",
      "TITLE     beta-Lactam formation by a non-ribosomal peptide synthetase during antibiotic biosynthesis.\n",
      "JOURNAL   Nature 520:383-7 (2015)\n",
      "DOI:10.1038/nature14100\n",
      "REFERENCE   PMID:24531841\n",
      "AUTHORS   Gaudelli NM, Townsend CA\n",
      "TITLE     Epimerization and substrate gating by a TE domain in beta-lactam antibiotic biosynthesis.\n",
      "JOURNAL   Nat Chem Biol 10:251-8 (2014)\n",
      "DOI:10.1038/nchembio.1456\n",
      "REFERENCE   PMID:22365611\n",
      "AUTHORS   Davidsen JM, Townsend CA\n",
      "TITLE     In vivo characterization of nonribosomal peptide synthetases NocA and NocB in the biosynthesis of nocardicin A.\n",
      "JOURNAL   Chem Biol 19:297-306 (2012)\n",
      "DOI:10.1016/j.chembiol.2011.10.020\n",
      "REFERENCE   PMID:23330869\n",
      "AUTHORS   Davidsen JM, Bartley DM, Townsend CA\n",
      "TITLE     Non-ribosomal propeptide precursor in nocardicin A biosynthesis predicted from adenylation domain specificity dependent on the MbtH family protein NocI.\n",
      "JOURNAL   J Am Chem Soc 135:1749-59 (2013)\n",
      "DOI:10.1021/ja307710d\n",
      "REFERENCE   PMID:6805424\n",
      "AUTHORS   O'Sullivan J, Gillum AM, Aklonis CA, Souser ML, Sykes RB\n",
      "TITLE     Biosynthesis of monobactam compounds: origin of the carbon atoms in the beta-lactam ring.\n",
      "JOURNAL   Antimicrob Agents Chemother 21:558-64 (1982)\n",
      "DOI:10.1128/AAC.21.4.558\n",
      "REFERENCE   PMID:6859838\n",
      "AUTHORS   O'Sullivan J, Souser ML, Kao CC, Aklonis CA\n",
      "TITLE     Sulfur metabolism in the biosynthesis of monobactams.\n",
      "JOURNAL   Antimicrob Agents Chemother 23:598-602 (1983)\n",
      "DOI:10.1128/AAC.23.4.598\n",
      "REFERENCE   PMID:16506699\n",
      "AUTHORS   Kinscherf TG, Willis DK\n",
      "TITLE     The biosynthetic gene cluster for the beta-lactam antibiotic tabtoxin in Pseudomonas syringae.\n",
      "JOURNAL   J Antibiot (Tokyo) 58:817-21 (2005)\n",
      "DOI:10.1038/ja.2005.109\n",
      "REFERENCE   PMID:22994681\n",
      "AUTHORS   Wencewicz TA, Walsh CT\n",
      "TITLE     Pseudomonas syringae self-protection from tabtoxinine-beta-lactam by ligase TblF and acetylase Ttr.\n",
      "JOURNAL   Biochemistry 51:7712-25 (2012)\n",
      "DOI:10.1021/bi3011384\n",
      "REFERENCE   PMID:23770908\n",
      "AUTHORS   Arai T, Arimura Y, Ishikura S, Kino K\n",
      "TITLE     L-amino acid ligase from Pseudomonas syringae producing tabtoxin can be used for enzymatic synthesis of various functional peptides.\n",
      "JOURNAL   Appl Environ Microbiol 79:5023-9 (2013)\n",
      "DOI:10.1128/AEM.01003-13\n",
      "REFERENCE   PMID:1515668\n",
      "AUTHORS   Engst K, Shaw PD\n",
      "TITLE     Identification of a lysA-like gene required for tabtoxin biosynthesis and pathogenicity in Pseudomonas syringae pv. tabaci strain PTBR2.024.\n",
      "JOURNAL   Mol Plant Microbe Interact 5:322-9 (1992)\n",
      "DOI:10.1094/mpmi-5-322\n",
      "REFERENCE   PMID:8990304\n",
      "AUTHORS   Liu L, Shaw PD\n",
      "TITLE     Characterization of dapB, a gene required by Pseudomonas syringae pv. tabaci BR2.024 for lysine and tabtoxinine-beta-lactam biosynthesis.\n",
      "JOURNAL   J Bacteriol 179:507-13 (1997)\n",
      "DOI:10.1128/JB.179.2.507-513.1997\n",
      "REFERENCE   PMID:9294453\n",
      "AUTHORS   Liu L, Shaw PD\n",
      "TITLE     A possible role for acetylated intermediates in diaminopimelate and tabtoxinine-beta-lactam biosynthesis in Pseudomonas syringae pv. tabaci BR2.024.\n",
      "JOURNAL   J Bacteriol 179:5922-7 (1997)\n",
      "DOI:10.1128/JB.179.18.5922-5927.1997\n",
      "REFERENCE   PMID:2881927\n",
      "AUTHORS   Unkefer CJ, London RE, Durbin RD, Uchytil TF, Langston-Unkefer PJ\n",
      "TITLE     The biosynthesis of tabtoxinine-beta-lactam. Use of specifically 13C-labeled glucose and 13C NMR spectroscopy to identify its biosynthetic precursors.\n",
      "JOURNAL   J Biol Chem 262:4994-9 (1987)\n",
      "REL_PATHWAY ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "KO_PATHWAY  ko00261\n",
      "///\n",
      "ENTRY       ath00270                    Pathway\n",
      "NAME        Cysteine and methionine metabolism - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Cysteine and methionine are sulfur-containing amino acids. Cysteine is synthesized from serine through different pathways in different organism groups. In bacteria and plants, cysteine is converted from serine (via acetylserine) by transfer of hydrogen sulfide [MD:M00021]. In animals, methionine-derived homocysteine is used as sulfur source and its condensation product with serine (cystathionine) is converted to cysteine [MD:M00338]. Cysteine is metabolized to pyruvate in multiple routes. Methionine is an essential amino acid, which animals cannot synthesize. In bacteria and plants, methionine is synthesized from aspartate [MD:M00017]. S-Adenosylmethionine (SAM), synthesized from methionine and ATP, is a methyl group donor in many important transfer reactions including DNA methylation for regulation of gene expression. SAM may also be used to regenerate methionine in the methionine salvage pathway [MD:M00034].\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00270  Cysteine and methionine metabolism\n",
      "MODULE      ath_M00021  Cysteine biosynthesis, serine => cysteine [PATH:ath00270]\n",
      "ath_M00034  Methionine salvage pathway [PATH:ath00270]\n",
      "ath_M00368  Ethylene biosynthesis, methionine => ethylene [PATH:ath00270]\n",
      "DBLINKS     GO: 0006534 0006555\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G55920  SERAT2;1; serine acetyltransferase 2;1 [KO:K00640] [EC:2.3.1.30]\n",
      "AT5G56760  SERAT1;1; serine acetyltransferase 1;1 [KO:K00640] [EC:2.3.1.30]\n",
      "AT3G13110  SERAT2;2; serine acetyltransferase 2;2 [KO:K00640] [EC:2.3.1.30]\n",
      "AT4G35640  SERAT3;2; serine acetyltransferase 3;2 [KO:K00640] [EC:2.3.1.30]\n",
      "AT2G17640  ATSERAT3;1; Trimeric LpxA-like enzymes superfamily protein [KO:K00640] [EC:2.3.1.30]\n",
      "AT1G55880  Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K01738] [EC:2.5.1.47]\n",
      "AT2G43750  OASB; O-acetylserine (thiol) lyase B [KO:K01738] [EC:2.5.1.47]\n",
      "AT3G04940  CYSD1; cysteine synthase D1 [KO:K01738] [EC:2.5.1.47]\n",
      "AT3G22460  OASA2; O-acetylserine (thiol) lyase (OAS-TL) isoform A2 [KO:K01738] [EC:2.5.1.47]\n",
      "AT5G28030  DES1; L-cysteine desulfhydrase 1 [KO:K01738] [EC:2.5.1.47]\n",
      "AT4G14880  OASA1; O-acetylserine (thiol) lyase (OAS-TL) isoform A1 [KO:K01738] [EC:2.5.1.47]\n",
      "AT3G59760  OASC; O-acetylserine (thiol) lyase isoform C [KO:K01738] [EC:2.5.1.47]\n",
      "AT5G28020  CYSD2; cysteine synthase D2 [KO:K01738] [EC:2.5.1.47]\n",
      "AT3G61440  CYSC1; cysteine synthase C1 [KO:K13034] [EC:2.5.1.47 4.4.1.9]\n",
      "AT3G57050  CBL; cystathionine beta-lyase [KO:K01760] [EC:4.4.1.13]\n",
      "AT4G23600  CORI3; Tyrosine transaminase family protein [KO:K21623] [EC:4.4.1.35]\n",
      "AT3G22740  HMT3; homocysteine S-methyltransferase 3 [KO:K00547] [EC:2.1.1.10]\n",
      "AT3G25900  HMT-1; Homocysteine S-methyltransferase family protein [KO:K00547] [EC:2.1.1.10]\n",
      "AT3G63250  HMT2; homocysteine methyltransferase 2 [KO:K00547] [EC:2.1.1.10]\n",
      "AT5G20980  MS3; methionine synthase 3 [KO:K00549] [EC:2.1.1.14]\n",
      "AT5G17920  ATMS1; Cobalamin-independent synthase family protein [KO:K00549] [EC:2.1.1.14]\n",
      "AT3G03780  MS2; methionine synthase 2 [KO:K00549] [EC:2.1.1.14]\n",
      "AT2G36880  MAT3; methionine adenosyltransferase 3 [KO:K00789] [EC:2.5.1.6]\n",
      "AT3G17390  MTO3; S-adenosylmethionine synthetase family protein [KO:K00789] [EC:2.5.1.6]\n",
      "AT4G01850  SAM-2; S-adenosylmethionine synthetase 2 [KO:K00789] [EC:2.5.1.6]\n",
      "AT1G02500  SAM1; S-adenosylmethionine synthetase 1 [KO:K00789] [EC:2.5.1.6]\n",
      "AT3G25570  Adenosylmethionine decarboxylase family protein [KO:K01611] [EC:4.1.1.50]\n",
      "AT5G18930  BUD2; Adenosylmethionine decarboxylase family protein [KO:K01611] [EC:4.1.1.50]\n",
      "AT5G15950  Adenosylmethionine decarboxylase family protein [KO:K01611] [EC:4.1.1.50]\n",
      "AT3G02470  SAMDC; S-adenosylmethionine decarboxylase [KO:K01611] [EC:4.1.1.50]\n",
      "AT1G23820  SPDS1; spermidine synthase [KO:K00797] [EC:2.5.1.16]\n",
      "AT1G70310  SPDS2; spermidine synthase 2 [KO:K00797] [EC:2.5.1.16]\n",
      "AT5G53120  SPDS3; spermidine synthase 3 [KO:K00797] [EC:2.5.1.16]\n",
      "AT5G04950  NAS1; nicotianamine synthase 1 [KO:K05953] [EC:2.5.1.43]\n",
      "AT5G56080  NAS2; nicotianamine synthase 2 [KO:K05953] [EC:2.5.1.43]\n",
      "AT1G09240  NAS3; nicotianamine synthase 3 [KO:K05953] [EC:2.5.1.43]\n",
      "AT1G56430  NAS4; nicotianamine synthase 4 [KO:K05953] [EC:2.5.1.43]\n",
      "AT4G34840  Phosphorylase superfamily protein [KO:K01244] [EC:3.2.2.16]\n",
      "AT4G38800  MTN1; methylthioadenosine nucleosidase 1 [KO:K01244] [EC:3.2.2.16]\n",
      "AT1G49820  MTK; S-methyl-5-thioribose kinase [KO:K00899] [EC:2.7.1.100]\n",
      "AT2G05830  NagB/RpiA/CoA transferase-like superfamily protein [KO:K08963] [EC:5.3.1.23]\n",
      "AT5G53850  haloacid dehalogenase-like hydrolase family protein [KO:K16054] [EC:4.2.1.109 3.1.3.77]\n",
      "AT4G14710  ATARD2; RmlC-like cupins superfamily protein [KO:K08967] [EC:1.13.11.53 1.13.11.54]\n",
      "AT4G14716  ARD1; acireductone dioxygenase 1 [KO:K08967] [EC:1.13.11.53 1.13.11.54]\n",
      "AT2G26400  ARD3; acireductone dioxygenase 3 [KO:K08967] [EC:1.13.11.53 1.13.11.54]\n",
      "AT5G43850  ARD4; RmlC-like cupins superfamily protein [KO:K08967] [EC:1.13.11.53 1.13.11.54]\n",
      "AT1G77670  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K23977] [EC:2.6.1.117]\n",
      "AT5G53970  TAT7; Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT5G36160  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT2G24850  TAT3; tyrosine aminotransferase 3 [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28410  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28420  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G23590  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT1G80360  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00837] [EC:2.6.1.-]\n",
      "AT1G64660  MGL; methionine gamma-lyase [KO:K01761] [EC:4.4.1.11]\n",
      "AT4G13610  MEE57; DNA (cytosine-5-)-methyltransferase family protein [KO:K00558] [EC:2.1.1.37]\n",
      "AT4G14140  DMT2; DNA methyltransferase 2 [KO:K00558] [EC:2.1.1.37]\n",
      "AT5G49160  MET1; methyltransferase 1 [KO:K00558] [EC:2.1.1.37]\n",
      "AT1G69770  CMT3; chromomethylase 3 [KO:K00558] [EC:2.1.1.37]\n",
      "AT1G80740  CMT1; chromomethylase 1 [KO:K00558] [EC:2.1.1.37]\n",
      "AT4G19020  CMT2; chromomethylase 2 [KO:K00558] [EC:2.1.1.37]\n",
      "AT4G08990  DNA (cytosine-5-)-methyltransferase family protein [KO:K00558] [EC:2.1.1.37]\n",
      "AT5G27650  Tudor/PWWP/MBT superfamily protein [KO:K17398] [EC:2.1.1.37]\n",
      "AT3G23810  SAHH2; S-adenosyl-l-homocysteine (SAH) hydrolase 2 [KO:K01251] [EC:3.3.1.1]\n",
      "AT4G13940  HOG1; S-adenosyl-L-homocysteine hydrolase [KO:K01251] [EC:3.3.1.1]\n",
      "AT4G11280  ACS6; 1-aminocyclopropane-1-carboxylic acid (acc) synthase 6 [KO:K20772] [EC:4.4.1.14]\n",
      "AT1G01480  ACS2; 1-amino-cyclopropane-1-carboxylate synthase 2 [KO:K20772] [EC:4.4.1.14]\n",
      "AT3G61510  ACS1; ACC synthase 1 [KO:K20772] [EC:4.4.1.14]\n",
      "AT4G37770  ACS8; 1-amino-cyclopropane-1-carboxylate synthase 8 [KO:K01762] [EC:4.4.1.14]\n",
      "AT3G49700  ACS9; 1-aminocyclopropane-1-carboxylate synthase 9 [KO:K01762] [EC:4.4.1.14]\n",
      "AT2G22810  ACS4; 1-aminocyclopropane-1-carboxylate synthase 4 [KO:K01762] [EC:4.4.1.14]\n",
      "AT5G65800  ACS5; ACC synthase 5 [KO:K01762] [EC:4.4.1.14]\n",
      "AT4G08040  ACS11; 1-aminocyclopropane-1-carboxylate synthase 11 [KO:K01762] [EC:4.4.1.14]\n",
      "AT4G26200  ACS7; 1-amino-cyclopropane-1-carboxylate synthase 7 [KO:K01762] [EC:4.4.1.14]\n",
      "AT1G05010  EFE; ethylene-forming enzyme [KO:K05933] [EC:1.14.17.4]\n",
      "AT1G12010  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K05933] [EC:1.14.17.4]\n",
      "AT2G19590  ACO1; ACC oxidase 1 [KO:K05933] [EC:1.14.17.4]\n",
      "AT1G62380  ACO2; ACC oxidase 2 [KO:K05933] [EC:1.14.17.4]\n",
      "AT1G77330  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K05933] [EC:1.14.17.4]\n",
      "AT5G13280  AK-LYS1; aspartate kinase 1 [KO:K00928] [EC:2.7.2.4]\n",
      "AT5G14060  CARAB-AK-LYS; Aspartate kinase family protein [KO:K00928] [EC:2.7.2.4]\n",
      "AT3G02020  AK3; aspartate kinase 3 [KO:K00928] [EC:2.7.2.4]\n",
      "AT1G31230  AK-HSDH_I; aspartate kinase-homoserine dehydrogenase i [KO:K12524] [EC:2.7.2.4 1.1.1.3]\n",
      "AT4G19710  AK-HSDH_II; aspartate kinase-homoserine dehydrogenase ii [KO:K12524] [EC:2.7.2.4 1.1.1.3]\n",
      "AT1G14810  semialdehyde dehydrogenase family protein [KO:K00133] [EC:1.2.1.11]\n",
      "AT3G01120  MTO1; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01739] [EC:2.5.1.48]\n",
      "AT1G33320  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01739] [EC:2.5.1.48]\n",
      "AT1G50090  BCAT7; D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G50110  D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10070  BCAT-2; branched-chain amino acid transaminase 2 [KO:K00826] [EC:2.6.1.42]\n",
      "AT3G49680  BCAT3; branched-chain aminotransferase 3 [KO:K00826] [EC:2.6.1.42]\n",
      "AT5G65780  ATBCAT-5; branched-chain amino acid aminotransferase 5 / branched-chain amino acid transaminase 5 (BCAT5) [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10060  BCAT-1; branched-chain amino acid transaminase 1 [KO:K00826] [EC:2.6.1.42]\n",
      "AT4G39660  AGT2; alanine:glyoxylate aminotransferase 2 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT3G08860  PYD4; PYRIMIDINE 4 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT2G38400  AGT3; alanine:glyoxylate aminotransferase 3 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT4G23100  GSH1; glutamate-cysteine ligase [KO:K01919] [EC:6.3.2.2]\n",
      "AT5G27380  GSH2; glutathione synthetase 2 [KO:K21456] [EC:6.3.2.3]\n",
      "AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "AT1G79230  MST1; mercaptopyruvate sulfurtransferase 1 [KO:K01011] [EC:2.8.1.1 2.8.1.2]\n",
      "AT1G16460  RDH2; rhodanese homologue 2 [KO:K01011] [EC:2.8.1.1 2.8.1.2]\n",
      "AT4G17260  Lactate/malate dehydrogenase family protein [KO:K00016] [EC:1.1.1.27]\n",
      "AT1G48420  D-CDES; D-cysteine desulfhydrase [KO:K05396] [EC:4.4.1.15]\n",
      "AT3G26115  Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K05396] [EC:4.4.1.15]\n",
      "AT3G62130  LCD; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K22207] [EC:4.4.1.28]\n",
      "AT5G26600  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K22207] [EC:4.4.1.28]\n",
      "AT1G04410  c-NAD-MDH1; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G56720  c-NAD-MDH3; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G43330  c-NAD-MDH2; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT3G47520  MDH; malate dehydrogenase [KO:K00026] [EC:1.1.1.37]\n",
      "AT1G53240  mMDH1; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT2G22780  PMDH1; peroxisomal NAD-malate dehydrogenase 1 [KO:K00026] [EC:1.1.1.37]\n",
      "AT3G15020  mMDH2; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT5G09660  PMDH2; peroxisomal NAD-malate dehydrogenase 2 [KO:K00026] [EC:1.1.1.37]\n",
      "AT3G03630  CS26; cysteine synthase 26 [KO:K22846] [EC:2.5.1.144]\n",
      "AT3G19480  D-3-phosphoglycerate dehydrogenase [KO:K00058] [EC:1.1.1.95 1.1.1.399]\n",
      "AT4G34200  EDA9; D-3-phosphoglycerate dehydrogenase [KO:K00058] [EC:1.1.1.95 1.1.1.399]\n",
      "AT1G17745  PGDH; D-3-phosphoglycerate dehydrogenase [KO:K00058] [EC:1.1.1.95 1.1.1.399]\n",
      "AT2G17630  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00831] [EC:2.6.1.52]\n",
      "AT4G35630  PSAT; phosphoserine aminotransferase [KO:K00831] [EC:2.6.1.52]\n",
      "COMPOUND    C00019  S-Adenosyl-L-methionine\n",
      "C00021  S-Adenosyl-L-homocysteine\n",
      "C00022  Pyruvate\n",
      "C00041  L-Alanine\n",
      "C00049  L-Aspartate\n",
      "C00051  Glutathione\n",
      "C00059  Sulfate\n",
      "C00065  L-Serine\n",
      "C00073  L-Methionine\n",
      "C00094  Sulfite\n",
      "C00097  L-Cysteine\n",
      "C00109  2-Oxobutanoate\n",
      "C00155  L-Homocysteine\n",
      "C00170  5'-Methylthioadenosine\n",
      "C00197  3-Phospho-D-glycerate\n",
      "C00263  L-Homoserine\n",
      "C00283  Hydrogen sulfide\n",
      "C00409  Methanethiol\n",
      "C00441  L-Aspartate 4-semialdehyde\n",
      "C00491  L-Cystine\n",
      "C00506  L-Cysteate\n",
      "C00606  3-Sulfino-L-alanine\n",
      "C00793  D-Cysteine\n",
      "C00957  Mercaptopyruvate\n",
      "C00979  O-Acetyl-L-serine\n",
      "C01005  O-Phospho-L-serine\n",
      "C01077  O-Acetyl-L-homoserine\n",
      "C01118  O-Succinyl-L-homoserine\n",
      "C01137  S-Adenosylmethioninamine\n",
      "C01180  4-Methylthio-2-oxobutanoic acid\n",
      "C01234  1-Aminocyclopropane-1-carboxylate\n",
      "C01817  L-Homocystine\n",
      "C01962  Thiocysteine\n",
      "C02218  Dehydroalanine\n",
      "C02291  L-Cystathionine\n",
      "C02356  (S)-2-Aminobutanoate\n",
      "C02989  L-Methionine S-oxide\n",
      "C03082  4-Phospho-L-aspartate\n",
      "C03089  5-Methylthio-D-ribose\n",
      "C03145  N-Formylmethionine\n",
      "C03232  3-Phosphonooxypyruvate\n",
      "C03539  S-Ribosyl-L-homocysteine\n",
      "C04188  S-Methyl-5-thio-D-ribose 1-phosphate\n",
      "C04582  S-Methyl-5-thio-D-ribulose 1-phosphate\n",
      "C05324  Nicotianamine\n",
      "C05524  Aminoacyl-L-methionine\n",
      "C05526  S-Glutathionyl-L-cysteine\n",
      "C05527  3-Sulfinylpyruvate\n",
      "C05528  3-Sulfopyruvate\n",
      "C05823  3-Mercaptolactate\n",
      "C05824  S-Sulfo-L-cysteine\n",
      "C06547  Ethylene\n",
      "C08276  3-(Methylthio)propanoate\n",
      "C09306  Sulfur dioxide\n",
      "C11481  HSO3-\n",
      "C11499  (S)-3-Sulfolactate\n",
      "C11537  (2R)-3-Sulfolactate\n",
      "C15606  1,2-Dihydroxy-5-(methylthio)pent-1-en-3-one\n",
      "C15650  2,3-Diketo-5-methylthiopentyl-1-phosphate\n",
      "C15651  2-Hydroxy-3-keto-5-methylthiopentenyl-1-phosphate\n",
      "C18049  N-Acyl-L-homoserine lactone\n",
      "C19787  5'-S-Methyl-5'-thioinosine\n",
      "C21015  gamma-L-Glutamyl-L-2-aminobutyrate\n",
      "C21016  Ophthalmate\n",
      "C22359  S-Methyl-1-thio-D-xylulose 5-phosphate\n",
      "REFERENCE   PMID:15102328\n",
      "AUTHORS   Sekowska A, Denervaud V, Ashida H, Michoud K, Haas D, Yokota A, Danchin A\n",
      "TITLE     Bacterial variations on the methionine salvage pathway.\n",
      "JOURNAL   BMC Microbiol 4:9 (2004)\n",
      "DOI:10.1186/1471-2180-4-9\n",
      "REFERENCE   PMID:14551435\n",
      "AUTHORS   Ashida H, Saito Y, Kojima C, Kobayashi K, Ogasawara N, Yokota A.\n",
      "TITLE     A functional link between RuBisCO-like protein of Bacillus and photosynthetic RuBisCO.\n",
      "JOURNAL   Science 302:286-90 (2003)\n",
      "DOI:10.1126/science.1086997\n",
      "REFERENCE   PMID:12022921\n",
      "AUTHORS   Sekowska A, Danchin A.\n",
      "TITLE     The methionine salvage pathway in Bacillus subtilis.\n",
      "JOURNAL   BMC Microbiol 2:8 (2002)\n",
      "DOI:10.1186/1471-2180-2-8\n",
      "REFERENCE   PMID:12670965\n",
      "AUTHORS   Berger BJ, English S, Chan G, Knodel MH.\n",
      "TITLE     Methionine regeneration and aminotransferases in Bacillus subtilis, Bacillus cereus, and Bacillus anthracis.\n",
      "JOURNAL   J Bacteriol 185:2418-31 (2003)\n",
      "DOI:10.1128/JB.185.8.2418-2431.2003\n",
      "REFERENCE   PMID:17169919\n",
      "AUTHORS   Goyer A, Collakova E, Shachar-Hill Y, Hanson AD\n",
      "TITLE     Functional characterization of a methionine gamma-lyase in Arabidopsis and its implication in an alternative to the reverse trans-sulfuration pathway.\n",
      "JOURNAL   Plant Cell Physiol 48:232-42 (2007)\n",
      "DOI:10.1093/pcp/pcl055\n",
      "REFERENCE   PMID:17030798\n",
      "AUTHORS   Rebeille F, Jabrin S, Bligny R, Loizeau K, Gambonnet B, Van Wilder V, Douce R, Ravanel S\n",
      "TITLE     Methionine catabolism in Arabidopsis cells is initiated by a gamma-cleavage process and leads to S-methylcysteine and isoleucine syntheses.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 103:15687-92 (2006)\n",
      "DOI:10.1073/pnas.0606195103\n",
      "REFERENCE   PMID:18625006\n",
      "AUTHORS   Pirkov I, Norbeck J, Gustafsson L, Albers E\n",
      "TITLE     A complete inventory of all enzymes in the eukaryotic methionine salvage pathway.\n",
      "JOURNAL   FEBS J 275:4111-20 (2008)\n",
      "DOI:10.1111/j.1742-4658.2008.06552.x\n",
      "REFERENCE   PMID:18391471\n",
      "AUTHORS   Ashida H, Saito Y, Kojima C, Yokota A\n",
      "TITLE     Enzymatic characterization of 5-methylthioribulose-1-phosphate dehydratase of the methionine salvage pathway in Bacillus subtilis.\n",
      "JOURNAL   Biosci Biotechnol Biochem 72:959-67 (2008)\n",
      "DOI:10.1271/bbb.70651\n",
      "REFERENCE   PMID:10613873\n",
      "AUTHORS   Kitabatake M, So MW, Tumbula DL, Soll D\n",
      "TITLE     Cysteine biosynthesis pathway in the archaeon Methanosarcina barkeri encoded by acquired bacterial genes?\n",
      "JOURNAL   J Bacteriol 182:143-5 (2000)\n",
      "DOI:10.1128/JB.182.1.143-145.2000\n",
      "REFERENCE   PMID:12644499\n",
      "AUTHORS   Mino K, Ishikawa K\n",
      "TITLE     Characterization of a novel thermostable O-acetylserine sulfhydrylase from Aeropyrum pernix K1.\n",
      "JOURNAL   J Bacteriol 185:2277-84 (2003)\n",
      "DOI:10.1128/JB.185.7.2277-2284.2003\n",
      "REFERENCE   PMID:18520135\n",
      "AUTHORS   Tanabe S\n",
      "TITLE     Development of assay methods for endogenous inorganic sulfur compounds and sulfurtransferases and evaluation of the physiological functions of bound sulfur.\n",
      "JOURNAL   Yakugaku Zasshi 128:881-900 (2008)\n",
      "DOI:10.1248/yakushi.128.881\n",
      "REFERENCE\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REFERENCE   PMID:19270684\n",
      "AUTHORS   Gutierrez JA, Crowder T, Rinaldo-Matthis A, Ho MC, Almo SC, Schramm VL\n",
      "TITLE     Transition state analogs of 5'-methylthioadenosine nucleosidase disrupt quorum sensing.\n",
      "JOURNAL   Nat Chem Biol 5:251-7 (2009)\n",
      "DOI:10.1038/nchembio.153\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00430  Taurine and hypotaurine metabolism\n",
      "ath00480  Glutathione metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00640  Propanoate metabolism\n",
      "ath00770  Pantothenate and CoA biosynthesis\n",
      "ath00920  Sulfur metabolism\n",
      "KO_PATHWAY  ko00270\n",
      "///\n",
      "ENTRY       ath00280                    Pathway\n",
      "NAME        Valine, leucine and isoleucine degradation - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00280  Valine, leucine and isoleucine degradation\n",
      "MODULE      ath_M00036  Leucine degradation, leucine => acetoacetate + acetyl-CoA [PATH:ath00280]\n",
      "DBLINKS     GO: 0006550 0006552 0006574\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G50090  BCAT7; D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G50110  D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10070  BCAT-2; branched-chain amino acid transaminase 2 [KO:K00826] [EC:2.6.1.42]\n",
      "AT3G49680  BCAT3; branched-chain aminotransferase 3 [KO:K00826] [EC:2.6.1.42]\n",
      "AT5G65780  ATBCAT-5; branched-chain amino acid aminotransferase 5 / branched-chain amino acid transaminase 5 (BCAT5) [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10060  BCAT-1; branched-chain amino acid transaminase 1 [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G21400  Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00166] [EC:1.2.4.4]\n",
      "AT5G09300  Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00166] [EC:1.2.4.4]\n",
      "AT5G34780  Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00166] [EC:1.2.4.4]\n",
      "AT1G55510  BCDH_BETA1; branched-chain alpha-keto acid decarboxylase E1 beta subunit [KO:K00167] [EC:1.2.4.4]\n",
      "AT3G13450  DIN4; Transketolase family protein [KO:K00167] [EC:1.2.4.4]\n",
      "AT3G06850  BCE2; 2-oxoacid dehydrogenases acyltransferase family protein [KO:K09699] [EC:2.3.1.168]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G06810  IBR3; acyl-CoA dehydrogenase-like protein [KO:K00249] [EC:1.3.8.7]\n",
      "AT3G45300  IVD; isovaleryl-CoA-dehydrogenase [KO:K00253] [EC:1.3.8.4]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "AT1G04710  PKT4; peroxisomal 3-ketoacyl-CoA thiolase 4 [KO:K07513] [EC:2.3.1.16]\n",
      "AT2G33150  PKT3; peroxisomal 3-ketoacyl-CoA thiolase 3 [KO:K07513] [EC:2.3.1.16]\n",
      "AT5G48880  KAT5; peroxisomal 3-keto-acyl-CoA thiolase 2 [KO:K07513] [EC:2.3.1.16]\n",
      "AT2G30650  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT5G65940  CHY1; beta-hydroxyisobutyryl-CoA hydrolase 1 [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G31810  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT1G06550  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT2G30660  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT3G60510  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT3G24360  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G13360  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G29120  6-phosphogluconate dehydrogenase family protein [KO:K00020] [EC:1.1.1.31]\n",
      "AT1G71170  6-phosphogluconate dehydrogenase family protein [KO:K00020] [EC:1.1.1.31]\n",
      "AT1G71180  6-phosphogluconate dehydrogenase family protein [KO:K00020] [EC:1.1.1.31]\n",
      "AT4G20930  6-phosphogluconate dehydrogenase family protein [KO:K23146] [EC:1.1.1.31 1.1.1.59]\n",
      "AT2G14170  ALDH6B2; aldehyde dehydrogenase 6B2 [KO:K00140] [EC:1.2.1.18 1.2.1.27]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT3G16170  AAE13; AMP-dependent synthetase and ligase family protein [KO:K18660] [EC:6.2.1.-]\n",
      "AT4G39660  AGT2; alanine:glyoxylate aminotransferase 2 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT3G08860  PYD4; PYRIMIDINE 4 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT2G38400  AGT3; alanine:glyoxylate aminotransferase 3 [KO:K00827] [EC:2.6.1.44 2.6.1.40]\n",
      "AT1G03090  MCCA; methylcrotonyl-CoA carboxylase alpha chain [KO:K01968] [EC:6.4.1.4]\n",
      "AT4G34030  MCCB; 3-methylcrotonyl-CoA carboxylase [KO:K01969] [EC:6.4.1.4]\n",
      "AT4G16800  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05607] [EC:4.2.1.18]\n",
      "AT2G26800  Aldolase superfamily protein [KO:K01640] [EC:4.1.3.4]\n",
      "AT5G48230  ACAT2; acetoacetyl-CoA thiolase 2 [KO:K00626] [EC:2.3.1.9]\n",
      "AT5G47720  AACT1; Thiolase family protein [KO:K00626] [EC:2.3.1.9]\n",
      "AT4G11820  HMGS; hydroxymethylglutaryl-CoA synthase / HMG-CoA synthase / 3-hydroxy-3-methylglutaryl coenzyme A synthase [KO:K01641] [EC:2.3.3.10]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00068  Thiamin diphosphate\n",
      "C00091  Succinyl-CoA\n",
      "C00100  Propanoyl-CoA\n",
      "C00123  L-Leucine\n",
      "C00141  3-Methyl-2-oxobutanoic acid\n",
      "C00164  Acetoacetate\n",
      "C00183  L-Valine\n",
      "C00233  4-Methyl-2-oxopentanoate\n",
      "C00332  Acetoacetyl-CoA\n",
      "C00349  2-Methyl-3-oxopropanoate\n",
      "C00356  (S)-3-Hydroxy-3-methylglutaryl-CoA\n",
      "C00407  L-Isoleucine\n",
      "C00630  2-Methylpropanoyl-CoA\n",
      "C00671  (S)-3-Methyl-2-oxopentanoic acid\n",
      "C00683  (S)-Methylmalonyl-CoA\n",
      "C01205  (R)-3-Amino-2-methylpropanoate\n",
      "C01213  (R)-Methylmalonyl-CoA\n",
      "C02170  Methylmalonate\n",
      "C02939  3-Methylbutanoyl-CoA\n",
      "C03069  3-Methylcrotonyl-CoA\n",
      "C03231  3-Methylglutaconyl-CoA\n",
      "C03284  L-3-Aminoisobutanoate\n",
      "C03344  2-Methylacetoacetyl-CoA\n",
      "C03345  2-Methylbut-2-enoyl-CoA\n",
      "C03460  2-Methylprop-2-enoyl-CoA\n",
      "C04405  (2S,3S)-3-Hydroxy-2-methylbutanoyl-CoA\n",
      "C05996  Branched chain fatty acid\n",
      "C05998  3-Hydroxyisovaleryl-CoA\n",
      "C06000  (S)-3-Hydroxyisobutyryl-CoA\n",
      "C06001  (S)-3-Hydroxyisobutyrate\n",
      "C06002  (S)-Methylmalonate semialdehyde\n",
      "C15972  Enzyme N6-(lipoyl)lysine\n",
      "C15973  Enzyme N6-(dihydrolipoyl)lysine\n",
      "C15974  3-Methyl-1-hydroxybutyl-ThPP\n",
      "C15975  [Dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] S-(3-methylbutanoyl)dihydrolipoyllysine\n",
      "C15976  2-Methyl-1-hydroxypropyl-ThPP\n",
      "C15977  [Dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] S-(2-methylpropanoyl)dihydrolipoyllysine\n",
      "C15978  2-Methyl-1-hydroxybutyl-ThPP\n",
      "C15979  [Dihydrolipoyllysine-residue (2-methylpropanoyl)transferase] S-(2-methylbutanoyl)dihydrolipoyllysine\n",
      "C15980  (S)-2-Methylbutanoyl-CoA\n",
      "C20827  3-Hydroxyisovalerate\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00240  Pyrimidine metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00640  Propanoate metabolism\n",
      "ath00900  Terpenoid backbone biosynthesis\n",
      "KO_PATHWAY  ko00280\n",
      "///\n",
      "ENTRY       ath00290                    Pathway\n",
      "NAME        Valine, leucine and isoleucine biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "MODULE      ath_M00019  Valine/isoleucine biosynthesis, pyruvate => valine / 2-oxobutanoate => isoleucine [PATH:ath00290]\n",
      "ath_M00432  Leucine biosynthesis, 2-oxoisovalerate => 2-oxoisocaproate [PATH:ath00290]\n",
      "ath_M00570  Isoleucine biosynthesis, threonine => 2-oxobutanoate => isoleucine [PATH:ath00290]\n",
      "DBLINKS     GO: 0009097 0009098 0009099\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G10050  OMR1; L-O-methylthreonine resistant 1 [KO:K01754] [EC:4.3.1.19]\n",
      "AT4G13430  IIL1; isopropyl malate isomerase large subunit 1 [KO:K01703] [EC:4.2.1.33 4.2.1.35]\n",
      "AT2G43090  Aconitase/3-isopropylmalate dehydratase protein [KO:K01704] [EC:4.2.1.33 4.2.1.35]\n",
      "AT1G31180  IMD3; isopropylmalate dehydrogenase 3 [KO:K00052] [EC:1.1.1.85]\n",
      "AT1G80560  IMD2; isopropylmalate dehydrogenase 2 [KO:K00052] [EC:1.1.1.85]\n",
      "AT3G48560  CSR1; chlorsulfuron/imidazolinone resistant 1 [KO:K01652] [EC:2.2.1.6]\n",
      "AT2G31810  ACT domain-containing small subunit of acetolactate synthase protein [KO:K01653] [EC:2.2.1.6]\n",
      "AT5G16290  VAT1; VALINE-TOLERANT 1 [KO:K01653] [EC:2.2.1.6]\n",
      "AT3G58610  ketol-acid reductoisomerase [KO:K00053] [EC:1.1.1.86]\n",
      "AT3G23940  dehydratase family [KO:K01687] [EC:4.2.1.9]\n",
      "AT1G50090  BCAT7; D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G50110  D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10070  BCAT-2; branched-chain amino acid transaminase 2 [KO:K00826] [EC:2.6.1.42]\n",
      "AT3G49680  BCAT3; branched-chain aminotransferase 3 [KO:K00826] [EC:2.6.1.42]\n",
      "AT5G65780  ATBCAT-5; branched-chain amino acid aminotransferase 5 / branched-chain amino acid transaminase 5 (BCAT5) [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10060  BCAT-1; branched-chain amino acid transaminase 1 [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G18500  MAML-4; methylthioalkylmalate synthase-like 4 [KO:K01649] [EC:2.3.3.13]\n",
      "AT1G74040  IMS1; 2-isopropylmalate synthase 1 [KO:K01649] [EC:2.3.3.13]\n",
      "AT2G43100  IPMI2; isopropylmalate isomerase 2 [KO:K21359] [EC:4.2.1.33 4.2.1.170]\n",
      "AT3G58990  IPMI1; isopropylmalate isomerase 1 [KO:K21359] [EC:4.2.1.33 4.2.1.170]\n",
      "AT5G14200  IMD1; isopropylmalate dehydrogenase 1 [KO:K21360] [EC:1.1.1.85 1.1.1.-]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00109  2-Oxobutanoate\n",
      "C00123  L-Leucine\n",
      "C00141  3-Methyl-2-oxobutanoic acid\n",
      "C00183  L-Valine\n",
      "C00188  L-Threonine\n",
      "C00233  4-Methyl-2-oxopentanoate\n",
      "C00407  L-Isoleucine\n",
      "C00671  (S)-3-Methyl-2-oxopentanoic acid\n",
      "C02226  2-Methylmaleate\n",
      "C02504  alpha-Isopropylmalate\n",
      "C02612  (R)-2-Methylmalate\n",
      "C02631  2-Isopropylmaleate\n",
      "C04181  3-Hydroxy-3-methyl-2-oxobutanoic acid\n",
      "C04236  (2S)-2-Isopropyl-3-oxosuccinate\n",
      "C04272  (R)-2,3-Dihydroxy-3-methylbutanoate\n",
      "C04411  (2R,3S)-3-Isopropylmalate\n",
      "C06006  (S)-2-Aceto-2-hydroxybutanoate\n",
      "C06007  (R)-2,3-Dihydroxy-3-methylpentanoate\n",
      "C06010  (S)-2-Acetolactate\n",
      "C06032  D-erythro-3-Methylmalate\n",
      "C14463  (R)-3-Hydroxy-3-methyl-2-oxopentanoate\n",
      "REFERENCE   PMID:15292141\n",
      "AUTHORS   Xu H, Zhang Y, Guo X, Ren S, Staempfli AA, Chiao J, Jiang W, Zhao G.\n",
      "TITLE     Isoleucine biosynthesis in Leptospira interrogans serotype lai strain 56601 proceeds via a threonine-independent pathway.\n",
      "JOURNAL   J Bacteriol 186:5400-9 (2004)\n",
      "DOI:10.1128/JB.186.16.5400-5409.2004\n",
      "REL_PATHWAY ath00260  Glycine, serine and threonine metabolism\n",
      "ath00280  Valine, leucine and isoleucine degradation\n",
      "ath00620  Pyruvate metabolism\n",
      "KO_PATHWAY  ko00290\n",
      "///\n",
      "ENTRY       ath00300                    Pathway\n",
      "NAME        Lysine biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00300  Lysine biosynthesis\n",
      "MODULE      ath_M00527  Lysine biosynthesis, DAP aminotransferase pathway, aspartate => lysine [PATH:ath00300]\n",
      "DBLINKS     GO: 0009085\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G13280  AK-LYS1; aspartate kinase 1 [KO:K00928] [EC:2.7.2.4]\n",
      "AT5G14060  CARAB-AK-LYS; Aspartate kinase family protein [KO:K00928] [EC:2.7.2.4]\n",
      "AT3G02020  AK3; aspartate kinase 3 [KO:K00928] [EC:2.7.2.4]\n",
      "AT1G31230  AK-HSDH_I; aspartate kinase-homoserine dehydrogenase i [KO:K12524] [EC:2.7.2.4 1.1.1.3]\n",
      "AT4G19710  AK-HSDH_II; aspartate kinase-homoserine dehydrogenase ii [KO:K12524] [EC:2.7.2.4 1.1.1.3]\n",
      "AT1G14810  semialdehyde dehydrogenase family protein [KO:K00133] [EC:1.2.1.11]\n",
      "AT2G45440  DHDPS2; dihydrodipicolinate synthase [KO:K01714] [EC:4.3.3.7]\n",
      "AT3G60880  DHDPS1; dihydrodipicolinate synthase 1 [KO:K01714] [EC:4.3.3.7]\n",
      "AT3G59890  Dihydrodipicolinate reductase, bacterial/plant [KO:K00215] [EC:1.17.1.8]\n",
      "AT2G44040  Dihydrodipicolinate reductase, bacterial/plant [KO:K00215] [EC:1.17.1.8]\n",
      "AT3G53580  diaminopimelate epimerase family protein [KO:K01778] [EC:5.1.1.7]\n",
      "AT4G33680  AGD2; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K10206] [EC:2.6.1.83]\n",
      "AT2G13810  ALD1; AGD2-like defense response protein 1 [KO:K10206] [EC:2.6.1.83]\n",
      "AT3G14390  Pyridoxal-dependent decarboxylase family protein [KO:K01586] [EC:4.1.1.20]\n",
      "AT5G11880  Pyridoxal-dependent decarboxylase family protein [KO:K01586] [EC:4.1.1.20]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00026  2-Oxoglutarate\n",
      "C00047  L-Lysine\n",
      "C00049  L-Aspartate\n",
      "C00263  L-Homoserine\n",
      "C00322  2-Oxoadipate\n",
      "C00441  L-Aspartate 4-semialdehyde\n",
      "C00449  N6-(L-1,3-Dicarboxypropyl)-L-lysine\n",
      "C00666  LL-2,6-Diaminoheptanedioate\n",
      "C00680  meso-2,6-Diaminoheptanedioate\n",
      "C00956  L-2-Aminoadipate\n",
      "C01251  (R)-2-Hydroxybutane-1,2,4-tricarboxylate\n",
      "C03082  4-Phospho-L-aspartate\n",
      "C03871  L-2-Amino-6-oxoheptanedioate\n",
      "C03972  2,3,4,5-Tetrahydrodipicolinate\n",
      "C04002  (Z)-But-1-ene-1,2,4-tricarboxylate\n",
      "C04076  L-2-Aminoadipate 6-semialdehyde\n",
      "C04390  N6-Acetyl-LL-2,6-diaminoheptanedioate\n",
      "C04421  N-Succinyl-LL-2,6-diaminoheptanedioate\n",
      "C04462  N-Succinyl-2-L-amino-6-oxoheptanedioate\n",
      "C04877  UDP-N-acetylmuramoyl-L-alanyl-gamma-D-glutamyl-meso-2,6-diaminopimelate\n",
      "C04882  UDP-N-acetylmuramoyl-L-alanyl-D-glutamyl-6-carboxy-L-lysyl-D-alanyl-D-alanine\n",
      "C05535  alpha-Aminoadipoyl-S-acyl enzyme\n",
      "C05539  N-Acetyl-L-2-amino-6-oxopimelate\n",
      "C05560  L-2-Aminoadipate adenylate\n",
      "C05662  Homoisocitrate\n",
      "C16138  L-Pyrrolysine\n",
      "C19886  LysW-gamma-L-alpha-aminoadipate\n",
      "C19887  LysW-gamma-L-alpha-aminoadipyl 6-phosphate\n",
      "C19888  LysW-gamma-L-alpha-aminoadipate 6-semialdehyde\n",
      "C19889  LysW-gamma-L-lysine\n",
      "C20258  (2S,4S)-4-Hydroxy-2,3,4,5-tetrahydrodipicolinate\n",
      "C20277  (2R,3R)-3-Methylornithine\n",
      "C20278  (2R,3R)-3-Methylornithinyl-N6-lysine\n",
      "C20279  (2R,3R)-3-Methylglutamyl-5-semialdehyde-N6-lysine\n",
      "REFERENCE   PMID:10074354\n",
      "AUTHORS   Ledwidge R, Blanchard JS.\n",
      "TITLE     The dual biosynthetic capability of N-acetylornithine aminotransferase in arginine and lysine biosynthesis.\n",
      "JOURNAL   Biochemistry 38:3019-3024 (1999)\n",
      "DOI:10.1021/bi982574a\n",
      "REFERENCE   PMID:10850974\n",
      "AUTHORS   Fuchs TM, Schneider B, Krumbach K, Eggeling L, Gross R.\n",
      "TITLE     Characterization of a bordetella pertussis diaminopimelate (DAP) biosynthesis locus identifies dapC, a novel gene coding for an N-succinyl-L,L-DAP aminotransferase.\n",
      "JOURNAL   J Bacteriol 182:3626-3631 (2000)\n",
      "DOI:10.1128/JB.182.13.3626-3631.2000\n",
      "REFERENCE   PMID:10714900\n",
      "AUTHORS   Zabriskie TM, Jackson MD.\n",
      "TITLE     Lysine biosynthesis and metabolism in fungi.\n",
      "JOURNAL   Nat Prod Rep 17:85-97 (2000)\n",
      "DOI:10.1039/A801345D\n",
      "REFERENCE   PMID:14627808\n",
      "AUTHORS   Rodionov DA, Vitreschak AG, Mironov AA, Gelfand MS.\n",
      "TITLE     Regulation of lysine biosynthesis and transport genes in bacteria: yet another RNA riboswitch?\n",
      "JOURNAL   Nucleic Acids Res 31:6748-57 (2003)\n",
      "DOI:10.1093/nar/gkg900\n",
      "REFERENCE   PMID:12355264\n",
      "AUTHORS   Velasco AM, Leguina JI, Lazcano A.\n",
      "TITLE     Molecular evolution of the lysine biosynthetic pathways.\n",
      "JOURNAL   J Mol Evol 55:445-59 (2002)\n",
      "DOI:10.1007/s00239-002-2340-2\n",
      "REFERENCE   PMID:15256574\n",
      "AUTHORS   Miyazaki T, Miyazaki J, Yamane H, Nishiyama M.\n",
      "TITLE     alpha-Aminoadipate aminotransferase from an extremely thermophilic bacterium, Thermus thermophilus.\n",
      "JOURNAL   Microbiology 150:2327-34 (2004)\n",
      "DOI:10.1099/mic.0.27037-0\n",
      "REFERENCE   PMID:12126930\n",
      "AUTHORS   Goh DL, Patel A, Thomas GH, Salomons GS, Schor DS, Jakobs C, Geraghty MT.\n",
      "TITLE     Characterization of the human gene encoding alpha-aminoadipate aminotransferase (AADAT).\n",
      "JOURNAL   Mol Genet Metab 76:172-80 (2002)\n",
      "DOI:10.1016/S1096-7192(02)00037-9\n",
      "REFERENCE   PMID:15160885\n",
      "AUTHORS   Nishiyama M.\n",
      "TITLE     [Novel lysine biosynthesis: a key to elucidate evolution of metabolic and biosynthetic pathways]\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 49:758-63 (2004)\n",
      "REFERENCE   PMID:10613839\n",
      "AUTHORS   Nishida H, Nishiyama M, Kobashi N, Kosuge T, Hoshino T, Yamane H.\n",
      "TITLE     A prokaryotic gene cluster involved in synthesis of lysine through the amino adipate pathway: a key to the evolution of amino acid biosynthesis.\n",
      "JOURNAL   Genome Res 9:1175-83 (1999)\n",
      "DOI:10.1101/gr.9.12.1175\n",
      "REFERENCE   PMID:12963379\n",
      "AUTHORS   Sakai H, Vassylyeva MN, Matsuura T, Sekine S, Gotoh K, Nishiyama M, Terada T, Shirouzu M, Kuramitsu S, Vassylyev DG, Yokoyama S.\n",
      "TITLE     Crystal structure of a lysine biosynthesis enzyme, LysX, from Thermus thermophilus HB8.\n",
      "JOURNAL   J Mol Biol 332:729-40 (2003)\n",
      "DOI:10.1016/S0022-2836(03)00946-X\n",
      "REFERENCE   PMID:16361515\n",
      "AUTHORS   Hudson AO, Singh BK, Leustek T, Gilvarg C.\n",
      "TITLE     An LL-diaminopimelate aminotransferase defines a novel variant of the lysine biosynthesis pathway in plants.\n",
      "JOURNAL   Plant Physiol 140:292-301 (2006)\n",
      "DOI:10.1104/pp.105.072629\n",
      "REFERENCE   PMID:19620981\n",
      "AUTHORS   Horie A, Tomita T, Saiki A, Kono H, Taka H, Mineki R, Fujimura T, Nishiyama C, Kuzuyama T, Nishiyama M\n",
      "TITLE     Discovery of proteinaceous N-modification in lysine biosynthesis of Thermus thermophilus.\n",
      "JOURNAL   Nat Chem Biol 5:673-9 (2009)\n",
      "DOI:10.1038/nchembio.198\n",
      "REFERENCE   PMID:21547300\n",
      "AUTHORS   Dairi T, Kuzuyama T, Nishiyama M, Fujii I\n",
      "TITLE     Convergent strategies in biosynthesis.\n",
      "JOURNAL   Nat Prod Rep 28:1054-86 (2011)\n",
      "DOI:10.1039/c0np00047g\n",
      "REFERENCE   PMID:21550296\n",
      "AUTHORS   Gaston MA, Jiang R, Krzycki JA\n",
      "TITLE     Functional context, biosynthesis, and genetic encoding of pyrrolysine.\n",
      "JOURNAL   Curr Opin Microbiol 14:342-9 (2011)\n",
      "DOI:10.1016/j.mib.2011.04.001\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00310  Lysine degradation\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "KO_PATHWAY  ko00300\n",
      "///\n",
      "ENTRY       ath00310                    Pathway\n",
      "NAME        Lysine degradation - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00310  Lysine degradation\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G33150  lysine-ketoglutarate reductase/saccharopine dehydrogenase bifunctional enzyme [KO:K14157] [EC:1.5.1.8 1.5.1.9]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT5G55070  Dihydrolipoamide succinyltransferase [KO:K00658] [EC:2.3.1.61]\n",
      "AT4G26910  Dihydrolipoamide succinyltransferase [KO:K00658] [EC:2.3.1.61]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "AT5G48230  ACAT2; acetoacetyl-CoA thiolase 2 [KO:K00626] [EC:2.3.1.9]\n",
      "AT5G47720  AACT1; Thiolase family protein [KO:K00626] [EC:2.3.1.9]\n",
      "AT2G24580  FAD-dependent oxidoreductase family protein [KO:K00306] [EC:1.5.3.1 1.5.3.7]\n",
      "AT4G35987  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K18826] [EC:2.1.1.60]\n",
      "AT2G17900  SDG37; SET domain group 37 [KO:K11426] [EC:2.1.1.354 2.1.1.357]\n",
      "AT4G15180  SDG2; SET domain protein 2 [KO:K22748] [EC:2.1.1.354]\n",
      "AT2G33290  SUVH2; SU(VAR)3-9 homolog 2 [KO:K11420] [EC:2.1.1.355]\n",
      "AT4G13460  SUVH9; SU(VAR)3-9 homolog 9 [KO:K11420] [EC:2.1.1.355]\n",
      "AT2G35160  SUVH5; SU(VAR)3-9 homolog 5 [KO:K11420] [EC:2.1.1.355]\n",
      "AT5G13960  SUVH4; histone-lysine N-methyltransferase, H3 lysine-9 specific SUVH4-like protein [KO:K11420] [EC:2.1.1.355]\n",
      "AT2G22740  SUVH6; SU(VAR)3-9 homolog 6 [KO:K11420] [EC:2.1.1.355]\n",
      "AT4G02020  SWN; SET domain-containing protein [KO:K11430] [EC:2.1.1.356]\n",
      "AT2G23380  CLF; SET domain-containing protein [KO:K11430] [EC:2.1.1.356]\n",
      "AT3G03750  SDG20; SET domain protein 20 [KO:K11433] [EC:2.1.1.357]\n",
      "AT5G04940  SUVH1; SU(VAR)3-9 homolog 1 [KO:K24405] [EC:2.1.1.368]\n",
      "AT5G24330  ATXR6; TRITHORAX-RELATED PROTEIN 6 [KO:K24406] [EC:2.1.1.369]\n",
      "AT5G09790  ATXR5; TRITHORAX-RELATED PROTEIN 5 [KO:K24406] [EC:2.1.1.369]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00026  2-Oxoglutarate\n",
      "C00037  Glycine\n",
      "C00042  Succinate\n",
      "C00047  L-Lysine\n",
      "C00136  Butanoyl-CoA\n",
      "C00164  Acetoacetate\n",
      "C00322  2-Oxoadipate\n",
      "C00332  Acetoacetyl-CoA\n",
      "C00408  L-Pipecolate\n",
      "C00431  5-Aminopentanoate\n",
      "C00449  N6-(L-1,3-Dicarboxypropyl)-L-lysine\n",
      "C00450  (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate\n",
      "C00487  Carnitine\n",
      "C00489  Glutarate\n",
      "C00527  Glutaryl-CoA\n",
      "C00739  D-Lysine\n",
      "C00877  Crotonoyl-CoA\n",
      "C00956  L-2-Aminoadipate\n",
      "C00990  5-Aminopentanamide\n",
      "C01142  (3S)-3,6-Diaminohexanoate\n",
      "C01144  (S)-3-Hydroxybutanoyl-CoA\n",
      "C01149  4-Trimethylammoniobutanal\n",
      "C01181  4-Trimethylammoniobutanoate\n",
      "C01186  (3S,5S)-3,5-Diaminohexanoate\n",
      "C01211  Procollagen 5-hydroxy-L-lysine\n",
      "C01259  (3S)-3-Hydroxy-N6,N6,N6-trimethyl-L-lysine\n",
      "C01672  Cadaverine\n",
      "C02188  Protein lysine\n",
      "C02727  N6-Acetyl-L-lysine\n",
      "C03087  5-Acetamidopentanoate\n",
      "C03196  (S)-2-Hydroxyglutarate\n",
      "C03239  6-Amino-2-oxohexanoate\n",
      "C03273  5-Oxopentanoate\n",
      "C03366  5-Phosphooxy-L-lysine\n",
      "C03656  (S)-5-Amino-3-oxohexanoic acid\n",
      "C03793  N6,N6,N6-Trimethyl-L-lysine\n",
      "C04020  D-Lysopine\n",
      "C04076  L-2-Aminoadipate 6-semialdehyde\n",
      "C04092  Delta1-Piperideine-2-carboxylate\n",
      "C04487  5-(D-Galactosyloxy)-L-lysine-procollagen\n",
      "C05161  (2R,5S)-2,5-Diaminohexanoate\n",
      "C05231  L-3-Aminobutyryl-CoA\n",
      "C05544  Protein N6-methyl-L-lysine\n",
      "C05545  Protein N6,N6-dimethyl-L-lysine\n",
      "C05546  Protein N6,N6,N6-trimethyl-L-lysine\n",
      "C05548  6-Acetamido-2-oxohexanoate\n",
      "C05825  2-Amino-5-oxohexanoate\n",
      "C12455  5-Aminopentanal\n",
      "C16741  L-Hydroxylysine\n",
      "REFERENCE   PMID:11802770\n",
      "AUTHORS   Vaz FM, Wanders RJ.\n",
      "TITLE     Carnitine biosynthesis in mammals.\n",
      "JOURNAL   Biochem J 361:417-29 (2002)\n",
      "DOI:10.1042/bj3610417\n",
      "REFERENCE   PMID:1682209\n",
      "AUTHORS   Large PJ, Robertson A.\n",
      "TITLE     The route of lysine breakdown in Candida tropicalis.\n",
      "JOURNAL   FEMS Microbiol Lett 66:209-13 (1991)\n",
      "DOI:10.1111/j.1574-6968.1990.tb03998.x\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00780  Biotin metabolism\n",
      "KO_PATHWAY  ko00310\n",
      "///\n",
      "ENTRY       ath00330                    Pathway\n",
      "NAME        Arginine and proline metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00330  Arginine and proline metabolism\n",
      "MODULE      ath_M00015  Proline biosynthesis, glutamate => proline [PATH:ath00330]\n",
      "DBLINKS     GO: 0006525 0006560\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G16500  ADC1; arginine decarboxylase 1 [KO:K01583] [EC:4.1.1.19]\n",
      "AT4G34710  ADC2; arginine decarboxylase 2 [KO:K01583] [EC:4.1.1.19]\n",
      "AT5G08170  EMB1873; porphyromonas-type peptidyl-arginine deiminase family protein [KO:K10536] [EC:3.5.3.12]\n",
      "AT2G27450  NLP1; nitrilase-like protein 1 [KO:K12251] [EC:3.5.1.53]\n",
      "AT5G61160  AACT1; anthocyanin 5-aromatic acyltransferase 1 [KO:K14329] [EC:2.3.1.64 2.3.1.-]\n",
      "AT1G23820  SPDS1; spermidine synthase [KO:K00797] [EC:2.5.1.16]\n",
      "AT1G70310  SPDS2; spermidine synthase 2 [KO:K00797] [EC:2.5.1.16]\n",
      "AT5G53120  SPDS3; spermidine synthase 3 [KO:K00797] [EC:2.5.1.16]\n",
      "AT3G25570  Adenosylmethionine decarboxylase family protein [KO:K01611] [EC:4.1.1.50]\n",
      "AT5G18930  BUD2; Adenosylmethionine decarboxylase family protein [KO:K01611] [EC:4.1.1.50]\n",
      "AT5G15950  Adenosylmethionine decarboxylase family protein [KO:K01611] [EC:4.1.1.50]\n",
      "AT3G02470  SAMDC; S-adenosylmethionine decarboxylase [KO:K01611] [EC:4.1.1.50]\n",
      "AT4G29720  PAO5; polyamine oxidase 5 [KO:K12259] [EC:1.5.3.16 1.5.3.-]\n",
      "AT5G13700  PAO1; polyamine oxidase 1 [KO:K13366] [EC:1.5.3.14 1.5.3.16 1.5.3.-]\n",
      "AT3G59050  PAO3; polyamine oxidase 3 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT1G65840  PAO4; polyamine oxidase 4 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT2G43020  PAO2; polyamine oxidase 2 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT4G34880  Amidase family protein [KO:K01426] [EC:3.5.1.4]\n",
      "AT1G08980  AMI1; amidase 1 [KO:K01426] [EC:3.5.1.4]\n",
      "AT3G47450  NOA1; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K13427] [EC:1.14.13.39]\n",
      "AT4G08900  ARGAH1; arginase [KO:K01476] [EC:3.5.3.1]\n",
      "AT4G08870  ARGAH2; Arginase/deacetylase superfamily protein [KO:K01476] [EC:3.5.3.1]\n",
      "AT5G46180  DELTA-OAT; ornithine-delta-aminotransferase [KO:K00819] [EC:2.6.1.13]\n",
      "AT5G14800  P5CR; pyrroline-5- carboxylate (P5C) reductase [KO:K00286] [EC:1.5.1.2]\n",
      "AT3G30775  ERD5; Methylenetetrahydrofolate reductase family protein [KO:K00318] [EC:1.5.5.2]\n",
      "AT5G38710  Methylenetetrahydrofolate reductase family protein [KO:K00318] [EC:1.5.5.2]\n",
      "AT5G62530  ALDH12A1; aldehyde dehydrogenase 12A1 [KO:K00294] [EC:1.2.1.88]\n",
      "AT2G39800  P5CS1; delta1-pyrroline-5-carboxylate synthase 1 [KO:K12657] [EC:2.7.2.11 1.2.1.41]\n",
      "AT3G55610  P5CS2; delta 1-pyrroline-5-carboxylate synthase 2 [KO:K12657] [EC:2.7.2.11 1.2.1.41]\n",
      "AT2G14260  PIP; proline iminopeptidase [KO:K01259] [EC:3.4.11.5]\n",
      "AT3G28480  Oxoglutarate/iron-dependent oxygenase [KO:K00472] [EC:1.14.11.2]\n",
      "AT4G35810  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K00472] [EC:1.14.11.2]\n",
      "AT1G20270  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K00472] [EC:1.14.11.2]\n",
      "AT2G17720  P4H5; 2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K00472] [EC:1.14.11.2]\n",
      "AT5G66060  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K00472] [EC:1.14.11.2]\n",
      "AT3G28490  Oxoglutarate/iron-dependent oxygenase [KO:K00472] [EC:1.14.11.2]\n",
      "AT5G18900  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K00472] [EC:1.14.11.2]\n",
      "AT3G06300  P4H2; P4H isoform 2 [KO:K00472] [EC:1.14.11.2]\n",
      "AT2G43080  AT-P4H-1; P4H isoform 1 [KO:K00472] [EC:1.14.11.2]\n",
      "AT2G23096  P4H13; 2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K00472] [EC:1.14.11.2]\n",
      "AT4G33910  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K00472] [EC:1.14.11.2]\n",
      "AT4G35820  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K00472] [EC:1.14.11.2]\n",
      "AT4G25600  Oxoglutarate/iron-dependent oxygenase [KO:K00472] [EC:1.14.11.2]\n",
      "AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "COMPOUND    C00012  Peptide\n",
      "C00019  S-Adenosyl-L-methionine\n",
      "C00022  Pyruvate\n",
      "C00025  L-Glutamate\n",
      "C00048  Glyoxylate\n",
      "C00062  L-Arginine\n",
      "C00077  L-Ornithine\n",
      "C00086  Urea\n",
      "C00134  Putrescine\n",
      "C00148  L-Proline\n",
      "C00179  Agmatine\n",
      "C00213  Sarcosine\n",
      "C00223  p-Coumaroyl-CoA\n",
      "C00300  Creatine\n",
      "C00315  Spermidine\n",
      "C00334  4-Aminobutanoate\n",
      "C00406  Feruloyl-CoA\n",
      "C00431  5-Aminopentanoate\n",
      "C00436  N-Carbamoylputrescine\n",
      "C00441  L-Aspartate 4-semialdehyde\n",
      "C00533  Nitric oxide\n",
      "C00555  4-Aminobutyraldehyde\n",
      "C00581  Guanidinoacetate\n",
      "C00750  Spermine\n",
      "C00763  D-Proline\n",
      "C00791  Creatinine\n",
      "C00884  Homocarnosine\n",
      "C00986  1,3-Diaminopropane\n",
      "C01035  4-Guanidinobutanoate\n",
      "C01043  N-Carbamoylsarcosine\n",
      "C01110  5-Amino-2-oxopentanoic acid\n",
      "C01137  S-Adenosylmethioninamine\n",
      "C01157  Hydroxyproline\n",
      "C01165  L-Glutamate 5-semialdehyde\n",
      "C01250  N-Acetyl-L-glutamate 5-semialdehyde\n",
      "C01682  Nopaline\n",
      "C01877  4-Oxoproline\n",
      "C02305  Phosphocreatine\n",
      "C02565  N-Methylhydantoin\n",
      "C02647  4-Guanidinobutanal\n",
      "C02714  N-Acetylputrescine\n",
      "C02946  4-Acetamidobutanoate\n",
      "C03078  4-Guanidinobutanamide\n",
      "C03166  Phosphoguanidinoacetate\n",
      "C03287  L-Glutamyl 5-phosphate\n",
      "C03296  N2-Succinyl-L-arginine\n",
      "C03375  Norspermidine\n",
      "C03415  N2-Succinyl-L-ornithine\n",
      "C03564  1-Pyrroline-2-carboxylate\n",
      "C03771  5-Guanidino-2-oxopentanoate\n",
      "C03912  (S)-1-Pyrroline-5-carboxylate\n",
      "C04137  Octopine\n",
      "C04281  L-1-Pyrroline-3-hydroxy-5-carboxylate\n",
      "C04498  p-Coumaroylagmatine\n",
      "C05147  trans-3-Hydroxy-L-proline\n",
      "C05931  N-Succinyl-L-glutamate\n",
      "C05932  N-Succinyl-L-glutamate 5-semialdehyde\n",
      "C05933  N(omega)-Hydroxyarginine\n",
      "C05936  N4-Acetylaminobutanal\n",
      "C05938  L-4-Hydroxyglutamate semialdehyde\n",
      "C05945  L-Arginine phosphate\n",
      "C05946  (4R)-4-Hydroxy-2-oxoglutarate\n",
      "C05947  L-erythro-4-Hydroxyglutamate\n",
      "C10497  Feruloylputrescine\n",
      "C15699  gamma-L-Glutamylputrescine\n",
      "C15700  gamma-Glutamyl-gamma-aminobutyraldehyde\n",
      "C15767  4-(L-gamma-Glutamylamino)butanoate\n",
      "C18172  Carboxyspermidine\n",
      "C18174  Carboxynorspermidine\n",
      "C18325  Feruloylagmatine\n",
      "C18326  p-Coumaroylputrescine\n",
      "C19706  cis-3-Hydroxy-L-proline\n",
      "REFERENCE   PMID:5288244\n",
      "AUTHORS   Haake P, Allen GW.\n",
      "TITLE     Studies on phosphorylation by phosphoroguanidinates. The mechanism of action of creatine: ATP transphosphorylase (creatine kinase).\n",
      "JOURNAL   Proc Natl Acad Sci U S A 68:2691-3 (1971)\n",
      "DOI:10.1073/pnas.68.11.2691\n",
      "REFERENCE   PMID:16499623\n",
      "AUTHORS   Kurihara S, Oda S, Kumagai H, Suzuki H.\n",
      "TITLE     Gamma-glutamyl-gamma-aminobutyrate hydrolase in the putrescine utilization pathway of Escherichia coli K-12.\n",
      "JOURNAL   FEMS Microbiol Lett 256:318-23 (2006)\n",
      "DOI:10.1111/j.1574-6968.2006.00137.x\n",
      "REFERENCE   PMID:15590624\n",
      "AUTHORS   Kurihara S, Oda S, Kato K, Kim HG, Koyanagi T, Kumagai H, Suzuki H.\n",
      "TITLE     A novel putrescine utilization pathway involves gamma-glutamylated intermediates of Escherichia coli K-12.\n",
      "JOURNAL   J Biol Chem 280:4602-8 (2005)\n",
      "DOI:10.1074/jbc.M411114200\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00310  Lysine degradation\n",
      "ath00410  beta-Alanine metabolism\n",
      "ath00470  D-Amino acid metabolism\n",
      "ath00480  Glutathione metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00650  Butanoate metabolism\n",
      "ath00770  Pantothenate and CoA biosynthesis\n",
      "ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "KO_PATHWAY  ko00330\n",
      "///\n",
      "ENTRY       ath00340                    Pathway\n",
      "NAME        Histidine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00340  Histidine metabolism\n",
      "MODULE      ath_M00026  Histidine biosynthesis, PRPP => histidine [PATH:ath00340]\n",
      "DBLINKS     GO: 0006547\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G09795  ATP-PRT2; ATP phosphoribosyl transferase 2 [KO:K00765] [EC:2.4.2.17]\n",
      "AT1G58080  ATP-PRT1; ATP phosphoribosyl transferase 1 [KO:K00765] [EC:2.4.2.17]\n",
      "AT1G31860  AT-IE; histidine biosynthesis bifunctional protein (HISIE) [KO:K11755] [EC:3.5.4.19 3.6.1.31]\n",
      "AT2G36230  APG10; Aldolase-type TIM barrel family protein [KO:K01814] [EC:5.3.1.16]\n",
      "AT4G26900  AT-HF; HIS HF [KO:K01663] [EC:4.3.2.10]\n",
      "AT4G14910  HISN5B; HISTIDINE BIOSYNTHESIS 5B [KO:K01693] [EC:4.2.1.19]\n",
      "AT3G22425  IGPD; imidazoleglycerol-phosphate dehydratase [KO:K01693] [EC:4.2.1.19]\n",
      "AT1G71920  HISN6B; HISTIDINE BIOSYNTHESIS 6B [KO:K00817] [EC:2.6.1.9]\n",
      "AT5G10330  HPA1; histidinol phosphate aminotransferase 1 [KO:K00817] [EC:2.6.1.9]\n",
      "AT4G39120  IMPL2; inositol monophosphatase family protein [KO:K18649] [EC:3.1.3.25 3.1.3.93 3.1.3.15]\n",
      "AT5G63890  HDH; histidinol dehydrogenase [KO:K00013] [EC:1.1.1.23]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT2G32160  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K19787] [EC:2.1.1.22]\n",
      "AT2G32170  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K19787] [EC:2.1.1.22]\n",
      "COMPOUND    C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00049  L-Aspartate\n",
      "C00119  5-Phospho-alpha-D-ribose 1-diphosphate\n",
      "C00135  L-Histidine\n",
      "C00386  Carnosine\n",
      "C00388  Histamine\n",
      "C00439  N-Formimino-L-glutamate\n",
      "C00785  Urocanate\n",
      "C00860  L-Histidinol\n",
      "C01044  N-Formyl-L-aspartate\n",
      "C01045  N-Formyl-L-glutamate\n",
      "C01100  L-Histidinol phosphate\n",
      "C01152  N(pi)-Methyl-L-histidine\n",
      "C01262  beta-Alanyl-N(pi)-methyl-L-histidine\n",
      "C01267  3-(Imidazol-4-yl)-2-oxopropyl phosphate\n",
      "C01929  L-Histidinal\n",
      "C02739  1-(5-Phospho-D-ribosyl)-ATP\n",
      "C02741  Phosphoribosyl-AMP\n",
      "C02835  Imidazole-4-acetate\n",
      "C03277  Imidazol-5-yl-pyruvate\n",
      "C03409  N-Formimino-L-aspartate\n",
      "C03680  4-Imidazolone-5-propanoate\n",
      "C04437  1-(5-Phosphoribosyl)imidazole-4-acetate\n",
      "C04666  D-erythro-1-(Imidazol-4-yl)glycerol 3-phosphate\n",
      "C04677  1-(5'-Phosphoribosyl)-5-amino-4-imidazolecarboxamide\n",
      "C04896  5-(5-Phospho-D-ribosylaminoformimino)-1-(5-phosphoribosyl)-imidazole-4-carboxamide\n",
      "C04916  N-(5'-Phospho-D-1'-ribulosylformimino)-5-amino-1-(5''-phospho-D-ribosyl)-4-imidazolecarboxamide\n",
      "C05127  N-Methylhistamine\n",
      "C05130  Imidazole-4-acetaldehyde\n",
      "C05131  (1-Ribosylimidazole)-4-acetate\n",
      "C05133  4-Imidazolone-5-acetate\n",
      "C05135  4-(beta-Acetylaminoethyl)imidazole\n",
      "C05565  Hydantoin-5-propionate\n",
      "C05568  Imidazole lactate\n",
      "C05570  Ergothioneine\n",
      "C05571  Thiourocanic acid\n",
      "C05572  4-Oxoglutaramate\n",
      "C05575  Hercynine\n",
      "C05827  Methylimidazole acetaldehyde\n",
      "C05828  Methylimidazoleacetic acid\n",
      "C05829  N-Carbamyl-L-glutamate\n",
      "C16673  Isoglutamine\n",
      "C16674  Formylisoglutamine\n",
      "C20522  Dihydrourocanate\n",
      "C20994  S-(Hercyn-2-yl)-L-cysteine S-oxide\n",
      "C20995  gamma-L-Glutamyl-S-(hercyn-2-yl)-L-cysteine S-oxide\n",
      "REL_PATHWAY ath00030  Pentose phosphate pathway\n",
      "ath00230  Purine metabolism\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "KO_PATHWAY  ko00340\n",
      "///\n",
      "ENTRY       ath00350                    Pathway\n",
      "NAME        Tyrosine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00350  Tyrosine metabolism\n",
      "MODULE      ath_M00044  Tyrosine degradation, tyrosine => homogentisate [PATH:ath00350]\n",
      "DBLINKS     GO: 0006570\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "AT2G22250  AAT; aspartate aminotransferase [KO:K15849] [EC:2.6.1.1 2.6.1.78 2.6.1.79]\n",
      "AT5G53970  TAT7; Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT5G36160  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT2G24850  TAT3; tyrosine aminotransferase 3 [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28410  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28420  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G23590  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT1G71920  HISN6B; HISTIDINE BIOSYNTHESIS 6B [KO:K00817] [EC:2.6.1.9]\n",
      "AT5G10330  HPA1; histidinol phosphate aminotransferase 1 [KO:K00817] [EC:2.6.1.9]\n",
      "AT1G06570  PDS1; 4-hydroxyphenylpyruvate dioxygenase [KO:K00457] [EC:1.13.11.27]\n",
      "AT5G54080  HGO; homogentisate 1,2-dioxygenase [KO:K00451] [EC:1.13.11.5]\n",
      "AT2G02380  GSTZ2; glutathione S-transferase (class zeta) 2 [KO:K01800] [EC:5.2.1.2]\n",
      "AT2G02390  GSTZ1; glutathione S-transferase zeta 1 [KO:K01800] [EC:5.2.1.2]\n",
      "AT1G12050  fumarylacetoacetase [KO:K01555] [EC:3.7.1.2]\n",
      "AT4G28680  TYRDC; L-tyrosine decarboxylase [KO:K01592] [EC:4.1.1.25]\n",
      "AT2G20340  AAS; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01593] [EC:4.1.1.28 4.1.1.105]\n",
      "AT2G42490  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31710  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31690  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G14940  AO1; amine oxidase 1 [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G62810  CuAO1; Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT3G43670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G12290  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G22440  Zinc-binding alcohol dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT5G42250  Zinc-binding alcohol dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT5G43940  HOT5; GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT4G22110  GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT1G22430  GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT1G77120  ADH1; alcohol dehydrogenase 1 [KO:K18857] [EC:1.1.1.1]\n",
      "AT1G32780  GroES-like zinc-binding dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT5G24760  GroES-like zinc-binding dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT1G64710  GroES-like zinc-binding alcohol dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT4G15940  Fumarylacetoacetate (FAA) hydrolase family [KO:K01557] [EC:3.7.1.5]\n",
      "AT3G16700  Fumarylacetoacetate (FAA) hydrolase family [KO:K01557] [EC:3.7.1.5]\n",
      "AT5G57170  Tautomerase/MIF superfamily protein [KO:K07253] [EC:5.3.2.1]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00042  Succinate\n",
      "C00082  L-Tyrosine\n",
      "C00122  Fumarate\n",
      "C00146  Phenol\n",
      "C00164  Acetoacetate\n",
      "C00232  Succinate semialdehyde\n",
      "C00355  3,4-Dihydroxy-L-phenylalanine\n",
      "C00483  Tyramine\n",
      "C00530  Hydroquinone\n",
      "C00544  Homogentisate\n",
      "C00547  L-Noradrenaline\n",
      "C00628  2,5-Dihydroxybenzoate\n",
      "C00642  4-Hydroxyphenylacetate\n",
      "C00788  L-Adrenaline\n",
      "C00811  4-Coumarate\n",
      "C00822  Dopaquinone\n",
      "C01036  4-Maleylacetoacetate\n",
      "C01060  3,5-Diiodo-L-tyrosine\n",
      "C01061  4-Fumarylacetoacetate\n",
      "C01161  3,4-Dihydroxyphenylacetate\n",
      "C01179  3-(4-Hydroxyphenyl)pyruvate\n",
      "C01384  Maleic acid\n",
      "C01693  L-Dopachrome\n",
      "C01829  Thyroxine\n",
      "C01850  Rosmarinate\n",
      "C02167  Maleylpyruvate\n",
      "C02442  N-Methyltyramine\n",
      "C02465  Triiodothyronine\n",
      "C02514  3-Fumarylpyruvate\n",
      "C02515  3-Iodo-L-tyrosine\n",
      "C03063  2-Oxohept-3-enedioate\n",
      "C03077  4-Chlorophenylacetate\n",
      "C03758  Dopamine\n",
      "C03765  4-Hydroxyphenylacetaldehyde\n",
      "C03964  (R)-3-(4-Hydroxyphenyl)lactate\n",
      "C04043  3,4-Dihydroxyphenylacetaldehyde\n",
      "C04045  3-(3,4-Dihydroxyphenyl)pyruvate\n",
      "C04052  5-Carboxy-2-oxohept-3-enedioate\n",
      "C04185  5,6-Dihydroxyindole-2-carboxylate\n",
      "C04186  5-Carboxymethyl-2-hydroxymuconate\n",
      "C04368  3-Amino-3-(4-hydroxyphenyl)propanoate\n",
      "C04642  2-Hydroxy-5-carboxymethylmuconate semialdehyde\n",
      "C04796  4-(L-Alanin-3-yl)-2-hydroxy-cis,cis-muconate 6-semialdehyde\n",
      "C04797  5-(L-Alanin-3-yl)-2-hydroxy-cis,cis-muconate 6-semialdehyde\n",
      "C05338  4-Hydroxyphenylacetyl-CoA\n",
      "C05350  2-Hydroxy-3-(4-hydroxyphenyl)propenoate\n",
      "C05576  3,4-Dihydroxyphenylethyleneglycol\n",
      "C05577  3,4-Dihydroxymandelaldehyde\n",
      "C05578  5,6-Dihydroxyindole\n",
      "C05579  Indole-5,6-quinone\n",
      "C05580  3,4-Dihydroxymandelate\n",
      "C05581  3-Methoxy-4-hydroxyphenylacetaldehyde\n",
      "C05582  Homovanillate\n",
      "C05583  3-Methoxy-4-hydroxyphenylglycolaldehyde\n",
      "C05584  3-Methoxy-4-hydroxymandelate\n",
      "C05585  Gentisate aldehyde\n",
      "C05587  3-Methoxytyramine\n",
      "C05588  L-Metanephrine\n",
      "C05589  L-Normetanephrine\n",
      "C05593  3-Hydroxyphenylacetate\n",
      "C05594  3-Methoxy-4-hydroxyphenylethyleneglycol\n",
      "C05595  4-Hydroxyphenylacetylglutamic acid\n",
      "C05596  p-Hydroxyphenylacetylglycine\n",
      "C05600  2-Hydroxyhepta-2,4-dienedioate\n",
      "C05604  2-Carboxy-2,3-dihydro-5,6-dihydroxyindole\n",
      "C06044  4-Hydroxyphenylethanol\n",
      "C06046  Salidroside\n",
      "C06047  Stizolobate\n",
      "C06048  Stizolobinate\n",
      "C06199  Hordenine\n",
      "C06201  2,4-Dihydroxyhept-2-enedioate\n",
      "C10447  3,4-Dihydroxyphenylpropanoate\n",
      "C17935  Cysteinyldopa\n",
      "C17936  Phaeomelanin\n",
      "C17937  Eumelanin\n",
      "C17938  5,6-Indolequinone-2-carboxylic acid\n",
      "C22038  (R)-3-(3,4-Dihydroxyphenyl)lactate\n",
      "REFERENCE   PMID:8550403\n",
      "AUTHORS   Prieto MA, Diaz E, Garcia JL.\n",
      "TITLE     Molecular characterization of the 4-hydroxyphenylacetate catabolic pathway of Escherichia coli W: engineering a mobile aromatic degradative cluster.\n",
      "JOURNAL   J Bacteriol 178:111-20 (1996)\n",
      "DOI:10.1128/JB.178.1.111-120.1996\n",
      "REFERENCE   PMID:1194238\n",
      "AUTHORS   Sparnins VL, Dagley S.\n",
      "TITLE     Alternative routes of aromatic catabolism in Pseudomonas acidovorans and Pseudomonas putida: gallic acid as a substrate and inhibitor of dioxygenases.\n",
      "JOURNAL   J Bacteriol 124:1374-81 (1975)\n",
      "DOI:10.1128/JB.124.3.1374-1381.1975\n",
      "REFERENCE   PMID:18435614\n",
      "AUTHORS   Ito S, Wakamatsu K\n",
      "TITLE     Chemistry of mixed melanogenesis--pivotal roles of dopaquinone.\n",
      "JOURNAL   Photochem Photobiol 84:582-92 (2008)\n",
      "DOI:10.1111/j.1751-1097.2007.00238.x\n",
      "REFERENCE   PMID:11171088\n",
      "AUTHORS   Olivares C, Jimenez-Cervantes C, Lozano JA, Solano F, Garcia-Borron JC\n",
      "TITLE     The 5,6-dihydroxyindole-2-carboxylic acid (DHICA) oxidase activity of human tyrosinase.\n",
      "JOURNAL   Biochem J 354:131-9 (2001)\n",
      "DOI:10.1042/bj3540131\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00360  Phenylalanine metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00650  Butanoate metabolism\n",
      "ath00940  Phenylpropanoid biosynthesis\n",
      "ath00950  Isoquinoline alkaloid biosynthesis\n",
      "KO_PATHWAY  ko00350\n",
      "///\n",
      "ENTRY       ath00360                    Pathway\n",
      "NAME        Phenylalanine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00360  Phenylalanine metabolism\n",
      "DBLINKS     GO: 0006558\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G20340  AAS; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01593] [EC:4.1.1.28 4.1.1.105]\n",
      "AT2G42490  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31710  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31690  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G14940  AO1; amine oxidase 1 [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G62810  CuAO1; Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT3G43670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G12290  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "AT3G15290  3-hydroxyacyl-CoA dehydrogenase family protein [KO:K00074] [EC:1.1.1.157]\n",
      "AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "AT2G22250  AAT; aspartate aminotransferase [KO:K15849] [EC:2.6.1.1 2.6.1.78 2.6.1.79]\n",
      "AT5G53970  TAT7; Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT5G36160  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT2G24850  TAT3; tyrosine aminotransferase 3 [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28410  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28420  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G23590  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT1G71920  HISN6B; HISTIDINE BIOSYNTHESIS 6B [KO:K00817] [EC:2.6.1.9]\n",
      "AT5G10330  HPA1; histidinol phosphate aminotransferase 1 [KO:K00817] [EC:2.6.1.9]\n",
      "AT1G06570  PDS1; 4-hydroxyphenylpyruvate dioxygenase [KO:K00457] [EC:1.13.11.27]\n",
      "AT5G57170  Tautomerase/MIF superfamily protein [KO:K07253] [EC:5.3.2.1]\n",
      "AT4G34880  Amidase family protein [KO:K01426] [EC:3.5.1.4]\n",
      "AT1G08980  AMI1; amidase 1 [KO:K01426] [EC:3.5.1.4]\n",
      "AT3G10340  PAL4; phenylalanine ammonia-lyase 4 [KO:K10775] [EC:4.3.1.24]\n",
      "AT3G53260  PAL2; phenylalanine ammonia-lyase 2 [KO:K10775] [EC:4.3.1.24]\n",
      "AT2G37040  PAL1; PHE ammonia lyase 1 [KO:K10775] [EC:4.3.1.24]\n",
      "AT5G04230  PAL3; phenyl alanine ammonia-lyase 3 [KO:K10775] [EC:4.3.1.24]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00042  Succinate\n",
      "C00079  L-Phenylalanine\n",
      "C00082  L-Tyrosine\n",
      "C00084  Acetaldehyde\n",
      "C00091  Succinyl-CoA\n",
      "C00122  Fumarate\n",
      "C00166  Phenylpyruvate\n",
      "C00180  Benzoate\n",
      "C00423  trans-Cinnamate\n",
      "C00512  Benzoyl-CoA\n",
      "C00582  Phenylacetyl-CoA\n",
      "C00596  2-Hydroxy-2,4-pentadienoate\n",
      "C00601  Phenylacetaldehyde\n",
      "C00642  4-Hydroxyphenylacetate\n",
      "C00805  Salicylate\n",
      "C01198  3-(2-Hydroxyphenyl)propanoate\n",
      "C01575  Ephedrine\n",
      "C01586  Hippurate\n",
      "C01772  trans-2-Hydroxycinnamate\n",
      "C02137  alpha-Oxo-benzeneacetic acid\n",
      "C02232  3-Oxoadipyl-CoA\n",
      "C02265  D-Phenylalanine\n",
      "C02505  2-Phenylacetamide\n",
      "C02763  2-Hydroxy-3-phenylpropenoate\n",
      "C02765  (+)-(1S,2S)-Pseudoephedrine\n",
      "C03519  N-Acetyl-L-phenylalanine\n",
      "C03589  4-Hydroxy-2-oxopentanoate\n",
      "C04044  3-(2,3-Dihydroxyphenyl)propanoate\n",
      "C04148  Phenylacetylglutamine\n",
      "C04351  (R)-2-Methylimino-1-phenylpropan-1-ol\n",
      "C04479  2-Hydroxy-6-oxonona-2,4-diene-1,9-dioate\n",
      "C05332  Phenethylamine\n",
      "C05593  3-Hydroxyphenylacetate\n",
      "C05598  Phenylacetylglycine\n",
      "C05607  Phenyllactate\n",
      "C05629  Phenylpropanoate\n",
      "C05852  2-Hydroxyphenylacetate\n",
      "C05853  Phenylethyl alcohol\n",
      "C06207  2,6-Dihydroxyphenylacetate\n",
      "C07086  Phenylacetic acid\n",
      "C08300  D-Cathine\n",
      "C08301  D-Cathinone\n",
      "C11457  3-(3-Hydroxyphenyl)propanoic acid\n",
      "C11588  cis-3-(Carboxy-ethyl)-3,5-cyclo-hexadiene-1,2-diol\n",
      "C12621  trans-3-Hydroxycinnamate\n",
      "C12622  cis-3-(3-Carboxyethenyl)-3,5-cyclohexadiene-1,2-diol\n",
      "C12623  trans-2,3-Dihydroxycinnamate\n",
      "C12624  2-Hydroxy-6-ketononatrienedioate\n",
      "C14144  5-Carboxy-2-pentenoyl-CoA\n",
      "C14145  (3S)-3-Hydroxyadipyl-CoA\n",
      "C15524  Phenylglyoxylyl-CoA\n",
      "C16719  (-)-Norephedrine\n",
      "C17268  Pyruvophenone\n",
      "C19945  3-Oxo-5,6-dehydrosuberyl-CoA\n",
      "C19946  3-Oxo-5,6-dehydrosuberyl-CoA semialdehyde\n",
      "C19975  2-Oxepin-2(3H)-ylideneacetyl-CoA\n",
      "C20062  2-(1,2-Epoxy-1,2-dihydrophenyl)acetyl-CoA\n",
      "REFERENCE\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REFERENCE   PMID:10537203\n",
      "AUTHORS   Arai H, Yamamoto T, Ohishi T, Shimizu T, Nakata T, Kudo T.\n",
      "TITLE     Genetic organization and characteristics of the 3-(3-hydroxyphenyl)propionic acid degradation pathway of Comamonas testosteroni TA441.\n",
      "JOURNAL   Microbiology 145 ( Pt 10):2813-20 (1999)\n",
      "DOI:10.1099/00221287-145-10-2813\n",
      "REFERENCE   PMID:9603882\n",
      "AUTHORS   Diaz E, Ferrandez A, Garcia JL.\n",
      "TITLE     Characterization of the hca cluster encoding the dioxygenolytic pathway for initial catabolism of 3-phenylpropionic acid in Escherichia coli K-12.\n",
      "JOURNAL   J Bacteriol 180:2915-23 (1998)\n",
      "DOI:10.1128/JB.180.11.2915-2923.1998\n",
      "REFERENCE   PMID:9098055\n",
      "AUTHORS   Ferrandez A, Garcia JL, Diaz E.\n",
      "TITLE     Genetic characterization and expression in heterologous hosts of the 3-(3-hydroxyphenyl)propionate catabolic pathway of Escherichia coli K-12.\n",
      "JOURNAL   J Bacteriol 179:2573-81 (1997)\n",
      "DOI:10.1128/JB.179.8.2573-2581.1997\n",
      "REFERENCE   PMID:19016485\n",
      "AUTHORS   Hwang JY, Park J, Seo JH, Cha M, Cho BK, Kim J, Kim BG\n",
      "TITLE     Simultaneous synthesis of 2-phenylethanol and L-homophenylalanine using aromatic transaminase with yeast Ehrlich pathway.\n",
      "JOURNAL   Biotechnol Bioeng 102:1323-9 (2009)\n",
      "DOI:10.1002/bit.22178\n",
      "REFERENCE   PMID:20660314\n",
      "AUTHORS   Teufel R, Mascaraque V, Ismail W, Voss M, Perera J, Eisenreich W, Haehnel W, Fuchs G\n",
      "TITLE     Bacterial phenylalanine and phenylacetate catabolic pathway revealed.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 107:14390-5 (2010)\n",
      "DOI:10.1073/pnas.1005399107\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00350  Tyrosine metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "KO_PATHWAY  ko00360\n",
      "///\n",
      "ENTRY       ath00380                    Pathway\n",
      "NAME        Tryptophan metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00380  Tryptophan metabolism\n",
      "MODULE      ath_M00370  Glucosinolate biosynthesis, tryptophan => glucobrassicin [PATH:ath00380]\n",
      "DBLINKS     GO: 0006568 0009851\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G55070  Dihydrolipoamide succinyltransferase [KO:K00658] [EC:2.3.1.61]\n",
      "AT4G26910  Dihydrolipoamide succinyltransferase [KO:K00658] [EC:2.3.1.61]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "AT5G48230  ACAT2; acetoacetyl-CoA thiolase 2 [KO:K00626] [EC:2.3.1.9]\n",
      "AT5G47720  AACT1; Thiolase family protein [KO:K00626] [EC:2.3.1.9]\n",
      "AT2G20340  AAS; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01593] [EC:4.1.1.28 4.1.1.105]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT1G26220  Acyl-CoA N-acyltransferases (NAT) superfamily protein [KO:K22450] [EC:2.3.1.87]\n",
      "AT1G32070  NSI; nuclear shuttle interacting [KO:K22450] [EC:2.3.1.87]\n",
      "AT4G35150  O-methyltransferase family protein [KO:K22588] [EC:2.1.1.4]\n",
      "AT4G35160  O-methyltransferase family protein [KO:K22588] [EC:2.1.1.4]\n",
      "AT5G54160  OMT1; O-methyltransferase 1 [KO:K13066] [EC:2.1.1.68 2.1.1.4]\n",
      "AT4G39950  CYP79B2; cytochrome P450, family 79, subfamily B, polypeptide 2 [KO:K11812] [EC:1.14.14.156]\n",
      "AT2G22330  CYP79B3; cytochrome P450, family 79, subfamily B, polypeptide 3 [KO:K11813] [EC:1.14.14.156]\n",
      "AT4G13260  YUC2; Flavin-binding monooxygenase family protein [KO:K11816] [EC:1.14.13.168]\n",
      "AT5G11320  YUC4; Flavin-binding monooxygenase family protein [KO:K11816] [EC:1.14.13.168]\n",
      "AT5G43890  YUC5; Flavin-binding monooxygenase family protein [KO:K11816] [EC:1.14.13.168]\n",
      "AT5G25620  YUC6; Flavin-binding monooxygenase family protein [KO:K11816] [EC:1.14.13.168]\n",
      "AT4G32540  YUC1; Flavin-binding monooxygenase family protein [KO:K11816] [EC:1.14.13.168]\n",
      "AT1G04610  YUC3; YUCCA 3 [KO:K11816] [EC:1.14.13.168]\n",
      "AT2G33230  YUC7; YUCCA 7 [KO:K11816] [EC:1.14.13.168]\n",
      "AT4G28720  YUC8; Flavin-binding monooxygenase family protein [KO:K11816] [EC:1.14.13.168]\n",
      "AT1G04180  YUC9; YUCCA 9 [KO:K11816] [EC:1.14.13.168]\n",
      "AT1G48910  YUC10; Flavin-containing monooxygenase family protein [KO:K11816] [EC:1.14.13.168]\n",
      "AT1G21430  YUC11; Flavin-binding monooxygenase family protein [KO:K11816] [EC:1.14.13.168]\n",
      "AT1G70560  TAA1; tryptophan aminotransferase of Arabidopsis 1 [KO:K16903] [EC:2.6.1.99]\n",
      "AT4G24670  TAR2; tryptophan aminotransferase related 2 [KO:K16903] [EC:2.6.1.99]\n",
      "AT1G23320  TAR1; tryptophan aminotransferase related 1 [KO:K16903] [EC:2.6.1.99]\n",
      "AT1G80360  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00837] [EC:2.6.1.-]\n",
      "AT2G30770  CYP71A13; cytochrome P450 family 71 polypeptide [KO:K11868] [EC:4.99.1.6]\n",
      "AT2G30750  CYP71A12; cytochrome P450 family 71 polypeptide [KO:K11868] [EC:4.99.1.6]\n",
      "AT1G11610  CYP71A18; cytochrome P450, family 71, subfamily A, polypeptide 18 [KO:K11868] [EC:4.99.1.6]\n",
      "AT5G20960  AO1; aldehyde oxidase 1 [KO:K11817] [EC:1.2.3.7]\n",
      "AT3G43600  AAO2; aldehyde oxidase 2 [KO:K11817] [EC:1.2.3.7]\n",
      "AT1G04580  AO4; aldehyde oxidase 4 [KO:K22417] [EC:1.2.1.28 1.2.3.7]\n",
      "AT3G44320  NIT3; nitrilase 3 [KO:K01501] [EC:3.5.5.1]\n",
      "AT3G44300  NIT2; nitrilase 2 [KO:K01501] [EC:3.5.5.1]\n",
      "AT3G44310  NIT1; nitrilase 1 [KO:K01501] [EC:3.5.5.1]\n",
      "AT4G34880  Amidase family protein [KO:K01426] [EC:3.5.1.4]\n",
      "AT1G08980  AMI1; amidase 1 [KO:K01426] [EC:3.5.1.4]\n",
      "AT4G31500  CYP83B1; cytochrome P450, family 83, subfamily B, polypeptide 1 [KO:K11818] [EC:1.14.14.45]\n",
      "AT2G20610  SUR1; Tyrosine transaminase family protein [KO:K11819] [EC:4.4.1.-]\n",
      "AT1G24100  UGT74B1; UDP-glucosyl transferase 74B1 [KO:K11820] [EC:2.4.1.195]\n",
      "AT1G74100  SOT16; sulfotransferase 16 [KO:K11821] [EC:2.8.2.24]\n",
      "AT5G26000  TGG1; thioglucoside glucohydrolase 1 [KO:K01237] [EC:3.2.1.147]\n",
      "AT1G47600  BGLU34; beta glucosidase 34 [KO:K01237] [EC:3.2.1.147]\n",
      "AT1G51470  BGLU35; beta glucosidase 35 [KO:K01237] [EC:3.2.1.147]\n",
      "AT1G51490  BGLU36; beta glucosidase 36 [KO:K01237] [EC:3.2.1.147]\n",
      "AT5G25980  TGG2; glucoside glucohydrolase 2 [KO:K01237] [EC:3.2.1.147]\n",
      "AT5G48375  TGG3; thioglucoside glucosidase 3 [KO:K01237] [EC:3.2.1.147]\n",
      "AT1G20630  CAT1; catalase 1 [KO:K03781] [EC:1.11.1.6]\n",
      "AT4G35090  CAT2; catalase 2 [KO:K03781] [EC:1.11.1.6]\n",
      "AT1G20620  CAT3; catalase 3 [KO:K03781] [EC:1.11.1.6]\n",
      "AT1G14130  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K23947] [EC:1.14.11.-]\n",
      "AT1G14120  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K23947] [EC:1.14.11.-]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00078  L-Tryptophan\n",
      "C00108  Anthranilate\n",
      "C00322  2-Oxoadipate\n",
      "C00328  L-Kynurenine\n",
      "C00331  Indolepyruvate\n",
      "C00332  Acetoacetyl-CoA\n",
      "C00398  Tryptamine\n",
      "C00463  Indole\n",
      "C00527  Glutaryl-CoA\n",
      "C00632  3-Hydroxyanthranilate\n",
      "C00637  Indole-3-acetaldehyde\n",
      "C00643  5-Hydroxy-L-tryptophan\n",
      "C00780  Serotonin\n",
      "C00877  Crotonoyl-CoA\n",
      "C00954  Indole-3-acetate\n",
      "C00955  Indole-3-ethanol\n",
      "C00978  N-Acetylserotonin\n",
      "C01111  7,8-Dihydroxykynurenate\n",
      "C01144  (S)-3-Hydroxybutanoyl-CoA\n",
      "C01249  7,8-Dihydro-7,8-dihydroxykynurenate\n",
      "C01252  4-(2-Aminophenyl)-2,4-dioxobutanoate\n",
      "C01598  Melatonin\n",
      "C01717  4-Hydroxy-2-quinolinecarboxylic acid\n",
      "C01987  2-Aminophenol\n",
      "C02161  2-Aminophenoxazin-3-one\n",
      "C02172  N-Acetylisatin\n",
      "C02220  2-Aminomuconate\n",
      "C02298  N-Acetylindoxyl\n",
      "C02470  Xanthurenic acid\n",
      "C02693  (Indol-3-yl)acetamide\n",
      "C02700  L-Formylkynurenine\n",
      "C02775  2,3-Dihydroxyindole\n",
      "C02937  Indole-3-acetaldehyde oxime\n",
      "C02938  3-Indoleacetonitrile\n",
      "C03227  3-Hydroxy-L-kynurenine\n",
      "C03230  (Indol-3-yl)glycolaldehyde\n",
      "C03453  gamma-Oxalocrotonate\n",
      "C03574  2-Formylaminobenzaldehyde\n",
      "C03722  Quinolinate\n",
      "C03824  2-Aminomuconate semialdehyde\n",
      "C04409  2-Amino-3-carboxymuconate semialdehyde\n",
      "C05634  5-Hydroxyindoleacetaldehyde\n",
      "C05635  5-Hydroxyindoleacetate\n",
      "C05636  3-Hydroxykynurenamine\n",
      "C05637  4,8-Dihydroxyquinoline\n",
      "C05638  5-Hydroxykynurenamine\n",
      "C05639  4,6-Dihydroxyquinoline\n",
      "C05640  Cinnavalininate\n",
      "C05641  5-(3'-Carboxy-3'-oxopropenyl)-4,6-dihydroxypicolinate\n",
      "C05642  Formyl-N-acetyl-5-methoxykynurenamine\n",
      "C05643  6-Hydroxymelatonin\n",
      "C05644  3-Methylindolepyruvate\n",
      "C05645  4-(2-Amino-3-hydroxyphenyl)-2,4-dioxobutanoate\n",
      "C05646  5-Hydroxyindolepyruvate\n",
      "C05647  Formyl-5-hydroxykynurenamine\n",
      "C05648  5-Hydroxy-N-formylkynurenine\n",
      "C05651  5-Hydroxykynurenine\n",
      "C05652  4-(2-Amino-5-hydroxyphenyl)-2,4-dioxobutanoate\n",
      "C05653  Formylanthranilate\n",
      "C05654  5-(2'-Formylethyl)-4,6-dihydroxypicolinate\n",
      "C05655  5-(2'-Carboxyethyl)-4,6-dihydroxypicolinate\n",
      "C05656  5-(3'-Carboxy-3'-oxopropyl)-4,6-dihydroxypicolinate\n",
      "C05658  Indoxyl\n",
      "C05659  5-Methoxytryptamine\n",
      "C05660  5-Methoxyindoleacetate\n",
      "C05663  6-Hydroxykynurenate\n",
      "C05830  8-Methoxykynurenate\n",
      "C05831  3-Methoxyanthranilate\n",
      "C05832  5-Hydroxyindoleacetylglycine\n",
      "C05834  3-Methyldioxyindole\n",
      "C05835  2-Formaminobenzoylacetate\n",
      "C05837  Glucobrassicin\n",
      "C06212  N-Methylserotonin\n",
      "C06213  N-Methyltryptamine\n",
      "C08313  Skatole\n",
      "C10164  Picolinic acid\n",
      "C16516  Indolylmethylthiohydroximate\n",
      "C16517  Indolylmethyl-desulfoglucosinolate\n",
      "C16518  S-(Indolylmethylthiohydroximoyl)-L-cysteine\n",
      "C17203  N-Hydroxyl-tryptamine\n",
      "C22006  (R)-(Indol-3-yl)lactate\n",
      "C22202  2-Oxoindole-3-acetate\n",
      "REFERENCE   PMID:15749753\n",
      "AUTHORS   Woodward AW, Bartel B\n",
      "TITLE     Auxin: regulation, action, and interaction.\n",
      "JOURNAL   Ann Bot 95:707-35 (2005)\n",
      "DOI:10.1093/aob/mci083\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00760  Nicotinate and nicotinamide metabolism\n",
      "KO_PATHWAY  ko00380\n",
      "///\n",
      "ENTRY       ath00400                    Pathway\n",
      "NAME        Phenylalanine, tyrosine and tryptophan biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Amino acid metabolism\n",
      "PATHWAY_MAP ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "MODULE      ath_M00022  Shikimate pathway, phosphoenolpyruvate + erythrose-4P => chorismate [PATH:ath00400]\n",
      "ath_M00023  Tryptophan biosynthesis, chorismate => tryptophan [PATH:ath00400]\n",
      "ath_M00040  Tyrosine biosynthesis, chorismate => arogenate => tyrosine [PATH:ath00400]\n",
      "ath_M00910  Phenylalanine biosynthesis, chorismate => arogenate => phenylalanine [PATH:ath00400]\n",
      "DBLINKS     GO: 0009094 0006571 0000162\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G22410  Class-II DAHP synthetase family protein [KO:K01626] [EC:2.5.1.54]\n",
      "AT4G33510  DHS2; 3-deoxy-d-arabino-heptulosonate 7-phosphate synthase [KO:K01626] [EC:2.5.1.54]\n",
      "AT4G39980  DHS1; 3-deoxy-D-arabino-heptulosonate 7-phosphate synthase 1 [KO:K01626] [EC:2.5.1.54]\n",
      "AT5G66120  3-dehydroquinate synthase [KO:K01735] [EC:4.2.3.4]\n",
      "AT3G06350  MEE32; dehydroquinate dehydratase, putative / shikimate dehydrogenase [KO:K13832] [EC:4.2.1.10 1.1.1.25]\n",
      "AT2G21940  SK1; shikimate kinase 1 [KO:K00891] [EC:2.7.1.71]\n",
      "AT4G39540  SK2; shikimate kinase 2 [KO:K00891] [EC:2.7.1.71]\n",
      "AT1G48860  RNA 3'-terminal phosphate cyclase/enolpyruvate transferase, alpha/beta [KO:K00800] [EC:2.5.1.19]\n",
      "AT2G45300  RNA 3'-terminal phosphate cyclase/enolpyruvate transferase, alpha/beta [KO:K00800] [EC:2.5.1.19]\n",
      "AT1G48850  EMB1144; chorismate synthase, putative / 5-enolpyruvylshikimate-3-phosphate phospholyase [KO:K01736] [EC:4.2.3.5]\n",
      "AT3G55870  ADC synthase superfamily protein [KO:K01657] [EC:4.1.3.27]\n",
      "AT5G05730  ASA1; anthranilate synthase alpha subunit 1 [KO:K01657] [EC:4.1.3.27]\n",
      "AT2G29690  ASA2; anthranilate synthase 2 [KO:K01657] [EC:4.1.3.27]\n",
      "AT5G57890  Glutamine amidotransferase type 1 family protein [KO:K01658] [EC:4.1.3.27]\n",
      "AT1G25220  ASB1; anthranilate synthase beta subunit 1 [KO:K01658] [EC:4.1.3.27]\n",
      "AT1G24807  Glutamine amidotransferase type 1 family protein [KO:K01658] [EC:4.1.3.27]\n",
      "AT1G24909  Glutamine amidotransferase type 1 family protein [KO:K01658] [EC:4.1.3.27]\n",
      "AT1G25083  Glutamine amidotransferase type 1 family protein [KO:K01658] [EC:4.1.3.27]\n",
      "AT1G25155  Glutamine amidotransferase type 1 family protein [KO:K01658] [EC:4.1.3.27]\n",
      "AT5G17990  TRP1; tryptophan biosynthesis 1 [KO:K00766] [EC:2.4.2.18]\n",
      "AT1G07780  PAI1; phosphoribosylanthranilate isomerase 1 [KO:K01817] [EC:5.3.1.24]\n",
      "AT5G05590  PAI2; phosphoribosylanthranilate isomerase 2 [KO:K01817] [EC:5.3.1.24]\n",
      "AT1G29410  PAI3; phosphoribosylanthranilate isomerase 3 [KO:K01817] [EC:5.3.1.24]\n",
      "AT5G48220  Aldolase-type TIM barrel family protein [KO:K01609] [EC:4.1.1.48]\n",
      "AT2G04400  Aldolase-type TIM barrel family protein [KO:K01609] [EC:4.1.1.48]\n",
      "AT4G02610  Aldolase-type TIM barrel family protein [KO:K01695] [EC:4.2.1.20]\n",
      "AT3G54640  TSA1; tryptophan synthase alpha chain [KO:K01695] [EC:4.2.1.20]\n",
      "AT5G54810  TSB1; tryptophan synthase beta-subunit 1 [KO:K01696] [EC:4.2.1.20]\n",
      "AT4G27070  TSB2; tryptophan synthase beta-subunit 2 [KO:K01696] [EC:4.2.1.20]\n",
      "AT5G28237  Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K01696] [EC:4.2.1.20]\n",
      "AT5G38530  TSBtype2; tryptophan synthase beta type 2 [KO:K06001] [EC:4.2.1.20]\n",
      "AT1G69370  CM3; chorismate mutase 3 [KO:K01850] [EC:5.4.99.5]\n",
      "AT5G10870  CM2; chorismate mutase 2 [KO:K01850] [EC:5.4.99.5]\n",
      "AT3G29200  CM1; chorismate mutase 1 [KO:K01850] [EC:5.4.99.5]\n",
      "AT5G34930  arogenate dehydrogenase [KO:K15227] [EC:1.3.1.78]\n",
      "AT1G15710  prephenate dehydrogenase family protein [KO:K15227] [EC:1.3.1.78]\n",
      "AT2G27820  PD1; prephenate dehydratase 1 [KO:K05359] [EC:4.2.1.91 4.2.1.51]\n",
      "AT5G22630  ADT5; arogenate dehydratase 5 [KO:K05359] [EC:4.2.1.91 4.2.1.51]\n",
      "AT3G44720  ADT4; arogenate dehydratase 4 [KO:K05359] [EC:4.2.1.91 4.2.1.51]\n",
      "AT1G08250  ADT6; arogenate dehydratase 6 [KO:K05359] [EC:4.2.1.91 4.2.1.51]\n",
      "AT1G11790  ADT1; arogenate dehydratase 1 [KO:K05359] [EC:4.2.1.91 4.2.1.51]\n",
      "AT3G07630  ADT2; arogenate dehydratase 2 [KO:K05359] [EC:4.2.1.91 4.2.1.51]\n",
      "AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "AT2G22250  AAT; aspartate aminotransferase [KO:K15849] [EC:2.6.1.1 2.6.1.78 2.6.1.79]\n",
      "AT5G53970  TAT7; Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT5G36160  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT2G24850  TAT3; tyrosine aminotransferase 3 [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28410  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28420  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G23590  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT1G71920  HISN6B; HISTIDINE BIOSYNTHESIS 6B [KO:K00817] [EC:2.6.1.9]\n",
      "AT5G10330  HPA1; histidinol phosphate aminotransferase 1 [KO:K00817] [EC:2.6.1.9]\n",
      "COMPOUND    C00074  Phosphoenolpyruvate\n",
      "C00078  L-Tryptophan\n",
      "C00079  L-Phenylalanine\n",
      "C00082  L-Tyrosine\n",
      "C00108  Anthranilate\n",
      "C00119  5-Phospho-alpha-D-ribose 1-diphosphate\n",
      "C00166  Phenylpyruvate\n",
      "C00251  Chorismate\n",
      "C00254  Prephenate\n",
      "C00279  D-Erythrose 4-phosphate\n",
      "C00296  Quinate\n",
      "C00354  D-Fructose 1,6-bisphosphate\n",
      "C00441  L-Aspartate 4-semialdehyde\n",
      "C00463  Indole\n",
      "C00493  Shikimate\n",
      "C00587  3-Hydroxybenzoate\n",
      "C00826  L-Arogenate\n",
      "C00944  3-Dehydroquinate\n",
      "C01094  D-Fructose 1-phosphate\n",
      "C01179  3-(4-Hydroxyphenyl)pyruvate\n",
      "C01269  5-O-(1-Carboxyvinyl)-3-phosphoshikimate\n",
      "C01302  1-(2-Carboxyphenylamino)-1-deoxy-D-ribulose 5-phosphate\n",
      "C02637  3-Dehydroshikimate\n",
      "C03175  Shikimate 3-phosphate\n",
      "C03506  Indoleglycerol phosphate\n",
      "C04302  N-(5-Phospho-D-ribosyl)anthranilate\n",
      "C04691  2-Dehydro-3-deoxy-D-arabino-heptonate 7-phosphate\n",
      "C16848  6-Deoxy-5-ketofructose 1-phosphate\n",
      "C16850  2-Amino-3,7-dideoxy-D-threo-hept-6-ulosonic acid\n",
      "C17235  L-Homophenylalanine\n",
      "C20327  2-Oxo-4-phenylbutyric acid\n",
      "C20653  2-Benzylmalic acid\n",
      "C20654  3-Benzylmalic acid\n",
      "C20710  (4R,5R)-4,5-Dihydroxycyclohexa-1(6),2-diene-1-carboxylate\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00130  Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "ath00350  Tyrosine metabolism\n",
      "ath00360  Phenylalanine metabolism\n",
      "ath00380  Tryptophan metabolism\n",
      "ath00710  Carbon fixation in photosynthetic organisms\n",
      "ath00790  Folate biosynthesis\n",
      "ath00901  Indole alkaloid biosynthesis\n",
      "ath00940  Phenylpropanoid biosynthesis\n",
      "ath00950  Isoquinoline alkaloid biosynthesis\n",
      "ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "ath00966  Glucosinolate biosynthesis\n",
      "KO_PATHWAY  ko00400\n",
      "///\n",
      "ENTRY       ath00410                    Pathway\n",
      "NAME        beta-Alanine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of other amino acids\n",
      "PATHWAY_MAP ath00410  beta-Alanine metabolism\n",
      "DBLINKS     GO: 0019482\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G02000  GAD3; glutamate decarboxylase 3 [KO:K01580] [EC:4.1.1.15]\n",
      "AT3G17760  GAD5; glutamate decarboxylase 5 [KO:K01580] [EC:4.1.1.15]\n",
      "AT5G17330  GAD; glutamate decarboxylase [KO:K01580] [EC:4.1.1.15]\n",
      "AT2G02010  GAD4; glutamate decarboxylase 4 [KO:K01580] [EC:4.1.1.15]\n",
      "AT1G65960  GAD2; glutamate decarboxylase 2 [KO:K01580] [EC:4.1.1.15]\n",
      "AT4G29720  PAO5; polyamine oxidase 5 [KO:K12259] [EC:1.5.3.16 1.5.3.-]\n",
      "AT5G13700  PAO1; polyamine oxidase 1 [KO:K13366] [EC:1.5.3.14 1.5.3.16 1.5.3.-]\n",
      "AT3G59050  PAO3; polyamine oxidase 3 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT1G65840  PAO4; polyamine oxidase 4 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT2G43020  PAO2; polyamine oxidase 2 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT2G42490  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31710  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31690  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G14940  AO1; amine oxidase 1 [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G62810  CuAO1; Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT3G43670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G12290  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT5G48840  PANC; pantoate-beta-alanine ligase [KO:K01918] [EC:6.3.2.1]\n",
      "AT3G17810  PYD1; pyrimidine 1 [KO:K00207] [EC:1.3.1.2]\n",
      "AT5G12200  PYD2; pyrimidine 2 [KO:K01464] [EC:3.5.2.2]\n",
      "AT5G64370  BETA-UP; beta-ureidopropionase [KO:K01431] [EC:3.5.1.6]\n",
      "AT4G20930  6-phosphogluconate dehydrogenase family protein [KO:K23146] [EC:1.1.1.31 1.1.1.59]\n",
      "AT2G30650  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT5G65940  CHY1; beta-hydroxyisobutyryl-CoA hydrolase 1 [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G31810  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT1G06550  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT2G30660  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT3G60510  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT3G24360  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G13360  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "AT2G35690  ACX5; acyl-CoA oxidase 5 [KO:K00232] [EC:1.3.3.6]\n",
      "AT4G16760  ACX1; acyl-CoA oxidase 1 [KO:K00232] [EC:1.3.3.6]\n",
      "AT5G65110  ACX2; acyl-CoA oxidase 2 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G51840  ACX4; acyl-CoA oxidase 4 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06290  ACX3; acyl-CoA oxidase 3 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06310  ACX6; acyl-CoA oxidase 6 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G06690  putative acyl-coenzyme A oxidase [KO:K00232] [EC:1.3.3.6]\n",
      "AT4G04320  malonyl-CoA decarboxylase family protein [KO:K01578] [EC:4.1.1.9]\n",
      "AT2G14170  ALDH6B2; aldehyde dehydrogenase 6B2 [KO:K00140] [EC:1.2.1.18 1.2.1.27]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00049  L-Aspartate\n",
      "C00083  Malonyl-CoA\n",
      "C00099  beta-Alanine\n",
      "C00100  Propanoyl-CoA\n",
      "C00106  Uracil\n",
      "C00135  L-Histidine\n",
      "C00222  3-Oxopropanoate\n",
      "C00315  Spermidine\n",
      "C00334  4-Aminobutanoate\n",
      "C00383  Malonate\n",
      "C00386  Carnosine\n",
      "C00429  5,6-Dihydrouracil\n",
      "C00555  4-Aminobutyraldehyde\n",
      "C00750  Spermine\n",
      "C00804  Propynoate\n",
      "C00864  Pantothenate\n",
      "C00894  Propenoyl-CoA\n",
      "C00986  1,3-Diaminopropane\n",
      "C01013  3-Hydroxypropanoate\n",
      "C01073  N-Acetyl-beta-alanine\n",
      "C01262  beta-Alanyl-N(pi)-methyl-L-histidine\n",
      "C02335  beta-Alanyl-CoA\n",
      "C02642  3-Ureidopropionate\n",
      "C03492  D-4'-Phosphopantothenate\n",
      "C03722  Quinolinate\n",
      "C05340  beta-Alanyl-L-arginine\n",
      "C05341  beta-Alanyl-L-lysine\n",
      "C05665  3-Aminopropanal\n",
      "C05668  3-Hydroxypropionyl-CoA\n",
      "C05669  beta-Nitropropanoate\n",
      "C05670  3-Aminopropiononitrile\n",
      "REL_PATHWAY ath00061  Fatty acid biosynthesis\n",
      "ath00240  Pyrimidine metabolism\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00460  Cyanoamino acid metabolism\n",
      "ath00640  Propanoate metabolism\n",
      "ath00770  Pantothenate and CoA biosynthesis\n",
      "KO_PATHWAY  ko00410\n",
      "///\n",
      "ENTRY       ath00430                    Pathway\n",
      "NAME        Taurine and hypotaurine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of other amino acids\n",
      "PATHWAY_MAP ath00430  Taurine and hypotaurine metabolism\n",
      "DBLINKS     GO: 0019530\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G02000  GAD3; glutamate decarboxylase 3 [KO:K01580] [EC:4.1.1.15]\n",
      "AT3G17760  GAD5; glutamate decarboxylase 5 [KO:K01580] [EC:4.1.1.15]\n",
      "AT5G17330  GAD; glutamate decarboxylase [KO:K01580] [EC:4.1.1.15]\n",
      "AT2G02010  GAD4; glutamate decarboxylase 4 [KO:K01580] [EC:4.1.1.15]\n",
      "AT1G65960  GAD2; glutamate decarboxylase 2 [KO:K01580] [EC:4.1.1.15]\n",
      "AT4G39640  GGT1; gamma-glutamyl transpeptidase 1 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT4G39650  GGT2; gamma-glutamyl transpeptidase 2 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT1G69820  GGT3; gamma-glutamyl transpeptidase 3 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT4G29210  GGT4; gamma-glutamyl transpeptidase 4 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00033  Acetate\n",
      "C00041  L-Alanine\n",
      "C00084  Acetaldehyde\n",
      "C00094  Sulfite\n",
      "C00097  L-Cysteine\n",
      "C00227  Acetyl phosphate\n",
      "C00245  Taurine\n",
      "C00506  L-Cysteate\n",
      "C00519  Hypotaurine\n",
      "C00593  Sulfoacetaldehyde\n",
      "C00606  3-Sulfino-L-alanine\n",
      "C01678  Cysteamine\n",
      "C01959  Taurocyamine\n",
      "C03149  N-Phosphotaurocyamine\n",
      "C05122  Taurocholate\n",
      "C05123  2-Hydroxyethanesulfonate\n",
      "C05844  5-L-Glutamyl-taurine\n",
      "C06735  Aminoacetaldehyde\n",
      "C14179  Sulfoacetate\n",
      "REFERENCE   PMID:12471498\n",
      "AUTHORS   Cook AM, Denger K.\n",
      "TITLE     Dissimilation of the C2 sulfonates.\n",
      "JOURNAL   Arch Microbiol 179:1-6 (2002)\n",
      "DOI:10.1007/s00203-002-0497-0\n",
      "REFERENCE   PMID:11082195\n",
      "AUTHORS   Laue H, Cook AM.\n",
      "TITLE     Biochemical and molecular characterization of taurine:pyruvate aminotransferase from the anaerobe Bilophila wadsworthia.\n",
      "JOURNAL   Eur J Biochem 267:6841-8 (2000)\n",
      "DOI:10.1046/j.1432-1033.2000.01782.x\n",
      "REFERENCE   PMID:12358600\n",
      "AUTHORS   Ruff J, Denger K, Cook AM.\n",
      "TITLE     Sulphoacetaldehyde acetyltransferase yields acetyl phosphate: purification from Alcaligenes defragrans and gene clusters in taurine degradation.\n",
      "JOURNAL   Biochem J 369:275-85 (2003)\n",
      "DOI:10.1042/BJ20021455\n",
      "REFERENCE   PMID:15073291\n",
      "AUTHORS   Bruggemann C, Denger K, Cook AM, Ruff J.\n",
      "TITLE     Enzymes and genes of taurine and isethionate dissimilation in Paracoccus denitrificans.\n",
      "JOURNAL   Microbiology 150:805-16 (2004)\n",
      "DOI:10.1099/mic.0.26795-0\n",
      "REFERENCE   PMID:15340795\n",
      "AUTHORS   Denger K, Weinitschke S, Hollemeyer K, Cook AM.\n",
      "TITLE     Sulfoacetate generated by Rhodopseudomonas palustris from taurine.\n",
      "JOURNAL   Arch Microbiol 182:254-8 (2004)\n",
      "DOI:10.1007/s00203-004-0678-0\n",
      "REFERENCE   PMID:15184572\n",
      "AUTHORS   Denger K, Ruff J, Schleheck D, Cook AM.\n",
      "TITLE     Rhodococcus opacus expresses the xsc gene to utilize taurine as a carbon source or as a nitrogen source but not as a sulfur source.\n",
      "JOURNAL   Microbiology 150:1859-67 (2004)\n",
      "DOI:10.1099/mic.0.27077-0\n",
      "REFERENCE   PMID:15883781\n",
      "AUTHORS   Styp von Rekowski K, Denger K, Cook AM.\n",
      "TITLE     Isethionate as a product from taurine during nitrogen-limited growth of Klebsiella oxytoca TauN1.\n",
      "JOURNAL   Arch Microbiol 183:325-30 (2005)\n",
      "DOI:10.1007/s00203-005-0776-7\n",
      "REFERENCE   PMID:4297098\n",
      "AUTHORS   Jacobsen JG, Smith LH\n",
      "TITLE     Biochemistry and physiology of taurine and taurine derivatives.\n",
      "JOURNAL   Physiol Rev 48:424-511 (1968)\n",
      "DOI:10.1152/physrev.1968.48.2.424\n",
      "REL_PATHWAY ath00270  Cysteine and methionine metabolism\n",
      "ath00460  Cyanoamino acid metabolism\n",
      "ath00480  Glutathione metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00920  Sulfur metabolism\n",
      "KO_PATHWAY  ko00430\n",
      "///\n",
      "ENTRY       ath00440                    Pathway\n",
      "NAME        Phosphonate and phosphinate metabolism - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Natural products containing carbon-phosphorous bonds, so-called C-P compounds, are derivatives of phosphonate and phosphinate with substitution of alkyl group for hydrogen of phosphorus-hydrogen bonds. C-P compounds have been found in many organisms, but only protists and bacteria, mostly Actinobacteria, have biosynthetic capacity. A common reaction in the biosynthetic pathway is C-P bond forming reaction from phosphoenolpyruvate (PEP) to phosphonopyruvate (PnPy) catalyzed by PEP phosphomutase. 2-Aminoethylphosphonate (AEP) is the most abundant C-P compound in the natural world. AEP derivatives include phosphonoprotein, phosphonoglycan, and phosphonolipid. Other known C-P compounds are bioactive substances used in medicine (antibiotics) and agriculture (herbicide) such as fosfomycin, FR-33289, rhizocticin, and bialaphos.\n",
      "CLASS       Metabolism; Metabolism of other amino acids\n",
      "PATHWAY_MAP ath00440  Phosphonate and phosphinate metabolism\n",
      "DBLINKS     GO: 0019634\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G38670  PECT1; phosphorylethanolamine cytidylyltransferase 1 [KO:K00967] [EC:2.7.7.14]\n",
      "AT1G13560  AAPT1; aminoalcoholphosphotransferase 1 [KO:K00993] [EC:2.7.8.1]\n",
      "AT3G25585  AAPT2; aminoalcoholphosphotransferase [KO:K00993] [EC:2.7.8.1]\n",
      "AT2G32260  CCT1; phosphorylcholine cytidylyltransferase [KO:K00968] [EC:2.7.7.15]\n",
      "AT4G15130  CCT2; phosphorylcholine cytidylyltransferase2 [KO:K00968] [EC:2.7.7.15]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00033  Acetate\n",
      "C00037  Glycine\n",
      "C00058  Formate\n",
      "C00074  Phosphoenolpyruvate\n",
      "C00084  Acetaldehyde\n",
      "C00117  D-Ribose 5-phosphate\n",
      "C01151  D-Ribose 1,5-bisphosphate\n",
      "C01438  Methane\n",
      "C02798  3-Phosphonopyruvate\n",
      "C03167  Phosphonoacetaldehyde\n",
      "C03557  2-Aminoethylphosphonate\n",
      "C04650  L-Phosphinothricin\n",
      "C05672  2-Amino-3-phosphonopropanoate\n",
      "C05673  CMP-2-aminoethylphosphonate\n",
      "C05674  CMP-N-trimethyl-2-aminoethylphosphonate\n",
      "C05675  Diacylglyceryl-2-aminoethylphosphonate\n",
      "C05676  Diacylglyceryl-N-trimethyl-2-aminoethylphosphonate\n",
      "C05677  Lipophosphonoglycan\n",
      "C05678  (2-Amino-1-hydroxyethyl)phosphonate\n",
      "C05679  N-Monomethyl-2-aminoethylphosphonate\n",
      "C05680  N-Dimethyl-2-aminoethylphosphonate\n",
      "C05681  Ceramide 2-aminoethylphosphonate\n",
      "C05682  Phosphonoacetate\n",
      "C05683  Ciliatocholate\n",
      "C06367  1-Carboxyvinyl carboxyphosphonate\n",
      "C06368  3-(Hydrohydroxyphosphoryl)pyruvate\n",
      "C06451  2-Hydroxyethylphosphonate\n",
      "C06455  Hydroxymethylphosphonate\n",
      "C06456  Phosphonoformate\n",
      "C06457  Bialaphos\n",
      "C06459  N-Trimethyl-2-aminoethylphosphonate\n",
      "C17940  Rhizocticin B\n",
      "C17941  2-Oxo-4-phosphonobutanoate\n",
      "C17942  FR 900098\n",
      "C17943  Formylphosphonate\n",
      "C17944  Rhizocticin A\n",
      "C17945  Phosphonoformyl-CMP\n",
      "C17947  2-Phosphinomethylmalate\n",
      "C17948  Deamino-alpha-keto-demethylphosphinothricin\n",
      "C17949  N-Acetyldemethylphosphinothricin\n",
      "C17950  N-Acetyldemethylphosphinothricin tripeptide\n",
      "C17951  N-Acetylbialaphos\n",
      "C17952  N-Acetylphosphinothricin\n",
      "C17960  Rhizocticin C\n",
      "C17961  Rhizocticin D\n",
      "C17962  Demethylphosphinothricin\n",
      "C20396  Methylphosphonate\n",
      "C20422  alpha-D-Ribose 1-methylphosphonate 5-triphosphate\n",
      "C20423  alpha-D-Ribose 1-methylphosphonate 5-phosphate\n",
      "C20440  alpha-D-Ribose 1,2-cyclic phosphate 5-phosphate\n",
      "C20986  D-Ribose 2,5-bisphosphate\n",
      "C21372  3-Phosphinomethylmalate\n",
      "C21403  2-Acetamidoethylphosphonate\n",
      "C21404  N-Ethylacetamide\n",
      "C21613  (R)-2-(Phosphonomethyl)malate\n",
      "REFERENCE   PMID:19489722\n",
      "AUTHORS   Metcalf WW, van der Donk WA\n",
      "TITLE     Biosynthesis of phosphonic and phosphinic acid natural products.\n",
      "JOURNAL   Annu Rev Biochem 78:65-94 (2009)\n",
      "DOI:10.1146/annurev.biochem.78.091707.100215\n",
      "REFERENCE   PMID:19053721\n",
      "AUTHORS   Xiao Y, Lee K, Liu P\n",
      "TITLE     Syntheses of the P-methylase substrates of the bialaphos biosynthetic pathway.\n",
      "JOURNAL   Org Lett 10:5521-4 (2008)\n",
      "DOI:10.1021/ol802269x\n",
      "REFERENCE   PMID:17168571\n",
      "AUTHORS   van der Donk WA\n",
      "TITLE     Rings, radicals, and regeneration: the early years of a bioorganic laboratory.\n",
      "JOURNAL   J Org Chem 71:9561-71 (2006)\n",
      "DOI:10.1021/jo0614240\n",
      "REFERENCE   PMID:15616300\n",
      "AUTHORS   Blodgett JA, Zhang JK, Metcalf WW\n",
      "TITLE     Molecular cloning, sequence analysis, and heterologous expression of the phosphinothricin tripeptide biosynthetic gene cluster from Streptomyces viridochromogenes DSM 40736.\n",
      "JOURNAL   Antimicrob Agents Chemother 49:230-40 (2005)\n",
      "DOI:10.1128/AAC.49.1.230-240.2005\n",
      "REFERENCE   PMID:15574905\n",
      "AUTHORS   Schwartz D, Berger S, Heinzelmann E, Muschko K, Welzel K, Wohlleben W\n",
      "TITLE     Biosynthetic gene cluster of the herbicide phosphinothricin tripeptide from Streptomyces viridochromogenes Tu494.\n",
      "JOURNAL   Appl Environ Microbiol 70:7093-102 (2004)\n",
      "DOI:10.1128/AEM.70.12.7093-7102.2004\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "KO_PATHWAY  ko00440\n",
      "///\n",
      "ENTRY       ath00450                    Pathway\n",
      "NAME        Selenocompound metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of other amino acids\n",
      "PATHWAY_MAP ath00450  Selenocompound metabolism\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G01120  MTO1; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01739] [EC:2.5.1.48]\n",
      "AT1G33320  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01739] [EC:2.5.1.48]\n",
      "AT3G57050  CBL; cystathionine beta-lyase [KO:K01760] [EC:4.4.1.13]\n",
      "AT5G20980  MS3; methionine synthase 3 [KO:K00549] [EC:2.1.1.14]\n",
      "AT5G17920  ATMS1; Cobalamin-independent synthase family protein [KO:K00549] [EC:2.1.1.14]\n",
      "AT3G03780  MS2; methionine synthase 2 [KO:K00549] [EC:2.1.1.14]\n",
      "AT1G64660  MGL; methionine gamma-lyase [KO:K01761] [EC:4.4.1.11]\n",
      "AT1G08490  CPNIFS; chloroplastic NIFS-like cysteine desulfurase [KO:K11717] [EC:2.8.1.7 4.4.1.16]\n",
      "AT5G49810  MMT; methionine S-methyltransferase [KO:K08247] [EC:2.1.1.12]\n",
      "AT4G35460  NTRB; NADPH-dependent thioredoxin reductase B [KO:K00384] [EC:1.8.1.9]\n",
      "AT2G17420  NTRA; NADPH-dependent thioredoxin reductase A [KO:K00384] [EC:1.8.1.9]\n",
      "AT2G41680  NTRC; NADPH-dependent thioredoxin reductase C [KO:K00384] [EC:1.8.1.9]\n",
      "AT4G14680  APS3; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT3G22890  APS1; ATP sulfurylase 1 [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT1G19920  APS2; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT5G43780  APS4; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT4G13780  methionine-tRNA ligase, putative / methionyl-tRNA synthetase, putative / MetRS [KO:K01874] [EC:6.1.1.10]\n",
      "AT3G55400  OVA1; methionyl-tRNA synthetase / methionine-tRNA ligase / MetRS (cpMetRS) [KO:K01874] [EC:6.1.1.10]\n",
      "COMPOUND    C00017  Protein\n",
      "C00041  L-Alanine\n",
      "C01528  Hydrogen selenide\n",
      "C02535  Dimethyl selenide\n",
      "C05172  Selenophosphoric acid\n",
      "C05335  L-Selenomethionine\n",
      "C05336  Selenomethionyl-tRNA(Met)\n",
      "C05684  Selenite\n",
      "C05686  Adenylylselenate\n",
      "C05688  L-Selenocysteine\n",
      "C05689  Se-Methyl-L-selenocysteine\n",
      "C05690  Se-Methylselenomethionine\n",
      "C05695  gamma-Glutamyl-Se-methylselenocysteine\n",
      "C05697  Selenate\n",
      "C05698  Selenohomocysteine\n",
      "C05699  L-Selenocystathionine\n",
      "C05703  Methaneselenol\n",
      "C06481  L-Seryl-tRNA(Sec)\n",
      "C06482  L-Selenocysteinyl-tRNA(Sec)\n",
      "C16638  O-Phosphoseryl-tRNA(Sec)\n",
      "C18870  Selenodiglutathione\n",
      "C18871  Glutathioselenol\n",
      "C18872  Trimethylselenonium\n",
      "C18893  1-Methylseleno-N-acetyl-D-galactosamine\n",
      "C18902  Methylselenic acid\n",
      "C18904  Methylselenopyruvate\n",
      "C18905  Methylselenocysteine Se-oxide\n",
      "REFERENCE   PMID:21190829\n",
      "AUTHORS   Hoefig CS, Renko K, Kohrle J, Birringer M, Schomburg L\n",
      "TITLE     Comparison of different selenocompounds with respect to nutritional value vs. toxicity using liver cells in culture.\n",
      "JOURNAL   J Nutr Biochem 22:945-55 (2011)\n",
      "DOI:10.1016/j.jnutbio.2010.08.006\n",
      "REFERENCE   PMID:19051051\n",
      "AUTHORS   McKenzie MJ, Hunter DA, Pathirana R, Watson LM, Joyce NI, Matich AJ, Rowan DD, Brummell DA\n",
      "TITLE     Accumulation of an organic anticancer selenium compound in a transgenic Solanaceous species shows wider applicability of the selenocysteine methyltransferase transgene from selenium hyperaccumulators.\n",
      "JOURNAL   Transgenic Res 18:407-24 (2009)\n",
      "DOI:10.1007/s11248-008-9233-0\n",
      "REFERENCE   PMID:20383543\n",
      "AUTHORS   Pinto JT, Lee JI, Sinha R, MacEwan ME, Cooper AJ\n",
      "TITLE     Chemopreventive mechanisms of alpha-keto acid metabolites of naturally occurring organoselenium compounds.\n",
      "JOURNAL   Amino Acids 41:29-41 (2011)\n",
      "DOI:10.1007/s00726-010-0578-3\n",
      "REFERENCE   PMID:17451884\n",
      "AUTHORS   Suzuki KT, Kurasaki K, Suzuki N\n",
      "TITLE     Selenocysteine beta-lyase and methylselenol demethylase in the metabolism of Se-methylated selenocompounds into selenide.\n",
      "JOURNAL   Biochim Biophys Acta 1770:1053-61 (2007)\n",
      "DOI:10.1016/j.bbagen.2007.03.007\n",
      "REFERENCE   PMID:11782468\n",
      "AUTHORS   Gromer S, Gross JH\n",
      "TITLE     Methylseleninate is a substrate rather than an inhibitor of mammalian thioredoxin reductase. Implications for the antitumor effects of selenium.\n",
      "JOURNAL   J Biol Chem 277:9701-6 (2002)\n",
      "DOI:10.1074/jbc.M109234200\n",
      "KO_PATHWAY  ko00450\n",
      "///\n",
      "ENTRY       ath00460                    Pathway\n",
      "NAME        Cyanoamino acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of other amino acids\n",
      "PATHWAY_MAP ath00460  Cyanoamino acid metabolism\n",
      "DBLINKS     GO: 0033052 0019756\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G42260  BGLU12; beta glucosidase 12 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G44640  BGLU13; beta glucosidase 13 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44450  BGLU15; beta glucosidase 15 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G26560  BGLU40; beta glucosidase 40 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44480  BGLU17; beta glucosidase 17 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G47050  Glycosyl hydrolase family protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G75940  ATA27; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G66280  BGLU22; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G66270  BGLU21; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G09260  PYK10; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G28510  BGLU24; beta glucosidase 24 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G36890  BGLU42; beta glucosidase 42 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G54570  BGLU41; beta glucosidase 41 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G32860  BGLU33; beta glucosidase 33 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G03640  BGLU25; beta glucosidase 25 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G21370  BGLU19; beta glucosidase 19 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60130  BGLU16; beta glucosidase 16 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G25630  BGLU14; beta glucosidase 14 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60140  DIN2; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44470  BGLU29; beta glucosidase 29 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44460  BGLU28; beta glucosidase 28 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44490  PEN2; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G24550  BGLU32; beta glucosidase 32 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G24540  BGLU31; beta glucosidase 31 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60120  BGLU27; beta glucosidase 27 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G02850  BGLU11; beta glucosidase 11 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G45191  BGLU1; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G22100  BGLU3; beta glucosidase 2 [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G27830  BGLU10; beta glucosidase 10 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G62750  BGLU8; beta glucosidase 8 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G16580  BGLU2; beta glucosidase 2 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G60270  BGLU6; beta glucosidase 6 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G62740  BGLU7; beta glucosidase 7 [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G27820  BGLU9; beta glucosidase 9 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G60260  BGLU5; beta glucosidase 5 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G47000  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G20950  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G20940  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G47040  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G04885  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G47010  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G18080  BGLU44; B-S glucosidase 44 [KO:K05350] [EC:3.2.1.21]\n",
      "AT1G61820  BGLU46; beta glucosidase 46 [KO:K05350] [EC:3.2.1.21]\n",
      "AT4G21760  BGLU47; beta-glucosidase 47 [KO:K05350] [EC:3.2.1.21]\n",
      "AT1G61810  BGLU45; beta-glucosidase 45 [KO:K05350] [EC:3.2.1.21]\n",
      "AT3G18070  BGLU43; beta glucosidase 43 [KO:K05350] [EC:3.2.1.21]\n",
      "AT5G05260  CYP79A2; phenylalanine N-monooxygenase-like protein [KO:K12153] [EC:1.14.14.40]\n",
      "AT5G35917  CYP79A3P; cytochrome P450, family 79, subfamily A, polypeptide 3 pseudogene [KO:K12153] [EC:1.14.14.40]\n",
      "AT1G73050  Glucose-methanol-choline (GMC) oxidoreductase family protein [KO:K08248] [EC:4.1.2.10]\n",
      "AT5G10300  MES5; methyl esterase 5 [KO:K20802] [EC:4.1.2.10]\n",
      "AT3G44320  NIT3; nitrilase 3 [KO:K01501] [EC:3.5.5.1]\n",
      "AT3G44300  NIT2; nitrilase 2 [KO:K01501] [EC:3.5.5.1]\n",
      "AT3G44310  NIT1; nitrilase 1 [KO:K01501] [EC:3.5.5.1]\n",
      "AT3G61440  CYSC1; cysteine synthase C1 [KO:K13034] [EC:2.5.1.47 4.4.1.9]\n",
      "AT4G39640  GGT1; gamma-glutamyl transpeptidase 1 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT4G39650  GGT2; gamma-glutamyl transpeptidase 2 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT1G69820  GGT3; gamma-glutamyl transpeptidase 3 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT4G29210  GGT4; gamma-glutamyl transpeptidase 4 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT5G22300  NIT4; nitrilase 4 [KO:K13035] [EC:3.5.5.4 4.2.1.65]\n",
      "AT5G08100  ASPGA1; N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K13051] [EC:3.5.1.1 3.4.19.5]\n",
      "AT3G16150  ASPGB1; N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K13051] [EC:3.5.1.1 3.4.19.5]\n",
      "AT4G13890  EDA36; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G13930  SHM4; serine hydroxymethyltransferase 4 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G22020  SHM6; serine hydroxymethyltransferase 6 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G32520  SHM3; serine hydroxymethyltransferase 3 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G37930  SHM1; serine transhydroxymethyltransferase 1 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G36370  SHM7; serine hydroxymethyltransferase 7 [KO:K00600] [EC:2.1.2.1]\n",
      "AT5G26780  SHM2; serine hydroxymethyltransferase 2 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G37550  Acetamidase/Formamidase family protein [KO:K01455] [EC:3.5.1.49]\n",
      "AT4G37560  Acetamidase/Formamidase family protein [KO:K01455] [EC:3.5.1.49]\n",
      "COMPOUND    C00014  Ammonia\n",
      "C00037  Glycine\n",
      "C00049  L-Aspartate\n",
      "C00065  L-Serine\n",
      "C00079  L-Phenylalanine\n",
      "C00082  L-Tyrosine\n",
      "C00097  L-Cysteine\n",
      "C00152  L-Asparagine\n",
      "C00183  L-Valine\n",
      "C00302  Glutamate\n",
      "C00407  L-Isoleucine\n",
      "C00488  Formamide\n",
      "C00561  Mandelonitrile\n",
      "C00844  Prunasin\n",
      "C01326  Hydrogen cyanide\n",
      "C01401  Alanine\n",
      "C01594  Linamarin\n",
      "C02512  3-Cyano-L-alanine\n",
      "C02659  Acetone cyanohydrin\n",
      "C03004  N-Hydroxy-L-tyrosine\n",
      "C03219  (E)-2-Methylpropanal oxime\n",
      "C03742  (S)-4-Hydroxymandelonitrile\n",
      "C04350  (E)-4-Hydroxyphenylacetaldehyde oxime\n",
      "C05143  Dhurrin\n",
      "C05670  3-Aminopropiononitrile\n",
      "C05711  gamma-Glutamyl-beta-cyanoalanine\n",
      "C05714  alpha-Aminopropiononitrile\n",
      "C05715  gamma-Amino-gamma-cyanobutanoate\n",
      "C06114  gamma-Glutamyl-beta-aminopropiononitrile\n",
      "C08325  Amygdalin\n",
      "C08334  Lotaustralin\n",
      "C15503  N,N-Dihydroxy-L-tyrosine\n",
      "C16074  Phenylacetonitrile\n",
      "C16075  (Z)-Phenylacetaldehyde oxime\n",
      "C18796  (2R)-2-Hydroxy-2-methylbutanenitrile\n",
      "C19491  (E)-2-Methylbutanal oxime\n",
      "C19712  N-Hydroxy-L-phenylalanine\n",
      "C19714  (E)-Phenylacetaldoxime\n",
      "C19715  N,N-Dihydroxy-L-phenylalanine\n",
      "C20310  N-Hydroxy-L-isoleucine\n",
      "C20311  N,N-Dihydroxy-L-isoleucine\n",
      "C20312  (Z)-2-Methylbutanal oxime\n",
      "C20313  N-Hydroxy-L-valine\n",
      "C20314  N,N-Dihydroxy-L-valine\n",
      "C20315  (Z)-2-Methylpropanal oxime\n",
      "REFERENCE   PMID:11575729\n",
      "AUTHORS   Maruyama A, Saito K, Ishizawa K\n",
      "TITLE     Beta-cyanoalanine synthase and cysteine synthase from potato: molecular cloning, biochemical characterization, and spatial and hormonal regulation.\n",
      "JOURNAL   Plant Mol Biol 46:749-60 (2001)\n",
      "DOI:10.1023/A:1011629703784\n",
      "REFERENCE   PMID:10636899\n",
      "AUTHORS   Andersen MD, Busk PK, Svendsen I, Moller BL\n",
      "TITLE     Cytochromes P-450 from cassava (Manihot esculenta Crantz) catalyzing the first steps in the biosynthesis of the cyanogenic glucosides linamarin and lotaustralin. Cloning, functional expression in Pichia pastoris, and substrate specificity of the isolated recombinant enzymes.\n",
      "JOURNAL   J Biol Chem 275:1966-75 (2000)\n",
      "DOI:10.1074/jbc.275.3.1966\n",
      "REFERENCE\n",
      "AUTHORS   Hickel A, Hasslacher M, Griengl H.\n",
      "TITLE     Hydroxynitrile lyases: Functions and properties\n",
      "JOURNAL   Physiol Plant 98:891-898 (1996)\n",
      "DOI:10.1111/j.1399-3054.1996.tb06700.x\n",
      "REL_PATHWAY ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00350  Tyrosine metabolism\n",
      "ath00360  Phenylalanine metabolism\n",
      "ath00410  beta-Alanine metabolism\n",
      "ath00450  Selenocompound metabolism\n",
      "ath00470  D-Amino acid metabolism\n",
      "ath00480  Glutathione metabolism\n",
      "ath00910  Nitrogen metabolism\n",
      "KO_PATHWAY  ko00460\n",
      "///\n",
      "ENTRY       ath00470                    Pathway\n",
      "NAME        D-Amino acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of other amino acids\n",
      "PATHWAY_MAP ath00470  D-Amino acid metabolism\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G11640  SR; serine racemase [KO:K12235] [EC:5.1.1.18]\n",
      "AT1G48420  D-CDES; D-cysteine desulfhydrase [KO:K05396] [EC:4.4.1.15]\n",
      "AT3G26115  Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K05396] [EC:4.4.1.15]\n",
      "AT3G53580  diaminopimelate epimerase family protein [KO:K01778] [EC:5.1.1.7]\n",
      "AT3G14390  Pyridoxal-dependent decarboxylase family protein [KO:K01586] [EC:4.1.1.20]\n",
      "AT5G11880  Pyridoxal-dependent decarboxylase family protein [KO:K01586] [EC:4.1.1.20]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00036  Oxaloacetate\n",
      "C00037  Glycine\n",
      "C00041  L-Alanine\n",
      "C00047  L-Lysine\n",
      "C00049  L-Aspartate\n",
      "C00062  L-Arginine\n",
      "C00064  L-Glutamine\n",
      "C00065  L-Serine\n",
      "C00073  L-Methionine\n",
      "C00077  L-Ornithine\n",
      "C00079  L-Phenylalanine\n",
      "C00084  Acetaldehyde\n",
      "C00097  L-Cysteine\n",
      "C00133  D-Alanine\n",
      "C00134  Putrescine\n",
      "C00135  L-Histidine\n",
      "C00148  L-Proline\n",
      "C00166  Phenylpyruvate\n",
      "C00188  L-Threonine\n",
      "C00217  D-Glutamate\n",
      "C00402  D-Aspartate\n",
      "C00431  5-Aminopentanoate\n",
      "C00433  2,5-Dioxopentanoate\n",
      "C00515  D-Ornithine\n",
      "C00666  LL-2,6-Diaminoheptanedioate\n",
      "C00680  meso-2,6-Diaminoheptanedioate\n",
      "C00692  UDP-N-acetylmuramoyl-L-alanyl-D-glutamate\n",
      "C00739  D-Lysine\n",
      "C00740  D-Serine\n",
      "C00763  D-Proline\n",
      "C00792  D-Arginine\n",
      "C00793  D-Cysteine\n",
      "C00819  D-Glutamine\n",
      "C00820  D-Threonine\n",
      "C00855  D-Methionine\n",
      "C00993  D-Alanyl-D-alanine\n",
      "C01110  5-Amino-2-oxopentanoic acid\n",
      "C01157  Hydroxyproline\n",
      "C01180  4-Methylthio-2-oxobutanoic acid\n",
      "C01667  Bacitracin\n",
      "C01672  Cadaverine\n",
      "C01726  D-Lombricine\n",
      "C02237  5-Oxo-D-proline\n",
      "C02265  D-Phenylalanine\n",
      "C02855  N-Phospho-D-lombricine\n",
      "C03239  6-Amino-2-oxohexanoate\n",
      "C03341  2-Amino-4-oxopentanoic acid\n",
      "C03440  cis-4-Hydroxy-D-proline\n",
      "C03564  1-Pyrroline-2-carboxylate\n",
      "C03771  5-Guanidino-2-oxopentanoate\n",
      "C03933  5-D-Glutamyl-D-glutamyl-peptide\n",
      "C03943  (2R,4S)-2,4-Diaminopentanoate\n",
      "C04260  O-D-Alanyl-poly(ribitol phosphate)\n",
      "C04282  1-Pyrroline-4-hydroxy-2-carboxylate\n",
      "C04457  D-Alanyl-alanyl-poly(glycerolphosphate)\n",
      "C05161  (2R,5S)-2,5-Diaminohexanoate\n",
      "C05620  N-Acetyl-D-phenylalanine\n",
      "C05825  2-Amino-5-oxohexanoate\n",
      "C05939  Linatine\n",
      "C05941  2-Oxo-4-hydroxy-5-aminovalerate\n",
      "C05942  Pyrrole-2-carboxylate\n",
      "C06419  D-Histidine\n",
      "C22024  Staphylopine\n",
      "C22025  (2S)-2-Amino-4-{[(1R)-1-carboxy-2-(1H-imidazol-4-yl)ethyl]amino}butanoate\n",
      "REFERENCE   PMID:33788945\n",
      "AUTHORS   Miyamoto T, Homma H\n",
      "TITLE     D-Amino acid metabolism in bacteria.\n",
      "JOURNAL   J Biochem 170:5-13 (2021)\n",
      "DOI:10.1093/jb/mvab043\n",
      "REFERENCE   PMID:27230378\n",
      "AUTHORS   Ghssein G, Brutesco C, Ouerdane L, Fojcik C, Izaute A, Wang S, Hajjar C, Lobinski R, Lemaire D, Richaud P, Voulhoux R, Espaillat A, Cava F, Pignol D, Borezee-Durant E, Arnoux P\n",
      "TITLE     Biosynthesis of a broad-spectrum nicotianamine-like metallophore in Staphylococcus aureus.\n",
      "JOURNAL   Science 352:1105-9 (2016)\n",
      "DOI:10.1126/science.aaf1018\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00310  Lysine degradation\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00340  Histidine metabolism\n",
      "ath00360  Phenylalanine metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "KO_PATHWAY  ko00470\n",
      "///\n",
      "ENTRY       ath00480                    Pathway\n",
      "NAME        Glutathione metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of other amino acids\n",
      "PATHWAY_MAP ath00480  Glutathione metabolism\n",
      "MODULE      ath_M00118  Glutathione biosynthesis, glutamate => glutathione [PATH:ath00480]\n",
      "DBLINKS     GO: 0006749\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G39640  GGT1; gamma-glutamyl transpeptidase 1 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT4G39650  GGT2; gamma-glutamyl transpeptidase 2 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT1G69820  GGT3; gamma-glutamyl transpeptidase 3 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT4G29210  GGT4; gamma-glutamyl transpeptidase 4 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT5G26220  ChaC-like family protein [KO:K22596] [EC:4.3.2.9]\n",
      "AT4G31290  ChaC-like family protein [KO:K22596] [EC:4.3.2.9]\n",
      "AT1G44790  ChaC-like family protein [KO:K22596] [EC:4.3.2.9]\n",
      "AT5G37830  OXP1; oxoprolinase 1 [KO:K01469] [EC:3.5.2.9]\n",
      "AT4G23100  GSH1; glutamate-cysteine ligase [KO:K01919] [EC:6.3.2.2]\n",
      "AT5G27380  GSH2; glutathione synthetase 2 [KO:K21456] [EC:6.3.2.3]\n",
      "AT4G30920  LAP2; Cytosol aminopeptidase family protein [KO:K01255] [EC:3.4.11.1]\n",
      "AT2G24200  LAP1; Cytosol aminopeptidase family protein [KO:K01255] [EC:3.4.11.1]\n",
      "AT4G30910  Cytosol aminopeptidase family protein [KO:K01255] [EC:3.4.11.1]\n",
      "AT1G63770  Peptidase M1 family protein [KO:K01256] [EC:3.4.11.2]\n",
      "AT1G65820  microsomal glutathione s-transferase [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G30860  GSTF9; glutathione S-transferase PHI 9 [KO:K00799] [EC:2.5.1.18]\n",
      "AT3G03190  GSTF11; glutathione S-transferase F11 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G29440  GSTU6; glutathione S-transferase tau 6 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G29490  GSTU1; glutathione S-transferase TAU 1 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G29460  GSTU4; glutathione S-transferase tau 4 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G29480  GSTU2; glutathione S-transferase tau 2 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G47730  GSTF8; glutathione S-transferase phi 8 [KO:K00799] [EC:2.5.1.18]\n",
      "AT3G43800  GSTU27; glutathione S-transferase tau 27 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G02930  GSTF3; glutathione S-transferase F3 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G02940  GSTF5; glutathione S-transferase (class phi) 5 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G02950  GSTF4; glutathione S-transferase F4 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G02920  GSTF7; glutathione S-transferase 7 [KO:K00799] [EC:2.5.1.18]\n",
      "AT5G17220  GSTF12; glutathione S-transferase phi 12 [KO:K00799] [EC:2.5.1.18]\n",
      "AT4G02520  GSTF2; glutathione S-transferase PHI 2 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G10370  ERD9; Glutathione S-transferase family protein [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G10360  GSTU18; glutathione S-transferase TAU 18 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G02930  GSTF6; glutathione S-transferase 6 [KO:K00799] [EC:2.5.1.18]\n",
      "AT3G62760  ATGSTF13; Glutathione S-transferase family protein [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G49860  GSTF14; glutathione S-transferase (class phi) 14 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G59670  GSTU15; glutathione S-transferase TAU 15 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G29420  GSTU7; glutathione S-transferase tau 7 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G29470  GSTU3; glutathione S-transferase tau 3 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G59700  GSTU16; glutathione S-transferase TAU 16 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G29450  GSTU5; glutathione S-transferase tau 5 [KO:K00799] [EC:2.5.1.18]\n",
      "AT2G30870  GSTF10; glutathione S-transferase PHI 10 [KO:K00799] [EC:2.5.1.18]\n",
      "AT3G09270  GSTU8; glutathione S-transferase TAU 8 [KO:K00799] [EC:2.5.1.18]\n",
      "AT3G55040  GSTL2; glutathione transferase lambda 2 [KO:K00799] [EC:2.5.1.18]\n",
      "AT5G02780  GSTL1; glutathione transferase lambda 1 [KO:K00799] [EC:2.5.1.18]\n",
      "AT5G02790  GSTL3; Glutathione S-transferase family protein [KO:K00799] [EC:2.5.1.18]\n",
      "AT5G41210  GSTT1; glutathione S-transferase THETA 1 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G17170  GSTU24; glutathione S-transferase TAU 24 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G17180  GSTU25; glutathione S-transferase TAU 25 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G17190  GSTU26; glutathione S-transferase tau 26 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G27130  GSTU13; glutathione S-transferase tau 13 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G27140  GSTU14; glutathione S-transferase tau 14 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G53680  GSTU28; glutathione S-transferase TAU 28 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G69920  GSTU12; glutathione S-transferase TAU 12 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G69930  GSTU11; glutathione S-transferase TAU 11 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G74590  GSTU10; glutathione S-transferase TAU 10 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G78320  GSTU23; glutathione S-transferase TAU 23 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G78340  GSTU22; glutathione S-transferase TAU 22 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G78360  GSTU21; glutathione S-transferase TAU 21 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G78370  GSTU20; glutathione S-transferase TAU 20 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G78380  GSTU19; glutathione S-transferase TAU 19 [KO:K00799] [EC:2.5.1.18]\n",
      "AT5G62480  GSTU9; glutathione S-transferase tau 9 [KO:K00799] [EC:2.5.1.18]\n",
      "AT5G41240  GSTT2; glutathione S-transferase THETA 2 [KO:K00799] [EC:2.5.1.18]\n",
      "AT5G41220  GSTT3; glutathione S-transferase THETA 3 [KO:K00799] [EC:2.5.1.18]\n",
      "AT1G19550  Glutathione S-transferase family protein [KO:K21888] [EC:1.8.5.1 2.5.1.18]\n",
      "AT1G19570  DHAR1; dehydroascorbate reductase [KO:K21888] [EC:1.8.5.1 2.5.1.18]\n",
      "AT1G75270  DHAR2; dehydroascorbate reductase 2 [KO:K21888] [EC:1.8.5.1 2.5.1.18]\n",
      "AT5G16710  DHAR3; dehydroascorbate reductase 1 [KO:K21888] [EC:1.8.5.1 2.5.1.18]\n",
      "AT3G54660  GR; glutathione reductase [KO:K00383] [EC:1.8.1.7]\n",
      "AT3G24170  GR1; glutathione-disulfide reductase [KO:K00383] [EC:1.8.1.7]\n",
      "AT1G65930  cICDH; cytosolic NADP+-dependent isocitrate dehydrogenase [KO:K00031] [EC:1.1.1.42]\n",
      "AT1G54340  ICDH; isocitrate dehydrogenase [KO:K00031] [EC:1.1.1.42]\n",
      "AT5G14590  Isocitrate/isopropylmalate dehydrogenase family protein [KO:K00031] [EC:1.1.1.42]\n",
      "AT1G64190  6-phosphogluconate dehydrogenase family protein [KO:K00033] [EC:1.1.1.44 1.1.1.343]\n",
      "AT5G41670  6-phosphogluconate dehydrogenase family protein [KO:K00033] [EC:1.1.1.44 1.1.1.343]\n",
      "AT3G02360  6-phosphogluconate dehydrogenase family protein [KO:K00033] [EC:1.1.1.44 1.1.1.343]\n",
      "AT3G27300  G6PD5; glucose-6-phosphate dehydrogenase 5 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT5G13110  G6PD2; glucose-6-phosphate dehydrogenase 2 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT1G09420  G6PD4; glucose-6-phosphate dehydrogenase 4 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT5G35790  G6PD1; glucose-6-phosphate dehydrogenase 1 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT1G24280  G6PD3; glucose-6-phosphate dehydrogenase 3 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT5G40760  G6PD6; glucose-6-phosphate dehydrogenase 6 [KO:K00036] [EC:1.1.1.49 1.1.1.363]\n",
      "AT3G63080  GPX5; glutathione peroxidase 5 [KO:K00432] [EC:1.11.1.9]\n",
      "AT2G48150  GPX4; glutathione peroxidase 4 [KO:K00432] [EC:1.11.1.9]\n",
      "AT2G31570  GPX2; glutathione peroxidase 2 [KO:K00432] [EC:1.11.1.9]\n",
      "AT1G63460  GPX8; glutathione peroxidase 8 [KO:K00432] [EC:1.11.1.9]\n",
      "AT4G11600  GPX6; glutathione peroxidase 6 [KO:K00432] [EC:1.11.1.9]\n",
      "AT2G25080  GPX1; glutathione peroxidase 1 [KO:K00432] [EC:1.11.1.9]\n",
      "AT4G31870  GPX7; glutathione peroxidase 7 [KO:K00432] [EC:1.11.1.9]\n",
      "AT2G43350  GPX3; glutathione peroxidase 3 [KO:K00432] [EC:1.11.1.9]\n",
      "AT1G48130  PER1; 1-cysteine peroxiredoxin 1 [KO:K11188] [EC:1.11.1.7 1.11.1.27 3.1.1.-]\n",
      "AT1G23820  SPDS1; spermidine synthase [KO:K00797] [EC:2.5.1.16]\n",
      "AT1G70310  SPDS2; spermidine synthase 2 [KO:K00797] [EC:2.5.1.16]\n",
      "AT5G53120  SPDS3; spermidine synthase 3 [KO:K00797] [EC:2.5.1.16]\n",
      "AT1G07890  APX1; ascorbate peroxidase 1 [KO:K00434] [EC:1.11.1.11]\n",
      "AT3G09640  APX2; ascorbate peroxidase 2 [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G35000  APX3; ascorbate peroxidase 3 [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G35970  APX5; ascorbate peroxidase 5 [KO:K00434] [EC:1.11.1.11]\n",
      "AT1G77490  TAPX; thylakoidal ascorbate peroxidase [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G08390  SAPX; stromal ascorbate peroxidase [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G09010  TL29; ascorbate peroxidase 4 [KO:K00434] [EC:1.11.1.11]\n",
      "AT4G32320  APX6; ascorbate peroxidase 6 [KO:K00434] [EC:1.11.1.11]\n",
      "AT2G21790  RNR1; ribonucleotide reductase 1 [KO:K10807] [EC:1.17.4.1]\n",
      "AT3G27060  TSO2; Ferritin/ribonucleotide reductase-like family protein [KO:K10808] [EC:1.17.4.1]\n",
      "AT3G23580  RNR2A; ribonucleotide reductase 2A [KO:K10808] [EC:1.17.4.1]\n",
      "COMPOUND    C00005  NADPH\n",
      "C00006  NADP+\n",
      "C00024  Acetyl-CoA\n",
      "C00025  L-Glutamate\n",
      "C00037  Glycine\n",
      "C00051  Glutathione\n",
      "C00072  Ascorbate\n",
      "C00077  L-Ornithine\n",
      "C00097  L-Cysteine\n",
      "C00127  Glutathione disulfide\n",
      "C00134  Putrescine\n",
      "C00151  L-Amino acid\n",
      "C00315  Spermidine\n",
      "C00669  gamma-L-Glutamyl-L-cysteine\n",
      "C00750  Spermine\n",
      "C01322  RX\n",
      "C01419  Cys-Gly\n",
      "C01672  Cadaverine\n",
      "C01879  5-Oxoproline\n",
      "C02090  Trypanothione\n",
      "C02320  R-S-Glutathione\n",
      "C03170  Trypanothione disulfide\n",
      "C03646  Bis-gamma-glutamylcystine\n",
      "C03740  (5-L-Glutamyl)-L-amino acid\n",
      "C05422  Dehydroascorbate\n",
      "C05726  S-Substituted L-cysteine\n",
      "C05727  S-Substituted N-acetyl-L-cysteine\n",
      "C05729  R-S-Cysteinylglycine\n",
      "C05730  Glutathionylspermidine\n",
      "C16562  Glutathionylspermine\n",
      "C16563  Bis(glutathionyl)spermine\n",
      "C16564  Bis(glutathionyl)spermine disulfide\n",
      "C16565  Aminopropylcadaverine\n",
      "C16566  Glutathionylaminopropylcadaverine\n",
      "C16567  Homotrypanothione\n",
      "C16568  Homotrypanothione disulfide\n",
      "C16663  Tryparedoxin\n",
      "C16664  Tryparedoxin disulfide\n",
      "REFERENCE   PMID:12675513\n",
      "AUTHORS   Josch C, Klotz LO, Sies H.\n",
      "TITLE     Identification of cytosolic leucyl aminopeptidase (EC 3.4.11.1) as the major cysteinylglycine-hydrolysing activity in rat liver.\n",
      "JOURNAL   Biol Chem 384:213-8 (2003)\n",
      "DOI:10.1515/BC.2003.023\n",
      "REFERENCE   PMID:18482986\n",
      "AUTHORS   Chu L, Lai Y, Xu X, Eddy S, Yang S, Song L, Kolodrubetz D.\n",
      "TITLE     A 52-kDa leucyl aminopeptidase from treponema denticola is a cysteinylglycinase that mediates the second step of glutathione metabolism.\n",
      "JOURNAL   J Biol Chem 283:19351-8 (2008)\n",
      "DOI:10.1074/jbc.M801034200\n",
      "REFERENCE   PMID:14583094\n",
      "AUTHORS   Cappiello M, Lazzarotti A, Buono F, Scaloni A, D'Ambrosio C, Amodeo P, Mendez BL, Pelosi P, Del Corso A, Mura U.\n",
      "TITLE     New role for leucyl aminopeptidase in glutathione turnover.\n",
      "JOURNAL   Biochem J 378:35-44 (2004)\n",
      "DOI:10.1042/BJ20031336\n",
      "REFERENCE   PMID:11157967\n",
      "AUTHORS   Suzuki H, Kamatani S, Kim ES, Kumagai H.\n",
      "TITLE     Aminopeptidases A, B, and N and dipeptidase D are the four cysteinylglycinases of Escherichia coli K-12.\n",
      "JOURNAL   J Bacteriol 183:1489-90 (2001)\n",
      "DOI:10.1128/JB.183.4.1489-1490.2001\n",
      "REFERENCE   PMID:15610825\n",
      "AUTHORS   Oza SL, Shaw MP, Wyllie S, Fairlamb AH.\n",
      "TITLE     Trypanothione biosynthesis in Leishmania major.\n",
      "JOURNAL   Mol Biochem Parasitol 139:107-16 (2005)\n",
      "DOI:10.1016/j.molbiopara.2004.10.004\n",
      "REFERENCE   PMID:12049631\n",
      "AUTHORS   Oza SL, Ariyanayagam MR, Fairlamb AH.\n",
      "TITLE     Characterization of recombinant glutathionylspermidine synthetase/amidase from Crithidia fasciculata.\n",
      "JOURNAL   Biochem J 364:679-86 (2002)\n",
      "DOI:10.1042/BJ20011370\n",
      "REFERENCE   PMID:9677355\n",
      "AUTHORS   Tetaud E, Manai F, Barrett MP, Nadeau K, Walsh CT, Fairlamb AH.\n",
      "TITLE     Cloning and characterization of the two enzymes responsible for trypanothione biosynthesis in Crithidia fasciculata.\n",
      "JOURNAL   J Biol Chem 273:19383-90 (1998)\n",
      "DOI:10.1074/jbc.273.31.19383\n",
      "REFERENCE   PMID:12750367\n",
      "AUTHORS   Ariyanayagam MR, Oza SL, Mehlert A, Fairlamb AH.\n",
      "TITLE     Bis(glutathionyl)spermine and other novel trypanothione analogues in Trypanosoma cruzi.\n",
      "JOURNAL   J Biol Chem 278:27612-9 (2003)\n",
      "DOI:10.1074/jbc.M302750200\n",
      "REFERENCE   PMID:12967709\n",
      "AUTHORS   Oza SL, Ariyanayagam MR, Aitcheson N, Fairlamb AH.\n",
      "TITLE     Properties of trypanothione synthetase from Trypanosoma brucei.\n",
      "JOURNAL   Mol Biochem Parasitol 131:25-33 (2003)\n",
      "DOI:10.1016/S0166-6851(03)00176-2\n",
      "REFERENCE   PMID:12121990\n",
      "AUTHORS   Oza SL, Tetaud E, Ariyanayagam MR, Warnon SS, Fairlamb AH.\n",
      "TITLE     A single enzyme catalyses formation of Trypanothione from glutathione and spermidine in Trypanosoma cruzi.\n",
      "JOURNAL   J Biol Chem 277:35853-61 (2002)\n",
      "DOI:10.1074/jbc.M204403200\n",
      "REFERENCE   PMID:15537651\n",
      "AUTHORS   Comini M, Menge U, Wissing J, Flohe L.\n",
      "TITLE     Trypanothione synthesis in crithidia revisited.\n",
      "JOURNAL   J Biol Chem 280:6850-60 (2005)\n",
      "DOI:10.1074/jbc.M404486200\n",
      "REFERENCE   PMID:7813456\n",
      "AUTHORS   Hunter KJ, Le Quesne SA, Fairlamb AH.\n",
      "TITLE     Identification and biosynthesis of N1,N9-bis(glutathionyl)aminopropylcadaverine (homotrypanothione) in Trypanosoma cruzi.\n",
      "JOURNAL   Eur J Biochem 226:1019-27 (1994)\n",
      "DOI:10.1111/j.1432-1033.1994.t01-1-01019.x\n",
      "REFERENCE   PMID:12751784\n",
      "AUTHORS   Krauth-Siegel RL, Meiering SK, Schmidt H.\n",
      "TITLE     The parasite-specific trypanothione metabolism of trypanosoma and leishmania.\n",
      "JOURNAL   Biol Chem 384:539-49 (2003)\n",
      "DOI:10.1515/BC.2003.062\n",
      "REFERENCE   PMID:18395526\n",
      "AUTHORS   Krauth-Siegel RL, Comini MA.\n",
      "TITLE     Redox control in trypanosomatids, parasitic protozoa with trypanothione-based thiol metabolism.\n",
      "JOURNAL   Biochim Biophys Acta 1780:1236-48 (2008)\n",
      "DOI:10.1016/j.bbagen.2008.03.006\n",
      "REFERENCE   PMID:8892297\n",
      "AUTHORS   Krauth-Siegel RL, Ludemann H.\n",
      "TITLE     Reduction of dehydroascorbate by trypanothione.\n",
      "JOURNAL   Mol Biochem Parasitol 80:203-8 (1996)\n",
      "DOI:10.1016/0166-6851(96)02689-8\n",
      "REFERENCE   PMID:11150302\n",
      "AUTHORS   Dormeyer M, Reckenfelderbaumer N, Ludemann H, Krauth-Siegel RL.\n",
      "TITLE     Trypanothione-dependent synthesis of deoxyribonucleotides by Trypanosoma brucei ribonucleotide reductase.\n",
      "JOURNAL   J Biol Chem 276:10602-6 (2001)\n",
      "DOI:10.1074/jbc.M010352200\n",
      "REFERENCE   PMID:12949079\n",
      "AUTHORS   Schmidt H, Krauth-Siegel RL.\n",
      "TITLE     Functional and physicochemical characterization of the thioredoxin system in Trypanosoma brucei.\n",
      "JOURNAL   J Biol Chem 278:46329-36 (2003)\n",
      "DOI:10.1074/jbc.M305338200\n",
      "REFERENCE   PMID:9851611\n",
      "AUTHORS   Tetaud E, Fairlamb AH.\n",
      "TITLE     Cloning, expression and reconstitution of the trypanothione-dependent peroxidase system of Crithidia fasciculata.\n",
      "JOURNAL   Mol Biochem Parasitol 96:111-23 (1998)\n",
      "DOI:10.1016/S0166-6851(98)00120-0\n",
      "REFERENCE   PMID:12446213\n",
      "AUTHORS   Castro H, Sousa C, Santos M, Cordeiro-da-Silva A, Flohe L, Tomas AM.\n",
      "TITLE     Complementary antioxidant defense by cytoplasmic and mitochondrial peroxiredoxins in Leishmania infantum.\n",
      "JOURNAL   Free Radic Biol Med 33:1552-62 (2002)\n",
      "DOI:10.1016/S0891-5849(02)01089-4\n",
      "REFERENCE   PMID:10713147\n",
      "AUTHORS   Wilkinson SR, Temperton NJ, Mondragon A, Kelly JM.\n",
      "TITLE     Distinct mitochondrial and cytosolic enzymes mediate trypanothione-dependent peroxide metabolism in Trypanosoma cruzi.\n",
      "JOURNAL   J Biol Chem 275:8220-5 (2000)\n",
      "DOI:10.1074/jbc.275.11.8220\n",
      "REFERENCE   PMID:17922848\n",
      "AUTHORS   Konig J, Fairlamb AH.\n",
      "TITLE     A comparative study of type I and type II tryparedoxin peroxidases in Leishmania major.\n",
      "JOURNAL   FEBS J 274:5643-58 (2007)\n",
      "DOI:10.1111/j.1742-4658.2007.06087.x\n",
      "REFERENCE   PMID:12466271\n",
      "AUTHORS   Hillebrand H, Schmidt A, Krauth-Siegel RL.\n",
      "TITLE     A second class of peroxidases linked to the trypanothione metabolism.\n",
      "JOURNAL   J Biol Chem 278:6809-15 (2003)\n",
      "DOI:10.1074/jbc.M210392200\n",
      "REFERENCE   PMID:14982633\n",
      "AUTHORS   Soksawatmaekhin W, Kuraishi A, Sakata K, Kashiwagi K, Igarashi K.\n",
      "TITLE     Excretion and uptake of cadaverine by CadB and its physiological functions in Escherichia coli.\n",
      "JOURNAL   Mol Microbiol 51:1401-12 (2004)\n",
      "DOI:10.1046/j.1365-2958.2003.03913.x\n",
      "REFERENCE   PMID:6798961\n",
      "AUTHORS   Pegg AE, Shuttleworth K, Hibasami H.\n",
      "TITLE     Specificity of mammalian spermidine synthase and spermine synthase.\n",
      "JOURNAL   Biochem J 197:315-20 (1981)\n",
      "DOI:10.1042/bj1970315\n",
      "REL_PATHWAY ath00220  Arginine biosynthesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00430  Taurine and hypotaurine metabolism\n",
      "ath00460  Cyanoamino acid metabolism\n",
      "KO_PATHWAY  ko00480\n",
      "///\n",
      "ENTRY       ath00500                    Pathway\n",
      "NAME        Starch and sucrose metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00500  Starch and sucrose metabolism\n",
      "MODULE      ath_M00854  Glycogen biosynthesis, glucose-1P => glycogen/starch [PATH:ath00500]\n",
      "DBLINKS     GO: 0005982 0005985\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G36190  cwINV4; cell wall invertase 4 [KO:K01193] [EC:3.2.1.26]\n",
      "AT3G13790  ATBFRUCT1; Glycosyl hydrolases family 32 protein [KO:K01193] [EC:3.2.1.26]\n",
      "AT1G12240  ATBETAFRUCT4; Glycosyl hydrolases family 32 protein [KO:K01193] [EC:3.2.1.26]\n",
      "AT1G62660  Glycosyl hydrolases family 32 protein [KO:K01193] [EC:3.2.1.26]\n",
      "AT3G13784  CWINV5; cell wall invertase 5 [KO:K01193] [EC:3.2.1.26]\n",
      "AT5G11920  cwINV6; 6-&1-fructan exohydrolase [KO:K01193] [EC:3.2.1.26]\n",
      "AT3G52600  CWINV2; cell wall invertase 2 [KO:K01193] [EC:3.2.1.26]\n",
      "AT1G55120  FRUCT5; beta-fructofuranosidase 5 [KO:K01193] [EC:3.2.1.26]\n",
      "AT5G20280  SPS1F; sucrose phosphate synthase 1F [KO:K00696] [EC:2.4.1.14]\n",
      "AT1G04920  SPS3F; sucrose phosphate synthase 3F [KO:K00696] [EC:2.4.1.14]\n",
      "AT4G10120  ATSPS4F; Sucrose-phosphate synthase family protein [KO:K00696] [EC:2.4.1.14]\n",
      "AT5G11110  SPS2F; sucrose phosphate synthase 2F [KO:K00696] [EC:2.4.1.14]\n",
      "AT1G51420  SPP1; sucrose-phosphatase 1 [KO:K07024] [EC:3.1.3.24]\n",
      "AT3G52340  SPP2; sucrose-6F-phosphate phosphohydrolase 2 [KO:K07024] [EC:3.1.3.24]\n",
      "AT2G35840  Sucrose-6F-phosphate phosphohydrolase family protein [KO:K07024] [EC:3.1.3.24]\n",
      "AT3G54270  sucrose-6F-phosphate phosphohydrolase family protein [KO:K07024] [EC:3.1.3.24]\n",
      "AT3G43190  SUS4; sucrose synthase 4 [KO:K00695] [EC:2.4.1.13]\n",
      "AT5G20830  SUS1; sucrose synthase 1 [KO:K00695] [EC:2.4.1.13]\n",
      "AT5G49190  SUS2; sucrose synthase 2 [KO:K00695] [EC:2.4.1.13]\n",
      "AT1G73370  SUS6; sucrose synthase 6 [KO:K00695] [EC:2.4.1.13]\n",
      "AT4G02280  SUS3; sucrose synthase 3 [KO:K00695] [EC:2.4.1.13]\n",
      "AT5G37180  SUS5; sucrose synthase 5 [KO:K00695] [EC:2.4.1.13]\n",
      "AT5G11720  Glycosyl hydrolases family 31 protein [KO:K01187] [EC:3.2.1.20]\n",
      "AT5G17310  UGP2; UDP-glucose pyrophosphorylase 2 [KO:K00963] [EC:2.7.7.9]\n",
      "AT3G03250  UGP1; UDP-GLUCOSE PYROPHOSPHORYLASE 1 [KO:K00963] [EC:2.7.7.9]\n",
      "AT4G29680  Alkaline-phosphatase-like family protein [KO:K01513] [EC:3.1.4.1 3.6.1.9]\n",
      "AT1G11820  O-Glycosyl hydrolases family 17 protein [KO:K19891] [EC:3.2.1.39]\n",
      "AT1G66250  O-Glycosyl hydrolases family 17 protein [KO:K19891] [EC:3.2.1.39]\n",
      "AT2G01630  O-Glycosyl hydrolases family 17 protein [KO:K19891] [EC:3.2.1.39]\n",
      "AT3G13560  O-Glycosyl hydrolases family 17 protein [KO:K19892] [EC:3.2.1.39]\n",
      "AT5G58090  O-Glycosyl hydrolases family 17 protein [KO:K19893] [EC:3.2.1.39]\n",
      "AT4G31140  O-Glycosyl hydrolases family 17 protein [KO:K19893] [EC:3.2.1.39]\n",
      "AT5G42260  BGLU12; beta glucosidase 12 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G44640  BGLU13; beta glucosidase 13 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44450  BGLU15; beta glucosidase 15 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G26560  BGLU40; beta glucosidase 40 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44480  BGLU17; beta glucosidase 17 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G47050  Glycosyl hydrolase family protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G75940  ATA27; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G66280  BGLU22; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G66270  BGLU21; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G09260  PYK10; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G28510  BGLU24; beta glucosidase 24 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G36890  BGLU42; beta glucosidase 42 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G54570  BGLU41; beta glucosidase 41 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G32860  BGLU33; beta glucosidase 33 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G03640  BGLU25; beta glucosidase 25 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G21370  BGLU19; beta glucosidase 19 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60130  BGLU16; beta glucosidase 16 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G25630  BGLU14; beta glucosidase 14 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60140  DIN2; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44470  BGLU29; beta glucosidase 29 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44460  BGLU28; beta glucosidase 28 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44490  PEN2; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G24550  BGLU32; beta glucosidase 32 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G24540  BGLU31; beta glucosidase 31 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60120  BGLU27; beta glucosidase 27 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G02850  BGLU11; beta glucosidase 11 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G45191  BGLU1; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G22100  BGLU3; beta glucosidase 2 [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G27830  BGLU10; beta glucosidase 10 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G62750  BGLU8; beta glucosidase 8 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G16580  BGLU2; beta glucosidase 2 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G60270  BGLU6; beta glucosidase 6 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G62740  BGLU7; beta glucosidase 7 [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G27820  BGLU9; beta glucosidase 9 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G60260  BGLU5; beta glucosidase 5 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G47000  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G20950  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G20940  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G47040  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G04885  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G47010  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G18080  BGLU44; B-S glucosidase 44 [KO:K05350] [EC:3.2.1.21]\n",
      "AT1G61820  BGLU46; beta glucosidase 46 [KO:K05350] [EC:3.2.1.21]\n",
      "AT4G21760  BGLU47; beta-glucosidase 47 [KO:K05350] [EC:3.2.1.21]\n",
      "AT1G61810  BGLU45; beta-glucosidase 45 [KO:K05350] [EC:3.2.1.21]\n",
      "AT3G18070  BGLU43; beta glucosidase 43 [KO:K05350] [EC:3.2.1.21]\n",
      "AT3G43860  GH9A4; glycosyl hydrolase 9A4 [KO:K01179] [EC:3.2.1.4]\n",
      "AT4G39000  GH9B17; glycosyl hydrolase 9B17 [KO:K01179] [EC:3.2.1.4]\n",
      "AT4G39010  GH9B18; glycosyl hydrolase 9B18 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G65610  KOR2; Six-hairpin glycosidases superfamily protein [KO:K01179] [EC:3.2.1.4]\n",
      "AT2G44560  GH9B11; glycosyl hydrolase 9B11 [KO:K01179] [EC:3.2.1.4]\n",
      "AT2G44570  GH9B12; glycosyl hydrolase 9B12 [KO:K01179] [EC:3.2.1.4]\n",
      "AT2G44540  GH9B9; glycosyl hydrolase 9B9 [KO:K01179] [EC:3.2.1.4]\n",
      "AT2G44550  GH9B10; glycosyl hydrolase 9B10 [KO:K01179] [EC:3.2.1.4]\n",
      "AT4G24260  GH9A3; glycosyl hydrolase 9A3 [KO:K01179] [EC:3.2.1.4]\n",
      "AT5G49720  GH9A1; glycosyl hydrolase 9A1 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G22880  CEL5; cellulase 5 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G23210  GH9B6; glycosyl hydrolase 9B6 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G70710  GH9B1; glycosyl hydrolase 9B1 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G71380  CEL3; cellulase 3 [KO:K01179] [EC:3.2.1.4]\n",
      "AT4G38990  GH9B16; glycosyl hydrolase 9B16 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G64390  GH9C2; glycosyl hydrolase 9C2 [KO:K01179] [EC:3.2.1.4]\n",
      "AT4G11050  GH9C3; glycosyl hydrolase 9C3 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G48930  GH9C1; glycosyl hydrolase 9C1 [KO:K01179] [EC:3.2.1.4]\n",
      "AT4G02290  GH9B13; glycosyl hydrolase 9B13 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G02800  CEL2; cellulase 2 [KO:K01179] [EC:3.2.1.4]\n",
      "AT1G27680  APL2; ADPGLC-PPase large subunit [KO:K00975] [EC:2.7.7.27]\n",
      "AT5G48300  ADG1; ADP glucose pyrophosphorylase 1 [KO:K00975] [EC:2.7.7.27]\n",
      "AT1G05610  APS2; ADP-glucose pyrophosphorylase small subunit 2 [KO:K00975] [EC:2.7.7.27]\n",
      "AT4G39210  APL3; Glucose-1-phosphate adenylyltransferase family protein [KO:K00975] [EC:2.7.7.27]\n",
      "AT5G19220  APL1; ADP glucose pyrophosphorylase large subunit 1 [KO:K00975] [EC:2.7.7.27]\n",
      "AT2G21590  APL4; Glucose-1-phosphate adenylyltransferase family protein [KO:K00975] [EC:2.7.7.27]\n",
      "AT4G11980  NUDX14; nudix hydrolase homolog 14 [KO:K18447] [EC:3.6.1.21]\n",
      "AT5G24300  SS1; Glycogen/starch synthases, ADP-glucose type [KO:K00703] [EC:2.4.1.21]\n",
      "AT3G01180  SS2; starch synthase 2 [KO:K00703] [EC:2.4.1.21]\n",
      "AT1G11720  SS3; starch synthase 3 [KO:K00703] [EC:2.4.1.21]\n",
      "AT4G18240  SS4; starch synthase 4 [KO:K00703] [EC:2.4.1.21]\n",
      "AT1G32900  GBSS1; UDP-Glycosyltransferase superfamily protein [KO:K13679] [EC:2.4.1.242]\n",
      "AT5G03650  SBE2.2; starch branching enzyme 2.2 [KO:K00700] [EC:2.4.1.18]\n",
      "AT2G36390  SBE2.1; starch branching enzyme 2.1 [KO:K00700] [EC:2.4.1.18]\n",
      "AT3G20440  EMB2729; Alpha amylase family protein [KO:K00700] [EC:2.4.1.18]\n",
      "AT3G29320  PHS1; Glycosyl transferase, family 35 [KO:K00688] [EC:2.4.1.1]\n",
      "AT3G46970  PHS2; alpha-glucan phosphorylase 2 [KO:K00688] [EC:2.4.1.1]\n",
      "AT4G25000  AMY1; alpha-amylase-like protein [KO:K01176] [EC:3.2.1.1]\n",
      "AT1G76130  AMY2; alpha-amylase-like 2 [KO:K01176] [EC:3.2.1.1]\n",
      "AT1G69830  AMY3; alpha-amylase-like 3 [KO:K01176] [EC:3.2.1.1]\n",
      "AT4G17090  CT-BMY; chloroplast beta-amylase [KO:K01177] [EC:3.2.1.2]\n",
      "AT3G23920  BAM1; beta-amylase 1 [KO:K01177] [EC:3.2.1.2]\n",
      "AT4G00490  BAM2; beta-amylase 2 [KO:K01177] [EC:3.2.1.2]\n",
      "AT2G45880  BAM7; beta-amylase 7 [KO:K01177] [EC:3.2.1.2]\n",
      "AT5G45300  BMY2; beta-amylase 2 [KO:K01177] [EC:3.2.1.2]\n",
      "AT4G15210  BAM5; beta-amylase 5 [KO:K01177] [EC:3.2.1.2]\n",
      "AT2G32290  BAM6; beta-amylase 6 [KO:K01177] [EC:3.2.1.2]\n",
      "AT2G40840  DPE2; disproportionating enzyme 2 [KO:K00705] [EC:2.4.1.25]\n",
      "AT5G64860  DPE1; disproportionating enzyme [KO:K00705] [EC:2.4.1.25]\n",
      "AT2G39930  ISA1; isoamylase 1 [KO:K01214] [EC:3.2.1.68]\n",
      "AT4G09020  ISA3; isoamylase 3 [KO:K01214] [EC:3.2.1.68]\n",
      "AT1G03310  DBE1; debranching enzyme 1 [KO:K01214] [EC:3.2.1.68]\n",
      "AT1G78090  TPPB; trehalose-6-phosphate phosphatase [KO:K01087] [EC:3.1.3.12]\n",
      "AT2G22190  TPPE; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT5G65140  TPPJ; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT1G22210  TPPC; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT1G35910  TPPD; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT4G22590  TPPG; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT5G10100  TPPI; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT4G12430  TPPF; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT4G39770  TPPH; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT5G51460  ATTPPA; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K01087] [EC:3.1.3.12]\n",
      "AT1G70290  TPS8; trehalose-6-phosphatase synthase S8 [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT4G17770  TPS5; trehalose phosphatase/synthase 5 [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT1G23870  TPS9; trehalose-phosphatase/synthase 9 [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT2G18700  TPS11; trehalose phosphatase/synthase 11 [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT1G16980  TPS2; trehalose-6-phosphate synthase [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT1G17000  TPS3; trehalose-phosphatase/synthase 3 [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT1G68020  ATTPS6; UDP-Glycosyltransferase / trehalose-phosphatase family protein [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT1G06410  TPS7; trehalose-phosphatase/synthase 7 [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT1G78580  TPS1; trehalose-6-phosphate synthase [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT1G60140  TPS10; trehalose phosphate synthase [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT4G27550  TPS4; trehalose-6-phosphatase synthase S4 [KO:K16055] [EC:2.4.1.15 3.1.3.12]\n",
      "AT4G24040  TRE1; trehalase 1 [KO:K01194] [EC:3.2.1.28]\n",
      "AT5G51820  PGM; phosphoglucomutase [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G23190  PGM3; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G70730  PGM2; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT4G37840  HKL3; hexokinase-like 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G47840  HXK3; hexokinase 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT2G19860  HXK2; hexokinase 2 [KO:K00844] [EC:2.7.1.1]\n",
      "AT4G29130  HXK1; hexokinase 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G50460  HKL1; hexokinase-like 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT3G20040  ATHXK4; Hexokinase [KO:K00844] [EC:2.7.1.1]\n",
      "AT4G24620  PGI1; phosphoglucose isomerase 1 [KO:K01810] [EC:5.3.1.9]\n",
      "AT5G42740  Sugar isomerase (SIS) family protein [KO:K01810] [EC:5.3.1.9]\n",
      "AT5G51830  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT2G31390  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT4G10260  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G06030  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT3G59480  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G66430  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G06020  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G50390  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "COMPOUND    C00029  UDP-glucose\n",
      "C00031  D-Glucose\n",
      "C00085  D-Fructose 6-phosphate\n",
      "C00089  Sucrose\n",
      "C00092  D-Glucose 6-phosphate\n",
      "C00095  D-Fructose\n",
      "C00103  D-Glucose 1-phosphate\n",
      "C00185  Cellobiose\n",
      "C00208  Maltose\n",
      "C00252  Isomaltose\n",
      "C00369  Starch\n",
      "C00372  Dextran\n",
      "C00394  GDP-glucose\n",
      "C00498  ADP-glucose\n",
      "C00501  CDP-glucose\n",
      "C00663  beta-D-Glucose 1-phosphate\n",
      "C00689  alpha,alpha'-Trehalose 6-phosphate\n",
      "C00718  Amylose\n",
      "C00721  Dextrin\n",
      "C00760  Cellulose\n",
      "C00963  beta-D-Glucoside\n",
      "C00965  1,3-beta-D-Glucan\n",
      "C00973  Cyclomaltodextrin\n",
      "C01083  alpha,alpha-Trehalose\n",
      "C01231  alpha-D-Glucose 1,6-bisphosphate\n",
      "C01725  Levanbiose\n",
      "C01898  Cellodextrin\n",
      "C01935  Maltodextrin\n",
      "C02591  Sucrose 6'-phosphate\n",
      "C02995  Maltose 6'-phosphate\n",
      "C03323  (2,1-beta-D-Fructosyl)n\n",
      "C04534  6-Phospho-beta-D-glucosyl-(1,4)-D-glucose\n",
      "C05731  3-Ketosucrose\n",
      "C06215  Levan\n",
      "C06400  1-alpha-D-[(1->4)-alpha-D-Glucosyl](n-1)-alpha-D-glucopyranoside\n",
      "C16688  Sucrose 6-phosphate\n",
      "C20237  alpha-Maltose 1-phosphate\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "KO_PATHWAY  ko00500\n",
      "///\n",
      "ENTRY       ath00510                    Pathway\n",
      "NAME        N-Glycan biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION N-glycans or asparagine-linked glycans are major constituents of glycoproteins in eukaryotes. N-glycans are covalently attached to asparagine with the consensus sequence of Asn-X-Ser/Thr by an N-glycosidic bond, GlcNAc b1- Asn. Biosynthesis of N-glycans begins on the cytoplasmic face of the ER membrane with the transferase reaction of UDP-GlcNAc and the lipid-like precursor P-Dol (dolichol phosphate) to generate GlcNAc a1- PP-Dol. After sequential addition of monosaccharides by ALG glycosyltransferases [MD:M00055], the N-glycan precursor is attached by the OST (oligosaccharyltransferase) complex to the polypeptide chain that is being synthesized and translocated through the ER membrane. The protein-bound N-glycan precursor is subsequently trimmed, extended, and modified in the ER and Golgi by a complex series of reactions catalyzed by membrane-bound glycosidases and glycosyltransferases. N-glycans thus synthesized are classified into three types: high-mannose type, complex type, and hybrid type. Defects in N-glycan biosynthesis lead to a variety of human diseases known as congenital disorders of glycosylation [DS:H00118 H00119].\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00510  N-Glycan biosynthesis\n",
      "MODULE      ath_M00055  N-glycan precursor biosynthesis [PATH:ath00510]\n",
      "ath_M00072  N-glycosylation by oligosaccharyltransferase [PATH:ath00510]\n",
      "ath_M00073  N-glycan precursor trimming [PATH:ath00510]\n",
      "DBLINKS     GO: 0006487\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G16530  3-oxo-5-alpha-steroid 4-dehydrogenase family protein [KO:K12345] [EC:1.3.1.22 1.3.1.94]\n",
      "AT1G72590  3-oxo-5-alpha-steroid 4-dehydrogenase family protein [KO:K12345] [EC:1.3.1.22 1.3.1.94]\n",
      "AT3G45040  phosphatidate cytidylyltransferase family protein [KO:K00902] [EC:2.7.1.108]\n",
      "AT2G41490  GPT; UDP-glcnac-adolichol phosphate glcnac-1-p-transferase [KO:K01001] [EC:2.7.8.15]\n",
      "AT3G57220  Glycosyl transferase family 4 protein [KO:K01001] [EC:2.7.8.15]\n",
      "AT2G39630  Nucleotide-diphospho-sugar transferases superfamily protein [KO:K00729] [EC:2.4.1.117]\n",
      "AT4G16710  glycosyltransferase family protein 28 [KO:K07432] [EC:2.4.1.141]\n",
      "AT4G18230  UDP-N-acetylglucosamine transferase subunit ALG14-like protein [KO:K07441] [EC:2.4.1.141]\n",
      "AT1G20575  DPMS1; Nucleotide-diphospho-sugar transferases superfamily protein [KO:K00721] [EC:2.4.1.83]\n",
      "AT1G74340  DPMS2; dolichol phosphate-mannose biosynthesis regulatory protein-like protein [KO:K09658]\n",
      "AT1G48140  DPMS3; dolichol-phosphate mannosyltransferase-like protein [KO:K09659]\n",
      "AT1G16570  UDP-Glycosyltransferase superfamily protein [KO:K03842] [EC:2.4.1.142]\n",
      "AT1G78800  UDP-Glycosyltransferase superfamily protein [KO:K03843] [EC:2.4.1.132 2.4.1.257]\n",
      "AT2G40190  LEW3; UDP-Glycosyltransferase superfamily protein [KO:K03844] [EC:2.4.1.131]\n",
      "AT2G47760  ALG3; asparagine-linked glycosylation 3 [KO:K03845] [EC:2.4.1.258]\n",
      "AT1G16900  EBS3; Alg9-like mannosyltransferase family [KO:K03846] [EC:2.4.1.259 2.4.1.261]\n",
      "AT1G02145  ALG12; homolog of asparagine-linked glycosylation 12 [KO:K03847] [EC:2.4.1.260]\n",
      "AT5G38460  ALG6, ALG8 glycosyltransferase family [KO:K03848] [EC:2.4.1.267]\n",
      "AT2G44660  ALG6, ALG8 glycosyltransferase family [KO:K03849] [EC:2.4.1.265]\n",
      "AT5G02410  ALG10; DIE2/ALG10 family [KO:K03850] [EC:2.4.1.256]\n",
      "AT1G34130  STT3B; staurosporin and temperature sensitive 3-like b [KO:K07151] [EC:2.4.99.18]\n",
      "AT5G19690  STT3A; staurosporin and temperature sensitive 3-like A [KO:K07151] [EC:2.4.99.18]\n",
      "AT1G76400  Ribophorin I [KO:K12666]\n",
      "AT2G01720  Ribophorin I [KO:K12666]\n",
      "AT4G21150  HAP6; ribophorin II (RPN2) family protein [KO:K12667]\n",
      "AT1G32210  ATDAD1; Defender against death (DAD family) protein [KO:K12668]\n",
      "AT2G35520  DAD2; Defender against death (DAD family) protein [KO:K12668]\n",
      "AT1G61790  Oligosaccharyltransferase complex/magnesium transporter family protein [KO:K12669]\n",
      "AT1G11560  Oligosaccharyltransferase complex/magnesium transporter family protein [KO:K12669]\n",
      "AT5G66680  DGL1; dolichyl-diphosphooligosaccharide-protein glycosyltransferase 48kDa subunit family protein [KO:K12670]\n",
      "AT5G03080  LPPgamma; Phosphatidic acid phosphatase (PAP2) family protein [KO:K07252] [EC:3.6.1.43]\n",
      "AT1G24320  Six-hairpin glycosidases superfamily protein [KO:K01228] [EC:3.2.1.106]\n",
      "AT1G67490  GCS1; glucosidase 1 [KO:K01228] [EC:3.2.1.106]\n",
      "AT5G63840  RSW3; Glycosyl hydrolases family 31 protein [KO:K05546] [EC:3.2.1.207]\n",
      "AT1G30000  MNS3; alpha-mannosidase 3 [KO:K23741] [EC:3.2.1.209]\n",
      "AT3G21160  MNS2; alpha-mannosidase 2 [KO:K01230] [EC:3.2.1.113]\n",
      "AT1G51590  MNS1; alpha-mannosidase 1 [KO:K01230] [EC:3.2.1.113]\n",
      "AT4G38240  CGL1; alpha-1,3-mannosyl-glycoprotein beta-1,2-N-acetylglucosaminyltransferase [KO:K00726] [EC:2.4.1.101]\n",
      "AT5G14950  GMII; golgi alpha-mannosidase II [KO:K01231] [EC:3.2.1.114]\n",
      "AT2G05320  beta-1,2-N-acetylglucosaminyltransferase II [KO:K00736] [EC:2.4.1.143]\n",
      "AT2G13290  beta-1,4-N-acetylglucosaminyltransferase family protein [KO:K00737] [EC:2.4.1.144]\n",
      "AT1G67880  beta-1,4-N-acetylglucosaminyltransferase family protein [KO:K00737] [EC:2.4.1.144]\n",
      "AT3G01620  beta-1,4-N-acetylglucosaminyltransferase family protein [KO:K00737] [EC:2.4.1.144]\n",
      "AT5G14480  beta-1,4-N-acetylglucosaminyltransferase family protein [KO:K00737] [EC:2.4.1.144]\n",
      "AT3G27540  beta-1,4-N-acetylglucosaminyltransferase family protein [KO:K00737] [EC:2.4.1.144]\n",
      "AT1G12990  beta-1,4-N-acetylglucosaminyltransferase family protein [KO:K00737] [EC:2.4.1.144]\n",
      "COMPOUND    C00096  GDP-mannose\n",
      "C00110  Dolichyl phosphate\n",
      "C00381  Dolichol\n",
      "C00621  Dolichyl diphosphate\n",
      "C01246  Dolichyl beta-D-glucosyl phosphate\n",
      "C03021  Protein asparagine\n",
      "C03862  Dolichyl phosphate D-mannose\n",
      "C06081  Polyprenol\n",
      "G00001  N-Acetyl-D-glucosaminyldiphosphodolichol\n",
      "G00002  N,N'-Chitobiosyldiphosphodolichol\n",
      "G00003\n",
      "G00004\n",
      "G00005\n",
      "G00006\n",
      "G00007\n",
      "G00008\n",
      "G00009\n",
      "G00010\n",
      "G00011\n",
      "G00012\n",
      "G00013\n",
      "G00014\n",
      "G00015\n",
      "G00016\n",
      "G00017\n",
      "G00018  DS 3\n",
      "G00019\n",
      "G00020\n",
      "G00021\n",
      "G00022\n",
      "G00171\n",
      "G10526\n",
      "G10595\n",
      "G10596\n",
      "G10597\n",
      "G10598\n",
      "G10599\n",
      "G10694\n",
      "REFERENCE   PMID:9111134\n",
      "AUTHORS   Schutzbach JS.\n",
      "TITLE     The role of the lipid matrix in the biosynthesis of dolichyl-linked oligosaccharides.\n",
      "JOURNAL   Glycoconj J 14:175-82 (1997)\n",
      "DOI:10.1023/A:1018533620002\n",
      "REFERENCE   PMID:12200473\n",
      "AUTHORS   Oriol R, Martinez-Duncker I, Chantret I, Mollicone R, Codogno P.\n",
      "TITLE     Common origin and evolution of glycosyltransferases using Dol-P-monosaccharides as donor substrate.\n",
      "JOURNAL   Mol Biol Evol 19:1451-63 (2002)\n",
      "DOI:10.1093/oxfordjournals.molbev.a004208\n",
      "REFERENCE   PMID:12684507\n",
      "AUTHORS   Thiel C, Schwarz M, Peng J, Grzmil M, Hasilik M, Braulke T, Kohlschutter A, von Figura K, Lehle L, Korner C.\n",
      "TITLE     A new type of congenital disorders of glycosylation (CDG-Ii) provides new insights into the early steps of dolichol-linked oligosaccharide biosynthesis.\n",
      "JOURNAL   J Biol Chem 278:22498-505 (2003)\n",
      "DOI:10.1074/jbc.M302850200\n",
      "REFERENCE   PMID:11278778\n",
      "AUTHORS   Cipollo JF, Trimble RB, Chi JH, Yan Q, Dean N.\n",
      "TITLE     The yeast ALG11 gene specifies addition of the terminal alpha 1,2-Man to the Man5GlcNAc2-PP-dolichol N-glycosylation intermediate formed on the cytosolic side of the endoplasmic reticulum.\n",
      "JOURNAL   J Biol Chem 276:21828-40 (2001)\n",
      "DOI:10.1074/jbc.M010896200\n",
      "REFERENCE   PMID:15987956\n",
      "AUTHORS   Frank CG, Aebi M.\n",
      "TITLE     ALG9 mannosyltransferase is involved in two different steps of lipid-linked oligosaccharide biosynthesis.\n",
      "JOURNAL   Glycobiology 15:1156-63 (2005)\n",
      "DOI:10.1093/glycob/cwj002\n",
      "REFERENCE   PMID:16878994\n",
      "AUTHORS   O'Reilly MK, Zhang G, Imperiali B.\n",
      "TITLE     In vitro evidence for the dual function of Alg2 and Alg11: essential mannosyltransferases in N-linked glycoprotein biosynthesis.\n",
      "JOURNAL   Biochemistry 45:9593-603 (2006)\n",
      "DOI:10.1021/bi060878o\n",
      "REL_PATHWAY ath00051  Fructose and mannose metabolism\n",
      "ath00511  Other glycan degradation\n",
      "ath00513  Various types of N-glycan biosynthesis\n",
      "ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "ath00563  Glycosylphosphatidylinositol (GPI)-anchor biosynthesis\n",
      "ath00900  Terpenoid backbone biosynthesis\n",
      "KO_PATHWAY  ko00510\n",
      "///\n",
      "ENTRY       ath00511                    Pathway\n",
      "NAME        Other glycan degradation - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00511  Other glycan degradation\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G54440  glycoside hydrolase family 2 protein [KO:K01190] [EC:3.2.1.23]\n",
      "AT1G72990  BGAL17; beta-galactosidase 17 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G52840  BGAL2; beta-galactosidase 2 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G55260  HEXO1; beta-hexosaminidase 1 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G65590  HEXO3; beta-hexosaminidase 3 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G05590  HEXO2; beta-hexosaminidase 2 [KO:K12373] [EC:3.2.1.52]\n",
      "AT3G26720  Glycosyl hydrolase family 38 protein [KO:K01191] [EC:3.2.1.24]\n",
      "AT5G66150  Glycosyl hydrolase family 38 protein [KO:K01191] [EC:3.2.1.24]\n",
      "AT5G13980  Glycosyl hydrolase family 38 protein [KO:K01191] [EC:3.2.1.24]\n",
      "AT5G05460  ENGase85A; Glycosyl hydrolase family 85 [KO:K01227] [EC:3.2.1.96]\n",
      "AT3G11040  ENGase85B; Glycosyl hydrolase family 85 [KO:K01227] [EC:3.2.1.96]\n",
      "AT2G28100  FUC1; alpha-L-fucosidase 1 [KO:K01206] [EC:3.2.1.51]\n",
      "AT1G67830  FXG1; alpha-fucosidase 1 [KO:K01206] [EC:3.2.1.51]\n",
      "AT4G34260  FUC95A; 1,2-alpha-L-fucosidase [KO:K15923] [EC:3.2.1.51]\n",
      "AT5G61540  N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K01444] [EC:3.5.1.26]\n",
      "AT3G24180  Beta-glucosidase, GBA2 type family protein [KO:K17108] [EC:3.2.1.45]\n",
      "AT5G49900  Beta-glucosidase, GBA2 type family protein [KO:K17108] [EC:3.2.1.45]\n",
      "AT4G10060  Beta-glucosidase, GBA2 type family protein [KO:K17108] [EC:3.2.1.45]\n",
      "AT1G33700  Beta-glucosidase, GBA2 type family protein [KO:K17108] [EC:3.2.1.45]\n",
      "REL_PATHWAY ath00510  N-Glycan biosynthesis\n",
      "ath00604  Glycosphingolipid biosynthesis - ganglio series\n",
      "KO_PATHWAY  ko00511\n",
      "///\n",
      "ENTRY       ath00513                    Pathway\n",
      "NAME        Various types of N-glycan biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00513  Various types of N-glycan biosynthesis\n",
      "MODULE      ath_M00072  N-glycosylation by oligosaccharyltransferase [PATH:ath00513]\n",
      "DBLINKS     GO: 0006487\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G16710  glycosyltransferase family protein 28 [KO:K07432] [EC:2.4.1.141]\n",
      "AT4G18230  UDP-N-acetylglucosamine transferase subunit ALG14-like protein [KO:K07441] [EC:2.4.1.141]\n",
      "AT1G16570  UDP-Glycosyltransferase superfamily protein [KO:K03842] [EC:2.4.1.142]\n",
      "AT1G78800  UDP-Glycosyltransferase superfamily protein [KO:K03843] [EC:2.4.1.132 2.4.1.257]\n",
      "AT2G40190  LEW3; UDP-Glycosyltransferase superfamily protein [KO:K03844] [EC:2.4.1.131]\n",
      "AT2G47760  ALG3; asparagine-linked glycosylation 3 [KO:K03845] [EC:2.4.1.258]\n",
      "AT1G16900  EBS3; Alg9-like mannosyltransferase family [KO:K03846] [EC:2.4.1.259 2.4.1.261]\n",
      "AT1G02145  ALG12; homolog of asparagine-linked glycosylation 12 [KO:K03847] [EC:2.4.1.260]\n",
      "AT1G34130  STT3B; staurosporin and temperature sensitive 3-like b [KO:K07151] [EC:2.4.99.18]\n",
      "AT5G19690  STT3A; staurosporin and temperature sensitive 3-like A [KO:K07151] [EC:2.4.99.18]\n",
      "AT1G76400  Ribophorin I [KO:K12666]\n",
      "AT2G01720  Ribophorin I [KO:K12666]\n",
      "AT4G21150  HAP6; ribophorin II (RPN2) family protein [KO:K12667]\n",
      "AT1G32210  ATDAD1; Defender against death (DAD family) protein [KO:K12668]\n",
      "AT2G35520  DAD2; Defender against death (DAD family) protein [KO:K12668]\n",
      "AT1G61790  Oligosaccharyltransferase complex/magnesium transporter family protein [KO:K12669]\n",
      "AT1G11560  Oligosaccharyltransferase complex/magnesium transporter family protein [KO:K12669]\n",
      "AT5G66680  DGL1; dolichyl-diphosphooligosaccharide-protein glycosyltransferase 48kDa subunit family protein [KO:K12670]\n",
      "AT1G30000  MNS3; alpha-mannosidase 3 [KO:K23741] [EC:3.2.1.209]\n",
      "AT3G21160  MNS2; alpha-mannosidase 2 [KO:K01230] [EC:3.2.1.113]\n",
      "AT1G51590  MNS1; alpha-mannosidase 1 [KO:K01230] [EC:3.2.1.113]\n",
      "AT4G38240  CGL1; alpha-1,3-mannosyl-glycoprotein beta-1,2-N-acetylglucosaminyltransferase [KO:K00726] [EC:2.4.1.101]\n",
      "AT5G14950  GMII; golgi alpha-mannosidase II [KO:K01231] [EC:3.2.1.114]\n",
      "AT2G05320  beta-1,2-N-acetylglucosaminyltransferase II [KO:K00736] [EC:2.4.1.143]\n",
      "AT5G55500  XYLT; beta-1,2-xylosyltransferase [KO:K03714] [EC:2.4.2.38]\n",
      "AT3G19280  FUT11; fucosyltransferase 11 [KO:K00753] [EC:2.4.1.214]\n",
      "AT1G49710  FUT12; fucosyltransferase 12 [KO:K00753] [EC:2.4.1.214]\n",
      "AT1G26810  GALT1; galactosyltransferase1 [KO:K14413] [EC:2.4.1.-]\n",
      "AT1G71990  FUT13; fucosyltransferase 13 [KO:K14412] [EC:2.4.1.65]\n",
      "AT3G55260  HEXO1; beta-hexosaminidase 1 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G65590  HEXO3; beta-hexosaminidase 3 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G05590  HEXO2; beta-hexosaminidase 2 [KO:K12373] [EC:3.2.1.52]\n",
      "COMPOUND    G00011\n",
      "G00012\n",
      "G00013\n",
      "G00014\n",
      "G00015\n",
      "G00016\n",
      "G00017\n",
      "G00179\n",
      "G01813\n",
      "G05477\n",
      "G10665\n",
      "G10694\n",
      "G10716\n",
      "G10770\n",
      "G10841\n",
      "G10920\n",
      "G11000\n",
      "G11040\n",
      "G11933\n",
      "G12617\n",
      "G12625\n",
      "G12626\n",
      "G13047\n",
      "G13048\n",
      "G13049\n",
      "G13050\n",
      "G13055\n",
      "G13056\n",
      "G13057\n",
      "G13058\n",
      "G13059\n",
      "REFERENCE   PMID:11412862\n",
      "AUTHORS   Munro S.\n",
      "TITLE     What can yeast tell us about N-linked glycosylation in the Golgi apparatus?\n",
      "JOURNAL   FEBS Lett 498:223-7 (2001)\n",
      "DOI:10.1016/S0014-5793(01)02488-7\n",
      "REFERENCE   PMID:9878803\n",
      "AUTHORS   Dean N.\n",
      "TITLE     Asparagine-linked glycosylation in the yeast Golgi.\n",
      "JOURNAL   Biochim Biophys Acta 1426:309-22 (1999)\n",
      "DOI:10.1016/S0304-4165(98)00132-9\n",
      "REFERENCE   PMID:9878780\n",
      "AUTHORS   Herscovics A.\n",
      "TITLE     Processing glycosidases of Saccharomyces cerevisiae.\n",
      "JOURNAL   Biochim Biophys Acta 1426:275-85 (1999)\n",
      "DOI:10.1016/S0304-4165(98)00129-9\n",
      "REFERENCE   PMID:9218445\n",
      "AUTHORS   Wang XH, Nakayama K, Shimma Y, Tanaka A, Jigami Y.\n",
      "TITLE     MNN6, a member of the KRE2/MNT1 family, is the gene for mannosylphosphate transfer in Saccharomyces cerevisiae.\n",
      "JOURNAL   J Biol Chem 272:18117-24 (1997)\n",
      "DOI:10.1074/jbc.272.29.18117\n",
      "REFERENCE   PMID:9756928\n",
      "AUTHORS   Rayner JC, Munro S.\n",
      "TITLE     Identification of the MNN2 and MNN5 mannosyltransferases required for forming and extending the mannose branches of the outer chain mannans of Saccharomyces cerevisiae.\n",
      "JOURNAL   J Biol Chem 273:26836-43 (1998)\n",
      "DOI:10.1074/jbc.273.41.26836\n",
      "REFERENCE   PMID:10037752\n",
      "AUTHORS   Jungmann J, Rayner JC, Munro S.\n",
      "TITLE     The Saccharomyces cerevisiae protein Mnn10p/Bed1p is a subunit of a Golgi mannosyltransferase complex.\n",
      "JOURNAL   J Biol Chem 274:6579-85 (1999)\n",
      "DOI:10.1074/jbc.274.10.6579\n",
      "REFERENCE   PMID:8253757\n",
      "AUTHORS   Nakanishi-Shindo Y, Nakayama K, Tanaka A, Toda Y, Jigami Y.\n",
      "TITLE     Structure of the N-linked oligosaccharides that show the complete loss of alpha-1,6-polymannose outer chain from och1, och1 mnn1, and och1 mnn1 alg3 mutants of Saccharomyces cerevisiae.\n",
      "JOURNAL   J Biol Chem 268:26338-45 (1993)\n",
      "REFERENCE   PMID:20233335\n",
      "AUTHORS   Gomord V, Fitchette AC, Menu-Bouaouiche L, Saint-Jore-Dupas C, Plasson C, Michaud D, Faye L\n",
      "TITLE     Plant-specific glycosylation patterns in the context of therapeutic protein production.\n",
      "JOURNAL   Plant Biotechnol J 8:564-87 (2010)\n",
      "DOI:10.1111/j.1467-7652.2009.00497.x\n",
      "REL_PATHWAY ath00510  N-Glycan biosynthesis\n",
      "KO_PATHWAY  ko00513\n",
      "///\n",
      "ENTRY       ath00514                    Pathway\n",
      "NAME        Other types of O-glycan biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION O-linked glycosylation is the attachment of monosaccharides to the hydroxyl groups of amino acids, mostly serine and threonine, and is found in eukaryotes, archaea and bacteria. O-glycans exhibit diverse types of modifications where the innermost monosaccharide is N-acetylgalactosamine (map00512), mannose in mammals (map00515), and others (this map) including N-acetylglucosamine, fucose, glucose, galactose, mannose in yeast and arabinose in plants.\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00514  Other types of O-glycan biosynthesis\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G04240  SEC; Tetratricopeptide repeat (TPR)-like superfamily protein [KO:K09667] [EC:2.4.1.255]\n",
      "AT3G11540  SPY; Tetratricopeptide repeat (TPR)-like superfamily protein [KO:K09667] [EC:2.4.1.255]\n",
      "AT5G50420  O-fucosyltransferase family protein [KO:K03691] [EC:2.4.1.221]\n",
      "AT1G17270  O-fucosyltransferase family protein [KO:K03691] [EC:2.4.1.221]\n",
      "AT1G53770  O-fucosyltransferase family protein [KO:K03691] [EC:2.4.1.221]\n",
      "AT3G01720  peptidyl serine alpha-galactosyltransferase [KO:K20781] [EC:2.4.1.-]\n",
      "AT5G25265  Hyp O-arabinosyltransferase-like protein [KO:K20782] [EC:2.4.2.58]\n",
      "AT2G25260  Hyp O-arabinosyltransferase-like protein [KO:K20782] [EC:2.4.2.58]\n",
      "AT5G13500  Hyp O-arabinosyltransferase-like protein [KO:K20782] [EC:2.4.2.58]\n",
      "AT1G75120  RRA1; Nucleotide-diphospho-sugar transferase family protein [KO:K20783] [EC:2.4.2.-]\n",
      "AT1G75110  RRA2; Nucleotide-diphospho-sugar transferase family protein [KO:K20783] [EC:2.4.2.-]\n",
      "AT1G19360  RRA3; Nucleotide-diphospho-sugar transferase family protein [KO:K20783] [EC:2.4.2.-]\n",
      "AT2G35610  XEG113; xyloglucanase 113 [KO:K20784] [EC:2.4.2.-]\n",
      "REFERENCE   PMID:22669261\n",
      "AUTHORS   Kizuka Y, Oka S\n",
      "TITLE     Regulated expression and neural functions of human natural killer-1 (HNK-1) carbohydrate.\n",
      "JOURNAL   Cell Mol Life Sci 69:4135-47 (2012)\n",
      "DOI:10.1007/s00018-012-1036-z\n",
      "REFERENCE   PMID:19429925\n",
      "AUTHORS   Lommel M, Strahl S\n",
      "TITLE     Protein O-mannosylation: conserved from bacteria to humans.\n",
      "JOURNAL   Glycobiology 19:816-28 (2009)\n",
      "DOI:10.1093/glycob/cwp066\n",
      "KO_PATHWAY  ko00514\n",
      "///\n",
      "ENTRY       ath00520                    Pathway\n",
      "NAME        Amino sugar and nucleotide sugar metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "MODULE      ath_M00549  Nucleotide sugar biosynthesis, glucose => UDP-glucose [PATH:ath00520]\n",
      "ath_M00892  UDP-N-acetyl-D-glucosamine biosynthesis, eukaryotes, glucose => UDP-GlcNAc [PATH:ath00520]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G54420  EP3; homolog of carrot EP3-3 chitinase [KO:K01183] [EC:3.2.1.14]\n",
      "AT5G24090  CHIA; chitinase A [KO:K01183] [EC:3.2.1.14]\n",
      "AT4G19810  ChiC; Glycosyl hydrolase family protein with chitinase insertion domain-containing protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT2G43590  Chitinase family protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT2G43580  Chitinase family protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT2G43620  Chitinase family protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT4G19800  Glycosyl hydrolase family protein with chitinase insertion domain-containing protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT4G19820  Glycosyl hydrolase family protein with chitinase insertion domain-containing protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT4G19760  Glycosyl hydrolase family protein with chitinase insertion domain-containing protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT4G19750  Glycosyl hydrolase family protein with chitinase insertion domain-containing protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT4G19720  Glycosyl hydrolase family protein with chitinase insertion domain-containing protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT4G19730  Glycosyl hydrolase superfamily protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT2G43570  CHI; chitinase [KO:K01183] [EC:3.2.1.14]\n",
      "AT2G43600  Chitinase family protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT3G47540  Chitinase family protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT1G56680  Chitinase family protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT2G43610  Chitinase family protein [KO:K01183] [EC:3.2.1.14]\n",
      "AT3G12500  HCHIB; basic chitinase [KO:K20547] [EC:3.2.1.14]\n",
      "AT3G55260  HEXO1; beta-hexosaminidase 1 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G65590  HEXO3; beta-hexosaminidase 3 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G05590  HEXO2; beta-hexosaminidase 2 [KO:K12373] [EC:3.2.1.52]\n",
      "AT4G37840  HKL3; hexokinase-like 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G47840  HXK3; hexokinase 3 [KO:K00844] [EC:2.7.1.1]\n",
      "AT2G19860  HXK2; hexokinase 2 [KO:K00844] [EC:2.7.1.1]\n",
      "AT4G29130  HXK1; hexokinase 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT1G50460  HKL1; hexokinase-like 1 [KO:K00844] [EC:2.7.1.1]\n",
      "AT3G20040  ATHXK4; Hexokinase [KO:K00844] [EC:2.7.1.1]\n",
      "AT5G51830  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT2G31390  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT4G10260  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G06030  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT3G59480  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G66430  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G06020  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT1G50390  pfkB-like carbohydrate kinase family protein [KO:K00847] [EC:2.7.1.4]\n",
      "AT3G24090  GFAT; putative glucosamine-fructose-6-phosphate aminotransferase [KO:K00820] [EC:2.6.1.16]\n",
      "AT5G15770  GNA1; glucose-6-phosphate acetyltransferase 1 [KO:K00621] [EC:2.3.1.4]\n",
      "AT5G18070  DRT101; phosphoglucosamine mutase-like protein [KO:K01836] [EC:5.4.2.3]\n",
      "AT3G06580  GALK; Mevalonate/galactokinase family protein [KO:K18674] [EC:2.7.1.157]\n",
      "AT1G31070  GlcNAc1pUT1; N-acetylglucosamine-1-phosphate uridylyltransferase 1 [KO:K00972] [EC:2.7.7.23 2.7.7.83]\n",
      "AT2G35020  GlcNAc1pUT2; N-acetylglucosamine-1-phosphate uridylyltransferase 2 [KO:K00972] [EC:2.7.7.23 2.7.7.83]\n",
      "AT5G17770  CBR; NADH:cytochrome B5 reductase 1 [KO:K00326] [EC:1.6.2.2]\n",
      "AT5G20080  FAD/NAD(P)-binding oxidoreductase [KO:K00326] [EC:1.6.2.2]\n",
      "AT4G16130  ARA1; arabinose kinase [KO:K12446] [EC:2.7.1.46]\n",
      "AT3G42850  Mevalonate/galactokinase family protein [KO:K12446] [EC:2.7.1.46]\n",
      "AT5G52560  USP; UDP-sugar pyrophosphorylase [KO:K12447] [EC:2.7.7.64]\n",
      "AT3G02230  RGP1; reversibly glycosylated polypeptide 1 [KO:K13379] [EC:2.4.1.- 5.4.99.30]\n",
      "AT5G15650  RGP2; reversibly glycosylated polypeptide 2 [KO:K13379] [EC:2.4.1.- 5.4.99.30]\n",
      "AT3G08900  RGP3; reversibly glycosylated polypeptide 3 [KO:K13379] [EC:2.4.1.- 5.4.99.30]\n",
      "AT5G50750  RGP4; reversibly glycosylated polypeptide 4 [KO:K13379] [EC:2.4.1.- 5.4.99.30]\n",
      "AT5G16510  RGP5; Alpha-1,4-glucan-protein synthase family protein [KO:K13379] [EC:2.4.1.- 5.4.99.30]\n",
      "AT3G10740  ASD1; alpha-L-arabinofuranosidase 1 [KO:K01209] [EC:3.2.1.55]\n",
      "AT5G26120  ASD2; alpha-L-arabinofuranosidase 2 [KO:K01209] [EC:3.2.1.55]\n",
      "AT5G14470  GHMP kinase family protein [KO:K16190] [EC:2.7.1.43]\n",
      "AT3G01640  GLCAK; glucuronokinase G [KO:K16190] [EC:2.7.1.43]\n",
      "AT3G53520  UXS1; UDP-glucuronic acid decarboxylase 1 [KO:K08678] [EC:4.1.1.35]\n",
      "AT2G47650  UXS4; UDP-xylose synthase 4 [KO:K08678] [EC:4.1.1.35]\n",
      "AT5G59290  UXS3; UDP-glucuronic acid decarboxylase 3 [KO:K08678] [EC:4.1.1.35]\n",
      "AT2G28760  UXS6; UDP-XYL synthase 6 [KO:K08678] [EC:4.1.1.35]\n",
      "AT3G62830  AUD1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K08678] [EC:4.1.1.35]\n",
      "AT3G46440  UXS5; UDP-XYL synthase 5 [KO:K08678] [EC:4.1.1.35]\n",
      "AT2G27860  AXS1; UDP-D-apiose/UDP-D-xylose synthase 1 [KO:K12449]\n",
      "AT1G08200  AXS2; UDP-D-apiose/UDP-D-xylose synthase 2 [KO:K12449]\n",
      "AT1G02730  CSLD5; cellulose synthase-like D5 [KO:K00770] [EC:2.4.2.24]\n",
      "AT5G64570  XYL4; beta-D-xylosidase 4 [KO:K15920] [EC:3.2.1.37]\n",
      "AT3G19620  Glycosyl hydrolase family protein [KO:K15920] [EC:3.2.1.37]\n",
      "AT5G09700  Glycosyl hydrolase family protein [KO:K15920] [EC:3.2.1.37]\n",
      "AT5G09730  BXL3; beta-xylosidase 3 [KO:K15920] [EC:3.2.1.37]\n",
      "AT2G34850  MEE25; NAD(P)-binding Rossmann-fold superfamily protein [KO:K12448] [EC:5.1.3.5]\n",
      "AT1G30620  MUR4; NAD(P)-binding Rossmann-fold superfamily protein [KO:K12448] [EC:5.1.3.5]\n",
      "AT4G20460  NAD(P)-binding Rossmann-fold superfamily protein [KO:K12448] [EC:5.1.3.5]\n",
      "AT5G44480  DUR; NAD(P)-binding Rossmann-fold superfamily protein [KO:K12448] [EC:5.1.3.5]\n",
      "AT5G51820  PGM; phosphoglucomutase [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G23190  PGM3; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT1G70730  PGM2; Phosphoglucomutase/phosphomannomutase family protein [KO:K01835] [EC:5.4.2.2]\n",
      "AT5G17310  UGP2; UDP-glucose pyrophosphorylase 2 [KO:K00963] [EC:2.7.7.9]\n",
      "AT3G03250  UGP1; UDP-GLUCOSE PYROPHOSPHORYLASE 1 [KO:K00963] [EC:2.7.7.9]\n",
      "AT1G53500  MUM4; NAD-dependent epimerase/dehydratase family protein [KO:K12450] [EC:4.2.1.76]\n",
      "AT3G14790  RHM3; rhamnose biosynthesis 3 [KO:K12450] [EC:4.2.1.76]\n",
      "AT1G78570  RHM1; rhamnose biosynthesis 1 [KO:K12450] [EC:4.2.1.76]\n",
      "AT1G63000  NRS/ER; nucleotide-rhamnose synthase/epimerase-reductase [KO:K12451] [EC:5.1.3.- 1.1.1.-]\n",
      "AT5G39320  UDG4; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT1G26570  UGD1; UDP-glucose dehydrogenase 1 [KO:K00012] [EC:1.1.1.22]\n",
      "AT5G15490  UGD3; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT3G29360  UGD2; UDP-glucose 6-dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT3G01010  UDP-glucose/GDP-mannose dehydrogenase family protein [KO:K00012] [EC:1.1.1.22]\n",
      "AT4G33030  SQD1; sulfoquinovosyldiacylglycerol 1 [KO:K06118] [EC:3.13.1.1]\n",
      "AT5G18200  putative galactose-1-phosphate uridyltransferase [KO:K00965] [EC:2.7.7.12]\n",
      "AT1G63180  UGE3; UDP-D-glucose/UDP-D-galactose 4-epimerase 3 [KO:K01784] [EC:5.1.3.2]\n",
      "AT1G64440  RHD1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K01784] [EC:5.1.3.2]\n",
      "AT1G12780  UGE1; UDP-D-glucose/UDP-D-galactose 4-epimerase 1 [KO:K01784] [EC:5.1.3.2]\n",
      "AT4G23920  UGE2; UDP-D-glucose/UDP-D-galactose 4-epimerase 2 [KO:K01784] [EC:5.1.3.2]\n",
      "AT4G10960  UGE5; UDP-D-glucose/UDP-D-galactose 4-epimerase 5 [KO:K01784] [EC:5.1.3.2]\n",
      "AT4G00110  GAE3; UDP-D-glucuronate 4-epimerase 3 [KO:K08679] [EC:5.1.3.6]\n",
      "AT4G30440  GAE1; UDP-D-glucuronate 4-epimerase 1 [KO:K08679] [EC:5.1.3.6]\n",
      "AT4G12250  GAE5; UDP-D-glucuronate 4-epimerase 5 [KO:K08679] [EC:5.1.3.6]\n",
      "AT2G45310  GAE4; UDP-D-glucuronate 4-epimerase 4 [KO:K08679] [EC:5.1.3.6]\n",
      "AT1G02000  GAE2; UDP-D-glucuronate 4-epimerase 2 [KO:K08679] [EC:5.1.3.6]\n",
      "AT3G23820  GAE6; UDP-D-glucuronate 4-epimerase 6 [KO:K08679] [EC:5.1.3.6]\n",
      "AT3G10700  GalAK; galacturonic acid kinase [KO:K18677] [EC:2.7.1.44]\n",
      "AT4G38270  GAUT3; galacturonosyltransferase 3 [KO:K13648] [EC:2.4.1.43]\n",
      "AT3G02350  GAUT9; galacturonosyltransferase 9 [KO:K13648] [EC:2.4.1.43]\n",
      "AT2G46480  GAUT2; galacturonosyltransferase-like protein [KO:K13648] [EC:2.4.1.43]\n",
      "AT5G47780  GAUT4; galacturonosyltransferase 4 [KO:K13648] [EC:2.4.1.43]\n",
      "AT1G06780  GAUT6; galacturonosyltransferase 6 [KO:K13648] [EC:2.4.1.43]\n",
      "AT2G20810  GAUT10; galacturonosyltransferase 10 [KO:K13648] [EC:2.4.1.43]\n",
      "AT1G18580  GAUT11; galacturonosyltransferase 11 [KO:K13648] [EC:2.4.1.43]\n",
      "AT3G61130  GAUT1; galacturonosyltransferase 1 [KO:K13648] [EC:2.4.1.43]\n",
      "AT3G25140  QUA1; Nucleotide-diphospho-sugar transferases superfamily protein [KO:K13648] [EC:2.4.1.43]\n",
      "AT2G30575  LGT5; los glycosyltransferase 5 [KO:K13648] [EC:2.4.1.43]\n",
      "AT2G38650  GAUT7; galacturonosyltransferase 7 [KO:K13648] [EC:2.4.1.43]\n",
      "AT4G24620  PGI1; phosphoglucose isomerase 1 [KO:K01810] [EC:5.3.1.9]\n",
      "AT5G42740  Sugar isomerase (SIS) family protein [KO:K01810] [EC:5.3.1.9]\n",
      "AT3G02570  MEE31; Mannose-6-phosphate isomerase, type I [KO:K01809] [EC:5.3.1.8]\n",
      "AT1G67070  DIN9; Mannose-6-phosphate isomerase, type I [KO:K01809] [EC:5.3.1.8]\n",
      "AT2G45790  PMM; phosphomannomutase [KO:K17497] [EC:5.4.2.8]\n",
      "AT1G74910  ADP-glucose pyrophosphorylase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT2G39770  CYT1; Glucose-1-phosphate adenylyltransferase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT3G55590  Glucose-1-phosphate adenylyltransferase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT4G30570  Glucose-1-phosphate adenylyltransferase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT2G04650  ADP-glucose pyrophosphorylase family protein [KO:K00966] [EC:2.7.7.13]\n",
      "AT5G66280  GMD1; GDP-D-mannose 4,6-dehydratase 1 [KO:K01711] [EC:4.2.1.47]\n",
      "AT3G51160  MUR1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K01711] [EC:4.2.1.47]\n",
      "AT1G17890  GER2; NAD(P)-binding Rossmann-fold superfamily protein [KO:K02377] [EC:1.1.1.271]\n",
      "AT1G73250  GER1; GDP-4-keto-6-deoxymannose-3, 5-epimerase-4-reductase 1 [KO:K02377] [EC:1.1.1.271]\n",
      "AT5G28840  GME; GDP-D-mannose 3',5'-epimerase [KO:K10046] [EC:5.1.3.18 5.1.3.-]\n",
      "AT1G01220  FKGP; L-fucokinase/GDP-L-fucose pyrophosphorylase [KO:K25314] [EC:2.7.1.52 2.7.7.30]\n",
      "AT1G27680  APL2; ADPGLC-PPase large subunit [KO:K00975] [EC:2.7.7.27]\n",
      "AT5G48300  ADG1; ADP glucose pyrophosphorylase 1 [KO:K00975] [EC:2.7.7.27]\n",
      "AT1G05610  APS2; ADP-glucose pyrophosphorylase small subunit 2 [KO:K00975] [EC:2.7.7.27]\n",
      "AT4G39210  APL3; Glucose-1-phosphate adenylyltransferase family protein [KO:K00975] [EC:2.7.7.27]\n",
      "AT5G19220  APL1; ADP glucose pyrophosphorylase large subunit 1 [KO:K00975] [EC:2.7.7.27]\n",
      "AT2G21590  APL4; Glucose-1-phosphate adenylyltransferase family protein [KO:K00975] [EC:2.7.7.27]\n",
      "COMPOUND    C00029  UDP-glucose\n",
      "C00031  D-Glucose\n",
      "C00043  UDP-N-acetyl-alpha-D-glucosamine\n",
      "C00052  UDP-alpha-D-galactose\n",
      "C00096  GDP-mannose\n",
      "C00103  D-Glucose 1-phosphate\n",
      "C00128  CMP-N-acetylneuraminate\n",
      "C00140  N-Acetyl-D-glucosamine\n",
      "C00159  D-Mannose\n",
      "C00167  UDP-glucuronate\n",
      "C00181  D-Xylose\n",
      "C00190  UDP-D-xylose\n",
      "C00191  D-Glucuronate\n",
      "C00203  UDP-N-acetyl-D-galactosamine\n",
      "C00216  D-Arabinose\n",
      "C00259  L-Arabinose\n",
      "C00267  alpha-D-Glucose\n",
      "C00270  N-Acetylneuraminate\n",
      "C00275  D-Mannose 6-phosphate\n",
      "C00325  GDP-L-fucose\n",
      "C00329  D-Glucosamine\n",
      "C00333  D-Galacturonate\n",
      "C00352  D-Glucosamine 6-phosphate\n",
      "C00357  N-Acetyl-D-glucosamine 6-phosphate\n",
      "C00394  GDP-glucose\n",
      "C00446  alpha-D-Galactose 1-phosphate\n",
      "C00461  Chitin\n",
      "C00498  ADP-glucose\n",
      "C00501  CDP-glucose\n",
      "C00617  UDP-D-galacturonate\n",
      "C00636  D-Mannose 1-phosphate\n",
      "C00645  N-Acetyl-D-mannosamine\n",
      "C00668  alpha-D-Glucose 6-phosphate\n",
      "C00714  Pectin\n",
      "C00734  Chitosan\n",
      "C00935  UDP-L-arabinose\n",
      "C00976  GDP-D-mannuronate\n",
      "C00984  alpha-D-Galactose\n",
      "C00996  Ferricytochrome b5\n",
      "C00999  Ferrocytochrome b5\n",
      "C01019  6-Deoxy-L-galactose\n",
      "C01050  UDP-N-acetylmuramate\n",
      "C01132  N-Acetyl-D-galactosamine\n",
      "C01170  UDP-N-acetyl-D-mannosamine\n",
      "C01219  CDP-4-dehydro-6-deoxy-D-glucose\n",
      "C01222  GDP-4-dehydro-6-deoxy-D-mannose\n",
      "C01623  UDP-apiose\n",
      "C01674  Chitobiose\n",
      "C01788  CDP-abequose\n",
      "C02199  UDP-L-rhamnose\n",
      "C02280  GDP-L-galactose\n",
      "C02336  beta-D-Fructose\n",
      "C02352  1,4-beta-D-Xylan\n",
      "C02474  Arabinan\n",
      "C02713  N-Acetylmuramate\n",
      "C02977  GDP-6-deoxy-D-talose\n",
      "C02985  L-Fucose 1-phosphate\n",
      "C03117  GDP-6-deoxy-D-mannose\n",
      "C03410  N-Glycoloyl-neuraminate\n",
      "C03598  CDP-3,6-dideoxy-D-glucose\n",
      "C03599  CDP-3,6-dideoxy-D-mannose\n",
      "C03691  CMP-N-glycoloylneuraminate\n",
      "C03733  UDP-alpha-D-galactofuranose\n",
      "C03906  beta-L-Arabinose 1-phosphate\n",
      "C04037  1-Phospho-alpha-D-galacturonate\n",
      "C04089  UDP-4-dehydro-6-deoxy-D-glucose\n",
      "C04257  N-Acetyl-D-mannosamine 6-phosphate\n",
      "C04297  CDP-4-dehydro-3,6-dideoxy-D-glucose\n",
      "C04501  N-Acetyl-alpha-D-glucosamine 1-phosphate\n",
      "C04573  UDP-N-acetyl-2-amino-2-deoxy-D-glucuronate\n",
      "C04613  UDP-2-acetamido-4-dehydro-2,6-dideoxyglucose\n",
      "C04630  UDP-2-acetamido-4-amino-2,4,6-trideoxy-alpha-D-glucose\n",
      "C04631  UDP-N-acetyl-3-(1-carboxyvinyl)-D-glucosamine\n",
      "C05345  beta-D-Fructose 6-phosphate\n",
      "C05385  D-Glucuronate 1-phosphate\n",
      "C06023  D-Glucosaminide\n",
      "C06156  alpha-D-Glucosamine 1-phosphate\n",
      "C06240  UDP-N-acetyl-D-mannosaminouronate\n",
      "C06241  N-Acetylneuraminate 9-phosphate\n",
      "C06377  D-Galactosamine 6-phosphate\n",
      "C11521  UDP-6-sulfoquinovose\n",
      "C13952  UDP-N-acetyl-D-galactosaminuronic acid\n",
      "C15925  GDP-L-gulose\n",
      "C16153  UDP-L-Ara4N\n",
      "C16154  UDP-L-Ara4FN\n",
      "C16155  UDP-L-Ara4O\n",
      "C16156  Undecaprenyl phosphate alpha-L-Ara4FN\n",
      "C16157  Undecaprenyl phosphate alpha-L-Ara4N\n",
      "C16183  alpha-D-Arabinopyranose 1-phosphate\n",
      "C16698  N-Acetylmuramic acid 6-phosphate\n",
      "C17269  GDP-D-arabinopyranose\n",
      "C17326  CDP-4-dehydro-3,6-dideoxy-D-glucose epimer\n",
      "C17327  CDP-ascarylose\n",
      "C17328  UDP-4-keto-rhamnose\n",
      "C18060  N-Acetyl-alpha-D-galactosamine 1-phosphate\n",
      "C18094  UDP-L-arabinofuranose\n",
      "C19725  UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-glucuronate\n",
      "C19745  UDP-2,3-diacetamido-2,3-dideoxy-alpha-D-mannuronate\n",
      "C19769  1,6-Anhydro-N-acetyl-beta-muramate\n",
      "C19823  UDP-2-acetamido-2,6-dideoxy-beta-L-arabino-hexos-4-ulose\n",
      "C19961  UDP-4-amino-4,6-dideoxy-N-acetyl-beta-L-altrosamine\n",
      "C19971  UDP-2,4-bis(acetamido)-2,4,6-trideoxy-beta-L-altropyranose\n",
      "C19972  2,4-Bis(acetamido)-2,4,6-trideoxy-beta-L-altropyranose\n",
      "C20082  Pseudaminic acid\n",
      "C20083  CMP-pseudaminic acid\n",
      "C20357  UDP-N,N'-diacetylbacillosamine\n",
      "C20359  UDP-2-acetamido-3-amino-2,3-dideoxy-alpha-D-glucuronate\n",
      "C20395  UDP-2-acetamido-2-deoxy-alpha-D-ribo-hex-3-uluronate\n",
      "C20418  N,N'-Diacetyllegionaminate\n",
      "C20419  CMP-N,N'-diacetyllegionaminate\n",
      "C20424  2,4-Diacetamido-2,4,6-trideoxy-D-mannopyranose\n",
      "C20638  GDP-4-amino-4,6-dideoxy-alpha-D-mannose\n",
      "C20672  GDP-4-acetamido-4,6-dideoxy-alpha-D-mannose\n",
      "C20934  3-Deoxy-D-glycero-D-galacto-non-2-ulopyranosonate\n",
      "C21027  N-Acetylmuramic acid alpha-1-phosphate\n",
      "C21383  3-Deoxy-D-glycero-D-galacto-non-2-ulopyranosonate 9-phosphate\n",
      "C21384  CMP-3-deoxy-D-glycero-D-galacto-non-2-ulopyranosonate\n",
      "C22029  alpha-D-Galactosamine 1-phosphate\n",
      "REFERENCE   PMID:2793832\n",
      "AUTHORS   Wyk P, Reeves P\n",
      "TITLE     Identification and sequence of the gene for abequose synthase, which confers antigenic specificity on group B salmonellae: homology with galactose epimerase.\n",
      "JOURNAL   J Bacteriol 171:5687-93 (1989)\n",
      "DOI:10.1128/JB.171.10.5687-5693.1989\n",
      "REFERENCE   PMID:8071227\n",
      "AUTHORS   Thorson JS, Lo SF, Ploux O, He X, Liu HW\n",
      "TITLE     Studies of the biosynthesis of 3,6-dideoxyhexoses: molecular cloning and characterization of the asc (ascarylose) region from Yersinia pseudotuberculosis serogroup VA.\n",
      "JOURNAL   J Bacteriol 176:5483-93 (1994)\n",
      "DOI:10.1128/JB.176.17.5483-5493.1994\n",
      "REFERENCE   PMID:8444803\n",
      "AUTHORS   Kessler AC, Haase A, Reeves PR\n",
      "TITLE     Molecular analysis of the 3,6-dideoxyhexose pathway genes of Yersinia pseudotuberculosis serogroup IIA.\n",
      "JOURNAL   J Bacteriol 175:1412-22 (1993)\n",
      "DOI:10.1128/JB.175.5.1412-1422.1993\n",
      "REFERENCE   PMID:9811644\n",
      "AUTHORS   Danese PN, Oliver GR, Barr K, Bowman GD, Rick PD, Silhavy TJ\n",
      "TITLE     Accumulation of the enterobacterial common antigen lipid II biosynthetic intermediate stimulates degP transcription in Escherichia coli.\n",
      "JOURNAL   J Bacteriol 180:5875-84 (1998)\n",
      "DOI:10.1128/JB.180.22.5875-5884.1998\n",
      "REFERENCE   PMID:10931327\n",
      "AUTHORS   Skurnik M, Peippo A, Ervela E\n",
      "TITLE     Characterization of the O-antigen gene clusters of Yersinia pseudotuberculosis and the cryptic O-antigen gene cluster of Yersinia pestis shows that the plague bacillus is most closely related to and has evolved from Y. pseudotuberculosis serotype O:1b.\n",
      "JOURNAL   Mol Microbiol 37:316-30 (2000)\n",
      "DOI:10.1046/j.1365-2958.2000.01993.x\n",
      "REFERENCE   PMID:10984043\n",
      "AUTHORS   Stover CK, Pham XQ, Erwin AL, Mizoguchi SD, Warrener P, Hickey MJ, Brinkman FS, Hufnagle WO, Kowalik DJ, Lagrou M, Garber RL, Goltry L, Tolentino E, Westbrock-Wadman S, Yuan Y, Brody LL, Coulter SN, Folger KR, Kas A, Larbig K, Lim R, Smith K, Spencer D, Wong GK, Wu Z, Paulsen IT, Reizer J, Saier MH, Hancock RE, Lory S, Olson MV\n",
      "TITLE     Complete genome sequence of Pseudomonas aeruginosa PAO1, an opportunistic pathogen.\n",
      "JOURNAL   Nature 406:959-64 (2000)\n",
      "DOI:10.1038/35023079\n",
      "REFERENCE   PMID:11447132\n",
      "AUTHORS   Ringenberg M, Lichtensteiger C, Vimr E\n",
      "TITLE     Redirection of sialic acid metabolism in genetically engineered Escherichia coli.\n",
      "JOURNAL   Glycobiology 11:533-9 (2001)\n",
      "DOI:10.1093/glycob/11.7.533\n",
      "REFERENCE   PMID:12562791\n",
      "AUTHORS   McCoy AJ, Sandlin RC, Maurelli AT\n",
      "TITLE     In vitro and in vivo functional activity of Chlamydia MurA, a UDP-N-acetylglucosamine enolpyruvyl transferase involved in peptidoglycan synthesis and fosfomycin resistance.\n",
      "JOURNAL   J Bacteriol 185:1218-28 (2003)\n",
      "DOI:10.1128/JB.185.4.1218-1228.2003\n",
      "REFERENCE   PMID:14450717\n",
      "AUTHORS   JACOBSON B, DAVIDSON EA\n",
      "TITLE     Biosynthesis of uronic acids by skin enzymes. I. Uridine diphosphate-D-glucuronic acid-5-epimerase.\n",
      "JOURNAL   J Biol Chem 237:638-42 (1962)\n",
      "REFERENCE   PMID:15134748\n",
      "AUTHORS   Seifert GJ\n",
      "TITLE     Nucleotide sugar interconversions and cell wall biosynthesis: how to bring the inside to the outside.\n",
      "JOURNAL   Curr Opin Plant Biol 7:277-84 (2004)\n",
      "DOI:10.1016/j.pbi.2004.03.004\n",
      "REFERENCE   PMID:15489439\n",
      "AUTHORS   Uehara T, Park JT\n",
      "TITLE     The N-acetyl-D-glucosamine kinase of Escherichia coli and its role in murein recycling.\n",
      "JOURNAL   J Bacteriol 186:7273-9 (2004)\n",
      "DOI:10.1128/JB.186.21.7273-7279.2004\n",
      "REFERENCE   PMID:15695810\n",
      "AUTHORS   Breazeale SD, Ribeiro AA, McClerren AL, Raetz CR.\n",
      "TITLE     A formyltransferase required for polymyxin resistance in Escherichia coli and the modification of lipid A with 4-Amino-4-deoxy-L-arabinose. Identification and function oF UDP-4-deoxy-4-formamido-L-arabinose.\n",
      "JOURNAL   J Biol Chem 280:14154-67 (2005)\n",
      "DOI:10.1074/jbc.M414265200\n",
      "REFERENCE   PMID:15809294\n",
      "AUTHORS   Williams GJ, Breazeale SD, Raetz CR, Naismith JH.\n",
      "TITLE     Structure and function of both domains of ArnA, a dual function decarboxylase and a formyltransferase, involved in 4-amino-4-deoxy-L-arabinose biosynthesis.\n",
      "JOURNAL   J Biol Chem 280:23000-8 (2005)\n",
      "DOI:10.1074/jbc.M501534200\n",
      "REFERENCE   PMID:16237198\n",
      "AUTHORS   Maliekal P, Vertommen D, Delpierre G, Van Schaftingen E\n",
      "TITLE     Identification of the sequence encoding N-acetylneuraminate-9-phosphate phosphatase.\n",
      "JOURNAL   Glycobiology 16:165-72 (2006)\n",
      "DOI:10.1093/glycob/cwj050\n",
      "REFERENCE   PMID:16277754\n",
      "AUTHORS   Campbell CT, Yarema KJ\n",
      "TITLE     Large-scale approaches for glycobiology.\n",
      "JOURNAL   Genome Biol 6:236 (2005)\n",
      "DOI:10.1186/gb-2005-6-11-236\n",
      "REFERENCE   PMID:16452451\n",
      "AUTHORS   Uehara T, Suefuji K, Jaeger T, Mayer C, Park JT\n",
      "TITLE     MurQ Etherase is required by Escherichia coli in order to metabolize anhydro-N-acetylmuramic acid obtained either from the environment or from its own cell wall.\n",
      "JOURNAL   J Bacteriol 188:1660-2 (2006)\n",
      "DOI:10.1128/JB.188.4.1660-1662.2006\n",
      "REFERENCE   PMID:16995900\n",
      "AUTHORS   Lanot A, Hodge D, Jackson RG, George GL, Elias L, Lim EK, Vaistij FE, Bowles DJ\n",
      "TITLE     The glucosyltransferase UGT72E2 is responsible for monolignol 4-O-glucoside production in Arabidopsis thaliana.\n",
      "JOURNAL   Plant J 48:286-95 (2006)\n",
      "DOI:10.1111/j.1365-313X.2006.02872.x\n",
      "REFERENCE   PMID:17085508\n",
      "AUTHORS   Muller R, Morant M, Jarmer H, Nilsson L, Nielsen TH\n",
      "TITLE     Genome-wide analysis of the Arabidopsis leaf transcriptome reveals interaction of phosphate and sugar metabolism.\n",
      "JOURNAL   Plant Physiol 143:156-71 (2007)\n",
      "DOI:10.1104/pp.106.090167\n",
      "REFERENCE   PMID:17190829\n",
      "AUTHORS   Oka T, Nemoto T, Jigami Y\n",
      "TITLE     Functional analysis of Arabidopsis thaliana RHM2/MUM4, a multidomain protein involved in UDP-D-glucose to UDP-L-rhamnose conversion.\n",
      "JOURNAL   J Biol Chem 282:5389-403 (2007)\n",
      "DOI:10.1074/jbc.M610196200\n",
      "REFERENCE   PMID:17827295\n",
      "AUTHORS   Greenberg DE, Porcella SF, Zelazny AM, Virtaneva K, Sturdevant DE, Kupko JJ 3rd, Barbian KD, Babar A, Dorward DW, Holland SM.\n",
      "TITLE     Genome sequence analysis of the emerging human pathogenic acetic acid bacterium Granulibacter bethesdensis.\n",
      "JOURNAL   J Bacteriol 189:8727-36 (2007)\n",
      "DOI:10.1128/JB.00793-07\n",
      "REFERENCE   PMID:18199744\n",
      "AUTHORS   Kotake T, Hojo S, Tajima N, Matsuoka K, Koyama T, Tsumuraya Y\n",
      "TITLE     A bifunctional enzyme with L-fucokinase and GDP-L-fucose pyrophosphorylase activities salvages free L-fucose in Arabidopsis.\n",
      "JOURNAL   J Biol Chem 283:8125-35 (2008)\n",
      "DOI:10.1074/jbc.M710078200\n",
      "REFERENCE   PMID:18304669\n",
      "AUTHORS   Vorholter FJ, Schneiker S, Goesmann A, Krause L, Bekel T, Kaiser O, Linke B, Patschkowski T, Ruckert C, Schmid J, Sidhu VK, Sieber V, Tauch A, Watt SA, Weisshaar B, Becker A, Niehaus K, Puhler A.\n",
      "TITLE     The genome of Xanthomonas campestris pv. campestris B100 and its use for the reconstruction of metabolic pathways involved in xanthan biosynthesis.\n",
      "JOURNAL   J Biotechnol 134:33-45 (2008)\n",
      "DOI:10.1016/j.jbiotec.2007.12.013\n",
      "REFERENCE   PMID:11479276\n",
      "AUTHORS   Albermann C, Piepersberg W\n",
      "TITLE     Expression and identification of the RfbE protein from Vibrio cholerae O1 and its use for the enzymatic synthesis of GDP-D-perosamine.\n",
      "JOURNAL   Glycobiology 11:655-61 (2001)\n",
      "DOI:10.1093/glycob/11.8.655\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00040  Pentose and glucuronate interconversions\n",
      "ath00051  Fructose and mannose metabolism\n",
      "ath00052  Galactose metabolism\n",
      "ath00053  Ascorbate and aldarate metabolism\n",
      "ath00500  Starch and sucrose metabolism\n",
      "ath00510  N-Glycan biosynthesis\n",
      "KO_PATHWAY  ko00520\n",
      "///\n",
      "ENTRY       ath00531                    Pathway\n",
      "NAME        Glycosaminoglycan degradation - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00531  Glycosaminoglycan degradation\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G07830  GUS2; glucuronidase 2 [KO:K07964] [EC:3.2.1.166]\n",
      "AT5G34940  GUS3; glucuronidase 3 [KO:K07964] [EC:3.2.1.166]\n",
      "AT5G61250  GUS1; glucuronidase 1 [KO:K07964] [EC:3.2.1.166]\n",
      "AT5G27730  heparan-alpha-glucosaminide N-acetyltransferase-like protein (DUF1624) [KO:K10532] [EC:2.3.1.78]\n",
      "AT5G13690  CYL1; alpha-N-acetylglucosaminidase family / NAGLU family [KO:K01205] [EC:3.2.1.50]\n",
      "AT1G72990  BGAL17; beta-galactosidase 17 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G52840  BGAL2; beta-galactosidase 2 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G55260  HEXO1; beta-hexosaminidase 1 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G65590  HEXO3; beta-hexosaminidase 3 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G05590  HEXO2; beta-hexosaminidase 2 [KO:K12373] [EC:3.2.1.52]\n",
      "COMPOUND    C00426  Dermatan sulfate\n",
      "C00573  Keratan sulfate\n",
      "C00634  Chondroitin 4-sulfate\n",
      "C00925  Heparan sulfate\n",
      "G00526\n",
      "G00711\n",
      "G00872  DWA-2\n",
      "G01391\n",
      "G01945\n",
      "G01977\n",
      "G02632\n",
      "G09660\n",
      "G10008\n",
      "G10505  Hyaluronic acid\n",
      "G12336\n",
      "G13034\n",
      "G13035\n",
      "G13036\n",
      "G13037\n",
      "G13038\n",
      "G13039\n",
      "G13040\n",
      "G13041\n",
      "G13042\n",
      "G13043\n",
      "G13073\n",
      "G13074\n",
      "REFERENCE   PMID:27570\n",
      "AUTHORS   Greiling H, Stuhlsatz HW, Cantz M, Gehler J\n",
      "TITLE     Increased urinary excretion of keratan sulfate in fucosidosis.\n",
      "JOURNAL   J Clin Chem Clin Biochem 16:329-34 (1978)\n",
      "DOI:10.1515/cclm.1978.16.6.329\n",
      "REFERENCE   PMID:3161730\n",
      "AUTHORS   Fuchs W, Beck M, Kresse H\n",
      "TITLE     Intralysosomal formation and metabolic fate of N-acetylglucosamine 6-sulfate from keratan sulfate.\n",
      "JOURNAL   Eur J Biochem 151:551-6 (1985)\n",
      "DOI:10.1111/j.1432-1033.1985.tb09138.x\n",
      "KO_PATHWAY  ko00531\n",
      "///\n",
      "ENTRY       ath00561                    Pathway\n",
      "NAME        Glycerolipid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00561  Glycerolipid metabolism\n",
      "MODULE      ath_M00089  Triacylglycerol biosynthesis [PATH:ath00561]\n",
      "ath_M00098  Acylglycerol degradation [PATH:ath00561]\n",
      "DBLINKS     GO: 0045017 0046486 0046503\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G80380  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K15918] [EC:2.7.1.31]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT2G37790  AKR4C10; NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT5G01670  NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT1G48430  Dihydroxyacetone kinase [KO:K00863] [EC:2.7.1.28 2.7.1.29 4.6.1.15]\n",
      "AT1G80460  NHO1; Actin-like ATPase superfamily protein [KO:K00864] [EC:2.7.1.30]\n",
      "AT4G25840  GPP1; glycerol-3-phosphatase 1 [KO:K24189] [EC:3.1.3.21]\n",
      "AT5G57440  GS1; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K24189] [EC:3.1.3.21]\n",
      "AT5G60620  GPAT9; glycerol-3-phosphate acyltransferase 9 [KO:K13506] [EC:2.3.1.15]\n",
      "AT1G32200  ATS1; phospholipid/glycerol acyltransferase family protein [KO:K00630] [EC:2.3.1.15]\n",
      "AT1G06520  GPAT1; glycerol-3-phosphate acyltransferase 1 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT1G02390  GPAT2; glycerol-3-phosphate acyltransferase 2 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT4G01950  GPAT3; glycerol-3-phosphate acyltransferase 3 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT1G01610  GPAT4; glycerol-3-phosphate acyltransferase 4 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT2G38110  GPAT6; glycerol-3-phosphate acyltransferase 6 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT5G06090  GPAT7; glycerol-3-phosphate acyltransferase 7 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT3G11430  GPAT5; glycerol-3-phosphate acyltransferase 5 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT4G00400  GPAT8; glycerol-3-phosphate acyltransferase 8 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT4G30580  ATS2; Phospholipid/glycerol acyltransferase family protein [KO:K00655] [EC:2.3.1.51]\n",
      "AT1G51260  LPAT3; lysophosphatidyl acyltransferase 3 [KO:K13523] [EC:2.3.1.51 2.3.1.-]\n",
      "AT3G57650  LPAT2; lysophosphatidyl acyltransferase 2 [KO:K13523] [EC:2.3.1.51 2.3.1.-]\n",
      "AT1G75020  LPAT4; lysophosphatidyl acyltransferase 4 [KO:K13513] [EC:2.3.1.- 2.3.1.51]\n",
      "AT3G18850  LPAT5; lysophosphatidyl acyltransferase 5 [KO:K13513] [EC:2.3.1.- 2.3.1.51]\n",
      "AT1G12640  LPLAT1; MBOAT (membrane bound O-acyl transferase) family protein [KO:K13519] [EC:2.3.1.51 2.3.1.23 2.3.1.-]\n",
      "AT1G63050  LPLAT2; MBOAT (membrane bound O-acyl transferase) family protein [KO:K13519] [EC:2.3.1.51 2.3.1.23 2.3.1.-]\n",
      "AT3G09560  PAH1; Lipin family protein [KO:K15728] [EC:3.1.3.4]\n",
      "AT5G42870  PAH2; phosphatidic acid phosphohydrolase 2 [KO:K15728] [EC:3.1.3.4]\n",
      "AT3G02600  LPP3; lipid phosphate phosphatase 3 [KO:K18693] [EC:3.1.3.81 3.1.3.4]\n",
      "AT1G15080  LPP2; lipid phosphate phosphatase 2 [KO:K18693] [EC:3.1.3.81 3.1.3.4]\n",
      "AT2G01180  PAP1; phosphatidic acid phosphatase 1 [KO:K18693] [EC:3.1.3.81 3.1.3.4]\n",
      "AT3G18220  LPP4; Phosphatidic acid phosphatase (PAP2) family protein [KO:K18693] [EC:3.1.3.81 3.1.3.4]\n",
      "AT5G63770  DGK2; diacylglycerol kinase 2 [KO:K00901] [EC:2.7.1.107]\n",
      "AT5G07920  DGK1; diacylglycerol kinase1 [KO:K00901] [EC:2.7.1.107]\n",
      "AT5G57690  DGK4; diacylglycerol kinase 4 [KO:K00901] [EC:2.7.1.107]\n",
      "AT2G18730  DGK3; diacylglycerol kinase 3 [KO:K00901] [EC:2.7.1.107]\n",
      "AT4G30340  DGK7; diacylglycerol kinase 7 [KO:K00901] [EC:2.7.1.107]\n",
      "AT4G28130  DGK6; diacylglycerol kinase 6 [KO:K00901] [EC:2.7.1.107]\n",
      "AT2G20900  DGK5; diacylglycerol kinase 5 [KO:K00901] [EC:2.7.1.107]\n",
      "AT2G19450  TAG1; membrane bound O-acyl transferase (MBOAT) family protein [KO:K11155] [EC:2.3.1.20 2.3.1.75 2.3.1.76]\n",
      "AT3G51520  DGAT2; diacylglycerol acyltransferase family [KO:K22848] [EC:2.3.1.20]\n",
      "AT1G48300  DGAT3; diacylglycerol acyltransferase [KO:K22849] [EC:2.3.1.20]\n",
      "AT3G44830  Lecithin:cholesterol acyltransferase family protein [KO:K00679] [EC:2.3.1.158]\n",
      "AT5G13640  PDAT; phospholipid:diacylglycerol acyltransferase [KO:K00679] [EC:2.3.1.158]\n",
      "AT3G57140  SDP1-LIKE; sugar-dependent 1-like protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G04040  SDP1; Patatin-like phospholipase family protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT3G03520  NPC3; non-specific phospholipase C3 [KO:K23737] [EC:3.1.3.106]\n",
      "AT1G73480  alpha/beta-Hydrolases superfamily protein [KO:K01054] [EC:3.1.1.23]\n",
      "AT1G18360  alpha/beta-Hydrolases superfamily protein [KO:K01054] [EC:3.1.1.23]\n",
      "AT1G11090  alpha/beta-Hydrolases superfamily protein [KO:K01054] [EC:3.1.1.23]\n",
      "AT3G56040  UGP3; UDP-glucose pyrophosphorylase 3 [KO:K22920] [EC:2.7.7.9]\n",
      "AT4G33030  SQD1; sulfoquinovosyldiacylglycerol 1 [KO:K06118] [EC:3.13.1.1]\n",
      "AT5G01220  SQD2; sulfoquinovosyldiacylglycerol 2 [KO:K06119] [EC:2.4.1.-]\n",
      "AT2G11810  MGDC; monogalactosyldiacylglycerol synthase type C [KO:K03715] [EC:2.4.1.46]\n",
      "AT4G31780  MGD1; monogalactosyl diacylglycerol synthase 1 [KO:K03715] [EC:2.4.1.46]\n",
      "AT5G20410  MGD2; monogalactosyldiacylglycerol synthase 2 [KO:K03715] [EC:2.4.1.46]\n",
      "AT3G11670  DGD1; UDP-Glycosyltransferase superfamily protein [KO:K09480] [EC:2.4.1.241]\n",
      "AT4G00550  DGD2; digalactosyl diacylglycerol deficient 2 [KO:K09480] [EC:2.4.1.241]\n",
      "AT3G56310  Melibiase family protein [KO:K07407] [EC:3.2.1.22]\n",
      "AT5G08370  AGAL2; alpha-galactosidase 2 [KO:K07407] [EC:3.2.1.22]\n",
      "AT5G08380  AGAL1; alpha-galactosidase 1 [KO:K07407] [EC:3.2.1.22]\n",
      "AT3G06510  SFR2; Glycosyl hydrolase superfamily protein [KO:K21362] [EC:2.4.1.184]\n",
      "COMPOUND    C00029  UDP-glucose\n",
      "C00040  Acyl-CoA\n",
      "C00093  sn-Glycerol 3-phosphate\n",
      "C00103  D-Glucose 1-phosphate\n",
      "C00111  Glycerone phosphate\n",
      "C00116  Glycerol\n",
      "C00162  Fatty acid\n",
      "C00173  Acyl-[acyl-carrier protein]\n",
      "C00184  Glycerone\n",
      "C00197  3-Phospho-D-glycerate\n",
      "C00258  D-Glycerate\n",
      "C00416  Phosphatidate\n",
      "C00422  Triacylglycerol\n",
      "C00577  D-Glyceraldehyde\n",
      "C00631  2-Phospho-D-glycerate\n",
      "C00641  1,2-Diacyl-sn-glycerol\n",
      "C00681  1-Acyl-sn-glycerol 3-phosphate\n",
      "C00969  3-Hydroxypropanal\n",
      "C01885  1-Acylglycerol\n",
      "C02133  Acyl phosphate\n",
      "C02457  Propane-1,3-diol\n",
      "C03692  1,2-Diacyl-3-beta-D-galactosyl-sn-glycerol\n",
      "C04046  3-D-Glucosyl-1,2-diacylglycerol\n",
      "C04315  2-Acyl-3-O-(beta-D-galactosyl)-sn-glycerol\n",
      "C04872  1,2-Diacyl-3-[3-(alpha-D-N-acetylneuraminyl)-beta-D-galactosyl]-sn-glycerol\n",
      "C05401  3-beta-D-Galactosyl-sn-glycerol\n",
      "C06037  Digalactosyl-diacylglycerol\n",
      "C06038  Acyl1-monogalactosyl-diacylglycerol\n",
      "C06040  Diglucosyldiacylglycerol\n",
      "C06041  Glycerophosphoglycoglycerolipid\n",
      "C06042  Lipoteichoic acid\n",
      "C06364  1,2-Diacyl-3-alpha-D-glucosyl-sn-glycerol\n",
      "C06365  alpha-Kojibiosyldiacylglycerol\n",
      "C11521  UDP-6-sulfoquinovose\n",
      "C13508  Sulfoquinovosyldiacylglycerol\n",
      "C20897  Glycerophosphoglycoglycerolipid\n",
      "C20898  Lipoteichoic acid\n",
      "C20991  1,2-Diacyl-3-O-[beta-D-galactosyl-(1->6)-beta-D-galactosyl]-sn-glycerol\n",
      "REFERENCE   PMID:8662716\n",
      "AUTHORS   Norbeck J, Pahlman AK, Akhtar N, Blomberg A, Adler L.\n",
      "TITLE     Purification and characterization of two isoenzymes of DL-glycerol-3-phosphatase from Saccharomyces cerevisiae. Identification of the corresponding GPP1 and GPP2 genes and evidence for osmotic regulation of Gpp2p expression by the osmosensing mitogen-activated protein kinase signal transduction pathway.\n",
      "JOURNAL   J Biol Chem 271:13875-81 (1996)\n",
      "DOI:10.1074/jbc.271.23.13875\n",
      "REFERENCE   PMID:8995384\n",
      "AUTHORS   Karlsson OP, Dahlqvist A, Vikstrom S, Wieslander A.\n",
      "TITLE     Lipid dependence and basic kinetics of the purified 1,2-diacylglycerol 3-glucosyltransferase from membranes of Acholeplasma laidlawii.\n",
      "JOURNAL   J Biol Chem 272:929-36 (1997)\n",
      "DOI:10.1074/jbc.272.2.929\n",
      "REFERENCE   PMID:11294844\n",
      "AUTHORS   Berg S, Edman M, Li L, Wikstrom M, Wieslander A.\n",
      "TITLE     Sequence properties of the 1,2-diacylglycerol 3-glucosyltransferase from Acholeplasma laidlawii membranes. Recognition of a large group of lipid glycosyltransferases in eubacteria and archaea.\n",
      "JOURNAL   J Biol Chem 276:22056-63 (2001)\n",
      "DOI:10.1074/jbc.M102576200\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00071  Fatty acid degradation\n",
      "ath00564  Glycerophospholipid metabolism\n",
      "KO_PATHWAY  ko00561\n",
      "///\n",
      "ENTRY       ath00562                    Pathway\n",
      "NAME        Inositol phosphate metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00562  Inositol phosphate metabolism\n",
      "MODULE      ath_M00132  Inositol phosphate metabolism, Ins(1,3,4)P3 => phytate [PATH:ath00562]\n",
      "DBLINKS     GO: 0006020\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G60490  VPS34; vacuolar protein sorting 34 [KO:K00914] [EC:2.7.1.137]\n",
      "AT5G04540  MTM2; Myotubularin-like phosphatases II superfamily [KO:K18081] [EC:3.1.3.64 3.1.3.95]\n",
      "AT3G10550  MTM1; Myotubularin-like phosphatases II superfamily [KO:K18081] [EC:3.1.3.64 3.1.3.95]\n",
      "AT1G49340  ATPI4K_ALPHA; Phosphatidylinositol 3- and 4-kinase family protein [KO:K00888] [EC:2.7.1.67]\n",
      "AT1G51040  Protein kinase superfamily protein [KO:K00888] [EC:2.7.1.67]\n",
      "AT5G09350  PI-4KBETA2; phosphatidylinositol 4-OH kinase beta2 [KO:K19801] [EC:2.7.1.67]\n",
      "AT5G64070  PI-4KBETA1; phosphatidylinositol 4-OH kinase beta1 [KO:K19801] [EC:2.7.1.67]\n",
      "AT3G51460  RHD4; Phosphoinositide phosphatase family protein [KO:K21797] [EC:3.1.3.-]\n",
      "AT5G66020  ATSAC1B; Phosphoinositide phosphatase family protein [KO:K21797] [EC:3.1.3.-]\n",
      "AT3G51830  SAC8; SAC domain-containing protein 8 [KO:K21797] [EC:3.1.3.-]\n",
      "AT2G26420  PIP5K3; 1-phosphatidylinositol-4-phosphate 5-kinase 3 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G10900  Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G21980  PIP5K1; phosphatidylinositol-4-phosphate 5-kinase 1 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G60890  Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G77740  PIP5K2; phosphatidylinositol-4-phosphate 5-kinase 2 [KO:K00889] [EC:2.7.1.68]\n",
      "AT2G41210  PIP5K5; phosphatidylinositol- 4-phosphate 5-kinase 5 [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G07960  PIP5K6; Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G56960  PIP5K4; phosphatidyl inositol monophosphate 5 kinase 4 [KO:K00889] [EC:2.7.1.68]\n",
      "AT4G01190  PIPK10; phosphatidylinositol phosphate kinase 10 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G01460  PIPK11; Phosphatidylinositol-4-phosphate 5-kinase, core [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G09920  PIP5K9; phosphatidyl inositol monophosphate 5 kinase [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G19420  PEN2; PTEN 2 [KO:K01110] [EC:3.1.3.16 3.1.3.48 3.1.3.67]\n",
      "AT3G50110  PEN3; PTEN 3 [KO:K01110] [EC:3.1.3.16 3.1.3.48 3.1.3.67]\n",
      "AT5G39400  PTEN1; Calcium/lipid-binding (CaLB) phosphatase [KO:K01110] [EC:3.1.3.16 3.1.3.48 3.1.3.67]\n",
      "AT4G33240  FAB1A; 1-phosphatidylinositol-3-phosphate 5-kinase FAB1A [KO:K00921] [EC:2.7.1.150]\n",
      "AT3G14270  FAB1B; phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00921] [EC:2.7.1.150]\n",
      "AT1G34260  FAB1D; FORMS APLOID AND BINUCLEATE CELLS 1A [KO:K00921] [EC:2.7.1.150]\n",
      "AT1G71010  FAB1C; FORMS APLOID AND BINUCLEATE CELLS 1C [KO:K00921] [EC:2.7.1.150]\n",
      "AT3G14205  Phosphoinositide phosphatase family protein [KO:K22913] [EC:3.1.3.-]\n",
      "AT5G20840  Phosphoinositide phosphatase family protein [KO:K22913] [EC:3.1.3.-]\n",
      "AT1G22620  ATSAC1; Phosphoinositide phosphatase family protein [KO:K22913] [EC:3.1.3.-]\n",
      "AT3G43220  Phosphoinositide phosphatase family protein [KO:K22913] [EC:3.1.3.-]\n",
      "AT1G17340  Phosphoinositide phosphatase family protein [KO:K22913] [EC:3.1.3.-]\n",
      "AT2G40116  Phosphoinositide-specific phospholipase C family protein [KO:K05857] [EC:3.1.4.11]\n",
      "AT3G08510  PLC2; phospholipase C 2 [KO:K05857] [EC:3.1.4.11]\n",
      "AT5G58690  PLC5; phosphatidylinositol-speciwc phospholipase C5 [KO:K05857] [EC:3.1.4.11]\n",
      "AT3G55940  Phosphoinositide-specific phospholipase C family protein [KO:K05857] [EC:3.1.4.11]\n",
      "AT5G58700  PLC4; phosphatidylinositol-speciwc phospholipase C4 [KO:K05857] [EC:3.1.4.11]\n",
      "AT5G58670  PLC1; phospholipase C1 [KO:K05857] [EC:3.1.4.11]\n",
      "AT4G38530  PLC1; phospholipase C1 [KO:K05857] [EC:3.1.4.11]\n",
      "AT1G07230  NPC1; non-specific phospholipase C1 [KO:K01114] [EC:3.1.4.3]\n",
      "AT2G26870  NPC2; non-specific phospholipase C2 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G48610  NPC6; non-specific phospholipase C6 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G03540  NPC5; non-specific phospholipase C5 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G03530  NPC4; non-specific phospholipase C4 [KO:K01114] [EC:3.1.4.3]\n",
      "AT1G31190  IMPL1; myo-inositol monophosphatase like 1 [KO:K01092] [EC:3.1.3.25]\n",
      "AT3G02870  VTC4; Inositol monophosphatase family protein [KO:K10047] [EC:3.1.3.25 3.1.3.93]\n",
      "AT4G39120  IMPL2; inositol monophosphatase family protein [KO:K18649] [EC:3.1.3.25 3.1.3.93 3.1.3.15]\n",
      "AT4G38570  PIS2; Putative CDP-diacylglycerol-inositol 3-phosphatidyltransferase 2 [KO:K00999] [EC:2.7.8.11]\n",
      "AT1G68000  PIS1; phosphatidylinositol synthase 1 [KO:K00999] [EC:2.7.8.11]\n",
      "AT4G39800  MIPS1; myo-inositol-1-phosphate synthase 1 [KO:K01858] [EC:5.5.1.4]\n",
      "AT5G10170  MIPS3; myo-inositol-1-phosphate synthase 3 [KO:K01858] [EC:5.5.1.4]\n",
      "AT2G22240  MIPS2; myo-inositol-1-phosphate synthase 2 [KO:K01858] [EC:5.5.1.4]\n",
      "AT5G09290  Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G63980  SAL1; SAL1 phosphatase-like protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G64000  SAL2; Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G63990  Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G58730  Mik; pfkB-like carbohydrate kinase family protein [KO:K19517] [EC:2.7.1.64]\n",
      "AT1G34120  IP5PI; inositol polyphosphate 5-phosphatase I [KO:K24221] [EC:3.1.3.56]\n",
      "AT4G18010  IP5PII; myo-inositol polyphosphate 5-phosphatase 2 [KO:K24221] [EC:3.1.3.56]\n",
      "AT1G09870  histidine acid phosphatase family protein [KO:K03103] [EC:3.1.3.62 3.1.3.80]\n",
      "AT4G08170  Inositol 1,3,4-trisphosphate 5/6-kinase family protein [KO:K00913] [EC:2.7.1.159 2.7.1.134]\n",
      "AT5G16760  ITPK1; Inositol 1,3,4-trisphosphate 5/6-kinase family protein [KO:K00913] [EC:2.7.1.159 2.7.1.134]\n",
      "AT4G33770  Inositol 1,3,4-trisphosphate 5/6-kinase family protein [KO:K00913] [EC:2.7.1.159 2.7.1.134]\n",
      "AT2G43980  ITPK4; inositol 1,3,4-trisphosphate 5/6-kinase 4 [KO:K01765] [EC:2.7.1.159]\n",
      "AT5G61760  IPK2BETA; inositol polyphosphate kinase 2 beta [KO:K00915] [EC:2.7.1.140 2.7.1.151]\n",
      "AT5G07370  IPK2a; inositol polyphosphate kinase 2 alpha [KO:K00915] [EC:2.7.1.140 2.7.1.151]\n",
      "AT5G42810  IPK1; inositol-pentakisphosphate 2-kinase 1 [KO:K10572] [EC:2.7.1.158]\n",
      "AT1G22100  Inositol-pentakisphosphate 2-kinase family protein [KO:K10572] [EC:2.7.1.158]\n",
      "AT1G59312  Inositol-pentakisphosphate 2-kinase family protein [KO:K10572] [EC:2.7.1.158]\n",
      "AT1G58643  Inositol-pentakisphosphate 2-kinase family protein [KO:K10572] [EC:2.7.1.158]\n",
      "AT1G58936  Inositol-pentakisphosphate 2-kinase family protein [KO:K10572] [EC:2.7.1.158]\n",
      "AT2G19800  MIOX2; myo-inositol oxygenase 2 [KO:K00469] [EC:1.13.99.1]\n",
      "AT1G14520  MIOX1; myo-inositol oxygenase 1 [KO:K00469] [EC:1.13.99.1]\n",
      "AT4G26260  MIOX4; myo-inositol oxygenase 4 [KO:K00469] [EC:1.13.99.1]\n",
      "AT5G56640  MIOX5; myo-inositol oxygenase 5 [KO:K00469] [EC:1.13.99.1]\n",
      "AT2G14170  ALDH6B2; aldehyde dehydrogenase 6B2 [KO:K00140] [EC:1.2.1.18 1.2.1.27]\n",
      "AT3G55440  TPI; triosephosphate isomerase [KO:K01803] [EC:5.3.1.1]\n",
      "AT2G21170  TIM; triosephosphate isomerase [KO:K01803] [EC:5.3.1.1]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00092  D-Glucose 6-phosphate\n",
      "C00111  Glycerone phosphate\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00137  myo-Inositol\n",
      "C00191  D-Glucuronate\n",
      "C00222  3-Oxopropanoate\n",
      "C00558  D-Tagaturonate\n",
      "C00641  1,2-Diacyl-sn-glycerol\n",
      "C00691  2,4,6/3,5-Pentahydroxycyclohexanone\n",
      "C01177  Inositol 1-phosphate\n",
      "C01194  1-Phosphatidyl-D-myo-inositol\n",
      "C01204  Phytic acid\n",
      "C01220  1D-myo-Inositol 1,4-bisphosphate\n",
      "C01243  1D-myo-Inositol 1,3,4-trisphosphate\n",
      "C01245  D-myo-Inositol 1,4,5-trisphosphate\n",
      "C01272  1D-myo-Inositol 1,3,4,5-tetrakisphosphate\n",
      "C01277  1-Phosphatidyl-1D-myo-inositol 4-phosphate\n",
      "C01284  1D-myo-Inositol 1,3,4,5,6-pentakisphosphate\n",
      "C03546  myo-Inositol 4-phosphate\n",
      "C03659  D-Bornesitol\n",
      "C03660  L-Bornesitol\n",
      "C04006  1D-myo-Inositol 3-phosphate\n",
      "C04062  D-myo-Inositol 1,3-bisphosphate\n",
      "C04063  D-myo-Inositol 3,4-bisphosphate\n",
      "C04287  3D-3,5/4-Trihydroxycyclohexane-1,2-dione\n",
      "C04477  1D-myo-Inositol 1,3,4,6-tetrakisphosphate\n",
      "C04520  1D-myo-Inositol 3,4,5,6-tetrakisphosphate\n",
      "C04549  1-Phosphatidyl-1D-myo-inositol 3-phosphate\n",
      "C04563  D-myo-Inositol 1,2,4,5,6-pentakisphosphate\n",
      "C04579  Inositol 1,2,3,5,6-pentakisphosphate\n",
      "C04637  1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\n",
      "C05981  Phosphatidylinositol-3,4,5-trisphosphate\n",
      "C06153  scyllo-Inositol\n",
      "C06892  2-Deoxy-5-keto-D-gluconic acid\n",
      "C06893  2-Deoxy-5-keto-D-gluconic acid 6-phosphate\n",
      "C11554  1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\n",
      "C11555  1D-myo-Inositol 1,4,5,6-tetrakisphosphate\n",
      "C11556  1-Phosphatidyl-1D-myo-inositol 3,5-bisphosphate\n",
      "C11557  1-Phosphatidyl-1D-myo-inositol 5-phosphate\n",
      "C16737  5-Deoxy-D-glucuronate\n",
      "C19794  CDP-1L-myo-inositol\n",
      "C19799  Bis(1L-myo-inositol)-3,1'-phosphate 1-phosphate\n",
      "C19891  1D-chiro-Inositol\n",
      "C20251  1-Keto-D-chiro-inositol\n",
      "C21643  3-Dehydro-scyllo-inosose\n",
      "C21644  5-Dehydro-L-gluconate\n",
      "REFERENCE   PMID:14993306\n",
      "AUTHORS   Yoshida K, Yamaguchi M, Ikeda H, Omae K, Tsurusaki K, Fujita Y\n",
      "TITLE     The fifth gene of the iol operon of Bacillus subtilis, iolE, encodes 2-keto-myo-inositol dehydratase.\n",
      "JOURNAL   Microbiology 150:571-80 (2004)\n",
      "DOI:10.1099/mic.0.26768-0\n",
      "REFERENCE   PMID:18310071\n",
      "AUTHORS   Yoshida K, Yamaguchi M, Morinaga T, Kinehara M, Ikeuchi M, Ashida H, Fujita Y.\n",
      "TITLE     myo-Inositol catabolism in Bacillus subtilis.\n",
      "JOURNAL   J Biol Chem 283:10415-24 (2008)\n",
      "DOI:10.1074/jbc.M708043200\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00040  Pentose and glucuronate interconversions\n",
      "ath00564  Glycerophospholipid metabolism\n",
      "KO_PATHWAY  ko00562\n",
      "///\n",
      "ENTRY       ath00563                    Pathway\n",
      "NAME        Glycosylphosphatidylinositol (GPI)-anchor biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Cell surface proteins can be attached to the cell membrane via the glycolipid structure called glycosylphosphatidylinositol (GPI) anchor. Hundreds of GPI-anchored proteins have been identified in many eukaryotes ranging from protozoa and fungi to mammals. All protein-linked GPI anchors share a common core structure, characterized by the substructure Man (a1-4) GlcN (a1-6) myo-inositol-1P-lipid. Biosynthesis of GPI anchors proceeds in three stages: (i) preassembly of a GPI precursor in the ER membrane, (ii) attachment of the GPI to the C-terminus of a newly synthesized protein in the lumen of the ER, and (iii) lipid remodeling and/or carbohydrate side-chain modifications in the ER and the Golgi. Defects of GPI anchor biosynthesis gene result in a genetic disorder, paroxysmal nocturnal hemoglobinuria.\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00563  Glycosylphosphatidylinositol (GPI)-anchor biosynthesis\n",
      "DBLINKS     GO: 0006506\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G45100  SETH2; UDP-Glycosyltransferase superfamily protein [KO:K03857] [EC:2.4.1.198]\n",
      "AT2G34980  SETH1; phosphatidylinositolglycan synthase family protein [KO:K03859]\n",
      "AT4G35530  phosphatidylinositolglycan-like protein [KO:K03858]\n",
      "AT1G61280  Phosphatidylinositol N-acetylglucosaminyltransferase, GPI19/PIG-P subunit [KO:K03861]\n",
      "AT2G39445  Phosphatidylinositol N-acetylglucosaminyltransferase, GPI19/PIG-P subunit [KO:K03861]\n",
      "AT3G57170  N-acetylglucosaminyl transferase component family protein / Gpi1 family protein [KO:K03860]\n",
      "AT1G74340  DPMS2; dolichol phosphate-mannose biosynthesis regulatory protein-like protein [KO:K09658]\n",
      "AT2G27340  N-acetylglucosaminylphosphatidylinositol de-N-acetylase family protein [KO:K03434] [EC:3.5.1.89]\n",
      "AT3G58130  N-acetylglucosaminylphosphatidylinositol de-N-acetylase family protein [KO:K03434] [EC:3.5.1.89]\n",
      "AT4G17910  transferases, transferring acyl groups [KO:K05283] [EC:2.3.-.-]\n",
      "AT5G22130  PNT1; mannosyltransferase family protein [KO:K05284] [EC:2.4.1.-]\n",
      "AT5G46850  phosphatidylinositol-glycan biosynthesis class X-like protein [KO:K07541]\n",
      "AT1G11880  transferases, transferring hexosyl groups [KO:K07542] [EC:2.4.1.-]\n",
      "AT3G01380  sulfatase and phosphatidylinositolglycan class N domain-containing protein [KO:K05285] [EC:2.7.-.-]\n",
      "AT5G14850  Alg9-like mannosyltransferase family [KO:K05286] [EC:2.4.1.-]\n",
      "AT5G17250  Alkaline-phosphatase-like family protein [KO:K05288] [EC:2.7.-.-]\n",
      "AT1G16040  phosphatidylinositol-glycan biosynthesis class F-like protein [KO:K05287]\n",
      "AT2G22530  Alkaline-phosphatase-like family protein [KO:K05310] [EC:2.7.-.-]\n",
      "AT5G19130  GPI transamidase component family protein / Gaa1-like family protein [KO:K05289]\n",
      "AT1G08750  Peptidase C13 family [KO:K05290]\n",
      "AT3G07180  GPI transamidase component PIG-S-like protein [KO:K05291]\n",
      "AT3G07140  GPI transamidase component Gpi16 subunit family protein [KO:K05292]\n",
      "AT1G63110  GPI transamidase subunit PIG-U [KO:K05293]\n",
      "AT1G12730  GPI transamidase subunit PIG-U [KO:K05293]\n",
      "AT3G27325  hydrolases, acting on ester bond [KO:K05294] [EC:3.-.-.-]\n",
      "AT1G53710  Calcineurin-like metallo-phosphoesterase superfamily protein [KO:K23362] [EC:3.1.-.-]\n",
      "COMPOUND    C00154  Palmitoyl-CoA\n",
      "C00346  Ethanolamine phosphate\n",
      "C00350  Phosphatidylethanolamine\n",
      "C01194  1-Phosphatidyl-D-myo-inositol\n",
      "G00143\n",
      "G00144\n",
      "G00145\n",
      "G00146\n",
      "G00148\n",
      "G00149\n",
      "G10610  UDP-N-acetyl-D-glucosamine\n",
      "G10617  Dolichyl phosphate D-mannose\n",
      "G12396  6-(alpha-D-glucosaminyl)-1D-myo-inositol\n",
      "G13044\n",
      "G13046\n",
      "G13128\n",
      "G13150\n",
      "G13151\n",
      "G13152\n",
      "REFERENCE   PMID:12655644\n",
      "AUTHORS   Eisenhaber B, Maurer-Stroh S, Novatchkova M, Schneider G, Eisenhaber F.\n",
      "TITLE     Enzymes and auxiliary factors for GPI lipid anchor biosynthesis and post-translational transfer to proteins.\n",
      "JOURNAL   Bioessays 25:367-85 (2003)\n",
      "DOI:10.1002/bies.10254\n",
      "REFERENCE   PMID:11102867\n",
      "AUTHORS   Kinoshita T, Inoue N.\n",
      "TITLE     Dissecting and manipulating the pathway for glycosylphos-phatidylinositol-anchor biosynthesis.\n",
      "JOURNAL   Curr Opin Chem Biol 4:632-8 (2000)\n",
      "DOI:10.1016/S1367-5931(00)00151-4\n",
      "REFERENCE   PMID:8253761\n",
      "AUTHORS   Fankhauser C, Homans SW, Thomas-Oates JE, McConville MJ, Desponds C, Conzelmann A, Ferguson MA.\n",
      "TITLE     Structures of glycosylphosphatidylinositol membrane anchors from Saccharomyces cerevisiae.\n",
      "JOURNAL   J Biol Chem 268:26365-74 (1993)\n",
      "REFERENCE   PMID:14517336\n",
      "AUTHORS   Murakami Y, Siripanyapinyo U, Hong Y, Kang JY, Ishihara S, Nakakuma H, Maeda Y, Kinoshita T.\n",
      "TITLE     PIG-W is critical for inositol acylation but not for flipping of glycosylphosphatidylinositol-anchor.\n",
      "JOURNAL   Mol Biol Cell 14:4285-95 (2003)\n",
      "DOI:10.1091/mbc.E03-03-0193\n",
      "REFERENCE   PMID:10329735\n",
      "AUTHORS   Benachour A, Sipos G, Flury I, Reggiori F, Canivenc-Gansel E, Vionnet C, Conzelmann A, Benghezal M.\n",
      "TITLE     Deletion of GPI7, a yeast gene required for addition of a side chain to the glycosylphosphatidylinositol (GPI) core structure, affects GPI protein transport, remodeling, and cell wall integrity.\n",
      "JOURNAL   J Biol Chem 274:15251-61 (1999)\n",
      "DOI:10.1074/jbc.274.21.15251\n",
      "REFERENCE   PMID:14734546\n",
      "AUTHORS   Tanaka S, Maeda Y, Tashima Y, Kinoshita T.\n",
      "TITLE     Inositol deacylation of glycosylphosphatidylinositol-anchored proteins is mediated by mammalian PGAP1 and yeast Bst1p.\n",
      "JOURNAL   J Biol Chem 279:14256-63 (2004)\n",
      "DOI:10.1074/jbc.M313755200\n",
      "REFERENCE   PMID:12802054\n",
      "AUTHORS   Hong Y, Ohishi K, Kang JY, Tanaka S, Inoue N, Nishimura J, Maeda Y, Kinoshita T.\n",
      "TITLE     Human PIG-U and yeast Cdc91p are the fifth subunit of GPI transamidase that attaches GPI-anchors to proteins.\n",
      "JOURNAL   Mol Biol Cell 14:1780-9 (2003)\n",
      "DOI:10.1091/mbc.E02-12-0794\n",
      "REFERENCE   PMID:12807000\n",
      "AUTHORS   Tanaka S, Maeda Y, Kinoshita T.\n",
      "TITLE     [Biosynthesis of GPI-anchored proteins]\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 48:990-6 (2003)\n",
      "REFERENCE   PMID:15623507\n",
      "AUTHORS   Kang JY, Hong Y, Ashida H, Shishioh N, Murakami Y, Morita YS, Maeda Y, Kinoshita T.\n",
      "TITLE     PIG-V involved in transferring the second mannose in glycosylphosphatidylinositol.\n",
      "JOURNAL   J Biol Chem 280:9489-97 (2005)\n",
      "DOI:10.1074/jbc.M413867200\n",
      "REFERENCE   PMID:15635094\n",
      "AUTHORS   Ashida H, Hong Y, Murakami Y, Shishioh N, Sugimoto N, Kim YU, Maeda Y, Kinoshita T.\n",
      "TITLE     Mammalian PIG-X and yeast Pbn1p are the essential components of glycosylphosphatidylinositol-mannosyltransferase I.\n",
      "JOURNAL   Mol Biol Cell 16:1439-48 (2005)\n",
      "DOI:10.1091/mbc.E04-09-0802\n",
      "REFERENCE   PMID:24364251\n",
      "AUTHORS   Fujita M\n",
      "TITLE     [Biosynthesis and remodeling of GPI-anchored proteins].\n",
      "JOURNAL   Seikagaku 85:985-95 (2013)\n",
      "REFERENCE   PMID:25509331\n",
      "AUTHORS   Kinoshita T\n",
      "TITLE     [Biochemistry of glycosylphosphatidylinositol (GPI) anchored proteins].\n",
      "JOURNAL   Seikagaku 86:626-36 (2014)\n",
      "REFERENCE   PMID:19837036\n",
      "AUTHORS   Fujita M, Maeda Y, Ra M, Yamaguchi Y, Taguchi R, Kinoshita T\n",
      "TITLE     GPI glycan remodeling by PGAP5 regulates transport of GPI-anchored proteins from the ER to the Golgi.\n",
      "JOURNAL   Cell 139:352-65 (2009)\n",
      "DOI:10.1016/j.cell.2009.08.040\n",
      "REFERENCE   PMID:26563290\n",
      "AUTHORS   Kinoshita T, Fujita M\n",
      "TITLE     Biosynthesis of GPI-anchored proteins: special emphasis on GPI lipid remodeling.\n",
      "JOURNAL   J Lipid Res 57:6-24 (2016)\n",
      "DOI:10.1194/jlr.R063313\n",
      "REL_PATHWAY ath00510  N-Glycan biosynthesis\n",
      "ath00564  Glycerophospholipid metabolism\n",
      "KO_PATHWAY  ko00563\n",
      "///\n",
      "ENTRY       ath00564                    Pathway\n",
      "NAME        Glycerophospholipid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00564  Glycerophospholipid metabolism\n",
      "MODULE      ath_M00092  Phosphatidylethanolamine (PE) biosynthesis, ethanolamine => PE [PATH:ath00564]\n",
      "DBLINKS     GO: 0006650 0046474 0046475\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G40610  GPDHp; NAD-dependent glycerol-3-phosphate dehydrogenase family protein [KO:K00006] [EC:1.1.1.8]\n",
      "AT2G40690  GLY1; NAD-dependent glycerol-3-phosphate dehydrogenase family protein [KO:K00006] [EC:1.1.1.8]\n",
      "AT2G41540  GPDHC1; 6-phosphogluconate dehydrogenase family protein [KO:K00006] [EC:1.1.1.8]\n",
      "AT3G07690  6-phosphogluconate dehydrogenase family protein [KO:K00006] [EC:1.1.1.8]\n",
      "AT3G10370  SDP6; FAD-dependent oxidoreductase family protein [KO:K00111] [EC:1.1.5.3]\n",
      "AT5G60620  GPAT9; glycerol-3-phosphate acyltransferase 9 [KO:K13506] [EC:2.3.1.15]\n",
      "AT1G32200  ATS1; phospholipid/glycerol acyltransferase family protein [KO:K00630] [EC:2.3.1.15]\n",
      "AT1G06520  GPAT1; glycerol-3-phosphate acyltransferase 1 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT1G02390  GPAT2; glycerol-3-phosphate acyltransferase 2 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT4G01950  GPAT3; glycerol-3-phosphate acyltransferase 3 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT1G01610  GPAT4; glycerol-3-phosphate acyltransferase 4 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT2G38110  GPAT6; glycerol-3-phosphate acyltransferase 6 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT5G06090  GPAT7; glycerol-3-phosphate acyltransferase 7 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT3G11430  GPAT5; glycerol-3-phosphate acyltransferase 5 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT4G00400  GPAT8; glycerol-3-phosphate acyltransferase 8 [KO:K13508] [EC:2.3.1.15 2.3.1.198]\n",
      "AT4G30580  ATS2; Phospholipid/glycerol acyltransferase family protein [KO:K00655] [EC:2.3.1.51]\n",
      "AT1G51260  LPAT3; lysophosphatidyl acyltransferase 3 [KO:K13523] [EC:2.3.1.51 2.3.1.-]\n",
      "AT3G57650  LPAT2; lysophosphatidyl acyltransferase 2 [KO:K13523] [EC:2.3.1.51 2.3.1.-]\n",
      "AT1G75020  LPAT4; lysophosphatidyl acyltransferase 4 [KO:K13513] [EC:2.3.1.- 2.3.1.51]\n",
      "AT3G18850  LPAT5; lysophosphatidyl acyltransferase 5 [KO:K13513] [EC:2.3.1.- 2.3.1.51]\n",
      "AT1G12640  LPLAT1; MBOAT (membrane bound O-acyl transferase) family protein [KO:K13519] [EC:2.3.1.51 2.3.1.23 2.3.1.-]\n",
      "AT1G63050  LPLAT2; MBOAT (membrane bound O-acyl transferase) family protein [KO:K13519] [EC:2.3.1.51 2.3.1.23 2.3.1.-]\n",
      "AT3G57140  SDP1-LIKE; sugar-dependent 1-like protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G04040  SDP1; Patatin-like phospholipase family protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT4G24730  manganese-dependent ADP-ribose/CDP-alcohol diphosphatase-like protein [KO:K01517] [EC:3.6.1.13 3.6.1.16 3.6.1.53]\n",
      "AT3G09560  PAH1; Lipin family protein [KO:K15728] [EC:3.1.3.4]\n",
      "AT5G42870  PAH2; phosphatidic acid phosphohydrolase 2 [KO:K15728] [EC:3.1.3.4]\n",
      "AT3G02600  LPP3; lipid phosphate phosphatase 3 [KO:K18693] [EC:3.1.3.81 3.1.3.4]\n",
      "AT1G15080  LPP2; lipid phosphate phosphatase 2 [KO:K18693] [EC:3.1.3.81 3.1.3.4]\n",
      "AT2G01180  PAP1; phosphatidic acid phosphatase 1 [KO:K18693] [EC:3.1.3.81 3.1.3.4]\n",
      "AT3G18220  LPP4; Phosphatidic acid phosphatase (PAP2) family protein [KO:K18693] [EC:3.1.3.81 3.1.3.4]\n",
      "AT5G63770  DGK2; diacylglycerol kinase 2 [KO:K00901] [EC:2.7.1.107]\n",
      "AT5G07920  DGK1; diacylglycerol kinase1 [KO:K00901] [EC:2.7.1.107]\n",
      "AT5G57690  DGK4; diacylglycerol kinase 4 [KO:K00901] [EC:2.7.1.107]\n",
      "AT2G18730  DGK3; diacylglycerol kinase 3 [KO:K00901] [EC:2.7.1.107]\n",
      "AT4G30340  DGK7; diacylglycerol kinase 7 [KO:K00901] [EC:2.7.1.107]\n",
      "AT4G28130  DGK6; diacylglycerol kinase 6 [KO:K00901] [EC:2.7.1.107]\n",
      "AT2G20900  DGK5; diacylglycerol kinase 5 [KO:K00901] [EC:2.7.1.107]\n",
      "AT1G07230  NPC1; non-specific phospholipase C1 [KO:K01114] [EC:3.1.4.3]\n",
      "AT2G26870  NPC2; non-specific phospholipase C2 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G48610  NPC6; non-specific phospholipase C6 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G03540  NPC5; non-specific phospholipase C5 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G03530  NPC4; non-specific phospholipase C4 [KO:K01114] [EC:3.1.4.3]\n",
      "AT2G42010  PLDBETA1; phospholipase D beta 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT5G25370  PLDALPHA3; phospholipase D alpha 3 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11840  PLDGAMMA3; phospholipase D gamma 3 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11850  PLDGAMMA1; phospholipase D gamma 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT1G55180  PLDEPSILON; phospholipase D alpha 4 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G05630  PLDP2; phospholipase D P2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G15730  PLDALPHA1; phospholipase D alpha 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G16785  PLDP1; phospholipase D P1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT1G52570  PLDALPHA2; phospholipase D alpha 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G35790  PLDDELTA; phospholipase D delta [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G00240  PLDBETA2; phospholipase D beta 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11830  PLDGAMMA2; phospholipase D gamma 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT2G06925  PLA2-ALPHA; Phospholipase A2 family protein [KO:K01047] [EC:3.1.1.4]\n",
      "AT2G45670  calcineurin B subunit-like protein [KO:K13510] [EC:2.3.1.23 2.3.1.67]\n",
      "AT1G80950  Phospholipid/glycerol acyltransferase family protein [KO:K13510] [EC:2.3.1.23 2.3.1.67]\n",
      "AT2G44810  DAD1; alpha/beta-Hydrolases superfamily protein [KO:K16818] [EC:3.1.1.32]\n",
      "AT3G03310  LCAT3; lecithin:cholesterol acyltransferase 3 [KO:K22389] [EC:3.1.1.32]\n",
      "AT4G19860  alpha/beta-Hydrolases superfamily protein [KO:K22389] [EC:3.1.1.32]\n",
      "AT5G20060  alpha/beta-Hydrolases superfamily protein [KO:K06130] [EC:3.1.1.5]\n",
      "AT3G15650  alpha/beta-Hydrolases superfamily protein [KO:K06130] [EC:3.1.1.5]\n",
      "AT5G41080  GDPD2; PLC-like phosphodiesterases superfamily protein [KO:K18696] [EC:3.1.4.46]\n",
      "AT3G02040  SRG3; senescence-related gene 3 [KO:K18696] [EC:3.1.4.46]\n",
      "AT5G43300  GDPD3; PLC-like phosphodiesterases superfamily protein [KO:K18696] [EC:3.1.4.46]\n",
      "AT1G74210  GDPD5; PLC-like phosphodiesterases superfamily protein [KO:K01126] [EC:3.1.4.46]\n",
      "AT5G08030  GDPD6; PLC-like phosphodiesterases superfamily protein [KO:K01126] [EC:3.1.4.46]\n",
      "AT1G74320  Protein kinase superfamily protein [KO:K14156] [EC:2.7.1.32 2.7.1.82]\n",
      "AT4G09760  Protein kinase superfamily protein [KO:K14156] [EC:2.7.1.32 2.7.1.82]\n",
      "AT1G71697  CK1; choline kinase 1 [KO:K14156] [EC:2.7.1.32 2.7.1.82]\n",
      "AT2G32260  CCT1; phosphorylcholine cytidylyltransferase [KO:K00968] [EC:2.7.7.15]\n",
      "AT4G15130  CCT2; phosphorylcholine cytidylyltransferase2 [KO:K00968] [EC:2.7.7.15]\n",
      "AT1G13560  AAPT1; aminoalcoholphosphotransferase 1 [KO:K00993] [EC:2.7.8.1]\n",
      "AT3G25585  AAPT2; aminoalcoholphosphotransferase [KO:K00993] [EC:2.7.8.1]\n",
      "AT1G43710  EMB1075; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K25193] [EC:4.1.1.-]\n",
      "AT2G26830  emb1187; Protein kinase superfamily protein [KO:K00894] [EC:2.7.1.82]\n",
      "AT2G38670  PECT1; phosphorylethanolamine cytidylyltransferase 1 [KO:K00967] [EC:2.7.7.14]\n",
      "AT3G18000  XPL1; S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K05929] [EC:2.1.1.103]\n",
      "AT1G48600  PMEAMT; S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K05929] [EC:2.1.1.103]\n",
      "AT1G73600  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K05929] [EC:2.1.1.103]\n",
      "AT1G80860  PLMT; phospholipid N-methyltransferase [KO:K00550] [EC:2.1.1.71]\n",
      "AT4G26770  Phosphatidate cytidylyltransferase family protein [KO:K00981] [EC:2.7.7.41]\n",
      "AT1G62430  CDS1; CDP-diacylglycerol synthase 1 [KO:K00981] [EC:2.7.7.41]\n",
      "AT4G22340  CDS2; cytidinediphosphate diacylglycerol synthase 2 [KO:K00981] [EC:2.7.7.41]\n",
      "AT2G45150  CDS4; cytidinediphosphate diacylglycerol synthase 4 [KO:K00981] [EC:2.7.7.41]\n",
      "AT3G60620  CDS5; cytidinediphosphate diacylglycerol synthase 5 [KO:K00981] [EC:2.7.7.41]\n",
      "AT1G15110  PSS1; phosphatidyl serine synthase family protein [KO:K08730] [EC:2.7.8.29]\n",
      "AT4G16700  PSD1; phosphatidylserine decarboxylase 1 [KO:K01613] [EC:4.1.1.65]\n",
      "AT4G25970  PSD3; phosphatidylserine decarboxylase 3 [KO:K01613] [EC:4.1.1.65]\n",
      "AT5G57190  PSD2; phosphatidylserine decarboxylase 2 [KO:K01613] [EC:4.1.1.65]\n",
      "AT2G39290  PGP1; phosphatidylglycerolphosphate synthase 1 [KO:K00995] [EC:2.7.8.5]\n",
      "AT3G55030  PGPS2; phosphatidylglycerolphosphate synthase 2 [KO:K00995] [EC:2.7.8.5]\n",
      "AT3G58830  haloacid dehalogenase (HAD) superfamily protein [KO:K01094] [EC:3.1.3.27]\n",
      "AT4G04870  CLS; cardiolipin synthase [KO:K08744] [EC:2.7.8.41]\n",
      "AT1G78690  At1g78690p; Phospholipid/glycerol acyltransferase family protein [KO:K13511] [EC:2.3.1.-]\n",
      "AT3G05510  Phospholipid/glycerol acyltransferase family protein [KO:K13511] [EC:2.3.1.-]\n",
      "AT4G38570  PIS2; Putative CDP-diacylglycerol-inositol 3-phosphatidyltransferase 2 [KO:K00999] [EC:2.7.8.11]\n",
      "AT1G68000  PIS1; phosphatidylinositol synthase 1 [KO:K00999] [EC:2.7.8.11]\n",
      "COMPOUND    C00040  Acyl-CoA\n",
      "C00065  L-Serine\n",
      "C00084  Acetaldehyde\n",
      "C00093  sn-Glycerol 3-phosphate\n",
      "C00111  Glycerone phosphate\n",
      "C00114  Choline\n",
      "C00157  Phosphatidylcholine\n",
      "C00189  Ethanolamine\n",
      "C00269  CDP-diacylglycerol\n",
      "C00307  CDP-choline\n",
      "C00344  Phosphatidylglycerol\n",
      "C00346  Ethanolamine phosphate\n",
      "C00350  Phosphatidylethanolamine\n",
      "C00416  Phosphatidate\n",
      "C00513  CDP-glycerol\n",
      "C00570  CDP-ethanolamine\n",
      "C00588  Choline phosphate\n",
      "C00623  sn-Glycerol 1-phosphate\n",
      "C00641  1,2-Diacyl-sn-glycerol\n",
      "C00670  sn-Glycero-3-phosphocholine\n",
      "C00681  1-Acyl-sn-glycerol 3-phosphate\n",
      "C01194  1-Phosphatidyl-D-myo-inositol\n",
      "C01210  N-Methylethanolamine phosphate\n",
      "C01233  sn-Glycero-3-phosphoethanolamine\n",
      "C01241  Phosphatidyl-N-methylethanolamine\n",
      "C01996  Acetylcholine\n",
      "C02737  Phosphatidylserine\n",
      "C03372  Acylglycerone phosphate\n",
      "C03819  1-Acylglycerophosphoinositol\n",
      "C03872  L-Serine-phosphoethanolamine\n",
      "C03892  Phosphatidylglycerophosphate\n",
      "C03974  2-Acyl-sn-glycerol 3-phosphate\n",
      "C04230  1-Acyl-sn-glycero-3-phosphocholine\n",
      "C04233  2-Acyl-sn-glycero-3-phosphocholine\n",
      "C04308  Phosphatidyl-N-dimethylethanolamine\n",
      "C04438  1-Acyl-sn-glycero-3-phosphoethanolamine\n",
      "C04590  3-(O-Geranylgeranyl)-sn-glycerol 1-phosphate\n",
      "C04638  2,3-Bis-O-(geranylgeranyl)glycerol 1-phosphate\n",
      "C05973  2-Acyl-sn-glycero-3-phosphoethanolamine\n",
      "C05974  2-Acyl-sn-glycero-3-phosphoserine\n",
      "C05976  Aminoacyl-phosphatidylglycerol\n",
      "C05980  Cardiolipin\n",
      "C06771  Triethanolamine\n",
      "C06772  Diethanolamine\n",
      "C13482  Phosphodimethylethanolamine\n",
      "C18059  CDP-2,3-bis-O-(geranylgeranyl)-sn-glycerol\n",
      "C18125  1-Acyl-sn-glycero-3-phosphoserine\n",
      "C18126  1-Acyl-sn-glycero-3-phosphoglycerol\n",
      "C18129  Monolysocardiolipin\n",
      "C18168  Diacylglycerylhomoserine\n",
      "C18169  Diacylglyceryl-N,N,N-trimethylhomoserine\n",
      "C20466  2,3-Bis-O-(geranylgeranyl)-sn-glycero-1-phospho-L-serine\n",
      "C20518  2,3-Bis-(O-phytanyl)-sn-glycerol 1-phosphate\n",
      "C22340  Heptaprenylglyceryl phosphate\n",
      "C22341  Heptaprenylglycerol\n",
      "C22342  Diacetylheptaprenylglycerol\n",
      "REFERENCE   PMID:1512203\n",
      "AUTHORS   Racenis PV, Lai JL, Das AK, Mullick PC, Hajra AK, Greenberg ML.\n",
      "TITLE     The acyl dihydroxyacetone phosphate pathway enzymes for glycerolipid biosynthesis are present in the yeast Saccharomyces cerevisiae.\n",
      "JOURNAL   J Bacteriol 174:5702-10 (1992)\n",
      "DOI:10.1128/JB.174.17.5702-5710.1992\n",
      "REFERENCE   PMID:10092508\n",
      "AUTHORS   Taniyama Y, Shibata S, Kita S, Horikoshi K, Fuse H, Shirafuji H, Sumino Y, Fujino M.\n",
      "TITLE     Cloning and expression of a novel lysophospholipase which structurally resembles lecithin cholesterol acyltransferase.\n",
      "JOURNAL   Biochem Biophys Res Commun 257:50-6 (1999)\n",
      "DOI:10.1006/bbrc.1999.0411\n",
      "REFERENCE   PMID:10542045\n",
      "AUTHORS   Athenstaedt K, Daum G.\n",
      "TITLE     Phosphatidic acid, a key intermediate in lipid metabolism.\n",
      "JOURNAL   Eur J Biochem 266:1-16 (1999)\n",
      "DOI:10.1046/j.1432-1327.1999.00822.x\n",
      "REFERENCE   PMID:12431977\n",
      "AUTHORS   Shields DJ, Lehner R, Agellon LB, Vance DE.\n",
      "TITLE     Membrane topography of human phosphatidylethanolamine N-methyltransferase.\n",
      "JOURNAL   J Biol Chem 278:2956-62 (2003)\n",
      "DOI:10.1074/jbc.M210904200\n",
      "REFERENCE   PMID:14663079\n",
      "AUTHORS   Martinez-Morales F, Schobert M, Lopez-Lara IM, Geiger O.\n",
      "TITLE     Pathways for phosphatidylcholine biosynthesis in bacteria.\n",
      "JOURNAL   Microbiology 149:3461-71 (2003)\n",
      "DOI:10.1099/mic.0.26522-0\n",
      "REFERENCE   PMID:15522832\n",
      "AUTHORS   Burgermeister M, Birner-Grunberger R, Nebauer R, Daum G.\n",
      "TITLE     Contribution of different pathways to the supply of phosphatidylethanolamine and phosphatidylcholine to mitochondrial membranes of the yeast Saccharomyces cerevisiae.\n",
      "JOURNAL   Biochim Biophys Acta 1686:161-8 (2004)\n",
      "DOI:10.1016/j.bbalip.2004.09.007\n",
      "REFERENCE   PMID:1649829\n",
      "AUTHORS   Hsu L, Jackowski S, Rock CO\n",
      "TITLE     Isolation and characterization of Escherichia coli K-12 mutants lacking both 2-acyl-glycerophosphoethanolamine acyltransferase and acyl-acyl carrier protein synthetase activity.\n",
      "JOURNAL   J Biol Chem 266:13783-8 (1991)\n",
      "REFERENCE   PMID:10938271\n",
      "AUTHORS   Stone SJ, Vance JE.\n",
      "TITLE     Phosphatidylserine synthase-1 and -2 are localized to mitochondria-associated membranes.\n",
      "JOURNAL   J Biol Chem 275:34534-40 (2000)\n",
      "DOI:10.1074/jbc.M002865200\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00561  Glycerolipid metabolism\n",
      "ath00562  Inositol phosphate metabolism\n",
      "ath00563  Glycosylphosphatidylinositol (GPI)-anchor biosynthesis\n",
      "ath00565  Ether lipid metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "KO_PATHWAY  ko00564\n",
      "///\n",
      "ENTRY       ath00565                    Pathway\n",
      "NAME        Ether lipid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00565  Ether lipid metabolism\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G13560  AAPT1; aminoalcoholphosphotransferase 1 [KO:K00993] [EC:2.7.8.1]\n",
      "AT3G25585  AAPT2; aminoalcoholphosphotransferase [KO:K00993] [EC:2.7.8.1]\n",
      "AT2G06925  PLA2-ALPHA; Phospholipase A2 family protein [KO:K01047] [EC:3.1.1.4]\n",
      "AT3G57140  SDP1-LIKE; sugar-dependent 1-like protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G04040  SDP1; Patatin-like phospholipase family protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT1G07230  NPC1; non-specific phospholipase C1 [KO:K01114] [EC:3.1.4.3]\n",
      "AT2G26870  NPC2; non-specific phospholipase C2 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G48610  NPC6; non-specific phospholipase C6 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G03540  NPC5; non-specific phospholipase C5 [KO:K01114] [EC:3.1.4.3]\n",
      "AT3G03530  NPC4; non-specific phospholipase C4 [KO:K01114] [EC:3.1.4.3]\n",
      "AT2G42010  PLDBETA1; phospholipase D beta 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT5G25370  PLDALPHA3; phospholipase D alpha 3 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11840  PLDGAMMA3; phospholipase D gamma 3 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11850  PLDGAMMA1; phospholipase D gamma 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT1G55180  PLDEPSILON; phospholipase D alpha 4 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G05630  PLDP2; phospholipase D P2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G15730  PLDALPHA1; phospholipase D alpha 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G16785  PLDP1; phospholipase D P1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT1G52570  PLDALPHA2; phospholipase D alpha 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G35790  PLDDELTA; phospholipase D delta [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G00240  PLDBETA2; phospholipase D beta 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11830  PLDGAMMA2; phospholipase D gamma 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT2G45670  calcineurin B subunit-like protein [KO:K13510] [EC:2.3.1.23 2.3.1.67]\n",
      "AT1G80950  Phospholipid/glycerol acyltransferase family protein [KO:K13510] [EC:2.3.1.23 2.3.1.67]\n",
      "AT1G12640  LPLAT1; MBOAT (membrane bound O-acyl transferase) family protein [KO:K13519] [EC:2.3.1.51 2.3.1.23 2.3.1.-]\n",
      "AT1G63050  LPLAT2; MBOAT (membrane bound O-acyl transferase) family protein [KO:K13519] [EC:2.3.1.51 2.3.1.23 2.3.1.-]\n",
      "COMPOUND    C00670  sn-Glycero-3-phosphocholine\n",
      "C00958  Plasmenylcholine\n",
      "C01233  sn-Glycero-3-phosphoethanolamine\n",
      "C01264  2-Acetyl-1-alkyl-sn-glycero-3-phosphate\n",
      "C02773  1-Alkyl-sn-glycerol\n",
      "C03201  1-Alkyl-2-acylglycerol\n",
      "C03372  Acylglycerone phosphate\n",
      "C03454  1-Alkenyl-2-acylglycerol\n",
      "C03715  O-Alkylglycerone phosphate\n",
      "C03820  1-Alkyl-2-acetyl-sn-glycerol\n",
      "C03968  1-Alkyl-sn-glycero-3-phosphate\n",
      "C04317  1-Organyl-2-lyso-sn-glycero-3-phosphocholine\n",
      "C04361  1-O-Alkyl-2-acetyl-3-acyl-sn-glycerol\n",
      "C04475  1-Alkyl-2-acylglycerophosphoethanolamine\n",
      "C04517  1-(1-Alkenyl)-sn-glycero-3-phosphocholine\n",
      "C04598  2-Acetyl-1-alkyl-sn-glycero-3-phosphocholine\n",
      "C04635  1-(1-Alkenyl)-sn-glycero-3-phosphoethanolamine\n",
      "C04756  O-1-Alk-1-enyl-2-acyl-sn-glycero-3-phosphoethanolamine\n",
      "C05212  1-Radyl-2-acyl-sn-glycero-3-phosphocholine\n",
      "C05977  2-Acyl-1-alkyl-sn-glycero-3-phosphate\n",
      "C15645  1-(1-Alkenyl)-sn-glycerol\n",
      "C15646  1-(1-Alkenyl)-sn-glycero-3-phosphate\n",
      "C15647  2-Acyl-1-(1-alkenyl)-sn-glycero-3-phosphate\n",
      "C20825  Seminolipid\n",
      "C20859  Galactosyl-1-alkyl-2-acylglycerol\n",
      "REFERENCE   PMID:11275267\n",
      "AUTHORS   Nagan N, Zoeller RA.\n",
      "TITLE     Plasmalogens: biosynthesis and functions.\n",
      "JOURNAL   Prog Lipid Res 40:199-229 (2001)\n",
      "DOI:10.1016/S0163-7827(01)00003-0\n",
      "REFERENCE   PMID:11015622\n",
      "AUTHORS   Montrucchio G, Alloatti G, Camussi G.\n",
      "TITLE     Role of platelet-activating factor in cardiovascular pathophysiology.\n",
      "JOURNAL   Physiol Rev 80:1669-99 (2000)\n",
      "DOI:10.1152/physrev.2000.80.4.1669\n",
      "REFERENCE   PMID:15788404\n",
      "AUTHORS   Sakagami H, Aoki J, Natori Y, Nishikawa K, Kakehi Y, Natori Y, Arai H.\n",
      "TITLE     Biochemical and molecular characterization of a novel choline-specific glycerophosphodiester phosphodiesterase belonging to the nucleotide pyrophosphatase/phosphodiesterase family.\n",
      "JOURNAL   J Biol Chem 280:23084-93 (2005)\n",
      "DOI:10.1074/jbc.M413438200\n",
      "REL_PATHWAY ath00564  Glycerophospholipid metabolism\n",
      "KO_PATHWAY  ko00565\n",
      "///\n",
      "ENTRY       ath00590                    Pathway\n",
      "NAME        Arachidonic acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00590  Arachidonic acid metabolism\n",
      "DBLINKS     GO: 0019369\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G06925  PLA2-ALPHA; Phospholipase A2 family protein [KO:K01047] [EC:3.1.1.4]\n",
      "AT3G57140  SDP1-LIKE; sugar-dependent 1-like protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G04040  SDP1; Patatin-like phospholipase family protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G42150  Glutathione S-transferase family protein [KO:K05309] [EC:5.3.99.3]\n",
      "AT3G59710  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00079] [EC:1.1.1.184 1.1.1.189 1.1.1.197]\n",
      "AT5G61830  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00079] [EC:1.1.1.184 1.1.1.189 1.1.1.197]\n",
      "AT5G51030  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00079] [EC:1.1.1.184 1.1.1.189 1.1.1.197]\n",
      "AT5G13520  peptidase M1 family protein [KO:K01254] [EC:3.3.2.6]\n",
      "AT4G39640  GGT1; gamma-glutamyl transpeptidase 1 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT4G39650  GGT2; gamma-glutamyl transpeptidase 2 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT1G69820  GGT3; gamma-glutamyl transpeptidase 3 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT4G29210  GGT4; gamma-glutamyl transpeptidase 4 [KO:K18592] [EC:2.3.2.2 3.4.19.13 3.4.19.14]\n",
      "AT3G63080  GPX5; glutathione peroxidase 5 [KO:K00432] [EC:1.11.1.9]\n",
      "AT2G48150  GPX4; glutathione peroxidase 4 [KO:K00432] [EC:1.11.1.9]\n",
      "AT2G31570  GPX2; glutathione peroxidase 2 [KO:K00432] [EC:1.11.1.9]\n",
      "AT1G63460  GPX8; glutathione peroxidase 8 [KO:K00432] [EC:1.11.1.9]\n",
      "AT4G11600  GPX6; glutathione peroxidase 6 [KO:K00432] [EC:1.11.1.9]\n",
      "AT2G25080  GPX1; glutathione peroxidase 1 [KO:K00432] [EC:1.11.1.9]\n",
      "AT4G31870  GPX7; glutathione peroxidase 7 [KO:K00432] [EC:1.11.1.9]\n",
      "AT2G43350  GPX3; glutathione peroxidase 3 [KO:K00432] [EC:1.11.1.9]\n",
      "COMPOUND    C00157  Phosphatidylcholine\n",
      "C00219  Arachidonate\n",
      "C00427  Prostaglandin H2\n",
      "C00584  Prostaglandin E2\n",
      "C00639  Prostaglandin F2alpha\n",
      "C00696  Prostaglandin D2\n",
      "C00909  Leukotriene A4\n",
      "C01312  Prostaglandin I2\n",
      "C02165  Leukotriene B4\n",
      "C02166  Leukotriene C4\n",
      "C02198  Thromboxane A2\n",
      "C03577  20-Hydroxyleukotriene E4\n",
      "C04577  15-OxoETE\n",
      "C04742  (15S)-15-Hydroxy-5,8,11-cis-13-trans-eicosatetraenoate\n",
      "C04805  5(S)-HETE\n",
      "C04822  8(R)-HPETE\n",
      "C04853  20-OH-Leukotriene B4\n",
      "C05356  5(S)-HPETE\n",
      "C05949  12-Keto-leukotriene B4\n",
      "C05950  20-COOH-Leukotriene B4\n",
      "C05951  Leukotriene D4\n",
      "C05952  Leukotriene E4\n",
      "C05953  Prostaglandin A2\n",
      "C05954  Prostaglandin B2\n",
      "C05955  Prostaglandin C2\n",
      "C05956  Prostaglandin G2\n",
      "C05957  Prostaglandin J2\n",
      "C05958  Delta-12-Prostaglandin J2\n",
      "C05959  11-epi-Prostaglandin F2alpha\n",
      "C05960  15-Keto-prostaglandin F2alpha\n",
      "C05961  6-Keto-prostaglandin F1alpha\n",
      "C05962  6-Keto-prostaglandin E1\n",
      "C05963  Thromboxane B2\n",
      "C05964  11-Dehydro-thromboxane B2\n",
      "C05965  12(S)-HPETE\n",
      "C05966  15(S)-HPETE\n",
      "C06314  Lipoxin A4\n",
      "C06315  Lipoxin B4\n",
      "C06462  Leukotriene F4\n",
      "C13809  9,11,15-Trihydroxy-prosta-5,13-dien-1-oic acid\n",
      "C14717  15-Deoxy-Delta12,14-PGJ2\n",
      "C14732  5-OxoETE\n",
      "C14748  20-HETE\n",
      "C14749  19(S)-HETE\n",
      "C14768  5,6-EET\n",
      "C14769  8,9-EET\n",
      "C14770  11,12-EET\n",
      "C14771  14,15-EET\n",
      "C14772  5,6-DHET\n",
      "C14773  8,9-DHET\n",
      "C14774  11,12-DHET\n",
      "C14775  14,15-DHET\n",
      "C14776  8(S)-HETE\n",
      "C14777  12(S)-HETE\n",
      "C14778  16(R)-HETE\n",
      "C14779  9(S)-HETE\n",
      "C14780  11(R)-HETE\n",
      "C14781  15H-11,12-EETA\n",
      "C14782  11,12,15-THETA\n",
      "C14794  2,3-Dinor-8-iso prostaglandin F2alpha\n",
      "C14795  2,3-Dinor-8-iso prostaglandin F1alpha\n",
      "C14807  12-OxoETE\n",
      "C14808  Hepoxilin A3\n",
      "C14809  Trioxilin A3\n",
      "C14810  Hepoxilin B3\n",
      "C14811  Trioxilin B3\n",
      "C14812  12(R)-HPETE\n",
      "C14813  11H-14,15-EETA\n",
      "C14814  11,14,15-THETA\n",
      "C14815  5,6-Epoxytetraene\n",
      "C14820  11(R)-HPETE\n",
      "C14821  9(S)-HPETE\n",
      "C14822  12(R)-HETE\n",
      "C14823  8(S)-HPETE\n",
      "C14824  8(R)-HETE\n",
      "REFERENCE   PMID:14636671\n",
      "AUTHORS   Spector AA, Fang X, Snyder GD, Weintraub NL.\n",
      "TITLE     Epoxyeicosatrienoic acids (EETs): metabolism and biochemical function.\n",
      "JOURNAL   Prog Lipid Res 43:55-90 (2004)\n",
      "DOI:10.1016/S0163-7827(03)00049-3\n",
      "REFERENCE   PMID:15292194\n",
      "AUTHORS   Harman CA, Rieke CJ, Garavito RM, Smith WL.\n",
      "TITLE     Crystal structure of arachidonic acid bound to a mutant of prostaglandin endoperoxide H synthase-1 that forms predominantly 11-hydroperoxyeicosatetraenoic acid.\n",
      "JOURNAL   J Biol Chem 279:42929-35 (2004)\n",
      "DOI:10.1074/jbc.M403013200\n",
      "REFERENCE   PMID:15205388\n",
      "AUTHORS   Poloyac SM, Tortorici MA, Przychodzin DI, Reynolds RB, Xie W, Frye RF, Zemaitis MA.\n",
      "TITLE     The effect of isoniazid on CYP2E1- and CYP4A-mediated hydroxylation of arachidonic acid in the rat liver and kidney.\n",
      "JOURNAL   Drug Metab Dispos 32:727-33 (2004)\n",
      "DOI:10.1124/dmd.32.7.727\n",
      "REFERENCE   PMID:14660610\n",
      "AUTHORS   Chuang SS, Helvig C, Taimi M, Ramshaw HA, Collop AH, Amad M, White JA, Petkovich M, Jones G, Korczak B.\n",
      "TITLE     CYP2U1, a novel human thymus- and brain-specific cytochrome P450, catalyzes omega- and (omega-1)-hydroxylation of fatty acids.\n",
      "JOURNAL   J Biol Chem 279:6305-14 (2004)\n",
      "DOI:10.1074/jbc.M311830200\n",
      "REFERENCE   PMID:14626496\n",
      "AUTHORS   Sacerdoti D, Gatta A, McGiff JC.\n",
      "TITLE     Role of cytochrome P450-dependent arachidonic acid metabolites in liver physiology and pathophysiology.\n",
      "JOURNAL   Prostaglandins Other Lipid Mediat 72:51-71 (2003)\n",
      "DOI:10.1016/S1098-8823(03)00077-7\n",
      "REFERENCE   PMID:14622984\n",
      "AUTHORS   Pfister SL, Spitzbarth N, Zeldin DC, Lafite P, Mansuy D, Campbell WB.\n",
      "TITLE     Rabbit aorta converts 15-HPETE to trihydroxyeicosatrienoic acids: potential role of cytochrome P450.\n",
      "JOURNAL   Arch Biochem Biophys 420:142-52 (2003)\n",
      "DOI:10.1016/j.abb.2003.09.026\n",
      "REFERENCE   PMID:11679404\n",
      "AUTHORS   Fleming I.\n",
      "TITLE     Cytochrome p450 and vascular homeostasis.\n",
      "JOURNAL   Circ Res 89:753-62 (2001)\n",
      "DOI:10.1161/hh2101.099268\n",
      "REFERENCE   PMID:11328810\n",
      "AUTHORS   Qu W, Bradbury JA, Tsao CC, Maronpot R, Harry GJ, Parker CE, Davis LS, Breyer MD, Waalkes MP, Falck JR, Chen J, Rosenberg RL, Zeldin DC.\n",
      "TITLE     Cytochrome P450 CYP2J9, a new mouse arachidonic acid omega-1 hydroxylase predominantly expressed in brain.\n",
      "JOURNAL   J Biol Chem 276:25467-79 (2001)\n",
      "DOI:10.1074/jbc.M100545200\n",
      "REFERENCE   PMID:10908295\n",
      "AUTHORS   Tsao CC, Foley J, Coulter SJ, Maronpot R, Zeldin DC, Goldstein JA.\n",
      "TITLE     CYP2C40, a unique arachidonic acid 16-hydroxylase, is the major CYP2C in murine intestinal tract.\n",
      "JOURNAL   Mol Pharmacol 58:279-87 (2000)\n",
      "DOI:10.1124/mol.58.2.279\n",
      "REFERENCE   PMID:9812980\n",
      "AUTHORS   Pfister SL, Spitzbarth N, Nithipatikom K, Edgemond WS, Falck JR, Campbell WB.\n",
      "TITLE     Identification of the 11,14,15- and 11,12, 15-trihydroxyeicosatrienoic acids as endothelium-derived relaxing factors of rabbit aorta.\n",
      "JOURNAL   J Biol Chem 273:30879-87 (1998)\n",
      "DOI:10.1074/jbc.273.47.30879\n",
      "REL_PATHWAY ath00591  Linoleic acid metabolism\n",
      "KO_PATHWAY  ko00590\n",
      "///\n",
      "ENTRY       ath00591                    Pathway\n",
      "NAME        Linoleic acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00591  Linoleic acid metabolism\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G06925  PLA2-ALPHA; Phospholipase A2 family protein [KO:K01047] [EC:3.1.1.4]\n",
      "AT3G57140  SDP1-LIKE; sugar-dependent 1-like protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G04040  SDP1; Patatin-like phospholipase family protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT1G55020  LOX1; lipoxygenase 1 [KO:K15718] [EC:1.13.11.58]\n",
      "AT3G22400  LOX5; PLAT/LH2 domain-containing lipoxygenase family protein [KO:K15718] [EC:1.13.11.58]\n",
      "AT3G45140  LOX2; lipoxygenase 2 [KO:K00454] [EC:1.13.11.12]\n",
      "AT1G17420  LOX3; lipoxygenase 3 [KO:K00454] [EC:1.13.11.12]\n",
      "AT1G67560  LOX6; PLAT/LH2 domain-containing lipoxygenase family protein [KO:K00454] [EC:1.13.11.12]\n",
      "AT1G72520  LOX4; PLAT/LH2 domain-containing lipoxygenase family protein [KO:K00454] [EC:1.13.11.12]\n",
      "COMPOUND    C00157  Phosphatidylcholine\n",
      "C00219  Arachidonate\n",
      "C01595  Linoleate\n",
      "C03242  Dihomo-gamma-linolenate\n",
      "C04056  Rumenic acid\n",
      "C04717  (9Z,11E)-(13S)-13-Hydroperoxyoctadeca-9,11-dienoic acid\n",
      "C06426  (6Z,9Z,12Z)-Octadecatrienoic acid\n",
      "C07289  Crepenynate\n",
      "C07338  (9Z,12Z)-(11S)-11-Hydroperoxyoctadeca-9,12-dienoic acid\n",
      "C07354  (7S,8S)-DiHODE\n",
      "C14762  13(S)-HODE\n",
      "C14765  13-OxoODE\n",
      "C14766  9-OxoODE\n",
      "C14767  9(S)-HODE\n",
      "C14825  9(10)-EpOME\n",
      "C14826  12(13)-EpOME\n",
      "C14827  9(S)-HPODE\n",
      "C14828  9,10-DHOME\n",
      "C14829  12,13-DHOME\n",
      "C14831  8(R)-HPODE\n",
      "C14832  12,13-Epoxy-9-hydroxy-10-octadecenoate\n",
      "C14833  9,12,13-TriHOME\n",
      "C14834  9,10-Epoxy-13-hydroxy-11-octadecenoate\n",
      "C14835  9,10,13-TriHOME\n",
      "C14836  9,10-12,13-Diepoxyoctadecanoate\n",
      "C14837  9,10-Dihydroxy-12,13-epoxyoctadecanoate\n",
      "C19937  (8E,10R,12Z)-10-Hydroperoxy-8,12-octadecadienoate\n",
      "C20186  5S,8R-DiHODE\n",
      "REFERENCE   PMID:14757320\n",
      "AUTHORS   Nowak G, Grant DF, Moran JH.\n",
      "TITLE     Linoleic acid epoxide promotes the maintenance of mitochondrial function and active Na+ transport following hypoxia.\n",
      "JOURNAL   Toxicol Lett 147:161-75 (2004)\n",
      "DOI:10.1016/j.toxlet.2003.11.002\n",
      "REFERENCE   PMID:12909723\n",
      "AUTHORS   Shureiqi I, Jiang W, Zuo X, Wu Y, Stimmel JB, Leesnitzer LM, Morris JS, Fan HZ, Fischer SM, Lippman SM.\n",
      "TITLE     The 15-lipoxygenase-1 product 13-S-hydroxyoctadecadienoic acid down-regulates PPAR-delta to induce apoptosis in colorectal cancer cells.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 100:9968-73 (2003)\n",
      "DOI:10.1073/pnas.1631086100\n",
      "REFERENCE   PMID:12324235\n",
      "AUTHORS   Spiteller G.\n",
      "TITLE     Do changes in the cell membrane structure induce the generation of lipid peroxidation products which serve as first signalling molecules in cell to cell communication?\n",
      "JOURNAL   Prostaglandins Leukot Essent Fatty Acids 67:151-62 (2002)\n",
      "DOI:10.1054/plef.2002.0413\n",
      "REFERENCE   PMID:11302930\n",
      "AUTHORS   Jude AR, Little JM, Bull AW, Podgorski I, Radominska-Pandya A.\n",
      "TITLE     13-hydroxy- and 13-oxooctadecadienoic acids: novel substrates for human UDP-glucuronosyltransferases.\n",
      "JOURNAL   Drug Metab Dispos 29:652-5 (2001)\n",
      "REFERENCE   PMID:9694817\n",
      "AUTHORS   Su C, Sahlin M, Oliw EH.\n",
      "TITLE     A protein radical and ferryl intermediates are generated by linoleate diol synthase, a ferric hemeprotein with dioxygenase and hydroperoxide isomerase activities.\n",
      "JOURNAL   J Biol Chem 273:20744-51 (1998)\n",
      "DOI:10.1074/jbc.273.33.20744\n",
      "REFERENCE   PMID:1634517\n",
      "AUTHORS   Brodowsky ID, Hamberg M, Oliw EH.\n",
      "TITLE     A linoleic acid (8R)-dioxygenase and hydroperoxide isomerase of the fungus Gaeumannomyces graminis. Biosynthesis of (8R)-hydroxylinoleic acid and (7S,8S)-dihydroxylinoleic acid from (8R)-hydroperoxylinoleic acid.\n",
      "JOURNAL   J Biol Chem 267:14738-45 (1992)\n",
      "REFERENCE   PMID:12226220\n",
      "AUTHORS   Hamberg M, Hamberg G.\n",
      "TITLE     Peroxygenase-Catalyzed Fatty Acid Epoxidation in Cereal Seeds (Sequential Oxidation of Linoleic Acid into 9(S),12(S),13(S)-Trihydroxy-10(E)-Octadecenoic Acid).\n",
      "JOURNAL   Plant Physiol 110:807-815 (1996)\n",
      "DOI:10.1104/pp.110.3.807\n",
      "REFERENCE   PMID:8765137\n",
      "AUTHORS   Moghaddam M, Motoba K, Borhan B, Pinot F, Hammock BD.\n",
      "TITLE     Novel metabolic pathways for linoleic and arachidonic acid metabolism.\n",
      "JOURNAL   Biochim Biophys Acta 1290:327-39 (1996)\n",
      "DOI:10.1016/0304-4165(96)00037-2\n",
      "REL_PATHWAY ath00590  Arachidonic acid metabolism\n",
      "KO_PATHWAY  ko00591\n",
      "///\n",
      "ENTRY       ath00592                    Pathway\n",
      "NAME        alpha-Linolenic acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00592  alpha-Linolenic acid metabolism\n",
      "MODULE      ath_M00113  Jasmonic acid biosynthesis [PATH:ath00592]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G06925  PLA2-ALPHA; Phospholipase A2 family protein [KO:K01047] [EC:3.1.1.4]\n",
      "AT3G57140  SDP1-LIKE; sugar-dependent 1-like protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT5G04040  SDP1; Patatin-like phospholipase family protein [KO:K14674] [EC:3.1.1.3 3.1.1.13 3.1.1.4 2.3.1.51]\n",
      "AT2G44810  DAD1; alpha/beta-Hydrolases superfamily protein [KO:K16818] [EC:3.1.1.32]\n",
      "AT3G03310  LCAT3; lecithin:cholesterol acyltransferase 3 [KO:K22389] [EC:3.1.1.32]\n",
      "AT4G19860  alpha/beta-Hydrolases superfamily protein [KO:K22389] [EC:3.1.1.32]\n",
      "AT3G45140  LOX2; lipoxygenase 2 [KO:K00454] [EC:1.13.11.12]\n",
      "AT1G17420  LOX3; lipoxygenase 3 [KO:K00454] [EC:1.13.11.12]\n",
      "AT1G67560  LOX6; PLAT/LH2 domain-containing lipoxygenase family protein [KO:K00454] [EC:1.13.11.12]\n",
      "AT1G72520  LOX4; PLAT/LH2 domain-containing lipoxygenase family protein [KO:K00454] [EC:1.13.11.12]\n",
      "AT5G42650  AOS; allene oxide synthase [KO:K01723] [EC:4.2.1.92]\n",
      "AT3G25770  AOC2; allene oxide cyclase 2 [KO:K10525] [EC:5.3.99.6]\n",
      "AT3G25780  AOC3; allene oxide cyclase 3 [KO:K10525] [EC:5.3.99.6]\n",
      "AT1G13280  AOC4; allene oxide cyclase 4 [KO:K10525] [EC:5.3.99.6]\n",
      "AT3G25760  AOC1; allene oxide cyclase 1 [KO:K10525] [EC:5.3.99.6]\n",
      "AT1G76690  OPR2; 12-oxophytodienoate reductase 2 [KO:K05894] [EC:1.3.1.42]\n",
      "AT2G06050  OPR3; oxophytodienoate-reductase 3 [KO:K05894] [EC:1.3.1.42]\n",
      "AT1G76680  OPR1; 12-oxophytodienoate reductase 1 [KO:K05894] [EC:1.3.1.42]\n",
      "AT1G09400  FMN-linked oxidoreductases superfamily protein [KO:K05894] [EC:1.3.1.42]\n",
      "AT1G18020  FMN-linked oxidoreductases superfamily protein [KO:K05894] [EC:1.3.1.42]\n",
      "AT1G20510  OPCL1; OPC-8:0 CoA ligase1 [KO:K10526] [EC:6.2.1.-]\n",
      "AT1G20480  AMP-dependent synthetase and ligase family protein [KO:K10526] [EC:6.2.1.-]\n",
      "AT1G20490  AMP-dependent synthetase and ligase family protein [KO:K10526] [EC:6.2.1.-]\n",
      "AT1G20500  AMP-dependent synthetase and ligase family protein [KO:K10526] [EC:6.2.1.-]\n",
      "AT5G38120  4CL8; AMP-dependent synthetase and ligase family protein [KO:K10526] [EC:6.2.1.-]\n",
      "AT2G35690  ACX5; acyl-CoA oxidase 5 [KO:K00232] [EC:1.3.3.6]\n",
      "AT4G16760  ACX1; acyl-CoA oxidase 1 [KO:K00232] [EC:1.3.3.6]\n",
      "AT5G65110  ACX2; acyl-CoA oxidase 2 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G51840  ACX4; acyl-CoA oxidase 4 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06290  ACX3; acyl-CoA oxidase 3 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06310  ACX6; acyl-CoA oxidase 6 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G06690  putative acyl-coenzyme A oxidase [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G06860  MFP2; multifunctional protein 2 [KO:K10527] [EC:4.2.1.17 1.1.1.35 1.1.1.211]\n",
      "AT4G29010  AIM1; Enoyl-CoA hydratase/isomerase family [KO:K10527] [EC:4.2.1.17 1.1.1.35 1.1.1.211]\n",
      "AT1G04710  PKT4; peroxisomal 3-ketoacyl-CoA thiolase 4 [KO:K07513] [EC:2.3.1.16]\n",
      "AT2G33150  PKT3; peroxisomal 3-ketoacyl-CoA thiolase 3 [KO:K07513] [EC:2.3.1.16]\n",
      "AT5G48880  KAT5; peroxisomal 3-keto-acyl-CoA thiolase 2 [KO:K07513] [EC:2.3.1.16]\n",
      "AT1G19640  JMT; jasmonic acid carboxyl methyltransferase [KO:K08241] [EC:2.1.1.141]\n",
      "AT1G77120  ADH1; alcohol dehydrogenase 1 [KO:K18857] [EC:1.1.1.1]\n",
      "AT3G03480  CHAT; acetyl CoA:(Z)-3-hexen-1-ol acetyltransferase [KO:K18858] [EC:2.3.1.195]\n",
      "AT4G13010  Oxidoreductase, zinc-binding dehydrogenase family protein [KO:K00224] [EC:1.3.1.-]\n",
      "AT1G73680  ALPHA_DOX2; alpha dioxygenase [KO:K10529] [EC:1.13.11.92]\n",
      "AT3G01420  DOX1; Peroxidase superfamily protein [KO:K10529] [EC:1.13.11.92]\n",
      "COMPOUND    C00157  Phosphatidylcholine\n",
      "C01226  12-OPDA\n",
      "C04672  (9Z,15Z)-(13S)-12,13-Epoxyoctadeca-9,11,15-trienoic acid\n",
      "C04780  8-[(1R,2R)-3-Oxo-2-{(Z)-pent-2-enyl}cyclopentyl]octanoate\n",
      "C04785  13(S)-HPOT\n",
      "C06427  (9Z,12Z,15Z)-Octadecatrienoic acid\n",
      "C08491  (-)-Jasmonic acid\n",
      "C08492  3-Hexenol\n",
      "C11512  Methyl jasmonate\n",
      "C16300  Stearidonic acid\n",
      "C16308  Traumatic acid\n",
      "C16309  Traumatin\n",
      "C16310  3-Hexenal\n",
      "C16311  12-Oxo-9(Z)-dodecenoic acid\n",
      "C16316  13(S)-HOT\n",
      "C16317  (+)-7-Isojasmonic acid\n",
      "C16318  (+)-7-Isomethyljasmonate\n",
      "C16319  Etherolenic acid\n",
      "C16320  Colnelenic acid\n",
      "C16321  9(S)-HPOT\n",
      "C16322  9-Oxononanoic acid\n",
      "C16323  3,6-Nonadienal\n",
      "C16324  9,10-EOT\n",
      "C16325  10-OPDA\n",
      "C16326  9(S)-HOT\n",
      "C16327  OPC8-CoA\n",
      "C16328  trans-2-Enoyl-OPC8-CoA\n",
      "C16330  3-Oxo-OPC8-CoA\n",
      "C16331  OPC6-CoA\n",
      "C16332  trans-2-Enoyl-OPC6-CoA\n",
      "C16334  3-Oxo-OPC6-CoA\n",
      "C16335  OPC4-CoA\n",
      "C16336  trans-2-Enoyl-OPC4-CoA\n",
      "C16338  3-Oxo-OPC4-CoA\n",
      "C16339  (+)-7-Isojasmonic acid CoA\n",
      "C16341  2(R)-HPOT\n",
      "C16342  2(R)-HOT\n",
      "C16343  Heptadecatrienal\n",
      "C16344  Norlinolenic acid\n",
      "C16345  Volicitin\n",
      "C16346  17-Hydroxylinolenic acid\n",
      "C19757  (Z)-3-Hexen-1-ol acetate\n",
      "C21923  9-Hydroxy-12-oxo-10(E),15(Z)-octadecadienoic acid\n",
      "C21924  9-Hydroxy-12-oxo-15(Z)-octadecenoic acid\n",
      "REFERENCE   PMID:17172287\n",
      "AUTHORS   Schilmiller AL, Koo AJ, Howe GA.\n",
      "TITLE     Functional diversification of acyl-coenzyme A oxidases in jasmonic acid biosynthesis and action.\n",
      "JOURNAL   Plant Physiol 143:812-24 (2007)\n",
      "DOI:10.1104/pp.106.092916\n",
      "REFERENCE   PMID:15141068\n",
      "AUTHORS   Cruz Castillo M, Martinez C, Buchala A, Metraux JP, Leon J.\n",
      "TITLE     Gene-specific involvement of beta-oxidation in wound-activated responses in Arabidopsis.\n",
      "JOURNAL   Plant Physiol 135:85-94 (2004)\n",
      "DOI:10.1104/pp.104.039925\n",
      "REFERENCE   PMID:17215350\n",
      "AUTHORS   Truman W, Bennett MH, Kubigsteltig I, Turnbull C, Grant M.\n",
      "TITLE     Arabidopsis systemic immunity uses conserved defense signaling pathways and is mediated by jasmonates.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 104:1075-80 (2007)\n",
      "DOI:10.1073/pnas.0605423104\n",
      "REFERENCE   PMID:16116284\n",
      "AUTHORS   Kuroda H, Oshima T, Kaneda H, Takashio M.\n",
      "TITLE     Identification and functional analyses of two cDNAs that encode fatty acid 9-/13-hydroperoxide lyase (CYP74C) in rice.\n",
      "JOURNAL   Biosci Biotechnol Biochem 69:1545-54 (2005)\n",
      "DOI:10.1271/bbb.69.1545\n",
      "REFERENCE   PMID:15670154\n",
      "AUTHORS   Hamberg M.\n",
      "TITLE     Hidden stereospecificity in the biosynthesis of divinyl ether fatty acids.\n",
      "JOURNAL   FEBS J 272:736-43 (2005)\n",
      "DOI:10.1111/j.1742-4658.2004.04510.x\n",
      "REFERENCE   PMID:14522973\n",
      "AUTHORS   Hamberg M, Sanz A, Rodriguez MJ, Calvo AP, Castresana C.\n",
      "TITLE     Activation of the fatty acid alpha-dioxygenase pathway during bacterial infection of tobacco leaves. Formation of oxylipins protecting against cell death.\n",
      "JOURNAL   J Biol Chem 278:51796-805 (2003)\n",
      "DOI:10.1074/jbc.M310514200\n",
      "REFERENCE\n",
      "AUTHORS   Creelman RA, Mulpuri R.\n",
      "TITLE     The Oxylipin Pathway in Arabidopsis\n",
      "JOURNAL   The Arabidopsis Book 1:e0012 (2002)\n",
      "DOI:10.1199/tab.0012\n",
      "KO_PATHWAY  ko00592\n",
      "///\n",
      "ENTRY       ath00600                    Pathway\n",
      "NAME        Sphingolipid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath00600  Sphingolipid metabolism\n",
      "MODULE      ath_M00100  Sphingosine degradation [PATH:ath00600]\n",
      "DBLINKS     GO: 0006665 0030148\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G23670  LCB2; long chain base2 [KO:K00654] [EC:2.3.1.50]\n",
      "AT3G48780  SPT1; serine palmitoyltransferase 1 [KO:K00654] [EC:2.3.1.50]\n",
      "AT3G48790  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00654] [EC:2.3.1.50]\n",
      "AT4G36480  LCB1; long-chain base1 [KO:K00654] [EC:2.3.1.50]\n",
      "AT3G06060  TSC10A; NAD(P)-binding Rossmann-fold superfamily protein [KO:K04708] [EC:1.1.1.102]\n",
      "AT5G19200  TSC10B; NAD(P)-binding Rossmann-fold superfamily protein [KO:K04708] [EC:1.1.1.102]\n",
      "AT1G07380  Neutral/alkaline non-lysosomal ceramidase [KO:K12349] [EC:3.5.1.23]\n",
      "AT2G38010  Neutral/alkaline non-lysosomal ceramidase [KO:K12349] [EC:3.5.1.23]\n",
      "AT5G58980  Neutral/alkaline non-lysosomal ceramidase [KO:K12349] [EC:3.5.1.23]\n",
      "AT4G22330  ATCES1; Alkaline phytoceramidase (aPHC) [KO:K04711] [EC:3.5.1.-]\n",
      "AT4G04930  DES-1-LIKE; fatty acid desaturase family protein [KO:K04712] [EC:1.14.19.17 1.14.18.5]\n",
      "AT1G14290  SBH2; sphingoid base hydroxylase 2 [KO:K04713] [EC:1.14.18.5]\n",
      "AT1G69640  SBH1; sphingoid base hydroxylase 1 [KO:K04713] [EC:1.14.18.5]\n",
      "AT5G51290  ACD5; Diacylglycerol kinase family protein [KO:K04715] [EC:2.7.1.138]\n",
      "AT4G21540  SPHK1; sphingosine kinase 1 [KO:K04718] [EC:2.7.1.91]\n",
      "AT4G21534  SPHK2; Diacylglycerol kinase family protein [KO:K04718] [EC:2.7.1.91]\n",
      "AT1G27980  DPL1; dihydrosphingosine phosphate lyase [KO:K01634] [EC:4.1.2.27]\n",
      "AT3G24180  Beta-glucosidase, GBA2 type family protein [KO:K17108] [EC:3.2.1.45]\n",
      "AT5G49900  Beta-glucosidase, GBA2 type family protein [KO:K17108] [EC:3.2.1.45]\n",
      "AT4G10060  Beta-glucosidase, GBA2 type family protein [KO:K17108] [EC:3.2.1.45]\n",
      "AT1G33700  Beta-glucosidase, GBA2 type family protein [KO:K17108] [EC:3.2.1.45]\n",
      "AT3G54440  glycoside hydrolase family 2 protein [KO:K01190] [EC:3.2.1.23]\n",
      "AT1G72990  BGAL17; beta-galactosidase 17 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G52840  BGAL2; beta-galactosidase 2 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G56310  Melibiase family protein [KO:K07407] [EC:3.2.1.22]\n",
      "AT5G08370  AGAL2; alpha-galactosidase 2 [KO:K07407] [EC:3.2.1.22]\n",
      "AT5G08380  AGAL1; alpha-galactosidase 1 [KO:K07407] [EC:3.2.1.22]\n",
      "AT3G51730  saposin B domain-containing protein [KO:K12382]\n",
      "AT5G01800  saposin B domain-containing protein [KO:K12382]\n",
      "COMPOUND    C00065  L-Serine\n",
      "C00154  Palmitoyl-CoA\n",
      "C00195  N-Acylsphingosine\n",
      "C00319  Sphingosine\n",
      "C00346  Ethanolamine phosphate\n",
      "C00550  Sphingomyelin\n",
      "C00836  Sphinganine\n",
      "C01120  Sphinganine 1-phosphate\n",
      "C01190  Glucosylceramide\n",
      "C01290  Lactosylceramide\n",
      "C01747  Psychosine\n",
      "C02686  Galactosylceramide\n",
      "C02934  3-Dehydrosphinganine\n",
      "C02960  Ceramide 1-phosphate\n",
      "C03640  Sphingosyl-phosphocholine\n",
      "C05681  Ceramide 2-aminoethylphosphonate\n",
      "C06062  Ceramide phosphoethanolamine\n",
      "C06124  Sphingosine 1-phosphate\n",
      "C06125  Sulfatide\n",
      "C06126  Digalactosylceramide\n",
      "C06127  Digalactosylceramide sulfate\n",
      "C06128  GM4\n",
      "C12126  Dihydroceramide\n",
      "C12144  Phytosphingosine\n",
      "C12145  Phytoceramide\n",
      "REFERENCE   PMID:12011104\n",
      "AUTHORS   Merrill AH Jr.\n",
      "TITLE     De novo sphingolipid biosynthesis: a necessary, but dangerous, pathway.\n",
      "JOURNAL   J Biol Chem 277:25843-6 (2002)\n",
      "DOI:10.1074/jbc.R200009200\n",
      "REFERENCE   PMID:10652340\n",
      "AUTHORS   Tani M, Okino N, Mitsutake S, Tanigawa T, Izu H, Ito M.\n",
      "TITLE     Purification and characterization of a neutral ceramidase from mouse liver. A single protein catalyzes the reversible reaction in which ceramide is both hydrolyzed and synthesized.\n",
      "JOURNAL   J Biol Chem 275:3462-8 (2000)\n",
      "DOI:10.1074/jbc.275.5.3462\n",
      "REFERENCE   PMID:11305904\n",
      "AUTHORS   Hannun YA, Luberto C, Argraves KM.\n",
      "TITLE     Enzymes of sphingolipid metabolism: from modular to integrative signaling.\n",
      "JOURNAL   Biochemistry 40:4893-903 (2001)\n",
      "DOI:10.1021/bi002836k\n",
      "REFERENCE   PMID:1495423\n",
      "AUTHORS   Merrill AH Jr, Wang E.\n",
      "TITLE     Enzymes of ceramide biosynthesis.\n",
      "JOURNAL   Methods Enzymol 209:427-37 (1992)\n",
      "DOI:10.1016/0076-6879(92)09053-6\n",
      "REFERENCE   PMID:8457204\n",
      "AUTHORS   Hirschberg K, Rodger J, Futerman AH.\n",
      "TITLE     The long-chain sphingoid base of sphingolipids is acylated at the cytosolic surface of the endoplasmic reticulum in rat liver.\n",
      "JOURNAL   Biochem J 290 ( Pt 3):751-7 (1993)\n",
      "DOI:10.1042/bj2900751\n",
      "REFERENCE   PMID:10366774\n",
      "AUTHORS   Dickson RC, Lester RL.\n",
      "TITLE     Metabolism and selected functions of sphingolipids in the yeast Saccharomyces cerevisiae.\n",
      "JOURNAL   Biochim Biophys Acta 1438:305-21 (1999)\n",
      "DOI:10.1016/S1388-1981(99)00068-2\n",
      "REL_PATHWAY ath00260  Glycine, serine and threonine metabolism\n",
      "ath00601  Glycosphingolipid biosynthesis - lacto and neolacto series\n",
      "ath00603  Glycosphingolipid biosynthesis - globo and isoglobo series\n",
      "ath00604  Glycosphingolipid biosynthesis - ganglio series\n",
      "KO_PATHWAY  ko00600\n",
      "///\n",
      "ENTRY       ath00601                    Pathway\n",
      "NAME        Glycosphingolipid biosynthesis - lacto and neolacto series - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00601  Glycosphingolipid biosynthesis - lacto and neolacto series\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G38150  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT3G09020  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT1G61050  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT5G01250  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT2G38152  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "COMPOUND    G00036  Lc3Cer\n",
      "G00037  Lc4Cer\n",
      "G00038\n",
      "G00039  Type IB glycolipid\n",
      "G00040\n",
      "G00042  Type IA glycolipid\n",
      "G00043\n",
      "G00044  IV2Fuc-Lc4Cer\n",
      "G00045  IV2Fuc,III4Fuc-Lc4Cer\n",
      "G00046  Fuc-Lc4Cer\n",
      "G00047  3'-isoLM1\n",
      "G00048  Fuc-3'-isoLM1\n",
      "G00050  Paragloboside\n",
      "G00051  nLc5Cer\n",
      "G00052  Type II B antigen\n",
      "G00054  Type II A antigen\n",
      "G00055  IV2Fuc-nLc4Cer\n",
      "G00056  III3,IV2Fuc-nLc4Cer\n",
      "G00057\n",
      "G00058  Type IIIH glycolipid\n",
      "G00059  Type IIIA glycolipid\n",
      "G00060  III3Fuc-nLc4Cer\n",
      "G00061  Sialyl-6-paragloboside\n",
      "G00062  Sialyl-3-paragloboside\n",
      "G00063  IV3NeuAc,III3Fuc-nLc4Cer\n",
      "G00064  3',8'-LD1\n",
      "G00066  nLc5Cer\n",
      "G00067  nLc6Cer\n",
      "G00068  nLc7Cer\n",
      "G00069  nLc8Cer\n",
      "G00071  VI2Fuc-nLc6\n",
      "G00072\n",
      "G00073\n",
      "G00074\n",
      "G00075  Type IIIAb\n",
      "G00076  III3Fuc-nLc6Cer\n",
      "G00077\n",
      "G00078  iso-nLc8Cer\n",
      "G00079\n",
      "G00080\n",
      "G00081\n",
      "G00082\n",
      "G00083\n",
      "G00084\n",
      "G00085\n",
      "G00086\n",
      "G00087  VI6NeuAc-nLc6Cer\n",
      "G00088  VI3NeuAc-nLc6Cer\n",
      "G00089  V3Fuc-nLc6Cer\n",
      "G00090  V3Fuc,III3Fuc-nLc6Cer\n",
      "G00091\n",
      "G00092  Lactosylceramide\n",
      "G00428  IV4Gala-nLc4-Cer\n",
      "G00889  PX2\n",
      "REL_PATHWAY ath00600  Sphingolipid metabolism\n",
      "KO_PATHWAY  ko00601\n",
      "///\n",
      "ENTRY       ath00603                    Pathway\n",
      "NAME        Glycosphingolipid biosynthesis - globo and isoglobo series - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00603  Glycosphingolipid biosynthesis - globo and isoglobo series\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G38150  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT3G09020  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT1G61050  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT5G01250  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT2G38152  alpha 1,4-glycosyltransferase family protein [KO:K01988] [EC:2.4.1.228]\n",
      "AT3G56310  Melibiase family protein [KO:K07407] [EC:3.2.1.22]\n",
      "AT5G08370  AGAL2; alpha-galactosidase 2 [KO:K07407] [EC:3.2.1.22]\n",
      "AT5G08380  AGAL1; alpha-galactosidase 1 [KO:K07407] [EC:3.2.1.22]\n",
      "AT3G55260  HEXO1; beta-hexosaminidase 1 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G65590  HEXO3; beta-hexosaminidase 3 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G05590  HEXO2; beta-hexosaminidase 2 [KO:K12373] [EC:3.2.1.52]\n",
      "COMPOUND    G00092  Lactosylceramide\n",
      "G00093  Globotriaosylceramide\n",
      "G00094  Globoside\n",
      "G00095  Forssman antigen, IV3GalNAca-Gb4Cer\n",
      "G00096  IV3GalNAcb-Gb4Cer\n",
      "G00097  Galactosylgloboside\n",
      "G00098  Monosialylgalactosylgloboside\n",
      "G00099  Globo-H\n",
      "G00100  V3(NeuAc)2-Gb5Cer\n",
      "G00101\n",
      "G00102\n",
      "G00103\n",
      "G00104\n",
      "G00105  GalGb3Cer\n",
      "G00106\n",
      "G00107\n",
      "G00181  Disialosylglobopentaosylceramide\n",
      "G00392  iGb4-Cer\n",
      "G00403  iGb3-Cer\n",
      "G02977\n",
      "G11492  IV4-alpha-Gal-Gb4Cer\n",
      "G13080  NORint\n",
      "G13081  NOR2 antigen\n",
      "REFERENCE   PMID:12716912\n",
      "AUTHORS   Saito S, Aoki H, Ito A, Ueno S, Wada T, Mitsuzuka K, Satoh M, Arai Y, Miyagi T.\n",
      "TITLE     Human alpha2,3-sialyltransferase (ST3Gal II) is a stage-specific embryonic antigen-4 synthase.\n",
      "JOURNAL   J Biol Chem 278:26474-9 (2003)\n",
      "DOI:10.1074/jbc.M213223200\n",
      "REFERENCE   PMID:10837462\n",
      "AUTHORS   Zhou D, Henion TR, Jungalwala FB, Berger EG, Hennet T.\n",
      "TITLE     The beta 1,3-galactosyltransferase beta 3GalT-V is a stage-specific embryonic antigen-3 (SSEA-3) synthase.\n",
      "JOURNAL   J Biol Chem 275:22631-4 (2000)\n",
      "DOI:10.1074/jbc.C000263200\n",
      "REFERENCE   PMID:10748143\n",
      "AUTHORS   Kojima Y, Fukumoto S, Furukawa K, Okajima T, Wiels J, Yokoyama K, Suzuki Y, Urano T, Ohta M, Furukawa K.\n",
      "TITLE     Molecular cloning of globotriaosylceramide/CD77 synthase, aglycosyltransferase that initiates the synthesis of globo seriesglycosphingolipids.\n",
      "JOURNAL   J Biol Chem 275:15152-6 (2000)\n",
      "DOI:10.1074/jbc.M909620199\n",
      "REFERENCE   PMID:10854428\n",
      "AUTHORS   Keusch JJ, Manzella SM, Nyame KA, Cummings RD, Baenziger JU.\n",
      "TITLE     Cloning of Gb3 synthase, the key enzyme in globo-series glycosphingolipidsynthesis, predicts a family of alpha 1, 4-glycosyltransferases conservedin plants, insects, and mammals.\n",
      "JOURNAL   J Biol Chem 275:25315-21 (2000)\n",
      "DOI:10.1074/jbc.M002630200\n",
      "REFERENCE   PMID:10854427\n",
      "AUTHORS   Keusch JJ, Manzella SM, Nyame KA, Cummings RD, Baenziger JU\n",
      "TITLE     Expression cloning of a new member of the ABO blood group glycosyltransferases, iGb3 synthase, that directs the synthesis of isoglobo-glycosphingolipids.\n",
      "JOURNAL   J Biol Chem 275:25308-14 (2000)\n",
      "DOI:10.1074/jbc.M002629200\n",
      "REL_PATHWAY ath00600  Sphingolipid metabolism\n",
      "KO_PATHWAY  ko00603\n",
      "///\n",
      "ENTRY       ath00604                    Pathway\n",
      "NAME        Glycosphingolipid biosynthesis - ganglio series - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Glycan biosynthesis and metabolism\n",
      "PATHWAY_MAP ath00604  Glycosphingolipid biosynthesis - ganglio series\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G72990  BGAL17; beta-galactosidase 17 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G52840  BGAL2; beta-galactosidase 2 [KO:K12309] [EC:3.2.1.23]\n",
      "AT3G55260  HEXO1; beta-hexosaminidase 1 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G65590  HEXO3; beta-hexosaminidase 3 [KO:K12373] [EC:3.2.1.52]\n",
      "AT1G05590  HEXO2; beta-hexosaminidase 2 [KO:K12373] [EC:3.2.1.52]\n",
      "COMPOUND    G00092  Lactosylceramide\n",
      "G00108  GM3\n",
      "G00109  GM2\n",
      "G00110  GM1\n",
      "G00111  GD1a\n",
      "G00112  GT1a\n",
      "G00113  GD3\n",
      "G00114  GD2\n",
      "G00115  GD1b\n",
      "G00116  GT1b\n",
      "G00117  GQ1b\n",
      "G00118  GT3\n",
      "G00119  GT2\n",
      "G00120  GT1c\n",
      "G00121  GQ1c\n",
      "G00122  GP1c\n",
      "G00123  GA2\n",
      "G00124  GA1\n",
      "G00125  GM1b\n",
      "G00126  GD1c\n",
      "G00127  GD1a\n",
      "G00128  GT1aalpha\n",
      "G00129  GQ1balpha\n",
      "G00130  GM2alpha\n",
      "G00131  GM1alpha\n",
      "G00132  GD1beta\n",
      "G00166  Fucosyl-GM1\n",
      "G00167  N-Acetylgalactosaminyl GM1b\n",
      "G00168  N-Acetylgalactosaminyl GD1a\n",
      "G00169  9-O-Acetylated GD3\n",
      "G00170  9-O-Acetylated GT3\n",
      "REL_PATHWAY ath00600  Sphingolipid metabolism\n",
      "KO_PATHWAY  ko00604\n",
      "///\n",
      "ENTRY       ath00620                    Pathway\n",
      "NAME        Pyruvate metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00620  Pyruvate metabolism\n",
      "MODULE      ath_M00168  CAM (Crassulacean acid metabolism), dark [PATH:ath00620]\n",
      "ath_M00169  CAM (Crassulacean acid metabolism), light [PATH:ath00620]\n",
      "ath_M00172  C4-dicarboxylic acid cycle, NADP - malic enzyme type [PATH:ath00620]\n",
      "ath_M00307  Pyruvate oxidation, pyruvate => acetyl-CoA [PATH:ath00620]\n",
      "DBLINKS     GO: 0006090\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G36880  ACS; acetyl-CoA synthetase [KO:K01895] [EC:6.2.1.1]\n",
      "AT3G16910  AAE7; acyl-activating enzyme 7 [KO:K01913] [EC:6.2.1.1 6.2.1.2]\n",
      "AT1G59900  E1_ALPHA; pyruvate dehydrogenase complex E1 alpha subunit [KO:K00161] [EC:1.2.4.1]\n",
      "AT1G01090  PDH-E1_ALPHA; pyruvate dehydrogenase E1 alpha [KO:K00161] [EC:1.2.4.1]\n",
      "AT1G24180  IAR4; Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00161] [EC:1.2.4.1]\n",
      "AT5G50850  MAB1; Transketolase family protein [KO:K00162] [EC:1.2.4.1]\n",
      "AT2G34590  Transketolase family protein [KO:K00162] [EC:1.2.4.1]\n",
      "AT1G30120  PDH-E1_BETA; pyruvate dehydrogenase E1 beta [KO:K00162] [EC:1.2.4.1]\n",
      "AT1G34430  EMB3003; 2-oxoacid dehydrogenases acyltransferase family protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G52200  LTA3; Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT1G54220  Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G13930  Dihydrolipoamide acetyltransferase, long form protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G25860  LTA2; 2-oxoacid dehydrogenases acyltransferase family protein [KO:K00627] [EC:2.3.1.12]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G32780  GroES-like zinc-binding dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT5G24760  GroES-like zinc-binding dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT1G64710  GroES-like zinc-binding alcohol dehydrogenase family protein [KO:K00001] [EC:1.1.1.1]\n",
      "AT1G22440  Zinc-binding alcohol dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT5G42250  Zinc-binding alcohol dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT5G43940  HOT5; GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT4G22110  GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT1G22430  GroES-like zinc-binding dehydrogenase family protein [KO:K00121] [EC:1.1.1.284 1.1.1.1]\n",
      "AT1G77120  ADH1; alcohol dehydrogenase 1 [KO:K18857] [EC:1.1.1.1]\n",
      "AT2G37790  AKR4C10; NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT5G01670  NAD(P)-linked oxidoreductase superfamily protein [KO:K00002] [EC:1.1.1.2]\n",
      "AT3G04050  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT5G52920  PKP-BETA1; plastidic pyruvate kinase beta subunit 1 [KO:K00873] [EC:2.7.1.40]\n",
      "AT5G56350  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G25960  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G52990  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G55650  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G55810  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT5G08570  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT5G63680  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT4G26390  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT1G32440  PKp3; plastidial pyruvate kinase 3 [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G49160  pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT3G22960  PKP-ALPHA; Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT2G36580  Pyruvate kinase family protein [KO:K00873] [EC:2.7.1.40]\n",
      "AT1G36160  ACC1; acetyl-CoA carboxylase 1 [KO:K11262] [EC:6.4.1.2 6.3.4.14 2.1.3.15]\n",
      "AT1G36180  ACC2; acetyl-CoA carboxylase 2 [KO:K11262] [EC:6.4.1.2 6.3.4.14 2.1.3.15]\n",
      "AT2G38040  CAC3; acetyl Co-enzyme a carboxylase carboxyltransferase alpha subunit [KO:K01962] [EC:6.4.1.2 2.1.3.15]\n",
      "AT5G16390  CAC1; chloroplastic acetylcoenzyme A carboxylase 1 [KO:K02160]\n",
      "AT5G15530  BCCP2; biotin carboxyl carrier protein 2 [KO:K02160]\n",
      "AT5G35360  CAC2; acetyl Co-enzyme a carboxylase biotin carboxylase subunit [KO:K01961] [EC:6.4.1.2 6.3.4.14]\n",
      "ArthCp031  accD; acetyl-CoA carboxylase beta subunit [KO:K01963] [EC:6.4.1.2 2.1.3.15]\n",
      "AT5G03370  acylphosphatase family [KO:K01512] [EC:3.6.1.7]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G54100  ALDH7B4; aldehyde dehydrogenase 7B4 [KO:K14085] [EC:1.2.1.31 1.2.1.8 1.2.1.3]\n",
      "AT4G17260  Lactate/malate dehydrogenase family protein [KO:K00016] [EC:1.1.1.27]\n",
      "AT5G06580  FAD-linked oxidases family protein [KO:K00102] [EC:1.1.2.4]\n",
      "AT1G08110  lactoylglutathione lyase family protein / glyoxalase I family protein [KO:K01759] [EC:4.4.1.5]\n",
      "AT1G67280  Glyoxalase/Bleomycin resistance protein/Dioxygenase superfamily protein [KO:K01759] [EC:4.4.1.5]\n",
      "AT1G11840  GLX1; glyoxalase/bleomycin resistance protein/dioxygenase superfamily protein [KO:K01759] [EC:4.4.1.5]\n",
      "AT3G10850  GLY2; Metallo-hydrolase/oxidoreductase superfamily protein [KO:K01069] [EC:3.1.2.6]\n",
      "AT1G06130  GLX2-4; glyoxalase 2-4 [KO:K01069] [EC:3.1.2.6]\n",
      "AT2G43430  GLX2-1; glyoxalase 2-1 [KO:K01069] [EC:3.1.2.6]\n",
      "AT2G31350  GLX2-5; glyoxalase 2-5 [KO:K01069] [EC:3.1.2.6]\n",
      "AT3G02720  DJ1D; Class I glutamine amidotransferase-like superfamily protein [KO:K18881] [EC:4.2.1.130]\n",
      "AT2G38860  YLS5; Class I glutamine amidotransferase-like superfamily protein [KO:K18881] [EC:4.2.1.130]\n",
      "AT3G54600  DJ1F; Class I glutamine amidotransferase-like superfamily protein [KO:K18881] [EC:4.2.1.130]\n",
      "AT4G00570  NAD-ME2; NAD-dependent malic enzyme 2 [KO:K00028] [EC:1.1.1.39]\n",
      "AT2G13560  NAD-ME1; NAD-dependent malic enzyme 1 [KO:K00028] [EC:1.1.1.39]\n",
      "AT5G25880  NADP-ME3; NADP-malic enzyme 3 [KO:K00029] [EC:1.1.1.40]\n",
      "AT5G11670  NADP-ME2; NADP-malic enzyme 2 [KO:K00029] [EC:1.1.1.40]\n",
      "AT2G19900  NADP-ME1; NADP-malic enzyme 1 [KO:K00029] [EC:1.1.1.40]\n",
      "AT1G79750  NADP-ME4; NADP-malic enzyme 4 [KO:K00029] [EC:1.1.1.40]\n",
      "AT1G04410  c-NAD-MDH1; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G56720  c-NAD-MDH3; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G43330  c-NAD-MDH2; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT3G47520  MDH; malate dehydrogenase [KO:K00026] [EC:1.1.1.37]\n",
      "AT1G53240  mMDH1; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT2G22780  PMDH1; peroxisomal NAD-malate dehydrogenase 1 [KO:K00026] [EC:1.1.1.37]\n",
      "AT3G15020  mMDH2; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT5G09660  PMDH2; peroxisomal NAD-malate dehydrogenase 2 [KO:K00026] [EC:1.1.1.37]\n",
      "AT5G58330  lactate/malate dehydrogenase family protein [KO:K00051] [EC:1.1.1.82]\n",
      "AT5G50950  FUM2; FUMARASE 2 [KO:K01679] [EC:4.2.1.2]\n",
      "AT2G47510  FUM1; fumarase 1 [KO:K01679] [EC:4.2.1.2]\n",
      "AT1G53310  PPC1; phosphoenolpyruvate carboxylase 1 [KO:K01595] [EC:4.1.1.31]\n",
      "AT3G14940  PPC3; phosphoenolpyruvate carboxylase 3 [KO:K01595] [EC:4.1.1.31]\n",
      "AT2G42600  PPC2; phosphoenolpyruvate carboxylase 2 [KO:K01595] [EC:4.1.1.31]\n",
      "AT1G68750  PPC4; phosphoenolpyruvate carboxylase 4 [KO:K01595] [EC:4.1.1.31]\n",
      "AT5G65690  PCK2; phosphoenolpyruvate carboxykinase 2 [KO:K01610] [EC:4.1.1.49]\n",
      "AT4G37870  PCK1; phosphoenolpyruvate carboxykinase 1 [KO:K01610] [EC:4.1.1.49]\n",
      "AT4G15530  PPDK; pyruvate orthophosphate dikinase [KO:K01006] [EC:2.7.9.1]\n",
      "AT5G03860  MLS; malate synthase [KO:K01638] [EC:2.3.3.9]\n",
      "AT5G48230  ACAT2; acetoacetyl-CoA thiolase 2 [KO:K00626] [EC:2.3.1.9]\n",
      "AT5G47720  AACT1; Thiolase family protein [KO:K00626] [EC:2.3.1.9]\n",
      "AT1G18500  MAML-4; methylthioalkylmalate synthase-like 4 [KO:K01649] [EC:2.3.3.13]\n",
      "AT1G74040  IMS1; 2-isopropylmalate synthase 1 [KO:K01649] [EC:2.3.3.13]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00033  Acetate\n",
      "C00036  Oxaloacetate\n",
      "C00042  Succinate\n",
      "C00058  Formate\n",
      "C00068  Thiamin diphosphate\n",
      "C00074  Phosphoenolpyruvate\n",
      "C00083  Malonyl-CoA\n",
      "C00084  Acetaldehyde\n",
      "C00122  Fumarate\n",
      "C00149  (S)-Malate\n",
      "C00186  (S)-Lactate\n",
      "C00227  Acetyl phosphate\n",
      "C00256  (R)-Lactate\n",
      "C00332  Acetoacetyl-CoA\n",
      "C00424  (S)-Lactaldehyde\n",
      "C00469  Ethanol\n",
      "C00546  Methylglyoxal\n",
      "C00937  (R)-Lactaldehyde\n",
      "C01251  (R)-2-Hydroxybutane-1,2,4-tricarboxylate\n",
      "C02488  (R)-2-Ethylmalate\n",
      "C02504  alpha-Isopropylmalate\n",
      "C03248  Acetylenedicarboxylate\n",
      "C03451  (R)-S-Lactoylglutathione\n",
      "C03981  2-Hydroxyethylenedicarboxylate\n",
      "C05125  2-(alpha-Hydroxyethyl)thiamine diphosphate\n",
      "C05993  Acetyl adenylate\n",
      "C05994  2-Propylmalate\n",
      "C15972  Enzyme N6-(lipoyl)lysine\n",
      "C15973  Enzyme N6-(dihydrolipoyl)lysine\n",
      "C16255  [Dihydrolipoyllysine-residue acetyltransferase] S-acetyldihydrolipoyllysine\n",
      "REFERENCE\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00061  Fatty acid biosynthesis\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00640  Propanoate metabolism\n",
      "ath00650  Butanoate metabolism\n",
      "ath00760  Nicotinate and nicotinamide metabolism\n",
      "KO_PATHWAY  ko00620\n",
      "///\n",
      "ENTRY       ath00630                    Pathway\n",
      "NAME        Glyoxylate and dicarboxylate metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "MODULE      ath_M00012  Glyoxylate cycle [PATH:ath00630]\n",
      "ath_M00532  Photorespiration [PATH:ath00630]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G21720  ICL; isocitrate lyase [KO:K01637] [EC:4.1.3.1]\n",
      "AT5G36880  ACS; acetyl-CoA synthetase [KO:K01895] [EC:6.2.1.1]\n",
      "AT3G16910  AAE7; acyl-activating enzyme 7 [KO:K01913] [EC:6.2.1.1 6.2.1.2]\n",
      "AT5G03860  MLS; malate synthase [KO:K01638] [EC:2.3.3.9]\n",
      "AT1G04410  c-NAD-MDH1; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G56720  c-NAD-MDH3; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G43330  c-NAD-MDH2; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT3G47520  MDH; malate dehydrogenase [KO:K00026] [EC:1.1.1.37]\n",
      "AT1G53240  mMDH1; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT2G22780  PMDH1; peroxisomal NAD-malate dehydrogenase 1 [KO:K00026] [EC:1.1.1.37]\n",
      "AT3G15020  mMDH2; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT5G09660  PMDH2; peroxisomal NAD-malate dehydrogenase 2 [KO:K00026] [EC:1.1.1.37]\n",
      "AT2G42790  CSY3; citrate synthase 3 [KO:K01647] [EC:2.3.3.1]\n",
      "AT3G58740  CSY1; citrate synthase 1 [KO:K01647] [EC:2.3.3.1]\n",
      "AT3G58750  CSY2; citrate synthase 2 [KO:K01647] [EC:2.3.3.1]\n",
      "AT3G60100  CSY5; citrate synthase 5 [KO:K01647] [EC:2.3.3.1]\n",
      "AT2G44350  ATCS; Citrate synthase family protein [KO:K01647] [EC:2.3.3.1]\n",
      "AT4G35830  ACO1; aconitase 1 [KO:K01681] [EC:4.2.1.3]\n",
      "AT2G05710  ACO3; aconitase 3 [KO:K01681] [EC:4.2.1.3]\n",
      "AT4G26970  ACO2; aconitase 2 [KO:K01681] [EC:4.2.1.3]\n",
      "AT5G48230  ACAT2; acetoacetyl-CoA thiolase 2 [KO:K00626] [EC:2.3.1.9]\n",
      "AT5G47720  AACT1; Thiolase family protein [KO:K00626] [EC:2.3.1.9]\n",
      "AT3G14415  GOX2; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT4G18360  GOX3; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT3G14150  HAOX2; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT3G14130  HAOX1; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT3G14420  GOX1; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT1G20630  CAT1; catalase 1 [KO:K03781] [EC:1.11.1.6]\n",
      "AT4G35090  CAT2; catalase 2 [KO:K03781] [EC:1.11.1.6]\n",
      "AT1G20620  CAT3; catalase 3 [KO:K03781] [EC:1.11.1.6]\n",
      "AT1G79870  D-isomer specific 2-hydroxyacid dehydrogenase family protein [KO:K15919] [EC:1.1.1.79 1.1.1.81]\n",
      "AT1G12550  HPR3; D-isomer specific 2-hydroxyacid dehydrogenase family protein [KO:K15919] [EC:1.1.1.79 1.1.1.81]\n",
      "AT3G25530  GLYR1; glyoxylate reductase 1 [KO:K18121] [EC:1.1.1.79 1.1.1.-]\n",
      "AT1G17650  GLYR2; glyoxylate reductase 2 [KO:K18121] [EC:1.1.1.79 1.1.1.-]\n",
      "AT1G68010  HPR; hydroxypyruvate reductase [KO:K15893] [EC:1.1.1.29]\n",
      "AT5G47760  PGLP2; 2-phosphoglycolate phosphatase 2 [KO:K19269] [EC:3.1.3.18 3.1.3.48]\n",
      "AT5G36790  Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K19269] [EC:3.1.3.18 3.1.3.48]\n",
      "AT5G36700  PGLP1; 2-phosphoglycolate phosphatase 1 [KO:K19269] [EC:3.1.3.18 3.1.3.48]\n",
      "ArthCp030  rbcL; ribulose-1,5-bisphosphate carboxylase/oxygenase large subunit [KO:K01601] [EC:4.1.1.39]\n",
      "AT1G67090  RBCS1A; ribulose bisphosphate carboxylase small chain 1A [KO:K01602] [EC:4.1.1.39]\n",
      "AT5G38420  RBCS2B; Ribulose bisphosphate carboxylase (small chain) family protein [KO:K01602] [EC:4.1.1.39]\n",
      "AT5G38410  RBCS3B; Ribulose bisphosphate carboxylase (small chain) family protein [KO:K01602] [EC:4.1.1.39]\n",
      "AT5G38430  RBCS1B; Ribulose bisphosphate carboxylase (small chain) family protein [KO:K01602] [EC:4.1.1.39]\n",
      "AT2G13360  AGT; alanine:glyoxylate aminotransferase [KO:K00830] [EC:2.6.1.44 2.6.1.45 2.6.1.51]\n",
      "AT1G70580  AOAT2; alanine-2-oxoglutarate aminotransferase 2 [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT1G23310  GGT1; glutamate:glyoxylate aminotransferase [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT2G41220  GLU2; glutamate synthase 2 [KO:K00284] [EC:1.4.7.1]\n",
      "AT5G04140  GLU1; glutamate synthase 1 [KO:K00284] [EC:1.4.7.1]\n",
      "AT5G37600  GSR_1; hypothetical protein [KO:K01915] [EC:6.3.1.2]\n",
      "AT3G17820  GLN1.3; glutamine synthetase 1.3 [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G48470  GLN1;5; glutamine synthetase 1;5 [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G35630  GS2; glutamine synthetase 2 [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G16570  GLN1;4; glutamine synthetase 1;4 [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G66200  GSR2; hypothetical protein [KO:K01915] [EC:6.3.1.2]\n",
      "AT4G13890  EDA36; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G13930  SHM4; serine hydroxymethyltransferase 4 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G22020  SHM6; serine hydroxymethyltransferase 6 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G32520  SHM3; serine hydroxymethyltransferase 3 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G37930  SHM1; serine transhydroxymethyltransferase 1 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G36370  SHM7; serine hydroxymethyltransferase 7 [KO:K00600] [EC:2.1.2.1]\n",
      "AT5G26780  SHM2; serine hydroxymethyltransferase 2 [KO:K00600] [EC:2.1.2.1]\n",
      "AT2G26080  GLDP2; glycine decarboxylase P-protein 2 [KO:K00281] [EC:1.4.4.2]\n",
      "AT4G33010  GLDP1; glycine decarboxylase P-protein 1 [KO:K00281] [EC:1.4.4.2]\n",
      "AT1G11860  Glycine cleavage T-protein family [KO:K00605] [EC:2.1.2.10]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G32470  Single hybrid motif superfamily protein [KO:K02437]\n",
      "AT2G35120  Single hybrid motif superfamily protein [KO:K02437]\n",
      "AT2G35370  GDCH; glycine decarboxylase complex H [KO:K02437]\n",
      "AT1G80380  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K15918] [EC:2.7.1.31]\n",
      "AT5G14780  FDH; formate dehydrogenase [KO:K00122] [EC:1.17.1.9]\n",
      "AT4G17360  Formyl transferase [KO:K01433] [EC:3.5.1.10]\n",
      "AT5G47435  formyltetrahydrofolate deformylase [KO:K01433] [EC:3.5.1.10]\n",
      "AT4G37550  Acetamidase/Formamidase family protein [KO:K01455] [EC:3.5.1.49]\n",
      "AT4G37560  Acetamidase/Formamidase family protein [KO:K01455] [EC:3.5.1.49]\n",
      "AT3G48990  AAE3; AMP-dependent synthetase and ligase family protein [KO:K22133] [EC:6.2.1.8]\n",
      "COMPOUND    C00007  Oxygen\n",
      "C00011  CO2\n",
      "C00014  Ammonia\n",
      "C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00027  Hydrogen peroxide\n",
      "C00033  Acetate\n",
      "C00036  Oxaloacetate\n",
      "C00037  Glycine\n",
      "C00042  Succinate\n",
      "C00048  Glyoxylate\n",
      "C00058  Formate\n",
      "C00064  L-Glutamine\n",
      "C00065  L-Serine\n",
      "C00091  Succinyl-CoA\n",
      "C00100  Propanoyl-CoA\n",
      "C00136  Butanoyl-CoA\n",
      "C00149  (S)-Malate\n",
      "C00158  Citrate\n",
      "C00160  Glycolate\n",
      "C00168  Hydroxypyruvate\n",
      "C00197  3-Phospho-D-glycerate\n",
      "C00209  Oxalate\n",
      "C00258  D-Glycerate\n",
      "C00266  Glycolaldehyde\n",
      "C00311  Isocitrate\n",
      "C00313  Oxalyl-CoA\n",
      "C00332  Acetoacetyl-CoA\n",
      "C00417  cis-Aconitate\n",
      "C00552  meso-Tartaric acid\n",
      "C00631  2-Phospho-D-glycerate\n",
      "C00683  (S)-Methylmalonyl-CoA\n",
      "C00798  Formyl-CoA\n",
      "C00877  Crotonoyl-CoA\n",
      "C00888  Pentanoyl-CoA\n",
      "C00898  (R,R)-Tartaric acid\n",
      "C00975  Dihydroxyfumarate\n",
      "C00988  2-Phosphoglycolate\n",
      "C01127  4-Hydroxy-2-oxoglutarate\n",
      "C01146  2-Hydroxy-3-oxopropanoate\n",
      "C01182  D-Ribulose 1,5-bisphosphate\n",
      "C01213  (R)-Methylmalonyl-CoA\n",
      "C01380  Ethylene glycol\n",
      "C01732  Mesaconate\n",
      "C01989  3-Ethylmalate\n",
      "C01990  3-Oxalomalate\n",
      "C02123  3-Propylmalate\n",
      "C02405  Formyl phosphate\n",
      "C03217  2-Hydroxy-3-oxoadipate\n",
      "C03459  2-Hydroxy-3-oxosuccinate\n",
      "C03548  trans-2,3-Epoxysuccinate\n",
      "C03561  (R)-3-Hydroxybutanoyl-CoA\n",
      "C03618  L-threo-3-Methylaspartate\n",
      "C04348  L-Malyl-CoA\n",
      "C06027  L-erythro-3-Methylmalyl-CoA\n",
      "C06028  2-Methylfumaryl-CoA\n",
      "C06049  N-Formylderivatives\n",
      "C18026  (2S)-Ethylmalonyl-CoA\n",
      "C18324  (2S)-Methylsuccinyl-CoA\n",
      "C20238  (2R)-Ethylmalonyl-CoA\n",
      "REFERENCE   PMID:10978349 (EC5.3.1.22)\n",
      "AUTHORS   Njau RK, Herndon CA, Hawes JW.\n",
      "TITLE     Novel beta -hydroxyacid dehydrogenases in Escherichia coli and Haemophilusinfluenzae.\n",
      "JOURNAL   J Biol Chem 275:38780-6 (2000)\n",
      "DOI:10.1074/jbc.M007432200\n",
      "REFERENCE   PMID:18065535\n",
      "AUTHORS   Zarzycki J, Schlichting A, Strychalsky N, Muller M, Alber BE, Fuchs G\n",
      "TITLE     Mesaconyl-coenzyme A hydratase, a new enzyme of two central carbon metabolic pathways in bacteria.\n",
      "JOURNAL   J Bacteriol 190:1366-74 (2008)\n",
      "DOI:10.1128/JB.01621-07\n",
      "REFERENCE   PMID:18819910\n",
      "AUTHORS   Erb TJ, Retey J, Fuchs G, Alber BE\n",
      "TITLE     Ethylmalonyl-CoA mutase from Rhodobacter sphaeroides defines a new subclade of coenzyme B12-dependent acyl-CoA mutases.\n",
      "JOURNAL   J Biol Chem 283:32283-93 (2008)\n",
      "DOI:10.1074/jbc.M805527200\n",
      "REFERENCE   PMID:9596633\n",
      "AUTHORS   Coschigano KT, Melo-Oliveira R, Lim J, Coruzzi GM\n",
      "TITLE     Arabidopsis gls mutants and distinct Fd-GOGAT genes. Implications for photorespiration and primary nitrogen assimilation.\n",
      "JOURNAL   Plant Cell 10:741-52 (1998)\n",
      "DOI:10.1105/tpc.10.5.741\n",
      "REFERENCE   PMID:16407450\n",
      "AUTHORS   Masclaux-Daubresse C, Reisdorf-Cren M, Pageau K, Lelandais M, Grandjean O, Kronenberger J, Valadier MH, Feraud M, Jouglet T, Suzuki A\n",
      "TITLE     Glutamine synthetase-glutamate synthase pathway and glutamate dehydrogenase play distinct roles in the sink-source nitrogen cycle in tobacco.\n",
      "JOURNAL   Plant Physiol 140:444-56 (2006)\n",
      "DOI:10.1104/pp.105.071910\n",
      "REFERENCE   PMID:21252347\n",
      "AUTHORS   Khomyakova M, Bukmez O, Thomas LK, Erb TJ, Berg IA\n",
      "TITLE     A methylaspartate cycle in haloarchaea.\n",
      "JOURNAL   Science 331:334-7 (2011)\n",
      "DOI:10.1126/science.1196544\n",
      "REFERENCE   PMID:11513957\n",
      "AUTHORS   Serrano JA, Bonete MJ\n",
      "TITLE     Sequencing, phylogenetic and transcriptional analysis of the glyoxylate bypass operon (ace) in the halophilic archaeon Haloferax volcanii.\n",
      "JOURNAL   Biochim Biophys Acta 1520:154-62 (2001)\n",
      "DOI:10.1016/S0167-4781(01)00263-9\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00053  Ascorbate and aldarate metabolism\n",
      "ath00071  Fatty acid degradation\n",
      "ath00230  Purine metabolism\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00710  Carbon fixation in photosynthetic organisms\n",
      "ath00750  Vitamin B6 metabolism\n",
      "ath00910  Nitrogen metabolism\n",
      "KO_PATHWAY  ko00630\n",
      "///\n",
      "ENTRY       ath00640                    Pathway\n",
      "NAME        Propanoate metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00640  Propanoate metabolism\n",
      "DBLINKS     GO: 0019541\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G36880  ACS; acetyl-CoA synthetase [KO:K01895] [EC:6.2.1.1]\n",
      "AT1G21400  Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00166] [EC:1.2.4.4]\n",
      "AT5G09300  Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00166] [EC:1.2.4.4]\n",
      "AT5G34780  Thiamin diphosphate-binding fold (THDP-binding) superfamily protein [KO:K00166] [EC:1.2.4.4]\n",
      "AT1G55510  BCDH_BETA1; branched-chain alpha-keto acid decarboxylase E1 beta subunit [KO:K00167] [EC:1.2.4.4]\n",
      "AT3G13450  DIN4; Transketolase family protein [KO:K00167] [EC:1.2.4.4]\n",
      "AT3G06850  BCE2; 2-oxoacid dehydrogenases acyltransferase family protein [KO:K09699] [EC:2.3.1.168]\n",
      "AT3G17240  mtLPD2; lipoamide dehydrogenase 2 [KO:K00382] [EC:1.8.1.4]\n",
      "AT3G16950  LPD1; lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT1G48030  mtLPD1; mitochondrial lipoamide dehydrogenase 1 [KO:K00382] [EC:1.8.1.4]\n",
      "AT4G16155  dihydrolipoamide dehydrogenase [KO:K00382] [EC:1.8.1.4]\n",
      "AT2G35690  ACX5; acyl-CoA oxidase 5 [KO:K00232] [EC:1.3.3.6]\n",
      "AT4G16760  ACX1; acyl-CoA oxidase 1 [KO:K00232] [EC:1.3.3.6]\n",
      "AT5G65110  ACX2; acyl-CoA oxidase 2 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G51840  ACX4; acyl-CoA oxidase 4 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06290  ACX3; acyl-CoA oxidase 3 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06310  ACX6; acyl-CoA oxidase 6 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G06690  putative acyl-coenzyme A oxidase [KO:K00232] [EC:1.3.3.6]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "AT2G30650  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT5G65940  CHY1; beta-hydroxyisobutyryl-CoA hydrolase 1 [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G31810  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT1G06550  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT2G30660  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT3G60510  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT3G24360  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G13360  ATP-dependent caseinolytic (Clp) protease/crotonase family protein [KO:K05605] [EC:3.1.2.4]\n",
      "AT4G20930  6-phosphogluconate dehydrogenase family protein [KO:K23146] [EC:1.1.1.31 1.1.1.59]\n",
      "AT1G36160  ACC1; acetyl-CoA carboxylase 1 [KO:K11262] [EC:6.4.1.2 6.3.4.14 2.1.3.15]\n",
      "AT1G36180  ACC2; acetyl-CoA carboxylase 2 [KO:K11262] [EC:6.4.1.2 6.3.4.14 2.1.3.15]\n",
      "AT2G38040  CAC3; acetyl Co-enzyme a carboxylase carboxyltransferase alpha subunit [KO:K01962] [EC:6.4.1.2 2.1.3.15]\n",
      "AT5G16390  CAC1; chloroplastic acetylcoenzyme A carboxylase 1 [KO:K02160]\n",
      "AT5G15530  BCCP2; biotin carboxyl carrier protein 2 [KO:K02160]\n",
      "AT5G35360  CAC2; acetyl Co-enzyme a carboxylase biotin carboxylase subunit [KO:K01961] [EC:6.4.1.2 6.3.4.14]\n",
      "ArthCp031  accD; acetyl-CoA carboxylase beta subunit [KO:K01963] [EC:6.4.1.2 2.1.3.15]\n",
      "AT4G04320  malonyl-CoA decarboxylase family protein [KO:K01578] [EC:4.1.1.9]\n",
      "AT5G08300  Succinyl-CoA ligase, alpha subunit [KO:K01899] [EC:6.2.1.4 6.2.1.5]\n",
      "AT5G23250  Succinyl-CoA ligase, alpha subunit [KO:K01899] [EC:6.2.1.4 6.2.1.5]\n",
      "AT2G20420  ATP citrate lyase (ACL) family protein [KO:K01900] [EC:6.2.1.4 6.2.1.5]\n",
      "AT2G14170  ALDH6B2; aldehyde dehydrogenase 6B2 [KO:K00140] [EC:1.2.1.18 1.2.1.27]\n",
      "AT4G17260  Lactate/malate dehydrogenase family protein [KO:K00016] [EC:1.1.1.27]\n",
      "COMPOUND    C00024  Acetyl-CoA\n",
      "C00042  Succinate\n",
      "C00068  Thiamin diphosphate\n",
      "C00083  Malonyl-CoA\n",
      "C00091  Succinyl-CoA\n",
      "C00099  beta-Alanine\n",
      "C00100  Propanoyl-CoA\n",
      "C00109  2-Oxobutanoate\n",
      "C00111  Glycerone phosphate\n",
      "C00163  Propanoate\n",
      "C00186  (S)-Lactate\n",
      "C00222  3-Oxopropanoate\n",
      "C00424  (S)-Lactaldehyde\n",
      "C00479  Propanal\n",
      "C00546  Methylglyoxal\n",
      "C00583  Propane-1,2-diol\n",
      "C00683  (S)-Methylmalonyl-CoA\n",
      "C00804  Propynoate\n",
      "C00827  Lactoyl-CoA\n",
      "C00894  Propenoyl-CoA\n",
      "C01013  3-Hydroxypropanoate\n",
      "C01213  (R)-Methylmalonyl-CoA\n",
      "C02170  Methylmalonate\n",
      "C02225  2-Methylcitrate\n",
      "C02335  beta-Alanyl-CoA\n",
      "C02876  Propanoyl phosphate\n",
      "C04225  (Z)-But-2-ene-1,2,3-tricarboxylate\n",
      "C04593  (2S,3R)-3-Hydroxybutane-1,2,3-tricarboxylate\n",
      "C05235  Hydroxyacetone\n",
      "C05668  3-Hydroxypropionyl-CoA\n",
      "C05979  Propane-1-ol\n",
      "C05983  Propionyladenylate\n",
      "C05984  2-Hydroxybutanoic acid\n",
      "C05989  3-Oxopropionyl-CoA\n",
      "C06002  (S)-Methylmalonate semialdehyde\n",
      "C15972  Enzyme N6-(lipoyl)lysine\n",
      "C15973  Enzyme N6-(dihydrolipoyl)lysine\n",
      "C21017  2-(alpha-Hydroxypropyl)thiamine diphosphate\n",
      "C21018  Enzyme N6-(S-propyldihydrolipoyl)lysine\n",
      "C21250  2-Methyl-trans-aconitate\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00280  Valine, leucine and isoleucine degradation\n",
      "ath00410  beta-Alanine metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00660  C5-Branched dibasic acid metabolism\n",
      "ath00770  Pantothenate and CoA biosynthesis\n",
      "KO_PATHWAY  ko00640\n",
      "///\n",
      "ENTRY       ath00650                    Pathway\n",
      "NAME        Butanoate metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00650  Butanoate metabolism\n",
      "MODULE      ath_M00027  GABA (gamma-Aminobutyrate) shunt [PATH:ath00650]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G48230  ACAT2; acetoacetyl-CoA thiolase 2 [KO:K00626] [EC:2.3.1.9]\n",
      "AT5G47720  AACT1; Thiolase family protein [KO:K00626] [EC:2.3.1.9]\n",
      "AT3G15290  3-hydroxyacyl-CoA dehydrogenase family protein [KO:K00074] [EC:1.1.1.157]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "AT3G16910  AAE7; acyl-activating enzyme 7 [KO:K01913] [EC:6.2.1.1 6.2.1.2]\n",
      "AT3G56840  FAD-dependent oxidoreductase family protein [KO:K00109] [EC:1.1.99.2]\n",
      "AT2G02000  GAD3; glutamate decarboxylase 3 [KO:K01580] [EC:4.1.1.15]\n",
      "AT3G17760  GAD5; glutamate decarboxylase 5 [KO:K01580] [EC:4.1.1.15]\n",
      "AT5G17330  GAD; glutamate decarboxylase [KO:K01580] [EC:4.1.1.15]\n",
      "AT2G02010  GAD4; glutamate decarboxylase 4 [KO:K01580] [EC:4.1.1.15]\n",
      "AT1G65960  GAD2; glutamate decarboxylase 2 [KO:K01580] [EC:4.1.1.15]\n",
      "AT3G22200  POP2; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K16871] [EC:2.6.1.96]\n",
      "AT1G79440  ALDH5F1; aldehyde dehydrogenase 5F1 [KO:K17761] [EC:1.2.1.24]\n",
      "AT3G25530  GLYR1; glyoxylate reductase 1 [KO:K18121] [EC:1.1.1.79 1.1.1.-]\n",
      "AT1G17650  GLYR2; glyoxylate reductase 2 [KO:K18121] [EC:1.1.1.79 1.1.1.-]\n",
      "AT4G11820  HMGS; hydroxymethylglutaryl-CoA synthase / HMG-CoA synthase / 3-hydroxy-3-methylglutaryl coenzyme A synthase [KO:K01641] [EC:2.3.3.10]\n",
      "AT2G26800  Aldolase superfamily protein [KO:K01640] [EC:4.1.3.4]\n",
      "AT3G48560  CSR1; chlorsulfuron/imidazolinone resistant 1 [KO:K01652] [EC:2.2.1.6]\n",
      "AT2G31810  ACT domain-containing small subunit of acetolactate synthase protein [KO:K01653] [EC:2.2.1.6]\n",
      "AT5G16290  VAT1; VALINE-TOLERANT 1 [KO:K01653] [EC:2.2.1.6]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00042  Succinate\n",
      "C00091  Succinyl-CoA\n",
      "C00122  Fumarate\n",
      "C00132  Methanol\n",
      "C00136  Butanoyl-CoA\n",
      "C00164  Acetoacetate\n",
      "C00207  Acetone\n",
      "C00232  Succinate semialdehyde\n",
      "C00246  Butanoic acid\n",
      "C00332  Acetoacetyl-CoA\n",
      "C00334  4-Aminobutanoate\n",
      "C00356  (S)-3-Hydroxy-3-methylglutaryl-CoA\n",
      "C00497  (R)-Malate\n",
      "C00741  Diacetyl\n",
      "C00810  (R)-Acetoin\n",
      "C00877  Crotonoyl-CoA\n",
      "C00989  4-Hydroxybutanoic acid\n",
      "C01089  (R)-3-Hydroxybutanoate\n",
      "C01144  (S)-3-Hydroxybutanoyl-CoA\n",
      "C01384  Maleic acid\n",
      "C01412  Butanal\n",
      "C01769  (S)-Acetoin\n",
      "C01845  Propan-2-ol\n",
      "C02331  Vinylacetyl-CoA\n",
      "C02411  Glutaconyl-1-CoA\n",
      "C02527  Butanoylphosphate\n",
      "C02630  2-Hydroxyglutarate\n",
      "C03044  (R,R)-Butane-2,3-diol\n",
      "C03046  (S,S)-Butane-2,3-diol\n",
      "C03058  2-Hydroxyglutaryl-CoA\n",
      "C03561  (R)-3-Hydroxybutanoyl-CoA\n",
      "C04546  (R)-3-((R)-3-Hydroxybutanoyloxy)butanoate\n",
      "C06010  (S)-2-Acetolactate\n",
      "C06142  1-Butanol\n",
      "C06143  Poly-beta-hydroxybutyrate\n",
      "C06144  3-Butynoate\n",
      "C06145  3-Butyn-1-al\n",
      "C06146  3-Butyn-1-ol\n",
      "C11062  4-Hydroxybutyryl-CoA\n",
      "C17530  Methyl acetate\n",
      "C20657  meso-2,3-Butanediol\n",
      "C20783  Propane\n",
      "C21390  Butane\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00750  Vitamin B6 metabolism\n",
      "KO_PATHWAY  ko00650\n",
      "///\n",
      "ENTRY       ath00660                    Pathway\n",
      "NAME        C5-Branched dibasic acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Carbohydrate metabolism\n",
      "PATHWAY_MAP ath00660  C5-Branched dibasic acid metabolism\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G48560  CSR1; chlorsulfuron/imidazolinone resistant 1 [KO:K01652] [EC:2.2.1.6]\n",
      "AT2G31810  ACT domain-containing small subunit of acetolactate synthase protein [KO:K01653] [EC:2.2.1.6]\n",
      "AT5G16290  VAT1; VALINE-TOLERANT 1 [KO:K01653] [EC:2.2.1.6]\n",
      "AT4G13430  IIL1; isopropyl malate isomerase large subunit 1 [KO:K01703] [EC:4.2.1.33 4.2.1.35]\n",
      "AT2G43090  Aconitase/3-isopropylmalate dehydratase protein [KO:K01704] [EC:4.2.1.33 4.2.1.35]\n",
      "AT1G31180  IMD3; isopropylmalate dehydrogenase 3 [KO:K00052] [EC:1.1.1.85]\n",
      "AT1G80560  IMD2; isopropylmalate dehydrogenase 2 [KO:K00052] [EC:1.1.1.85]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00033  Acetate\n",
      "C00048  Glyoxylate\n",
      "C00100  Propanoyl-CoA\n",
      "C00109  2-Oxobutanoate\n",
      "C00417  cis-Aconitate\n",
      "C00490  Itaconate\n",
      "C00531  Itaconyl-CoA\n",
      "C00651  4-Methylene-L-glutamate\n",
      "C00810  (R)-Acetoin\n",
      "C01011  (3S)-Citramalyl-CoA\n",
      "C01109  4-Methylene-L-glutamine\n",
      "C01732  Mesaconate\n",
      "C02226  2-Methylmaleate\n",
      "C02341  trans-Aconitate\n",
      "C02612  (R)-2-Methylmalate\n",
      "C02614  (S)-2-Methylmalate\n",
      "C02630  2-Hydroxyglutarate\n",
      "C02876  Propanoyl phosphate\n",
      "C03618  L-threo-3-Methylaspartate\n",
      "C06010  (S)-2-Acetolactate\n",
      "C06027  L-erythro-3-Methylmalyl-CoA\n",
      "C06028  2-Methylfumaryl-CoA\n",
      "C06029  L-threo-3-Methylmalate\n",
      "C06030  Methyloxaloacetate\n",
      "C06031  D-threo-3-Methylmalate\n",
      "C06032  D-erythro-3-Methylmalate\n",
      "C06033  Parapyruvate\n",
      "C06034  4-Hydroxy-4-methylglutamate\n",
      "C06035  4-Methylene-2-oxoglutarate\n",
      "C06234  4-Methyl-L-glutamate\n",
      "C21632  2-Hydroxyparaconate\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00640  Propanoate metabolism\n",
      "ath00760  Nicotinate and nicotinamide metabolism\n",
      "KO_PATHWAY  ko00660\n",
      "///\n",
      "ENTRY       ath00670                    Pathway\n",
      "NAME        One carbon pool by folate - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00670  One carbon pool by folate\n",
      "MODULE      ath_M00126  Tetrahydrofolate biosynthesis, GTP => THF [PATH:ath00670]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G34570  THY-2; thymidylate synthase 2 [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "AT2G16370  THY-1; thymidylate synthase 1 [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "AT2G21550  Bifunctional dihydrofolate reductase/thymidylate synthase [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "AT1G50480  THFS; 10-formyltetrahydrofolate synthetase [KO:K01938] [EC:6.3.4.3]\n",
      "AT4G13890  EDA36; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G13930  SHM4; serine hydroxymethyltransferase 4 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G22020  SHM6; serine hydroxymethyltransferase 6 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G32520  SHM3; serine hydroxymethyltransferase 3 [KO:K00600] [EC:2.1.2.1]\n",
      "AT4G37930  SHM1; serine transhydroxymethyltransferase 1 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G36370  SHM7; serine hydroxymethyltransferase 7 [KO:K00600] [EC:2.1.2.1]\n",
      "AT5G26780  SHM2; serine hydroxymethyltransferase 2 [KO:K00600] [EC:2.1.2.1]\n",
      "AT1G31220  Formyl transferase [KO:K00601] [EC:2.1.2.2]\n",
      "AT2G35040  AICARFT/IMPCHase bienzyme family protein [KO:K00602] [EC:2.1.2.3 3.5.4.10]\n",
      "AT1G66520  pde194; formyltransferase [KO:K00604] [EC:2.1.2.9]\n",
      "AT1G11860  Glycine cleavage T-protein family [KO:K00605] [EC:2.1.2.10]\n",
      "AT4G17360  Formyl transferase [KO:K01433] [EC:3.5.1.10]\n",
      "AT5G47435  formyltetrahydrofolate deformylase [KO:K01433] [EC:3.5.1.10]\n",
      "AT3G59970  MTHFR1; methylenetetrahydrofolate reductase 1 [KO:K00297] [EC:1.5.1.54]\n",
      "AT2G44160  MTHFR2; methylenetetrahydrofolate reductase 2 [KO:K00297] [EC:1.5.1.54]\n",
      "AT1G76730  COG0212; NagB/RpiA/CoA transferase-like superfamily protein [KO:K01934] [EC:6.3.3.2]\n",
      "AT5G13050  5-FCL; 5-formyltetrahydrofolate cycloligase [KO:K01934] [EC:6.3.3.2]\n",
      "COMPOUND    C00101  Tetrahydrofolate\n",
      "C00143  5,10-Methylenetetrahydrofolate\n",
      "C00234  10-Formyltetrahydrofolate\n",
      "C00415  Dihydrofolate\n",
      "C00440  5-Methyltetrahydrofolate\n",
      "C00445  5,10-Methenyltetrahydrofolate\n",
      "C00504  Folate\n",
      "C00664  5-Formiminotetrahydrofolate\n",
      "C03479  Folinic acid\n",
      "REL_PATHWAY ath00790  Folate biosynthesis\n",
      "KO_PATHWAY  ko00670\n",
      "///\n",
      "ENTRY       ath00710                    Pathway\n",
      "NAME        Carbon fixation in photosynthetic organisms - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Energy metabolism\n",
      "PATHWAY_MAP ath00710  Carbon fixation in photosynthetic organisms\n",
      "MODULE      ath_M00165  Reductive pentose phosphate cycle (Calvin cycle) [PATH:ath00710]\n",
      "ath_M00166  Reductive pentose phosphate cycle, ribulose-5P => glyceraldehyde-3P [PATH:ath00710]\n",
      "ath_M00167  Reductive pentose phosphate cycle, glyceraldehyde-3P => ribulose-5P [PATH:ath00710]\n",
      "ath_M00168  CAM (Crassulacean acid metabolism), dark [PATH:ath00710]\n",
      "ath_M00169  CAM (Crassulacean acid metabolism), light [PATH:ath00710]\n",
      "ath_M00170  C4-dicarboxylic acid cycle, phosphoenolpyruvate carboxykinase type [PATH:ath00710]\n",
      "ath_M00171  C4-dicarboxylic acid cycle, NAD - malic enzyme type [PATH:ath00710]\n",
      "ath_M00172  C4-dicarboxylic acid cycle, NADP - malic enzyme type [PATH:ath00710]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        ArthCp030  rbcL; ribulose-1,5-bisphosphate carboxylase/oxygenase large subunit [KO:K01601] [EC:4.1.1.39]\n",
      "AT1G67090  RBCS1A; ribulose bisphosphate carboxylase small chain 1A [KO:K01602] [EC:4.1.1.39]\n",
      "AT5G38420  RBCS2B; Ribulose bisphosphate carboxylase (small chain) family protein [KO:K01602] [EC:4.1.1.39]\n",
      "AT5G38410  RBCS3B; Ribulose bisphosphate carboxylase (small chain) family protein [KO:K01602] [EC:4.1.1.39]\n",
      "AT5G38430  RBCS1B; Ribulose bisphosphate carboxylase (small chain) family protein [KO:K01602] [EC:4.1.1.39]\n",
      "AT3G12780  PGK1; phosphoglycerate kinase 1 [KO:K00927] [EC:2.7.2.3]\n",
      "AT1G56190  Phosphoglycerate kinase family protein [KO:K00927] [EC:2.7.2.3]\n",
      "AT1G79550  PGK; phosphoglycerate kinase [KO:K00927] [EC:2.7.2.3]\n",
      "AT1G13440  GAPC2; glyceraldehyde-3-phosphate dehydrogenase C2 [KO:K00134] [EC:1.2.1.12]\n",
      "AT1G16300  GAPCP-2; glyceraldehyde-3-phosphate dehydrogenase of plastid 2 [KO:K00134] [EC:1.2.1.12]\n",
      "AT1G79530  GAPCP-1; glyceraldehyde-3-phosphate dehydrogenase of plastid 1 [KO:K00134] [EC:1.2.1.12]\n",
      "AT3G04120  GAPC1; glyceraldehyde-3-phosphate dehydrogenase C subunit 1 [KO:K00134] [EC:1.2.1.12]\n",
      "AT3G26650  GAPA; glyceraldehyde 3-phosphate dehydrogenase A subunit [KO:K05298] [EC:1.2.1.13]\n",
      "AT1G42970  GAPB; glyceraldehyde-3-phosphate dehydrogenase B subunit [KO:K05298] [EC:1.2.1.13]\n",
      "AT1G12900  GAPA-2; glyceraldehyde 3-phosphate dehydrogenase A subunit 2 [KO:K05298] [EC:1.2.1.13]\n",
      "AT4G26520  FBA7; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G36460  FBA6; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT3G52930  FBA8; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G01140  PDE345; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT4G38970  FBA2; fructose-bisphosphate aldolase 2 [KO:K01623] [EC:4.1.2.13]\n",
      "AT4G26530  FBA5; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT5G03690  FBA4; Aldolase superfamily protein [KO:K01623] [EC:4.1.2.13]\n",
      "AT2G21330  FBA1; fructose-bisphosphate aldolase 1 [KO:K01623] [EC:4.1.2.13]\n",
      "AT1G43670  FBP; Inositol monophosphatase family protein [KO:K03841] [EC:3.1.3.11]\n",
      "AT3G54050  HCEF1; high cyclic electron flow 1 [KO:K03841] [EC:3.1.3.11]\n",
      "AT5G64380  Inositol monophosphatase family protein [KO:K03841] [EC:3.1.3.11]\n",
      "AT3G60750  Transketolase [KO:K00615] [EC:2.2.1.1]\n",
      "AT2G45290  Transketolase [KO:K00615] [EC:2.2.1.1]\n",
      "AT3G55800  SBPASE; sedoheptulose-bisphosphatase [KO:K01100] [EC:3.1.3.37]\n",
      "AT1G71100  RSW10; Ribose 5-phosphate isomerase, type A protein [KO:K01807] [EC:5.3.1.6]\n",
      "AT2G01290  RPI2; ribose-5-phosphate isomerase 2 [KO:K01807] [EC:5.3.1.6]\n",
      "AT3G04790  EMB3119; Ribose 5-phosphate isomerase, type A protein [KO:K01807] [EC:5.3.1.6]\n",
      "AT5G44520  NagB/RpiA/CoA transferase-like superfamily protein [KO:K01807] [EC:5.3.1.6]\n",
      "AT1G32060  PRK; phosphoribulokinase [KO:K00855] [EC:2.7.1.19]\n",
      "AT3G55440  TPI; triosephosphate isomerase [KO:K01803] [EC:5.3.1.1]\n",
      "AT2G21170  TIM; triosephosphate isomerase [KO:K01803] [EC:5.3.1.1]\n",
      "AT1G63290  Aldolase-type TIM barrel family protein [KO:K01783] [EC:5.1.3.1]\n",
      "AT5G61410  RPE; D-ribulose-5-phosphate-3-epimerase [KO:K01783] [EC:5.1.3.1]\n",
      "AT3G01850  Aldolase-type TIM barrel family protein [KO:K01783] [EC:5.1.3.1]\n",
      "AT1G53310  PPC1; phosphoenolpyruvate carboxylase 1 [KO:K01595] [EC:4.1.1.31]\n",
      "AT3G14940  PPC3; phosphoenolpyruvate carboxylase 3 [KO:K01595] [EC:4.1.1.31]\n",
      "AT2G42600  PPC2; phosphoenolpyruvate carboxylase 2 [KO:K01595] [EC:4.1.1.31]\n",
      "AT1G68750  PPC4; phosphoenolpyruvate carboxylase 4 [KO:K01595] [EC:4.1.1.31]\n",
      "AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G65690  PCK2; phosphoenolpyruvate carboxykinase 2 [KO:K01610] [EC:4.1.1.49]\n",
      "AT4G37870  PCK1; phosphoenolpyruvate carboxykinase 1 [KO:K01610] [EC:4.1.1.49]\n",
      "AT1G17290  AlaAT1; alanine aminotransferas [KO:K00814] [EC:2.6.1.2]\n",
      "AT1G72330  ALAAT2; alanine aminotransferase 2 [KO:K00814] [EC:2.6.1.2]\n",
      "AT1G70580  AOAT2; alanine-2-oxoglutarate aminotransferase 2 [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT1G23310  GGT1; glutamate:glyoxylate aminotransferase [KO:K14272] [EC:2.6.1.4 2.6.1.2 2.6.1.44]\n",
      "AT4G15530  PPDK; pyruvate orthophosphate dikinase [KO:K01006] [EC:2.7.9.1]\n",
      "AT5G25880  NADP-ME3; NADP-malic enzyme 3 [KO:K00029] [EC:1.1.1.40]\n",
      "AT5G11670  NADP-ME2; NADP-malic enzyme 2 [KO:K00029] [EC:1.1.1.40]\n",
      "AT2G19900  NADP-ME1; NADP-malic enzyme 1 [KO:K00029] [EC:1.1.1.40]\n",
      "AT1G79750  NADP-ME4; NADP-malic enzyme 4 [KO:K00029] [EC:1.1.1.40]\n",
      "AT5G58330  lactate/malate dehydrogenase family protein [KO:K00051] [EC:1.1.1.82]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT1G04410  c-NAD-MDH1; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G56720  c-NAD-MDH3; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT5G43330  c-NAD-MDH2; Lactate/malate dehydrogenase family protein [KO:K00025] [EC:1.1.1.37]\n",
      "AT3G47520  MDH; malate dehydrogenase [KO:K00026] [EC:1.1.1.37]\n",
      "AT1G53240  mMDH1; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT2G22780  PMDH1; peroxisomal NAD-malate dehydrogenase 1 [KO:K00026] [EC:1.1.1.37]\n",
      "AT3G15020  mMDH2; Lactate/malate dehydrogenase family protein [KO:K00026] [EC:1.1.1.37]\n",
      "AT5G09660  PMDH2; peroxisomal NAD-malate dehydrogenase 2 [KO:K00026] [EC:1.1.1.37]\n",
      "AT4G00570  NAD-ME2; NAD-dependent malic enzyme 2 [KO:K00028] [EC:1.1.1.39]\n",
      "AT2G13560  NAD-ME1; NAD-dependent malic enzyme 1 [KO:K00028] [EC:1.1.1.39]\n",
      "COMPOUND    C00001  H2O\n",
      "C00011  CO2\n",
      "C00022  Pyruvate\n",
      "C00036  Oxaloacetate\n",
      "C00041  L-Alanine\n",
      "C00049  L-Aspartate\n",
      "C00074  Phosphoenolpyruvate\n",
      "C00085  D-Fructose 6-phosphate\n",
      "C00111  Glycerone phosphate\n",
      "C00117  D-Ribose 5-phosphate\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00149  (S)-Malate\n",
      "C00197  3-Phospho-D-glycerate\n",
      "C00199  D-Ribulose 5-phosphate\n",
      "C00231  D-Xylulose 5-phosphate\n",
      "C00236  3-Phospho-D-glyceroyl phosphate\n",
      "C00279  D-Erythrose 4-phosphate\n",
      "C00354  D-Fructose 1,6-bisphosphate\n",
      "C00369  Starch\n",
      "C00447  Sedoheptulose 1,7-bisphosphate\n",
      "C01182  D-Ribulose 1,5-bisphosphate\n",
      "C02076  Sedoheptulose\n",
      "C05382  Sedoheptulose 7-phosphate\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "KO_PATHWAY  ko00710\n",
      "///\n",
      "ENTRY       ath00730                    Pathway\n",
      "NAME        Thiamine metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00730  Thiamine metabolism\n",
      "MODULE      ath_M00897  Thiamine biosynthesis, plants, AIR (+ NAD+) => TMP/thiamine/TPP [PATH:ath00730]\n",
      "ath_M00899  Thiamine salvage pathway, HMP/HET => TMP [PATH:ath00730]\n",
      "DBLINKS     GO: 0006772\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G29630  THIC; thiaminC [KO:K03147] [EC:4.1.99.17]\n",
      "AT1G22940  TH1; thiamin biosynthesis protein [KO:K14153] [EC:2.7.1.49 2.7.4.7 2.5.1.3]\n",
      "AT5G65720  NFS1; nitrogen fixation S (NIFS)-like 1 [KO:K04487] [EC:2.8.1.7]\n",
      "AT3G21500  DXPS1; 1-deoxy-D-xylulose 5-phosphate synthase 1 [KO:K01662] [EC:2.2.1.7]\n",
      "AT4G15560  CLA1; Deoxyxylulose-5-phosphate synthase [KO:K01662] [EC:2.2.1.7]\n",
      "AT5G11380  DXPS3; 1-deoxy-D-xylulose 5-phosphate synthase 3 [KO:K01662] [EC:2.2.1.7]\n",
      "AT5G54770  THI1; thiazole biosynthetic enzyme, chloroplast (ARA6) (THI1) (THI4) [KO:K03146] [EC:2.4.2.60]\n",
      "AT3G24030  hydroxyethylthiazole kinase family protein [KO:K00878] [EC:2.7.1.50]\n",
      "AT5G66005  Expressed protein [KO:K06928] [EC:3.6.1.15]\n",
      "AT1G67440  emb1688; Minichromosome maintenance (MCM2/3/5) family protein [KO:K06949] [EC:3.6.1.- 3.1.3.100]\n",
      "AT5G32470  heme oxygenase-like, multi-helical [KO:K22911] [EC:3.1.3.100 3.5.99.-]\n",
      "AT2G44750  TPK2; thiamin pyrophosphokinase 2 [KO:K00949] [EC:2.7.6.2]\n",
      "AT1G02880  TPK1; thiamin pyrophosphokinase1 [KO:K00949] [EC:2.7.6.2]\n",
      "AT5G47840  AMK2; adenosine monophosphate kinase [KO:K00939] [EC:2.7.4.3]\n",
      "AT5G50370  Adenylate kinase family protein [KO:K00939] [EC:2.7.4.3]\n",
      "AT5G63400  ADK1; adenylate kinase 1 [KO:K00939] [EC:2.7.4.3]\n",
      "AT5G35170  adenylate kinase family protein [KO:K00939] [EC:2.7.4.3]\n",
      "AT2G39270  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K00939] [EC:2.7.4.3]\n",
      "AT2G37250  ADK; adenosine kinase [KO:K00939] [EC:2.7.4.3]\n",
      "AT3G01820  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K00939] [EC:2.7.4.3]\n",
      "AT3G16990  heme oxygenase-like, multi-helical [KO:K20896] [EC:3.5.1.- 3.5.99.-]\n",
      "COMPOUND    C00003  NAD+\n",
      "C00018  Pyridoxal phosphate\n",
      "C00022  Pyruvate\n",
      "C00037  Glycine\n",
      "C00068  Thiamin diphosphate\n",
      "C00082  L-Tyrosine\n",
      "C00097  L-Cysteine\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00378  Thiamine\n",
      "C01081  Thiamin monophosphate\n",
      "C01279  4-Amino-5-hydroxymethyl-2-methylpyrimidine\n",
      "C02892  Thiamine acetic acid\n",
      "C03028  Thiamin triphosphate\n",
      "C03373  Aminoimidazole ribotide\n",
      "C04294  5-(2-Hydroxyethyl)-4-methylthiazole\n",
      "C04327  4-Methyl-5-(2-phosphooxyethyl)thiazole\n",
      "C04556  4-Amino-2-methyl-5-(phosphooxymethyl)pyrimidine\n",
      "C04752  4-Amino-5-hydroxymethyl-2-methylpyrimidine diphosphate\n",
      "C05856  Thiamine aldehyde\n",
      "C11437  1-Deoxy-D-xylulose 5-phosphate\n",
      "C15809  Iminoglycine\n",
      "C15810  Sulfur-carrier protein\n",
      "C15811  [Enzyme]-cysteine\n",
      "C15812  [Enzyme]-S-sulfanylcysteine\n",
      "C15813  Adenylyl-[sulfur-carrier protein]\n",
      "C15814  Thiocarboxy-[sulfur-carrier protein]\n",
      "C19872  N-Formyl-4-amino-5-aminomethyl-2-methylpyrimidine\n",
      "C20246  2-[(2R,5Z)-2-Carboxy-4-methylthiazol-5(2H)-ylidene]ethyl phosphate\n",
      "C20247  2-(2-Carboxy-4-methylthiazol-5-yl)ethyl phosphate\n",
      "C20267  4-Amino-5-aminomethyl-2-methylpyrimidine\n",
      "C20784  ADP-5-ethyl-4-methylthiazole-2-carboxylate\n",
      "REFERENCE   PMID:15614489\n",
      "AUTHORS   Kawasaki Y, Onozuka M, Mizote T, Nosaka K.\n",
      "TITLE     Biosynthesis of hydroxymethylpyrimidine pyrophosphate in Saccharomyces cerevisiae.\n",
      "JOURNAL   Curr Genet 47:156-62 (2005)\n",
      "DOI:10.1007/s00294-004-0557-x\n",
      "REFERENCE   PMID:14567704\n",
      "AUTHORS   Park JH, Dorrestein PC, Zhai H, Kinsland C, McLafferty FW, Begley TP.\n",
      "TITLE     Biosynthesis of the thiazole moiety of thiamin pyrophosphate (vitamin B1).\n",
      "JOURNAL   Biochemistry 42:12430-8 (2003)\n",
      "DOI:10.1021/bi034902z\n",
      "REFERENCE   PMID:22616866\n",
      "AUTHORS   Begley TP, Ealick SE, McLafferty FW\n",
      "TITLE     Thiamin biosynthesis: still yielding fascinating biological chemistry.\n",
      "JOURNAL   Biochem Soc Trans 40:555-60 (2012)\n",
      "DOI:10.1042/BST20120084\n",
      "REFERENCE   PMID:12777485\n",
      "AUTHORS   Wightman R, Meacock PA\n",
      "TITLE     The THI5 gene family of Saccharomyces cerevisiae: distribution of homologues among the hemiascomycetes and functional redundancy in the aerobic biosynthesis of thiamin from pyridoxine.\n",
      "JOURNAL   Microbiology 149:1447-60 (2003)\n",
      "DOI:10.1099/mic.0.26194-0\n",
      "REFERENCE   PMID:22031445\n",
      "AUTHORS   Chatterjee A, Abeydeera ND, Bale S, Pai PJ, Dorrestein PC, Russell DH, Ealick SE, Begley TP\n",
      "TITLE     Saccharomyces cerevisiae THI4p is a suicide thiamine thiazole synthase.\n",
      "JOURNAL   Nature 478:542-6 (2011)\n",
      "DOI:10.1038/nature10503\n",
      "REFERENCE   PMID:19348578\n",
      "AUTHORS   Jurgenson CT, Begley TP, Ealick SE\n",
      "TITLE     The structural and biochemical foundations of thiamin biosynthesis.\n",
      "JOURNAL   Annu Rev Biochem 78:569-603 (2009)\n",
      "DOI:10.1146/annurev.biochem.78.072407.102340\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00230  Purine metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00750  Vitamin B6 metabolism\n",
      "KO_PATHWAY  ko00730\n",
      "///\n",
      "ENTRY       ath00740                    Pathway\n",
      "NAME        Riboflavin metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00740  Riboflavin metabolism\n",
      "MODULE      ath_M00125  Riboflavin biosynthesis, plants and bacteria, GTP => riboflavin/FMN/FAD [PATH:ath00740]\n",
      "DBLINKS     GO: 0006771\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G59750  RIBA3; monofunctional riboflavin biosynthesis protein RIBA 3 [KO:K14652] [EC:4.1.99.12 3.5.4.25]\n",
      "AT2G22450  RIBA2; riboflavin biosynthesis protein [KO:K14652] [EC:4.1.99.12 3.5.4.25]\n",
      "AT5G64300  GCH; GTP cyclohydrolase II [KO:K14652] [EC:4.1.99.12 3.5.4.25]\n",
      "AT4G20960  PyrD; Cytidine/deoxycytidylate deaminase family protein [KO:K11752] [EC:3.5.4.26 1.1.1.193]\n",
      "AT3G47390  PHS1; cytidine/deoxycytidylate deaminase family protein [KO:K11752] [EC:3.5.4.26 1.1.1.193]\n",
      "AT4G11570  Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K22912] [EC:3.1.3.104]\n",
      "AT1G79790  FHY1; Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K20860] [EC:3.1.3.102 3.1.3.104]\n",
      "AT2G44050  COS1; 6,7-dimethyl-8-ribityllumazine synthase / DMRL synthase / lumazine synthase / riboflavin synthase [KO:K00794] [EC:2.5.1.78]\n",
      "AT2G20690  lumazine-binding family protein [KO:K00793] [EC:2.5.1.9]\n",
      "AT4G21470  FMN/FHY; riboflavin kinase/FMN hydrolase [KO:K20884] [EC:2.7.1.26 3.1.3.102]\n",
      "AT3G17790  PAP17; purple acid phosphatase 17 [KO:K14379] [EC:3.1.3.2]\n",
      "AT2G01890  PAP8; purple acid phosphatase 8 [KO:K14379] [EC:3.1.3.2]\n",
      "AT1G14700  PAP3; purple acid phosphatase 3 [KO:K14379] [EC:3.1.3.2]\n",
      "AT1G25230  Calcineurin-like metallo-phosphoesterase superfamily protein [KO:K14379] [EC:3.1.3.2]\n",
      "AT2G01880  PAP7; purple acid phosphatase 7 [KO:K14379] [EC:3.1.3.2]\n",
      "AT5G08340  Nucleotidylyl transferase superfamily protein [KO:K22949] [EC:2.7.7.2]\n",
      "AT5G23330  Nucleotidylyl transferase superfamily protein [KO:K22949] [EC:2.7.7.2]\n",
      "AT4G29680  Alkaline-phosphatase-like family protein [KO:K01513] [EC:3.1.4.1 3.6.1.9]\n",
      "AT2G42070  NUDX23; nudix hydrolase homolog 23 [KO:K18453] [EC:3.6.1.13 3.6.1.18]\n",
      "COMPOUND    C00016  FAD\n",
      "C00044  GTP\n",
      "C00061  FMN\n",
      "C00199  D-Ribulose 5-phosphate\n",
      "C00255  Riboflavin\n",
      "C00474  Ribitol\n",
      "C01007  Reduced riboflavin\n",
      "C01268  5-Amino-6-(5'-phosphoribosylamino)uracil\n",
      "C01304  2,5-Diamino-6-(5-phospho-D-ribosylamino)pyrimidin-4(3H)-one\n",
      "C01352  FADH2\n",
      "C01727  Lumichrome\n",
      "C01847  Reduced FMN\n",
      "C03114  Dimethylbenzimidazole\n",
      "C04332  6,7-Dimethyl-8-(D-ribityl)lumazine\n",
      "C04454  5-Amino-6-(5'-phospho-D-ribitylamino)uracil\n",
      "C04732  5-Amino-6-(1-D-ribitylamino)uracil\n",
      "C05995  7-Hydroxy-6-methyl-8-ribityllumazine\n",
      "C15556  L-3,4-Dihydroxybutan-2-one 4-phosphate\n",
      "C15563  2-Amino-5-formylamino-6-(5-phospho-D-ribosylamino)pyrimidin-4(3H)-one\n",
      "C18910  2,5-Diamino-6-(5-phospho-D-ribitylamino)pyrimidin-4(3H)-one\n",
      "REFERENCE   PMID:12136096\n",
      "AUTHORS   Vitreschak AG, Rodionov DA, Mironov AA, Gelfand MS.\n",
      "TITLE     Regulation of riboflavin biosynthesis and transport genes in bacteria by transcriptional and translational attenuation.\n",
      "JOURNAL   Nucleic Acids Res 30:3141-51 (2002)\n",
      "DOI:10.1093/nar/30.14.3141\n",
      "REFERENCE   PMID:11856310\n",
      "AUTHORS   Fischer M, Haase I, Feicht R, Richter G, Gerhardt S, Changeux JP, Huber R, Bacher A.\n",
      "TITLE     Biosynthesis of riboflavin: 6,7-dimethyl-8-ribityllumazine synthase of Schizosaccharomyces pombe.\n",
      "JOURNAL   Eur J Biochem 269:519-26 (2002)\n",
      "DOI:10.1046/j.0014-2956.2001.02674.x\n",
      "REFERENCE   PMID:12867427\n",
      "AUTHORS   Haase I, Fischer M, Bacher A, Schramek N.\n",
      "TITLE     Temperature-dependent presteady state kinetics of lumazine synthase from the hyperthermophilic eubacterium Aquifex aeolicus.\n",
      "JOURNAL   J Biol Chem 278:37909-15 (2003)\n",
      "DOI:10.1074/jbc.M303090200\n",
      "REFERENCE   PMID:16042598\n",
      "AUTHORS   Fischer M, Romisch W, Illarionov B, Eisenreich W, Bacher A.\n",
      "TITLE     Structures and reaction mechanisms of riboflavin synthases of eubacterial and archaeal origin.\n",
      "JOURNAL   Biochem Soc Trans 33:780-4 (2005)\n",
      "DOI:10.1042/BST0330780\n",
      "REL_PATHWAY ath00030  Pentose phosphate pathway\n",
      "ath00040  Pentose and glucuronate interconversions\n",
      "ath00230  Purine metabolism\n",
      "ath00860  Porphyrin and chlorophyll metabolism\n",
      "KO_PATHWAY  ko00740\n",
      "///\n",
      "ENTRY       ath00750                    Pathway\n",
      "NAME        Vitamin B6 metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00750  Vitamin B6 metabolism\n",
      "MODULE      ath_M00916  Pyridoxal-P biosynthesis, R5P + glyceraldehyde-3P + glutamine => pyridoxal-P [PATH:ath00750]\n",
      "DBLINKS     GO: 0042816\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G46580  Pyridoxamine 5'-phosphate oxidase family protein [KO:K00275] [EC:1.4.3.5]\n",
      "AT5G49970  PPOX; pyridoxin (pyrodoxamine) 5'-phosphate oxidase [KO:K23998] [EC:1.4.3.5 5.1.99.6]\n",
      "AT5G37850  SOS4; pfkB-like carbohydrate kinase family protein [KO:K00868] [EC:2.7.1.35]\n",
      "AT1G73010  PS2; inorganic pyrophosphatase 1 [KO:K13248] [EC:3.1.3.74]\n",
      "AT1G17710  PEPC1; Pyridoxal phosphate phosphatase-related protein [KO:K13248] [EC:3.1.3.74]\n",
      "AT4G29530  Pyridoxal phosphate phosphatase-related protein [KO:K13248] [EC:3.1.3.74]\n",
      "AT5G53580  PLR1; NAD(P)-linked oxidoreductase superfamily protein [KO:K05275] [EC:1.1.1.65]\n",
      "AT2G17630  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K00831] [EC:2.6.1.52]\n",
      "AT4G35630  PSAT; phosphoserine aminotransferase [KO:K00831] [EC:2.6.1.52]\n",
      "AT1G72810  Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K01733] [EC:4.2.3.1]\n",
      "AT4G29840  MTO2; Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K01733] [EC:4.2.3.1]\n",
      "AT2G38230  PDX1.1; pyridoxine biosynthesis 1.1 [KO:K06215] [EC:4.3.3.6]\n",
      "AT5G01410  RSR4; Aldolase-type TIM barrel family protein [KO:K06215] [EC:4.3.3.6]\n",
      "AT3G16050  PDX1.2; pyridoxine biosynthesis 1.2 [KO:K06215] [EC:4.3.3.6]\n",
      "AT5G60540  PDX2; pyridoxine biosynthesis 2 [KO:K08681] [EC:4.3.3.6]\n",
      "COMPOUND    C00018  Pyridoxal phosphate\n",
      "C00064  L-Glutamine\n",
      "C00117  D-Ribose 5-phosphate\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00232  Succinate semialdehyde\n",
      "C00250  Pyridoxal\n",
      "C00266  Glycolaldehyde\n",
      "C00279  D-Erythrose 4-phosphate\n",
      "C00314  Pyridoxine\n",
      "C00534  Pyridoxamine\n",
      "C00627  Pyridoxine phosphate\n",
      "C00647  Pyridoxamine phosphate\n",
      "C00847  4-Pyridoxate\n",
      "C00971  4-Pyridoxolactone\n",
      "C01215  2-(Acetamidomethylene)succinate\n",
      "C01270  3-Hydroxy-2-methylpyridine-5-carboxylate\n",
      "C03393  4-Phospho-D-erythronate\n",
      "C04106  2-(Hydroxymethyl)-4-oxobutanoate\n",
      "C04604  3-Hydroxy-2-methylpyridine-4,5-dicarboxylate\n",
      "C04690  2-(Hydroxymethyl)-3-(acetamidomethylene)succinate\n",
      "C04773  3-Hydroxy-4-hydroxymethyl-2-methylpyridine-5-carboxylate\n",
      "C06050  2-Methyl-3-hydroxy-5-formylpyridine-4-carboxylate\n",
      "C06051  Isopyridoxal\n",
      "C06052  5-Pyridoxolactone\n",
      "C06054  2-Oxo-3-hydroxy-4-phosphobutanoate\n",
      "C06055  O-Phospho-4-hydroxy-L-threonine\n",
      "C06056  4-Hydroxy-L-threonine\n",
      "C07335  2-Amino-3-oxo-4-phosphonooxybutyrate\n",
      "C11638  3-Amino-2-oxopropyl phosphate\n",
      "REFERENCE   PMID:10753941\n",
      "AUTHORS   Tazoe M, Ichikawa K, Hoshino T.\n",
      "TITLE     Biosynthesis of vitamin B(6) in rhizobium.\n",
      "JOURNAL   J Biol Chem 275:11300-5 (2000)\n",
      "DOI:10.1074/jbc.275.15.11300\n",
      "REFERENCE   PMID:9696782\n",
      "AUTHORS   Yang Y, Zhao G, Man TK, Winkler ME.\n",
      "TITLE     Involvement of the gapA- and epd (gapB)-encoded dehydrogenases in pyridoxal 5'-phosphate coenzyme biosynthesis in Escherichia coli K-12.\n",
      "JOURNAL   J Bacteriol 180:4294-9 (1998)\n",
      "DOI:10.1128/JB.180.16.4294-4299.1998\n",
      "REFERENCE   PMID:12269807\n",
      "AUTHORS   Yeh JI, Du S, Pohl E, Cane DE.\n",
      "TITLE     Multistate binding in pyridoxine 5'-phosphate synthase: 1.96 A crystal structure in complex with 1-deoxy-D-xylulose phosphate.\n",
      "JOURNAL   Biochemistry 41:11649-57 (2002)\n",
      "DOI:10.1021/bi026292t\n",
      "REFERENCE   PMID:10225425\n",
      "AUTHORS   Laber B, Maurer W, Scharf S, Stepusin K, Schmidt FS.\n",
      "TITLE     Vitamin B6 biosynthesis: formation of pyridoxine 5'-phosphate from 4-(phosphohydroxy)-L-threonine and 1-deoxy-D-xylulose-5-phosphate by PdxA and PdxJ protein.\n",
      "JOURNAL   FEBS Lett 449:45-8 (1999)\n",
      "DOI:10.1016/S0014-5793(99)00393-2\n",
      "REFERENCE   PMID:16157873\n",
      "AUTHORS   Tambasco-Studart M, Titiz O, Raschle T, Forster G, Amrhein N, Fitzpatrick TB.\n",
      "TITLE     Vitamin B6 biosynthesis in higher plants.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 102:13687-92 (2005)\n",
      "DOI:10.1073/pnas.0506228102\n",
      "REFERENCE   PMID:22276911\n",
      "AUTHORS   Nagase T, Mugo AN, Chu HN, Yoshikane Y, Ohnishi K, Yagi T\n",
      "TITLE     The mll6786 gene encodes a repressor protein controlling the degradation pathway for vitamin B6 in Mesorhizobium loti.\n",
      "JOURNAL   FEMS Microbiol Lett 329:116-22 (2012)\n",
      "DOI:10.1111/j.1574-6968.2012.02510.x\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00650  Butanoate metabolism\n",
      "ath00730  Thiamine metabolism\n",
      "KO_PATHWAY  ko00750\n",
      "///\n",
      "ENTRY       ath00760                    Pathway\n",
      "NAME        Nicotinate and nicotinamide metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00760  Nicotinate and nicotinamide metabolism\n",
      "MODULE      ath_M00115  NAD biosynthesis, aspartate => quinolinate => NAD [PATH:ath00760]\n",
      "DBLINKS     UMBBD: nic\n",
      "GO: 0006769\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G14760  AO; L-aspartate oxidase [KO:K00278] [EC:1.4.3.16]\n",
      "AT5G50210  QS; quinolinate synthase [KO:K03517] [EC:2.5.1.72]\n",
      "AT2G01350  QPT; quinolinate phoshoribosyltransferase [KO:K00767] [EC:2.4.2.19]\n",
      "AT2G23420  NAPRT2; nicotinate phosphoribosyltransferase 2 [KO:K00763] [EC:6.3.4.21]\n",
      "AT4G36940  NAPRT1; nicotinate phosphoribosyltransferase 1 [KO:K00763] [EC:6.3.4.21]\n",
      "AT1G72880  Survival protein SurE-like phosphatase/nucleotidase [KO:K03787] [EC:3.1.3.5]\n",
      "AT4G14930  Survival protein SurE-like phosphatase/nucleotidase [KO:K03787] [EC:3.1.3.5]\n",
      "AT3G62040  Haloacid dehalogenase-like hydrolase (HAD) superfamily protein [KO:K18551] [EC:3.1.3.-]\n",
      "AT2G36310  URH1; uridine-ribohydrolase 1 [KO:K01240] [EC:3.2.2.3]\n",
      "AT5G55810  NMNAT; nicotinate/nicotinamide mononucleotide adenyltransferase [KO:K06210] [EC:2.7.7.1 2.7.7.18]\n",
      "AT4G29680  Alkaline-phosphatase-like family protein [KO:K01513] [EC:3.1.4.1 3.6.1.9]\n",
      "AT5G20070  NUDX19; nudix hydrolase homolog 19 [KO:K03426] [EC:3.6.1.22]\n",
      "AT1G55090  carbon-nitrogen hydrolase family protein [KO:K01950] [EC:6.3.5.1]\n",
      "AT5G09230  SRT2; sirtuin 2 [KO:K11414] [EC:2.3.1.286]\n",
      "AT5G55760  SRT1; sirtuin 1 [KO:K11416] [EC:2.3.1.286]\n",
      "AT3G21070  NADK1; NAD kinase 1 [KO:K00858] [EC:2.7.1.23]\n",
      "AT1G21640  NADK2; NAD kinase 2 [KO:K00858] [EC:2.7.1.23]\n",
      "AT1G78590  NADK3; NAD(H) kinase 3 [KO:K00858] [EC:2.7.1.23]\n",
      "COMPOUND    C00003  NAD+\n",
      "C00006  NADP+\n",
      "C00022  Pyruvate\n",
      "C00042  Succinate\n",
      "C00049  L-Aspartate\n",
      "C00111  Glycerone phosphate\n",
      "C00122  Fumarate\n",
      "C00153  Nicotinamide\n",
      "C00163  Propanoate\n",
      "C00232  Succinate semialdehyde\n",
      "C00253  Nicotinate\n",
      "C00334  4-Aminobutanoate\n",
      "C00455  Nicotinamide D-ribonucleotide\n",
      "C00857  Deamino-NAD+\n",
      "C00922  2,3-Dimethylmaleate\n",
      "C01004  N-Methylnicotinate\n",
      "C01020  6-Hydroxynicotinate\n",
      "C01056  (S)-6-Hydroxynicotine\n",
      "C01059  2,5-Dihydroxypyridine\n",
      "C01185  Nicotinate D-ribonucleotide\n",
      "C01297  6-Hydroxypseudooxynicotine\n",
      "C01384  Maleic acid\n",
      "C01596  Maleamate\n",
      "C02295  Methylitaconate\n",
      "C02918  1-Methylnicotinamide\n",
      "C02930  2-Methyleneglutarate\n",
      "C03043  (R)-6-Hydroxynicotine\n",
      "C03056  2,6-Dihydroxypyridine\n",
      "C03150  Nicotinamide-beta-riboside\n",
      "C03458  2,3,6-Trihydroxypyridine\n",
      "C03652  (2R,3S)-2,3-Dimethylmalate\n",
      "C03722  Quinolinate\n",
      "C04226  6-Oxo-1,4,5,6-tetrahydronicotinate\n",
      "C05380  Nicotinurate\n",
      "C05840  Iminoaspartate\n",
      "C05841  Nicotinate D-ribonucleoside\n",
      "C05842  N1-Methyl-2-pyridone-5-carboxamide\n",
      "C05843  N1-Methyl-4-pyridone-5-carboxamide\n",
      "C06178  1-Methylpyrrolinium\n",
      "C15523  2,6-Dihydroxynicotinate\n",
      "C15986  2,6-Dihydroxypseudooxynicotine\n",
      "C15987  4-Methylaminobutyrate\n",
      "C16150  (R,S)-Nicotine\n",
      "C16151  2,6-Dihydroxy-N-methylmyosmine\n",
      "C16152  Blue pigment\n",
      "C16159  2-Formylglutarate\n",
      "C16390  (S)-2-(Hydroxymethyl)glutarate\n",
      "C18232  N-Formylmaleamic acid\n",
      "C19567  4-Oxo-1-(3-pyridyl)-1-butanone\n",
      "C19569  3-Succinoylpyridine\n",
      "C19631  6-Hydroxy-3-succinoylpyridine\n",
      "C20361  Pseudooxynicotine\n",
      "C20381  3,3'-Bipyridine-2,2',5,5',6,6'-hexol\n",
      "C21075  5-(N-Methyl-4,5-dihydro-1H-pyrrol-2-yl)pyridin-2-ol\n",
      "C21154  N-Methylmyosmine\n",
      "REFERENCE   PMID:13475371\n",
      "AUTHORS   BEHRMAN EJ, STANIER RY.\n",
      "TITLE     The bacterial oxidation of nicotinic acid.\n",
      "JOURNAL   J Biol Chem 228:923-45 (1957)\n",
      "REFERENCE   PMID:16894175\n",
      "AUTHORS   Alhapel A, Darley DJ, Wagener N, Eckel E, Elsner N, Pierik AJ.\n",
      "TITLE     Molecular and functional analysis of nicotinate catabolism in Eubacterium barkeri.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 103:12341-6 (2006)\n",
      "DOI:10.1073/pnas.0601635103\n",
      "REFERENCE   PMID:11514508\n",
      "AUTHORS   Baitsch D, Sandu C, Brandsch R, Igloi GL.\n",
      "TITLE     Gene cluster on pAO1 of Arthrobacter nicotinovorans involved in degradation of the plant alkaloid nicotine: cloning, purification, and characterization of 2,6-dihydroxypyridine 3-hydroxylase.\n",
      "JOURNAL   J Bacteriol 183:5262-7 (2001)\n",
      "DOI:10.1128/JB.183.18.5262-5267.2001\n",
      "REFERENCE   PMID:16321959\n",
      "AUTHORS   Sachelaru P, Schiltz E, Igloi GL, Brandsch R.\n",
      "TITLE     An alpha/beta-fold C--C bond hydrolase is involved in a central step of nicotine catabolism by Arthrobacter nicotinovorans.\n",
      "JOURNAL   J Bacteriol 187:8516-9 (2005)\n",
      "DOI:10.1128/JB.187.24.8516-8519.2005\n",
      "REFERENCE   PMID:5835946\n",
      "AUTHORS   Gherna RL, Richardson SH, Rittenberg SC\n",
      "TITLE     The bacterial oxidation of nicotine. VI. The metabolism of 2,6-dihydroxypseudooxynicotine.\n",
      "JOURNAL   J Biol Chem 240:3669-74 (1965)\n",
      "REFERENCE   PMID:21953451\n",
      "AUTHORS   Galeazzi L, Bocci P, Amici A, Brunetti L, Ruggieri S, Romine M, Reed S, Osterman AL, Rodionov DA, Sorci L, Raffaelli N\n",
      "TITLE     Identification of nicotinamide mononucleotide deamidase of the bacterial pyridine nucleotide cycle reveals a novel broadly conserved amidohydrolase family.\n",
      "JOURNAL   J Biol Chem 286:40365-75 (2011)\n",
      "DOI:10.1074/jbc.M111.275818\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00380  Tryptophan metabolism\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00660  C5-Branched dibasic acid metabolism\n",
      "ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "KO_PATHWAY  ko00760\n",
      "///\n",
      "ENTRY       ath00770                    Pathway\n",
      "NAME        Pantothenate and CoA biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00770  Pantothenate and CoA biosynthesis\n",
      "MODULE      ath_M00019  Valine/isoleucine biosynthesis, pyruvate => valine / 2-oxobutanoate => isoleucine [PATH:ath00770]\n",
      "ath_M00120  Coenzyme A biosynthesis, pantothenate => CoA [PATH:ath00770]\n",
      "DBLINKS     GO: 0015940 0015937\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G48560  CSR1; chlorsulfuron/imidazolinone resistant 1 [KO:K01652] [EC:2.2.1.6]\n",
      "AT2G31810  ACT domain-containing small subunit of acetolactate synthase protein [KO:K01653] [EC:2.2.1.6]\n",
      "AT5G16290  VAT1; VALINE-TOLERANT 1 [KO:K01653] [EC:2.2.1.6]\n",
      "AT3G58610  ketol-acid reductoisomerase [KO:K00053] [EC:1.1.1.86]\n",
      "AT3G23940  dehydratase family [KO:K01687] [EC:4.2.1.9]\n",
      "AT2G46110  KPHMT1; ketopantoate hydroxymethyltransferase 1 [KO:K00606] [EC:2.1.2.11]\n",
      "AT3G61530  PANB2; Phosphoenolpyruvate carboxylase family protein [KO:K00606] [EC:2.1.2.11]\n",
      "AT5G48840  PANC; pantoate-beta-alanine ligase [KO:K01918] [EC:6.3.2.1]\n",
      "AT1G60440  PANK1; pantothenate kinase 1 [KO:K09680] [EC:2.7.1.33]\n",
      "AT5G02080  phosphopantothenate-cysteine ligase-like protein [KO:K01922] [EC:6.3.2.51]\n",
      "AT1G12350  COAB; 4-phospho-panto-thenoylcysteine synthetase [KO:K01922] [EC:6.3.2.51]\n",
      "AT3G18030  HAL3A; HAL3-like protein A [KO:K01598] [EC:4.1.1.36]\n",
      "AT1G48605  ATHAL3B; Flavoprotein [KO:K01598] [EC:4.1.1.36]\n",
      "AT4G29680  Alkaline-phosphatase-like family protein [KO:K01513] [EC:3.1.4.1 3.6.1.9]\n",
      "AT2G18250  COAD; 4-phosphopantetheine adenylyltransferase [KO:K02201] [EC:2.7.7.3]\n",
      "AT2G27490  ATCOAE; dephospho-CoA kinase family [KO:K00859] [EC:2.7.1.24]\n",
      "AT3G11470  4'-phosphopantetheinyl transferase superfamily [KO:K06133] [EC:2.7.8.-]\n",
      "AT1G50090  BCAT7; D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G50110  D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10070  BCAT-2; branched-chain amino acid transaminase 2 [KO:K00826] [EC:2.6.1.42]\n",
      "AT3G49680  BCAT3; branched-chain aminotransferase 3 [KO:K00826] [EC:2.6.1.42]\n",
      "AT5G65780  ATBCAT-5; branched-chain amino acid aminotransferase 5 / branched-chain amino acid transaminase 5 (BCAT5) [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10060  BCAT-1; branched-chain amino acid transaminase 1 [KO:K00826] [EC:2.6.1.42]\n",
      "AT3G17810  PYD1; pyrimidine 1 [KO:K00207] [EC:1.3.1.2]\n",
      "AT5G12200  PYD2; pyrimidine 2 [KO:K01464] [EC:3.5.2.2]\n",
      "AT5G64370  BETA-UP; beta-ureidopropionase [KO:K01431] [EC:3.5.1.6]\n",
      "AT3G59050  PAO3; polyamine oxidase 3 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT1G65840  PAO4; polyamine oxidase 4 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT2G43020  PAO2; polyamine oxidase 2 [KO:K17839] [EC:1.5.3.17 1.5.3.-]\n",
      "AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "COMPOUND    C00010  CoA\n",
      "C00022  Pyruvate\n",
      "C00049  L-Aspartate\n",
      "C00054  Adenosine 3',5'-bisphosphate\n",
      "C00097  L-Cysteine\n",
      "C00099  beta-Alanine\n",
      "C00106  Uracil\n",
      "C00141  3-Methyl-2-oxobutanoic acid\n",
      "C00183  L-Valine\n",
      "C00229  Acyl-carrier protein\n",
      "C00429  5,6-Dihydrouracil\n",
      "C00522  (R)-Pantoate\n",
      "C00750  Spermine\n",
      "C00831  Pantetheine\n",
      "C00864  Pantothenate\n",
      "C00882  Dephospho-CoA\n",
      "C00966  2-Dehydropantoate\n",
      "C01053  (R)-4-Dehydropantoate\n",
      "C01088  (R)-3,3-Dimethylmalate\n",
      "C01134  Pantetheine 4'-phosphate\n",
      "C02642  3-Ureidopropionate\n",
      "C03492  D-4'-Phosphopantothenate\n",
      "C03688  Apo-[acyl-carrier-protein]\n",
      "C04079  N-((R)-Pantothenoyl)-L-cysteine\n",
      "C04272  (R)-2,3-Dihydroxy-3-methylbutanoate\n",
      "C04352  (R)-4'-Phosphopantothenoyl-L-cysteine\n",
      "C05665  3-Aminopropanal\n",
      "C05944  Pantothenol\n",
      "C06010  (S)-2-Acetolactate\n",
      "C18911  (R)-4-Phosphopantoate\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00410  beta-Alanine metabolism\n",
      "ath00640  Propanoate metabolism\n",
      "KO_PATHWAY  ko00770\n",
      "///\n",
      "ENTRY       ath00780                    Pathway\n",
      "NAME        Biotin metabolism - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Biotin (vitamin H or vitamin B7) is the essential cofactor of biotin-dependent carboxylases, such as pyruvate carboxylase and acetyl-CoA carboxylase. Mammals cannot synthesize biotin, while in bacteria, fungi, and plants it is synthesized from pimelate thioester through different pathways. In E. coli and many organisms, pimelate thioester is derived from malonyl-ACP. The pathway starts with the methylation to malonyl-ACP methyl ester, followed by the fatty acid chain elongation cycle to form pimeloyl-ACP methyl ester, which is then demethylated to form pimeloyl-ACP [MD:M00572]. Pimeloyl-ACP is converted to biotin through the final four steps in the biotin bicyclic ring assembly, which are conserved among biotin-producing organisms [MD:M00123]. In B. subtilis, biotin is derived from pimeloyl-ACP formed by oxidative cleavage of long-chain acyl-ACPs [MD:M00573]. Some bacteria synthesize biotin from pimeloyl-CoA derived from pimelate [MD:M00577]. Biotin is covalently attached to biotin-dependent carboxylase by biotin protein ligase, also known as holocarboxylase synthase in mammals, to form an active holocarboxylase. After degradation of the biotinylated carboxylase into biocytin, it is further degraded by biotinidase to release free biotin, which is recycled in holocarboxylase synthesis. Biotin is catabolized by beta-oxidation of the valeric acid side chain or oxidation of sulfur in the heterocyclic ring.\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00780  Biotin metabolism\n",
      "MODULE      ath_M00123  Biotin biosynthesis, pimeloyl-ACP/CoA => biotin [PATH:ath00780]\n",
      "DBLINKS     GO: 0006768\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G04540  Beta-ketoacyl synthase [KO:K09458] [EC:2.3.1.179]\n",
      "AT1G74960  FAB1; fatty acid biosynthesis 1 [KO:K09458] [EC:2.3.1.179]\n",
      "AT5G46290  KASI; 3-ketoacyl-acyl carrier protein synthase I [KO:K09458] [EC:2.3.1.179]\n",
      "AT4G13180  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT3G03980  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT1G24360  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT3G04000  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT5G18210  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00059] [EC:1.1.1.100]\n",
      "AT5G10160  Thioesterase superfamily protein [KO:K02372] [EC:4.2.1.59]\n",
      "AT2G22230  Thioesterase superfamily protein [KO:K02372] [EC:4.2.1.59]\n",
      "AT2G05990  MOD1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K00208] [EC:1.3.1.9 1.3.1.10]\n",
      "AT5G04620  BIOF; biotin F [KO:K00652] [EC:2.3.1.47]\n",
      "AT5G57590  BIO1; adenosylmethionine-8-amino-7-oxononanoate transaminase [KO:K19562] [EC:6.3.3.3 2.6.1.62]\n",
      "AT2G43360  BIO2; Radical SAM superfamily protein [KO:K01012] [EC:2.8.1.6]\n",
      "AT1G37150  HCS2; holocarboxylase synthetase 2 [KO:K01942] [EC:6.3.4.9 6.3.4.10 6.3.4.11 6.3.4.15]\n",
      "AT2G25710  HCS1; holocarboxylase synthase 1 [KO:K01942] [EC:6.3.4.9 6.3.4.10 6.3.4.11 6.3.4.15]\n",
      "COMPOUND    C00047  L-Lysine\n",
      "C00086  Urea\n",
      "C00120  Biotin\n",
      "C01037  7,8-Diaminononanoate\n",
      "C01063  6-Carboxyhexanoyl-CoA\n",
      "C01092  8-Amino-7-oxononanoate\n",
      "C01209  Malonyl-[acyl-carrier protein]\n",
      "C01894  Biotinyl-CoA\n",
      "C01909  Dethiobiotin\n",
      "C02656  Pimelate\n",
      "C05552  Biocytin\n",
      "C05921  Biotinyl-5'-AMP\n",
      "C06250  Holo-[carboxylase]\n",
      "C19673  Malonyl-[acp] methyl ester\n",
      "C19845  Pimeloyl-[acyl-carrier protein]\n",
      "C19846  Pimeloyl-[acyl-carrier protein] methyl ester\n",
      "C20372  3-Ketoglutaryl-[acp] methyl ester\n",
      "C20373  3-Hydroxyglutaryl-[acp] methyl ester\n",
      "C20374  Enoylglutaryl-[acp] methyl ester\n",
      "C20375  Glutaryl-[acp] methyl ester\n",
      "C20376  3-Ketopimeloyl-[acp] methyl ester\n",
      "C20377  3-Hydroxypimeloyl-[acp] methyl ester\n",
      "C20378  Enoylpimeloyl-[acp] methyl ester\n",
      "C20384  Bisnorbiotin\n",
      "C20385  Tetranorbiotin\n",
      "C20386  Biotin sulfoxide\n",
      "C20387  Biotin sulfone\n",
      "C20683  Long-chain acyl-[acyl-carrier protein]\n",
      "REFERENCE   PMID:21437340\n",
      "AUTHORS   Lin S, Cronan JE\n",
      "TITLE     Closing in on complete pathways of biotin biosynthesis.\n",
      "JOURNAL   Mol Biosyst 7:1811-21 (2011)\n",
      "DOI:10.1039/c1mb05022b\n",
      "REFERENCE   PMID:22965231\n",
      "AUTHORS   Lin S, Cronan JE\n",
      "TITLE     The BioC O-methyltransferase catalyzes methyl esterification of malonyl-acyl carrier protein, an essential step in biotin synthesis.\n",
      "JOURNAL   J Biol Chem 287:37010-20 (2012)\n",
      "DOI:10.1074/jbc.M112.410290\n",
      "REFERENCE   PMID:20693992\n",
      "AUTHORS   Lin S, Hanson RE, Cronan JE\n",
      "TITLE     Biotin synthesis begins by hijacking the fatty acid synthetic pathway.\n",
      "JOURNAL   Nat Chem Biol 6:682-8 (2010)\n",
      "DOI:10.1038/nchembio.420\n",
      "REFERENCE   PMID:21435937\n",
      "AUTHORS   Cronan JE, Lin S\n",
      "TITLE     Synthesis of the alpha,omega-dicarboxylic acid precursor of biotin by the canonical fatty acid biosynthetic pathway.\n",
      "JOURNAL   Curr Opin Chem Biol 15:407-13 (2011)\n",
      "DOI:10.1016/j.cbpa.2011.03.001\n",
      "REFERENCE   PMID:11368323\n",
      "AUTHORS   Stok JE, De Voss J\n",
      "TITLE     Expression, purification, and characterization of BioI: a carbon-carbon bond cleaving cytochrome P450 involved in biotin biosynthesis in Bacillus subtilis.\n",
      "JOURNAL   Arch Biochem Biophys 384:351-60 (2000)\n",
      "DOI:10.1006/abbi.2000.2067\n",
      "REFERENCE   PMID:18838690\n",
      "AUTHORS   Cryle MJ, Schlichting I\n",
      "TITLE     Structural insights from a P450 Carrier Protein complex reveal how specificity is achieved in the P450(BioI) ACP complex.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 105:15696-701 (2008)\n",
      "DOI:10.1073/pnas.0805983105\n",
      "REFERENCE   PMID:8763940\n",
      "AUTHORS   Bower S, Perkins JB, Yocum RR, Howitt CL, Rahaim P, Pero J\n",
      "TITLE     Cloning, sequencing, and characterization of the Bacillus subtilis biotin biosynthetic operon.\n",
      "JOURNAL   J Bacteriol 178:4122-30 (1996)\n",
      "DOI:10.1128/JB.178.14.4122-4130.1996\n",
      "REFERENCE   PMID:1445232\n",
      "AUTHORS   Ploux O, Soularue P, Marquet A, Gloeckler R, Lemoine Y\n",
      "TITLE     Investigation of the first step of biotin biosynthesis in Bacillus sphaericus. Purification and characterization of the pimeloyl-CoA synthase, and uptake of pimelate.\n",
      "JOURNAL   Biochem J 287 ( Pt 3):685-90 (1992)\n",
      "DOI:10.1042/bj2870685\n",
      "REFERENCE   PMID:12368242\n",
      "AUTHORS   Rodionov DA, Mironov AA, Gelfand MS\n",
      "TITLE     Conservation of the biotin regulon and the BirA regulatory signal in Eubacteria and Archaea.\n",
      "JOURNAL   Genome Res 12:1507-16 (2002)\n",
      "DOI:10.1101/gr.314502\n",
      "REFERENCE   PMID:11320134\n",
      "AUTHORS   Sullivan JT, Brown SD, Yocum RR, Ronson CW\n",
      "TITLE     The bio operon on the acquired symbiosis island of Mesorhizobium sp. strain R7A includes a novel gene involved in pimeloyl-CoA synthesis.\n",
      "JOURNAL   Microbiology 147:1315-22 (2001)\n",
      "DOI:10.1099/00221287-147-5-1315\n",
      "REFERENCE   PMID:12039741\n",
      "AUTHORS   Entcheva P, Phillips DA, Streit WR\n",
      "TITLE     Functional analysis of Sinorhizobium meliloti genes involved in biotin synthesis and transport.\n",
      "JOURNAL   Appl Environ Microbiol 68:2843-8 (2002)\n",
      "DOI:10.1128/AEM.68.6.2843-2848.2002\n",
      "REFERENCE   PMID:19319844\n",
      "AUTHORS   Zempleni J, Wijeratne SS, Hassan YI\n",
      "TITLE     Biotin.\n",
      "JOURNAL   Biofactors 35:36-46 (2009)\n",
      "DOI:10.1002/biof.8\n",
      "REFERENCE   PMID:9022537\n",
      "AUTHORS   Zempleni J, McCormick DB, Mock DM\n",
      "TITLE     Identification of biotin sulfone, bisnorbiotin methyl ketone, and tetranorbiotin-l-sulfoxide in human urine.\n",
      "JOURNAL   Am J Clin Nutr 65:508-11 (1997)\n",
      "DOI:10.1093/ajcn/65.2.508\n",
      "REFERENCE   PMID:5775781\n",
      "AUTHORS   Iwahara S, McCormick DB, Wright LD, Li HC\n",
      "TITLE     Bacterial degradation of biotin. 3. Metabolism of 14C-carbonyl-labeled biotin.\n",
      "JOURNAL   J Biol Chem 244:1393-8 (1969)\n",
      "REFERENCE   PMID:2180922\n",
      "AUTHORS   Pierson DE, Campbell A\n",
      "TITLE     Cloning and nucleotide sequence of bisC, the structural gene for biotin sulfoxide reductase in Escherichia coli.\n",
      "JOURNAL   J Bacteriol 172:2194-8 (1990)\n",
      "DOI:10.1128/JB.172.4.2194-2198.1990\n",
      "REFERENCE   PMID:15601707\n",
      "AUTHORS   Ezraty B, Bos J, Barras F, Aussel L\n",
      "TITLE     Methionine sulfoxide reduction and assimilation in Escherichia coli: new role for the biotin sulfoxide reductase BisC.\n",
      "JOURNAL   J Bacteriol 187:231-7 (2005)\n",
      "DOI:10.1128/JB.187.1.231-237.2005\n",
      "REFERENCE   PMID:374979\n",
      "AUTHORS   del Campillo-Campbell A, Dykhuizen D, Cleary PP\n",
      "TITLE     Enzymic reduction of d-biotin d-sulfoxide to d-biotin.\n",
      "JOURNAL   Methods Enzymol 62:379-85 (1979)\n",
      "DOI:10.1016/0076-6879(79)62244-9\n",
      "REFERENCE   PMID:9013576\n",
      "AUTHORS   Pollock VV, Barber MJ\n",
      "TITLE     Biotin sulfoxide reductase. Heterologous expression and characterization of a functional molybdopterin guanine dinucleotide-containing enzyme.\n",
      "JOURNAL   J Biol Chem 272:3355-62 (1997)\n",
      "DOI:10.1074/jbc.272.6.3355\n",
      "REFERENCE   PMID:15366932\n",
      "AUTHORS   Nelson KJ, Rajagopalan KV\n",
      "TITLE     Studies on the interaction of NADPH with Rhodobacter sphaeroides biotin sulfoxide reductase.\n",
      "JOURNAL   Biochemistry 43:11226-37 (2004)\n",
      "DOI:10.1021/bi0490845\n",
      "REFERENCE   PMID:16299174\n",
      "AUTHORS   Pinon V, Ravanel S, Douce R, Alban C\n",
      "TITLE     Biotin synthesis in plants. The first committed step of the pathway is catalyzed by a cytosolic 7-keto-8-aminopelargonic acid synthase.\n",
      "JOURNAL   Plant Physiol 139:1666-76 (2005)\n",
      "DOI:10.1104/pp.105.070144\n",
      "REL_PATHWAY ath00061  Fatty acid biosynthesis\n",
      "ath00310  Lysine degradation\n",
      "ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "KO_PATHWAY  ko00780\n",
      "///\n",
      "ENTRY       ath00785                    Pathway\n",
      "NAME        Lipoic acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00785  Lipoic acid metabolism\n",
      "MODULE      ath_M00881  Lipoic acid biosynthesis, plants and bacteria, octanoyl-ACP => dihydrolipoyl-E2/H [PATH:ath00785]\n",
      "ath_M00882  Lipoic acid biosynthesis, eukaryotes, octanoyl-ACP => dihydrolipoyl-H [PATH:ath00785]\n",
      "DBLINKS     GO: 0009106\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G31050  Biotin/lipoate A/B protein ligase family [KO:K03801] [EC:2.3.1.181]\n",
      "AT1G47578  Biotin/lipoate A/B protein ligase family [KO:K03801] [EC:2.3.1.181]\n",
      "AT1G04640  LIP2; lipoyltransferase 2 [KO:K23735] [EC:2.3.1.181]\n",
      "AT2G20860  LIP1; lipoic acid synthase 1 [KO:K03644] [EC:2.8.1.8]\n",
      "AT5G08415  Radical SAM superfamily protein [KO:K03644] [EC:2.8.1.8]\n",
      "COMPOUND    C01944  Octanoyl-CoA\n",
      "C02051  Lipoylprotein\n",
      "C02147  Dihydrolipoate\n",
      "C02972  Dihydrolipoylprotein\n",
      "C05752  Octanoyl-[acp]\n",
      "C06423  Octanoic acid\n",
      "C15972  Enzyme N6-(lipoyl)lysine\n",
      "C15973  Enzyme N6-(dihydrolipoyl)lysine\n",
      "C16238  Lipoyl-AMP\n",
      "C16241  (R)-Lipoate\n",
      "C22159  [Glycine cleavage system H]-N6-octanoyl-L-lysine\n",
      "C22160  [Lipoyl-carrier protein E2]-N6-octanoyl-L-lysine\n",
      "C22161  Dihydrolipoyl-AMP\n",
      "REFERENCE   PMID:16246025\n",
      "AUTHORS   Gunther S, McMillan PJ, Wallace LJ, Muller S.\n",
      "TITLE     Plasmodium falciparum possesses organelle-specific alpha-keto acid dehydrogenase complexes and lipoylation pathways.\n",
      "JOURNAL   Biochem Soc Trans 33:977-80 (2005)\n",
      "DOI:10.1042/BST20050977\n",
      "REFERENCE   PMID:11382754\n",
      "AUTHORS   Fujiwara K, Takeuchi S, Okamura-Ikeda K, Motokawa Y.\n",
      "TITLE     Purification, characterization, and cDNA cloning of lipoate-activating enzyme from bovine liver.\n",
      "JOURNAL   J Biol Chem 276:28819-23 (2001)\n",
      "DOI:10.1074/jbc.M101748200\n",
      "REFERENCE   PMID:9808738\n",
      "AUTHORS   Yasuno R, Wada H.\n",
      "TITLE     Biosynthesis of lipoic acid in Arabidopsis: cloning and characterization of the cDNA for lipoic acid synthase.\n",
      "JOURNAL   Plant Physiol 118:935-43 (1998)\n",
      "DOI:10.1104/pp.118.3.935\n",
      "REFERENCE   PMID:10671542\n",
      "AUTHORS   Gueguen V, Macherel D, Jaquinod M, Douce R, Bourguignon J.\n",
      "TITLE     Fatty acid and lipoic acid biosynthesis in higher plant mitochondria.\n",
      "JOURNAL   J Biol Chem 275:5016-25 (2000)\n",
      "DOI:10.1074/jbc.275.7.5016\n",
      "REFERENCE   PMID:11427685\n",
      "AUTHORS   Wada M, Yasuno R, Jordan SW, Cronan JE Jr, Wada H.\n",
      "TITLE     Lipoic acid metabolism in Arabidopsis thaliana: cloning and characterization of a cDNA encoding lipoyltransferase.\n",
      "JOURNAL   Plant Cell Physiol 42:650-6 (2001)\n",
      "DOI:10.1093/pcp/pce081\n",
      "REFERENCE   PMID:8206909\n",
      "AUTHORS   Morris TW, Reed KE, Cronan JE Jr.\n",
      "TITLE     Identification of the gene encoding lipoate-protein ligase A of Escherichia coli. Molecular cloning and characterization of the lplA gene and gene product.\n",
      "JOURNAL   J Biol Chem 269:16091-100 (1994)\n",
      "REFERENCE   PMID:14700636\n",
      "AUTHORS   Zhao X, Miller JR, Jiang Y, Marletta MA, Cronan JE.\n",
      "TITLE     Assembly of the covalent linkage between lipoic acid and its cognate enzymes.\n",
      "JOURNAL   Chem Biol 10:1293-302 (2003)\n",
      "DOI:10.1016/j.chembiol.2003.11.016\n",
      "REFERENCE   PMID:15642479\n",
      "AUTHORS   Nesbitt NM, Baleanu-Gogonea C, Cicchillo RM, Goodson K, Iwig DF, Broadwater JA, Haas JA, Fox BG, Booker SJ.\n",
      "TITLE     Expression, purification, and physical characterization of Escherichia coli lipoyl(octanoyl)transferase.\n",
      "JOURNAL   Protein Expr Purif 39:269-82 (2005)\n",
      "DOI:10.1016/j.pep.2004.10.021\n",
      "REFERENCE   PMID:29191830\n",
      "AUTHORS   Solmonson A, DeBerardinis RJ\n",
      "TITLE     Lipoic acid metabolism and mitochondrial redox regulation.\n",
      "JOURNAL   J Biol Chem 293:7522-7530 (2018)\n",
      "DOI:10.1074/jbc.TM117.000259\n",
      "REFERENCE   PMID:27074917\n",
      "AUTHORS   Cronan JE\n",
      "TITLE     Assembly of Lipoic Acid on Its Cognate Enzymes: an Extraordinary and Essential Biosynthetic Pathway.\n",
      "JOURNAL   Microbiol Mol Biol Rev 80:429-50 (2016)\n",
      "DOI:10.1128/MMBR.00073-15\n",
      "REFERENCE   PMID:24777537\n",
      "AUTHORS   Mayr JA, Feichtinger RG, Tort F, Ribes A, Sperl W\n",
      "TITLE     Lipoic acid biosynthesis defects.\n",
      "JOURNAL   J Inherit Metab Dis 37:553-63 (2014)\n",
      "DOI:10.1007/s10545-014-9705-8\n",
      "REFERENCE   PMID:29987032\n",
      "AUTHORS   Cao X, Zhu L, Song X, Hu Z, Cronan JE\n",
      "TITLE     Protein moonlighting elucidates the essential human pathway catalyzing lipoic acid assembly on its cognate enzymes.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 115:E7063-E7072 (2018)\n",
      "DOI:10.1073/pnas.1805862115\n",
      "REFERENCE   PMID:24341803\n",
      "AUTHORS   Soreze Y, Boutron A, Habarou F, Barnerias C, Nonnenmacher L, Delpech H, Mamoune A, Chretien D, Hubert L, Bole-Feysot C, Nitschke P, Correia I, Sardet C, Boddaert N, Hamel Y, Delahodde A, Ottolenghi C, de Lonlay P\n",
      "TITLE     Mutations in human lipoyltransferase gene LIPT1 cause a Leigh disease with secondary deficiency for pyruvate and alpha-ketoglutarate dehydrogenase.\n",
      "JOURNAL   Orphanet J Rare Dis 8:192 (2013)\n",
      "DOI:10.1186/1750-1172-8-192\n",
      "REFERENCE   PMID:28757203\n",
      "AUTHORS   Habarou F, Hamel Y, Haack TB, Feichtinger RG, Lebigot E, Marquardt I, Busiah K, Laroche C, Madrange M, Grisel C, Pontoizeau C, Eisermann M, Boutron A, Chretien D, Chadefaux-Vekemans B, Barouki R, Bole-Feysot C, Nitschke P, Goudin N, Boddaert N, Nemazanyy I, Delahodde A, Kolker S, Rodenburg RJ, Korenke GC, Meitinger T, Strom TM, Prokisch H, Rotig A, Ottolenghi C, Mayr JA, de Lonlay P\n",
      "TITLE     Biallelic Mutations in LIPT2 Cause a Mitochondrial Lipoylation Defect Associated with Severe Neonatal Encephalopathy.\n",
      "JOURNAL   Am J Hum Genet 101:283-290 (2017)\n",
      "DOI:10.1016/j.ajhg.2017.07.001\n",
      "REFERENCE   PMID:20508247\n",
      "AUTHORS   Spalding MD, Prigge ST\n",
      "TITLE     Lipoic acid metabolism in microbial pathogens.\n",
      "JOURNAL   Microbiol Mol Biol Rev 74:200-28 (2010)\n",
      "DOI:10.1128/MMBR.00008-10\n",
      "REFERENCE   PMID:28543853\n",
      "AUTHORS   Guerra AJ, Afanador GA, Prigge ST\n",
      "TITLE     Crystal structure of lipoate-bound lipoate ligase 1, LipL1, from Plasmodium falciparum.\n",
      "JOURNAL   Proteins 85:1777-1783 (2017)\n",
      "DOI:10.1002/prot.25324\n",
      "REFERENCE   PMID:25116855\n",
      "AUTHORS   Afanador GA, Matthews KA, Bartee D, Gisselberg JE, Walters MS, Freel Meyers CL, Prigge ST\n",
      "TITLE     Redox-dependent lipoylation of mitochondrial proteins in Plasmodium falciparum.\n",
      "JOURNAL   Mol Microbiol 94:156-71 (2014)\n",
      "DOI:10.1111/mmi.12753\n",
      "REFERENCE   PMID:28836704\n",
      "AUTHORS   Afanador GA, Guerra AJ, Swift RP, Rodriguez RE, Bartee D, Matthews KA, Schon A, Freire E, Freel Meyers CL, Prigge ST\n",
      "TITLE     A novel lipoate attachment enzyme is shared by Plasmodium and Chlamydia species.\n",
      "JOURNAL   Mol Microbiol 106:439-451 (2017)\n",
      "DOI:10.1111/mmi.13776\n",
      "REFERENCE   PMID:22607141\n",
      "AUTHORS   Storm J, Muller S\n",
      "TITLE     Lipoic acid metabolism of Plasmodium--a suitable drug target.\n",
      "JOURNAL   Curr Pharm Des 18:3480-9 (2012)\n",
      "DOI:10.2174/138161212801327266\n",
      "REFERENCE   PMID:18069893\n",
      "AUTHORS   Gunther S, Wallace L, Patzewitz EM, McMillan PJ, Storm J, Wrenger C, Bissett R, Smith TK, Muller S\n",
      "TITLE     Apicoplast lipoic acid protein ligase B is not essential for Plasmodium falciparum.\n",
      "JOURNAL   PLoS Pathog 3:e189 (2007)\n",
      "DOI:10.1371/journal.ppat.0030189\n",
      "REFERENCE   PMID:15225307\n",
      "AUTHORS   Wrenger C, Muller S\n",
      "TITLE     The human malaria parasite Plasmodium falciparum has distinct organelle-specific lipoylation pathways.\n",
      "JOURNAL   Mol Microbiol 53:103-13 (2004)\n",
      "DOI:10.1111/j.1365-2958.2004.04112.x\n",
      "REFERENCE   PMID:21338420\n",
      "AUTHORS   Martin N, Christensen QH, Mansilla MC, Cronan JE, de Mendoza D\n",
      "TITLE     A novel two-gene requirement for the octanoyltransfer reaction of Bacillus subtilis lipoic acid biosynthesis.\n",
      "JOURNAL   Mol Microbiol 80:335-49 (2011)\n",
      "DOI:10.1111/j.1365-2958.2011.07597.x\n",
      "REFERENCE   PMID:21338421\n",
      "AUTHORS   Christensen QH, Martin N, Mansilla MC, de Mendoza D, Cronan JE\n",
      "TITLE     A novel amidotransferase required for lipoic acid cofactor assembly in Bacillus subtilis.\n",
      "JOURNAL   Mol Microbiol 80:350-63 (2011)\n",
      "DOI:10.1111/j.1365-2958.2011.07598.x\n",
      "REL_PATHWAY ath00061  Fatty acid biosynthesis\n",
      "KO_PATHWAY  ko00785\n",
      "///\n",
      "ENTRY       ath00790                    Pathway\n",
      "NAME        Folate biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00790  Folate biosynthesis\n",
      "MODULE      ath_M00126  Tetrahydrofolate biosynthesis, GTP => THF [PATH:ath00790]\n",
      "ath_M00880  Molybdenum cofactor biosynthesis, GTP => molybdenum cofactor [PATH:ath00790]\n",
      "DBLINKS     GO: 0046656\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G07270  GTP cyclohydrolase I [KO:K01495] [EC:3.5.4.16]\n",
      "AT5G42370  Calcineurin-like metallo-phosphoesterase superfamily protein [KO:K01113] [EC:3.1.3.1]\n",
      "AT3G11750  FOLB1; Dihydroneopterin aldolase [KO:K01633] [EC:4.1.2.25 5.1.99.8 1.13.11.81]\n",
      "AT5G62980  FOLB2; Dihydroneopterin aldolase [KO:K01633] [EC:4.1.2.25 5.1.99.8 1.13.11.81]\n",
      "AT3G21730  FOLB3; Dihydroneopterin aldolase [KO:K01633] [EC:4.1.2.25 5.1.99.8 1.13.11.81]\n",
      "AT1G69190  Dihydropterin pyrophosphokinase / Dihydropteroate synthase [KO:K13941] [EC:2.7.6.3 2.5.1.15]\n",
      "AT4G30000  Dihydropterin pyrophosphokinase / Dihydropteroate synthase [KO:K13941] [EC:2.7.6.3 2.5.1.15]\n",
      "AT4G34570  THY-2; thymidylate synthase 2 [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "AT2G16370  THY-1; thymidylate synthase 1 [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "AT2G21550  Bifunctional dihydrofolate reductase/thymidylate synthase [KO:K13998] [EC:1.5.1.3 2.1.1.45]\n",
      "AT5G05980  DFB; DHFS-FPGS homolog B [KO:K01930] [EC:6.3.2.17]\n",
      "AT3G55630  DFD; DHFS-FPGS homolog D [KO:K01930] [EC:6.3.2.17]\n",
      "AT3G10160  DFC; DHFS-FPGS homolog C [KO:K01930] [EC:6.3.2.17]\n",
      "AT5G41480  GLA1; Folylpolyglutamate synthetase family protein [KO:K20457] [EC:6.3.2.12]\n",
      "AT1G78680  GGH2; gamma-glutamyl hydrolase 2 [KO:K01307] [EC:3.4.19.9]\n",
      "AT1G78670  GGH3; gamma-glutamyl hydrolase 3 [KO:K01307] [EC:3.4.19.9]\n",
      "AT1G78660  GGH1; gamma-glutamyl hydrolase 1 [KO:K01307] [EC:3.4.19.9]\n",
      "AT5G59750  RIBA3; monofunctional riboflavin biosynthesis protein RIBA 3 [KO:K14652] [EC:4.1.99.12 3.5.4.25]\n",
      "AT2G22450  RIBA2; riboflavin biosynthesis protein [KO:K14652] [EC:4.1.99.12 3.5.4.25]\n",
      "AT5G64300  GCH; GTP cyclohydrolase II [KO:K14652] [EC:4.1.99.12 3.5.4.25]\n",
      "AT2G28880  emb1997; para-aminobenzoate (PABA) synthase family protein [KO:K13950] [EC:2.6.1.85]\n",
      "AT5G57850  ADCL; D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K18482] [EC:4.1.3.38]\n",
      "AT3G59710  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00079] [EC:1.1.1.184 1.1.1.189 1.1.1.197]\n",
      "AT5G61830  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00079] [EC:1.1.1.184 1.1.1.189 1.1.1.197]\n",
      "AT5G51030  NAD(P)-binding Rossmann-fold superfamily protein [KO:K00079] [EC:1.1.1.184 1.1.1.189 1.1.1.197]\n",
      "AT2G31955  CNX2; cofactor of nitrate reductase and xanthine dehydrogenase 2 [KO:K03639] [EC:4.1.99.22]\n",
      "AT1G01290  CNX3; cofactor of nitrate reductase and xanthine dehydrogenase 3 [KO:K03637] [EC:4.6.1.17]\n",
      "AT2G43760  molybdopterin biosynthesis MoaE family protein [KO:K03635] [EC:2.8.1.12]\n",
      "AT5G20990  B73; molybdopterin biosynthesis CNX1 protein / molybdenum cofactor biosynthesis enzyme CNX1 (CNX1) [KO:K15376] [EC:2.10.1.1 2.7.7.75]\n",
      "AT1G16540  ABA3; molybdenum cofactor sulfurase (LOS5) (ABA3) [KO:K15631] [EC:2.8.1.9]\n",
      "COMPOUND    C00044  GTP\n",
      "C00101  Tetrahydrofolate\n",
      "C00119  5-Phospho-alpha-D-ribose 1-diphosphate\n",
      "C00156  4-Hydroxybenzoate\n",
      "C00251  Chorismate\n",
      "C00266  Glycolaldehyde\n",
      "C00268  Dihydrobiopterin\n",
      "C00272  Tetrahydrobiopterin\n",
      "C00415  Dihydrofolate\n",
      "C00504  Folate\n",
      "C00568  4-Aminobenzoate\n",
      "C00835  Sepiapterin\n",
      "C00921  Dihydropteroate\n",
      "C01217  5,6,7,8-Tetrahydromethanopterin\n",
      "C01300  6-(Hydroxymethyl)-7,8-dihydropterin\n",
      "C01304  2,5-Diamino-6-(5-phospho-D-ribosylamino)pyrimidin-4(3H)-one\n",
      "C02587  Pyrimidodiazepine\n",
      "C02953  7,8-Dihydrobiopterin\n",
      "C03541  THF-polyglutamate\n",
      "C03684  6-Pyruvoyltetrahydropterin\n",
      "C04244  6-Lactoyl-5,6,7,8-tetrahydropterin\n",
      "C04807  6-Hydroxymethyl-7,8-dihydropterin diphosphate\n",
      "C04874  7,8-Dihydroneopterin\n",
      "C04895  7,8-Dihydroneopterin 3'-triphosphate\n",
      "C05922  Formamidopyrimidine nucleoside triphosphate\n",
      "C05923  2,5-Diaminopyrimidine nucleoside triphosphate\n",
      "C05924  Molybdopterin\n",
      "C05925  Dihydroneopterin phosphate\n",
      "C05926  Neopterin\n",
      "C05927  7,8-Dihydromethanopterin\n",
      "C06148  2,5-Diamino-6-(5'-triphosphoryl-3',4'-trihydroxy-2'-oxopentyl)-amino-4-oxopyrimidine\n",
      "C06313  Biopterin\n",
      "C09332  THF-L-glutamate\n",
      "C11355  4-Amino-4-deoxychorismate\n",
      "C15522  4a-Hydroxytetrahydrobiopterin\n",
      "C15996  7-Cyano-7-carbaguanine\n",
      "C16675  7-Aminomethyl-7-carbaguanine\n",
      "C18237  Molybdoenzyme molybdenum cofactor\n",
      "C18239  Precursor Z\n",
      "C18802  Tetrahydrosarcinapterin\n",
      "C19848  Adenylated molybdopterin\n",
      "C19871  Guanylyl molybdenum cofactor\n",
      "C20239  6-Carboxy-5,6,7,8-tetrahydropterin\n",
      "C20248  7-Carboxy-7-carbaguanine\n",
      "C20264  L-threo-Tetrahydrobiopterin\n",
      "C20559  4-(beta-D-Ribofuranosyl)aniline 5'-phosphate\n",
      "C20562  N-[(7,8-Dihydropterin-6-yl)methyl]-4-(beta-D-ribofuranosyl)aniline 5'-phosphate\n",
      "C20566  7,8-Dihydroneopterin 2'-phosphate\n",
      "C20567  7,8-Dihydroneopterin 2',3'-cyclic phosphate\n",
      "C21007  5,6,7,8-Tetrahydromonapterin\n",
      "C21008  7,8-Dihydromonapterin\n",
      "C21065  7,8-Dihydroxanthopterin\n",
      "C21094  7,8-Dihydromonapterin 3'-triphosphate\n",
      "C21107  4-(beta-D-Ribofuranosyl)phenol 5'-phosphate\n",
      "C21310  (8S)-3',8-Cyclo-7,8-dihydroguanosine 5'-triphosphate\n",
      "C21485  Cytidylyl molybdenum cofactor\n",
      "C21486  Thio-molybdenum cofactor\n",
      "C21640  6-(1'-Hydroxy-2'-oxopropyl)-tetrahydropterin\n",
      "REFERENCE\n",
      "AUTHORS   Nishizuka Y, Seyama Y, Ikai A, Ishimura Y, Kawaguchi A (eds).\n",
      "TITLE     [Cellular Functions and Metabolic Maps] (In Japanese)\n",
      "JOURNAL   Tokyo Kagaku Dojin (1997)\n",
      "REFERENCE   PMID:19675644\n",
      "AUTHORS   Schwarz G, Mendel RR, Ribbe MW\n",
      "TITLE     Molybdenum cofactors, enzymes and pathways.\n",
      "JOURNAL   Nature 460:839-47 (2009)\n",
      "DOI:10.1038/nature08302\n",
      "REFERENCE   PMID:16632608\n",
      "AUTHORS   Hanzelmann P, Schindelin H\n",
      "TITLE     Binding of 5'-GTP to the C-terminal FeS cluster of the radical S-adenosylmethionine enzyme MoaA provides insights into its mechanism.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 103:6829-34 (2006)\n",
      "DOI:10.1073/pnas.0510711103\n",
      "REFERENCE   PMID:18092812\n",
      "AUTHORS   Daniels JN, Wuebbens MM, Rajagopalan KV, Schindelin H\n",
      "TITLE     Crystal structure of a molybdopterin synthase-precursor Z complex: insight into its sulfur transfer mechanism and its role in molybdenum cofactor deficiency.\n",
      "JOURNAL   Biochemistry 47:615-26 (2008)\n",
      "DOI:10.1021/bi701734g\n",
      "REFERENCE   PMID:12571226\n",
      "AUTHORS   Wuebbens MM, Rajagopalan KV\n",
      "TITLE     Mechanistic and mutational studies of Escherichia coli molybdopterin synthase clarify the final step of molybdopterin biosynthesis.\n",
      "JOURNAL   J Biol Chem 278:14523-32 (2003)\n",
      "DOI:10.1074/jbc.M300453200\n",
      "REFERENCE   PMID:21867484\n",
      "AUTHORS   Werner ER, Blau N, Thony B\n",
      "TITLE     Tetrahydrobiopterin: biochemistry and pathophysiology.\n",
      "JOURNAL   Biochem J 438:397-414 (2011)\n",
      "DOI:10.1042/BJ20110293\n",
      "REFERENCE   PMID:12893295\n",
      "AUTHORS   Iino T, Tabata M, Takikawa S, Sawada H, Shintaku H, Ishikura S, Hara A\n",
      "TITLE     Tetrahydrobiopterin is synthesized from 6-pyruvoyl-tetrahydropterin by the human aldo-keto reductase AKR1 family members.\n",
      "JOURNAL   Arch Biochem Biophys 416:180-7 (2003)\n",
      "DOI:10.1016/S0003-9861(03)00295-9\n",
      "REFERENCE   PMID:1902669\n",
      "AUTHORS   Park YS, Heizmann CW, Wermuth B, Levine RA, Steinerstauch P, Guzman J, Blau N\n",
      "TITLE     Human carbonyl and aldose reductases: new catalytic functions in tetrahydrobiopterin biosynthesis.\n",
      "JOURNAL   Biochem Biophys Res Commun 175:738-44 (1991)\n",
      "DOI:10.1016/0006-291X(91)91628-P\n",
      "REL_PATHWAY ath00230  Purine metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00670  One carbon pool by folate\n",
      "KO_PATHWAY  ko00790\n",
      "///\n",
      "ENTRY       ath00860                    Pathway\n",
      "NAME        Porphyrin and chlorophyll metabolism - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of cofactors and vitamins\n",
      "PATHWAY_MAP ath00860  Porphyrin and chlorophyll metabolism\n",
      "MODULE      ath_M00121  Heme biosynthesis, plants and bacteria, glutamate => heme [PATH:ath00860]\n",
      "DBLINKS     GO: 0006778\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G64050  ERS; glutamate tRNA synthetase [KO:K01885] [EC:6.1.1.17]\n",
      "AT5G26710  Glutamyl/glutaminyl-tRNA synthetase, class Ic [KO:K01885] [EC:6.1.1.17]\n",
      "AT1G09940  HEMA2; Glutamyl-tRNA reductase family protein [KO:K02492] [EC:1.2.1.70]\n",
      "AT1G58290  HEMA1; Glutamyl-tRNA reductase family protein [KO:K02492] [EC:1.2.1.70]\n",
      "AT2G31250  HEMA3; Glutamyl-tRNA reductase family protein [KO:K02492] [EC:1.2.1.70]\n",
      "AT5G63570  GSA1; glutamate-1-semialdehyde-2,1-aminomutase [KO:K01845] [EC:5.4.3.8]\n",
      "AT3G48730  GSA2; glutamate-1-semialdehyde 2,1-aminomutase 2 [KO:K01845] [EC:5.4.3.8]\n",
      "AT1G44318  hemb2; Aldolase superfamily protein [KO:K01698] [EC:4.2.1.24]\n",
      "AT1G69740  HEMB1; Aldolase superfamily protein [KO:K01698] [EC:4.2.1.24]\n",
      "AT5G08280  HEMC; hydroxymethylbilane synthase [KO:K01749] [EC:2.5.1.61]\n",
      "AT2G26540  HEMD; uroporphyrinogen-III synthase family protein [KO:K01719] [EC:4.2.1.75]\n",
      "AT2G40490  HEME2; Uroporphyrinogen decarboxylase [KO:K01599] [EC:4.1.1.37]\n",
      "AT3G14930  HEME1; Uroporphyrinogen decarboxylase [KO:K01599] [EC:4.1.1.37]\n",
      "AT1G03475  LIN2; Coproporphyrinogen III oxidase [KO:K00228] [EC:1.3.3.3]\n",
      "AT4G03205  hemf2; Coproporphyrinogen III oxidase [KO:K00228] [EC:1.3.3.3]\n",
      "AT5G14220  HEMG2; Flavin containing amine oxidoreductase family [KO:K00231] [EC:1.3.3.4 1.3.3.15]\n",
      "AT4G01690  PPOX; Flavin containing amine oxidoreductase family [KO:K00231] [EC:1.3.3.4 1.3.3.15]\n",
      "AT5G26030  FC1; ferrochelatase 1 [KO:K01772] [EC:4.99.1.1 4.99.1.9]\n",
      "AT2G30390  FC2; ferrochelatase 2 [KO:K01772] [EC:4.99.1.1 4.99.1.9]\n",
      "AT5G40850  UPM1; urophorphyrin methylase 1 [KO:K02303] [EC:2.1.1.107]\n",
      "AT1G50170  SIRB; sirohydrochlorin ferrochelatase B [KO:K03794] [EC:4.99.1.4]\n",
      "AT2G44520  COX10; cytochrome c oxidase 10 [KO:K02257] [EC:2.5.1.141]\n",
      "AT5G56090  COX15; cytochrome c oxidase 15 [KO:K02259] [EC:1.17.99.9]\n",
      "AT5G13630  GUN5; magnesium-chelatase subunit chlH, chloroplast, putative / Mg-protoporphyrin IX chelatase, putative (CHLH) [KO:K03403] [EC:6.6.1.1]\n",
      "AT1G08520  ALB1; ALBINA 1 [KO:K03404] [EC:6.6.1.1]\n",
      "AT4G18480  CHLI1; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K03405] [EC:6.6.1.1]\n",
      "AT5G45930  CHLI2; magnesium chelatase i2 [KO:K03405] [EC:6.6.1.1]\n",
      "AT4G25080  CHLM; magnesium-protoporphyrin IX methyltransferase [KO:K03428] [EC:2.1.1.11]\n",
      "AT3G56940  CRD1; dicarboxylate diiron protein, putative (Crd1) [KO:K04035] [EC:1.14.13.81]\n",
      "AT5G18660  PCB2; NAD(P)-binding Rossmann-fold superfamily protein [KO:K19073] [EC:1.3.1.75]\n",
      "AT5G54190  PORA; protochlorophyllide oxidoreductase A [KO:K00218] [EC:1.3.1.33]\n",
      "AT4G27440  PORB; protochlorophyllide oxidoreductase B [KO:K00218] [EC:1.3.1.33]\n",
      "AT1G03630  POR_C; protochlorophyllide oxidoreductase C [KO:K00218] [EC:1.3.1.33]\n",
      "AT3G51820  G4; UbiA prenyltransferase family protein [KO:K04040] [EC:2.5.1.62 2.5.1.133]\n",
      "AT1G19670  CLH1; chlorophyllase 1 [KO:K08099] [EC:3.1.1.14]\n",
      "AT5G43860  CLH2; chlorophyllase 2 [KO:K08099] [EC:3.1.1.14]\n",
      "AT1G44446  CH1; Pheophorbide a oxygenase family protein with Rieske 2Fe-2S domain-containing protein [KO:K13600] [EC:1.14.13.122]\n",
      "AT4G13250  NYC1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K13606] [EC:1.1.1.294]\n",
      "AT5G04900  NOL; NYC1-like protein [KO:K13606] [EC:1.1.1.294]\n",
      "AT1G04620  HCAR; coenzyme F420 hydrogenase family / dehydrogenase, beta subunit family [KO:K18010] [EC:1.17.7.2]\n",
      "AT4G11910  STAY-GREEN-like protein [KO:K22013] [EC:4.99.1.10]\n",
      "AT1G44000  STAY-GREEN-like protein [KO:K22013] [EC:4.99.1.10]\n",
      "AT4G22920  NYE1; non-yellowing 1 [KO:K22013] [EC:4.99.1.10]\n",
      "AT4G11911  STAY-GREEN-like protein [KO:K22013] [EC:4.99.1.10]\n",
      "AT4G16690  MES16; methyl esterase 16 [KO:K13544] [EC:3.1.1.82]\n",
      "AT3G44880  ACD1; Pheophorbide a oxygenase family protein with Rieske 2Fe-2S domain-containing protein [KO:K13071] [EC:1.14.15.17]\n",
      "AT4G37000  ACD2; accelerated cell death 2 (ACD2) [KO:K13545] [EC:1.3.7.12]\n",
      "AT1G74470  Pyridine nucleotide-disulfide oxidoreductase family protein [KO:K10960] [EC:1.3.1.83 1.3.1.111]\n",
      "AT1G58300  HO4; heme oxygenase 4 [KO:K21480] [EC:1.14.15.20]\n",
      "AT2G26670  TED4; Plant heme oxygenase (decyclizing) family protein [KO:K21480] [EC:1.14.15.20]\n",
      "AT1G69720  HO3; heme oxygenase 3 [KO:K21480] [EC:1.14.15.20]\n",
      "AT3G09150  HY2; phytochromobilin:ferredoxin oxidoreductase, chloroplast / phytochromobilin synthase (HY2) [KO:K08101] [EC:1.3.7.4]\n",
      "AT4G03240  FH; frataxin-like protein [KO:K19054] [EC:1.16.3.1]\n",
      "COMPOUND    C00025  L-Glutamate\n",
      "C00032  Heme\n",
      "C00037  Glycine\n",
      "C00188  L-Threonine\n",
      "C00194  Cobamide coenzyme\n",
      "C00430  5-Aminolevulinate\n",
      "C00486  Bilirubin\n",
      "C00500  Biliverdin\n",
      "C00524  Cytochrome c\n",
      "C00541  Cob(II)alamin\n",
      "C00748  Siroheme\n",
      "C00853  Cob(I)alamin\n",
      "C00931  Porphobilinogen\n",
      "C00992  Aquacob(III)alamin\n",
      "C01024  Hydroxymethylbilane\n",
      "C01051  Uroporphyrinogen III\n",
      "C01079  Protoporphyrinogen IX\n",
      "C01708  Hemoglobin\n",
      "C02139  Chlorophyllide\n",
      "C02191  Protoporphyrin\n",
      "C02463  Precorrin 2\n",
      "C02469  Uroporphyrin III\n",
      "C02800  4,5-Dioxopentanoate\n",
      "C02823  Cyanocobalamin\n",
      "C02880  Protochlorophyllide\n",
      "C02987  L-Glutamyl-tRNA(Glu)\n",
      "C03029  Transferrin[Fe(II)]2\n",
      "C03114  Dimethylbenzimidazole\n",
      "C03179  Transferrin[Fe(III)]2\n",
      "C03194  (R)-1-Aminopropan-2-ol\n",
      "C03263  Coproporphyrinogen III\n",
      "C03516  Magnesium protoporphyrin\n",
      "C03741  (S)-4-Amino-5-oxopentanoate\n",
      "C04122  D-1-Aminopropan-2-ol O-phosphate\n",
      "C04536  Magnesium protoporphyrin monomethyl ester\n",
      "C04778  N1-(5-Phospho-alpha-D-ribosyl)-5,6-dimethylbenzimidazole\n",
      "C05306  Chlorophyll a\n",
      "C05307  Chlorophyll b\n",
      "C05766  Uroporphyrinogen I\n",
      "C05767  Uroporphyrin I\n",
      "C05768  Coproporphyrinogen I\n",
      "C05769  Coproporphyrin I\n",
      "C05770  Coproporphyrin III\n",
      "C05772  Precorrin 3A\n",
      "C05773  Cobyrinate\n",
      "C05774  Cobinamide\n",
      "C05775  alpha-Ribazole\n",
      "C05777  Coenzyme F430\n",
      "C05778  Sirohydrochlorin\n",
      "C05779  Apoferritin\n",
      "C05780  Apotransferrin\n",
      "C05781  Oxyhemoglobin\n",
      "C05782  Myoglobin\n",
      "C05783  Cytochrome a\n",
      "C05784  Catalase\n",
      "C05785  Peroxidase\n",
      "C05786  (3Z)-Phycocyanobilin\n",
      "C05787  Bilirubin beta-diglucuronide\n",
      "C05789  L-Urobilinogen\n",
      "C05790  I-Urobilinogen\n",
      "C05791  D-Urobilinogen\n",
      "C05793  L-Urobilin\n",
      "C05794  Urobilin\n",
      "C05795  D-Urobilin\n",
      "C05797  Pheophytin a\n",
      "C05798  Bacteriopheophytin\n",
      "C05912  (3Z)-Phycoerythrobilin\n",
      "C05913  (3Z)-Phytochromobilin\n",
      "C06319  Precorrin 6Y\n",
      "C06320  Precorrin 6X\n",
      "C06399  Hydrogenobyrinate\n",
      "C06406  Precorrin 3B\n",
      "C06407  Precorrin 4\n",
      "C06408  Precorrin 8X\n",
      "C06416  Precorrin 5\n",
      "C06503  Hydrogenobyrinate a,c diamide\n",
      "C06504  Cob(II)yrinate a,c diamide\n",
      "C06505  Cob(I)yrinate a,c diamide\n",
      "C06506  Adenosyl cobyrinate a,c diamide\n",
      "C06507  Adenosyl cobyrinate hexaamide\n",
      "C06508  Adenosyl cobinamide\n",
      "C06509  Adenosyl cobinamide phosphate\n",
      "C06510  Adenosine-GDP-cobinamide\n",
      "C11242  Bacteriochlorophyll a\n",
      "C11243  Bacteriochlorophyll b\n",
      "C11538  Cobalt-sirohydrochlorin\n",
      "C11540  Cobalt-precorrin 4\n",
      "C11542  Cobalt-precorrin 6\n",
      "C11543  Cobalt-dihydro-precorrin 6\n",
      "C11545  Cobalt-precorrin 8\n",
      "C11630  15,16-Dihydrobiliverdin\n",
      "C11829  13(1)-Hydroxy-magnesium-protoporphyrin IX 13-monomethyl ester\n",
      "C11830  13(1)-Oxo-magnesium-protoporphyrin IX 13-monomethyl ester\n",
      "C11831  Divinylprotochlorophyllide\n",
      "C11832  Divinyl chlorophyllide a\n",
      "C11850  Divinylchlorophyll a\n",
      "C11851  Zn-Bacteriochlorophyll a\n",
      "C12147  L-Threonine O-3-phosphate\n",
      "C14818  Fe2+\n",
      "C14819  Fe3+\n",
      "C15670  Heme A\n",
      "C15672  Heme O\n",
      "C16242  Cobalt-precorrin 5A\n",
      "C16243  Cobalt-precorrin 5B\n",
      "C16244  Cobalt-precorrin 7\n",
      "C16540  7(1)-Hydroxychlorophyllide a\n",
      "C16541  Chlorophyllide b\n",
      "C17401  Cobalt-factor III\n",
      "C18021  Pheophorbide a\n",
      "C18022  Red chlorophyll catabolite\n",
      "C18064  Pyropheophorbide a\n",
      "C18098  Primary fluorescent chlorophyll catabolite\n",
      "C18151  7(1)-Hydroxychlorophyll a\n",
      "C18152  3-Vinylbacteriochlorophyllide a\n",
      "C18153  3-Hydroxyethylbacteriochlorophyllide a\n",
      "C18154  3-Hydroxyethylchlorophyllide a\n",
      "C18155  Bacteriochlorophyllide a\n",
      "C18156  Bacteriochlorophyllide b\n",
      "C18157  8-Ethyl-12-methyl-3-vinylbacteriochlorophyllide d\n",
      "C18160  Bacteriochlorophyllide d\n",
      "C18161  Bacteriochlorophyllide c\n",
      "C18162  Bacteriochlorophyll d\n",
      "C18163  Bacteriochlorophyll c\n",
      "C19608  Ferritin\n",
      "C20666  5-Oxo-delta-bilirubin\n",
      "C21217  Geranylgeranyl bacteriochlorophyllide a\n",
      "C21284  Fe-coproporphyrin III\n",
      "C21427  8,12-Diethyl-3-vinylbacteriochlorophyllide d\n",
      "C21428  12-Ethyl-8-propyl-3-vinylbacteriochlorophyllide d\n",
      "C21429  12-Ethyl-8-isobutyl-3-vinylbacteriochlorophyllide d\n",
      "C21431  8-Ethyl-12-methylbacteriochlorophyllide d\n",
      "C21432  8,12-Diethylbacteriochlorophyllide d\n",
      "C21433  12-Ethyl-8-propylbacteriochlorophyllide d\n",
      "C21434  12-Ethyl-8-isobutylbacteriochlorophyllide d\n",
      "C21435  Geranylgeranyl bacteriochlorophyllide b\n",
      "C21510  Nickel-sirohydrochlorin\n",
      "C21511  Nickel-sirohydrochlorin a,c-diamide\n",
      "C21512  15,17(3)-Seco-F430-17(3)-acid\n",
      "C21582  Bacteriochlorophyllide e\n",
      "C21723  Biliverdin-IX-delta\n",
      "C21724  Biliverdin-IX-beta\n",
      "C21835  12,18-Didecarboxysiroheme\n",
      "C22338  Dihydro-heme d1\n",
      "C22339  Heme d1\n",
      "REFERENCE   PMID:9461500\n",
      "AUTHORS   Woodcock SC, Raux E, Levillayer F, Thermes C, Rambach A, Warren MJ.\n",
      "TITLE     Effect of mutations in the transmethylase and dehydrogenase/chelatase domains of sirohaem synthase (CysG) on sirohaem and cobalamin biosynthesis.\n",
      "JOURNAL   Biochem J 330:121-9 (1998)\n",
      "DOI:10.1042/bj3300121\n",
      "REFERENCE   PMID:10217486\n",
      "AUTHORS   Johansson P, Hederstedt L\n",
      "TITLE     Organization of genes for tetrapyrrole biosynthesis in gram--positive bacteria.\n",
      "JOURNAL   Microbiology 145 ( Pt 3):529-38 (1999)\n",
      "DOI:10.1099/13500872-145-3-529\n",
      "REFERENCE   PMID:10518530\n",
      "AUTHORS   Maggio-Hall LA, Escalante-Semerena JC.\n",
      "TITLE     In vitro synthesis of the nucleotide loop of cobalamin by Salmonella typhimurium enzymes.\n",
      "JOURNAL   Proc Natl Acad Sci USA 96:11798-803 (1999)\n",
      "DOI:10.1073/pnas.96.21.11798\n",
      "REFERENCE   PMID:11007789\n",
      "AUTHORS   Roper JM, Raux E, Brindley AA, Schubert HL, Gharbia SE, Shah HN, Warren MJ.\n",
      "TITLE     The enigma of cobalamin (Vitamin B12) biosynthesis in Porphyromonas gingivalis. Identification and characterization of a functional corrin pathway.\n",
      "JOURNAL   J Biol Chem 275:40316-23 (2000)\n",
      "DOI:10.1074/jbc.M007146200\n",
      "REFERENCE   PMID:12196148\n",
      "AUTHORS   Scott AI, Roessner CA.\n",
      "TITLE     Biosynthesis of cobalamin (vitamin B(12)).\n",
      "JOURNAL   Biochem Soc Trans 30:613-20 (2002)\n",
      "DOI:10.1042/bst0300613\n",
      "REFERENCE   PMID:12195810\n",
      "AUTHORS   Warren MJ, Raux E, Schubert HL, Escalante-Semerena JC.\n",
      "TITLE     The biosynthesis of adenosylcobalamin (vitamin B12).\n",
      "JOURNAL   Nat Prod Rep 19:390-412 (2002)\n",
      "DOI:10.1039/b108967f\n",
      "REFERENCE   PMID:12055304\n",
      "AUTHORS   Roessner CA, Huang KX, Warren MJ, Raux E, Scott AI.\n",
      "TITLE     Isolation and characterization of 14 additional genes specifying the anaerobic biosynthesis of cobalamin (vitamin B12) in Propionibacterium freudenreichii (P. shermanii).\n",
      "JOURNAL   Microbiology 148:1845-53 (2002)\n",
      "DOI:10.1099/00221287-148-6-1845\n",
      "REFERENCE   PMID:11283349\n",
      "AUTHORS   Frankenberg N, Mukougawa K, Kohchi T, Lagarias JC.\n",
      "TITLE     Functional genomic analysis of the HY2 family of ferredoxin-dependent bilin reductases from oxygenic photosynthetic organisms.\n",
      "JOURNAL   Plant Cell 13:965-78 (2001)\n",
      "DOI:10.1105/tpc.13.4.965\n",
      "REFERENCE   PMID:17506685\n",
      "AUTHORS   Chew AG, Bryant DA\n",
      "TITLE     Chlorophyll biosynthesis in bacteria: the origins of structural and functional diversity.\n",
      "JOURNAL   Annu Rev Microbiol 61:113-29 (2007)\n",
      "DOI:10.1146/annurev.micro.61.080706.093242\n",
      "REFERENCE   PMID:16245042\n",
      "AUTHORS   Frigaard NU, Chew AG, Li H, Maresca JA, Bryant DA\n",
      "TITLE     Chlorobium tepidum: insights into the structure, physiology, and metabolism of a green sulfur bacterium derived from the complete genome sequence.\n",
      "JOURNAL   Photosynth Res 78:93-117 (2003)\n",
      "DOI:10.1023/B:PRES.0000004310.96189.b4\n",
      "REFERENCE   PMID:23042036\n",
      "AUTHORS   Deery E, Schroeder S, Lawrence AD, Taylor SL, Seyedarabi A, Waterman J, Wilson KS, Brown D, Geeves MA, Howard MJ, Pickersgill RW, Warren MJ\n",
      "TITLE     An enzyme-trap approach allows isolation of intermediates in cobalamin biosynthesis.\n",
      "JOURNAL   Nat Chem Biol 8:933-40 (2012)\n",
      "DOI:10.1038/nchembio.1086\n",
      "REFERENCE   PMID:23922391\n",
      "AUTHORS   Moore SJ, Lawrence AD, Biedendieck R, Deery E, Frank S, Howard MJ, Rigby SE, Warren MJ\n",
      "TITLE     Elucidation of the anaerobic pathway for the corrin component of cobalamin (vitamin B12).\n",
      "JOURNAL   Proc Natl Acad Sci U S A 110:14906-11 (2013)\n",
      "DOI:10.1073/pnas.1308098110\n",
      "REL_PATHWAY ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00740  Riboflavin metabolism\n",
      "KO_PATHWAY  ko00860\n",
      "///\n",
      "ENTRY       ath00900                    Pathway\n",
      "NAME        Terpenoid backbone biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Terpenoids, also known as isoprenoids, are a large class of natural products consisting of isoprene (C5) units. There are two biosynthetic pathways, the mevalonate pathway [MD:M00095] and the non-mevalonate pathway or the MEP/DOXP pathway [MD:M00096], for the terpenoid building blocks: isopentenyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP). The action of prenyltransferases then generates higher-order building blocks: geranyl diphosphate (GPP), farsenyl diphosphate (FPP), and geranylgeranyl diphosphate (GGPP), which are the precursors of monoterpenoids (C10), sesquiterpenoids (C15), and diterpenoids (C20), respectively. Condensation of these building blocks gives rise to the precursors of sterols (C30) and carotenoids (C40). The MEP/DOXP pathway is absent in higher animals and fungi, but in green plants the MEP/DOXP and mevalonate pathways co-exist in separate cellular compartments. The MEP/DOXP pathway, operating in the plastids, is responsible for the formation of essential oil monoterpenes and linalyl acetate, some sesquiterpenes, diterpenes, and carotenoids and phytol. The mevalonate pathway, operating in the cytosol, gives rise to triterpenes, sterols, and most sesquiterpenes.\n",
      "CLASS       Metabolism; Metabolism of terpenoids and polyketides\n",
      "PATHWAY_MAP ath00900  Terpenoid backbone biosynthesis\n",
      "MODULE      ath_M00095  C5 isoprenoid biosynthesis, mevalonate pathway [PATH:ath00900]\n",
      "ath_M00096  C5 isoprenoid biosynthesis, non-mevalonate pathway [PATH:ath00900]\n",
      "ath_M00366  C10-C20 isoprenoid biosynthesis, plants [PATH:ath00900]\n",
      "DBLINKS     GO: 0009240 0016091\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G21500  DXPS1; 1-deoxy-D-xylulose 5-phosphate synthase 1 [KO:K01662] [EC:2.2.1.7]\n",
      "AT4G15560  CLA1; Deoxyxylulose-5-phosphate synthase [KO:K01662] [EC:2.2.1.7]\n",
      "AT5G11380  DXPS3; 1-deoxy-D-xylulose 5-phosphate synthase 3 [KO:K01662] [EC:2.2.1.7]\n",
      "AT5G62790  DXR; 1-deoxy-D-xylulose 5-phosphate reductoisomerase [KO:K00099] [EC:1.1.1.267]\n",
      "AT2G02500  ISPD; Nucleotide-diphospho-sugar transferases superfamily protein [KO:K00991] [EC:2.7.7.60]\n",
      "AT2G26930  CDPMEK; 4-(cytidine 5'-phospho)-2-C-methyl-D-erithritol kinase [KO:K00919] [EC:2.7.1.148]\n",
      "AT1G63970  ISPF; isoprenoid F [KO:K01770] [EC:4.6.1.12]\n",
      "AT5G60600  HDS; 4-hydroxy-3-methylbut-2-enyl diphosphate synthase [KO:K03526] [EC:1.17.7.1 1.17.7.3]\n",
      "AT4G34350  HDR; 4-hydroxy-3-methylbut-2-enyl diphosphate reductase [KO:K03527] [EC:1.17.7.4]\n",
      "AT5G48230  ACAT2; acetoacetyl-CoA thiolase 2 [KO:K00626] [EC:2.3.1.9]\n",
      "AT5G47720  AACT1; Thiolase family protein [KO:K00626] [EC:2.3.1.9]\n",
      "AT4G11820  HMGS; hydroxymethylglutaryl-CoA synthase / HMG-CoA synthase / 3-hydroxy-3-methylglutaryl coenzyme A synthase [KO:K01641] [EC:2.3.3.10]\n",
      "AT2G17370  HMG2; 3-hydroxy-3-methylglutaryl-CoA reductase 2 [KO:K00021] [EC:1.1.1.34]\n",
      "AT1G76490  HMG1; hydroxy methylglutaryl CoA reductase 1 [KO:K00021] [EC:1.1.1.34]\n",
      "AT5G27450  MK; mevalonate kinase [KO:K00869] [EC:2.7.1.36]\n",
      "AT1G31910  GHMP kinase family protein [KO:K00938] [EC:2.7.4.2]\n",
      "AT3G54250  GHMP kinase family protein [KO:K01597] [EC:4.1.1.33]\n",
      "AT2G38700  MVD1; mevalonate diphosphate decarboxylase 1 [KO:K01597] [EC:4.1.1.33]\n",
      "AT1G26640  Amino acid kinase family protein [KO:K06981] [EC:2.7.4.26]\n",
      "AT3G02780  IPP2; isopentenyl pyrophosphate:dimethylallyl pyrophosphate isomerase 2 [KO:K01823] [EC:5.3.3.2]\n",
      "AT5G16440  IPP1; isopentenyl diphosphate isomerase 1 [KO:K01823] [EC:5.3.3.2]\n",
      "AT5G47770  FPS1; farnesyl diphosphate synthase 1 [KO:K00787] [EC:2.5.1.1 2.5.1.10]\n",
      "AT4G17190  FPS2; farnesyl diphosphate synthase 2 [KO:K00787] [EC:2.5.1.1 2.5.1.10]\n",
      "AT2G34630  GPS1; geranyl diphosphate synthase 1 [KO:K14066] [EC:2.5.1.1]\n",
      "AT4G38460  GGR; geranylgeranyl reductase [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT1G49530  GGPS6; geranylgeranyl pyrophosphate synthase 6 [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT2G18620  Terpenoid synthases superfamily protein [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT2G18640  GGPS4; geranylgeranyl pyrophosphate synthase 4 [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT2G23800  GGPS2; geranylgeranyl pyrophosphate synthase 2 [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT3G14510  Polyprenyl synthetase family protein [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT3G14530  Terpenoid synthases superfamily protein [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT3G14550  GGPS3; geranylgeranyl pyrophosphate synthase 3 [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT3G20160  Terpenoid synthases superfamily protein [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT3G29430  Terpenoid synthases superfamily protein [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT3G32040  Terpenoid synthases superfamily protein [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT4G36810  GGPS1; geranylgeranyl pyrophosphate synthase 1 [KO:K13789] [EC:2.5.1.1 2.5.1.10 2.5.1.29]\n",
      "AT1G78510  SPS1; solanesyl diphosphate synthase 1 [KO:K05356] [EC:2.5.1.84 2.5.1.85]\n",
      "AT1G17050  SPS2; solanesyl diphosphate synthase 2 [KO:K05356] [EC:2.5.1.84 2.5.1.85]\n",
      "AT5G58770  cPT4; Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT5G58780  cPT5; Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT5G58782  cPT6; Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT5G58784  Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT2G17570  cPT1; Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT5G60510  cPT9; Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT1G11755  LEW1; Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT5G60500  cPT8; Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT2G23400  Undecaprenyl pyrophosphate synthetase family protein [KO:K11778] [EC:2.5.1.87]\n",
      "AT2G23410  CPT; cis-prenyltransferase [KO:K11778] [EC:2.5.1.87]\n",
      "AT1G74470  Pyridine nucleotide-disulfide oxidoreductase family protein [KO:K10960] [EC:1.3.1.83 1.3.1.111]\n",
      "AT3G59380  FTA; farnesyltransferase A [KO:K05955] [EC:2.5.1.58 2.5.1.59]\n",
      "AT5G40280  ERA1; Prenyltransferase family protein [KO:K05954] [EC:2.5.1.58]\n",
      "AT2G36305  FACE2; farnesylated protein-converting enzyme 2 [KO:K08658] [EC:3.4.22.-]\n",
      "AT4G01320  ATSTE24; Peptidase family M48 family protein [KO:K06013] [EC:3.4.24.84]\n",
      "AT5G23320  STE14A; S-isoprenylcysteine O-methyltransferase [KO:K00587] [EC:2.1.1.100]\n",
      "AT5G08335  ATSTE14B; Isoprenylcysteine carboxyl methyltransferase (ICMT) family [KO:K00587] [EC:2.1.1.100]\n",
      "AT5G15860  PCME; prenylcysteine methylesterase [KO:K15889] [EC:3.1.1.-]\n",
      "AT3G02410  ICME-LIKE2; alpha/beta-Hydrolases superfamily protein [KO:K15889] [EC:3.1.1.-]\n",
      "AT1G26120  ICME-LIKE1; alpha/beta-Hydrolases superfamily protein [KO:K15889] [EC:3.1.1.-]\n",
      "AT5G63910  FCLY; farnesylcysteine lyase [KO:K05906] [EC:1.8.3.5 1.8.3.6]\n",
      "AT4G33360  FLDH; NAD(P)-binding Rossmann-fold superfamily protein [KO:K15891] [EC:1.1.1.354]\n",
      "AT5G58560  FOLK; Phosphatidate cytidylyltransferase family protein [KO:K15892] [EC:2.7.1.216]\n",
      "COMPOUND    C00022  Pyruvate\n",
      "C00024  Acetyl-CoA\n",
      "C00118  D-Glyceraldehyde 3-phosphate\n",
      "C00129  Isopentenyl diphosphate\n",
      "C00235  Dimethylallyl diphosphate\n",
      "C00332  Acetoacetyl-CoA\n",
      "C00341  Geranyl diphosphate\n",
      "C00353  Geranylgeranyl diphosphate\n",
      "C00356  (S)-3-Hydroxy-3-methylglutaryl-CoA\n",
      "C00418  (R)-Mevalonate\n",
      "C00448  trans,trans-Farnesyl diphosphate\n",
      "C01107  (R)-5-Phosphomevalonate\n",
      "C01126  (2E,6E)-Farnesol\n",
      "C01143  (R)-5-Diphosphomevalonate\n",
      "C01230  all-trans-Hexaprenyl diphosphate\n",
      "C02321  cis-1,4-Polyisoprene\n",
      "C03461  2-trans,6-trans-Farnesal\n",
      "C04145  all-trans-Nonaprenyl diphosphate\n",
      "C04146  all-trans-Octaprenyl diphosphate\n",
      "C04216  all-trans-Heptaprenyl diphosphate\n",
      "C04506  Protein C-terminal S-farnesyl-L-cysteine\n",
      "C04574  di-trans,poly-cis-Undecaprenyl diphosphate\n",
      "C04748  Protein C-terminal S-farnesyl-L-cysteine methyl ester\n",
      "C05427  Phytyl diphosphate\n",
      "C05859  Dehydrodolichol diphosphate\n",
      "C06081  Polyprenol\n",
      "C11434  2-C-Methyl-D-erythritol 4-phosphate\n",
      "C11435  4-(Cytidine 5'-diphospho)-2-C-methyl-D-erythritol\n",
      "C11436  2-Phospho-4-(cytidine 5'-diphospho)-2-C-methyl-D-erythritol\n",
      "C11437  1-Deoxy-D-xylulose 5-phosphate\n",
      "C11453  2-C-Methyl-D-erythritol 2,4-cyclodiphosphate\n",
      "C11811  1-Hydroxy-2-methyl-2-butenyl 4-diphosphate\n",
      "C16521  Isoprene\n",
      "C16826  2-cis,6-trans-Farnesyl diphosphate\n",
      "C17432  all-trans-Decaprenyl diphosphate\n",
      "C18321  trans,octacis-Decaprenyl diphosphate\n",
      "C19691  Farnesylcysteine\n",
      "C19760  (2Z,6Z)-Farnesyl diphosphate\n",
      "C19852  tritrans,heptacis-Undecaprenyl diphosphate\n",
      "C19853  trans,polycis-Polyprenyl diphosphate\n",
      "C20120  S-Farnesyl protein\n",
      "C20121  trans,trans-Farnesyl phosphate\n",
      "C20345  Isopentenyl phosphate\n",
      "C20847  (R)-Mevalonate 3-phosphate\n",
      "C20848  (R)-Mevalonate 3,5-bisphosphate\n",
      "REFERENCE   PMID:12777052\n",
      "AUTHORS   Lange BM, Ghassemian M.\n",
      "TITLE     Genome organization in Arabidopsis thaliana: a survey for genes involved in isoprenoid and chlorophyll metabolism.\n",
      "JOURNAL   Plant Mol Biol 51:925-48 (2003)\n",
      "DOI:10.1023/A:1023005504702\n",
      "REFERENCE   PMID:16262699\n",
      "AUTHORS   Saiki R, Nagata A, Kainou T, Matsuda H, Kawamukai M\n",
      "TITLE     Characterization of solanesyl and decaprenyl diphosphate synthases in mice and humans.\n",
      "JOURNAL   FEBS J 272:5606-22 (2005)\n",
      "DOI:10.1111/j.1742-4658.2005.04956.x\n",
      "REFERENCE   PMID:9858571\n",
      "AUTHORS   Sato M, Sato K, Nishikawa S, Hirata A, Kato J, Nakano A\n",
      "TITLE     The yeast RER2 gene, identified by endoplasmic reticulum protein localization mutations, encodes cis-prenyltransferase, a key enzyme in dolichol synthesis.\n",
      "JOURNAL   Mol Cell Biol 19:471-83 (1999)\n",
      "DOI:10.1128/MCB.19.1.471\n",
      "REFERENCE   PMID:24375100\n",
      "AUTHORS   Vannice JC, Skaff DA, Keightley A, Addo JK, Wyckoff GJ, Miziorko HM\n",
      "TITLE     Identification in Haloferax volcanii of phosphomevalonate decarboxylase and isopentenyl phosphate kinase as catalysts of the terminal enzyme reactions in an  archaeal alternate mevalonate pathway.\n",
      "JOURNAL   J Bacteriol 196:1055-63 (2014)\n",
      "DOI:10.1128/JB.01230-13\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00100  Steroid biosynthesis\n",
      "ath00130  Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "ath00510  N-Glycan biosynthesis\n",
      "ath00902  Monoterpenoid biosynthesis\n",
      "ath00904  Diterpenoid biosynthesis\n",
      "ath00906  Carotenoid biosynthesis\n",
      "ath00908  Zeatin biosynthesis\n",
      "ath00909  Sesquiterpenoid and triterpenoid biosynthesis\n",
      "KO_PATHWAY  ko00900\n",
      "///\n",
      "ENTRY       ath00901                    Pathway\n",
      "NAME        Indole alkaloid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00901  Indole alkaloid biosynthesis\n",
      "DBLINKS     GO: 0009709\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G20340  AAS; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01593] [EC:4.1.1.28 4.1.1.105]\n",
      "AT1G74010  Calcium-dependent phosphotriesterase superfamily protein [KO:K01757] [EC:4.3.3.2]\n",
      "AT1G74020  SS2; strictosidine synthase 2 [KO:K01757] [EC:4.3.3.2]\n",
      "AT1G74000  SS3; strictosidine synthase 3 [KO:K01757] [EC:4.3.3.2]\n",
      "COMPOUND    C00031  D-Glucose\n",
      "C00078  L-Tryptophan\n",
      "C00132  Methanol\n",
      "C00398  Tryptamine\n",
      "C01091  Deacetylvindoline\n",
      "C01626  Vindoline\n",
      "C01761  Vomilenine\n",
      "C01852  Secologanin\n",
      "C02074  Raucaffricine\n",
      "C02151  Geissoschizine\n",
      "C02673  Desacetoxyvindoline\n",
      "C03309  Strictosidine aglycone\n",
      "C03470  3-alpha(S)-Strictosidine\n",
      "C03677  4,21-Dehydrogeissoschizine\n",
      "C04290  4-(3-Methylbut-2-enyl)-L-tryptophan\n",
      "C04578  16-Methoxy-2,3-dihydro-3-hydroxytabersonine\n",
      "C06542  Ajmaline\n",
      "C07201  Vinblastine\n",
      "C07204  Vincristine\n",
      "C07576  Psilocybin\n",
      "C08312  Psilocin\n",
      "C09023  Agroclavine\n",
      "C09024  Ajmalicine\n",
      "C09028  Alstonine\n",
      "C09107  Catharanthine\n",
      "C09131  Chanoclavine-I\n",
      "C09239  Sarpagine\n",
      "C09241  Serpentine\n",
      "C09244  Tabersonine\n",
      "C11632  Polyneuridine aldehyde\n",
      "C11633  16-Epivellosimine\n",
      "C11634  Vellosimine\n",
      "C11635  10-Deoxysarpagine\n",
      "C11641  3',4'-Anhydrovinblastine\n",
      "C11642  19-Hydroxytabersonine\n",
      "C11643  16-Hydroxytabersonine\n",
      "C11675  16-Methoxytabersonine\n",
      "C11676  Lochnericine\n",
      "C11677  Horhammericine\n",
      "C11678  Dialdehyde\n",
      "C11679  4,21-Dehydrocorynantheine aldehyde\n",
      "C11680  Cathenamine\n",
      "C11681  19-epi-Cathenamine\n",
      "C11682  Tetrahydroalstonine\n",
      "C11683  19-epi-Ajmalicine\n",
      "C11691  Stemmadenine\n",
      "C11783  Minovincinine\n",
      "C11784  Echitovenine\n",
      "C11807  Vinorine\n",
      "C11808  1,2-Dihydrovomilenine\n",
      "C11809  17-O-Acetylnorajmaline\n",
      "C11810  Norajmaline\n",
      "C11812  Lochnerinine\n",
      "C15985  17-O-Acetylajmaline\n",
      "C20379  Chanoclavine-I aldehyde\n",
      "C20408  Festuclavine\n",
      "C20409  6,8-Dimethyl-6,7-didehydroergoline\n",
      "C20410  4-Dimethylallyl-L-abrine\n",
      "C20436  Fumigaclavine A\n",
      "C20437  Fumigaclavine B\n",
      "C20438  Fumigaclavine C\n",
      "C20457  6,8-Dimethyl-6,7,8,9-tetradehydroergoline\n",
      "C21678  (3R)-3-Hydroxy-16-methoxy-1,2-didehydro-2,3-dihydrotabersonine\n",
      "C21682  (3R)-3-Hydroxy-2,3-dihydrotabersonine\n",
      "C21683  (3R)-3-Hydroxy-1,2-didehydro-2,3-dihydrotabersonine\n",
      "C21762  4-Hydroxytryptamine\n",
      "C21777  4-Hydroxy-L-tryptophan\n",
      "C21778  Norbaeocystin\n",
      "C21779  Baeocystin\n",
      "C21897  Dihydrochanoclavine-I aldehyde\n",
      "REFERENCE   PMID:9622525\n",
      "AUTHORS   Rijhwani SK, Shanks JV.\n",
      "TITLE     Effect of elicitor dosage and exposure time on biosynthesis of indole alkaloids by Catharanthus roseus hairy root cultures.\n",
      "JOURNAL   Biotechnol Prog 14:442-9 (1998)\n",
      "DOI:10.1021/bp980029v\n",
      "REFERENCE   PMID:4698228\n",
      "AUTHORS   Madyastha KM, Guarnaccia R, Baxter C, Coscia CJ.\n",
      "TITLE     S-Adenosyl-L-methionine: loganic acid methyltransferase. A carboxyl-alkylating enzyme from Vinca rosea.\n",
      "JOURNAL   J Biol Chem 248:2497-501 (1973)\n",
      "REFERENCE   PMID:510306\n",
      "AUTHORS   Treimer JF, Zenk MH.\n",
      "TITLE     Purification and properties of strictosidine synthase, the key enzyme in indole alkaloid formation.\n",
      "JOURNAL   Eur J Biochem 101:225-33 (1979)\n",
      "DOI:10.1111/j.1432-1033.1979.tb04235.x\n",
      "REFERENCE   PMID:10652285\n",
      "AUTHORS   Geerlings A, Ibanez MM, Memelink J, van Der Heijden R, Verpoorte R.\n",
      "TITLE     Molecular cloning and analysis of strictosidine beta-D-glucosidase, an enzyme in terpenoid indole alkaloid biosynthesis in Catharanthus roseus.\n",
      "JOURNAL   J Biol Chem 275:3051-6 (2000)\n",
      "DOI:10.1074/jbc.275.5.3051\n",
      "REFERENCE   PMID:11128\n",
      "AUTHORS   Stockigt J, Treimer J, Zenk MH.\n",
      "TITLE     Synthesis of ajmalicine and related indole alkaloids by cell free extracts of Catharanthus roseus cell suspension cultures.\n",
      "JOURNAL   FEBS Lett 70:267-70 (1976)\n",
      "DOI:10.1016/0014-5793(76)80772-7\n",
      "REFERENCE   PMID:12228585\n",
      "AUTHORS   St-Pierre B, De Luca V.\n",
      "TITLE     A Cytochrome P-450 Monooxygenase Catalyzes the First Step in the Conversion of Tabersonine to Vindoline in Catharanthus roseus.\n",
      "JOURNAL   Plant Physiol 109:131-139 (1995)\n",
      "DOI:10.1104/pp.109.1.131\n",
      "REFERENCE   PMID:10481044\n",
      "AUTHORS   Schroder G, Unterbusch E, Kaltenbach M, Schmidt J, Strack D, De Luca V, Schroder J.\n",
      "TITLE     Light-induced cytochrome P450-dependent enzyme in indole alkaloid biosynthesis: tabersonine 16-hydroxylase.\n",
      "JOURNAL   FEBS Lett 458:97-102 (1999)\n",
      "DOI:10.1016/S0014-5793(99)01138-2\n",
      "REFERENCE   PMID:9654153\n",
      "AUTHORS   Sottomayor M, Lopez-Serrano M, DiCosmo F, Ros Barcelo A.\n",
      "TITLE     Purification and characterization of alpha-3',4'-anhydrovinblastine synthase (peroxidase-like) from Catharanthus roseus (L.) G. Don.\n",
      "JOURNAL   FEBS Lett 428:299-303 (1998)\n",
      "DOI:10.1016/S0014-5793(98)00551-1\n",
      "REFERENCE   PMID:1368411\n",
      "AUTHORS   Furuya T, Sakamoto K, Iida K, Asada Y, Yoshikawa T, Sakai S, Aimi N.\n",
      "TITLE     Biotransformation of tabersonine in cell suspension cultures of Catharanthus roseus.\n",
      "JOURNAL   Phytochemistry 31:3065-8 (1992)\n",
      "DOI:10.1016/0031-9422(92)83447-7\n",
      "REFERENCE   PMID:11154328\n",
      "AUTHORS   Laflamme P, St-Pierre B, De Luca V.\n",
      "TITLE     Molecular and biochemical analysis of a madagascar periwinkle root-specific minovincinine-19-hydroxy-o-acetyltransferase.\n",
      "JOURNAL   Plant Physiol 125:189-98 (2001)\n",
      "DOI:10.1104/pp.125.1.189\n",
      "REFERENCE   PMID:12150812\n",
      "AUTHORS   Jain S, Sinha A, Bhakuni DS.\n",
      "TITLE     The biosynthesis of beta-carboline and quinolizidine alkaloids of Alangium lamarckii.\n",
      "JOURNAL   Phytochemistry 60:853-9 (2002)\n",
      "DOI:10.1016/S0031-9422(02)00057-2\n",
      "REFERENCE   PMID:28763571\n",
      "AUTHORS   Fricke J, Blei F, Hoffmeister D\n",
      "TITLE     Enzymatic Synthesis of Psilocybin.\n",
      "JOURNAL   Angew Chem Int Ed Engl 56:12352-12355 (2017)\n",
      "DOI:10.1002/anie.201705489\n",
      "REL_PATHWAY ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00902  Monoterpenoid biosynthesis\n",
      "KO_PATHWAY  ko00901\n",
      "///\n",
      "ENTRY       ath00902                    Pathway\n",
      "NAME        Monoterpenoid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Monoterpenoids (C10 terpenoids) are a group of terpenoids consisting of two isoprene units. They are derived from geranyl diphosphate (GPP). Most monoterpenoids are volatile oils with highly distinctive aromas and flavors, such as essential oils, turpentine, and oleoresins of coniferous plants. This map shows some examples. The monoterpene ketone l-menthone is specifically converted to l-menthol and d-neomenthol in mature peppermint leaves. The iridoids constitute a family of highly oxygenated monoterpenes, mixtures of which are present in many medicinal plants, such as valerian. They are derived from geraniol or nerol via oxidation of a terminal methyl group. The cyclopentane ring of loganin can itself be cleaved in a further P450-dependent step, leading to secologanin, which provides the carbon skeleton for many powerfully bioactive secondary metabolites of indole alkaloids.\n",
      "CLASS       Metabolism; Metabolism of terpenoids and polyketides\n",
      "PATHWAY_MAP ath00902  Monoterpenoid biosynthesis\n",
      "DBLINKS     GO: 0016099\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G24210  TPS10; terpene synthase 10 [KO:K12467] [EC:4.2.3.15]\n",
      "AT3G25810  Terpenoid cyclases/Protein prenyltransferases superfamily protein [KO:K12467] [EC:4.2.3.15]\n",
      "AT1G61680  TPS14; terpene synthase 14 [KO:K15086] [EC:4.2.3.25]\n",
      "AT3G61220  SDR1; NAD(P)-binding Rossmann-fold superfamily protein [KO:K15095] [EC:1.1.1.208]\n",
      "AT2G24190  SDR2; NAD(P)-binding Rossmann-fold superfamily protein [KO:K15095] [EC:1.1.1.208]\n",
      "AT3G25830  TPS-CIN; terpene synthase-like sequence-1,8-cineole [KO:K07385] [EC:4.2.3.108]\n",
      "AT3G25820  TPS-CIN; terpene synthase-like sequence-1,8-cineole [KO:K07385] [EC:4.2.3.108]\n",
      "COMPOUND    C00341  Geranyl diphosphate\n",
      "C00400  (-)-Menthol\n",
      "C00521  (S)-Limonene\n",
      "C00553  (+)-Neomenthol\n",
      "C00808  (+)-Camphor\n",
      "C00843  (-)-Menthone\n",
      "C00848  6-Oxocineole\n",
      "C00964  (-)-trans-Carveol\n",
      "C01123  (-)-trans-Isopiperitenol\n",
      "C01433  Loganin\n",
      "C01500  Geraniol\n",
      "C01512  Loganate\n",
      "C01765  (+)-Borneol\n",
      "C01767  (-)-Carvone\n",
      "C01852  Secologanin\n",
      "C01957  Secologanate\n",
      "C02344  (-)-endo-Fenchol\n",
      "C02452  (-)-Perillyl alcohol\n",
      "C02462  Sabinene hydrate\n",
      "C02485  (-)-Isopiperitenone\n",
      "C03092  6-endo-Hydroxycineole\n",
      "C03190  (+)-Bornyl diphosphate\n",
      "C04433  (E)-3,7-Dimethylocta-1,6-diene-3,8-diol\n",
      "C04718  1,6,6-Trimethyl-2,7-dioxabicyclo[3.2.2]nonan-3-one\n",
      "C06066  4,5-Dihydro-5,5-dimethyl-4-(3-oxobutyl)furan-2(3H)-one\n",
      "C06070  Iridotrial\n",
      "C06071  Deoxyloganin\n",
      "C06074  Myrcene\n",
      "C06099  (R)-Limonene\n",
      "C06304  (+)-Camphene\n",
      "C06305  (-)-Camphene\n",
      "C06307  (-)-beta-Pinene\n",
      "C06308  (-)-alpha-Pinene\n",
      "C09769  Asperuloside\n",
      "C09804  cis-trans-Nepetalactol\n",
      "C09844  1,8-Cineole\n",
      "C09893  Pulegone\n",
      "C11383  (+)-(S)-Carvone\n",
      "C11388  (-)-Linalool\n",
      "C11389  (+)-Linalool\n",
      "C11393  (-)-alpha-Terpineol\n",
      "C11409  (+)-trans-Carveol\n",
      "C11636  7-Deoxyloganate\n",
      "C11672  10-Deoxygeniposidic acid\n",
      "C11673  Geniposidic acid\n",
      "C11951  (+)-cis-Isopulegone\n",
      "C11952  (+)-Isomenthone\n",
      "C17621  (6E)-8-Hydroxygeraniol\n",
      "C17622  (6E)-8-Oxogeranial\n",
      "C18025  (+)-Menthofuran\n",
      "C18027  (4R,7S)-7-Isopropyl-4-methyloxepan-2-one\n",
      "C20221  (6E)-8-Oxolinalool\n",
      "C20789  7-Deoxyloganetate\n",
      "C20790  7-Deoxyloganetin\n",
      "C20943  (R)-Ipsdienol\n",
      "C20944  Ipsdienone\n",
      "C21203  Geranyl phosphate\n",
      "C22014  7-Deoxyloganetic alcohol\n",
      "C22225  (S)-8-Oxocitronellyl enol\n",
      "REFERENCE   PMID:15728344\n",
      "AUTHORS   Davis EM, Ringer KL, McConkey ME, Croteau R\n",
      "TITLE     Monoterpene metabolism. Cloning, expression, and characterization of menthone reductases from peppermint.\n",
      "JOURNAL   Plant Physiol 137:873-81 (2005)\n",
      "DOI:10.1104/pp.104.053306\n",
      "REFERENCE   PMID:11375201\n",
      "AUTHORS   Duetz WA, Fjallman AH, Ren S, Jourdat C, Witholt B\n",
      "TITLE     Biotransformation of D-limonene to (+) trans-carveol by toluene-grown Rhodococcus opacus PWD4 cells.\n",
      "JOURNAL   Appl Environ Microbiol 67:2829-32 (2001)\n",
      "DOI:10.1128/AEM.67.6.2829-2832.2001\n",
      "REFERENCE   PMID:15299125\n",
      "AUTHORS   Chen F, Ro DK, Petri J, Gershenzon J, Bohlmann J, Pichersky E, Tholl D\n",
      "TITLE     Characterization of a root-specific Arabidopsis terpene synthase responsible for the formation of the volatile monoterpene 1,8-cineole.\n",
      "JOURNAL   Plant Physiol 135:1956-66 (2004)\n",
      "DOI:10.1104/pp.104.044388\n",
      "REFERENCE   PMID:1368150\n",
      "AUTHORS   Trudgill PW\n",
      "TITLE     Microbial metabolism of monoterpenes--recent developments.\n",
      "JOURNAL   Biodegradation 1:93-105 (1990)\n",
      "DOI:10.1007/BF00058829\n",
      "REFERENCE   PMID:1864846\n",
      "AUTHORS   Ikeda H, Esaki N, Nakai S, Hashimoto K, Uesato S, Soda K, Fujita T\n",
      "TITLE     Acyclic monoterpene primary alcohol:NADP+ oxidoreductase of Rauwolfia serpentina cells: the key enzyme in biosynthesis of monoterpene alcohols.\n",
      "JOURNAL   J Biochem 109:341-7 (1991)\n",
      "REFERENCE   PMID:10656401\n",
      "AUTHORS   Yamamoto H, Katano N, Ooi A, Inoue K.\n",
      "TITLE     Secologanin synthase which catalyzes the oxidative cleavage of loganin into secologanin is a cytochrome P450.\n",
      "JOURNAL   Phytochemistry 53:7-12 (2000)\n",
      "DOI:10.1016/S0031-9422(99)00471-9\n",
      "REFERENCE   PMID:11524113\n",
      "AUTHORS   Katano N, Yamamoto H, Iio R, Inoue K.\n",
      "TITLE     7-Deoxyloganin 7-hydroxylase in Lonicera japonica cell cultures.\n",
      "JOURNAL   Phytochemistry 58:53-8 (2001)\n",
      "DOI:10.1016/S0031-9422(01)00181-9\n",
      "REL_PATHWAY ath00900  Terpenoid backbone biosynthesis\n",
      "ath00901  Indole alkaloid biosynthesis\n",
      "ath00903  Limonene and pinene degradation\n",
      "KO_PATHWAY  ko00902\n",
      "///\n",
      "ENTRY       ath00903                    Pathway\n",
      "NAME        Limonene and pinene degradation - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of terpenoids and polyketides\n",
      "PATHWAY_MAP ath00903  Limonene and pinene degradation\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G23800  ALDH2B7; aldehyde dehydrogenase 2B7 [KO:K00128] [EC:1.2.1.3]\n",
      "AT3G48000  ALDH2B4; aldehyde dehydrogenase 2B4 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G34240  ALDH3I1; aldehyde dehydrogenase 3I1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT1G44170  ALDH3H1; aldehyde dehydrogenase 3H1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G36250  ALDH3F1; aldehyde dehydrogenase 3F1 [KO:K00128] [EC:1.2.1.3]\n",
      "AT4G16210  ECHIA; enoyl-CoA hydratase/isomerase A [KO:K01692] [EC:4.2.1.17]\n",
      "COMPOUND    C00521  (S)-Limonene\n",
      "C00964  (-)-trans-Carveol\n",
      "C01767  (-)-Carvone\n",
      "C02452  (-)-Perillyl alcohol\n",
      "C02576  Perillyl aldehyde\n",
      "C02759  alpha-Pinene-oxide\n",
      "C04435  (Z)-2-Methyl-5-isopropylhexa-2,5-dienal\n",
      "C06099  (R)-Limonene\n",
      "C07271  Limonene-1,2-epoxide\n",
      "C07276  Limonene-1,2-diol\n",
      "C09880  alpha-Pinene\n",
      "C09884  Pinocarvone\n",
      "C11383  (+)-(S)-Carvone\n",
      "C11395  (4R,6R)-cis-Carveol\n",
      "C11396  (1R,2R,4R)-Dihydrocarveol\n",
      "C11397  (1R,2S,4R)-Neo-dihydrocarveol\n",
      "C11398  (1R,4R)-Dihydrocarvone\n",
      "C11399  (1S,2S,4R)-Iso-dihydrocarveol\n",
      "C11400  (1S,2R,4R)-Neoiso-dihydrocarveol\n",
      "C11401  (1S,4R)-Iso-dihydrocarvone\n",
      "C11402  (4R,7R)-4-Isopropenyl-7-methyl-2-oxo-oxepanone\n",
      "C11403  (3S,6R)-6-Isopropenyl-3-methyl-2-oxo-oxepanone\n",
      "C11404  (3R)-6-Hydroxy-3-isopropenyl-heptanoate\n",
      "C11405  (3R)-3-Isopropenyl-6-oxoheptanoate\n",
      "C11406  (5R)-6-Hydroxy-5-isopropenyl-2-methylhexanoate\n",
      "C11407  (3R)-3-Isopropenyl-6-oxoheptanoyl-CoA\n",
      "C11408  (4S,6S)-cis-Carveol\n",
      "C11409  (+)-trans-Carveol\n",
      "C11410  (1R,2S,4S)-Neoiso-dihydrocarveol\n",
      "C11411  (1R,2R,4S)-Iso-dihydrocarveol\n",
      "C11412  (1R,4S)-Iso-dihydrocarvone\n",
      "C11413  (1S,2S,4S)-Dihydrocarveol\n",
      "C11414  (4S,7R)-4-Isopropenyl-7-methyl-2-oxo-oxepanone\n",
      "C11415  (1S,4S)-Dihydrocarvone\n",
      "C11416  (1S,2R,4S)-Neo-dihydrocarveol\n",
      "C11417  (3S)-6-Hydroxy-3-isopropenyl-heptanoate\n",
      "C11418  (3S,6S)-6-Isopropenyl-3-methyl-2-oxo-oxepanone\n",
      "C11419  (3S)-3-Isopropenyl-6-oxoheptanoate\n",
      "C11420  (5S)-6-Hydroxy-5-isopropenyl-2-methylhexanoate\n",
      "C11421  (3S)-3-Isopropenyl-6-oxoheptanoyl-CoA\n",
      "C11924  Perillic acid\n",
      "C11929  Perillyl-CoA\n",
      "C11934  2-Hydroxy-4-isopropenylcyclohexane-1-carboxyl-CoA\n",
      "C11935  4-Isopropenyl-2-oxy-cyclohexanecarboxyl-CoA\n",
      "C11936  3-Isopropenylpimelyl-CoA\n",
      "C11937  (1S,4R)-1-Hydroxy-2-oxolimonene\n",
      "C11938  Myrtenol\n",
      "C11939  Myrtenal\n",
      "C11940  Myrtenic acid\n",
      "C11941  Pinocarveol\n",
      "C11942  trans-2-Methyl-5-isopropylhexa-2,5-dienal\n",
      "C11943  trans-2-Methyl-5-isopropylhexa-2,5-dienoic acid\n",
      "C11944  cis-2-Methyl-5-isopropylhexa-2,5-dienoic acid\n",
      "C11945  trans-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA\n",
      "C11946  cis-2-Methyl-5-isopropylhexa-2,5-dienoyl-CoA\n",
      "C11947  3-Hydroxy-2,6-dimethyl-5-methylene-heptanoyl-CoA\n",
      "C11948  2,6-Dimethyl-5-methylene-3-oxo-heptanoyl-CoA\n",
      "C11949  3-Isopropylbut-3-enoyl-CoA\n",
      "C11950  3-Isopropyl-3-butenoic acid\n",
      "C19079  (4R)-7-Hydroxy-4-isopropenyl-7-methyl-2-oxo-oxepanone\n",
      "C19081  (4S)-Limonene-1,2-epoxide\n",
      "C19082  (1R,2R,4S)-Limonene-1,2-diol\n",
      "C19083  (1R,4S)-1-Hydroxy-2-oxolimonene\n",
      "C19084  (4S)-7-Hydroxy-4-isopropenyl-7-methyl-2-oxo-oxepanone\n",
      "REFERENCE   PMID:10224006\n",
      "AUTHORS   van der Werf MJ, Swarts HJ, de Bont JA\n",
      "TITLE     Rhodococcus erythropolis DCL14 contains a novel degradation pathway for limonene.\n",
      "JOURNAL   Appl Environ Microbiol 65:2092-102 (1999)\n",
      "DOI:10.1128/AEM.65.5.2092-2102.1999\n",
      "REFERENCE   PMID:10832640\n",
      "AUTHORS   van der Werf MJ, Boot AM\n",
      "TITLE     Metabolism of carveol and dihydrocarveol in Rhodococcus erythropolis DCL14.\n",
      "JOURNAL   Microbiology 146 ( Pt 5):1129-41 (2000)\n",
      "DOI:10.1099/00221287-146-5-1129\n",
      "REL_PATHWAY ath00902  Monoterpenoid biosynthesis\n",
      "KO_PATHWAY  ko00903\n",
      "///\n",
      "ENTRY       ath00904                    Pathway\n",
      "NAME        Diterpenoid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of terpenoids and polyketides\n",
      "PATHWAY_MAP ath00904  Diterpenoid biosynthesis\n",
      "MODULE      ath_M00927  Gibberellin A12 biosynthesis, GAPP => GA12 [PATH:ath00904]\n",
      "ath_M00928  Gibberellin A4/A1 biosynthesis, GA12 => GA4 / GA53 => GA1 [PATH:ath00904]\n",
      "DBLINKS     GO: 0016102\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G02780  GA1; Terpenoid cyclases/Protein prenyltransferases superfamily protein [KO:K04120] [EC:5.5.1.13]\n",
      "AT1G79460  GA2; Terpenoid cyclases/Protein prenyltransferases superfamily protein [KO:K04121] [EC:4.2.3.19]\n",
      "AT5G25900  GA3; GA requiring 3 [KO:K04122] [EC:1.14.14.86]\n",
      "AT2G32440  KAO2; ent-kaurenoic acid hydroxylase 2 [KO:K04123] [EC:1.14.14.107]\n",
      "AT1G05160  CYP88A3; cytochrome P450, family 88, subfamily A, polypeptide 3 [KO:K04123] [EC:1.14.14.107]\n",
      "AT4G25420  GA20OX1; 2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K05282] [EC:1.14.11.12]\n",
      "AT5G51810  GA20OX2; gibberellin 20 oxidase 2 [KO:K05282] [EC:1.14.11.12]\n",
      "AT5G07200  GA20OX3; gibberellin 20-oxidase 3 [KO:K05282] [EC:1.14.11.12]\n",
      "AT1G60980  GA20OX4; gibberellin 20-oxidase 4 [KO:K05282] [EC:1.14.11.12]\n",
      "AT1G44090  GA20OX5; gibberellin 20-oxidase 5 [KO:K05282] [EC:1.14.11.12]\n",
      "AT1G15550  GA3OX1; gibberellin 3-oxidase 1 [KO:K04124] [EC:1.14.11.15]\n",
      "AT1G80340  GA3OX2; gibberellin 3-oxidase 2 [KO:K04124] [EC:1.14.11.15]\n",
      "AT1G80330  GA3OX4; gibberellin 3-oxidase 4 [KO:K04124] [EC:1.14.11.15]\n",
      "AT4G21690  GA3OX3; gibberellin 3-oxidase 3 [KO:K04124] [EC:1.14.11.15]\n",
      "AT1G78440  ATGA2OX1; gibberellin 2-beta-dioxygenase [KO:K04125] [EC:1.14.11.13]\n",
      "AT2G34555  ATGA2OX3; gibberellin 2-oxidase 3 [KO:K04125] [EC:1.14.11.13]\n",
      "AT1G02400  GA2OX6; gibberellin 2-oxidase 6 [KO:K04125] [EC:1.14.11.13]\n",
      "AT1G30040  GA2OX2; gibberellin 2-oxidase [KO:K04125] [EC:1.14.11.13]\n",
      "AT1G47990  GA2OX4; gibberellin 2-oxidase 4 [KO:K04125] [EC:1.14.11.13]\n",
      "AT1G61120  TPS04; terpene synthase 04 [KO:K17982] [EC:4.2.3.144]\n",
      "AT3G25180  CYP82G1; cytochrome P450, family 82, subfamily G, polypeptide 1 [KO:K17961] [EC:1.14.14.58 1.14.14.59]\n",
      "COMPOUND    C00353  Geranylgeranyl diphosphate\n",
      "C00859  Gibberellin A1\n",
      "C01414  Casbene\n",
      "C01699  Gibberellin A3\n",
      "C02034  Gibberellin A19\n",
      "C02035  Gibberellin A20\n",
      "C03579  Gibberellin A8\n",
      "C06086  Pimaradiene\n",
      "C06087  Abietate\n",
      "C06088  Aphidicolin\n",
      "C06089  ent-Copalyl diphosphate\n",
      "C06090  ent-Kaurene\n",
      "C06091  Aconitine\n",
      "C06092  Veatchine\n",
      "C06093  Gibberellin A12 aldehyde\n",
      "C06094  Gibberellin A53\n",
      "C06095  Gibberellin A44 diacid\n",
      "C06096  Gibberellin A29\n",
      "C07394  Paclitaxel\n",
      "C09092  Ferruginol\n",
      "C09094  Geranylgeraniol\n",
      "C09118  Isopimaric acid\n",
      "C09163  Pisiferic acid\n",
      "C09183  Sclareol\n",
      "C11700  10-Deacetylbaccatin III\n",
      "C11853  Gibberellin A14 aldehyde\n",
      "C11854  Gibberellin A51-catabolite\n",
      "C11855  Gibberellin A29-catabolite\n",
      "C11857  Gibberellin A12\n",
      "C11858  Gibberellin A14\n",
      "C11859  Gibberellin A37 open lactone\n",
      "C11860  Gibberellin A15 open lactone\n",
      "C11861  Gibberellin A24\n",
      "C11862  Gibberellin A36\n",
      "C11863  Gibberellin A9\n",
      "C11864  Gibberellin A4\n",
      "C11865  Gibberellin A51\n",
      "C11866  2,3-Dehydro-gibberellin A9\n",
      "C11867  Gibberellin A7\n",
      "C11868  Gibberellin A34\n",
      "C11869  Gibberellin A34-catabolite\n",
      "C11870  Gibberellin A8-catabolite\n",
      "C11871  Gibberellin A5\n",
      "C11872  Kaur-16-en-18-ol\n",
      "C11873  Kaur-16-en-18-al\n",
      "C11874  Kaur-16-en-18-oic acid\n",
      "C11875  ent-7alpha-Hydroxykaur-16-en-19-oic acid\n",
      "C11876  6beta,7beta-Dihydroxykaurenoic acid\n",
      "C11877  (+)-Sandaracopimaradiene\n",
      "C11878  (-)-Abietadiene\n",
      "C11879  Levopimaradiene\n",
      "C11880  Neoabietadiene\n",
      "C11881  Palustradiene\n",
      "C11882  Abietinol\n",
      "C11883  Levopimarinol\n",
      "C11884  Neoabietinol\n",
      "C11885  Neoabietal\n",
      "C11886  Levopimarinal\n",
      "C11887  Abietal\n",
      "C11888  Levopimaric acid\n",
      "C11889  Neoabietic acid\n",
      "C11890  Aphidicolan-16beta-ol\n",
      "C11893  Cembrene\n",
      "C11894  Taxa-4(5),11(12)-diene\n",
      "C11895  Taxa-4(20),11(12)-dien-5alpha-ol\n",
      "C11896  Taxa-4(20),11(12)-dien-5alpha-yl acetate\n",
      "C11897  Taxa-4(20),11(12)-dien-5alpha,13alpha-diol\n",
      "C11898  Taxa-4(20),11(12)-dien-5alpha-acetoxy-10beta-ol\n",
      "C11899  10-Deacetyl-2-debenzoylbaccatin III\n",
      "C11900  Baccatin III\n",
      "C11901  Copalyl diphosphate\n",
      "C11902  9alpha-Copalyl diphosphate\n",
      "C11905  Gibberellin A53 aldehyde\n",
      "C12077  Palustric acid\n",
      "C13273  Plaunotol\n",
      "C16523  3'-N-Debenzoyl-2'-deoxytaxol\n",
      "C16524  3'-N-Debenzoyltaxol\n",
      "C18015  Momilactone A\n",
      "C18016  3beta-Hydroxy-9beta-pimara-7,15-diene-19,6beta-olide\n",
      "C18211  Isopimara-7,15-diene\n",
      "C18219  Palustradienol\n",
      "C18220  Palustradienal\n",
      "C18221  Isopimaradienol\n",
      "C18222  Isopimaradienal\n",
      "C18223  Stemar-13-ene\n",
      "C18224  Stemod-13(17)-ene\n",
      "C18225  9beta-Pimara-7,15-diene\n",
      "C18226  ent-Cassa-12,15-diene\n",
      "C18228  ent-Pimara-8(14),15-diene\n",
      "C18229  Fusicocca-2,10(14)-diene\n",
      "C19817  Phyllocladan-16alpha-ol\n",
      "C20145  ent-Isokaurene\n",
      "C20146  ent-2alpha-Hydroxyisokaurene\n",
      "C20149  9beta-Pimara-7,15-dien-19-oate\n",
      "C20150  ent-11beta-Hydroxycassa-12,15-diene\n",
      "C20151  10beta,14beta-Dihydroxytaxa-4(20),11-dien-5alpha-yl acetate\n",
      "C20158  (13E)-Labda-7,13-dien-15-ol\n",
      "C20185  (12E)-9alpha-Labda-8(17),12,14-triene\n",
      "C20208  syn-Stemoden-19-oate\n",
      "C20270  Copal-8-ol diphosphate\n",
      "C20481  cis-Abienol\n",
      "C20681  (E,E)-Geranyllinalool\n",
      "C20700  (E,E)-4,8,12-Trimethyltrideca-1,3,7,11-tetraene\n",
      "C20711  Miltiradiene\n",
      "C21425  Abieta-8,11,13-triene\n",
      "C21560  ent-Sandaracopimaradien-3beta-ol\n",
      "C21561  Oryzalexin E\n",
      "C21562  Oryzalexin D\n",
      "C21656  ent-8alpha-Hydroxylabd-13-en-15-yl diphosphate\n",
      "C21709  Nezukol\n",
      "C21714  ent-13-epi-Manoyl oxide\n",
      "C21735  (-)-Kolavenyl diphosphate\n",
      "C21796  11-Hydroxyferruginol\n",
      "C21818  Carnosic acid\n",
      "C21819  Salviol\n",
      "C21822  Sugiol\n",
      "C21823  11-Hydroxysugiol\n",
      "C21824  11,20-Dihydroxysugiol\n",
      "C21825  9beta-Pimara-7,15-diene-3beta-ol\n",
      "C21826  ent-3beta-Hydroxycassa-12,15-dien-2-one\n",
      "C21830  11,20-Dihydroxyferruginol\n",
      "C21831  ent-2alpha,3alpha-Dihydroxyisokaurene\n",
      "REFERENCE   PMID:16137660\n",
      "AUTHORS   Long RM, Croteau R.\n",
      "TITLE     Preliminary assessment of the C13-side chain 2'-hydroxylase involved in taxol biosynthesis.\n",
      "JOURNAL   Biochem Biophys Res Commun 338:410-7 (2005)\n",
      "DOI:10.1016/j.bbrc.2005.08.119\n",
      "REFERENCE   PMID:12089320\n",
      "AUTHORS   Walker K, Long R, Croteau R.\n",
      "TITLE     The final acylation step in taxol biosynthesis: cloning of the taxoid C13-side-chain N-benzoyltransferase from Taxus.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 99:9166-71 (2002)\n",
      "DOI:10.1073/pnas.082115799\n",
      "REL_PATHWAY ath00900  Terpenoid backbone biosynthesis\n",
      "KO_PATHWAY  ko00904\n",
      "///\n",
      "ENTRY       ath00905                    Pathway\n",
      "NAME        Brassinosteroid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Brassinosteroids are a group of plant steroid hormones that regulate growth and development. More than fifty naturally occurring brassinosteroids have been identified in a wide range of plant species. The most abundant and widely occurring brassinosteroids are C28 steroids, and among them brassinolide (BL) is the most biologically active. Plants have multiple pathways for biosynthesis of BL, which are derived from the steroid biosynthetic pathway. Two pathways from campestanol to castasterone (CS), C6 oxidation and the late-C6 oxidation pathways, operate in many plants. Another branching pathway, the early-C22 oxidation pathway, was demonstrated using a brassinosteroid-deficient mutant of Arabidopsis thaliana. Recently, a new shortcut pathway from campesterol to 6-deoxotyphasterol was demonstrated by a functional analysis of cytochrome P450 monooxygenases responsible for brassinosteroid biosynthesis. Thus, at least four pathways are involved in the biosynthesis of CS, and CS is further metabolized to BL by lactonization of the B ring.\n",
      "CLASS       Metabolism; Metabolism of terpenoids and polyketides\n",
      "PATHWAY_MAP ath00905  Brassinosteroid biosynthesis\n",
      "DBLINKS     GO: 0016132\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G50660  DWF4; Cytochrome P450 superfamily protein [KO:K09587] [EC:1.14.13.-]\n",
      "AT5G05690  CPD; Cytochrome P450 superfamily protein [KO:K09588] [EC:1.14.-.-]\n",
      "AT2G38050  DET2; 3-oxo-5-alpha-steroid 4-dehydrogenase family protein [KO:K09591] [EC:1.3.1.22]\n",
      "AT4G36380  ROT3; Cytochrome P450 superfamily protein [KO:K12637] [EC:1.14.14.147]\n",
      "AT3G13730  CYP90D1; cytochrome P450, family 90, subfamily D, polypeptide 1 [KO:K12638] [EC:1.14.14.147]\n",
      "AT5G38970  BR6OX1; brassinosteroid-6-oxidase 1 [KO:K09590] [EC:1.14.-.-]\n",
      "AT3G30180  BR6OX2; brassinosteroid-6-oxidase 2 [KO:K12640] [EC:1.14.-.-]\n",
      "AT2G26710  BAS1; Cytochrome P450 superfamily protein [KO:K15639] [EC:1.14.-.-]\n",
      "COMPOUND    C01789  Campesterol\n",
      "C08814  Brassinolide\n",
      "C15784  Campest-4-en-3beta-ol\n",
      "C15785  Campest-4-en-3-one\n",
      "C15786  5alpha-Campestan-3-one\n",
      "C15787  Campestanol\n",
      "C15788  6alpha-Hydroxycampestanol\n",
      "C15789  6-Oxocampestanol\n",
      "C15790  Cathasterone\n",
      "C15791  Teasterone\n",
      "C15792  3-Dehydroteasterone\n",
      "C15793  Typhasterol\n",
      "C15794  Castasterone\n",
      "C15795  22alpha-Hydroxy-campesterol\n",
      "C15796  22alpha-Hydroxy-campest-4-en-3-one\n",
      "C15797  22alpha-Hydroxy-5alpha-campestan-3-one\n",
      "C15798  6-Deoxocathasterone\n",
      "C15799  6-Deoxoteasterone\n",
      "C15800  3-Dehydro-6-deoxoteasterone\n",
      "C15801  6-Deoxotyphasterol\n",
      "C15802  6-Deoxocastasterone\n",
      "C15803  6alpha-Hydroxy-castasterone\n",
      "C16251  3-epi-6-Deoxocathasterone\n",
      "C16252  (22R,23R)-22,23-Dihydroxycampesterol\n",
      "C16253  (22R,23R)-22,23-Dihydroxy-campest-4-en-3-one\n",
      "C17733  7-Oxateasterone\n",
      "C17735  7-Oxatyphasterol\n",
      "C19873  26-Hydroxycastasterone\n",
      "C19874  26-Hydroxybrassinolide\n",
      "REFERENCE\n",
      "AUTHORS   Koshiba K, Kamiya Y.\n",
      "TITLE     [Atarasii Syokubutuhorumon no Kagaku] (In Japanese)\n",
      "JOURNAL   Koudansya saientific:199-138 (2002)\n",
      "REFERENCE\n",
      "AUTHORS   Fukuda H, Matida Y, Kamiya Y, Kakimoto T.\n",
      "TITLE     [Shinpan Syokubutuhorumon no Sigunarudentatu] (In Japanese)\n",
      "JOURNAL   Syujyunsya:165-186 (2004)\n",
      "REFERENCE   PMID:12529536\n",
      "AUTHORS   Shimada Y, Goda H, Nakamura A, Takatsuto S, Fujioka S, Yoshida S.\n",
      "TITLE     Organ-specific expression of brassinosteroid-biosynthetic genes and distribution of endogenous brassinosteroids in Arabidopsis.\n",
      "JOURNAL   Plant Physiol 131:287-97 (2003)\n",
      "DOI:10.1104/pp.013029\n",
      "REFERENCE\n",
      "AUTHORS   Choe, S.\n",
      "TITLE     Brassinosteroid biosynthesis and inactivation.\n",
      "JOURNAL   Physiol Plant 126:539-548 (2006)\n",
      "DOI:10.1111/j.1399-3054.2006.00681.x\n",
      "REFERENCE   PMID:17138693\n",
      "AUTHORS   Ohnishi T, Szatmari AM, Watanabe B, Fujita S, Bancos S, Koncz C, Lafos M, Shibata K, Yokota T, Sakata K, Szekeres M, Mizutani M\n",
      "TITLE     C-23 hydroxylation by Arabidopsis CYP90C1 and CYP90D1 reveals a novel shortcut in brassinosteroid biosynthesis.\n",
      "JOURNAL   Plant Cell 18:3275-88 (2006)\n",
      "DOI:10.1105/tpc.106.045443\n",
      "REFERENCE   PMID:17693126\n",
      "AUTHORS   Bishop GJ\n",
      "TITLE     Refining the plant steroid hormone biosynthesis pathway.\n",
      "JOURNAL   Trends Plant Sci 12:377-80 (2007)\n",
      "DOI:10.1016/j.tplants.2007.07.001\n",
      "REFERENCE   PMID:15710611\n",
      "AUTHORS   Nomura T, Kushiro T, Yokota T, Kamiya Y, Bishop GJ, Yamaguchi S\n",
      "TITLE     The last reaction producing brassinolide is catalyzed by cytochrome P-450s, CYP85A3 in tomato and CYP85A2 in Arabidopsis.\n",
      "JOURNAL   J Biol Chem 280:17873-9 (2005)\n",
      "DOI:10.1074/jbc.M414592200\n",
      "REFERENCE   PMID:18685225\n",
      "AUTHORS   Katsumata T, Hasegawa A, Fujiwara T, Komatsu T, Notomi M, Abe H, Natsume M, Kawaide H\n",
      "TITLE     Arabidopsis CYP85A2 catalyzes lactonization reactions in the biosynthesis of 2-deoxy-7-oxalactone brassinosteroids.\n",
      "JOURNAL   Biosci Biotechnol Biochem 72:2110-7 (2008)\n",
      "DOI:10.1271/bbb.80192\n",
      "REFERENCE   PMID:15705958\n",
      "AUTHORS   Tanabe S, Ashikari M, Fujioka S, Takatsuto S, Yoshida S, Yano M, Yoshimura A, Kitano H, Matsuoka M, Fujisawa Y, Kato H, Iwasaki Y\n",
      "TITLE     A novel cytochrome P450 is implicated in brassinosteroid biosynthesis via the characterization of a rice dwarf mutant, dwarf11, with reduced seed length.\n",
      "JOURNAL   Plant Cell 17:776-90 (2005)\n",
      "DOI:10.1105/tpc.104.024950\n",
      "REFERENCE   PMID:14615594\n",
      "AUTHORS   Hong Z, Ueguchi-Tanaka M, Umemura K, Uozu S, Fujioka S, Takatsuto S, Yoshida S, Ashikari M, Kitano H, Matsuoka M\n",
      "TITLE     A rice brassinosteroid-deficient mutant, ebisu dwarf (d2), is caused by a loss of function of a new member of cytochrome P450.\n",
      "JOURNAL   Plant Cell 15:2900-10 (2003)\n",
      "DOI:10.1105/tpc.014712\n",
      "REL_PATHWAY ath00100  Steroid biosynthesis\n",
      "KO_PATHWAY  ko00905\n",
      "///\n",
      "ENTRY       ath00906                    Pathway\n",
      "NAME        Carotenoid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Metabolism of terpenoids and polyketides\n",
      "PATHWAY_MAP ath00906  Carotenoid biosynthesis\n",
      "MODULE      ath_M00097  beta-Carotene biosynthesis, GGAP => beta-carotene [PATH:ath00906]\n",
      "ath_M00372  Abscisic acid biosynthesis, beta-carotene => abscisic acid [PATH:ath00906]\n",
      "DBLINKS     GO: 0016117\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G17230  PSY; PHYTOENE SYNTHASE [KO:K02291] [EC:2.5.1.32]\n",
      "AT4G14210  PDS3; phytoene desaturase 3 [KO:K02293] [EC:1.3.5.5]\n",
      "AT1G10830  Z-ISO; 15-cis-zeta-carotene isomerase [KO:K15744] [EC:5.2.1.12]\n",
      "AT3G04870  ZDS; zeta-carotene desaturase [KO:K00514] [EC:1.3.5.6]\n",
      "AT1G06820  CRTISO; carotenoid isomerase [KO:K09835] [EC:5.2.1.13]\n",
      "AT5G57030  LUT2; Lycopene beta/epsilon cyclase protein [KO:K06444] [EC:5.5.1.18]\n",
      "AT3G10230  LYC; lycopene cyclase [KO:K06443] [EC:5.5.1.19]\n",
      "AT4G25700  BETA-OHASE_1; beta-hydroxylase 1 [KO:K15746] [EC:1.14.15.24]\n",
      "AT5G52570  BETA-OHASE_2; beta-carotene hydroxylase 2 [KO:K15746] [EC:1.14.15.24]\n",
      "AT1G31800  CYP97A3; cytochrome P450, family 97, subfamily A, polypeptide 3 [KO:K15747] [EC:1.14.-.-]\n",
      "AT3G53130  LUT1; Cytochrome P450 superfamily protein [KO:K09837] [EC:1.14.14.158]\n",
      "AT5G67030  ABA1; zeaxanthin epoxidase (ZEP) (ABA1) [KO:K09838] [EC:1.14.15.21]\n",
      "AT1G08550  NPQ1; non-photochemical quenching 1 [KO:K09839] [EC:1.23.5.1]\n",
      "AT1G30100  NCED5; nine-cis-epoxycarotenoid dioxygenase 5 [KO:K09840] [EC:1.13.11.51]\n",
      "AT3G14440  NCED3; nine-cis-epoxycarotenoid dioxygenase 3 [KO:K09840] [EC:1.13.11.51]\n",
      "AT3G24220  NCED6; nine-cis-epoxycarotenoid dioxygenase 6 [KO:K09840] [EC:1.13.11.51]\n",
      "AT1G78390  NCED9; nine-cis-epoxycarotenoid dioxygenase 9 [KO:K09840] [EC:1.13.11.51]\n",
      "AT4G18350  NCED2; nine-cis-epoxycarotenoid dioxygenase 2 [KO:K09840] [EC:1.13.11.51]\n",
      "AT4G19170  NCED4; nine-cis-epoxycarotenoid dioxygenase 4 [KO:K09840] [EC:1.13.11.51]\n",
      "AT1G52340  ABA2; NAD(P)-binding Rossmann-fold superfamily protein [KO:K09841] [EC:1.1.1.288]\n",
      "AT2G27150  AAO3; abscisic aldehyde oxidase 3 [KO:K09842] [EC:1.2.3.14]\n",
      "AT2G29090  CYP707A2; cytochrome P450, family 707, subfamily A, polypeptide 2 [KO:K09843] [EC:1.14.14.137]\n",
      "AT3G19270  CYP707A4; cytochrome P450, family 707, subfamily A, polypeptide 4 [KO:K09843] [EC:1.14.14.137]\n",
      "AT5G45340  CYP707A3; cytochrome P450, family 707, subfamily A, polypeptide 3 [KO:K09843] [EC:1.14.14.137]\n",
      "AT4G19230  CYP707A1; cytochrome P450, family 707, subfamily A, polypeptide 1 [KO:K09843] [EC:1.14.14.137]\n",
      "AT1G52400  BGLU18; beta glucosidase 18 [KO:K15748] [EC:3.2.1.175]\n",
      "AT1G03055  D27; beta-carotene isomerase D27-like protein [KO:K17911] [EC:5.2.1.14]\n",
      "AT2G44990  CCD7; carotenoid cleavage dioxygenase 7 [KO:K17912] [EC:1.13.11.68]\n",
      "AT4G32810  CCD8; carotenoid cleavage dioxygenase 8 [KO:K17913] [EC:1.13.11.69 1.13.11.70]\n",
      "COMPOUND    C00353  Geranylgeranyl diphosphate\n",
      "C00448  trans,trans-Farnesyl diphosphate\n",
      "C02094  beta-Carotene\n",
      "C03427  Prephytoene diphosphate\n",
      "C03428  Presqualene diphosphate\n",
      "C05413  all-trans-Phytoene\n",
      "C05414  all-trans-Phytofluene\n",
      "C05421  15-cis-Phytoene\n",
      "C05430  all-trans-zeta-Carotene\n",
      "C05431  all-trans-Neurosporene\n",
      "C05432  Lycopene\n",
      "C05433  alpha-Carotene\n",
      "C05434  beta-Zeacarotene\n",
      "C05435  gamma-Carotene\n",
      "C06082  Abscisate\n",
      "C06098  Zeaxanthin\n",
      "C08579  Antheraxanthin\n",
      "C08580  Astaxanthin\n",
      "C08583  Canthaxanthin\n",
      "C08584  Capsanthin\n",
      "C08585  Capsorubin\n",
      "C08586  delta-Carotene\n",
      "C08590  Zeinoxanthin\n",
      "C08591  beta-Cryptoxanthin\n",
      "C08592  Echinenone\n",
      "C08601  Lutein\n",
      "C08606  Neoxanthin\n",
      "C08607  Neurosporaxanthin\n",
      "C08613  Torulene\n",
      "C08614  Violaxanthin\n",
      "C09190  Strigol\n",
      "C09707  Phaseic acid\n",
      "C13431  9'-cis-Neoxanthin\n",
      "C13433  9-cis-Violaxanthin\n",
      "C13453  Xanthoxin\n",
      "C13454  Xanthoxic acid\n",
      "C13455  Abscisic aldehyde\n",
      "C13456  Abscisic alcohol\n",
      "C15514  8'-Hydroxyabscisate\n",
      "C15857  9,9'-dicis-zeta-Carotene\n",
      "C15858  7,9,7',9'-tetracis-Lycopene\n",
      "C15859  1'-Hydroxy-gamma-carotene\n",
      "C15860  1'-Hydroxytorulene\n",
      "C15861  1'-Hydroxy-gamma-carotene glucoside\n",
      "C15864  1'-Hydroxy-gamma-carotene glucoside ester\n",
      "C15867  3,4-Dehydrolycopene\n",
      "C15874  3,4-Dehydrorhodopin\n",
      "C15877  Anhydrorhodovibrin\n",
      "C15878  Rhodovibrin\n",
      "C15879  Hydroxyspirilloxanthin\n",
      "C15881  Spirilloxanthin\n",
      "C15884  2-Ketospirilloxanthin\n",
      "C15885  2,2'-Diketospirilloxanthin\n",
      "C15886  3,4-Dihydroanhydrorhodovibrin\n",
      "C15887  3',4'-Dihydrorhodovibrin\n",
      "C15888  3,4,3',4'-Tetrahydrospirilloxanthin\n",
      "C15892  Chloroxanthin\n",
      "C15895  3,4-Dihydrospheroidene\n",
      "C15898  Demethylspheroidene\n",
      "C15900  Spheroidene\n",
      "C15902  Hydroxyspheroidene\n",
      "C15903  Spheroidenone\n",
      "C15905  Hydroxyspheroidenone\n",
      "C15908  Chlorobactene\n",
      "C15911  Hydroxychlorobactene\n",
      "C15914  Hydroxychlorobactene glucoside\n",
      "C15917  Hydroxychlorobactene glucoside ester\n",
      "C15935  (2'S)-Deoxymyxol 2'-(2,4-di-O-methyl-alpha-L-fucoside)\n",
      "C15937  (3R,2'S)-Myxol 2'-(2,4-di-O-methyl-alpha-L-fucoside)\n",
      "C15940  (2'S)-Deoxymyxol 2'-alpha-L-fucoside\n",
      "C15941  (3R,2'S)-Myxol 2'-alpha-L-fucoside\n",
      "C15942  (3S,2'S)-4-Ketomyxol 2'-alpha-L-fucoside\n",
      "C15943  Isorenieratene\n",
      "C15965  3'-Hydroxyechinenone\n",
      "C15966  3-Hydroxyechinenone\n",
      "C15967  Phoenicoxanthin\n",
      "C15968  Adonixanthin\n",
      "C15969  Zeaxanthin diglucoside\n",
      "C15970  Abscisic acid glucose ester\n",
      "C15971  Dihydrophaseic acid\n",
      "C15981  alpha-Cryptoxanthin\n",
      "C15982  Thermocryptoxanthin\n",
      "C15983  Thermozeaxanthin\n",
      "C15984  Thermobiszeaxanthin\n",
      "C16144  4,4'-Diapophytoene\n",
      "C16145  4,4'-Diaponeurosporene\n",
      "C16146  4,4'-Diaponeurosporenic acid\n",
      "C16147  Glycosyl-4,4'-diaponeurosporenoate\n",
      "C16148  Staphyloxanthin\n",
      "C16270  Rhodopinal\n",
      "C16271  Rhodopinal glucoside\n",
      "C16274  (2S,2'S)-Oscillol\n",
      "C16275  (2S,2'S)-Oscillol 2,2'-di(alpha-L-fucoside)\n",
      "C16276  epsilon-Carotene\n",
      "C16277  R.g.-Keto I\n",
      "C16278  R.g.-Keto III\n",
      "C16279  Thiothece-474\n",
      "C16280  Okenone\n",
      "C16282  Caloxanthin\n",
      "C16284  Nostoxanthin\n",
      "C16291  7,8-Dihydro-beta-carotene\n",
      "C16340  beta-Isorenieratene\n",
      "C16486  Astaxanthin diester\n",
      "C18036  Strigolactone ABC-rings\n",
      "C18037  5-Deoxystrigol\n",
      "C19759  7,9,9'-tricis-Neurosporene\n",
      "C19764  9,15,9'-tricis-zeta-Carotene\n",
      "C19765  15,9'-dicis-Phytofluene\n",
      "C19795  1-Hydroxy-1,2-dihydrolycopene\n",
      "C19797  4,4'-Diapolycopene\n",
      "C19798  4,4'-Diapolycopenedial\n",
      "C19892  4'-Apo-beta,psi-caroten-4'-al\n",
      "C20484  9-cis-beta-Carotene\n",
      "C20692  9-cis-10'-Apo-beta-carotenal\n",
      "C20693  Carlactone\n",
      "C21770  4,4'-Diapolycopene-4,4'-dioate\n",
      "C22058  Dihydroisopentenyldehydrorhodopin\n",
      "C22059  Isopentenyldehydrorhodopin\n",
      "C22060  Dihydrobisanhydrobacterioruberin\n",
      "C22346  1'-Hydroxy-1',2'-dihydro-beta,psi-caroten-4'-one\n",
      "C22347  Bisanhydrobacterioruberin\n",
      "C22348  Monoanhydrobacterioruberin\n",
      "C22349  Bacterioruberin\n",
      "REFERENCE\n",
      "AUTHORS   Takaichi S, Mimuro M, Tomita Y.\n",
      "TITLE     [Karotenoido-sono Tayousei to Seirikattsei] (In Japanese)\n",
      "JOURNAL   SHOKABO:109-156 (2006)\n",
      "REFERENCE\n",
      "AUTHORS   Fukuda H, Matida Y, Kamiya Y, Kakimoto T.\n",
      "TITLE     [Shinpan Syokubutuhorumon no Sigunarudentatu] (In Japanese)\n",
      "JOURNAL   Syujyunsya:165-186 (2004)\n",
      "REFERENCE\n",
      "AUTHORS   Takaichi S.\n",
      "TITLE     Distribution and biosynthesis of carotenoids in plants and photosynthetic bacteria.\n",
      "JOURNAL   Foods and Food Ingredients Journal of Japan 212-7:524-531 (2007)\n",
      "DOI:10.1007/978-1-4020-8815-5_6\n",
      "REFERENCE   PMID:11804826 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Seo M, Koshiba T.\n",
      "TITLE     Complex regulation of ABA biosynthesis in plants.\n",
      "JOURNAL   Trends Plant Sci 7:41-8 (2002)\n",
      "DOI:10.1016/S1360-1385(01)02187-2\n",
      "REFERENCE   PMID:17085635 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Cunningham FX Jr, Lee H, Gantt E.\n",
      "TITLE     Carotenoid biosynthesis in the primitive red alga Cyanidioschyzon merolae.\n",
      "JOURNAL   Eukaryot Cell 6:533-45 (2007)\n",
      "DOI:10.1128/EC.00265-06\n",
      "REFERENCE   PMID:9914519 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Bartley GE, Scolnik PA, Beyer P.\n",
      "TITLE     Two Arabidopsis thaliana carotene desaturases, phytoene desaturase and zeta-carotene desaturase, expressed in Escherichia coli, catalyze a poly-cis pathway to yield pro-lycopene.\n",
      "JOURNAL   Eur J Biochem 259:396-403 (1999)\n",
      "DOI:10.1046/j.1432-1327.1999.00051.x\n",
      "REFERENCE   PMID:11884677 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Park H, Kreunen SS, Cuttriss AJ, DellaPenna D, Pogson BJ.\n",
      "TITLE     Identification of the carotenoid isomerase provides insight into carotenoid biosynthesis, prolamellar body formation, and photomorphogenesis.\n",
      "JOURNAL   Plant Cell 14:321-32 (2002)\n",
      "DOI:10.1105/tpc.010302\n",
      "REFERENCE   PMID:15557094 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Isaacson T, Ohad I, Beyer P, Hirschberg J.\n",
      "TITLE     Analysis in vitro of the enzyme CRTISO establishes a poly-cis-carotenoid biosynthesis pathway in plants.\n",
      "JOURNAL   Plant Physiol 136:4246-55 (2004)\n",
      "DOI:10.1104/pp.104.052092\n",
      "REFERENCE   PMID:12782726 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Tian L, Magallanes-Lundback M, Musetti V, DellaPenna D.\n",
      "TITLE     Functional analysis of beta- and epsilon-ring carotenoid hydroxylases in Arabidopsis.\n",
      "JOURNAL   Plant Cell 15:1320-32 (2003)\n",
      "DOI:10.1105/tpc.011403\n",
      "REFERENCE   PMID:14709673 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Tian L, Musetti V, Kim J, Magallanes-Lundback M, DellaPenna D.\n",
      "TITLE     The Arabidopsis LUT1 locus encodes a member of the cytochrome p450 family that is required for carotenoid epsilon-ring hydroxylation activity.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 101:402-7 (2004)\n",
      "DOI:10.1073/pnas.2237237100\n",
      "REFERENCE   PMID:7920703 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Bouvier F, Hugueney P, d'Harlingue A, Kuntz M, Camara B.\n",
      "TITLE     Xanthophyll biosynthesis in chromoplasts: isolation and molecular cloning of an enzyme catalyzing the conversion of 5,6-epoxycarotenoid into ketocarotenoid.\n",
      "JOURNAL   Plant J 6:45-54 (1994)\n",
      "DOI:10.1046/j.1365-313X.1994.6010045.x\n",
      "REFERENCE   PMID:9159947 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Schwartz SH, Leon-Kloosterziel KM, Koornneef M, Zeevaart JA.\n",
      "TITLE     Biochemical characterization of the aba2 and aba3 mutants in Arabidopsis thaliana.\n",
      "JOURNAL   Plant Physiol 114:161-6 (1997)\n",
      "DOI:10.1104/pp.114.1.161\n",
      "REFERENCE   PMID:12114582 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Xu ZJ, Nakajima M, Suzuki Y, Yamaguchi I.\n",
      "TITLE     Cloning and characterization of the abscisic acid-specific glucosyltransferase gene from adzuki bean seedlings.\n",
      "JOURNAL   Plant Physiol 129:1285-95 (2002)\n",
      "DOI:10.1104/pp.001784\n",
      "REFERENCE   PMID:9808729 (Plant-Carotenoid pathway)\n",
      "AUTHORS   Krochko JE, Abrams GD, Loewen MK, Abrams SR, Cutler AJ.\n",
      "TITLE     (+)-Abscisic acid 8'-hydroxylase is a cytochrome P450 monooxygenase\n",
      "JOURNAL   Plant Physiol 118:849-60 (1998)\n",
      "DOI:10.1104/pp.118.3.849\n",
      "REFERENCE   PMID:10851005 (Purple bacteria-canthaxanthin,spirilloxanthin pathway)\n",
      "AUTHORS   Hannibal L, Lorquin J, D'Ortoli NA, Garcia N, Chaintreuil C, Masson-Boivin C, Dreyfus B, Giraud E.\n",
      "TITLE     Isolation and characterization of canthaxanthin biosynthesis genes from the photosynthetic bacterium Bradyrhizobium sp. strain ORS278.\n",
      "JOURNAL   J Bacteriol 182:3850-3 (2000)\n",
      "DOI:10.1128/JB.182.13.3850-3853.2000\n",
      "REFERENCE   PMID:12664193 (Purple bacteria-Spheroidene,spirilloxanthin pathway)\n",
      "AUTHORS   Pinta V, Ouchane S, Picaud M, Takaichi S, Astier C, Reiss-Husson F.\n",
      "TITLE     Characterization of unusual hydroxy- and ketocarotenoids in Rubrivivax gelatinosus: involvement of enzyme CrtF or CrtA.\n",
      "JOURNAL   Arch Microbiol 179:354-62 (2003)\n",
      "DOI:10.1007/s00203-003-0538-3\n",
      "REFERENCE   PMID:7721699 (Purple bacteria-Spheroidene pathway)\n",
      "AUTHORS   Lang HP, Cogdell RJ, Takaichi S, Hunter CN.\n",
      "TITLE     Complete DNA sequence, specific Tn5 insertion map, and gene assignment of the carotenoid biosynthesis pathway of Rhodobacter sphaeroides.\n",
      "JOURNAL   J Bacteriol 177:2064-73 (1995)\n",
      "DOI:10.1128/JB.177.8.2064-2073.1995\n",
      "REFERENCE   PMID:9636041 (Purple bacteria-Unusual spirilloxanthin pathway)\n",
      "AUTHORS   Komori M, Ghosh R, Takaichi S, Hu Y, Mizoguchi T, Koyama Y, Kuki M.\n",
      "TITLE     A null lesion in the rhodopin 3,4-desaturase of Rhodospirillum rubrum unmasks a cryptic branch of the carotenoid biosynthetic pathway.\n",
      "JOURNAL   Biochemistry 37:8987-94 (1998)\n",
      "DOI:10.1021/bi9730947\n",
      "REFERENCE   PMID:14044829 (Purple bacteria-Okenone)\n",
      "AUTHORS   SCHMIDT K, LIAAENJENSEN S, SCHLEGEL HG.\n",
      "TITLE     [CAROTENOIDS IN THIORHODACEAE. I. OKENONEAS THE PRINCIPAL CAROTENOID IN CHROMATIUM OKENII PERTY.]\n",
      "JOURNAL   Arch Mikrobiol 46:117-26 (1963)\n",
      "REFERENCE   PMID:4932469 (Purple bacteria-Rhodopinal)\n",
      "AUTHORS   Schmidt K.\n",
      "TITLE     Carotenoids of purple nonsulfur bacteria. Composition and biosynthesis of the carotenoids of some strains of Rhodopseudomonas acidophila, Rhodospirillum tenue, and Rhodocyclus purpureus.\n",
      "JOURNAL   Arch Mikrobiol 77:231-8 (1971)\n",
      "DOI:10.1007/BF00408115\n",
      "REFERENCE   PMID:2254247 (Erwinia uredovora-Carotenoid pathway)\n",
      "AUTHORS   Misawa N, Nakagawa M, Kobayashi K, Yamano S, Izawa Y, Nakamura K, Harashima K.\n",
      "TITLE     Elucidation of the Erwinia uredovora carotenoid biosynthetic pathway by functional analysis of gene products expressed in Escherichia coli.\n",
      "JOURNAL   J Bacteriol 172:6704-12 (1990)\n",
      "DOI:10.1128/JB.172.12.6704-6712.1990\n",
      "REFERENCE   PMID:7588751 (Myxococcus xanthus-Carotenoid pathway)\n",
      "AUTHORS   Botella JA, Murillo FJ, Ruiz-Vazquez R.\n",
      "TITLE     A cluster of structural and regulatory genes for light-induced carotenogenesis in Myxococcus xanthus.\n",
      "JOURNAL   Eur J Biochem 233:238-48 (1995)\n",
      "DOI:10.1111/j.1432-1033.1995.238_1.x\n",
      "REFERENCE   PMID:7592436 (Marine bacteria-Astaxanthin pathway)\n",
      "AUTHORS   Misawa N, Satomi Y, Kondo K, Yokoyama A, Kajiwara S, Saito T, Ohtani T, Miki W.\n",
      "TITLE     Structure and functional analysis of a marine bacterial carotenoid biosynthesis gene cluster and astaxanthin biosynthetic pathway proposed at the gene level.\n",
      "JOURNAL   J Bacteriol 177:6575-84 (1995)\n",
      "DOI:10.1128/JB.177.22.6575-6584.1995\n",
      "REFERENCE   PMID:16020541 (Heriobacteria-Diapocarotene pathway)\n",
      "AUTHORS   Pelz A, Wieland KP, Putzbach K, Hentschel P, Albert K, Gotz F.\n",
      "TITLE     Structure and biosynthesis of staphyloxanthin from Staphylococcus aureus.\n",
      "JOURNAL   J Biol Chem 280:32493-8 (2005)\n",
      "DOI:10.1074/jbc.M505070200\n",
      "REFERENCE   PMID:8002598 (Heriobacteria-Diapocarotene pathway)\n",
      "AUTHORS   Wieland B, Feil C, Gloria-Maercker E, Thumm G, Lechner M, Bravo JM, Poralla K, Gotz F.\n",
      "TITLE     Genetic and biochemical analyses of the biosynthesis of the yellow carotenoid 4,4'-diaponeurosporene of Staphylococcus aureus.\n",
      "JOURNAL   J Bacteriol 176:7719-26 (1994)\n",
      "DOI:10.1128/JB.176.24.7719-7726.1994\n",
      "REFERENCE   PMID:10395965 (Streptomyces griseus-Carotenoid pathway)\n",
      "AUTHORS   Krugel H, Krubasik P, Weber K, Saluz HP, Sandmann G.\n",
      "TITLE     Functional analysis of genes from Streptomyces griseus involved in the synthesis of isorenieratene, a carotenoid with aromatic end groups, revealed a novel type of carotenoid desaturase.\n",
      "JOURNAL   Biochim Biophys Acta 1439:57-64 (1999)\n",
      "DOI:10.1016/S1388-1981(99)00075-X\n",
      "REFERENCE   PMID:11479383 (Cyanobacteria-Myxol pathway)\n",
      "AUTHORS   Takaichi S, Maoka T, Masamoto K.\n",
      "TITLE     Myxoxanthophyll in Synechocystis sp. PCC 6803 is myxol 2'-dimethyl-fucoside, (3R,2'S)-myxol 2'-(2,4-di-O-methyl-alpha-L-fucoside), not rhamnoside.\n",
      "JOURNAL   Plant Cell Physiol 42:756-62 (2001)\n",
      "DOI:10.1093/pcp/pce098\n",
      "REFERENCE   PMID:15695449 (Cyanobacteria-Myxol pathway)\n",
      "AUTHORS   Takaichi S, Mochimaru M, Maoka T, Katoh H.\n",
      "TITLE     Myxol and 4-ketomyxol 2'-fucosides, not rhamnosides, from Anabaena sp. PCC 7120 and Nostoc punctiforme PCC 73102, and proposal for the biosynthetic pathway of carotenoids.\n",
      "JOURNAL   Plant Cell Physiol 46:497-504 (2005)\n",
      "DOI:10.1093/pcp/pci049\n",
      "REFERENCE   PMID:9092504 (Cyanobacteria-Carotenoid pathway)\n",
      "AUTHORS   Fernandez-Gonzalez B, Sandmann G, Vioque A.\n",
      "TITLE     A new type of asymmetrically acting beta-carotene ketolase is required for the synthesis of echinenone in the cyanobacterium Synechocystis sp. PCC 6803.\n",
      "JOURNAL   J Biol Chem 272:9728-33 (1997)\n",
      "DOI:10.1074/jbc.272.15.9728\n",
      "REFERENCE   PMID:16242129 (Cyanobacteria-Carotenoid pathway)\n",
      "AUTHORS   Mochimaru M, Masukawa H, Takaichi S.\n",
      "TITLE     The cyanobacterium Anabaena sp. PCC 7120 has two distinct beta-carotene ketolases: CrtO for echinenone and CrtW for ketomyxol synthesis.\n",
      "JOURNAL   FEBS Lett 579:6111-4 (2005)\n",
      "DOI:10.1016/j.febslet.2005.09.081\n",
      "REFERENCE   PMID:7919981 (Cyanobacteria-Carotenoid pathway)\n",
      "AUTHORS   Cunningham FX Jr, Sun Z, Chamovitz D, Hirschberg J, Gantt E.\n",
      "TITLE     Molecular structure and enzymatic function of lycopene cyclase from the cyanobacterium Synechococcus sp strain PCC7942.\n",
      "JOURNAL   Plant Cell 6:1107-21 (1994)\n",
      "DOI:10.1105/tpc.6.8.1107\n",
      "REFERENCE   PMID:15811329 (Cyanobacteria-Carotenoid pathway)\n",
      "AUTHORS   Tsuchiya T, Takaichi S, Misawa N, Maoka T, Miyashita H, Mimuro M.\n",
      "TITLE     The cyanobacterium Gloeobacter violaceus PCC 7421 uses bacterial-type phytoene desaturase in carotenoid biosynthesis.\n",
      "JOURNAL   FEBS Lett 579:2125-9 (2005)\n",
      "DOI:10.1016/j.febslet.2005.02.066\n",
      "REFERENCE   PMID:15115301 (Green sulfer bacteria-Carotenoid pathway)\n",
      "AUTHORS   Hirabayashi H, Ishii T, Takaichi S, Inoue K, Uehara K.\n",
      "TITLE     The role of carotenoids in the photoadaptation of the brown-colored sulfur bacterium Chlorobium phaeobacteroides.\n",
      "JOURNAL   Photochem Photobiol 79:280-5 (2004)\n",
      "DOI:10.1111/j.1751-1097.2004.tb00396.x\n",
      "REFERENCE   PMID:16923888 (Green sulfer bacteria-Carotenoid pathway)\n",
      "AUTHORS   Maresca JA, Bryant DA.\n",
      "TITLE     Two genes encoding new carotenoid-modifying enzymes in the green sulfur bacterium Chlorobium tepidum.\n",
      "JOURNAL   J Bacteriol 188:6217-23 (2006)\n",
      "DOI:10.1128/JB.00766-06\n",
      "REFERENCE   PMID:15292122 (Green sulfer bacteria-Carotenoid pathway)\n",
      "AUTHORS   Frigaard NU, Maresca JA, Yunker CE, Jones AD, Bryant DA.\n",
      "TITLE     Genetic manipulation of carotenoid biosynthesis in the green sulfur bacterium Chlorobium tepidum.\n",
      "JOURNAL   J Bacteriol 186:5210-20 (2004)\n",
      "DOI:10.1128/JB.186.16.5210-5220.2004\n",
      "REFERENCE   PMID:9297463 (Green sulfer bacteria-Carotenoid pathway)\n",
      "AUTHORS   Takaichi S, Wang ZY, Umetsu M, Nozawa T, Shimada K, Madigan MT.\n",
      "TITLE     New carotenoids from the thermophilic green sulfur bacterium Chlorobium tepidum: 1',2'-dihydro-gamma-carotene, 1',2'-dihydrochlorobactene, and OH-chlorobactene glucoside ester, and the carotenoid composition of different strains.\n",
      "JOURNAL   Arch Microbiol 168:270-6 (1997)\n",
      "DOI:10.1007/s002030050498\n",
      "REFERENCE   PMID:8661926 (Thermophilic bacteria-Thermocryptoxanthins)\n",
      "AUTHORS   Yokoyama A, Shizuri Y, Hoshino T, Sandmann G.\n",
      "TITLE     Thermocryptoxanthins: novel intermediates in the carotenoid biosynthetic pathway of Thermus thermophilus\n",
      "JOURNAL   Arch Microbiol 165:342-5 (1996)\n",
      "DOI:10.1007/s002030050336\n",
      "REFERENCE   PMID:12763034\n",
      "AUTHORS   Hemmi H, Ikejiri S, Nakayama T, Nishino T\n",
      "TITLE     Fusion-type lycopene beta-cyclase from a thermoacidophilic archaeon Sulfolobus solfataricus.\n",
      "JOURNAL   Biochem Biophys Res Commun 305:586-91 (2003)\n",
      "DOI:10.1016/S0006-291X(03)00798-8\n",
      "REL_PATHWAY ath00900  Terpenoid backbone biosynthesis\n",
      "KO_PATHWAY  ko00906\n",
      "///\n",
      "ENTRY       ath00908                    Pathway\n",
      "NAME        Zeatin biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Zeatin is a member of the cytokinin family, a class of phytohormones involved in various processes of growth and development in plants. Most abundant cytokinins are adenine-type, where the N6 position of adenine is substituted with an isoprenoid, such as in zeatin, or an aromatic side chain, such as in kinetin. Zeatin can be synthesized in two different pathways: the tRNA pathway and the AMP pathway. In the tRNA pathway zeatin is a recycled product of isopentenylated tRNAs. In the AMP pathway zeatin is synthesized from an isopentenyl donor, dimethylallyl diphosphate (DMAPP), and AMP, ADP, or ATP by isopentenyltransferases. After synthesis cytokinins can be glucosylated.\n",
      "CLASS       Metabolism; Metabolism of terpenoids and polyketides\n",
      "PATHWAY_MAP ath00908  Zeatin biosynthesis\n",
      "DBLINKS     GO: 0009691\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G63110  IPT3; isopentenyltransferase 3 [KO:K10760] [EC:2.5.1.27 2.5.1.112]\n",
      "AT4G24650  IPT4; isopentenyltransferase 4 [KO:K10760] [EC:2.5.1.27 2.5.1.112]\n",
      "AT5G19040  IPT5; isopentenyltransferase 5 [KO:K10760] [EC:2.5.1.27 2.5.1.112]\n",
      "AT3G23630  IPT7; isopentenyltransferase 7 [KO:K10760] [EC:2.5.1.27 2.5.1.112]\n",
      "AT1G25410  IPT6; isopentenyltransferase 6 [KO:K10760] [EC:2.5.1.27 2.5.1.112]\n",
      "AT3G19160  IPT8; ATP/ADP isopentenyltransferase [KO:K10760] [EC:2.5.1.27 2.5.1.112]\n",
      "AT1G68460  IPT1; isopentenyltransferase 1 [KO:K10760] [EC:2.5.1.27 2.5.1.112]\n",
      "AT1G67110  CYP735A2; cytochrome P450, family 735, subfamily A, polypeptide 2 [KO:K10717]\n",
      "AT5G38450  CYP735A1; cytochrome P450, family 735, subfamily A, polypeptide 1 [KO:K10717]\n",
      "AT5G21482  CKX7; cytokinin oxidase 7 [KO:K00279] [EC:1.5.99.12]\n",
      "AT4G29740  CKX4; cytokinin oxidase 4 [KO:K00279] [EC:1.5.99.12]\n",
      "AT2G19500  CKX2; cytokinin oxidase 2 [KO:K00279] [EC:1.5.99.12]\n",
      "AT5G56970  CKX3; cytokinin oxidase 3 [KO:K00279] [EC:1.5.99.12]\n",
      "AT2G41510  CKX1; cytokinin oxidase/dehydrogenase 1 [KO:K00279] [EC:1.5.99.12]\n",
      "AT1G75450  CKX5; cytokinin oxidase 5 [KO:K00279] [EC:1.5.99.12]\n",
      "AT3G63440  CKX6; cytokinin oxidase/dehydrogenase 6 [KO:K00279] [EC:1.5.99.12]\n",
      "AT5G05860  UGT76C2; UDP-glucosyl transferase 76C2 [KO:K13493] [EC:2.4.1.-]\n",
      "AT5G05870  UGT76C1; UDP-glucosyl transferase 76C1 [KO:K13493] [EC:2.4.1.-]\n",
      "AT2G36750  UGT73C1; UDP-glucosyl transferase 73C1 [KO:K13496] [EC:2.4.1.-]\n",
      "AT3G53160  UGT73C7; UDP-glucosyl transferase 73C7 [KO:K13496] [EC:2.4.1.-]\n",
      "AT2G36800  DOGT1; don-glucosyltransferase 1 [KO:K13496] [EC:2.4.1.-]\n",
      "AT2G36770  UDP-Glycosyltransferase superfamily protein [KO:K13496] [EC:2.4.1.-]\n",
      "AT2G36780  UDP-Glycosyltransferase superfamily protein [KO:K13496] [EC:2.4.1.-]\n",
      "AT2G36760  UGT73C2; UDP-glucosyl transferase 73C2 [KO:K13496] [EC:2.4.1.-]\n",
      "AT1G22400  UGT85A1; UDP-Glycosyltransferase superfamily protein [KO:K23452] [EC:2.4.1.-]\n",
      "AT1G22380  UGT85A3; UDP-glucosyl transferase 85A3 [KO:K23452] [EC:2.4.1.-]\n",
      "AT1G22340  UGT85A7; UDP-glucosyl transferase 85A7 [KO:K23452] [EC:2.4.1.-]\n",
      "AT1G78270  UGT85A4; UDP-glucosyl transferase 85A4 [KO:K23452] [EC:2.4.1.-]\n",
      "AT1G22360  UGT85A2; UDP-glucosyl transferase 85A2 [KO:K23452] [EC:2.4.1.-]\n",
      "AT1G22370  UGT85A5; UDP-glucosyl transferase 85A5 [KO:K23452] [EC:2.4.1.-]\n",
      "AT2G27760  IPT2; tRNAisopentenyltransferase 2 [KO:K00791] [EC:2.5.1.75]\n",
      "AT5G20040  IPT9; isopentenyltransferase 9 [KO:K00791] [EC:2.5.1.75]\n",
      "COMPOUND    C00002  ATP\n",
      "C00008  ADP\n",
      "C00015  UDP\n",
      "C00019  S-Adenosyl-L-methionine\n",
      "C00020  AMP\n",
      "C00029  UDP-glucose\n",
      "C00033  Acetate\n",
      "C00147  Adenine\n",
      "C00170  5'-Methylthioadenosine\n",
      "C00190  UDP-D-xylose\n",
      "C00235  Dimethylallyl diphosphate\n",
      "C00371  Zeatin\n",
      "C00979  O-Acetyl-L-serine\n",
      "C01513  Lupinate\n",
      "C01804  Discadenine\n",
      "C02029  Dihydrozeatin\n",
      "C03300  O-beta-D-Xylosylzeatin\n",
      "C03423  O-beta-D-Glucosylzeatin\n",
      "C04083  N6-(Delta2-Isopentenyl)-adenine\n",
      "C04432  tRNA containing 6-isopentenyladenosine\n",
      "C04713  N6-(delta2-Isopentenyl)-adenosine 5'-monophosphate\n",
      "C07330  3-Methyl-2-butenal\n",
      "C15545  cis-Zeatin\n",
      "C15546  O-beta-D-Glucopyranosyl-cis-zeatin\n",
      "C16424  Isopentenyladenosine-5'-triphosphate\n",
      "C16426  Isopentenyladenosine-5'-diphosphate\n",
      "C16427  Isopentenyl adenosine\n",
      "C16428  trans-Zeatin riboside triphosphate\n",
      "C16429  trans-Zeatin riboside diphosphate\n",
      "C16430  trans-Zeatin riboside monophosphate\n",
      "C16431  trans-Zeatin riboside\n",
      "C16441  cis-Zeatin riboside monophosphate\n",
      "C16443  trans-Zeatin-7-beta-D-glucoside\n",
      "C16445  Dihydrozeatin riboside monophosphate\n",
      "C16447  Dihydrozeatin riboside\n",
      "C16448  Dihydrozeatin-O-glucoside\n",
      "C16449  cis-Zeatin riboside\n",
      "C16467  cis-Prenyl-tRNA\n",
      "C17324  tRNA adenine\n",
      "REFERENCE   PMID:14726522\n",
      "AUTHORS   Kasahara H, Takei K, Ueda N, Hishiyama S, Yamaya T, Kamiya Y, Yamaguchi S, Sakakibara H.\n",
      "TITLE     Distinct isoprenoid origins of cis- and trans-zeatin biosyntheses in Arabidopsis.\n",
      "JOURNAL   J Biol Chem 279:14049-54 (2004)\n",
      "DOI:10.1074/jbc.M314195200\n",
      "REFERENCE   PMID:11313355\n",
      "AUTHORS   Takei K, Sakakibara H, Sugiyama T.\n",
      "TITLE     Identification of genes encoding adenylate isopentenyltransferase, a cytokinin biosynthesis enzyme, in Arabidopsis thaliana.\n",
      "JOURNAL   J Biol Chem 276:26405-10 (2001)\n",
      "DOI:10.1074/jbc.M102130200\n",
      "REFERENCE   PMID:11479373\n",
      "AUTHORS   Kakimoto T.\n",
      "TITLE     Identification of plant cytokinin biosynthetic enzymes as dimethylallyl diphosphate:ATP/ADP isopentenyltransferases.\n",
      "JOURNAL   Plant Cell Physiol 42:677-85 (2001)\n",
      "DOI:10.1093/pcp/pce112\n",
      "REFERENCE   PMID:15280363\n",
      "AUTHORS   Takei K, Yamaya T, Sakakibara H.\n",
      "TITLE     Arabidopsis CYP735A1 and CYP735A2 encode cytokinin hydroxylases that catalyze the biosynthesis of trans-Zeatin.\n",
      "JOURNAL   J Biol Chem 279:41866-72 (2004)\n",
      "DOI:10.1074/jbc.M406337200\n",
      "REFERENCE   PMID:15342621\n",
      "AUTHORS   Hou B, Lim EK, Higgins GS, Bowles DJ.\n",
      "TITLE     N-glucosylation of cytokinins by glycosyltransferases of Arabidopsis thaliana.\n",
      "JOURNAL   J Biol Chem 279:47822-32 (2004)\n",
      "DOI:10.1074/jbc.M409569200\n",
      "REFERENCE   PMID:9874810\n",
      "AUTHORS   Martin RC, Mok MC, Mok DW\n",
      "TITLE     Isolation of a cytokinin gene, ZOG1, encoding zeatin O-glucosyltransferase from Phaseolus lunatus.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 96:284-9 (1999)\n",
      "DOI:10.1073/pnas.96.1.284\n",
      "REFERENCE   PMID:10364407\n",
      "AUTHORS   Martin RC, Mok MC, Mok DW\n",
      "TITLE     A gene encoding the cytokinin enzyme zeatin O-xylosyltransferase of Phaseolus vulgaris.\n",
      "JOURNAL   Plant Physiol 120:553-8 (1999)\n",
      "DOI:10.1104/pp.120.2.553\n",
      "REFERENCE   PMID:12644686\n",
      "AUTHORS   Veach YK, Martin RC, Mok DW, Malbeck J, Vankova R, Mok MC\n",
      "TITLE     O-glucosylation of cis-zeatin in maize. Characterization of genes, enzymes, and endogenous cytokinins.\n",
      "JOURNAL   Plant Physiol 131:1374-80 (2003)\n",
      "DOI:10.1104/pp.017210\n",
      "REFERENCE   PMID:14555694\n",
      "AUTHORS   Werner T, Motyka V, Laucou V, Smets R, Van Onckelen H, Schmulling T\n",
      "TITLE     Cytokinin-deficient transgenic Arabidopsis plants show multiple developmental alterations indicating opposite functions of cytokinins in the regulation of shoot and root meristem activity.\n",
      "JOURNAL   Plant Cell 15:2532-50 (2003)\n",
      "DOI:10.1105/tpc.014928\n",
      "REFERENCE   PMID:17062755\n",
      "AUTHORS   Miyawaki K, Tarkowski P, Matsumoto-Kitano M, Kato T, Sato S, Tarkowska D, Tabata S, Sandberg G, Kakimoto T\n",
      "TITLE     Roles of Arabidopsis ATP/ADP isopentenyltransferases and tRNA isopentenyltransferases in cytokinin biosynthesis.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 103:16598-603 (2006)\n",
      "DOI:10.1073/pnas.0603522103\n",
      "REL_PATHWAY ath00900  Terpenoid backbone biosynthesis\n",
      "KO_PATHWAY  ko00908\n",
      "///\n",
      "ENTRY       ath00909                    Pathway\n",
      "NAME        Sesquiterpenoid and triterpenoid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Sesquiterpenoids (C15 terpenoids) are a group of terpenoids consisting of three isoprene units. They are derive from farnesyl diphosphate (FPP) and can be cyclized to produce various skeletal structures. Sesquiterpenoid biosynthesis begins with the loss of diphosphate from FPP under the action of sesquiterpene synthesis enzymes, generating an allylic cation that is highly susceptible to intramolecular attacks. Cyclization of the farnesyl cation may take place onto either of the remaining double bonds with the result that 6-, 10-, or 11-membered rings may be formed. Many sesquiterpenoids have been isolated from plants, fungi, marine organisms, and Streptomyces species. This map shows a few examples of acyclic and cyclic sesquiterpenoids.\n",
      "CLASS       Metabolism; Metabolism of terpenoids and polyketides\n",
      "PATHWAY_MAP ath00909  Sesquiterpenoid and triterpenoid biosynthesis\n",
      "DBLINKS     GO: 0016106 0016104\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G34650  SQS2; squalene synthase 2 [KO:K00801] [EC:2.5.1.21]\n",
      "AT4G34640  SQS1; squalene synthase 1 [KO:K00801] [EC:2.5.1.21]\n",
      "AT1G58440  XF1; FAD/NAD(P)-binding oxidoreductase family protein [KO:K00511] [EC:1.14.14.17]\n",
      "AT4G37760  SQE3; squalene epoxidase 3 [KO:K00511] [EC:1.14.14.17]\n",
      "AT5G24140  SQP2; squalene monooxygenase 2 [KO:K00511] [EC:1.14.14.17]\n",
      "AT5G24160  SQE6; squalene monooxygenase 6 [KO:K00511] [EC:1.14.14.17]\n",
      "AT2G22830  SQE2; squalene epoxidase 2 [KO:K00511] [EC:1.14.14.17]\n",
      "AT5G24150  SQP1; FAD/NAD(P)-binding oxidoreductase family protein [KO:K00511] [EC:1.14.14.17]\n",
      "AT4G33360  FLDH; NAD(P)-binding Rossmann-fold superfamily protein [KO:K15891] [EC:1.1.1.354]\n",
      "AT4G13280  TPS12; terpenoid synthase 12 [KO:K15795] [EC:4.2.3.40]\n",
      "AT4G13300  TPS13; terpenoid synthase 13 [KO:K15795] [EC:4.2.3.40]\n",
      "AT5G44630  Terpenoid cyclases/Protein prenyltransferases superfamily protein [KO:K15799] [EC:4.2.3.69 4.2.3.78 4.2.3.79]\n",
      "AT5G23960  TPS21; terpene synthase 21 [KO:K14184] [EC:4.2.3.104 4.2.3.57]\n",
      "AT1G78960  LUP2; lupeol synthase 2 [KO:K15815] [EC:5.4.99.39 5.4.99.40 5.4.99.41]\n",
      "AT1G78950  BAS; Terpenoid cyclases family protein [KO:K15813] [EC:5.4.99.39]\n",
      "AT1G78970  LUP1; lupeol synthase 1 [KO:K15816] [EC:4.2.1.128 5.4.99.41]\n",
      "AT1G78500  Terpenoid cyclases family protein [KO:K16204] [EC:5.4.99.52 5.4.99.54]\n",
      "AT5G36150  PEN3; putative pentacyclic triterpene synthase 3 [KO:K16205] [EC:5.4.99.56]\n",
      "AT4G15370  BARS1; baruol synthase 1 [KO:K16206] [EC:5.4.99.57]\n",
      "AT5G48010  THAS1; thalianol synthase 1 [KO:K15821] [EC:5.4.99.31]\n",
      "AT1G78955  CAMS1; camelliol C synthase 1 [KO:K15822] [EC:5.4.99.38]\n",
      "AT4G15340  PEN1; pentacyclic triterpene synthase 1 [KO:K15823] [EC:4.2.1.124]\n",
      "AT5G42600  MRN1; marneral synthase [KO:K16207] [EC:5.4.99.53]\n",
      "COMPOUND    C00448  trans,trans-Farnesyl diphosphate\n",
      "C00751  Squalene\n",
      "C01054  (S)-2,3-Epoxysqualene\n",
      "C01126  (2E,6E)-Farnesol\n",
      "C01841  Pentalenene\n",
      "C01860  Trichodiene\n",
      "C02004  Aristolochene\n",
      "C03220  (2Z,6E)-Farnesol\n",
      "C03428  Presqualene diphosphate\n",
      "C03461  2-trans,6-trans-Farnesal\n",
      "C06079  PR-toxin\n",
      "C06080  Nivalenol\n",
      "C06083  Tetrahymanol\n",
      "C06309  Diplopterol\n",
      "C06310  Diploptene\n",
      "C06394  delta-Cadinene\n",
      "C07667  Gossypol\n",
      "C08615  alpha-Amyrin\n",
      "C08616  beta-Amyrin\n",
      "C08628  Lupeol\n",
      "C08637  Taraxerol\n",
      "C09382  Costunolide\n",
      "C09627  Capsidiol\n",
      "C09629  beta-Caryophyllene\n",
      "C09665  alpha-Farnesene\n",
      "C09666  beta-Farnesene\n",
      "C09672  Germacrene B\n",
      "C09684  Humulene\n",
      "C09699  Longifolene\n",
      "C09704  Nerolidol\n",
      "C09723  beta-Selinene\n",
      "C09737  Solavetivone\n",
      "C12142  Vetispiradiene\n",
      "C16028  Amorpha-4,11-diene\n",
      "C16141  (+)-Germacrene A\n",
      "C16142  (-)-Germacrene D\n",
      "C16143  (1E,4S,5E,7R)-Germacra-1(10),5-dien-11-ol\n",
      "C16269  (+)-epi-Isozizaene\n",
      "C16286  Geosmin\n",
      "C16775  (S)-beta-Bisabolene\n",
      "C16814  gamma-Bisabolene\n",
      "C16829  gamma-Humulene\n",
      "C17277  (+)-Valencene\n",
      "C17953  (5S)-Albaflavenol\n",
      "C17954  Albaflavenone\n",
      "C17955  (5R)-Albaflavenol\n",
      "C18245  8-epi-Cedrol\n",
      "C19676  Germacra-1(10),4,11(13)-trien-12-ol\n",
      "C19678  Germacrene A acid\n",
      "C19708  5-Epiaristolochene\n",
      "C19711  Solavetivol\n",
      "C19732  24-Hydroxy-beta-amyrin\n",
      "C19736  (+)-alpha-Santalene\n",
      "C19740  (+)-alpha-Barbatene\n",
      "C19742  (+)-delta-Selinene\n",
      "C19746  (3R,6E)-Nerolidol\n",
      "C19747  Germacrene C\n",
      "C19748  (E)-gamma-Bisabolene\n",
      "C19749  (E)-alpha-Bisabolene\n",
      "C19752  (S)-beta-Macrocarpene\n",
      "C19753  cis-Muurola-3,5-diene\n",
      "C19755  Patchoulol\n",
      "C19801  Isomultiflorenol\n",
      "C19819  Cucurbitadienol\n",
      "C19820  Protostadienol\n",
      "C19828  Arabidiol\n",
      "C19829  Dammarenediol II\n",
      "C19832  Thalianol\n",
      "C19833  Germanicol\n",
      "C19834  Dammara-20,24-diene\n",
      "C19835  Camelliol C\n",
      "C19863  Lupan-3beta,20-diol\n",
      "C19908  (+)-beta-Chamigrene\n",
      "C19918  Parkeol\n",
      "C19939  Pentalen-13-al\n",
      "C19973  (+)-beta-Caryophyllene\n",
      "C19974  (+)-Caryolan-1-ol\n",
      "C20159  7-epi-alpha-Selinene\n",
      "C20163  5-epi-alpha-Selinene\n",
      "C20181  Avermitilol\n",
      "C20187  Achilleol B\n",
      "C20191  alpha-seco-Amyrin\n",
      "C20192  Marneral\n",
      "C20193  beta-seco-Amyrin\n",
      "C20195  Tirucalla-7,24-dien-3beta-ol\n",
      "C20200  Baruol\n",
      "C20478  (E)-2-epi-beta-Caryophyllene\n",
      "C20724  (+)-Thujopsene\n",
      "REFERENCE   PMID:9371761\n",
      "AUTHORS   Crock J, Wildung M, Croteau R\n",
      "TITLE     Isolation and bacterial expression of a sesquiterpene synthase cDNA clone from peppermint (Mentha x piperita, L.) that produces the aphid alarm pheromone (E)-beta-farnesene.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 94:12833-8 (1997)\n",
      "DOI:10.1073/pnas.94.24.12833\n",
      "REFERENCE   PMID:18363779\n",
      "AUTHORS   Nagegowda DA, Gutensohn M, Wilkerson CG, Dudareva N\n",
      "TITLE     Two nearly identical terpene synthases catalyze the formation of nerolidol and linalool in snapdragon flowers.\n",
      "JOURNAL   Plant J 55:224-39 (2008)\n",
      "DOI:10.1111/j.1365-313X.2008.03496.x\n",
      "REFERENCE   PMID:16579958\n",
      "AUTHORS   Bertea CM, Voster A, Verstappen FW, Maffei M, Beekwilder J, Bouwmeester HJ\n",
      "TITLE     Isoprenoid biosynthesis in Artemisia annua: cloning and heterologous expression of a germacrene A synthase from a glandular trichome cDNA library.\n",
      "JOURNAL   Arch Biochem Biophys 448:3-12 (2006)\n",
      "DOI:10.1016/j.abb.2006.02.026\n",
      "REFERENCE   PMID:12011356\n",
      "AUTHORS   de Kraker JW, Franssen MC, Joerink M, de Groot A, Bouwmeester HJ\n",
      "TITLE     Biosynthesis of costunolide, dihydrocostunolide, and leucodin. Demonstration of cytochrome p450-catalyzed formation of the lactone ring present in sesquiterpene lactones of chicory.\n",
      "JOURNAL   Plant Physiol 129:257-68 (2002)\n",
      "DOI:10.1104/pp.010957\n",
      "REFERENCE   PMID:17226502\n",
      "AUTHORS   Jaroszewski JW, Strom-Hansen T, Hansen SH, Thastrup O, Kofod H\n",
      "TITLE     On the botanical distribution of chiral forms of gossypol.\n",
      "JOURNAL   Planta Med 58:454-8 (1992)\n",
      "DOI:10.1055/s-2006-961512\n",
      "REFERENCE   PMID:9442047\n",
      "AUTHORS   Steele CL, Crock J, Bohlmann J, Croteau R\n",
      "TITLE     Sesquiterpene synthases from grand fir (Abies grandis). Comparison of constitutive and wound-induced activities, and cDNA isolation, characterization, and bacterial expression of delta-selinene synthase and gamma-humulene synthase.\n",
      "JOURNAL   J Biol Chem 273:2078-89 (1998)\n",
      "DOI:10.1074/jbc.273.4.2078\n",
      "REFERENCE   PMID:19030608\n",
      "AUTHORS   Fraga BM\n",
      "TITLE     Natural sesquiterpenoids.\n",
      "JOURNAL   Nat Prod Rep 25:1180-209 (2008)\n",
      "DOI:10.1039/b806216c\n",
      "REFERENCE   PMID:21876149\n",
      "AUTHORS   Field B, Fiston-Lavier AS, Kemen A, Geisler K, Quesneville H, Osbourn AE\n",
      "TITLE     Formation of plant metabolic gene clusters within dynamic chromosomal regions.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 108:16116-21 (2011)\n",
      "DOI:10.1073/pnas.1109273108\n",
      "REFERENCE   PMID:18356490\n",
      "AUTHORS   Field B, Osbourn AE\n",
      "TITLE     Metabolic diversification--independent assembly of operon-like gene clusters in different plants.\n",
      "JOURNAL   Science 320:543-7 (2008)\n",
      "DOI:10.1126/science.1154990\n",
      "REFERENCE   PMID:15148404\n",
      "AUTHORS   Qi X, Bakht S, Leggett M, Maxwell C, Melton R, Osbourn A\n",
      "TITLE     A gene cluster for secondary metabolism in oat: implications for the evolution of metabolic diversity in plants.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 101:8233-8 (2004)\n",
      "DOI:10.1073/pnas.0401301101\n",
      "REL_PATHWAY ath00100  Steroid biosynthesis\n",
      "ath00900  Terpenoid backbone biosynthesis\n",
      "KO_PATHWAY  ko00909\n",
      "///\n",
      "ENTRY       ath00910                    Pathway\n",
      "NAME        Nitrogen metabolism - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The biological process of the nitrogen cycle is a complex interplay among many microorganisms catalyzing different reactions, where nitrogen is found in various oxidation states ranging from +5 in nitrate to -3 in ammonia. The core nitrogen cycle involves four reduction pathways and two oxidation pathways. Nitrogen fixation [MD:M00175] is the process of reducing atmospheric molecular nitrogen to ammonia, a biologically useful reduced form incorporated into amino acids and other vital compounds. The ability of fixing atmospheric nitrogen by the nitrogenase enzyme complex is present in restricted prokaryotes (diazotrophs). The other reduction pathways are assimilatory nitrate reduction [MD:M00531] and dissimilatory nitrate reduction [MD:M00530] both for conversion to ammonia, and denitrification [MD:M00529]. Denitrification is a respiration in which nitrate or nitrite is reduced as a terminal electron acceptor under low oxygen or anoxic conditions, producing gaseous nitrogen compounds (N2, NO and N2O) to the atmosphere. The two oxidation pathways are nitrification [MD:M00528] and anaerobic ammonium oxidation (anammox). Nitrification is the oxidation of ammonia (NH3) with oxygen into nitrite followed by the oxidation of nitrite into nitrate. The first step is performed by ammonia-oxidizing microorganisms (e.g., Nitrosomonas and Nitrosococcus) and the second step by nitrite-oxidizing microorganisms (e.g., Nitrobacter). Anammox is a biochemical process of oxidizing ammonium (NH4+) into dinitrogen gas (N2) using nitrite as an electron acceptor. It occurs in the anammoxosome, a membrane bound compartment inside the cytoplasm, of anammox bacteria (e.g., Planctomycetes).\n",
      "CLASS       Metabolism; Energy metabolism\n",
      "PATHWAY_MAP ath00910  Nitrogen metabolism\n",
      "MODULE      ath_M00531  Assimilatory nitrate reduction, nitrate => ammonia [PATH:ath00910]\n",
      "DBLINKS     GO: 0006807\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G37550  Acetamidase/Formamidase family protein [KO:K01455] [EC:3.5.1.49]\n",
      "AT4G37560  Acetamidase/Formamidase family protein [KO:K01455] [EC:3.5.1.49]\n",
      "AT1G08100  NRT2.2; nitrate transporter 2.2 [KO:K02575]\n",
      "AT1G08090  NRT2:1; nitrate transporter 2:1 [KO:K02575]\n",
      "AT5G14570  NRT2.7; high affinity nitrate transporter 2.7 [KO:K02575]\n",
      "AT5G60770  NRT2.4; nitrate transporter 2.4 [KO:K02575]\n",
      "AT3G45060  NRT2.6; high affinity nitrate transporter 2.6 [KO:K02575]\n",
      "AT5G60780  NRT2.3; nitrate transporter 2.3 [KO:K02575]\n",
      "AT1G12940  NRT2.5; nitrate transporter2.5 [KO:K02575]\n",
      "AT1G37130  NIA2; nitrate reductase 2 [KO:K10534] [EC:1.7.1.1 1.7.1.2 1.7.1.3]\n",
      "AT1G77760  NIA1; nitrate reductase 1 [KO:K10534] [EC:1.7.1.1 1.7.1.2 1.7.1.3]\n",
      "AT2G15620  NIR1; nitrite reductase 1 [KO:K00366] [EC:1.7.7.1]\n",
      "AT3G44320  NIT3; nitrilase 3 [KO:K01501] [EC:3.5.5.1]\n",
      "AT3G44300  NIT2; nitrilase 2 [KO:K01501] [EC:3.5.5.1]\n",
      "AT3G44310  NIT1; nitrilase 1 [KO:K01501] [EC:3.5.5.1]\n",
      "AT5G07440  GDH2; glutamate dehydrogenase 2 [KO:K00261] [EC:1.4.1.3]\n",
      "AT5G18170  GDH1; glutamate dehydrogenase 1 [KO:K00261] [EC:1.4.1.3]\n",
      "AT3G03910  GDH3; glutamate dehydrogenase 3 [KO:K00261] [EC:1.4.1.3]\n",
      "AT1G51720  Amino acid dehydrogenase family protein [KO:K00262] [EC:1.4.1.4]\n",
      "AT5G37600  GSR_1; hypothetical protein [KO:K01915] [EC:6.3.1.2]\n",
      "AT3G17820  GLN1.3; glutamine synthetase 1.3 [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G48470  GLN1;5; glutamine synthetase 1;5 [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G35630  GS2; glutamine synthetase 2 [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G16570  GLN1;4; glutamine synthetase 1;4 [KO:K01915] [EC:6.3.1.2]\n",
      "AT1G66200  GSR2; hypothetical protein [KO:K01915] [EC:6.3.1.2]\n",
      "AT5G53460  GLT1; NADH-dependent glutamate synthase 1 [KO:K00264] [EC:1.4.1.14]\n",
      "AT2G41220  GLU2; glutamate synthase 2 [KO:K00284] [EC:1.4.7.1]\n",
      "AT5G04140  GLU1; glutamate synthase 1 [KO:K00284] [EC:1.4.7.1]\n",
      "AT3G23490  CYN; cyanase [KO:K01725] [EC:4.2.1.104]\n",
      "AT5G14740  CA2; carbonic anhydrase 2 [KO:K01673] [EC:4.2.1.1]\n",
      "AT1G23730  BCA3; beta carbonic anhydrase 3 [KO:K01673] [EC:4.2.1.1]\n",
      "AT3G01500  CA1; carbonic anhydrase 1 [KO:K01673] [EC:4.2.1.1]\n",
      "AT1G58180  BCA6; beta carbonic anhydrase 6 [KO:K01673] [EC:4.2.1.1]\n",
      "AT1G70410  BCA4; beta carbonic anhydrase 4 [KO:K01673] [EC:4.2.1.1]\n",
      "AT4G33580  BCA5; beta carbonic anhydrase 5 [KO:K01673] [EC:4.2.1.1]\n",
      "AT3G52720  ACA1; alpha carbonic anhydrase 1 [KO:K01674] [EC:4.2.1.1]\n",
      "AT4G21000  ACA6; alpha carbonic anhydrase 6 [KO:K01674] [EC:4.2.1.1]\n",
      "AT1G08080  ACA7; alpha carbonic anhydrase 7 [KO:K01674] [EC:4.2.1.1]\n",
      "AT1G08065  ACA5; alpha carbonic anhydrase 5 [KO:K01674] [EC:4.2.1.1]\n",
      "AT4G20990  ACA4; alpha carbonic anhydrase 4 [KO:K01674] [EC:4.2.1.1]\n",
      "AT5G04180  ACA3; alpha carbonic anhydrase 3 [KO:K01674] [EC:4.2.1.1]\n",
      "AT2G28210  ACA2; alpha carbonic anhydrase 2 [KO:K01674] [EC:4.2.1.1]\n",
      "AT5G56330  ACA8; alpha carbonic anhydrase 8 [KO:K01674] [EC:4.2.1.1]\n",
      "COMPOUND    C00011  CO2\n",
      "C00014  Ammonia\n",
      "C00025  L-Glutamate\n",
      "C00058  Formate\n",
      "C00064  L-Glutamine\n",
      "C00088  Nitrite\n",
      "C00169  Carbamoyl phosphate\n",
      "C00192  Hydroxylamine\n",
      "C00244  Nitrate\n",
      "C00288  HCO3-\n",
      "C00488  Formamide\n",
      "C00533  Nitric oxide\n",
      "C00697  Nitrogen\n",
      "C00726  Nitrile\n",
      "C00887  Nitrous oxide\n",
      "C01417  Cyanate\n",
      "C01563  Carbamate\n",
      "C05361  Hydrazine\n",
      "C06058  Nitroalkane\n",
      "REFERENCE   PMID:14702404\n",
      "AUTHORS   Scott JD, Ludwig RA\n",
      "TITLE     Azorhizobium caulinodans electron-transferring flavoprotein N electrochemically couples pyruvate dehydrogenase complex activity to N2 fixation.\n",
      "JOURNAL   Microbiology 150:117-26 (2004)\n",
      "DOI:10.1099/mic.0.26603-0\n",
      "REFERENCE   PMID:17408485\n",
      "AUTHORS   Kneip C, Lockhart P, Voss C, Maier UG\n",
      "TITLE     Nitrogen fixation in eukaryotes--new models for symbiosis.\n",
      "JOURNAL   BMC Evol Biol 7:55 (2007)\n",
      "DOI:10.1186/1471-2148-7-55\n",
      "REFERENCE   PMID:11004450\n",
      "AUTHORS   Whittaker M, Bergmann D, Arciero D, Hooper AB\n",
      "TITLE     Electron transfer during the oxidation of ammonia by the chemolithotrophic bacterium Nitrosomonas europaea.\n",
      "JOURNAL   Biochim Biophys Acta 1459:346-55 (2000)\n",
      "DOI:10.1016/S0005-2728(00)00171-7\n",
      "REFERENCE   PMID:15528644\n",
      "AUTHORS   Cabello P, Roldan MD, Moreno-Vivian C\n",
      "TITLE     Nitrate reduction and the nitrogen cycle in archaea.\n",
      "JOURNAL   Microbiology 150:3527-46 (2004)\n",
      "DOI:10.1099/mic.0.27303-0\n",
      "REFERENCE   PMID:11921398\n",
      "AUTHORS   Stolz JF, Basu P\n",
      "TITLE     Evolution of nitrate reductase: molecular and structural variations on a common function.\n",
      "JOURNAL   Chembiochem 3:198-206 (2002)\n",
      "DOI:10.1002/1439-7633(20020301)3:2/3<198::AID-CBIC198>3.0.CO;2-C\n",
      "REFERENCE   PMID:18021072\n",
      "AUTHORS   Morozkina EV, Zvyagilskaya RA\n",
      "TITLE     Nitrate reductases: structure, functions, and effect of stress factors.\n",
      "JOURNAL   Biochemistry (Mosc) 72:1151-60 (2007)\n",
      "DOI:10.1134/S0006297907100124\n",
      "REFERENCE   PMID:19247843\n",
      "AUTHORS   Jetten MS, Niftrik L, Strous M, Kartal B, Keltjens JT, Op den Camp HJ\n",
      "TITLE     Biochemistry and molecular biology of anammox bacteria.\n",
      "JOURNAL   Crit Rev Biochem Mol Biol 44:65-84 (2009)\n",
      "DOI:10.1080/10409230902722783\n",
      "REFERENCE   PMID:22221911\n",
      "AUTHORS   Luesken FA, Wu ML, Op den Camp HJ, Keltjens JT, Stunnenberg H, Francoijs KJ, Strous M, Jetten MS\n",
      "TITLE     Effect of oxygen on the anaerobic methanotroph 'Candidatus Methylomirabilis oxyfera': kinetic and transcriptional analysis.\n",
      "JOURNAL   Environ Microbiol 14:1024-34 (2012)\n",
      "DOI:10.1111/j.1462-2920.2011.02682.x\n",
      "REFERENCE   PMID:26610024\n",
      "AUTHORS   Daims H, Lebedeva EV, Pjevac P, Han P, Herbold C, Albertsen M, Jehmlich N, Palatinszky M, Vierheilig J, Bulaev A, Kirkegaard RH, von Bergen M, Rattei T, Bendinger B, Nielsen PH, Wagner M\n",
      "TITLE     Complete nitrification by Nitrospira bacteria.\n",
      "JOURNAL   Nature 528:504-9 (2015)\n",
      "DOI:10.1038/nature16461\n",
      "REFERENCE   PMID:26610025\n",
      "AUTHORS   van Kessel MA, Speth DR, Albertsen M, Nielsen PH, Op den Camp HJ, Kartal B, Jetten MS, Lucker S\n",
      "TITLE     Complete nitrification by a single microorganism.\n",
      "JOURNAL   Nature 528:555-9 (2015)\n",
      "DOI:10.1038/nature16459\n",
      "REL_PATHWAY ath00220  Arginine biosynthesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00460  Cyanoamino acid metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "KO_PATHWAY  ko00910\n",
      "///\n",
      "ENTRY       ath00920                    Pathway\n",
      "NAME        Sulfur metabolism - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Sulfur is an essential element for life and the metabolism of organic sulfur compounds plays an important role in the global sulfur cycle. Sulfur occurs in various oxidation states ranging from +6 in sulfate to -2 in sulfide (H2S). Sulfate reduction can occur in both an energy consuming assimilatory pathway and an energy producing dissimilatory pathway. The assimilatory pathway, which is found in a wide range of organisms, produces reduced sulfur compounds for the biosynthesis of S-containing amino acids and does not lead to direct excretion of sulfide. In the dissimilatory pathway, which is restricted to obligatory anaerobic bacterial and archaeal lineages, sulfate (or sulfur) is the terminal electron acceptor of the respiratory chain producing large quantities of inorganic sulfide. Both pathways start from the activation of sulfate by reaction with ATP to form adenylyl sulfate (APS). In the assimilatory pathway [MD:M00176] APS is converted to 3'-phosphoadenylyl sulfate (PAPS) and then reduced to sulfite, and sulfite is further reduced to sulfide by the assimilatory sulfite reductase. In the dissimilatory pathway [MD:M00596] APS is directly reduced to sulfite, and sulfite is further reduced to sulfide by the dissimilatory sulfite reductase. The capacity for oxidation of sulfur is quite widespread among bacteria and archaea, comprising phototrophs and chemolithoautotrophs. The SOX (sulfur-oxidation) system [MD:M00595] is a well-known sulfur oxidation pathway and is found in both photosynthetic and non-photosynthetic sulfur-oxidizing bacteria. Green sulfur bacteria and purple sulfur bacteria carry out anoxygenic photosynthesis with reduced sulfur compounds such as sulfide and elemental sulfur, as well as thiosulfate (in some species with the SOX system), as the electron donor for photoautotrophic growth. In some chemolithoautotrophic sulfur oxidizers (such as Thiobacillus denitrificans), it has been suggested that dissimilatory sulfur reduction enzymes operate in the reverse direction, forming a sulfur oxidation pathway from sulfite to APS and then to sulfate.\n",
      "CLASS       Metabolism; Energy metabolism\n",
      "PATHWAY_MAP ath00920  Sulfur metabolism\n",
      "MODULE      ath_M00021  Cysteine biosynthesis, serine => cysteine [PATH:ath00920]\n",
      "DBLINKS     GO: 0006790\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT4G14680  APS3; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT3G22890  APS1; ATP sulfurylase 1 [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT1G19920  APS2; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT5G43780  APS4; Pseudouridine synthase/archaeosine transglycosylase-like family protein [KO:K13811] [EC:2.7.7.4 2.7.1.25]\n",
      "AT4G16566  HINT4; histidine triad nucleotide-binding 4 [KO:K22966] [EC:2.7.7.5 3.6.2.1]\n",
      "AT5G67520  APK4; adenosine-5'-phosphosulfate (APS) kinase 4 [KO:K00860] [EC:2.7.1.25]\n",
      "AT3G03900  APK3; adenosine-5'-phosphosulfate (APS) kinase 3 [KO:K00860] [EC:2.7.1.25]\n",
      "AT2G14750  APK; APS kinase [KO:K00860] [EC:2.7.1.25]\n",
      "AT4G39940  AKN2; APS-kinase 2 [KO:K00860] [EC:2.7.1.25]\n",
      "AT5G54390  HL; HAL2-like protein [KO:K01082] [EC:3.1.3.7]\n",
      "AT4G05090  Inositol monophosphatase family protein [KO:K01082] [EC:3.1.3.7]\n",
      "AT5G09290  Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G63980  SAL1; SAL1 phosphatase-like protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G64000  SAL2; Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G63990  Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT4G04610  APR1; APS reductase 1 [KO:K05907] [EC:1.8.4.9]\n",
      "AT1G62180  APR2; 5'adenylylphosphosulfate reductase 2 [KO:K05907] [EC:1.8.4.9]\n",
      "AT4G21990  APR3; APS reductase 3 [KO:K05907] [EC:1.8.4.9]\n",
      "AT3G01910  SOX; sulfite oxidase [KO:K00387] [EC:1.8.3.1]\n",
      "AT5G04590  SIR; sulfite reductase [KO:K00392] [EC:1.8.7.1]\n",
      "AT1G53580  GLY3; glyoxalase II 3 [KO:K17725] [EC:1.13.11.18]\n",
      "AT1G79230  MST1; mercaptopyruvate sulfurtransferase 1 [KO:K01011] [EC:2.8.1.1 2.8.1.2]\n",
      "AT1G16460  RDH2; rhodanese homologue 2 [KO:K01011] [EC:2.8.1.1 2.8.1.2]\n",
      "AT1G55920  SERAT2;1; serine acetyltransferase 2;1 [KO:K00640] [EC:2.3.1.30]\n",
      "AT5G56760  SERAT1;1; serine acetyltransferase 1;1 [KO:K00640] [EC:2.3.1.30]\n",
      "AT3G13110  SERAT2;2; serine acetyltransferase 2;2 [KO:K00640] [EC:2.3.1.30]\n",
      "AT4G35640  SERAT3;2; serine acetyltransferase 3;2 [KO:K00640] [EC:2.3.1.30]\n",
      "AT2G17640  ATSERAT3;1; Trimeric LpxA-like enzymes superfamily protein [KO:K00640] [EC:2.3.1.30]\n",
      "AT1G55880  Pyridoxal-5'-phosphate-dependent enzyme family protein [KO:K01738] [EC:2.5.1.47]\n",
      "AT2G43750  OASB; O-acetylserine (thiol) lyase B [KO:K01738] [EC:2.5.1.47]\n",
      "AT3G04940  CYSD1; cysteine synthase D1 [KO:K01738] [EC:2.5.1.47]\n",
      "AT3G22460  OASA2; O-acetylserine (thiol) lyase (OAS-TL) isoform A2 [KO:K01738] [EC:2.5.1.47]\n",
      "AT5G28030  DES1; L-cysteine desulfhydrase 1 [KO:K01738] [EC:2.5.1.47]\n",
      "AT4G14880  OASA1; O-acetylserine (thiol) lyase (OAS-TL) isoform A1 [KO:K01738] [EC:2.5.1.47]\n",
      "AT3G59760  OASC; O-acetylserine (thiol) lyase isoform C [KO:K01738] [EC:2.5.1.47]\n",
      "AT5G28020  CYSD2; cysteine synthase D2 [KO:K01738] [EC:2.5.1.47]\n",
      "AT3G61440  CYSC1; cysteine synthase C1 [KO:K13034] [EC:2.5.1.47 4.4.1.9]\n",
      "AT3G01120  MTO1; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01739] [EC:2.5.1.48]\n",
      "AT1G33320  Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01739] [EC:2.5.1.48]\n",
      "AT4G14030  SBP1; selenium-binding protein 1 [KO:K17285] [EC:1.8.3.4]\n",
      "AT4G14040  SBP2; selenium-binding protein 2 [KO:K17285] [EC:1.8.3.4]\n",
      "AT3G23800  SBP3; selenium-binding protein 3 [KO:K17285] [EC:1.8.3.4]\n",
      "COMPOUND    C00033  Acetate\n",
      "C00042  Succinate\n",
      "C00053  3'-Phosphoadenylyl sulfate\n",
      "C00054  Adenosine 3',5'-bisphosphate\n",
      "C00059  Sulfate\n",
      "C00065  L-Serine\n",
      "C00084  Acetaldehyde\n",
      "C00087  Sulfur\n",
      "C00094  Sulfite\n",
      "C00097  L-Cysteine\n",
      "C00155  L-Homocysteine\n",
      "C00224  Adenylyl sulfate\n",
      "C00245  Taurine\n",
      "C00263  L-Homoserine\n",
      "C00283  Hydrogen sulfide\n",
      "C00320  Thiosulfate\n",
      "C00409  Methanethiol\n",
      "C00580  Dimethyl sulfide\n",
      "C00979  O-Acetyl-L-serine\n",
      "C01118  O-Succinyl-L-homoserine\n",
      "C01861  Trithionate\n",
      "C02084  Tetrathionate\n",
      "C03920  2-(Methylthio)ethanesulfonate\n",
      "C04022  S,S-Dimethyl-beta-propiothetin\n",
      "C08276  3-(Methylthio)propanoate\n",
      "C11142  Dimethyl sulfone\n",
      "C11143  Dimethyl sulfoxide\n",
      "C11145  Methanesulfonic acid\n",
      "C15521  Alkanesulfonate\n",
      "C17267  S-Sulfanylglutathione\n",
      "C19692  Polysulfide\n",
      "C20870  3-(Methylthio)propanoyl-CoA\n",
      "C20955  3-(Methylthio)acryloyl-CoA\n",
      "REFERENCE   PMID:22982583\n",
      "AUTHORS   Grein F, Ramos AR, Venceslau SS, Pereira IA\n",
      "TITLE     Unifying concepts in anaerobic respiration: Insights from dissimilatory sulfur metabolism.\n",
      "JOURNAL   Biochim Biophys Acta 1827:145-60 (2013)\n",
      "DOI:10.1016/j.bbabio.2012.09.001\n",
      "REFERENCE   PMID:22633058\n",
      "AUTHORS   Fauque GD, Barton LL\n",
      "TITLE     Hemoproteins in dissimilatory sulfate- and sulfur-reducing prokaryotes.\n",
      "JOURNAL   Adv Microb Physiol 60:1-90 (2012)\n",
      "DOI:10.1016/B978-0-12-398264-3.00001-2\n",
      "REFERENCE   PMID:20143161\n",
      "AUTHORS   Sakurai H, Ogawa T, Shiga M, Inoue K\n",
      "TITLE     Inorganic sulfur oxidizing system in green sulfur bacteria.\n",
      "JOURNAL   Photosynth Res 104:163-76 (2010)\n",
      "DOI:10.1007/s11120-010-9531-2\n",
      "REFERENCE   PMID:22092713\n",
      "AUTHORS   Falkenby LG, Szymanska M, Holkenbrink C, Habicht KS, Andersen JS, Miller M, Frigaard NU\n",
      "TITLE     Quantitative proteomics of Chlorobaculum tepidum: insights into the sulfur metabolism of a phototrophic green sulfur bacterium.\n",
      "JOURNAL   FEMS Microbiol Lett 323:142-50 (2011)\n",
      "DOI:10.1111/j.1574-6968.2011.02370.x\n",
      "REFERENCE   PMID:21833341\n",
      "AUTHORS   Gregersen LH, Bryant DA, Frigaard NU\n",
      "TITLE     Mechanisms and evolution of oxidative sulfur metabolism in green sulfur bacteria.\n",
      "JOURNAL   Front Microbiol 2:116 (2011)\n",
      "DOI:10.3389/fmicb.2011.00116\n",
      "REFERENCE   PMID:16452431\n",
      "AUTHORS   Beller HR, Chain PS, Letain TE, Chakicherla A, Larimer FW, Richardson PM, Coleman MA, Wood AP, Kelly DP.\n",
      "TITLE     The genome sequence of the obligately chemolithoautotrophic, facultatively anaerobic bacterium Thiobacillus denitrificans.\n",
      "JOURNAL   J Bacteriol 188:1473-88 (2006)\n",
      "DOI:10.1128/JB.188.4.1473-1488.2006\n",
      "REFERENCE   PMID:9695921\n",
      "AUTHORS   Pott AS, Dahl C\n",
      "TITLE     Sirohaem sulfite reductase and other proteins encoded by genes at the dsr locus of Chromatium vinosum are involved in the oxidation of intracellular sulfur.\n",
      "JOURNAL   Microbiology 144 ( Pt 7):1881-94 (1998)\n",
      "DOI:10.1099/00221287-144-7-1881\n",
      "REFERENCE   PMID:18929068\n",
      "AUTHORS   Frigaard NU, Dahl C\n",
      "TITLE     Sulfur metabolism in phototrophic sulfur bacteria.\n",
      "JOURNAL   Adv Microb Physiol 54:103-200 (2009)\n",
      "DOI:10.1016/S0065-2911(08)00002-7\n",
      "REL_PATHWAY ath00260  Glycine, serine and threonine metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "KO_PATHWAY  ko00920\n",
      "///\n",
      "ENTRY       ath00940                    Pathway\n",
      "NAME        Phenylpropanoid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Phenylpropanoids are a group of plant secondary metabolites derived from phenylalanine and having a wide variety of functions both as structural and signaling molecules. Phenylalanine is first converted to cinnamic acid by deamination. It is followed by hydroxylation and frequent methylation to generate coumaric acid and other acids with a phenylpropane (C6-C3) unit. Reduction of the CoA-activated carboxyl groups of these acids results in the corresponding aldehydes and alcohols. The alcohols are called monolignols, the starting compounds for biosynthesis of lignin.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00940  Phenylpropanoid biosynthesis\n",
      "MODULE      ath_M00039  Monolignol biosynthesis, phenylalanine/tyrosine => monolignol [PATH:ath00940]\n",
      "ath_M00137  Flavanone biosynthesis, phenylalanine => naringenin [PATH:ath00940]\n",
      "DBLINKS     GO: 0009699\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G10340  PAL4; phenylalanine ammonia-lyase 4 [KO:K10775] [EC:4.3.1.24]\n",
      "AT3G53260  PAL2; phenylalanine ammonia-lyase 2 [KO:K10775] [EC:4.3.1.24]\n",
      "AT2G37040  PAL1; PHE ammonia lyase 1 [KO:K10775] [EC:4.3.1.24]\n",
      "AT5G04230  PAL3; phenyl alanine ammonia-lyase 3 [KO:K10775] [EC:4.3.1.24]\n",
      "AT2G30490  C4H; cinnamate-4-hydroxylase [KO:K00487] [EC:1.14.14.91]\n",
      "AT4G05160  AMP-dependent synthetase and ligase family protein [KO:K01904] [EC:6.2.1.12]\n",
      "AT1G65060  4CL3; 4-coumarate:CoA ligase 3 [KO:K01904] [EC:6.2.1.12]\n",
      "AT1G62940  ACOS5; acyl-CoA synthetase 5 [KO:K01904] [EC:6.2.1.12]\n",
      "AT3G21240  4CL2; 4-coumarate:CoA ligase 2 [KO:K01904] [EC:6.2.1.12]\n",
      "AT1G51680  4CL1; 4-coumarate:CoA ligase 1 [KO:K01904] [EC:6.2.1.12]\n",
      "AT3G21230  4CL5; 4-coumarate:CoA ligase 5 [KO:K01904] [EC:6.2.1.12]\n",
      "AT4G19010  AMP-dependent synthetase and ligase family protein [KO:K01904] [EC:6.2.1.12]\n",
      "AT5G63380  AMP-dependent synthetase and ligase family protein [KO:K01904] [EC:6.2.1.12]\n",
      "AT5G48930  HCT; hydroxycinnamoyl-CoA shikimate/quinate hydroxycinnamoyl transferase [KO:K13065] [EC:2.3.1.133]\n",
      "AT2G19070  SHT; spermidine hydroxycinnamoyl transferase [KO:K13065] [EC:2.3.1.133]\n",
      "AT5G57840  HXXXD-type acyl-transferase family protein [KO:K13065] [EC:2.3.1.133]\n",
      "AT2G40890  CYP98A3; cytochrome P450, family 98, subfamily A, polypeptide 3 [KO:K09754] [EC:1.14.14.96]\n",
      "AT1G52760  LysoPL2; lysophospholipase 2 [KO:K18368] [EC:3.1.1.-]\n",
      "AT2G47630  alpha/beta-Hydrolases superfamily protein [KO:K18368] [EC:3.1.1.-]\n",
      "AT2G39420  alpha/beta-Hydrolases superfamily protein [KO:K18368] [EC:3.1.1.-]\n",
      "AT5G16120  alpha/beta-Hydrolases superfamily protein [KO:K18368] [EC:3.1.1.-]\n",
      "AT3G62860  alpha/beta-Hydrolases superfamily protein [KO:K18368] [EC:3.1.1.-]\n",
      "AT1G77420  alpha/beta-Hydrolases superfamily protein [KO:K18368] [EC:3.1.1.-]\n",
      "AT3G55180  alpha/beta-Hydrolases superfamily protein [KO:K18368] [EC:3.1.1.-]\n",
      "AT4G34050  CCoAOMT1; S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "AT1G24735  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "AT1G67980  CCOAMT; caffeoyl-CoA 3-O-methyltransferase [KO:K00588] [EC:2.1.1.104]\n",
      "AT4G26220  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "AT1G15950  CCR1; cinnamoyl coa reductase 1 [KO:K09753] [EC:1.2.1.44]\n",
      "AT1G80820  CCR2; cinnamoyl coa reductase [KO:K09753] [EC:1.2.1.44]\n",
      "AT3G24503  ALDH2C4; aldehyde dehydrogenase 2C4 [KO:K12355] [EC:1.2.1.68]\n",
      "AT4G36220  FAH1; ferulic acid 5-hydroxylase 1 [KO:K09755] [EC:1.14.-.-]\n",
      "AT5G04330  CYP84A4; Cytochrome P450 superfamily protein [KO:K09755] [EC:1.14.-.-]\n",
      "AT5G54160  OMT1; O-methyltransferase 1 [KO:K13066] [EC:2.1.1.68 2.1.1.4]\n",
      "AT3G19450  ATCAD4; GroES-like zinc-binding alcohol dehydrogenase family protein [KO:K00083] [EC:1.1.1.195]\n",
      "AT4G34230  CAD5; cinnamyl alcohol dehydrogenase 5 [KO:K00083] [EC:1.1.1.195]\n",
      "AT2G21730  CAD2; cinnamyl alcohol dehydrogenase homolog 2 [KO:K00083] [EC:1.1.1.195]\n",
      "AT4G37970  CAD6; cinnamyl alcohol dehydrogenase 6 [KO:K00083] [EC:1.1.1.195]\n",
      "AT2G21890  CAD3; cinnamyl alcohol dehydrogenase homolog 3 [KO:K00083] [EC:1.1.1.195]\n",
      "AT1G72680  CAD1; cinnamyl-alcohol dehydrogenase [KO:K00083] [EC:1.1.1.195]\n",
      "AT4G39330  CAD9; cinnamyl alcohol dehydrogenase 9 [KO:K00083] [EC:1.1.1.195]\n",
      "AT4G37980  ELI3-1; cinnamyl alcohol dehydrogenase 7 [KO:K00083] [EC:1.1.1.195]\n",
      "AT4G37990  ELI3-2; cinnamyl alcohol dehydrogenase 8 [KO:K00083] [EC:1.1.1.195]\n",
      "AT1G30760  FAD-binding Berberine family protein [KO:K22395] [EC:1.1.1.195]\n",
      "AT2G34790  MEE23; FAD-binding Berberine family protein [KO:K22395] [EC:1.1.1.195]\n",
      "AT1G48130  PER1; 1-cysteine peroxiredoxin 1 [KO:K11188] [EC:1.11.1.7 1.11.1.27 3.1.1.-]\n",
      "AT5G19880  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G49570  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G68850  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G71695  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G18140  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G18150  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G18980  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G22420  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G24800  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G34060  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G37130  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G38390  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G39040  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G41480  peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G43480  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G01190  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G03670  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G17070  Peroxidase family protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G21770  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G28200  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G49120  PRXCB; peroxidase CB [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G49960  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G08770  Prx37; Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G08780  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G11290  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G16270  peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G17690  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G21960  PRXR1; Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G26010  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G30170  Peroxidase family protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G31760  peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G33420  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G36430  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G37520  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G37530  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G05340  PRX52; Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G06720  PA2; peroxidase 2 [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G06730  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G14130  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G15180  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G17820  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G19890  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G22410  RHS18; root hair specific 18 [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G24070  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G40150  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G42180  PER64; Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G47000  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G51890  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G58390  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G58400  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G64100  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G64110  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G64120  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G66390  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G67400  RHS19; root hair specific 19 [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G05260  RCI3; Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G14540  PER4; Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G14550  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G24110  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G30870  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G34510  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G44970  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G38380  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G49110  PRXCA; peroxidase CA [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G32980  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G05240  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G25980  peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G77100  peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT3G50990  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT1G05250  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT2G35380  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G39580  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT4G33870  Peroxidase superfamily protein [KO:K00430] [EC:1.11.1.7]\n",
      "AT5G26310  UGT72E3; UDP-Glycosyltransferase superfamily protein [KO:K12356] [EC:2.4.1.111]\n",
      "AT3G50740  UGT72E1; UDP-glucosyl transferase 72E1 [KO:K12356] [EC:2.4.1.111]\n",
      "AT5G66690  UGT72E2; UDP-Glycosyltransferase superfamily protein [KO:K12356] [EC:2.4.1.111]\n",
      "AT3G21560  UGT84A2; UDP-Glycosyltransferase superfamily protein [KO:K13068] [EC:2.4.1.120]\n",
      "AT5G09640  SCPL19; serine carboxypeptidase-like 19 [KO:K09756] [EC:3.4.16.- 2.3.1.91]\n",
      "AT2G22990  SNG1; sinapoylglucose 1 [KO:K09757] [EC:3.4.16.- 2.3.1.92]\n",
      "AT5G42260  BGLU12; beta glucosidase 12 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G44640  BGLU13; beta glucosidase 13 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44450  BGLU15; beta glucosidase 15 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G26560  BGLU40; beta glucosidase 40 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44480  BGLU17; beta glucosidase 17 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G47050  Glycosyl hydrolase family protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G75940  ATA27; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G66280  BGLU22; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G66270  BGLU21; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G09260  PYK10; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G28510  BGLU24; beta glucosidase 24 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G36890  BGLU42; beta glucosidase 42 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G54570  BGLU41; beta glucosidase 41 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G32860  BGLU33; beta glucosidase 33 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G03640  BGLU25; beta glucosidase 25 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G21370  BGLU19; beta glucosidase 19 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60130  BGLU16; beta glucosidase 16 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G25630  BGLU14; beta glucosidase 14 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60140  DIN2; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44470  BGLU29; beta glucosidase 29 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44460  BGLU28; beta glucosidase 28 [KO:K01188] [EC:3.2.1.21]\n",
      "AT2G44490  PEN2; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G24550  BGLU32; beta glucosidase 32 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G24540  BGLU31; beta glucosidase 31 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G60120  BGLU27; beta glucosidase 27 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G02850  BGLU11; beta glucosidase 11 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G45191  BGLU1; Glycosyl hydrolase superfamily protein [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G22100  BGLU3; beta glucosidase 2 [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G27830  BGLU10; beta glucosidase 10 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G62750  BGLU8; beta glucosidase 8 [KO:K01188] [EC:3.2.1.21]\n",
      "AT5G16580  BGLU2; beta glucosidase 2 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G60270  BGLU6; beta glucosidase 6 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G62740  BGLU7; beta glucosidase 7 [KO:K01188] [EC:3.2.1.21]\n",
      "AT4G27820  BGLU9; beta glucosidase 9 [KO:K01188] [EC:3.2.1.21]\n",
      "AT1G60260  BGLU5; beta glucosidase 5 [KO:K01188] [EC:3.2.1.21]\n",
      "AT3G47000  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G20950  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G20940  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G47040  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT5G04885  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G47010  Glycosyl hydrolase family protein [KO:K05349] [EC:3.2.1.21]\n",
      "AT3G18080  BGLU44; B-S glucosidase 44 [KO:K05350] [EC:3.2.1.21]\n",
      "AT1G61820  BGLU46; beta glucosidase 46 [KO:K05350] [EC:3.2.1.21]\n",
      "AT4G21760  BGLU47; beta-glucosidase 47 [KO:K05350] [EC:3.2.1.21]\n",
      "AT1G61810  BGLU45; beta-glucosidase 45 [KO:K05350] [EC:3.2.1.21]\n",
      "AT3G18070  BGLU43; beta glucosidase 43 [KO:K05350] [EC:3.2.1.21]\n",
      "AT1G74540  CYP98A8; cytochrome P450, family 98, subfamily A, polypeptide 8 [KO:K15506] [EC:1.14.13.-]\n",
      "AT1G74550  CYP98A9; cytochrome P450, family 98, subfamily A, polypeptide 9 [KO:K23662] [EC:1.14.13.-]\n",
      "AT1G67990  TSM1; S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K13067] [EC:2.1.1.-]\n",
      "AT3G13610  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K06892] [EC:1.14.11.61]\n",
      "AT1G55290  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K06892] [EC:1.14.11.61]\n",
      "COMPOUND    C00079  L-Phenylalanine\n",
      "C00082  L-Tyrosine\n",
      "C00223  p-Coumaroyl-CoA\n",
      "C00315  Spermidine\n",
      "C00323  Caffeoyl-CoA\n",
      "C00406  Feruloyl-CoA\n",
      "C00411  Sinapoyl-CoA\n",
      "C00423  trans-Cinnamate\n",
      "C00482  Sinapate\n",
      "C00540  Cinnamoyl-CoA\n",
      "C00590  Coniferyl alcohol\n",
      "C00761  Coniferin\n",
      "C00811  4-Coumarate\n",
      "C00852  Chlorogenate\n",
      "C00903  Cinnamaldehyde\n",
      "C00933  Sinapine\n",
      "C01175  1-O-Sinapoyl-beta-D-glucose\n",
      "C01197  Caffeate\n",
      "C01494  Ferulate\n",
      "C01527  Scopolin\n",
      "C01533  Syringin\n",
      "C01752  Scopoletin\n",
      "C01772  trans-2-Hydroxycinnamate\n",
      "C02325  Sinapyl alcohol\n",
      "C02646  4-Coumaryl alcohol\n",
      "C02666  Coniferyl aldehyde\n",
      "C02887  Sinapoyl malate\n",
      "C02947  4-Coumaroylshikimate\n",
      "C04366  3-(2-Carboxyethenyl)-cis,cis-muconate\n",
      "C05158  trans-beta-D-Glucosyl-2-hydroxycinnamate\n",
      "C05608  4-Hydroxycinnamyl aldehyde\n",
      "C05610  Sinapoyl aldehyde\n",
      "C05619  5-Hydroxyferulic acid methyl ester\n",
      "C05627  4-Hydroxystyrene\n",
      "C05838  cis-2-Hydroxycinnamate\n",
      "C05839  cis-beta-D-Glucosyl-2-hydroxycinnamate\n",
      "C05851  Coumarin\n",
      "C05855  4-Hydroxycinnamyl alcohol 4-D-glucoside\n",
      "C06224  3,4-Dihydroxystyrene\n",
      "C09315  Umbelliferone\n",
      "C10428  Anethole\n",
      "C10434  5-O-Caffeoylshikimic acid\n",
      "C10452  Estragole\n",
      "C10453  Eugenol\n",
      "C10454  Methyleugenol\n",
      "C10469  Isoeugenol\n",
      "C10478  Methylisoeugenol\n",
      "C10945  Caffeic aldehyde\n",
      "C12203  5-Hydroxyferuloyl-CoA\n",
      "C12204  5-Hydroxyconiferaldehyde\n",
      "C12205  5-Hydroxyconiferyl alcohol\n",
      "C12206  Caffeyl alcohol\n",
      "C12208  p-Coumaroyl quinic acid\n",
      "C15804  p-Hydroxyphenyl lignin\n",
      "C15805  Guaiacyl lignin\n",
      "C15806  Syringyl lignin\n",
      "C15807  5-Hydroxy-guaiacyl lignin\n",
      "C16930  Chavicol\n",
      "C18069  N1,N5,N10-Tricoumaroyl spermidine\n",
      "C18070  N1,N5,N10-Tricaffeoyl spermidine\n",
      "C18071  N1,N5,N10-Triferuloyl spermidine\n",
      "C18072  N1,N5,N10-Tri(hydroxyferuloyl) spermidine\n",
      "C18073  N1,N5-Di(hydroxyferuloyl)-N10-sinapoyl spermidine\n",
      "C20225  Coniferyl acetate\n",
      "C20464  Isochavicol\n",
      "C20465  Coumaryl acetate\n",
      "C21459  6-Hydroxyferuloyl-CoA\n",
      "C22070  2,4-Dihydroxycinnamoyl-CoA\n",
      "REFERENCE   PMID:14729911\n",
      "AUTHORS   Nair RB, Bastress KL, Ruegger MO, Denault JW, Chapple C.\n",
      "TITLE     The Arabidopsis thaliana REDUCED EPIDERMAL FLUORESCENCE1 gene encodes an aldehyde dehydrogenase involved in ferulic acid and sinapic acid biosynthesis.\n",
      "JOURNAL   Plant Cell 16:544-54 (2004)\n",
      "DOI:10.1105/tpc.017509\n",
      "REFERENCE   PMID:14551235\n",
      "AUTHORS   Baucher M, Halpin C, Petit-Conil M, Boerjan W.\n",
      "TITLE     Lignin: genetic engineering and impact on pulping.\n",
      "JOURNAL   Crit Rev Biochem Mol Biol 38:305-50 (2003)\n",
      "DOI:10.1080/10409230391036757\n",
      "REFERENCE   PMID:15878986\n",
      "AUTHORS   Rogers LA, Dubos C, Cullis IF, Surman C, Poole M, Willment J, Mansfield SD, Campbell MM.\n",
      "TITLE     Light, the circadian clock, and sugar perception in the control of lignin biosynthesis.\n",
      "JOURNAL   J Exp Bot 56:1651-63 (2005)\n",
      "DOI:10.1093/jxb/eri162\n",
      "REFERENCE   PMID:12242395\n",
      "AUTHORS   Whetten R, Sederoff R.\n",
      "TITLE     Lignin Biosynthesis.\n",
      "JOURNAL   Plant Cell 7:1001-1013 (1995)\n",
      "DOI:10.1105/tpc.7.7.1001\n",
      "REFERENCE   PMID:8972602\n",
      "AUTHORS   Lorenzen M, Racicot V, Strack D, Chapple C.\n",
      "TITLE     Sinapic acid ester metabolism in wild type and a sinapoylglucose-accumulating mutant of arabidopsis.\n",
      "JOURNAL   Plant Physiol 112:1625-30 (1996)\n",
      "DOI:10.1104/pp.112.4.1625\n",
      "REFERENCE   PMID:1477555\n",
      "AUTHORS   Chapple CC, Vogt T, Ellis BE, Somerville CR.\n",
      "TITLE     An Arabidopsis mutant defective in the general phenylpropanoid pathway.\n",
      "JOURNAL   Plant Cell 4:1413-24 (1992)\n",
      "DOI:10.1105/tpc.4.11.1413\n",
      "REFERENCE   PMID:14612585\n",
      "AUTHORS   Raes J, Rohde A, Christensen JH, Van de Peer Y, Boerjan W.\n",
      "TITLE     Genome-wide characterization of the lignification toolbox in Arabidopsis.\n",
      "JOURNAL   Plant Physiol 133:1051-71 (2003)\n",
      "DOI:10.1104/pp.103.026484\n",
      "REFERENCE   PMID:14568076\n",
      "AUTHORS   Costa MA, Collins RE, Anterola AM, Cochrane FC, Davin LB, Lewis NG.\n",
      "TITLE     An in silico assessment of gene function and organization of the phenylpropanoid pathway metabolic networks in Arabidopsis thaliana and limitations thereof.\n",
      "JOURNAL   Phytochemistry 64:1097-112 (2003)\n",
      "DOI:10.1016/S0031-9422(03)00517-X\n",
      "REFERENCE   PMID:10934215\n",
      "AUTHORS   Meyermans H, Morreel K, Lapierre C, Pollet B, De Bruyn A, Busson R, Herdewijn P, Devreese B, Van Beeumen J, Marita JM, Ralph J, Chen C, Burggraeve B, Van Montagu M, Messens E, Boerjan W.\n",
      "TITLE     Modifications in lignin and accumulation of phenolic glucosides in poplar xylem upon down-regulation of caffeoyl-coenzyme A O-methyltransferase, an enzyme involved in lignin biosynthesis.\n",
      "JOURNAL   J Biol Chem 275:36899-909 (2000)\n",
      "DOI:10.1074/jbc.M006915200\n",
      "REFERENCE   PMID:16027976\n",
      "AUTHORS   Gachon CM, Langlois-Meurinne M, Henry Y, Saindrenan P.\n",
      "TITLE     Transcriptional co-regulation of secondary metabolism enzymes in Arabidopsis: functional and evolutionary implications.\n",
      "JOURNAL   Plant Mol Biol 58:229-45 (2005)\n",
      "DOI:10.1007/s11103-005-5346-5\n",
      "REFERENCE   PMID:10692459\n",
      "AUTHORS   Li L, Popko JL, Umezawa T, Chiang VL.\n",
      "TITLE     5-hydroxyconiferyl aldehyde modulates enzymatic methylation for syringyl monolignol formation, a new view of monolignol biosynthesis in angiosperms.\n",
      "JOURNAL   J Biol Chem 275:6537-45 (2000)\n",
      "DOI:10.1074/jbc.275.9.6537\n",
      "REFERENCE   PMID:12009990\n",
      "AUTHORS   Yamauchi K, Yasuda S, Fukushima K.\n",
      "TITLE     Evidence for the biosynthetic pathway from sinapic acid to syringyl lignin using labeled sinapic acid with stable isotope at both methoxy groups in Robinia pseudoacacia and Nerium indicum.\n",
      "JOURNAL   J Agric Food Chem 50:3222-7 (2002)\n",
      "DOI:10.1021/jf011565x\n",
      "REFERENCE   PMID:12084820\n",
      "AUTHORS   Eckardt NA.\n",
      "TITLE     Probing the mysteries of lignin biosynthesis: the crystal structure of caffeic acid/5-hydroxyferulic acid 3/5-O-methyltransferase provides new insights.\n",
      "JOURNAL   Plant Cell 14:1185-9 (2002)\n",
      "DOI:10.1105/tpc.140610\n",
      "REL_PATHWAY ath00130  Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00941  Flavonoid biosynthesis\n",
      "ath00945  Stilbenoid, diarylheptanoid and gingerol biosynthesis\n",
      "KO_PATHWAY  ko00940\n",
      "///\n",
      "ENTRY       ath00941                    Pathway\n",
      "NAME        Flavonoid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Flavonoids are a major class of plant secondary metabolites that serves a multitude of functions including pigments and antioxidant activity. Flavonoids are synthesized from phenylpropanoid derivatives by condensation with malonyl-CoA. For example, condensation of p-coumaroyl-CoA (C6-C3) with three malonyl-CoA (C3) molecules results in naringenin chalcone with a diphenylpropane (C6-C3-C6) unit, which is converted to naringenin with the flavone (2-phenylchromen-4-one) backbone by conjugate ring closure. These and further modifications yield a variety of structural forms including chalcones, flavanones, dihyroflavonols, and flavans, anthocyanins, flavones and flavonols, and isoflavonoids.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00941  Flavonoid biosynthesis\n",
      "MODULE      ath_M00137  Flavanone biosynthesis, phenylalanine => naringenin [PATH:ath00941]\n",
      "ath_M00138  Flavonoid biosynthesis, naringenin => pelargonidin [PATH:ath00941]\n",
      "DBLINKS     GO: 0009813\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G13930  TT4; Chalcone and stilbene synthase family protein [KO:K00660] [EC:2.3.1.74]\n",
      "AT5G66220  Chalcone-flavanone isomerase family protein [KO:K01859] [EC:5.5.1.6]\n",
      "AT3G55120  TT5; Chalcone-flavanone isomerase family protein [KO:K01859] [EC:5.5.1.6]\n",
      "AT5G05270  CHIL; Chalcone-flavanone isomerase family protein [KO:K01859] [EC:5.5.1.6]\n",
      "AT3G51240  F3H; flavanone 3-hydroxylase [KO:K00475] [EC:1.14.11.9]\n",
      "AT5G08640  FLS1; flavonol synthase 1 [KO:K05278] [EC:1.14.20.6]\n",
      "AT5G63590  FLS3; flavonol synthase 3 [KO:K05278] [EC:1.14.20.6]\n",
      "AT5G43935  FLS6; flavonol synthase 6 [KO:K05278] [EC:1.14.20.6]\n",
      "AT5G63600  FLS5; flavonol synthase 5 [KO:K05278] [EC:1.14.20.6]\n",
      "AT5G63595  FLS4; flavonol synthase 4 [KO:K05278] [EC:1.14.20.6]\n",
      "AT5G63580  FLS2; flavonol synthase 2 [KO:K05278] [EC:1.14.20.6]\n",
      "AT2G30490  C4H; cinnamate-4-hydroxylase [KO:K00487] [EC:1.14.14.91]\n",
      "AT5G07990  TT7; Cytochrome P450 superfamily protein [KO:K05280] [EC:1.14.14.82]\n",
      "AT5G42800  DFR; dihydroflavonol 4-reductase [KO:K13082] [EC:1.1.1.219 1.1.1.234]\n",
      "AT4G22870  2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily protein [KO:K05277] [EC:1.14.20.4]\n",
      "AT4G22880  LDOX; leucoanthocyanidin dioxygenase [KO:K05277] [EC:1.14.20.4]\n",
      "AT1G61720  BAN; NAD(P)-binding Rossmann-fold superfamily protein [KO:K08695] [EC:1.3.1.77]\n",
      "AT5G48930  HCT; hydroxycinnamoyl-CoA shikimate/quinate hydroxycinnamoyl transferase [KO:K13065] [EC:2.3.1.133]\n",
      "AT2G19070  SHT; spermidine hydroxycinnamoyl transferase [KO:K13065] [EC:2.3.1.133]\n",
      "AT5G57840  HXXXD-type acyl-transferase family protein [KO:K13065] [EC:2.3.1.133]\n",
      "AT2G40890  CYP98A3; cytochrome P450, family 98, subfamily A, polypeptide 3 [KO:K09754] [EC:1.14.14.96]\n",
      "AT4G34050  CCoAOMT1; S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "AT1G24735  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "AT1G67980  CCOAMT; caffeoyl-CoA 3-O-methyltransferase [KO:K00588] [EC:2.1.1.104]\n",
      "AT4G26220  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "COMPOUND    C00223  p-Coumaroyl-CoA\n",
      "C00323  Caffeoyl-CoA\n",
      "C00389  Quercetin\n",
      "C00406  Feruloyl-CoA\n",
      "C00509  Naringenin\n",
      "C00540  Cinnamoyl-CoA\n",
      "C00774  Phloretin\n",
      "C00852  Chlorogenate\n",
      "C00974  Dihydrokaempferol\n",
      "C01378  Fustin\n",
      "C01460  Vitexin\n",
      "C01477  Apigenin\n",
      "C01514  Luteolin\n",
      "C01604  Phlorizin\n",
      "C01617  Taxifolin\n",
      "C01709  Hesperetin\n",
      "C02906  Dihydromyricetin\n",
      "C02947  4-Coumaroylshikimate\n",
      "C03648  cis-3,4-Leucopelargonidin\n",
      "C05334  Isosakuranetin\n",
      "C05631  Eriodictyol\n",
      "C05903  Kaempferol\n",
      "C05904  Pelargonidin\n",
      "C05905  Cyanidin\n",
      "C05906  Leucocyanidin\n",
      "C05907  Luteoforol\n",
      "C05908  Delphinidin\n",
      "C05909  Leucodelphinidin\n",
      "C05911  Pentahydroxyflavanone\n",
      "C06561  Naringenin chalcone\n",
      "C06562  (+)-Catechin\n",
      "C08578  Butein\n",
      "C08650  Isoliquiritigenin\n",
      "C09099  Naringenin 7-O-beta-D-glucoside\n",
      "C09320  Afzelechin\n",
      "C09614  Butin\n",
      "C09727  (-)-Epicatechin\n",
      "C09736  Fisetinidol-4beta-ol\n",
      "C09751  Garbanzol\n",
      "C09756  Homoeriodictyol\n",
      "C09762  Liquiritigenin\n",
      "C09789  Naringin\n",
      "C09806  Neohesperidin\n",
      "C09826  Pinobanksin\n",
      "C09827  Pinocembrin\n",
      "C09833  Sakuranetin\n",
      "C10028  Chrysin\n",
      "C10044  Galangin\n",
      "C10107  Myricetin\n",
      "C10192  Tricetin\n",
      "C10434  5-O-Caffeoylshikimic acid\n",
      "C12123  7,4'-Dihydroxyflavone\n",
      "C12124  Apiforol\n",
      "C12127  (+)-Gallocatechin\n",
      "C12128  (-)-Epiafzelechin\n",
      "C12136  (-)-Epigallocatechin\n",
      "C12208  p-Coumaroyl quinic acid\n",
      "C15525  Eriodictyol chalcone\n",
      "C16404  Pinocembrin chalcone\n",
      "C16405  Homoeriodictyol chalcone\n",
      "C16406  Phlorizin chalcone\n",
      "C16407  2',4,4',6'-Tetrahydroxychalcone 4'-O-glucoside\n",
      "C16408  2',3,4,4',6'-Peptahydroxychalcone 4'-O-glucoside\n",
      "C16409  Aureusidin 6-O-glucoside\n",
      "C16410  Bracteatin 6-O-glucoside\n",
      "C16415  5-Deoxyleucopelargonidin\n",
      "C16416  Desmethylxanthohumol\n",
      "C16417  Xanthohumol\n",
      "C16418  Pinobanksin 3-O-acetate\n",
      "C16419  Pinostrobin\n",
      "C16422  Hesperetin 7-O-glucoside\n",
      "C16492  8-C-Glucosylnaringenin\n",
      "C21913  Dihydro-4-coumaroyl-CoA\n",
      "C22048  4-O-Methylxanthohumol\n",
      "REFERENCE   PMID:16945088\n",
      "AUTHORS   Tsai CJ, Harding SA, Tschaplinski TJ, Lindroth RL, Yuan Y.\n",
      "TITLE     Genome-wide analysis of the structural genes regulating defense phenylpropanoid metabolism in Populus.\n",
      "JOURNAL   New Phytol 172:47-62 (2006)\n",
      "DOI:10.1111/j.1469-8137.2006.01798.x\n",
      "REFERENCE   PMID:16367960\n",
      "AUTHORS   Ono E, Hatayama M, Isono Y, Sato T, Watanabe R, Yonekura-Sakakibara K, Fukuchi-Mizutani M, Tanaka Y, Kusumi T, Nishino T, Nakayama T.\n",
      "TITLE     Localization of a flavonoid biosynthetic polyphenol oxidase in vacuoles.\n",
      "JOURNAL   Plant J 45:133-43 (2006)\n",
      "DOI:10.1111/j.1365-313X.2005.02625.x\n",
      "REFERENCE   PMID:16774647\n",
      "AUTHORS   Morreel K, Goeminne G, Storme V, Sterck L, Ralph J, Coppieters W, Breyne P, Steenackers M, Georges M, Messens E, Boerjan W.\n",
      "TITLE     Genetical metabolomics of flavonoid biosynthesis in Populus: a case study.\n",
      "JOURNAL   Plant J 47:224-37 (2006)\n",
      "DOI:10.1111/j.1365-313X.2006.02786.x\n",
      "REFERENCE   PMID:16832053\n",
      "AUTHORS   Ono E, Fukuchi-Mizutani M, Nakamura N, Fukui Y, Yonekura-Sakakibara K, Yamaguchi M, Nakayama T, Tanaka T, Kusumi T, Tanaka Y.\n",
      "TITLE     Yellow flowers generated by expression of the aurone biosynthetic pathway.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 103:11075-80 (2006)\n",
      "DOI:10.1073/pnas.0604246103\n",
      "REFERENCE\n",
      "AUTHORS   Halbwirth H, Martens S, Wienand U, Forkmann G, Stich K.\n",
      "TITLE     Synthesis of (14C)-labeled 5-deoxyflavonoids and their application in the study of dihydroflavonol/leucoanthocyanidin interconversion by dihydroflavonol 4-reductase.\n",
      "JOURNAL   Plant Sci 170:587-595 (2006)\n",
      "DOI:10.1016/j.plantsci.2005.10.013\n",
      "REFERENCE   PMID:15231405\n",
      "AUTHORS   Stevens JF, Page JE.\n",
      "TITLE     Xanthohumol and related prenylflavonoids from hops and beer: to your good health!\n",
      "JOURNAL   Phytochemistry 65:1317-30 (2004)\n",
      "DOI:10.1016/j.phytochem.2004.04.025\n",
      "REFERENCE\n",
      "AUTHORS   Ogata J, Itoh Y, Ishida M, Yoshida H, Ozeki Y\n",
      "TITLE     Cloning and heterologous expression of cDNAs encoding flavonoid glucosyltransferases from Dianthus caryophyllus.\n",
      "JOURNAL   Plant Biotechnol 21:367-375 (2004)\n",
      "DOI:10.5511/plantbiotechnology.21.367\n",
      "REFERENCE   PMID:12828368\n",
      "AUTHORS   Springob K, Nakajima J, Yamazaki M, Saito K.\n",
      "TITLE     Recent advances in the biosynthesis and accumulation of anthocyanins.\n",
      "JOURNAL   Nat Prod Rep 20:288-303 (2003)\n",
      "DOI:10.1039/b109542k\n",
      "REFERENCE   PMID:12644647\n",
      "AUTHORS   Shimada N, Aoki T, Sato S, Nakamura Y, Tabata S, Ayabe S.\n",
      "TITLE     A cluster of genes encodes the two types of chalcone isomerase involved in the biosynthesis of general flavonoids and legume-specific 5-deoxy(iso)flavonoids in Lotus japonicus.\n",
      "JOURNAL   Plant Physiol 131:941-51 (2003)\n",
      "DOI:10.1104/pp.004820\n",
      "REFERENCE\n",
      "AUTHORS   Halbwirth H, Martens S, Wienand U, Forkmann G, Stich K.\n",
      "TITLE     Biochemical formation of anthocyanins in silk tissue of Zea mays.\n",
      "JOURNAL   Plant Sci 164:489-495 (2003)\n",
      "DOI:10.1016/S0168-9452(02)00433-8\n",
      "REFERENCE\n",
      "AUTHORS   Martens S, Knott J, Seitz CA, Janvari L, Yu SN, Forkmann G.\n",
      "TITLE     Impact of biochemical pre-studies on specific metabolic engineering strategies of flavonoid biosynthesis in plant tissues.\n",
      "JOURNAL   Biochem Eng J 14:227-235 (2003)\n",
      "DOI:10.1016/S1369-703X(02)00224-3\n",
      "REFERENCE   PMID:11073455\n",
      "AUTHORS   Nakayama T, Yonekura-Sakakibara K, Sato T, Kikuchi S, Fukui Y, Fukuchi-Mizutani M, Ueda T, Nakao M, Tanaka Y, Kusumi T, Nishino T.\n",
      "TITLE     Aureusidin synthase: a polyphenol oxidase homolog responsible for flower coloration.\n",
      "JOURNAL   Science 290:1163-6 (2000)\n",
      "DOI:10.1126/science.290.5494.1163\n",
      "REFERENCE   PMID:16667183\n",
      "AUTHORS   Lewinsohn E, Britsch L, Mazur Y, Gressel J\n",
      "TITLE     Flavanone Glycoside Biosynthesis in Citrus: Chalcone Synthase, UDP-Glucose:Flavanone-7-O-Glucosyl-Transferase and -Rhamnosyl-Transferase Activities in Cell-Free Extracts.\n",
      "JOURNAL   Plant Physiol 91:1323-8 (1989)\n",
      "DOI:10.1104/pp.91.4.1323\n",
      "REL_PATHWAY ath00940  Phenylpropanoid biosynthesis\n",
      "ath00942  Anthocyanin biosynthesis\n",
      "ath00944  Flavone and flavonol biosynthesis\n",
      "KO_PATHWAY  ko00941\n",
      "///\n",
      "ENTRY       ath00942                    Pathway\n",
      "NAME        Anthocyanin biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Anthocyanidins (aglycones) and anthocyanins (glycosides) are common plant pigments and belong to a structural subclass of flavonoids characterized by a 2-phenylbenzopyrylium unit. They are derived along the flavonoid modification pathways and further separated into three types, pelargonidin, cyanidin, and delphinidin, due to the different number of hydroxyl groups in the phenyl group.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00942  Anthocyanin biosynthesis\n",
      "DBLINKS     GO: 0009718\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G54060  UF3GT; UDP-glucose:flavonoid 3-o-glucosyltransferase [KO:K17193] [EC:2.4.2.51]\n",
      "AT4G14090  UDP-Glycosyltransferase superfamily protein [KO:K12338] [EC:2.4.1.298]\n",
      "COMPOUND    C05904  Pelargonidin\n",
      "C05905  Cyanidin\n",
      "C05908  Delphinidin\n",
      "C08604  Chrysanthemin\n",
      "C08620  Cyanidin 3-O-rutinoside\n",
      "C08639  Cyanin\n",
      "C08641  Gentiodelphin\n",
      "C08725  Pelargonin\n",
      "C12095  Cyanidin 3-O-(6-O-p-coumaroyl)glucoside\n",
      "C12096  Cyanidin 3-O-(6-O-p-coumaroyl)glucoside-5-O-glucoside\n",
      "C12137  Pelargonidin 3-O-glucoside\n",
      "C12138  Delphinidin 3-O-glucoside\n",
      "C12139  Petunidin 3-O-glucoside\n",
      "C12140  Malvidin 3-O-glucoside\n",
      "C12141  Peonidin 3-O-glucoside\n",
      "C12640  Pelargonidin 3-O-(6-caffeoyl-beta-D-glucoside) 5-O-beta-D-glucoside\n",
      "C12641  4'''-Demalonylsalvianin\n",
      "C12642  Pelargonidin 3-O-(6-O-malonyl-beta-D-glucoside)\n",
      "C12643  Cyanidin 3-O-(6-O-malonyl-beta-D-glucoside)\n",
      "C12644  Pelargonidin 3-O-rutinoside\n",
      "C12645  Pelargonidin 3-O-rutinoside 5-O-beta-D-glucoside\n",
      "C12646  Cyanidin 3-O-rutinoside 5-O-beta-D-glucoside\n",
      "C12647  Salvianin\n",
      "C16288  Pelargonidin 3-O-3'',6''-O-dimalonylglucoside\n",
      "C16289  Cyanidin 3-O-3'',6''-O-dimalonylglucoside\n",
      "C16290  Delphinidin 3-O-3'',6''-O-dimalonylglucoside\n",
      "C16292  Cyanidin-3-(p-coumaroyl)-rutinoside-5-glucoside\n",
      "C16293  Peonidin-3-(p-coumaroyl)-rutinoside-5-glucoside\n",
      "C16294  Delphinidin-3-(p-coumaroyl)-rutinoside-5-glucoside\n",
      "C16295  Petunidin-3-(p-coumaroyl)-rutinoside-5-glucoside\n",
      "C16296  Malvidin-3-(p-coumaroyl)-rutinoside-5-glucoside\n",
      "C16297  Pelargonidin 3-O-(6-caffeoyl-beta-D-glucoside)\n",
      "C16298  Cyanidin 5-O-glucoside\n",
      "C16299  Malonylshisonin\n",
      "C16301  Delphinidin 3-O-(6''-O-malonyl)-beta-D-glucoside\n",
      "C16302  Ternatin A1\n",
      "C16303  Ternatin C5\n",
      "C16304  Delphinidin 3-O-(6''-O-malonyl)-beta-glucoside-3'-O-beta-glucoside\n",
      "C16305  Pelargonidin 3-O-sophoroside\n",
      "C16306  Cyanidin 3-O-sophoroside\n",
      "C16307  Delphinidin 3-O-sophoroside\n",
      "C16312  Delphin\n",
      "C16313  Delphinidin 3-glucoside 5-caffoyl-glucoside\n",
      "C16314  Delphinidin 3,5,3'-triglucoside\n",
      "C16315  Tulipanin\n",
      "C16349  Pelargonidin 3-O-beta-D-glucoside 5-O-(6-coumaroyl-beta-D-glucoside)\n",
      "C16350  Cyanidin 3-O-beta-D-glucoside 5-O-(6-coumaroyl-beta-D-glucoside)\n",
      "C16351  Delphinidin 3-O-beta-D-glucoside 5-O-(6-coumaroyl-beta-D-glucoside)\n",
      "C16354  Albireodelphin\n",
      "C16367  Delphinidin 3-O-(6-caffeoyl-beta-D-glucoside)\n",
      "C16368  Pelargonidin 3-(6-p-coumaroyl)glucoside\n",
      "C16369  Cyanidin 3-(6-p-caffeoyl)glucoside\n",
      "C16370  Delphinidin 3-(6-p-coumaroyl)glucoside\n",
      "C16371  Pelargonidin 3-glucoside 5-caffeoylglucoside\n",
      "C16372  Cyanidin 3-glucoside 5-caffeoylglucoside\n",
      "C16373  Cyanidin-3-O-(6''-O-malonyl-2''-O-glucuronyl)glucoside\n",
      "C19762  Cyanidin 3-O-(2-O-beta-D-glucuronosyl)-beta-D-glucoside\n",
      "C20469  Cyanidin 3,7-di-O-beta-D-glucoside\n",
      "C20489  Pelargonidin 3-O-beta-D-sambubioside\n",
      "C20490  Cyanidin 3-O-beta-D-sambubioside\n",
      "C20491  Delphinidin 3-O-beta-D-sambubioside\n",
      "C20492  Pelargonidin 5-O-beta-D-glucoside 3-O-beta-D-sambubioside\n",
      "C20493  Cyanidin 5-O-beta-D-glucoside 3-O-beta-D-sambubioside\n",
      "C20494  Delphinidin 5-O-beta-D-glucoside 3-O-beta-D-sambubioside\n",
      "C20495  Pelargonidin 3,7-di-O-beta-D-glucoside\n",
      "C20496  Delphinidin 3,7-di-O-beta-D-glucoside\n",
      "REFERENCE   PMID:17383962\n",
      "AUTHORS   Unno H, Ichimaida F, Suzuki H, Takahashi S, Tanaka Y, Saito A, Nishino T, Kusunoki M, Nakayama T.\n",
      "TITLE     Structural and mutational studies of anthocyanin malonyltransferases establish the features of BAHD enzyme catalysis.\n",
      "JOURNAL   J Biol Chem 282:15812-22 (2007)\n",
      "DOI:10.1074/jbc.M700638200\n",
      "REFERENCE   PMID:16887235\n",
      "AUTHORS   Kogawa K, Kazuma K, Kato N, Noda N, Suzuki M.\n",
      "TITLE     Biosynthesis of malonylated flavonoid glycosides on the basis of malonyltransferase activity in the petals of Clitoria ternatea.\n",
      "JOURNAL   J Plant Physiol 164:886-94 (2007)\n",
      "DOI:10.1016/j.jplph.2006.05.006\n",
      "REFERENCE   PMID:16945088\n",
      "AUTHORS   Tsai CJ, Harding SA, Tschaplinski TJ, Lindroth RL, Yuan Y.\n",
      "TITLE     Genome-wide analysis of the structural genes regulating defense phenylpropanoid metabolism in Populus.\n",
      "JOURNAL   New Phytol 172:47-62 (2006)\n",
      "DOI:10.1111/j.1469-8137.2006.01798.x\n",
      "REFERENCE   PMID:15944692\n",
      "AUTHORS   Ogata J, Kanno Y, Itoh Y, Tsugawa H, Suzuki M.\n",
      "TITLE     Plant biochemistry: anthocyanin biosynthesis in roses.\n",
      "JOURNAL   Nature 435:757-8 (2005)\n",
      "DOI:10.1038/nature435757a\n",
      "REFERENCE   PMID:15860422\n",
      "AUTHORS   Bowles D, Isayenkova J, Lim EK, Poppenberger B.\n",
      "TITLE     Glycosyltransferases: managers of small molecules.\n",
      "JOURNAL   Curr Opin Plant Biol 8:254-63 (2005)\n",
      "DOI:10.1016/j.pbi.2005.03.007\n",
      "REFERENCE   PMID:15842621\n",
      "AUTHORS   Morita Y, Hoshino A, Kikuchi Y, Okuhara H, Ono E, Tanaka Y, Fukui Y, Saito N, Nitasaka E, Noguchi H, Iida S.\n",
      "TITLE     Japanese morning glory dusky mutants displaying reddish-brown or purplish-gray flowers are deficient in a novel glycosylation enzyme for anthocyanin biosynthesis, UDP-glucose:anthocyanidin 3-O-glucoside-2''-O-glucosyltransferase, due to 4-bp insertions in the gene.\n",
      "JOURNAL   Plant J 42:353-63 (2005)\n",
      "DOI:10.1111/j.1365-313X.2005.02383.x\n",
      "REFERENCE   PMID:15509561\n",
      "AUTHORS   Sawada S, Suzuki H, Ichimaida F, Yamaguchi MA, Iwashita T, Fukui Y, Hemmi H, Nishino T, Nakayama T.\n",
      "TITLE     UDP-glucuronic acid:anthocyanin glucuronosyltransferase from red daisy (Bellis perennis) flowers. Enzymology and phylogenetics of a novel glucuronosyltransferase involved in flower pigment biosynthesis.\n",
      "JOURNAL   J Biol Chem 280:899-906 (2005)\n",
      "DOI:10.1074/jbc.M410537200\n",
      "REFERENCE   PMID:15165190\n",
      "AUTHORS   Suzuki H, Sawada S, Watanabe K, Nagae S, Yamaguchi MA, Nakayama T, Nishino T.\n",
      "TITLE     Identification and characterization of a novel anthocyanin malonyltransferase from scarlet sage (Salvia splendens) flowers: an enzyme that is phylogenetically separated from other anthocyanin acyltransferases.\n",
      "JOURNAL   Plant J 38:994-1003 (2004)\n",
      "DOI:10.1111/j.1365-313X.2004.02101.x\n",
      "REFERENCE\n",
      "AUTHORS   Suzuki H, Nakayama T, Yamaguchi M, Nishino T.\n",
      "TITLE     cDNA cloning and characterization of two Dendranthema x morifolium anthocyanin malonyltransferases with different functional activities.\n",
      "JOURNAL   Plant Sci 166:89-96 (2004)\n",
      "DOI:10.1016/j.plantsci.2003.08.010\n",
      "REFERENCE\n",
      "AUTHORS   Nakayama T, Suzuki H, Nishino T.\n",
      "TITLE     Anthocyanin acyltransferases: specificities, mechanism, phylogenetics, and applications.\n",
      "JOURNAL   J Mol Catal B Enzym 23:117-132 (2003)\n",
      "DOI:10.1016/S1381-1177(03)00078-X\n",
      "REFERENCE   PMID:12857844\n",
      "AUTHORS   Fukuchi-Mizutani M, Okuhara H, Fukui Y, Nakao M, Katsumoto Y, Yonekura-Sakakibara K, Kusumi T, Hase T, Tanaka Y.\n",
      "TITLE     Biochemical and molecular characterization of a novel UDP-glucose:anthocyanin 3'-O-glucosyltransferase, a key enzyme for blue anthocyanin biosynthesis, from gentian.\n",
      "JOURNAL   Plant Physiol 132:1652-63 (2003)\n",
      "DOI:10.1104/pp.102.018242\n",
      "REFERENCE   PMID:12828368\n",
      "AUTHORS   Springob K, Nakajima J, Yamazaki M, Saito K.\n",
      "TITLE     Recent advances in the biosynthesis and accumulation of anthocyanins.\n",
      "JOURNAL   Nat Prod Rep 20:288-303 (2003)\n",
      "DOI:10.1039/b109542k\n",
      "REFERENCE   PMID:11598135\n",
      "AUTHORS   Suzuki H, Nakayama T, Yonekura-Sakakibara K, Fukui Y, Nakamura N, Nakao M, Tanaka Y, Yamaguchi MA, Kusumi T, Nishino T\n",
      "TITLE     Malonyl-CoA:anthocyanin 5-O-glucoside-6\"'-O-malonyltransferase from scarlet sage (Salvia splendens) flowers. Enzyme purification, gene cloning, expression, and characterization.\n",
      "JOURNAL   J Biol Chem 276:49013-9 (2001)\n",
      "DOI:10.1074/jbc.M108444200\n",
      "REFERENCE   PMID:10536025\n",
      "AUTHORS   Burbulis IE, Winkel-Shirley B.\n",
      "TITLE     Interactions among enzymes of the Arabidopsis flavonoid biosynthetic pathway.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 96:12929-34 (1999)\n",
      "DOI:10.1073/pnas.96.22.12929\n",
      "REFERENCE   PMID:10504566\n",
      "AUTHORS   Brugliera F, Barri-Rewell G, Holton TA, Mason JG.\n",
      "TITLE     Isolation and characterization of a flavonoid 3'-hydroxylase cDNA clone corresponding to the Ht1 locus of Petunia hybrida.\n",
      "JOURNAL   Plant J 19:441-51 (1999)\n",
      "DOI:10.1046/j.1365-313X.1999.00539.x\n",
      "REFERENCE   PMID:12226348\n",
      "AUTHORS   Boss PK, Davies C, Robinson SP.\n",
      "TITLE     Analysis of the Expression of Anthocyanin Pathway Genes in Developing Vitis vinifera L. cv Shiraz Grape Berries and the Implications for Pathway Regulation.\n",
      "JOURNAL   Plant Physiol 111:1059-1066 (1996)\n",
      "DOI:10.1104/pp.111.4.1059\n",
      "REFERENCE   PMID:12242398\n",
      "AUTHORS   Holton TA, Cornish EC.\n",
      "TITLE     Genetics and Biochemistry of Anthocyanin Biosynthesis.\n",
      "JOURNAL   Plant Cell 7:1071-1083 (1995)\n",
      "DOI:10.1105/tpc.7.7.1071\n",
      "REFERENCE   PMID:8130799\n",
      "AUTHORS   Kroon J, Souer E, de Graaff A, Xue Y, Mol J, Koes R.\n",
      "TITLE     Cloning and structural analysis of the anthocyanin pigmentation locus Rt of Petunia hybrida: characterization of insertion sequences in two mutant alleles.\n",
      "JOURNAL   Plant J 5:69-80 (1994)\n",
      "DOI:10.1046/j.1365-313X.1994.5010069.x\n",
      "REL_PATHWAY ath00941  Flavonoid biosynthesis\n",
      "KO_PATHWAY  ko00942\n",
      "///\n",
      "ENTRY       ath00944                    Pathway\n",
      "NAME        Flavone and flavonol biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Flavones and flavonols (3-hydroxyflavones) are common flavonoids in the plant kingdom. They are synthesized as part of the flavonoid modification pathways in aglycone and glucoside forms.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00944  Flavone and flavonol biosynthesis\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G07990  TT7; Cytochrome P450 superfamily protein [KO:K05280] [EC:1.14.14.82]\n",
      "AT1G30530  UGT78D1; UDP-glucosyl transferase 78D1 [KO:K15787] [EC:2.4.1.-]\n",
      "AT2G36790  UGT73C6; UDP-glucosyl transferase 73C6 [KO:K22771] [EC:2.4.1.-]\n",
      "AT5G17050  UGT78D2; UDP-glucosyl transferase 78D2 [KO:K10757] [EC:2.4.1.91]\n",
      "COMPOUND    C00389  Quercetin\n",
      "C00616  Quercetin 3-sulfate\n",
      "C01155  Quercetin 3,3'-bissulfate\n",
      "C01156  Quercetin 3,4'-bissulfate\n",
      "C01265  3,7-Di-O-methylquercetin\n",
      "C01460  Vitexin\n",
      "C01470  Acacetin\n",
      "C01477  Apigenin\n",
      "C01514  Luteolin\n",
      "C01714  Isovitexin\n",
      "C01750  Quercitrin\n",
      "C03515  Luteolin 7-O-glucuronide\n",
      "C03897  Quercetin 3,3',7-trissulfate\n",
      "C03898  Quercetin 3,4',7-trissulfate\n",
      "C03951  Luteolin 7-O-beta-D-glucoside\n",
      "C04024  Vitexin 2''-O-beta-D-glucoside\n",
      "C04199  Isovitexin 2''-O-beta-D-glucoside\n",
      "C04293  Chrysoeriol\n",
      "C04443  3-O-Methylquercetin\n",
      "C04444  3,7,4'-Tri-O-methylquercetin\n",
      "C04608  Apigenin 7-O-beta-D-glucoside\n",
      "C04858  Apigenin 7-O-[beta-D-apiosyl-(1->2)-beta-D-glucoside]\n",
      "C04900  Luteolin 7-O-[beta-D-glucuronosyl-(1->2)-beta-D-glucuronide]-4'-O-beta-D-glucuronide\n",
      "C05622  Malonylapiin\n",
      "C05623  Quercetin 3-O-glucoside\n",
      "C05625  Rutin\n",
      "C05903  Kaempferol\n",
      "C10098  Kaempferide\n",
      "C10107  Myricetin\n",
      "C10175  Quercetin 3-(2G-xylosylrutinoside)\n",
      "C11620  Syringetin\n",
      "C12249  Kaempferol 3-O-glucoside\n",
      "C12626  Kaempferol-3-O-galactoside\n",
      "C12627  Apigenin 7-O-neohesperidoside\n",
      "C12628  Vitexin 2''-O-beta-L-rhamnoside\n",
      "C12629  Isoswertisin 2''-rhamnoside\n",
      "C12630  Scolymoside\n",
      "C12632  Luteolin 7-O-[beta-D-glucuronosyl-(1->2)-beta-D-glucuronide]\n",
      "C12633  Laricitrin\n",
      "C12634  Kaempferol 3-O-beta-D-glucosyl-(1->2)-beta-D-glucoside\n",
      "C12635  Kaempferol 3-sophorotrioside\n",
      "C12636  Kaempferol 3-O-[6-(4-coumaroyl)-beta-D-glucosyl-(1->2)-beta-D-glucosyl-(1->2)-beta-D-glucoside]\n",
      "C12637  Quercetin 3-O-[beta-D-xylosyl-(1->2)-beta-D-glucoside]\n",
      "C12638  Quercetin 3-O-(6-O-malonyl-beta-D-glucoside)\n",
      "C12667  Quercetin 3-O-beta-D-glucosyl-(1->2)-beta-D-glucoside\n",
      "C12668  Quercetin 3-O-beta-D-glucosyl-(1->2)-beta-D-glucosyl-(1->2)-beta-D-glucoside\n",
      "C16490  Kaempferol 3-O-beta-D-glucosylgalactoside\n",
      "C16911  Afzelin\n",
      "C19796  Quercetin 3-O-rhamnoside 7-O-glucoside\n",
      "C21833  Kaempferol-3-O-rutinoside\n",
      "C21854  Kaempferol 3-O-rhamnoside-7-O-glucoside\n",
      "REFERENCE   PMID:3994393\n",
      "AUTHORS   De Luca V, Ibrahim RK\n",
      "TITLE     Enzymatic synthesis of polymethylated flavonols in Chrysosplenium americanum. I. Partial purification and some properties of S-adenosyl-L-methionine:flavonol 3-, 6-, 7-, and 4'-O-methyltransferases.\n",
      "JOURNAL   Arch Biochem Biophys 238:596-605 (1985)\n",
      "DOI:10.1016/0003-9861(85)90205-X\n",
      "REFERENCE   PMID:6462109\n",
      "AUTHORS   Jourdan PS, Mansell RL\n",
      "TITLE     Isolation and partial characterization of three glucosyl transferases involved in the biosynthesis of flavonol triglucosides in Pisum sativum L.\n",
      "JOURNAL   Arch Biochem Biophys 213:434-43 (1982)\n",
      "DOI:10.1016/0003-9861(82)90569-0\n",
      "REFERENCE   PMID:10567367\n",
      "AUTHORS   Miller KD, Guyon V, Evans JN, Shuttleworth WA, Taylor LP\n",
      "TITLE     Purification, cloning, and heterologous expression of a catalytically efficient flavonol 3-O-galactosyltransferase expressed in the male gametophyte of Petunia hybrida.\n",
      "JOURNAL   J Biol Chem 274:34011-9 (1999)\n",
      "DOI:10.1074/jbc.274.48.34011\n",
      "REFERENCE   PMID:12482447\n",
      "AUTHORS   Cacace S, Schroder G, Wehinger E, Strack D, Schmidt J, Schroder J\n",
      "TITLE     A flavonol O-methyltransferase from Catharanthus roseus performing two sequential methylations.\n",
      "JOURNAL   Phytochemistry 62:127-37 (2003)\n",
      "DOI:10.1016/S0031-9422(02)00483-1\n",
      "REFERENCE   PMID:15807784\n",
      "AUTHORS   Tohge T, Nishiyama Y, Hirai MY, Yano M, Nakajima J, Awazuhara M, Inoue E, Takahashi H, Goodenowe DB, Kitayama M, Noji M, Yamazaki M, Saito K\n",
      "TITLE     Functional genomics by integrated analysis of metabolome and transcriptome of Arabidopsis plants over-expressing an MYB transcription factor.\n",
      "JOURNAL   Plant J 42:218-35 (2005)\n",
      "DOI:10.1111/j.1365-313X.2005.02371.x\n",
      "REFERENCE   PMID:17277900\n",
      "AUTHORS   Kubo H, Nawa N, Lupsea SA\n",
      "TITLE     Anthocyaninless1 gene of Arabidopsis thaliana encodes a UDP-glucose:flavonoid-3-O-glucosyltransferase.\n",
      "JOURNAL   J Plant Res 120:445-9 (2007)\n",
      "DOI:10.1007/s10265-006-0067-7\n",
      "REFERENCE   PMID:24241851\n",
      "AUTHORS   Knogge W, Weissenbock G\n",
      "TITLE     Tissue-distribution of secondary phenolic biosynthesis in developing primary leaves of Avena sativa L.\n",
      "JOURNAL   Planta 167:196-205 (1986)\n",
      "DOI:10.1007/BF00391415\n",
      "REFERENCE   PMID:24177970\n",
      "AUTHORS   Anhalt S, Weissenbock G\n",
      "TITLE     Subcellular localization of luteolin glucuronides and related enzymes in rye mesophyll.\n",
      "JOURNAL   Planta 187:83-8 (1992)\n",
      "DOI:10.1007/BF00201627\n",
      "REFERENCE\n",
      "AUTHORS   Suzuki H, Nakayama T, Nagae S, Yamaguchi M, Iwashita T, Fukui Y, Nishino T.\n",
      "TITLE     cDNA cloning and functional characterization of flavonol 3-O-glucoside-6\"-O-malonyltransferases from flowers of Verbena hybrida and Lamium purpureum.\n",
      "JOURNAL   J Mol Catal B 28:87-93 (2004)\n",
      "DOI:10.1016/j.molcatb.2004.01.005\n",
      "REL_PATHWAY ath00941  Flavonoid biosynthesis\n",
      "KO_PATHWAY  ko00944\n",
      "///\n",
      "ENTRY       ath00945                    Pathway\n",
      "NAME        Stilbenoid, diarylheptanoid and gingerol biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Stilbenoids are a group of phenolic compounds, biosynthetically interrelated through their common origin from a C6-C2-C6 intermediate, such as resveratol found in grapes. Stilbenoids can also exist as glycosides (e.g., piceid). Combretastatins are potentially useful stilbenoid natural products with known antitumor activity. Diarylheptanoid is a compound group having phenyl rings at 1,7 positions of n-heptane (C6-C7-C6), such as curcumin found in the ginger family. [6]-Gingerol is a major active component of ginger and has diverse pharmacologic effects.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00945  Stilbenoid, diarylheptanoid and gingerol biosynthesis\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G30490  C4H; cinnamate-4-hydroxylase [KO:K00487] [EC:1.14.14.91]\n",
      "AT5G48930  HCT; hydroxycinnamoyl-CoA shikimate/quinate hydroxycinnamoyl transferase [KO:K13065] [EC:2.3.1.133]\n",
      "AT2G19070  SHT; spermidine hydroxycinnamoyl transferase [KO:K13065] [EC:2.3.1.133]\n",
      "AT5G57840  HXXXD-type acyl-transferase family protein [KO:K13065] [EC:2.3.1.133]\n",
      "AT2G40890  CYP98A3; cytochrome P450, family 98, subfamily A, polypeptide 3 [KO:K09754] [EC:1.14.14.96]\n",
      "AT4G34050  CCoAOMT1; S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "AT1G24735  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "AT1G67980  CCOAMT; caffeoyl-CoA 3-O-methyltransferase [KO:K00588] [EC:2.1.1.104]\n",
      "AT4G26220  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00588] [EC:2.1.1.104]\n",
      "COMPOUND    C00223  p-Coumaroyl-CoA\n",
      "C00323  Caffeoyl-CoA\n",
      "C00406  Feruloyl-CoA\n",
      "C00540  Cinnamoyl-CoA\n",
      "C00852  Chlorogenate\n",
      "C01745  Pinosylvin\n",
      "C02947  4-Coumaroylshikimate\n",
      "C03582  Resveratrol\n",
      "C05901  Piceatannol\n",
      "C10287  Pterostilbene\n",
      "C10434  5-O-Caffeoylshikimic acid\n",
      "C10443  Curcumin\n",
      "C10462  [6]-Gingerol\n",
      "C12208  p-Coumaroyl quinic acid\n",
      "C17740  p-Coumaroyl-diketide-CoA\n",
      "C17741  Feruloyl-diketide-CoA\n",
      "C17742  Demethoxycurcumin\n",
      "C17743  Bisdemethoxycurcumin\n",
      "C17744  1-(4-Hydroxyphenyl)-1-decene-3,5-dione\n",
      "C17745  1-(3,4-Dihydroxyphenyl)-1-decene-3,5-dione\n",
      "C17746  1-Dehydro-[6]-gingerdione\n",
      "C17747  5-Hydroxy-1-(4-hydroxyphenyl)-3-decanone\n",
      "C17748  1-(3,4-Dihydroxyphenyl)-5-hydroxy-3-decanone\n",
      "C17749  Curcumin monoglucoside\n",
      "C17750  Curcumin diglucoside\n",
      "REFERENCE   PMID:19258320\n",
      "AUTHORS   Katsuyama Y, Kita T, Funa N, Horinouchi S\n",
      "TITLE     Curcuminoid biosynthesis by two type III polyketide synthases in the herb Curcuma longa.\n",
      "JOURNAL   J Biol Chem 284:11160-70 (2009)\n",
      "DOI:10.1074/jbc.M900070200\n",
      "REFERENCE   PMID:16890967\n",
      "AUTHORS   Ramirez-Ahumada Mdel C, Timmermann BN, Gang DR\n",
      "TITLE     Biosynthesis of curcuminoids and gingerols in turmeric (Curcuma longa) and ginger (Zingiber officinale): identification of curcuminoid synthase and hydroxycinnamoyl-CoA thioesterases.\n",
      "JOURNAL   Phytochemistry 67:2017-29 (2006)\n",
      "DOI:10.1016/j.phytochem.2006.06.028\n",
      "REFERENCE   PMID:17509574\n",
      "AUTHORS   Masada S, Terasaka K, Mizukami H\n",
      "TITLE     A single amino acid in the PSPG-box plays an important role in the catalytic function of CaUGT2 (Curcumin glucosyltransferase), a Group D Family 1 glucosyltransferase from Catharanthus roseus.\n",
      "JOURNAL   FEBS Lett 581:2605-10 (2007)\n",
      "DOI:10.1016/j.febslet.2007.05.002\n",
      "REL_PATHWAY ath00940  Phenylpropanoid biosynthesis\n",
      "KO_PATHWAY  ko00945\n",
      "///\n",
      "ENTRY       ath00950                    Pathway\n",
      "NAME        Isoquinoline alkaloid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Isoquinoline alkaloids are tyrosine-derived plant alkaloids with an isoquinoline skeleton. Among them benzylisoquinoline alkaloids form an important group with potent pharmacological activity, including analgesic compounds of morphine and codeine, and anti-infective agents of berberine, palmatine, and magnoflorine. Biosynthesis of isoquinoline alkaloids proceeds via decarboxylation of tyrosine or DOPA to yield dopamine, which together with 4-hydroxyphenylacetaldehyde, an aldehyde derived from tyrosine, is converted to reticuline, an important precursor of various benzylisoquinoline alkaloids.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00950  Isoquinoline alkaloid biosynthesis\n",
      "DBLINKS     GO: 0033075\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "AT2G22250  AAT; aspartate aminotransferase [KO:K15849] [EC:2.6.1.1 2.6.1.78 2.6.1.79]\n",
      "AT5G53970  TAT7; Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT5G36160  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT2G24850  TAT3; tyrosine aminotransferase 3 [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28410  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28420  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G23590  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28680  TYRDC; L-tyrosine decarboxylase [KO:K01592] [EC:4.1.1.25]\n",
      "AT2G20340  AAS; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01593] [EC:4.1.1.28 4.1.1.105]\n",
      "AT2G42490  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31710  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31690  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G14940  AO1; amine oxidase 1 [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G62810  CuAO1; Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT3G43670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G12290  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "COMPOUND    C00082  L-Tyrosine\n",
      "C00355  3,4-Dihydroxy-L-phenylalanine\n",
      "C00483  Tyramine\n",
      "C00757  Berberine\n",
      "C00811  4-Coumarate\n",
      "C01179  3-(4-Hydroxyphenyl)pyruvate\n",
      "C01516  Morphine\n",
      "C01735  Morphinone\n",
      "C01795  Columbamine\n",
      "C01852  Secologanin\n",
      "C02105  (S)-Reticuline\n",
      "C02106  (S)-Scoulerine\n",
      "C02134  Allocryptopine\n",
      "C02890  Tetrahydropalmatine\n",
      "C02915  (S)-N-Methylcanadine\n",
      "C02916  (S)-Norlaudanosoline\n",
      "C03329  (S)-Canadine\n",
      "C03758  Dopamine\n",
      "C03765  4-Hydroxyphenylacetaldehyde\n",
      "C04043  3,4-Dihydroxyphenylacetaldehyde\n",
      "C04118  Isocorypalmine\n",
      "C05174  (S)-Cheilanthifoline\n",
      "C05175  Stylopine\n",
      "C05176  (S)-N-Methylcoclaurine\n",
      "C05177  Berbamunine\n",
      "C05178  (R)-Reticuline\n",
      "C05179  Salutaridine\n",
      "C05189  Protopine\n",
      "C05190  6-Hydroxyprotopine\n",
      "C05191  Dihydrosanguinarine\n",
      "C05193  12-Hydroxydihydrochelirubine\n",
      "C05194  Dihydrochelirubine\n",
      "C05202  3'-Hydroxy-N-methyl-(S)-coclaurine\n",
      "C05220  Salutaridinol\n",
      "C05243  (R)-N-Methylcoclaurine\n",
      "C05247  10-Hydroxydihydrosanguinarine\n",
      "C05315  Palmatine\n",
      "C05316  Dihydromacarpine\n",
      "C05322  7-O-Acetylsalutaridinol\n",
      "C06160  (S)-Norcoclaurine\n",
      "C06161  (S)-Coclaurine\n",
      "C06162  Sanguinarine\n",
      "C06163  (S)-cis-N-Methylstylopine\n",
      "C06165  Macarpine\n",
      "C06167  1,2-Dehydroreticuline\n",
      "C06171  Codeinone\n",
      "C06172  Neopinone\n",
      "C06173  Thebaine\n",
      "C06174  Codeine\n",
      "C06175  Oripavine\n",
      "C06327  Chelirubine\n",
      "C06511  Guattegaumerine\n",
      "C06512  2'-Norberbamunine\n",
      "C06516  (R)-Norreticuline\n",
      "C06517  (S)-6-O-Methylnorlaudanosoline\n",
      "C06518  (S)-Nororientaline\n",
      "C06520  (S)-Norreticuline\n",
      "C06533  Papaverine\n",
      "C07304  Deacetylisoipecoside\n",
      "C07307  Deacetylipecoside\n",
      "C07592  Colchicine\n",
      "C08521  Caranine\n",
      "C08526  Galanthamine\n",
      "C08527  Haemanthamine\n",
      "C08532  Lycorine\n",
      "C09248  Tubulosine\n",
      "C09328  Alangimarckine\n",
      "C09330  Alangiside\n",
      "C09337  Ankorine\n",
      "C09390  Cephaeline\n",
      "C09421  Emetine\n",
      "C09581  Magnoflorine\n",
      "C09592  alpha-Narcotine\n",
      "C09593  Narcotoline\n",
      "C11250  Demecolcine\n",
      "C11813  Demethylalangiside\n",
      "C11814  Demethylisoalangiside\n",
      "C11815  Isoalangiside\n",
      "C11816  Protoemetine\n",
      "C11817  Deoxytubulosine\n",
      "C11818  (R)-Canadine\n",
      "C12162  Crinine\n",
      "C12173  Norgalanthamine\n",
      "C12180  (-)-8-Demethylmaritidine\n",
      "C12193  Vittatine\n",
      "C15530  Corydaline\n",
      "C16700  3,4-Dihydroxybenzaldehyde\n",
      "C16701  Norbelladine\n",
      "C16702  4'-O-Methylnorbelladine\n",
      "C16703  N-Demethylnarwedine\n",
      "C16706  4-Hydroxydihydrocinnamaldehyde\n",
      "C16707  (S)-Autumnaline\n",
      "C16708  Isoandrocymbine\n",
      "C16709  O-Methylandrocymbine\n",
      "C16710  N-Formyldemecolcine\n",
      "C16712  Deacetylcolchicine\n",
      "C16713  Floramultine\n",
      "C16714  Kreysigine\n",
      "C17591  (S)-Corytuberine\n",
      "C17592  Laudanine\n",
      "C20297  Narcotine hemiacetal\n",
      "C20299  Papaveroxine\n",
      "C21405  (4aS,10bR)-Noroxomaritidine\n",
      "C21406  (4aR,10bS)-Noroxomaritidine\n",
      "C21566  6-O-Methyldeacetylisoipecoside\n",
      "C21583  6-O-Methyl-N-deacetylisoipecoside aglycon\n",
      "C21584  7'-O-Dmethylcephaeline\n",
      "C21586  (S)-1-Hydroxy-N-methylcanadine\n",
      "C21587  (13S,14R)-1,13-Dihydroxy-N-methylcanadine\n",
      "C21588  (13S,14R)-1-Hydroxy-13-O-acetyl-N-methylcanadine\n",
      "C21589  (13S,14R)-1,8-Dihydroxy-13-O-acetyl-N-methylcanadine\n",
      "C21590  4'-O-Desmethyl-3-O-acetylpapaveroxine\n",
      "C21591  3-O-Acetylpapaveroxine\n",
      "C21599  4'-O-Desmethylpapaveroxine\n",
      "C21600  Narcotoline hemiacetal\n",
      "C21630  (S)-Norlaudanine\n",
      "C21631  (S)-Tetrahydropapaverine\n",
      "C21633  Noroxopluviine\n",
      "C21634  Norcraugsodine\n",
      "C21635  (4aR,10bS)-Normaritidine\n",
      "C21637  Elwesine\n",
      "C21639  Maritinamine\n",
      "REFERENCE   PMID:10811648\n",
      "AUTHORS   Morishige T, Tsujita T, Yamada Y, Sato F\n",
      "TITLE     Molecular characterization of the S-adenosyl-L-methionine:3'-hydroxy-N-methylcoclaurine 4'-O-methyltransferase involved in isoquinoline alkaloid biosynthesis in Coptis japonica.\n",
      "JOURNAL   J Biol Chem 275:23398-405 (2000)\n",
      "DOI:10.1074/jbc.M002439200\n",
      "REFERENCE   PMID:12423366\n",
      "AUTHORS   Morishige T, Dubouzet E, Choi KB, Yazaki K, Sato F\n",
      "TITLE     Molecular cloning of columbamine O-methyltransferase from cultured Coptis japonica cells.\n",
      "JOURNAL   Eur J Biochem 269:5659-67 (2002)\n",
      "DOI:10.1046/j.1432-1033.2002.03275.x\n",
      "REFERENCE   PMID:14675446\n",
      "AUTHORS   Ounaroon A, Decker G, Schmidt J, Lottspeich F, Kutchan TM\n",
      "TITLE     (R,S)-Reticuline 7-O-methyltransferase and (R,S)-norcoclaurine 6-O-methyltransferase of Papaver somniferum - cDNA cloning and characterization of methyl transfer enzymes of alkaloid biosynthesis in opium poppy.\n",
      "JOURNAL   Plant J 36:808-19 (2003)\n",
      "DOI:10.1046/j.1365-313X.2003.01928.x\n",
      "REFERENCE   PMID:18230623\n",
      "AUTHORS   Ikezawa N, Iwasa K, Sato F\n",
      "TITLE     Molecular cloning and characterization of CYP80G2, a cytochrome P450 that catalyzes an intramolecular C-C phenol coupling of (S)-reticuline in magnoflorine biosynthesis, from cultured Coptis japonica cells.\n",
      "JOURNAL   J Biol Chem 283:8810-21 (2008)\n",
      "DOI:10.1074/jbc.M705082200\n",
      "REFERENCE   PMID:18476877\n",
      "AUTHORS   Facchini PJ, De Luca V\n",
      "TITLE     Opium poppy and Madagascar periwinkle: model non-model systems to investigate alkaloid biosynthesis in plants.\n",
      "JOURNAL   Plant J 54:763-84 (2008)\n",
      "DOI:10.1111/j.1365-313X.2008.03438.x\n",
      "REFERENCE   PMID:18492807\n",
      "AUTHORS   Minami H, Kim JS, Ikezawa N, Takemura T, Katayama T, Kumagai H, Sato F\n",
      "TITLE     Microbial production of plant benzylisoquinoline alkaloids.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 105:7393-8 (2008)\n",
      "DOI:10.1073/pnas.0802981105\n",
      "REFERENCE\n",
      "AUTHORS   Rueffer M, Bauer W, Zenk MH.\n",
      "TITLE     The formation of corydaline and related alkaloids in Corydalis cava in vivo and in vitro.\n",
      "JOURNAL   Can J Chem 72:170-175 (1994)\n",
      "DOI:10.1139/v94-026\n",
      "REFERENCE\n",
      "AUTHORS   Sekine Y, Crevelling C, Bell M, Brossi A.\n",
      "TITLE     Mammalian alkaloids: O-methylation of (+-)-norcoclaurine-1-carboxylic acid and related isoquinolines including (S)- and (R)-norcoclaurine with 14C-labeled S-adenosyl-L-methionine in presence of mammalian catechol O-methyltransferase.\n",
      "JOURNAL   Helv Chim Acta 73:426-432 (1990)\n",
      "DOI:10.1002/hlca.19900730222\n",
      "REFERENCE\n",
      "AUTHORS   De-Eknamkul W, Tanahashi T, Zenk MH.\n",
      "TITLE     Enzymic 10-hydroxylation and 10-O-methylation of dihydrosanguinarine in dihydrochelirubine formation by Eschscholtzia.\n",
      "JOURNAL   Phytochemistry 31:2713-2717 (1992)\n",
      "DOI:10.1016/0031-9422(92)83617-8\n",
      "REFERENCE\n",
      "AUTHORS   Kammerer L, De-Eknamkul W, Zenk MH.\n",
      "TITLE     Enzymic 12-hydroxylation and 12-O-methylation of dihydrochelirubine in dihydromacarpine formation by Thalictrum bulgaricum.\n",
      "JOURNAL   Phytochemistry 36:1409-1416 (1994)\n",
      "DOI:10.1016/S0031-9422(00)89732-0\n",
      "REFERENCE   PMID:24241413\n",
      "AUTHORS   Schumacher HM, Zenk MH\n",
      "TITLE     Partial purification and characterization of dihydrobenzophenanthridine oxidase from Eschscholtzia californica cell suspension cultures.\n",
      "JOURNAL   Plant Cell Rep 7:43-6 (1988)\n",
      "DOI:10.1007/BF00272975\n",
      "REFERENCE\n",
      "AUTHORS   Rueffer M, Zenk MH.\n",
      "TITLE     Enzymatic formation of protopines by a microsomal cytochrome P-450 system of Corydalis vaginans.\n",
      "JOURNAL   Tetrahedron Lett 28:5307-5310 (1987)\n",
      "DOI:10.1016/S0040-4039(00)96715-7\n",
      "REL_PATHWAY ath00350  Tyrosine metabolism\n",
      "ath00360  Phenylalanine metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00902  Monoterpenoid biosynthesis\n",
      "KO_PATHWAY  ko00950\n",
      "///\n",
      "ENTRY       ath00960                    Pathway\n",
      "NAME        Tropane, piperidine and pyridine alkaloid biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "DBLINKS     UMBBD: caf  caf2\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G42490  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31710  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31690  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G14940  AO1; amine oxidase 1 [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G62810  CuAO1; Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT3G43670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT4G12290  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT1G31670  Copper amine oxidase family protein [KO:K00276] [EC:1.4.3.21]\n",
      "AT5G06060  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29260  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G30670  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29340  NAD-dependent epimerase/dehydratase family protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29360  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29290  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT1G07450  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29330  TRI; tropinone reductase [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29370  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29320  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29300  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT2G29310  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT1G07440  NAD(P)-binding Rossmann-fold superfamily protein [KO:K08081] [EC:1.1.1.206]\n",
      "AT5G19550  ASP2; aspartate aminotransferase 2 [KO:K14454] [EC:2.6.1.1]\n",
      "AT5G11520  ASP3; aspartate aminotransferase 3 [KO:K14454] [EC:2.6.1.1]\n",
      "AT1G62800  ASP4; aspartate aminotransferase 4 [KO:K14454] [EC:2.6.1.1]\n",
      "AT2G30970  ASP1; aspartate aminotransferase 1 [KO:K14455] [EC:2.6.1.1]\n",
      "AT4G31990  ASP5; aspartate aminotransferase 5 [KO:K00811] [EC:2.6.1.1]\n",
      "AT2G22250  AAT; aspartate aminotransferase [KO:K15849] [EC:2.6.1.1 2.6.1.78 2.6.1.79]\n",
      "AT5G53970  TAT7; Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT5G36160  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT2G24850  TAT3; tyrosine aminotransferase 3 [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28410  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G28420  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT4G23590  Tyrosine transaminase family protein [KO:K00815] [EC:2.6.1.5]\n",
      "AT1G71920  HISN6B; HISTIDINE BIOSYNTHESIS 6B [KO:K00817] [EC:2.6.1.9]\n",
      "AT5G10330  HPA1; histidinol phosphate aminotransferase 1 [KO:K00817] [EC:2.6.1.9]\n",
      "COMPOUND    C00047  L-Lysine\n",
      "C00079  L-Phenylalanine\n",
      "C00134  Putrescine\n",
      "C00166  Phenylpyruvate\n",
      "C00253  Nicotinate\n",
      "C00407  L-Isoleucine\n",
      "C00408  L-Pipecolate\n",
      "C00450  (S)-2,3,4,5-Tetrahydropyridine-2-carboxylate\n",
      "C00729  Tropine\n",
      "C00745  Nicotine\n",
      "C00783  Tropinone\n",
      "C01416  Cocaine\n",
      "C01456  Tropate\n",
      "C01479  Atropine\n",
      "C01672  Cadaverine\n",
      "C01746  Piperidine\n",
      "C01851  Scopolamine\n",
      "C02046  L-Hyoscyamine\n",
      "C02066  Pseudotropine\n",
      "C02621  13-Hydroxylupanine\n",
      "C02723  N-Methylputrescine\n",
      "C03239  6-Amino-2-oxohexanoate\n",
      "C03325  (6S)-Hydroxyhyoscyamine\n",
      "C03882  Piperine\n",
      "C04076  L-2-Aminoadipate 6-semialdehyde\n",
      "C04092  Delta1-Piperideine-2-carboxylate\n",
      "C04170  13-(2-Methylcrotonoyl)oxylupanine\n",
      "C05607  Phenyllactate\n",
      "C06176  Senecionine\n",
      "C06177  Retronecine\n",
      "C06178  1-Methylpyrrolinium\n",
      "C06179  Hygrine\n",
      "C06180  Anabasine\n",
      "C06181  Piperideine\n",
      "C06182  Pelletierine\n",
      "C06183  L-(+)-Anaferine\n",
      "C06184  N-Methylpelletierine\n",
      "C06185  Slaframine\n",
      "C06366  sym-Homospermidine\n",
      "C06521  Cuscohygrine\n",
      "C06524  Nornicotine\n",
      "C07475  Lobeline\n",
      "C10126  Anatabine\n",
      "C10157  Lobelanine\n",
      "C10171  (-)-Sedamine\n",
      "C10173  Swainsonine\n",
      "C10763  Cytisine\n",
      "C10772  Lupanine\n",
      "C10773  Lupinine\n",
      "C10783  (-)-Sparteine\n",
      "C10850  Calystegin A3\n",
      "C10858  Ecgonine\n",
      "C10860  Littorine\n",
      "C10865  Pseudopelletierine\n",
      "C12440  Trachelanthamidine\n",
      "C12448  Ecgonine methyl ester\n",
      "C12449  Pseudoecgonine\n",
      "C12450  Pseudoecgonyl-CoA\n",
      "C12452  Acetyltropine\n",
      "C12453  Acetylpseudotropine\n",
      "C12454  Anatalline\n",
      "C12455  5-Aminopentanal\n",
      "C15612  Senecionine N-oxide\n",
      "C16257  (R)-Phenyllactyl-CoA\n",
      "C16671  3,6-Dihydronicotinic acid\n",
      "C16697  1-Indolizidinone\n",
      "C16933  3,6-Dihydropyridine\n",
      "C18158  17-Oxosparteine\n",
      "REL_PATHWAY ath00300  Lysine biosynthesis\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00360  Phenylalanine metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00760  Nicotinate and nicotinamide metabolism\n",
      "ath00780  Biotin metabolism\n",
      "KO_PATHWAY  ko00960\n",
      "///\n",
      "ENTRY       ath00965                    Pathway\n",
      "NAME        Betalain biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Betalains are water-soluble nitrogen-containing pigments that are present in plants belonging to the order Caryophyllales (such as cactus and amaranth families) and in higher fungi. They contain betalamic acid as the chromophore and are classified into two types: betacyanins and betaxanthins. Betacyanins contain a cyclo-DOPA residue and exhibit red/violet coloration, while betaxanthins contain different amino acids or amino side chains and exhibit a yellow/orange coloration. The condensation of betalamic acid with amino acids (including cyclo-DOPA or amines) in plants is a spontaneous reaction, not an enzyme-catalyzed reaction.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00965  Betalain biosynthesis\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G20340  AAS; Pyridoxal phosphate (PLP)-dependent transferases superfamily protein [KO:K01593] [EC:4.1.1.28 4.1.1.105]\n",
      "AT4G15093  LigB; catalytic LigB subunit of aromatic ring-opening dioxygenase family [KO:K15777] [EC:1.13.11.-]\n",
      "COMPOUND    C00082  L-Tyrosine\n",
      "C00355  3,4-Dihydroxy-L-phenylalanine\n",
      "C00822  Dopaquinone\n",
      "C03758  Dopamine\n",
      "C05587  3-Methoxytyramine\n",
      "C05604  2-Carboxy-2,3-dihydro-5,6-dihydroxyindole\n",
      "C08537  Amaranthin\n",
      "C08538  Betalamic acid\n",
      "C08539  Betanidin\n",
      "C08540  Betanin\n",
      "C08542  Celosianin II\n",
      "C08543  Dopaxanthin\n",
      "C08544  Gomphrenin-I\n",
      "C08552  Lampranthin II\n",
      "C08557  Miraxanthin-V\n",
      "C08565  Portulacaxanthin II\n",
      "C17751  cyclo-Dopa 5-O-glucoside\n",
      "C17752  cyclo-Dopa-glucuronylglucoside\n",
      "C17753  Dopaxanthin quinone\n",
      "C17754  3-Methoxytyramine-betaxanthin\n",
      "C17755  Dopamine quinone\n",
      "C17756  2-Descarboxy-cyclo-dopa\n",
      "C17757  2-Descarboxy-betanidin\n",
      "C17758  4,5-seco-Dopa\n",
      "REFERENCE   PMID:15805475\n",
      "AUTHORS   Gandia-Herrero F, Escribano J, Garcia-Carmona F\n",
      "TITLE     Betaxanthins as substrates for tyrosinase. An approach to the role of tyrosinase in the biosynthetic pathway of betalains.\n",
      "JOURNAL   Plant Physiol 138:421-32 (2005)\n",
      "DOI:10.1104/pp.104.057992\n",
      "REFERENCE   PMID:16247652\n",
      "AUTHORS   Sasaki N, Abe Y, Wada K, Koda T, Goda Y, Adachi T, Ozeki Y\n",
      "TITLE     Amaranthin in feather cockscombs is synthesized via glucuronylation at the cyclo-DOPA glucoside step in the betacyanin biosynthetic pathway.\n",
      "JOURNAL   J Plant Res 118:439-42 (2005)\n",
      "DOI:10.1007/s10265-005-0237-z\n",
      "REFERENCE   PMID:12620337\n",
      "AUTHORS   Strack D, Vogt T, Schliemann W\n",
      "TITLE     Recent advances in betalain research.\n",
      "JOURNAL   Phytochemistry 62:247-69 (2003)\n",
      "DOI:10.1016/S0031-9422(02)00564-2\n",
      "REFERENCE   PMID:11855654\n",
      "AUTHORS   Vogt T\n",
      "TITLE     Substrate specificity and sequence analysis define a polyphyletic origin of betanidin 5- and 6-O-glucosyltransferase from Dorotheanthus bellidiformis.\n",
      "JOURNAL   Planta 214:492-5 (2002)\n",
      "DOI:10.1007/s00425-001-0685-1\n",
      "REL_PATHWAY ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "KO_PATHWAY  ko00965\n",
      "///\n",
      "ENTRY       ath00966                    Pathway\n",
      "NAME        Glucosinolate biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Glucosinolates are biologically active secondary metabolites found in Brassicaceae (mustard family) and related families.These compounds are genetically variable within plant species and used as natural pesticides, such as against insect herbivores. All glucosinolates share a common structure consisting of a beta-thioglucose moiety, a sulfonated oxime moiety, and a variable aglycone side chain derived from an alpha-amino acid. Genes encoding glucosinolate biosynthetic enzymes have been identified in Arabidopsis thaliana by genetic polymorphisms and loss-of-function mutations. This map shows examples of side chain elongation in methionine-derived glucosinolates and the core pathway for biosynthesis of glucosinolates from amino acids.\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00966  Glucosinolate biosynthesis\n",
      "MODULE      ath_M00370  Glucosinolate biosynthesis, tryptophan => glucobrassicin [PATH:ath00966]\n",
      "DBLINKS     GO: 0019761\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G19710  BCAT4; branched-chain aminotransferase4 [KO:K21346] [EC:2.6.1.88]\n",
      "AT5G23010  MAM1; methylthioalkylmalate synthase 1 [KO:K15741] [EC:2.3.3.17]\n",
      "AT5G23020  IMS2; 2-isopropylmalate synthase 2 [KO:K15742] [EC:2.3.3.17]\n",
      "AT4G13430  IIL1; isopropyl malate isomerase large subunit 1 [KO:K01703] [EC:4.2.1.33 4.2.1.35]\n",
      "AT2G43100  IPMI2; isopropylmalate isomerase 2 [KO:K21359] [EC:4.2.1.33 4.2.1.170]\n",
      "AT3G58990  IPMI1; isopropylmalate isomerase 1 [KO:K21359] [EC:4.2.1.33 4.2.1.170]\n",
      "AT5G14200  IMD1; isopropylmalate dehydrogenase 1 [KO:K21360] [EC:1.1.1.85 1.1.1.-]\n",
      "AT1G50090  BCAT7; D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G50110  D-aminoacid aminotransferase-like PLP-dependent enzymes superfamily protein [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10070  BCAT-2; branched-chain amino acid transaminase 2 [KO:K00826] [EC:2.6.1.42]\n",
      "AT3G49680  BCAT3; branched-chain aminotransferase 3 [KO:K00826] [EC:2.6.1.42]\n",
      "AT5G65780  ATBCAT-5; branched-chain amino acid aminotransferase 5 / branched-chain amino acid transaminase 5 (BCAT5) [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G10060  BCAT-1; branched-chain amino acid transaminase 1 [KO:K00826] [EC:2.6.1.42]\n",
      "AT1G16410  CYP79F1; cytochrome p450 79f1 [KO:K12154] [EC:1.14.14.42]\n",
      "AT4G13770  CYP83A1; cytochrome P450, family 83, subfamily A, polypeptide 1 [KO:K12156] [EC:1.14.14.43]\n",
      "AT2G20610  SUR1; Tyrosine transaminase family protein [KO:K11819] [EC:4.4.1.-]\n",
      "AT1G24100  UGT74B1; UDP-glucosyl transferase 74B1 [KO:K11820] [EC:2.4.1.195]\n",
      "AT1G74100  SOT16; sulfotransferase 16 [KO:K11821] [EC:2.8.2.24]\n",
      "AT1G18590  SOT17; sulfotransferase 17 [KO:K22321] [EC:2.8.2.38]\n",
      "AT1G74090  SOT18; desulfo-glucosinolate sulfotransferase 18 [KO:K22321] [EC:2.8.2.38]\n",
      "AT1G16400  CYP79F2; cytochrome P450, family 79, subfamily F, polypeptide 2 [KO:K12155] [EC:1.14.14.42]\n",
      "AT5G05260  CYP79A2; phenylalanine N-monooxygenase-like protein [KO:K12153] [EC:1.14.14.40]\n",
      "AT5G35917  CYP79A3P; cytochrome P450, family 79, subfamily A, polypeptide 3 pseudogene [KO:K12153] [EC:1.14.14.40]\n",
      "AT4G31500  CYP83B1; cytochrome P450, family 83, subfamily B, polypeptide 1 [KO:K11818] [EC:1.14.14.45]\n",
      "AT4G39950  CYP79B2; cytochrome P450, family 79, subfamily B, polypeptide 2 [KO:K11812] [EC:1.14.14.156]\n",
      "AT2G22330  CYP79B3; cytochrome P450, family 79, subfamily B, polypeptide 3 [KO:K11813] [EC:1.14.14.156]\n",
      "COMPOUND    C00073  L-Methionine\n",
      "C00078  L-Tryptophan\n",
      "C00079  L-Phenylalanine\n",
      "C00082  L-Tyrosine\n",
      "C00123  L-Leucine\n",
      "C00141  3-Methyl-2-oxobutanoic acid\n",
      "C00183  L-Valine\n",
      "C00233  4-Methyl-2-oxopentanoate\n",
      "C00407  L-Isoleucine\n",
      "C00671  (S)-3-Methyl-2-oxopentanoic acid\n",
      "C01069  Desulfoglucotropeolin\n",
      "C01180  4-Methylthio-2-oxobutanoic acid\n",
      "C02153  Glucotropaeolin\n",
      "C02937  Indole-3-acetaldehyde oxime\n",
      "C03219  (E)-2-Methylpropanal oxime\n",
      "C03719  Phenylacetothiohydroximate\n",
      "C04350  (E)-4-Hydroxyphenylacetaldehyde oxime\n",
      "C05837  Glucobrassicin\n",
      "C08401  Glucoberteroin\n",
      "C08407  Glucocochlearin\n",
      "C08409  Glucoerucin\n",
      "C08412  Glucoiberverin\n",
      "C08417  Gluconasturtiin\n",
      "C08418  Glucoputranjivin\n",
      "C08426  Sinalbin\n",
      "C16516  Indolylmethylthiohydroximate\n",
      "C16517  Indolylmethyl-desulfoglucosinolate\n",
      "C16518  S-(Indolylmethylthiohydroximoyl)-L-cysteine\n",
      "C17210  2-(2'-Methylthio)ethylmalic acid\n",
      "C17211  2-Oxo-5-methylthiopentanoic acid\n",
      "C17212  3-(2'-Methylthio)ethylmalic acid\n",
      "C17213  Homomethionine\n",
      "C17214  2-(3'-Methylthio)propylmalic acid\n",
      "C17215  3-(3'-Methylthio)propylmalic acid\n",
      "C17216  2-Oxo-6-methylthiohexanoic acid\n",
      "C17217  Dihomomethionine\n",
      "C17218  2-(4'-Methylthio)butylmalic acid\n",
      "C17219  3-(4'-Methylthio)butylmalic acid\n",
      "C17220  2-Oxo-7-methylthioheptanoic acid\n",
      "C17221  Trihomomethionine\n",
      "C17222  2-(5'-Methylthio)pentylmalic acid\n",
      "C17223  3-(5'-Methylthio)pentylmalic acid\n",
      "C17224  2-Oxo-8-methylthiooctanoic acid\n",
      "C17225  Tetrahomomethionine\n",
      "C17226  2-(6'-Methylthio)hexylmalic acid\n",
      "C17227  3-(6'-Methylthio)hexylmalic acid\n",
      "C17228  2-Oxo-9-methylthiononanoic acid\n",
      "C17229  Pentahomomethionine\n",
      "C17230  2-(7'-Methylthio)heptylmalic acid\n",
      "C17231  3-(7'-Methylthio)heptylmalic acid\n",
      "C17232  2-Oxo-10-methylthiodecanoic acid\n",
      "C17233  Hexahomomethionine\n",
      "C17235  L-Homophenylalanine\n",
      "C17236  3-Phenylpropionaldoxim\n",
      "C17237  S-(Phenylacetothiohydroximoyl)-L-cysteine\n",
      "C17238  S-(Hydroxyphenylacetothiohydroximoyl)-L-cysteine\n",
      "C17239  p-Hydroxyphenylacetothiohydroximate\n",
      "C17240  p-Hydroxybenzyldesulphoglucosinolate\n",
      "C17241  4-Methylthiobutanaldoxime\n",
      "C17242  S-(4-Methylthiobutylthiohydroximoyl)-L-cysteine\n",
      "C17243  4-Methylthiobutylthiohydroximate\n",
      "C17244  3-Methylthiopropyl-desulfoglucosinolate\n",
      "C17245  5-Methylthiopentanaldoxime\n",
      "C17246  6-Methylthiohexanaldoxime\n",
      "C17248  4-Methylthiobutyl-desulfoglucosinolate\n",
      "C17249  7-Methylthioheptanaldoxime\n",
      "C17250  Glucolesquerellin\n",
      "C17251  8-Methylthiooctanaldoxime\n",
      "C17252  7-Methylthioheptyl glucosinolate\n",
      "C17253  9-Methylthiononanaldoxime\n",
      "C17254  8-Methylthiooctyl glucosinolate\n",
      "C17255  3-Methylbutyraldehyde oxime\n",
      "C17256  2-Methylpropyl glucosinolate\n",
      "C19491  (E)-2-Methylbutanal oxime\n",
      "C19714  (E)-Phenylacetaldoxime\n",
      "C21650  7-Methylthioheptyl-desulfoglucosinolate\n",
      "C21746  8-Methylthiooctyl-desulfoglucosinolate\n",
      "REFERENCE   PMID:12049923\n",
      "AUTHORS   Wittstock U, Halkier BA\n",
      "TITLE     Glucosinolate research in the Arabidopsis era.\n",
      "JOURNAL   Trends Plant Sci 7:263-70 (2002)\n",
      "DOI:10.1016/S1360-1385(02)02273-2\n",
      "REFERENCE   PMID:15913672\n",
      "AUTHORS   Windsor AJ, Reichelt M, Figuth A, Svatos A, Kroymann J, Kliebenstein DJ, Gershenzon J, Mitchell-Olds T\n",
      "TITLE     Geographic and evolutionary diversification of glucosinolates among near relatives of Arabidopsis thaliana (Brassicaceae).\n",
      "JOURNAL   Phytochemistry 66:1321-33 (2005)\n",
      "DOI:10.1016/j.phytochem.2005.04.016\n",
      "REFERENCE   PMID:14740211\n",
      "AUTHORS   Textor S, Bartram S, Kroymann J, Falk KL, Hick A, Pickett JA, Gershenzon J\n",
      "TITLE     Biosynthesis of methionine-derived glucosinolates in Arabidopsis thaliana: recombinant expression and characterization of methylthioalkylmalate synthase, the condensing enzyme of the chain-elongation cycle.\n",
      "JOURNAL   Planta 218:1026-35 (2004)\n",
      "DOI:10.1007/s00425-003-1184-3\n",
      "REFERENCE   PMID:15358770\n",
      "AUTHORS   Piotrowski M, Schemenewitz A, Lopukhina A, Muller A, Janowitz T, Weiler EW, Oecking C\n",
      "TITLE     Desulfoglucosinolate sulfotransferases from Arabidopsis thaliana catalyze the final step in the biosynthesis of the glucosinolate core structure.\n",
      "JOURNAL   J Biol Chem 279:50717-25 (2004)\n",
      "DOI:10.1074/jbc.M407681200\n",
      "REFERENCE   PMID:12586901\n",
      "AUTHORS   Mikkelsen MD, Halkier BA\n",
      "TITLE     Metabolic engineering of valine- and isoleucine-derived glucosinolates in Arabidopsis expressing CYP79D2 from Cassava.\n",
      "JOURNAL   Plant Physiol 131:773-9 (2003)\n",
      "DOI:10.1104/pp.013425\n",
      "REFERENCE\n",
      "AUTHORS   Chen S, Andreasson E.\n",
      "TITLE     Update on glucosinolate metabolism and transport.\n",
      "JOURNAL   Plant Physiol Biochem 39:743-758 (2001)\n",
      "DOI:10.1016/S0981-9428(01)01301-8\n",
      "REL_PATHWAY ath00270  Cysteine and methionine metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "KO_PATHWAY  ko00966\n",
      "///\n",
      "ENTRY       ath00970                    Pathway\n",
      "NAME        Aminoacyl-tRNA biosynthesis - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Genetic Information Processing; Translation\n",
      "PATHWAY_MAP ath00970  Aminoacyl-tRNA biosynthesis\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        ArthCt108  trnA; tRNA-Ala [KO:K14218]\n",
      "ArthCt114  trnA; tRNA-Ala [KO:K14218]\n",
      "ArthCt093  trnR; tRNA-Arg [KO:K14219]\n",
      "ArthCt115  trnR; tRNA-Arg [KO:K14219]\n",
      "ArthCt127  trnR; tRNA-Arg [KO:K14219]\n",
      "Arthct120  trnN; tRNA-Asn [KO:K14220]\n",
      "ArthCt110  trnN; tRNA-Asn [KO:K14220]\n",
      "ArthCt095  trnD; tRNA-Asp [KO:K14221]\n",
      "ArthCt094  trnC; tRNA-Cys [KO:K14222]\n",
      "ArthCt090  trnQ; tRNA-Gln [KO:K14223]\n",
      "ArthCt097  trnE; tRNA-Glu [KO:K14224]\n",
      "ArthCt092  trnG; tRNA-Gly [KO:K14225]\n",
      "ArthCt116  trnG; tRNA-Gly [KO:K14225]\n",
      "ArthCt088  trnH; tRNA-His [KO:K14226]\n",
      "ArthCt107  trnI; tRNA-Ile [KO:K14227]\n",
      "ArthCt124  trnI; tRNA-Ile [KO:K14227]\n",
      "Arthct122  trnL; tRNA-Leu [KO:K14228]\n",
      "ArthCt101  trnL; tRNA-Leu [KO:K14228]\n",
      "ArthCt126  trnL; tRNA-Leu [KO:K14228]\n",
      "ArthCt118  trnL; tRNA-Leu [KO:K14228]\n",
      "ArthCt089  trnK; tRNA-Lys [KO:K14229]\n",
      "ArthCt104  trnM; tRNA-Met [KO:K14230]\n",
      "ArthCt099  trnfM; tRNA-Met [KO:K14230]\n",
      "ArthCt111  trnI; tRNA-Ile [KO:K14230]\n",
      "ArthCt113  trnI; tRNA-Ile [KO:K14230]\n",
      "ArthCt102  trnF; tRNA-Phe [KO:K14231]\n",
      "ArthCt106  trnP; tRNA-Pro [KO:K14232]\n",
      "ArthCt091  trnS; tRNA-Ser [KO:K14233]\n",
      "ArthCt125  trnS; tRNA-Ser [KO:K14233]\n",
      "ArthCt117  trnS; tRNA-Ser [KO:K14233]\n",
      "ArthCt098  trnT; tRNA-Thr [KO:K14234]\n",
      "ArthCt100  tRNA-Thr [KO:K14234]\n",
      "ArthCt105  trnW; tRNA-Trp [KO:K14235]\n",
      "ArthCt096  trnY; tRNA-Tyr [KO:K14236]\n",
      "Arthct121  trnV; tRNA-Val [KO:K14237]\n",
      "ArthCt112  tRNA-Val [KO:K14237]\n",
      "ArthCt103  trnV; tRNA-Val [KO:K14237]\n",
      "AT5G64050  ERS; glutamate tRNA synthetase [KO:K01885] [EC:6.1.1.17]\n",
      "AT5G26710  Glutamyl/glutaminyl-tRNA synthetase, class Ic [KO:K01885] [EC:6.1.1.17]\n",
      "AT3G25660  Amidase family protein [KO:K02433] [EC:6.3.5.6 6.3.5.7]\n",
      "AT1G48520  GATB; GLU-ADT subunit B [KO:K02434] [EC:6.3.5.6 6.3.5.7]\n",
      "AT4G32915  glutamyl-tRNA(Gln) amidotransferase subunit C [KO:K02435] [EC:6.3.5.6 6.3.5.7]\n",
      "AT1G25350  OVA9; glutamine-tRNA ligase, putative / glutaminyl-tRNA synthetase, putative / GlnRS [KO:K01886] [EC:6.1.1.18]\n",
      "AT5G19720  Ribosomal protein L25/Gln-tRNA synthetase, anti-codon-binding domain-containing protein [KO:K01886] [EC:6.1.1.18]\n",
      "AT1G50200  ALATS; Alanyl-tRNA synthetase [KO:K01872] [EC:6.1.1.7]\n",
      "AT5G22800  EMB1030; Alanyl-tRNA synthetase, class IIc [KO:K01872] [EC:6.1.1.7]\n",
      "AT4G26870  Class II aminoacyl-tRNA and biotin synthetases superfamily protein [KO:K22503] [EC:6.1.1.12]\n",
      "AT4G31180  Class II aminoacyl-tRNA and biotin synthetases superfamily protein [KO:K22503] [EC:6.1.1.12]\n",
      "AT4G33760  tRNA synthetase class II (D, K and N) family protein [KO:K01876] [EC:6.1.1.12]\n",
      "AT4G17300  NS1; Class II aminoacyl-tRNA and biotin synthetases superfamily protein [KO:K01893] [EC:6.1.1.22]\n",
      "AT1G70980  SYNC3; Class II aminoacyl-tRNA and biotin synthetases superfamily protein [KO:K01893] [EC:6.1.1.22]\n",
      "AT3G07420  NS2; asparaginyl-tRNA synthetase 2 [KO:K01893] [EC:6.1.1.22]\n",
      "AT5G56680  SYNC1; Class II aminoacyl-tRNA and biotin synthetases superfamily protein [KO:K01893] [EC:6.1.1.22]\n",
      "AT1G29870  tRNA synthetase class II (G, H, P and S) family protein [KO:K01880] [EC:6.1.1.14]\n",
      "AT1G29880  glycyl-tRNA synthetase / glycine-tRNA ligase [KO:K01880] [EC:6.1.1.14]\n",
      "AT3G44740  Class II aaRS and biotin synthetases superfamily protein [KO:K01880] [EC:6.1.1.14]\n",
      "AT3G48110  EDD1; glycine-tRNA ligase [KO:K14164] [EC:6.1.1.14]\n",
      "AT5G26830  Threonyl-tRNA synthetase [KO:K01868] [EC:6.1.1.3]\n",
      "AT2G04842  EMB2761; threonyl-tRNA synthetase, putative / threonine-tRNA ligase [KO:K01868] [EC:6.1.1.3]\n",
      "AT1G17960  Threonyl-tRNA synthetase [KO:K01868] [EC:6.1.1.3]\n",
      "AT1G18130  Class II aaRS and biotin synthetases superfamily protein [KO:K01868] [EC:6.1.1.3]\n",
      "AT5G27470  seryl-tRNA synthetase / serine-tRNA ligase [KO:K01875] [EC:6.1.1.11]\n",
      "AT1G11870  SRS; Seryl-tRNA synthetase [KO:K01875] [EC:6.1.1.11]\n",
      "AT2G31170  SYCO_ARATH; Cysteinyl-tRNA synthetase, class Ia family protein [KO:K01883] [EC:6.1.1.16]\n",
      "AT5G38830  Cysteinyl-tRNA synthetase, class Ia family protein [KO:K01883] [EC:6.1.1.16]\n",
      "AT3G56300  Cysteinyl-tRNA synthetase, class Ia family protein [KO:K01883] [EC:6.1.1.16]\n",
      "AT4G13780  methionine-tRNA ligase, putative / methionyl-tRNA synthetase, putative / MetRS [KO:K01874] [EC:6.1.1.10]\n",
      "AT3G55400  OVA1; methionyl-tRNA synthetase / methionine-tRNA ligase / MetRS (cpMetRS) [KO:K01874] [EC:6.1.1.10]\n",
      "AT1G66520  pde194; formyltransferase [KO:K00604] [EC:2.1.2.9]\n",
      "AT1G14610  TWN2; valyl-tRNA synthetase / valine-tRNA ligase (VALRS) [KO:K01873] [EC:6.1.1.9]\n",
      "AT5G16715  EMB2247; protein EMBRYO DEFECTIVE 2247 [KO:K01873] [EC:6.1.1.9]\n",
      "AT1G27160  valyl-tRNA synthetase / valine-tRNA ligase-like protein [KO:K01873] [EC:6.1.1.9]\n",
      "AT4G04350  EMB2369; tRNA synthetase class I (I, L, M and V) family protein [KO:K01869] [EC:6.1.1.4]\n",
      "AT1G09620  ATP binding/leucine-tRNA ligases/aminoacyl-tRNA ligase [KO:K01869] [EC:6.1.1.4]\n",
      "AT4G10320  tRNA synthetase class I (I, L, M and V) family protein [KO:K01870] [EC:6.1.1.5]\n",
      "AT5G49030  OVA2; tRNA synthetase class I (I, L, M and V) family protein [KO:K01870] [EC:6.1.1.5]\n",
      "AT3G13490  OVA5; Lysyl-tRNA synthetase, class II [KO:K04567] [EC:6.1.1.6]\n",
      "AT3G11710  ATKRS-1; lysyl-tRNA synthetase 1 [KO:K04567] [EC:6.1.1.6]\n",
      "AT1G66530  Arginyl-tRNA synthetase, class Ic [KO:K01887] [EC:6.1.1.19]\n",
      "AT4G26300  emb1027; Arginyl-tRNA synthetase, class Ic [KO:K01887] [EC:6.1.1.19]\n",
      "AT5G52520  OVA6; Class II aaRS and biotin synthetases superfamily protein [KO:K01881] [EC:6.1.1.15]\n",
      "AT3G62120  Class II aaRS and biotin synthetases superfamily protein [KO:K01881] [EC:6.1.1.15]\n",
      "AT5G10880  tRNA synthetase-related / tRNA ligase-like protein [KO:K01881] [EC:6.1.1.15]\n",
      "AT3G46100  HRS1; Histidyl-tRNA synthetase 1 [KO:K01892] [EC:6.1.1.21]\n",
      "AT3G02760  Class II aaRS and biotin synthetases superfamily protein [KO:K01892] [EC:6.1.1.21]\n",
      "AT5G03406  Class II aaRS and biotin synthetases superfamily protein [KO:K01892] [EC:6.1.1.21]\n",
      "AT3G58140  phenylalanyl-tRNA synthetase class IIc family protein [KO:K01889] [EC:6.1.1.20]\n",
      "AT4G39280  phenylalanyl-tRNA synthetase, putative / phenylalanine-tRNA ligase [KO:K01889] [EC:6.1.1.20]\n",
      "AT1G72550  tRNA synthetase beta subunit family protein [KO:K01890] [EC:6.1.1.20]\n",
      "AT2G33840  Tyrosyl-tRNA synthetase, class Ib, bacterial/mitochondrial [KO:K01866] [EC:6.1.1.1]\n",
      "AT3G02660  EMB2768; Tyrosyl-tRNA synthetase, class Ib, bacterial/mitochondrial [KO:K01866] [EC:6.1.1.1]\n",
      "AT1G28350  Nucleotidylyl transferase superfamily protein [KO:K01866] [EC:6.1.1.1]\n",
      "AT3G04600  Nucleotidylyl transferase superfamily protein [KO:K01867] [EC:6.1.1.2]\n",
      "AT2G25840  OVA4; Nucleotidylyl transferase superfamily protein [KO:K01867] [EC:6.1.1.2]\n",
      "COMPOUND    C00025  L-Glutamate\n",
      "C00037  Glycine\n",
      "C00041  L-Alanine\n",
      "C00047  L-Lysine\n",
      "C00049  L-Aspartate\n",
      "C00062  L-Arginine\n",
      "C00064  L-Glutamine\n",
      "C00065  L-Serine\n",
      "C00073  L-Methionine\n",
      "C00078  L-Tryptophan\n",
      "C00079  L-Phenylalanine\n",
      "C00082  L-Tyrosine\n",
      "C00097  L-Cysteine\n",
      "C00101  Tetrahydrofolate\n",
      "C00123  L-Leucine\n",
      "C00135  L-Histidine\n",
      "C00148  L-Proline\n",
      "C00152  L-Asparagine\n",
      "C00183  L-Valine\n",
      "C00188  L-Threonine\n",
      "C00234  10-Formyltetrahydrofolate\n",
      "C00407  L-Isoleucine\n",
      "C00886  L-Alanyl-tRNA\n",
      "C01005  O-Phospho-L-serine\n",
      "C01931  L-Lysyl-tRNA\n",
      "C02047  L-Leucyl-tRNA\n",
      "C02163  L-Arginyl-tRNA(Arg)\n",
      "C02282  Glutaminyl-tRNA\n",
      "C02412  Glycyl-tRNA(Gly)\n",
      "C02430  L-Methionyl-tRNA\n",
      "C02553  L-Seryl-tRNA(Ser)\n",
      "C02554  L-Valyl-tRNA(Val)\n",
      "C02702  L-Prolyl-tRNA(Pro)\n",
      "C02839  L-Tyrosyl-tRNA(Tyr)\n",
      "C02984  L-Aspartyl-tRNA(Asp)\n",
      "C02987  L-Glutamyl-tRNA(Glu)\n",
      "C02988  L-Histidyl-tRNA(His)\n",
      "C02992  L-Threonyl-tRNA(Thr)\n",
      "C03125  L-Cysteinyl-tRNA(Cys)\n",
      "C03127  L-Isoleucyl-tRNA(Ile)\n",
      "C03294  N-Formylmethionyl-tRNA\n",
      "C03402  L-Asparaginyl-tRNA(Asn)\n",
      "C03511  L-Phenylalanyl-tRNA(Phe)\n",
      "C03512  L-Tryptophanyl-tRNA(Trp)\n",
      "C06112  L-Glutamyl-tRNA(Gln)\n",
      "C06113  L-Aspartyl-tRNA(Asn)\n",
      "C06481  L-Seryl-tRNA(Sec)\n",
      "C06482  L-Selenocysteinyl-tRNA(Sec)\n",
      "C16138  L-Pyrrolysine\n",
      "C16638  O-Phosphoseryl-tRNA(Sec)\n",
      "C17022  O-Phosphoseryl-tRNA(Cys)\n",
      "C17027  L-Pyrrolysyl-tRNA(Pyl)\n",
      "REFERENCE   PMID:18252769\n",
      "AUTHORS   Sheppard K, Yuan J, Hohn MJ, Jester B, Devine KM, Soll D\n",
      "TITLE     From one amino acid to another: tRNA-dependent amino acid biosynthesis.\n",
      "JOURNAL   Nucleic Acids Res 36:1813-25 (2008)\n",
      "DOI:10.1093/nar/gkn015\n",
      "REFERENCE   PMID:15314242\n",
      "AUTHORS   Polycarpo C, Ambrogelly A, Berube A, Winbush SM, McCloskey JA, Crain PF, Wood JL, Soll D\n",
      "TITLE     An aminoacyl-tRNA synthetase that specifically activates pyrrolysine.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 101:12450-4 (2004)\n",
      "DOI:10.1073/pnas.0405362101\n",
      "REL_PATHWAY ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00340  Histidine metabolism\n",
      "ath00380  Tryptophan metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "KO_PATHWAY  ko00970\n",
      "///\n",
      "ENTRY       ath00997                    Pathway\n",
      "NAME        Biosynthesis of various secondary metabolites - part 3 - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00997  Biosynthesis of various secondary metabolites - part 3\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G36880  MAT3; methionine adenosyltransferase 3 [KO:K00789] [EC:2.5.1.6]\n",
      "AT3G17390  MTO3; S-adenosylmethionine synthetase family protein [KO:K00789] [EC:2.5.1.6]\n",
      "AT4G01850  SAM-2; S-adenosylmethionine synthetase 2 [KO:K00789] [EC:2.5.1.6]\n",
      "AT1G02500  SAM1; S-adenosylmethionine synthetase 1 [KO:K00789] [EC:2.5.1.6]\n",
      "AT5G04950  NAS1; nicotianamine synthase 1 [KO:K05953] [EC:2.5.1.43]\n",
      "AT5G56080  NAS2; nicotianamine synthase 2 [KO:K05953] [EC:2.5.1.43]\n",
      "AT1G09240  NAS3; nicotianamine synthase 3 [KO:K05953] [EC:2.5.1.43]\n",
      "AT1G56430  NAS4; nicotianamine synthase 4 [KO:K05953] [EC:2.5.1.43]\n",
      "AT1G59960  NAD(P)-linked oxidoreductase superfamily protein [KO:K22374] [EC:1.1.1.285]\n",
      "AT1G59950  NAD(P)-linked oxidoreductase superfamily protein [KO:K22374] [EC:1.1.1.285]\n",
      "COMPOUND    C00019  S-Adenosyl-L-methionine\n",
      "C00022  Pyruvate\n",
      "C00025  L-Glutamate\n",
      "C00026  2-Oxoglutarate\n",
      "C00036  Oxaloacetate\n",
      "C00037  Glycine\n",
      "C00047  L-Lysine\n",
      "C00049  L-Aspartate\n",
      "C00061  FMN\n",
      "C00062  L-Arginine\n",
      "C00065  L-Serine\n",
      "C00073  L-Methionine\n",
      "C00077  L-Ornithine\n",
      "C00158  Citrate\n",
      "C00188  L-Threonine\n",
      "C00197  3-Phospho-D-glycerate\n",
      "C00327  L-Citrulline\n",
      "C00441  L-Aspartate 4-semialdehyde\n",
      "C00515  D-Ornithine\n",
      "C00979  O-Acetyl-L-serine\n",
      "C01005  O-Phospho-L-serine\n",
      "C01028  N6-Hydroxy-L-lysine\n",
      "C03401  L-2,3-Diaminopropanoate\n",
      "C03955  N6-Acetyl-N6-hydroxy-L-lysine\n",
      "C04075  2-Amino-4-chloro-4-pentenoic acid\n",
      "C05324  Nicotianamine\n",
      "C05554  Aerobactin\n",
      "C05933  N(omega)-Hydroxyarginine\n",
      "C07044  Hydroxyurea\n",
      "C08057  Cycloserine\n",
      "C12115  3-Amino-4-hydroxybenzoate\n",
      "C15485  2'-Deoxymugineic acid\n",
      "C15486  3''-Deamino-3''-oxonicotianamine\n",
      "C15500  Mugineic acid\n",
      "C15501  3-Epihydroxymugineic acid\n",
      "C15502  3-Epihydroxy-2'-deoxymugineic acid\n",
      "C20333  N2-Citryl-N6-acetyl-N6-hydroxy-L-lysine\n",
      "C20639  O-Ureido-L-serine\n",
      "C20644  2-Amino-4,5-dihydroxy-6-oxo-7-(phosphooxy)heptanoate\n",
      "C20651  O-Ureido-D-serine\n",
      "C20799  Grixazone B\n",
      "C20961  N3-Fumaroyl-L-2,3-diaminopropanoate\n",
      "C20962  Dapdiamide A\n",
      "C20963  Dapdiamide B\n",
      "C20964  Dapdiamide C\n",
      "C20966  3-{[(2E)-4-Amino-4-oxobut-2-enoyl]amino}-L-alanine\n",
      "C21559  N-[(2S)-2-Amino-2-carboxyethyl]-L-glutamate\n",
      "C21646  8-Amino-8-demethylriboflavin\n",
      "C21647  Roseoflavin\n",
      "C21648  8-Demethyl-8-(methylamino)riboflavin\n",
      "C21803  8-Demethyl-8-formylriboflavin 5'-phosphate\n",
      "C21804  8-Carboxy-8-demethylriboflavin 5'-phosphate\n",
      "C21805  8-Amino-8-demethylriboflavin 5'-phosphate\n",
      "C22072  2-[(L-Alanin-3-ylcarbamoyl)methyl]-2-hydroxybutanedioate\n",
      "C22073  2-[(2-Aminoethylcarbamoyl)methyl]-2-hydroxybutanedioate\n",
      "C22074  2-[(L-Alanin-3-ylcarbamoyl)methyl]-3-(2-aminoethylcarbamoyl)-2-hydroxypropanoate\n",
      "C22075  Staphyloferrin B\n",
      "C22101  N5-Citryl-D-ornithine\n",
      "C22102  Staphyloferrin A\n",
      "C22137  4-Chloro-L-lysine\n",
      "C22138  L-Propargylglycine\n",
      "C22139  gamma-L-Glutamyl-L-propargylglycine\n",
      "C22140  L-gamma-Glutamyl-(3R)-L-beta-ethynylserine\n",
      "C22141  L-beta-Ethynylserine\n",
      "REFERENCE   PMID:19775248\n",
      "AUTHORS   Cheung J, Beasley FC, Liu S, Lajoie GA, Heinrichs DE\n",
      "TITLE     Molecular characterization of staphyloferrin B biosynthesis in Staphylococcus aureus.\n",
      "JOURNAL   Mol Microbiol 74:594-608 (2009)\n",
      "DOI:10.1111/j.1365-2958.2009.06880.x\n",
      "REFERENCE   PMID:19807062\n",
      "AUTHORS   Hollenhorst MA, Clardy J, Walsh CT\n",
      "TITLE     The ATP-dependent amide ligases DdaG and DdaF assemble the fumaramoyl-dipeptide scaffold of the dapdiamide antibiotics.\n",
      "JOURNAL   Biochemistry 48:10467-72 (2009)\n",
      "DOI:10.1021/bi9013165\n",
      "REFERENCE   PMID:17003031\n",
      "AUTHORS   Suzuki H, Ohnishi Y, Furusho Y, Sakuda S, Horinouchi S\n",
      "TITLE     Novel benzene ring biosynthesis from C(3) and C(4) primary metabolites by two enzymes.\n",
      "JOURNAL   J Biol Chem 281:36944-51 (2006)\n",
      "DOI:10.1074/jbc.M608103200\n",
      "REFERENCE   PMID:9952442\n",
      "AUTHORS   Higuchi K, Suzuki K, Nakanishi H, Yamaguchi H, Nishizawa NK, Mori S\n",
      "TITLE     Cloning of nicotianamine synthase genes, novel genes involved in the biosynthesis of phytosiderophores.\n",
      "JOURNAL   Plant Physiol 119:471-80 (1999)\n",
      "DOI:10.1104/pp.119.2.471\n",
      "REFERENCE   PMID:7622460\n",
      "AUTHORS   Ma JF, Shinada T, Matsuda C, Nomoto K\n",
      "TITLE     Biosynthesis of phytosiderophores, mugineic acids, associated with methionine cycling.\n",
      "JOURNAL   J Biol Chem 270:16549-54 (1995)\n",
      "DOI:10.1074/jbc.270.28.16549\n",
      "REFERENCE   PMID:20086163\n",
      "AUTHORS   Kumagai T, Koyama Y, Oda K, Noda M, Matoba Y, Sugiyama M\n",
      "TITLE     Molecular cloning and heterologous expression of a biosynthetic gene cluster for the antitubercular agent D-cycloserine produced by Streptomyces lavendulae.\n",
      "JOURNAL   Antimicrob Agents Chemother 54:1132-9 (2010)\n",
      "DOI:10.1128/AAC.01226-09\n",
      "REFERENCE   PMID:19138128\n",
      "AUTHORS   Cotton JL, Tao J, Balibar CJ\n",
      "TITLE     Identification and characterization of the Staphylococcus aureus gene cluster coding for staphyloferrin A.\n",
      "JOURNAL   Biochemistry 48:1025-35 (2009)\n",
      "DOI:10.1021/bi801844c\n",
      "REFERENCE   PMID:27062037\n",
      "AUTHORS   Schwarz J, Konjik V, Jankowitsch F, Sandhoff R, Mack M\n",
      "TITLE     Identification of the Key Enzyme of Roseoflavin Biosynthesis.\n",
      "JOURNAL   Angew Chem Int Ed Engl 55:6103-6 (2016)\n",
      "DOI:10.1002/anie.201600581\n",
      "REFERENCE   PMID:30867596\n",
      "AUTHORS   Marchand JA, Neugebauer ME, Ing MC, Lin CI, Pelton JG, Chang MCY\n",
      "TITLE     Discovery of a pathway for terminal-alkyne amino acid biosynthesis.\n",
      "JOURNAL   Nature 567:420-424 (2019)\n",
      "DOI:10.1038/s41586-019-1020-y\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00740  Riboflavin metabolism\n",
      "KO_PATHWAY  ko00997\n",
      "///\n",
      "ENTRY       ath00998                    Pathway\n",
      "NAME        Biosynthesis of various secondary metabolites - part 2 - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Biosynthesis of other secondary metabolites\n",
      "PATHWAY_MAP ath00998  Biosynthesis of various secondary metabolites - part 2\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G32100  PRR1; pinoresinol reductase 1 [KO:K21568] [EC:1.23.1.1 1.23.1.2 1.23.1.3 1.23.1.4]\n",
      "AT4G13660  PRR2; pinoresinol reductase 2 [KO:K21568] [EC:1.23.1.1 1.23.1.2 1.23.1.3 1.23.1.4]\n",
      "COMPOUND    C00002  ATP\n",
      "C00041  L-Alanine\n",
      "C00074  Phosphoenolpyruvate\n",
      "C00078  L-Tryptophan\n",
      "C00079  L-Phenylalanine\n",
      "C00082  L-Tyrosine\n",
      "C00083  Malonyl-CoA\n",
      "C00092  D-Glucose 6-phosphate\n",
      "C00108  Anthranilate\n",
      "C00251  Chorismate\n",
      "C00254  Prephenate\n",
      "C00279  D-Erythrose 4-phosphate\n",
      "C00406  Feruloyl-CoA\n",
      "C00423  trans-Cinnamate\n",
      "C00590  Coniferyl alcohol\n",
      "C00630  2-Methylpropanoyl-CoA\n",
      "C00755  4-Hydroxy-3-methoxy-benzaldehyde\n",
      "C00811  4-Coumarate\n",
      "C01179  3-(4-Hydroxyphenyl)pyruvate\n",
      "C01610  Puromycin\n",
      "C02247  Anthraniloyl-CoA\n",
      "C02798  3-Phosphonopyruvate\n",
      "C03167  Phosphonoacetaldehyde\n",
      "C05345  beta-D-Fructose 6-phosphate\n",
      "C05366  (+)-Pinoresinol\n",
      "C06451  2-Hydroxyethylphosphonate\n",
      "C06452  (S)-2-Hydroxypropylphosphonate\n",
      "C06454  Fosfomycin\n",
      "C06866  Capsaicin\n",
      "C07024  3'-Keto-3'-deoxy-ATP\n",
      "C07025  3'-Keto-3'-deoxy-AMP\n",
      "C07026  3'-Amino-3'-deoxy-AMP\n",
      "C07027  N6,N6,O-Tridemethylpuromycin-5'-phosphate\n",
      "C07028  N-Acetyl-N6,N6,O-tridemethylpuromycin-5'-phosphate\n",
      "C07029  N-Acetyl-N6,O-didemethylpuromycin-5'-phosphate\n",
      "C07030  N-Acetyl-O-demethylpuromycin-5'-phosphate\n",
      "C07031  N-Acetyl-O-demethylpuromycin\n",
      "C07032  N-Acetylpuromycin\n",
      "C07303  4-Hydroxy-3-methoxyphenyl-beta-hydroxypropanoyl-CoA\n",
      "C10556  Deoxypodophyllotoxin\n",
      "C10557  Dihydroanhydropodorhizol\n",
      "C10641  Aurachin D\n",
      "C10646  Lariciresinol\n",
      "C10682  Matairesinol\n",
      "C10874  Podophyllotoxin\n",
      "C12212  Kanosamine\n",
      "C12213  Kanosamine 6-phosphate\n",
      "C16666  Vanillylamine\n",
      "C18167  Secoisolariciresinol\n",
      "C18202  8-Methyl-6-nonenoic acid\n",
      "C20454  (-)-Lariciresinol\n",
      "C20455  (-)-Pinoresinol\n",
      "C20456  (+)-Secoisolariciresinol\n",
      "C20668  3-Dehydro-D-glucose 6-phosphate\n",
      "C20940  L-Dihydroanticapsin\n",
      "C20941  L-Anticapsin\n",
      "C20942  Bacilysin\n",
      "C20953  3-[(4R)-4-Hydroxycyclohexa-1,5-dien-1-yl]-2-oxopropanoate\n",
      "C21085  3-[(1E,4R)-4-Hydroxycyclohex-2-en-1-ylidene]-2-oxopropanoate\n",
      "C21086  (3E)-3-[(1R,5R,6S)-5-Hydroxy-7-oxabicyclo[4.1.0]heptan-2-ylidene]-2-oxopropanoate\n",
      "C21087  3-[(1R,2S,5R,6S)-5-Hydroxy-7-oxabicyclo[4.1.0]heptan-2-yl]-2-oxopropanoate\n",
      "C21140  Aurachin B\n",
      "C21141  4-Hydroxy-2-methyl-3-oxo-4-[(2E,6E)-farnesyl]-3,4-dihydroquinoline 1-oxide\n",
      "C21183  Bursehernin\n",
      "C21184  (-)-5'-Demethylyatein\n",
      "C21191  Pluviatolide\n",
      "C21330  Aurachin C\n",
      "C21873  2-Methylquinolin-4-ol\n",
      "C21874  Aurachin B epoxide\n",
      "C21875  Aurachin A\n",
      "C22142  Cyclo(L-Trp-L-Phe)\n",
      "C22143  N-Methyl-cyclo(L-Trp-L-Phe)\n",
      "C22144  (-)-Ditryptophenaline\n",
      "C22145  Fumiquinazoline F\n",
      "C22146  Fumiquinazoline F-indoline-2',3'-diol\n",
      "C22147  Fumiquinazoline A\n",
      "C22148  Fumiquinazoline C\n",
      "C22149  Fumiquinazoline D\n",
      "C22216  Cytidine 5'-{[hydroxy(2-hydroxyethyl)phosphonoyl]phosphate}\n",
      "C22217  Cytidine 5'-{[hydroxy(2-hydroxypropyl)phosphonoyl]phosphate}\n",
      "C22289  (6E)-8-Methylnon-6-enoyl-CoA\n",
      "REFERENCE   PMID:23586652\n",
      "AUTHORS   Vetter ND, Langill DM, Anjum S, Boisvert-Martel J, Jagdhane RC, Omene E, Zheng H, van Straaten KE, Asiamah I, Krol ES, Sanders DA, Palmer DR\n",
      "TITLE     A previously unrecognized kanosamine biosynthesis pathway in Bacillus subtilis.\n",
      "JOURNAL   J Am Chem Soc 135:5970-3 (2013)\n",
      "DOI:10.1021/ja4010255\n",
      "REFERENCE   PMID:24677498\n",
      "AUTHORS   Saruwatari T, Yagishita F, Mino T, Noguchi H, Hotta K, Watanabe K\n",
      "TITLE     Cytochrome P450 as dimerization catalyst in diketopiperazine alkaloid biosynthesis.\n",
      "JOURNAL   Chembiochem 15:656-9 (2014)\n",
      "DOI:10.1002/cbic.201300751\n",
      "REFERENCE   PMID:24612080\n",
      "AUTHORS   Lim FY, Ames B, Walsh CT, Keller NP\n",
      "TITLE     Co-ordination between BrlA regulation and secretion of the oxidoreductase FmqD directs selective accumulation of fumiquinazoline C to conidial tissues in Aspergillus fumigatus.\n",
      "JOURNAL   Cell Microbiol 16:1267-83 (2014)\n",
      "DOI:10.1111/cmi.12284\n",
      "REFERENCE   PMID:17335090\n",
      "AUTHORS   Sandmann A, Dickschat J, Jenke-Kodama H, Kunze B, Dittmann E, Muller R\n",
      "TITLE     A Type II polyketide synthase from the gram-negative Bacterium Stigmatella aurantiaca is involved in Aurachin alkaloid biosynthesis.\n",
      "JOURNAL   Angew Chem Int Ed Engl 46:2712-6 (2007)\n",
      "DOI:10.1002/anie.200603513\n",
      "REFERENCE   PMID:25644640\n",
      "AUTHORS   Ozcengiz G, Ogulur I\n",
      "TITLE     Biochemistry, genetics and regulation of bacilysin biosynthesis and its significance more than an antibiotic.\n",
      "JOURNAL   N Biotechnol 32:612-9 (2015)\n",
      "DOI:10.1016/j.nbt.2015.01.006\n",
      "REFERENCE   PMID:8576156\n",
      "AUTHORS   Tercero JA, Espinosa JC, Lacalle RA, Jimenez A.\n",
      "TITLE     The biosynthetic pathway of the aminonucleoside antibiotic puromycin, as deduced from the molecular analysis of the pur cluster of Streptomyces alboniger.\n",
      "JOURNAL   J Biol Chem 271:1579-90 (1996)\n",
      "DOI:10.1074/jbc.271.3.1579\n",
      "REFERENCE   PMID:19553373\n",
      "AUTHORS   Mazourek M, Pujar A, Borovsky Y, Paran I, Mueller L, Jahn MM\n",
      "TITLE     A dynamic interface for capsaicinoid systems biology.\n",
      "JOURNAL   Plant Physiol 150:1806-21 (2009)\n",
      "DOI:10.1104/pp.109.136549\n",
      "REFERENCE   PMID:26359402\n",
      "AUTHORS   Lau W, Sattely ES\n",
      "TITLE     Six enzymes from mayapple that complete the biosynthetic pathway to the etoposide aglycone.\n",
      "JOURNAL   Science 349:1224-8 (2015)\n",
      "DOI:10.1126/science.aac7202\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00280  Valine, leucine and isoleucine degradation\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00440  Phosphonate and phosphinate metabolism\n",
      "ath00940  Phenylpropanoid biosynthesis\n",
      "KO_PATHWAY  ko00998\n",
      "///\n",
      "ENTRY       ath01040                    Pathway\n",
      "NAME        Biosynthesis of unsaturated fatty acids - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Metabolism; Lipid metabolism\n",
      "PATHWAY_MAP ath01040  Biosynthesis of unsaturated fatty acids\n",
      "MODULE      ath_M00415  Fatty acid elongation in endoplasmic reticulum [PATH:ath01040]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G67730  KCR1; beta-ketoacyl reductase 1 [KO:K10251] [EC:1.1.1.62 1.1.1.330]\n",
      "AT1G24470  KCR2; beta-ketoacyl reductase 2 [KO:K10251] [EC:1.1.1.62 1.1.1.330]\n",
      "AT5G10480  PAS2; Protein-tyrosine phosphatase-like, PTPLA [KO:K10703] [EC:4.2.1.134]\n",
      "AT5G59770  Protein-tyrosine phosphatase-like, PTPLA [KO:K10703] [EC:4.2.1.134]\n",
      "AT3G55360  CER10; 3-oxo-5-alpha-steroid 4-dehydrogenase family protein [KO:K10258] [EC:1.3.1.93]\n",
      "AT1G43800  FTM1; Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT3G02610  plant stearoyl-acyl-carrier desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT3G02620  Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT3G02630  Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT5G16230  Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT5G16240  Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT2G43710  SSI2; Plant stearoyl-acyl-carrier-protein desaturase family protein [KO:K03921] [EC:1.14.19.2 1.14.19.11 1.14.19.26]\n",
      "AT3G12120  FAD2; fatty acid desaturase 2 [KO:K10256] [EC:1.14.19.6 1.14.19.22]\n",
      "AT2G35690  ACX5; acyl-CoA oxidase 5 [KO:K00232] [EC:1.3.3.6]\n",
      "AT4G16760  ACX1; acyl-CoA oxidase 1 [KO:K00232] [EC:1.3.3.6]\n",
      "AT5G65110  ACX2; acyl-CoA oxidase 2 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G51840  ACX4; acyl-CoA oxidase 4 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06290  ACX3; acyl-CoA oxidase 3 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06310  ACX6; acyl-CoA oxidase 6 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G06690  putative acyl-coenzyme A oxidase [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G04710  PKT4; peroxisomal 3-ketoacyl-CoA thiolase 4 [KO:K07513] [EC:2.3.1.16]\n",
      "AT2G33150  PKT3; peroxisomal 3-ketoacyl-CoA thiolase 3 [KO:K07513] [EC:2.3.1.16]\n",
      "AT5G48880  KAT5; peroxisomal 3-keto-acyl-CoA thiolase 2 [KO:K07513] [EC:2.3.1.16]\n",
      "AT1G01710  acyl-CoA thioesterase II [KO:K01068] [EC:3.1.2.2]\n",
      "AT4G00520  Acyl-CoA thioesterase family protein [KO:K01068] [EC:3.1.2.2]\n",
      "COMPOUND    C00154  Palmitoyl-CoA\n",
      "C00219  Arachidonate\n",
      "C00249  Hexadecanoic acid\n",
      "C00412  Stearoyl-CoA\n",
      "C00510  Oleoyl-CoA\n",
      "C00712  (9Z)-Octadecenoic acid\n",
      "C01203  Oleoyl-[acyl-carrier protein]\n",
      "C01530  Octadecanoic acid\n",
      "C01595  Linoleate\n",
      "C02041  Eicosanoyl-CoA\n",
      "C02050  Linoleoyl-CoA\n",
      "C02249  Arachidonyl-CoA\n",
      "C03035  gamma-Linolenoyl-CoA\n",
      "C03242  Dihomo-gamma-linolenate\n",
      "C03595  8,11,14-Eicosatrienoyl-CoA\n",
      "C04088  Octadecanoyl-[acyl-carrier protein]\n",
      "C05764  Hexadecanoyl-[acp]\n",
      "C06425  Icosanoic acid\n",
      "C06426  (6Z,9Z,12Z)-Octadecatrienoic acid\n",
      "C06427  (9Z,12Z,15Z)-Octadecatrienoic acid\n",
      "C06428  (5Z,8Z,11Z,14Z,17Z)-Icosapentaenoic acid\n",
      "C06429  (4Z,7Z,10Z,13Z,16Z,19Z)-Docosahexaenoic acid\n",
      "C08281  Docosanoic acid\n",
      "C08316  Erucic acid\n",
      "C08320  Tetracosanoic acid\n",
      "C08323  (15Z)-Tetracosenoic acid\n",
      "C16160  Linoleoyl-[acyl-carrier protein]\n",
      "C16161  alpha-Linolenoyl-ACP\n",
      "C16162  (9Z,12Z,15Z)-Octadecatrienoyl-CoA\n",
      "C16163  (6Z,9Z,12Z,15Z)-Octadecatetraenoyl-CoA\n",
      "C16164  (8Z,11Z,14Z,17Z)-Icosatetraenoyl-CoA\n",
      "C16165  (5Z,8Z,11Z,14Z,17Z)-Icosapentaenoyl-CoA\n",
      "C16166  (7Z,10Z,13Z,16Z,19Z)-Docosapentaenoyl-CoA\n",
      "C16167  (9Z,12Z,15Z,18Z,21Z)-Tetracosapentaenoyl-CoA\n",
      "C16168  (6Z,9Z,12Z,15Z,18Z,21Z)-Tetracosahexaenoyl-CoA\n",
      "C16169  (4Z,7Z,10Z,13Z,16Z,19Z)-Docosahexaenoyl-CoA\n",
      "C16170  (7Z,10Z,13Z,16Z)-Docosatetraenoyl-CoA\n",
      "C16171  (9Z,12Z,15Z,18Z)-Tetracosatetraenoyl-CoA\n",
      "C16172  (6Z,9Z,12Z,15Z,18Z)-Tetracosapentaenoyl-CoA\n",
      "C16173  (4Z,7Z,10Z,13Z,16Z)-Docosapentaenoyl-CoA\n",
      "C16179  (11Z,14Z,17Z)-Icosatrienoyl-CoA\n",
      "C16180  (11Z,14Z)-Icosadienoyl-CoA\n",
      "C16513  (7Z,10Z,13Z,16Z,19Z)-Docosapentaenoic acid\n",
      "C16522  Icosatrienoic acid\n",
      "C16525  Icosadienoic acid\n",
      "C16526  Icosenoic acid\n",
      "C16527  Adrenic acid\n",
      "C16528  Docosanoyl-CoA\n",
      "C16529  Tetracosanoyl-CoA\n",
      "C16530  Icosenoyl-CoA\n",
      "C16531  Docosenoyl-CoA\n",
      "C16532  Tetracosenoyl-CoA\n",
      "C16533  (13Z,16Z)-Docosadienoic acid\n",
      "C16645  (13Z,16Z)-Docosadienoyl-CoA\n",
      "C17279  Ximenoyl-CoA\n",
      "C21072  Palmitoleoyl-CoA\n",
      "C21932  Cerotoyl-CoA\n",
      "C21934  Montanoyl-CoA\n",
      "C21935  (6Z,9Z)-Octadecadienoyl-CoA\n",
      "C21937  (8Z,11Z)-Icosadienoyl-CoA\n",
      "C21939  (5Z,8Z,11Z)-Icosatrienoyl-CoA\n",
      "C21941  (7Z,10Z,13Z)-Docosatrienoyl-CoA\n",
      "C21943  Sapienoyl-CoA\n",
      "C21945  cis-Vaccenoyl-CoA\n",
      "C21947  Paullinoyl-CoA\n",
      "C21949  (11Z,14Z,17Z,20Z,23Z)-Hexacosapentaenoyl-CoA\n",
      "C21951  (13Z,16Z,19Z,22Z,25Z)-Octacosapentaenoyl-CoA\n",
      "C21953  (8Z,11Z,14Z,17Z,20Z,23Z)-Hexacosahexaenoyl-CoA\n",
      "C21959  (10Z,13Z,16Z,19Z,22Z,25Z)-Octacosahexaenoyl-CoA\n",
      "C21961  (11Z,14Z,17Z,20Z)-Hexacosatetraenoyl-CoA\n",
      "C21963  (13Z,16Z,19Z,22Z)-Octacosatetraenoyl-CoA\n",
      "C21965  (8Z,11Z,14Z,17Z,20Z)-Hexacosapentaenoyl-CoA\n",
      "C21967  (10Z,13Z,16Z,19Z,22Z)-Octacosapentaenoyl-CoA\n",
      "C21981  (6Z,9Z)-Hexadecadienoyl-CoA\n",
      "REFERENCE   PMID:16564093\n",
      "AUTHORS   Jakobsson A, Westerberg R, Jacobsson A.\n",
      "TITLE     Fatty acid elongases in mammals: their regulation and roles in metabolism.\n",
      "JOURNAL   Prog Lipid Res 45:237-49 (2006)\n",
      "DOI:10.1016/j.plipres.2006.01.004\n",
      "REFERENCE   PMID:12482854\n",
      "AUTHORS   Moon YA, Horton JD.\n",
      "TITLE     Identification of two mammalian reductases involved in the two-carbon fatty acyl elongation cascade.\n",
      "JOURNAL   J Biol Chem 278:7335-43 (2003)\n",
      "DOI:10.1074/jbc.M211684200\n",
      "REFERENCE   PMID:12087109\n",
      "AUTHORS   Han G, Gable K, Kohlwein SD, Beaudoin F, Napier JA, Dunn TM.\n",
      "TITLE     The Saccharomyces cerevisiae YBR159w gene encodes the 3-ketoreductase of the microsomal fatty acid elongase.\n",
      "JOURNAL   J Biol Chem 277:35440-9 (2002)\n",
      "DOI:10.1074/jbc.M205620200\n",
      "REFERENCE   PMID:11500517\n",
      "AUTHORS   Su HM, Moser AB, Moser HW, Watkins PA.\n",
      "TITLE     Peroxisomal straight-chain Acyl-CoA oxidase and D-bifunctional protein are essential for the retroconversion step in docosahexaenoic acid synthesis.\n",
      "JOURNAL   J Biol Chem 276:38115-20 (2001)\n",
      "DOI:10.1074/jbc.M106326200\n",
      "REFERENCE   PMID:16766224\n",
      "AUTHORS   Huyghe S, Mannaerts GP, Baes M, Van Veldhoven PP.\n",
      "TITLE     Peroxisomal multifunctional protein-2: the enzyme, the patients and the knockout mouse model.\n",
      "JOURNAL   Biochim Biophys Acta 1761:973-94 (2006)\n",
      "DOI:10.1016/j.bbalip.2006.04.006\n",
      "REFERENCE   PMID:17719544\n",
      "AUTHORS   Denic V, Weissman JS\n",
      "TITLE     A molecular caliper mechanism for determining very long-chain fatty acid length.\n",
      "JOURNAL   Cell 130:663-77 (2007)\n",
      "DOI:10.1016/j.cell.2007.06.031\n",
      "REFERENCE   PMID:24184513\n",
      "AUTHORS   Ichi I, Kono N, Arita Y, Haga S, Arisawa K, Yamano M, Nagase M, Fujiwara Y, Arai H\n",
      "TITLE     Identification of genes and pathways involved in the synthesis of Mead acid (20:3n-9), an indicator of essential fatty acid deficiency.\n",
      "JOURNAL   Biochim Biophys Acta 1841:204-13 (2014)\n",
      "DOI:10.1016/j.bbalip.2013.10.013\n",
      "REFERENCE   PMID:20018209\n",
      "AUTHORS   Guillou H, Zadravec D, Martin PG, Jacobsson A\n",
      "TITLE     The key roles of elongases and desaturases in mammalian fatty acid metabolism: Insights from transgenic mice.\n",
      "JOURNAL   Prog Lipid Res 49:186-99 (2010)\n",
      "DOI:10.1016/j.plipres.2009.12.002\n",
      "REFERENCE   PMID:22158834\n",
      "AUTHORS   Yu M, Benham A, Logan S, Brush RS, Mandal MN, Anderson RE, Agbaga MP\n",
      "TITLE     ELOVL4 protein preferentially elongates 20:5n3 to very long chain PUFAs over 20:4n6 and 22:6n3.\n",
      "JOURNAL   J Lipid Res 53:494-504 (2012)\n",
      "DOI:10.1194/jlr.M021386\n",
      "REFERENCE   PMID:28784662\n",
      "AUTHORS   Sawai M, Uchida Y, Ohno Y, Miyamoto M, Nishioka C, Itohara S, Sassa T, Kihara A\n",
      "TITLE     The 3-hydroxyacyl-CoA dehydratases HACD1 and HACD2 exhibit functional redundancy and are active in a wide range of fatty acid elongation pathways.\n",
      "JOURNAL   J Biol Chem 292:15538-15551 (2017)\n",
      "DOI:10.1074/jbc.M117.803171\n",
      "REFERENCE   PMID:20937905\n",
      "AUTHORS   Ohno Y, Suto S, Yamanaka M, Mizutani Y, Mitsutake S, Igarashi Y, Sassa T, Kihara A\n",
      "TITLE     ELOVL1 production of C24 acyl-CoAs is linked to C24 sphingolipid synthesis.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 107:18439-44 (2010)\n",
      "DOI:10.1073/pnas.1005572107\n",
      "REFERENCE   PMID:21959040\n",
      "AUTHORS   Naganuma T, Sato Y, Sassa T, Ohno Y, Kihara A\n",
      "TITLE     Biochemical characterization of the very long-chain fatty acid elongase ELOVL7.\n",
      "JOURNAL   FEBS Lett 585:3337-41 (2011)\n",
      "DOI:10.1016/j.febslet.2011.09.024\n",
      "REFERENCE   PMID:19826053\n",
      "AUTHORS   Tamura K, Makino A, Hullin-Matsuda F, Kobayashi T, Furihata M, Chung S, Ashida S, Miki T, Fujioka T, Shuin T, Nakamura Y, Nakagawa H\n",
      "TITLE     Novel lipogenic enzyme ELOVL7 is involved in prostate cancer growth through saturated long-chain fatty acid metabolism.\n",
      "JOURNAL   Cancer Res 69:8133-40 (2009)\n",
      "DOI:10.1158/0008-5472.CAN-09-0775\n",
      "REFERENCE   PMID:22562166\n",
      "AUTHORS   Uemura H\n",
      "TITLE     Synthesis and production of unsaturated and polyunsaturated fatty acids in yeast: current state and perspectives.\n",
      "JOURNAL   Appl Microbiol Biotechnol 95:1-12 (2012)\n",
      "DOI:10.1007/s00253-012-4105-1\n",
      "REFERENCE   PMID:11734571\n",
      "AUTHORS   Ferdinandusse S, Denis S, Mooijer PA, Zhang Z, Reddy JK, Spector AA, Wanders RJ\n",
      "TITLE     Identification of the peroxisomal beta-oxidation enzymes involved in the biosynthesis of docosahexaenoic acid.\n",
      "JOURNAL   J Lipid Res 42:1987-95 (2001)\n",
      "KO_PATHWAY  ko01040\n",
      "///\n",
      "ENTRY       ath01100                    Pathway\n",
      "NAME        Metabolic pathways - Arabidopsis thaliana (thale cress)\n",
      "PATHWAY_MAP ath01100  Metabolic pathways\n",
      "MODULE      ath_M00001  Glycolysis (Embden-Meyerhof pathway), glucose => pyruvate [PATH:ath01100]\n",
      "ath_M00002  Glycolysis, core module involving three-carbon compounds [PATH:ath01100]\n",
      "ath_M00003  Gluconeogenesis, oxaloacetate => fructose-6P [PATH:ath01100]\n",
      "ath_M00004  Pentose phosphate pathway (Pentose phosphate cycle) [PATH:ath01100]\n",
      "ath_M00005  PRPP biosynthesis, ribose 5P => PRPP [PATH:ath01100]\n",
      "ath_M00006  Pentose phosphate pathway, oxidative phase, glucose 6P => ribulose 5P [PATH:ath01100]\n",
      "ath_M00007  Pentose phosphate pathway, non-oxidative phase, fructose 6P => ribose 5P [PATH:ath01100]\n",
      "ath_M00010  Citrate cycle, first carbon oxidation, oxaloacetate => 2-oxoglutarate [PATH:ath01100]\n",
      "ath_M00012  Glyoxylate cycle [PATH:ath01100]\n",
      "ath_M00015  Proline biosynthesis, glutamate => proline [PATH:ath01100]\n",
      "ath_M00018  Threonine biosynthesis, aspartate => homoserine => threonine [PATH:ath01100]\n",
      "ath_M00019  Valine/isoleucine biosynthesis, pyruvate => valine / 2-oxobutanoate => isoleucine [PATH:ath01100]\n",
      "ath_M00020  Serine biosynthesis, glycerate-3P => serine [PATH:ath01100]\n",
      "ath_M00021  Cysteine biosynthesis, serine => cysteine [PATH:ath01100]\n",
      "ath_M00022  Shikimate pathway, phosphoenolpyruvate + erythrose-4P => chorismate [PATH:ath01100]\n",
      "ath_M00023  Tryptophan biosynthesis, chorismate => tryptophan [PATH:ath01100]\n",
      "ath_M00026  Histidine biosynthesis, PRPP => histidine [PATH:ath01100]\n",
      "ath_M00027  GABA (gamma-Aminobutyrate) shunt [PATH:ath01100]\n",
      "ath_M00028  Ornithine biosynthesis, glutamate => ornithine [PATH:ath01100]\n",
      "ath_M00034  Methionine salvage pathway [PATH:ath01100]\n",
      "ath_M00036  Leucine degradation, leucine => acetoacetate + acetyl-CoA [PATH:ath01100]\n",
      "ath_M00039  Monolignol biosynthesis, phenylalanine/tyrosine => monolignol [PATH:ath01100]\n",
      "ath_M00040  Tyrosine biosynthesis, chorismate => arogenate => tyrosine [PATH:ath01100]\n",
      "ath_M00044  Tyrosine degradation, tyrosine => homogentisate [PATH:ath01100]\n",
      "ath_M00046  Pyrimidine degradation, uracil => beta-alanine, thymine => 3-aminoisobutanoate [PATH:ath01100]\n",
      "ath_M00048  Inosine monophosphate biosynthesis, PRPP + glutamine => IMP [PATH:ath01100]\n",
      "ath_M00049  Adenine ribonucleotide biosynthesis, IMP => ADP,ATP [PATH:ath01100]\n",
      "ath_M00050  Guanine ribonucleotide biosynthesis, IMP => GDP,GTP [PATH:ath01100]\n",
      "ath_M00052  Pyrimidine ribonucleotide biosynthesis, UMP => UDP/UTP,CDP/CTP [PATH:ath01100]\n",
      "ath_M00053  Pyrimidine deoxyribonuleotide biosynthesis, CDP => dCTP [PATH:ath01100]\n",
      "ath_M00055  N-glycan precursor biosynthesis [PATH:ath01100]\n",
      "ath_M00072  N-glycosylation by oligosaccharyltransferase [PATH:ath01100]\n",
      "ath_M00073  N-glycan precursor trimming [PATH:ath01100]\n",
      "ath_M00081  Pectin degradation [PATH:ath01100]\n",
      "ath_M00082  Fatty acid biosynthesis, initiation [PATH:ath01100]\n",
      "ath_M00083  Fatty acid biosynthesis, elongation [PATH:ath01100]\n",
      "ath_M00086  beta-Oxidation, acyl-CoA synthesis [PATH:ath01100]\n",
      "ath_M00087  beta-Oxidation [PATH:ath01100]\n",
      "ath_M00089  Triacylglycerol biosynthesis [PATH:ath01100]\n",
      "ath_M00092  Phosphatidylethanolamine (PE) biosynthesis, ethanolamine => PE [PATH:ath01100]\n",
      "ath_M00095  C5 isoprenoid biosynthesis, mevalonate pathway [PATH:ath01100]\n",
      "ath_M00096  C5 isoprenoid biosynthesis, non-mevalonate pathway [PATH:ath01100]\n",
      "ath_M00097  beta-Carotene biosynthesis, GGAP => beta-carotene [PATH:ath01100]\n",
      "ath_M00098  Acylglycerol degradation [PATH:ath01100]\n",
      "ath_M00100  Sphingosine degradation [PATH:ath01100]\n",
      "ath_M00112  Tocopherol/tocotorienol biosynthesis, homogentisate + phytyl/geranylgeranyl-PP => tocopherol/tocotorienol [PATH:ath01100]\n",
      "ath_M00113  Jasmonic acid biosynthesis [PATH:ath01100]\n",
      "ath_M00114  Ascorbate biosynthesis, plants, fructose-6P => ascorbate [PATH:ath01100]\n",
      "ath_M00115  NAD biosynthesis, aspartate => quinolinate => NAD [PATH:ath01100]\n",
      "ath_M00118  Glutathione biosynthesis, glutamate => glutathione [PATH:ath01100]\n",
      "ath_M00120  Coenzyme A biosynthesis, pantothenate => CoA [PATH:ath01100]\n",
      "ath_M00121  Heme biosynthesis, plants and bacteria, glutamate => heme [PATH:ath01100]\n",
      "ath_M00123  Biotin biosynthesis, pimeloyl-ACP/CoA => biotin [PATH:ath01100]\n",
      "ath_M00125  Riboflavin biosynthesis, plants and bacteria, GTP => riboflavin/FMN/FAD [PATH:ath01100]\n",
      "ath_M00126  Tetrahydrofolate biosynthesis, GTP => THF [PATH:ath01100]\n",
      "ath_M00132  Inositol phosphate metabolism, Ins(1,3,4)P3 => phytate [PATH:ath01100]\n",
      "ath_M00137  Flavanone biosynthesis, phenylalanine => naringenin [PATH:ath01100]\n",
      "ath_M00138  Flavonoid biosynthesis, naringenin => pelargonidin [PATH:ath01100]\n",
      "ath_M00165  Reductive pentose phosphate cycle (Calvin cycle) [PATH:ath01100]\n",
      "ath_M00166  Reductive pentose phosphate cycle, ribulose-5P => glyceraldehyde-3P [PATH:ath01100]\n",
      "ath_M00167  Reductive pentose phosphate cycle, glyceraldehyde-3P => ribulose-5P [PATH:ath01100]\n",
      "ath_M00168  CAM (Crassulacean acid metabolism), dark [PATH:ath01100]\n",
      "ath_M00169  CAM (Crassulacean acid metabolism), light [PATH:ath01100]\n",
      "ath_M00170  C4-dicarboxylic acid cycle, phosphoenolpyruvate carboxykinase type [PATH:ath01100]\n",
      "ath_M00171  C4-dicarboxylic acid cycle, NAD - malic enzyme type [PATH:ath01100]\n",
      "ath_M00172  C4-dicarboxylic acid cycle, NADP - malic enzyme type [PATH:ath01100]\n",
      "ath_M00307  Pyruvate oxidation, pyruvate => acetyl-CoA [PATH:ath01100]\n",
      "ath_M00366  C10-C20 isoprenoid biosynthesis, plants [PATH:ath01100]\n",
      "ath_M00368  Ethylene biosynthesis, methionine => ethylene [PATH:ath01100]\n",
      "ath_M00370  Glucosinolate biosynthesis, tryptophan => glucobrassicin [PATH:ath01100]\n",
      "ath_M00372  Abscisic acid biosynthesis, beta-carotene => abscisic acid [PATH:ath01100]\n",
      "ath_M00415  Fatty acid elongation in endoplasmic reticulum [PATH:ath01100]\n",
      "ath_M00432  Leucine biosynthesis, 2-oxoisovalerate => 2-oxoisocaproate [PATH:ath01100]\n",
      "ath_M00527  Lysine biosynthesis, DAP aminotransferase pathway, aspartate => lysine [PATH:ath01100]\n",
      "ath_M00531  Assimilatory nitrate reduction, nitrate => ammonia [PATH:ath01100]\n",
      "ath_M00532  Photorespiration [PATH:ath01100]\n",
      "ath_M00549  Nucleotide sugar biosynthesis, glucose => UDP-glucose [PATH:ath01100]\n",
      "ath_M00555  Betaine biosynthesis, choline => betaine [PATH:ath01100]\n",
      "ath_M00570  Isoleucine biosynthesis, threonine => 2-oxobutanoate => isoleucine [PATH:ath01100]\n",
      "ath_M00844  Arginine biosynthesis, ornithine => arginine [PATH:ath01100]\n",
      "ath_M00854  Glycogen biosynthesis, glucose-1P => glycogen/starch [PATH:ath01100]\n",
      "ath_M00880  Molybdenum cofactor biosynthesis, GTP => molybdenum cofactor [PATH:ath01100]\n",
      "ath_M00881  Lipoic acid biosynthesis, plants and bacteria, octanoyl-ACP => dihydrolipoyl-E2/H [PATH:ath01100]\n",
      "ath_M00882  Lipoic acid biosynthesis, eukaryotes, octanoyl-ACP => dihydrolipoyl-H [PATH:ath01100]\n",
      "ath_M00892  UDP-N-acetyl-D-glucosamine biosynthesis, eukaryotes, glucose => UDP-GlcNAc [PATH:ath01100]\n",
      "ath_M00897  Thiamine biosynthesis, plants, AIR (+ NAD+) => TMP/thiamine/TPP [PATH:ath01100]\n",
      "ath_M00899  Thiamine salvage pathway, HMP/HET => TMP [PATH:ath01100]\n",
      "ath_M00910  Phenylalanine biosynthesis, chorismate => arogenate => phenylalanine [PATH:ath01100]\n",
      "ath_M00916  Pyridoxal-P biosynthesis, R5P + glyceraldehyde-3P + glutamine => pyridoxal-P [PATH:ath01100]\n",
      "ath_M00917  Phytosterol biosynthesis, squalene 2,3-epoxide => campesterol/sitosterol [PATH:ath01100]\n",
      "ath_M00927  Gibberellin A12 biosynthesis, GAPP => GA12 [PATH:ath01100]\n",
      "ath_M00928  Gibberellin A4/A1 biosynthesis, GA12 => GA4 / GA53 => GA1 [PATH:ath01100]\n",
      "ath_M00932  Phylloquinone biosynthesis, chorismate (+ phytyl-PP) => phylloquinol [PATH:ath01100]\n",
      "ath_M00933  Plastoquinone biosynthesis, homogentisate + solanesyl-PP => plastoquinol [PATH:ath01100]\n",
      "ath_M00938  Pyrimidine deoxyribonuleotide biosynthesis, UDP => dTTP [PATH:ath01100]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00040  Pentose and glucuronate interconversions\n",
      "ath00051  Fructose and mannose metabolism\n",
      "ath00052  Galactose metabolism\n",
      "ath00053  Ascorbate and aldarate metabolism\n",
      "ath00061  Fatty acid biosynthesis\n",
      "ath00062  Fatty acid elongation\n",
      "ath00071  Fatty acid degradation\n",
      "ath00073  Cutin, suberine and wax biosynthesis\n",
      "ath00100  Steroid biosynthesis\n",
      "ath00130  Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "ath00190  Oxidative phosphorylation\n",
      "ath00195  Photosynthesis\n",
      "ath00196  Photosynthesis - antenna proteins\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00230  Purine metabolism\n",
      "ath00232  Caffeine metabolism\n",
      "ath00240  Pyrimidine metabolism\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00261  Monobactam biosynthesis\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00280  Valine, leucine and isoleucine degradation\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00310  Lysine degradation\n",
      "ath00330  Arginine and proline metabolism\n",
      "ath00340  Histidine metabolism\n",
      "ath00350  Tyrosine metabolism\n",
      "ath00360  Phenylalanine metabolism\n",
      "ath00380  Tryptophan metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00410  beta-Alanine metabolism\n",
      "ath00430  Taurine and hypotaurine metabolism\n",
      "ath00440  Phosphonate and phosphinate metabolism\n",
      "ath00450  Selenocompound metabolism\n",
      "ath00460  Cyanoamino acid metabolism\n",
      "ath00470  D-Amino acid metabolism\n",
      "ath00480  Glutathione metabolism\n",
      "ath00500  Starch and sucrose metabolism\n",
      "ath00510  N-Glycan biosynthesis\n",
      "ath00511  Other glycan degradation\n",
      "ath00513  Various types of N-glycan biosynthesis\n",
      "ath00514  Other types of O-glycan biosynthesis\n",
      "ath00520  Amino sugar and nucleotide sugar metabolism\n",
      "ath00531  Glycosaminoglycan degradation\n",
      "ath00561  Glycerolipid metabolism\n",
      "ath00562  Inositol phosphate metabolism\n",
      "ath00563  Glycosylphosphatidylinositol (GPI)-anchor biosynthesis\n",
      "ath00564  Glycerophospholipid metabolism\n",
      "ath00565  Ether lipid metabolism\n",
      "ath00590  Arachidonic acid metabolism\n",
      "ath00591  Linoleic acid metabolism\n",
      "ath00592  alpha-Linolenic acid metabolism\n",
      "ath00600  Sphingolipid metabolism\n",
      "ath00601  Glycosphingolipid biosynthesis - lacto and neolacto series\n",
      "ath00603  Glycosphingolipid biosynthesis - globo and isoglobo series\n",
      "ath00604  Glycosphingolipid biosynthesis - ganglio series\n",
      "ath00620  Pyruvate metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00640  Propanoate metabolism\n",
      "ath00650  Butanoate metabolism\n",
      "ath00660  C5-Branched dibasic acid metabolism\n",
      "ath00670  One carbon pool by folate\n",
      "ath00710  Carbon fixation in photosynthetic organisms\n",
      "ath00730  Thiamine metabolism\n",
      "ath00740  Riboflavin metabolism\n",
      "ath00750  Vitamin B6 metabolism\n",
      "ath00760  Nicotinate and nicotinamide metabolism\n",
      "ath00770  Pantothenate and CoA biosynthesis\n",
      "ath00780  Biotin metabolism\n",
      "ath00785  Lipoic acid metabolism\n",
      "ath00790  Folate biosynthesis\n",
      "ath00860  Porphyrin and chlorophyll metabolism\n",
      "ath00900  Terpenoid backbone biosynthesis\n",
      "ath00901  Indole alkaloid biosynthesis\n",
      "ath00902  Monoterpenoid biosynthesis\n",
      "ath00903  Limonene and pinene degradation\n",
      "ath00904  Diterpenoid biosynthesis\n",
      "ath00905  Brassinosteroid biosynthesis\n",
      "ath00906  Carotenoid biosynthesis\n",
      "ath00908  Zeatin biosynthesis\n",
      "ath00909  Sesquiterpenoid and triterpenoid biosynthesis\n",
      "ath00910  Nitrogen metabolism\n",
      "ath00920  Sulfur metabolism\n",
      "ath00940  Phenylpropanoid biosynthesis\n",
      "ath00941  Flavonoid biosynthesis\n",
      "ath00942  Anthocyanin biosynthesis\n",
      "ath00944  Flavone and flavonol biosynthesis\n",
      "ath00950  Isoquinoline alkaloid biosynthesis\n",
      "ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "ath00965  Betalain biosynthesis\n",
      "ath00966  Glucosinolate biosynthesis\n",
      "ath00998  Biosynthesis of various secondary metabolites - part 2\n",
      "ath01040  Biosynthesis of unsaturated fatty acids\n",
      "KO_PATHWAY  ko01100\n",
      "///\n",
      "ENTRY       ath01110                    Pathway\n",
      "NAME        Biosynthesis of secondary metabolites - Arabidopsis thaliana (thale cress)\n",
      "PATHWAY_MAP ath01110  Biosynthesis of secondary metabolites\n",
      "MODULE      ath_M00012  Glyoxylate cycle [PATH:ath01110]\n",
      "ath_M00018  Threonine biosynthesis, aspartate => homoserine => threonine [PATH:ath01110]\n",
      "ath_M00019  Valine/isoleucine biosynthesis, pyruvate => valine / 2-oxobutanoate => isoleucine [PATH:ath01110]\n",
      "ath_M00021  Cysteine biosynthesis, serine => cysteine [PATH:ath01110]\n",
      "ath_M00022  Shikimate pathway, phosphoenolpyruvate + erythrose-4P => chorismate [PATH:ath01110]\n",
      "ath_M00023  Tryptophan biosynthesis, chorismate => tryptophan [PATH:ath01110]\n",
      "ath_M00026  Histidine biosynthesis, PRPP => histidine [PATH:ath01110]\n",
      "ath_M00039  Monolignol biosynthesis, phenylalanine/tyrosine => monolignol [PATH:ath01110]\n",
      "ath_M00040  Tyrosine biosynthesis, chorismate => arogenate => tyrosine [PATH:ath01110]\n",
      "ath_M00095  C5 isoprenoid biosynthesis, mevalonate pathway [PATH:ath01110]\n",
      "ath_M00096  C5 isoprenoid biosynthesis, non-mevalonate pathway [PATH:ath01110]\n",
      "ath_M00097  beta-Carotene biosynthesis, GGAP => beta-carotene [PATH:ath01110]\n",
      "ath_M00112  Tocopherol/tocotorienol biosynthesis, homogentisate + phytyl/geranylgeranyl-PP => tocopherol/tocotorienol [PATH:ath01110]\n",
      "ath_M00113  Jasmonic acid biosynthesis [PATH:ath01110]\n",
      "ath_M00114  Ascorbate biosynthesis, plants, fructose-6P => ascorbate [PATH:ath01110]\n",
      "ath_M00121  Heme biosynthesis, plants and bacteria, glutamate => heme [PATH:ath01110]\n",
      "ath_M00125  Riboflavin biosynthesis, plants and bacteria, GTP => riboflavin/FMN/FAD [PATH:ath01110]\n",
      "ath_M00137  Flavanone biosynthesis, phenylalanine => naringenin [PATH:ath01110]\n",
      "ath_M00138  Flavonoid biosynthesis, naringenin => pelargonidin [PATH:ath01110]\n",
      "ath_M00366  C10-C20 isoprenoid biosynthesis, plants [PATH:ath01110]\n",
      "ath_M00368  Ethylene biosynthesis, methionine => ethylene [PATH:ath01110]\n",
      "ath_M00370  Glucosinolate biosynthesis, tryptophan => glucobrassicin [PATH:ath01110]\n",
      "ath_M00372  Abscisic acid biosynthesis, beta-carotene => abscisic acid [PATH:ath01110]\n",
      "ath_M00432  Leucine biosynthesis, 2-oxoisovalerate => 2-oxoisocaproate [PATH:ath01110]\n",
      "ath_M00527  Lysine biosynthesis, DAP aminotransferase pathway, aspartate => lysine [PATH:ath01110]\n",
      "ath_M00532  Photorespiration [PATH:ath01110]\n",
      "ath_M00910  Phenylalanine biosynthesis, chorismate => arogenate => phenylalanine [PATH:ath01110]\n",
      "ath_M00917  Phytosterol biosynthesis, squalene 2,3-epoxide => campesterol/sitosterol [PATH:ath01110]\n",
      "ath_M00927  Gibberellin A12 biosynthesis, GAPP => GA12 [PATH:ath01110]\n",
      "ath_M00928  Gibberellin A4/A1 biosynthesis, GA12 => GA4 / GA53 => GA1 [PATH:ath01110]\n",
      "ath_M00932  Phylloquinone biosynthesis, chorismate (+ phytyl-PP) => phylloquinol [PATH:ath01110]\n",
      "ath_M00933  Plastoquinone biosynthesis, homogentisate + solanesyl-PP => plastoquinol [PATH:ath01110]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "REL_PATHWAY ath00010  Glycolysis / Gluconeogenesis\n",
      "ath00020  Citrate cycle (TCA cycle)\n",
      "ath00030  Pentose phosphate pathway\n",
      "ath00053  Ascorbate and aldarate metabolism\n",
      "ath00073  Cutin, suberine and wax biosynthesis\n",
      "ath00100  Steroid biosynthesis\n",
      "ath00130  Ubiquinone and other terpenoid-quinone biosynthesis\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00230  Purine metabolism\n",
      "ath00232  Caffeine metabolism\n",
      "ath00250  Alanine, aspartate and glutamate metabolism\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00261  Monobactam biosynthesis\n",
      "ath00270  Cysteine and methionine metabolism\n",
      "ath00280  Valine, leucine and isoleucine degradation\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00340  Histidine metabolism\n",
      "ath00380  Tryptophan metabolism\n",
      "ath00400  Phenylalanine, tyrosine and tryptophan biosynthesis\n",
      "ath00440  Phosphonate and phosphinate metabolism\n",
      "ath00460  Cyanoamino acid metabolism\n",
      "ath00500  Starch and sucrose metabolism\n",
      "ath00564  Glycerophospholipid metabolism\n",
      "ath00592  alpha-Linolenic acid metabolism\n",
      "ath00630  Glyoxylate and dicarboxylate metabolism\n",
      "ath00650  Butanoate metabolism\n",
      "ath00740  Riboflavin metabolism\n",
      "ath00770  Pantothenate and CoA biosynthesis\n",
      "ath00860  Porphyrin and chlorophyll metabolism\n",
      "ath00900  Terpenoid backbone biosynthesis\n",
      "ath00901  Indole alkaloid biosynthesis\n",
      "ath00902  Monoterpenoid biosynthesis\n",
      "ath00904  Diterpenoid biosynthesis\n",
      "ath00905  Brassinosteroid biosynthesis\n",
      "ath00906  Carotenoid biosynthesis\n",
      "ath00908  Zeatin biosynthesis\n",
      "ath00909  Sesquiterpenoid and triterpenoid biosynthesis\n",
      "ath00940  Phenylpropanoid biosynthesis\n",
      "ath00941  Flavonoid biosynthesis\n",
      "ath00942  Anthocyanin biosynthesis\n",
      "ath00944  Flavone and flavonol biosynthesis\n",
      "ath00945  Stilbenoid, diarylheptanoid and gingerol biosynthesis\n",
      "ath00950  Isoquinoline alkaloid biosynthesis\n",
      "ath00960  Tropane, piperidine and pyridine alkaloid biosynthesis\n",
      "ath00965  Betalain biosynthesis\n",
      "ath00966  Glucosinolate biosynthesis\n",
      "ath00998  Biosynthesis of various secondary metabolites - part 2\n",
      "KO_PATHWAY  ko01110\n",
      "///\n",
      "ENTRY       ath01200                    Pathway\n",
      "NAME        Carbon metabolism - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Carbon metabolism is the most basic aspect of life. This map presents an overall view of central carbon metabolism, where the number of carbons is shown for each compound denoted by a circle, excluding a cofactor (CoA, CoM, THF, or THMPT) that is replaced by an asterisk. The map contains carbon utilization pathways of glycolysis (map00010), pentose phosphate pathway (map00030), and citrate cycle (map00020), and six known carbon fixation pathways (map00710 and map00720) as well as some pathways of methane metabolism (map00680). The six carbon fixation pathways are: (1) reductive pentose phosphate cycle (Calvin cycle) in plants and cyanobacteria that perform oxygenic photosynthesis, (2) reductive citrate cycle in photosynthetic green sulfur bacteria and some chemolithoautotrophs, (3) 3-hydroxypropionate bi-cycle in photosynthetic green nonsulfur bacteria, two variants of 4-hydroxybutyrate pathways in Crenarchaeota called (4) hydroxypropionate-hydroxybutyrate cycle and (5) dicarboxylate-hydroxybutyrate cycle, and (6) reductive acetyl-CoA pathway in methanogenic bacteria.\n",
      "PATHWAY_MAP ath01200  Carbon metabolism\n",
      "MODULE      ath_M00001  Glycolysis (Embden-Meyerhof pathway), glucose => pyruvate [PATH:ath01200]\n",
      "ath_M00002  Glycolysis, core module involving three-carbon compounds [PATH:ath01200]\n",
      "ath_M00004  Pentose phosphate pathway (Pentose phosphate cycle) [PATH:ath01200]\n",
      "ath_M00005  PRPP biosynthesis, ribose 5P => PRPP [PATH:ath01200]\n",
      "ath_M00006  Pentose phosphate pathway, oxidative phase, glucose 6P => ribulose 5P [PATH:ath01200]\n",
      "ath_M00007  Pentose phosphate pathway, non-oxidative phase, fructose 6P => ribose 5P [PATH:ath01200]\n",
      "ath_M00010  Citrate cycle, first carbon oxidation, oxaloacetate => 2-oxoglutarate [PATH:ath01200]\n",
      "ath_M00012  Glyoxylate cycle [PATH:ath01200]\n",
      "ath_M00020  Serine biosynthesis, glycerate-3P => serine [PATH:ath01200]\n",
      "ath_M00021  Cysteine biosynthesis, serine => cysteine [PATH:ath01200]\n",
      "ath_M00165  Reductive pentose phosphate cycle (Calvin cycle) [PATH:ath01200]\n",
      "ath_M00166  Reductive pentose phosphate cycle, ribulose-5P => glyceraldehyde-3P [PATH:ath01200]\n",
      "ath_M00167  Reductive pentose phosphate cycle, glyceraldehyde-3P => ribulose-5P [PATH:ath01200]\n",
      "ath_M00168  CAM (Crassulacean acid metabolism), dark [PATH:ath01200]\n",
      "ath_M00169  CAM (Crassulacean acid metabolism), light [PATH:ath01200]\n",
      "ath_M00170  C4-dicarboxylic acid cycle, phosphoenolpyruvate carboxykinase type [PATH:ath01200]\n",
      "ath_M00171  C4-dicarboxylic acid cycle, NAD - malic enzyme type [PATH:ath01200]\n",
      "ath_M00172  C4-dicarboxylic acid cycle, NADP - malic enzyme type [PATH:ath01200]\n",
      "ath_M00307  Pyruvate oxidation, pyruvate => acetyl-CoA [PATH:ath01200]\n",
      "ath_M00532  Photorespiration [PATH:ath01200]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "KO_PATHWAY  ko01200\n",
      "///\n",
      "ENTRY       ath01210                    Pathway\n",
      "NAME        2-Oxocarboxylic acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION 2-Oxocarboxylic acids, also called 2-oxo acids and alpha-keto acids, are the most elementary set of metabolites that includes pyruvate (2-oxopropanoate), 2-oxobutanoate, oxaloacetate (2-oxosuccinate) and 2-oxoglutarate. This diagram illustrates the architecture of chain extension and modification reaction modules for 2-oxocarboxylic acids. The chain extension module RM001 is a tricarboxylic pathway where acetyl-CoA derived carbon is used to extend the chain length by one. The chain modification modules RM002 (including RM032) and RM033, together with a reductive amination step (RC00006 or RC00036), generate basic and branched-chain amino acids, respectively. The modification module RM030 is used in the biosynthesis of glucosinolates, a class of plant secondary metabolites, for conversion to oxime followed by addition of thio-glucose moiety. Furthermore, the chain extension from 2-oxoadipate to 2-oxosuberate is followed by coenzyme B biosynthesis in methonogenic archaea.\n",
      "PATHWAY_MAP ath01210  2-Oxocarboxylic acid metabolism\n",
      "MODULE      ath_M00010  Citrate cycle, first carbon oxidation, oxaloacetate => 2-oxoglutarate [PATH:ath01210]\n",
      "ath_M00019  Valine/isoleucine biosynthesis, pyruvate => valine / 2-oxobutanoate => isoleucine [PATH:ath01210]\n",
      "ath_M00028  Ornithine biosynthesis, glutamate => ornithine [PATH:ath01210]\n",
      "ath_M00370  Glucosinolate biosynthesis, tryptophan => glucobrassicin [PATH:ath01210]\n",
      "ath_M00432  Leucine biosynthesis, 2-oxoisovalerate => 2-oxoisocaproate [PATH:ath01210]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "ath00220  Arginine biosynthesis\n",
      "ath00260  Glycine, serine and threonine metabolism\n",
      "ath00290  Valine, leucine and isoleucine biosynthesis\n",
      "ath00300  Lysine biosynthesis\n",
      "ath00966  Glucosinolate biosynthesis\n",
      "KO_PATHWAY  ko01210\n",
      "///\n",
      "ENTRY       ath01212                    Pathway\n",
      "NAME        Fatty acid metabolism - Arabidopsis thaliana (thale cress)\n",
      "PATHWAY_MAP ath01212  Fatty acid metabolism\n",
      "MODULE      ath_M00082  Fatty acid biosynthesis, initiation [PATH:ath01212]\n",
      "ath_M00083  Fatty acid biosynthesis, elongation [PATH:ath01212]\n",
      "ath_M00086  beta-Oxidation, acyl-CoA synthesis [PATH:ath01212]\n",
      "ath_M00087  beta-Oxidation [PATH:ath01212]\n",
      "ath_M00415  Fatty acid elongation in endoplasmic reticulum [PATH:ath01212]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "REL_PATHWAY ath00020  Citrate cycle (TCA cycle)\n",
      "KO_PATHWAY  ko01212\n",
      "///\n",
      "ENTRY       ath01230                    Pathway\n",
      "NAME        Biosynthesis of amino acids - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION This map presents a modular architecture of the biosynthesis pathways of twenty amino acids, which may be viewed as consisting of the core part and its extensions. The core part is the KEGG module for conversion of three-carbon compounds from glyceraldehyde-3P to pyruvate [MD:M00002], together with the pathways around serine and glycine. This KEGG module is the most conserved one in the KEGG MODULE database and is found in almost all the completely sequenced genomes. The extensions are the pathways containing the reaction modules RM001, RM033, RM032, and RM002 for biosynthesis of branched-chain amino acids (left) and basic amino acids (bottom), and the pathways for biosynthesis of histidine and aromatic amino acids (top right). It is interesting to note that the so-called essential amino acids that cannot be synthesized in human and other organisms generally appear in these extensions. Furthermore, the bottom extension of basic amino acids appears to be most divergent containing multiple pathways for lysine biosynthesis and multiple gene sets for arginine biosynthesis.\n",
      "PATHWAY_MAP ath01230  Biosynthesis of amino acids\n",
      "MODULE      ath_M00002  Glycolysis, core module involving three-carbon compounds [PATH:ath01230]\n",
      "ath_M00005  PRPP biosynthesis, ribose 5P => PRPP [PATH:ath01230]\n",
      "ath_M00007  Pentose phosphate pathway, non-oxidative phase, fructose 6P => ribose 5P [PATH:ath01230]\n",
      "ath_M00010  Citrate cycle, first carbon oxidation, oxaloacetate => 2-oxoglutarate [PATH:ath01230]\n",
      "ath_M00015  Proline biosynthesis, glutamate => proline [PATH:ath01230]\n",
      "ath_M00018  Threonine biosynthesis, aspartate => homoserine => threonine [PATH:ath01230]\n",
      "ath_M00019  Valine/isoleucine biosynthesis, pyruvate => valine / 2-oxobutanoate => isoleucine [PATH:ath01230]\n",
      "ath_M00020  Serine biosynthesis, glycerate-3P => serine [PATH:ath01230]\n",
      "ath_M00021  Cysteine biosynthesis, serine => cysteine [PATH:ath01230]\n",
      "ath_M00022  Shikimate pathway, phosphoenolpyruvate + erythrose-4P => chorismate [PATH:ath01230]\n",
      "ath_M00023  Tryptophan biosynthesis, chorismate => tryptophan [PATH:ath01230]\n",
      "ath_M00026  Histidine biosynthesis, PRPP => histidine [PATH:ath01230]\n",
      "ath_M00028  Ornithine biosynthesis, glutamate => ornithine [PATH:ath01230]\n",
      "ath_M00040  Tyrosine biosynthesis, chorismate => arogenate => tyrosine [PATH:ath01230]\n",
      "ath_M00432  Leucine biosynthesis, 2-oxoisovalerate => 2-oxoisocaproate [PATH:ath01230]\n",
      "ath_M00527  Lysine biosynthesis, DAP aminotransferase pathway, aspartate => lysine [PATH:ath01230]\n",
      "ath_M00570  Isoleucine biosynthesis, threonine => 2-oxobutanoate => isoleucine [PATH:ath01230]\n",
      "ath_M00844  Arginine biosynthesis, ornithine => arginine [PATH:ath01230]\n",
      "ath_M00910  Phenylalanine biosynthesis, chorismate => arogenate => phenylalanine [PATH:ath01230]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "KO_PATHWAY  ko01230\n",
      "///\n",
      "ENTRY       ath01240                    Pathway\n",
      "NAME        Biosynthesis of cofactors - Arabidopsis thaliana (thale cress)\n",
      "PATHWAY_MAP ath01240  Biosynthesis of cofactors\n",
      "MODULE      ath_M00049  Adenine ribonucleotide biosynthesis, IMP => ADP,ATP [PATH:ath01240]\n",
      "ath_M00052  Pyrimidine ribonucleotide biosynthesis, UMP => UDP/UTP,CDP/CTP [PATH:ath01240]\n",
      "ath_M00112  Tocopherol/tocotorienol biosynthesis, homogentisate + phytyl/geranylgeranyl-PP => tocopherol/tocotorienol [PATH:ath01240]\n",
      "ath_M00114  Ascorbate biosynthesis, plants, fructose-6P => ascorbate [PATH:ath01240]\n",
      "ath_M00115  NAD biosynthesis, aspartate => quinolinate => NAD [PATH:ath01240]\n",
      "ath_M00118  Glutathione biosynthesis, glutamate => glutathione [PATH:ath01240]\n",
      "ath_M00120  Coenzyme A biosynthesis, pantothenate => CoA [PATH:ath01240]\n",
      "ath_M00121  Heme biosynthesis, plants and bacteria, glutamate => heme [PATH:ath01240]\n",
      "ath_M00123  Biotin biosynthesis, pimeloyl-ACP/CoA => biotin [PATH:ath01240]\n",
      "ath_M00125  Riboflavin biosynthesis, plants and bacteria, GTP => riboflavin/FMN/FAD [PATH:ath01240]\n",
      "ath_M00126  Tetrahydrofolate biosynthesis, GTP => THF [PATH:ath01240]\n",
      "ath_M00880  Molybdenum cofactor biosynthesis, GTP => molybdenum cofactor [PATH:ath01240]\n",
      "ath_M00881  Lipoic acid biosynthesis, plants and bacteria, octanoyl-ACP => dihydrolipoyl-E2/H [PATH:ath01240]\n",
      "ath_M00882  Lipoic acid biosynthesis, eukaryotes, octanoyl-ACP => dihydrolipoyl-H [PATH:ath01240]\n",
      "ath_M00897  Thiamine biosynthesis, plants, AIR (+ NAD+) => TMP/thiamine/TPP [PATH:ath01240]\n",
      "ath_M00916  Pyridoxal-P biosynthesis, R5P + glyceraldehyde-3P + glutamine => pyridoxal-P [PATH:ath01240]\n",
      "ath_M00932  Phylloquinone biosynthesis, chorismate (+ phytyl-PP) => phylloquinol [PATH:ath01240]\n",
      "ath_M00933  Plastoquinone biosynthesis, homogentisate + solanesyl-PP => plastoquinol [PATH:ath01240]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "KO_PATHWAY  ko01240\n",
      "///\n",
      "ENTRY       ath01250                    Pathway\n",
      "NAME        Biosynthesis of nucleotide sugars - Arabidopsis thaliana (thale cress)\n",
      "PATHWAY_MAP ath01250  Biosynthesis of nucleotide sugars\n",
      "MODULE      ath_M00549  Nucleotide sugar biosynthesis, glucose => UDP-glucose [PATH:ath01250]\n",
      "ath_M00892  UDP-N-acetyl-D-glucosamine biosynthesis, eukaryotes, glucose => UDP-GlcNAc [PATH:ath01250]\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "KO_PATHWAY  ko01250\n",
      "///\n",
      "ENTRY       ath02010                    Pathway\n",
      "NAME        ABC transporters - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The ATP-binding cassette (ABC) transporters form one of the largest known protein families, and are widespread in bacteria, archaea, and eukaryotes. They couple ATP hydrolysis to active transport of a wide variety of substrates such as ions, sugars, lipids, sterols, peptides, proteins, and drugs. The structure of a prokaryotic ABC transporter usually consists of three components; typically two integral membrane proteins each having six transmembrane segments, two peripheral proteins that bind and hydrolyze ATP, and a periplasmic (or lipoprotein) substrate-binding protein. Many of the genes for the three components form operons as in fact observed in many bacterial and archaeal genomes. On the other hand, in a typical eukaryotic ABC transporter, the membrane spanning protein and the ATP-binding protein are fused, forming a multi-domain protein with the membrane-spanning domain (MSD) and the nucleotide-binding domain (NBD).\n",
      "CLASS       Environmental Information Processing; Membrane transport\n",
      "PATHWAY_MAP ath02010  ABC transporters\n",
      "DBLINKS     GO: 0043190 0043191 0043192\n",
      "TC: 3.A.1\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G41700  ABCA1; ATP-binding cassette A1 [KO:K05643]\n",
      "AT4G25450  ABCB28; non-intrinsic ABC protein 8 [KO:K05657]\n",
      "AT4G01820  ABCB3; P-glycoprotein 3 [KO:K05658] [EC:7.6.2.2]\n",
      "AT3G28380  ABCB17; P-glycoprotein 17 [KO:K05658] [EC:7.6.2.2]\n",
      "AT5G46540  ABCB7; P-glycoprotein 7 [KO:K05658] [EC:7.6.2.2]\n",
      "AT3G28860  ABCB19; ATP binding cassette subfamily B19 [KO:K05658] [EC:7.6.2.2]\n",
      "AT1G27940  ABCB13; P-glycoprotein 13 [KO:K05658] [EC:7.6.2.2]\n",
      "AT2G39480  ABCB6; P-glycoprotein 6 [KO:K05658] [EC:7.6.2.2]\n",
      "AT4G01830  ABCB5; P-glycoprotein 5 [KO:K05658] [EC:7.6.2.2]\n",
      "AT2G36910  ABCB1; ATP binding cassette subfamily B1 [KO:K05658] [EC:7.6.2.2]\n",
      "AT1G28010  ABCB14; P-glycoprotein 14 [KO:K05658] [EC:7.6.2.2]\n",
      "AT3G28360  ABCB16; P-glycoprotein 16 [KO:K05658] [EC:7.6.2.2]\n",
      "AT3G28345  ABCB15; ABC transporter family protein [KO:K05658] [EC:7.6.2.2]\n",
      "AT4G25960  ABCB2; P-glycoprotein 2 [KO:K05658] [EC:7.6.2.2]\n",
      "AT1G02530  ABCB12; P-glycoprotein 12 [KO:K05658] [EC:7.6.2.2]\n",
      "AT3G28390  ABCB18; P-glycoprotein 18 [KO:K05658] [EC:7.6.2.2]\n",
      "AT3G62150  ABCB21; P-glycoprotein 21 [KO:K05658] [EC:7.6.2.2]\n",
      "AT3G28415  ABCB22; ABC transporter family protein [KO:K05658] [EC:7.6.2.2]\n",
      "AT3G55320  ABCB20; P-glycoprotein 20 [KO:K05658] [EC:7.6.2.2]\n",
      "AT1G02520  ABCB11; P-glycoprotein 11 [KO:K05658] [EC:7.6.2.2]\n",
      "AT4G18050  ABCB9; P-glycoprotein 9 [KO:K05658] [EC:7.6.2.2]\n",
      "AT2G47000  ABCB4; ATP binding cassette subfamily B4 [KO:K05658] [EC:7.6.2.2]\n",
      "AT1G10680  PGP10; P-glycoprotein 10 [KO:K05658] [EC:7.6.2.2]\n",
      "AT5G58270  ABCB25; ABC transporter of the mitochondrion 3 [KO:K05662]\n",
      "AT4G28630  ABCB23; ABC transporter of the mitochondrion 1 [KO:K05662]\n",
      "AT4G28620  ABCB24; ABC transporter of the mitochondrion 2 [KO:K05662]\n",
      "AT2G07680  ABCC13; multidrug resistance-associated protein 11 [KO:K05674]\n",
      "AT1G31770  ABCG14; ATP-binding cassette 14 [KO:K05681]\n",
      "AT5G06530  ABCG22; ABC-2 type transporter family protein [KO:K05681]\n",
      "AT3G25620  ABCG21; ABC-2 type transporter family protein [KO:K05681]\n",
      "AT1G71960  ABCG25; ATP-binding casette family G25 [KO:K05681]\n",
      "AT4G27420  ABCG9; ABC-2 type transporter family protein [KO:K05681]\n",
      "COMPOUND    C00009  Orthophosphate\n",
      "C00025  L-Glutamate\n",
      "C00031  D-Glucose\n",
      "C00032  Heme\n",
      "C00034  Manganese\n",
      "C00037  Glycine\n",
      "C00038  Zinc cation\n",
      "C00041  L-Alanine\n",
      "C00047  L-Lysine\n",
      "C00049  L-Aspartate\n",
      "C00051  Glutathione\n",
      "C00059  Sulfate\n",
      "C00062  L-Arginine\n",
      "C00064  L-Glutamine\n",
      "C00065  L-Serine\n",
      "C00070  Copper\n",
      "C00077  L-Ornithine\n",
      "C00079  L-Phenylalanine\n",
      "C00086  Urea\n",
      "C00088  Nitrite\n",
      "C00089  Sucrose\n",
      "C00093  sn-Glycerol 3-phosphate\n",
      "C00095  D-Fructose\n",
      "C00098  Oligopeptide\n",
      "C00107  Dipeptide\n",
      "C00114  Choline\n",
      "C00116  Glycerol\n",
      "C00120  Biotin\n",
      "C00121  D-Ribose\n",
      "C00123  L-Leucine\n",
      "C00134  Putrescine\n",
      "C00135  L-Histidine\n",
      "C00137  myo-Inositol\n",
      "C00140  N-Acetyl-D-glucosamine\n",
      "C00148  L-Proline\n",
      "C00151  L-Amino acid\n",
      "C00159  D-Mannose\n",
      "C00175  Cobalt ion\n",
      "C00181  D-Xylose\n",
      "C00183  L-Valine\n",
      "C00185  Cellobiose\n",
      "C00188  L-Threonine\n",
      "C00208  Maltose\n",
      "C00212  Adenosine\n",
      "C00243  Lactose\n",
      "C00244  Nitrate\n",
      "C00245  Taurine\n",
      "C00255  Riboflavin\n",
      "C00259  L-Arabinose\n",
      "C00288  HCO3-\n",
      "C00291  Nickel\n",
      "C00294  Inosine\n",
      "C00299  Uridine\n",
      "C00315  Spermidine\n",
      "C00320  Thiosulfate\n",
      "C00330  Deoxyguanosine\n",
      "C00333  D-Galacturonate\n",
      "C00338  Lipopolysaccharide\n",
      "C00378  Thiamine\n",
      "C00379  Xylitol\n",
      "C00387  Guanosine\n",
      "C00392  Mannitol\n",
      "C00407  L-Isoleucine\n",
      "C00430  5-Aminolevulinate\n",
      "C00470  Pectate\n",
      "C00475  Cytidine\n",
      "C00487  Carnitine\n",
      "C00491  L-Cystine\n",
      "C00492  Raffinose\n",
      "C00503  Erythritol\n",
      "C00526  Deoxyuridine\n",
      "C00559  Deoxyadenosine\n",
      "C00719  Betaine\n",
      "C00794  D-Sorbitol\n",
      "C00855  D-Methionine\n",
      "C00865  Phospholipid\n",
      "C00881  Deoxycytidine\n",
      "C00919  Choline sulfate\n",
      "C00973  Cyclomaltodextrin\n",
      "C01083  alpha,alpha-Trehalose\n",
      "C01153  Orthophosphoric monoester\n",
      "C01157  Hydroxyproline\n",
      "C01177  Inositol 1-phosphate\n",
      "C01181  4-Trimethylammoniobutanoate\n",
      "C01279  4-Amino-5-hydroxymethyl-2-methylpyrimidine\n",
      "C01330  Sodium cation\n",
      "C01417  Cyanate\n",
      "C01487  D-Allose\n",
      "C01606  Phthalate\n",
      "C01630  Xylobiose\n",
      "C01667  Bacitracin\n",
      "C01674  Chitobiose\n",
      "C01682  Nopaline\n",
      "C01684  D-Rhamnose\n",
      "C01762  Xanthosine\n",
      "C01834  Lipoprotein\n",
      "C01835  Maltotriose\n",
      "C01935  Maltodextrin\n",
      "C01946  Oleandomycin\n",
      "C02160  Isomaltotriose\n",
      "C02273  Digalacturonate\n",
      "C03557  2-Aminoethylphosphonate\n",
      "C03611  Galactose oligosaccharide\n",
      "C03619  Methyl beta-D-galactoside\n",
      "C04114  (E)-4-(Trimethylammonio)but-2-enoate\n",
      "C04137  Octopine\n",
      "C05349  Ciprofloxacin\n",
      "C05402  Melibiose\n",
      "C05512  Deoxyinosine\n",
      "C05776  Vitamin B12\n",
      "C06227  Fe(III)hydroxamate\n",
      "C06229  Fe(III)dicitrate\n",
      "C06230  Fe-enterobactin\n",
      "C06232  Molybdate\n",
      "C06687  Norfloxacin\n",
      "C06705  Capsular polysaccharide\n",
      "C06706  Lipo-oligosaccharide\n",
      "C06707  Teichoic acid\n",
      "C06767  Hemine\n",
      "C07662  Sparfloxacin\n",
      "C07663  Moxifloxacin\n",
      "C11612  Polymyxin B\n",
      "C13768  Colistin sulfate\n",
      "C14818  Fe2+\n",
      "C14819  Fe3+\n",
      "C15521  Alkanesulfonate\n",
      "C15719  Coelichelin\n",
      "C16421  AI-2\n",
      "C16692  Mannopine\n",
      "C19609  Nickel(2+)\n",
      "C19872  N-Formyl-4-amino-5-aminomethyl-2-methylpyrimidine\n",
      "C20570  alpha-1,5-L-Arabinobiose\n",
      "C20571  alpha-1,5-L-Arabinotriose\n",
      "C20572  alpha-1,5-L-Arabinotetraose\n",
      "C20573  Aldotetraouronic acid\n",
      "C20679  Tungstate\n",
      "C21066  D-Galactofuranose\n",
      "C22040  S-Methyl-L-cysteine\n",
      "G00457  Maltotetraose\n",
      "REFERENCE   PMID:9799792\n",
      "AUTHORS   Tomii K, Kanehisa M.\n",
      "TITLE     A comparative analysis of ABC transporters in complete microbial genomes.\n",
      "JOURNAL   Genome Res 8:1048-59 (1998)\n",
      "DOI:10.1101/gr.8.10.1048\n",
      "REFERENCE   PMID:18424520\n",
      "AUTHORS   Sperandeo P, Lau FK, Carpentieri A, De Castro C, Molinaro A, Deho G, Silhavy TJ, Polissi A\n",
      "TITLE     Functional analysis of the protein machinery required for transport of lipopolysaccharide to the outer membrane of Escherichia coli.\n",
      "JOURNAL   J Bacteriol 190:4460-9 (2008)\n",
      "DOI:10.1128/JB.00270-08\n",
      "REFERENCE   PMID:18375759\n",
      "AUTHORS   Ruiz N, Gronenberg LS, Kahne D, Silhavy TJ\n",
      "TITLE     Identification of two inner-membrane proteins required for the transport of lipopolysaccharide to the outer membrane of Escherichia coli.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 105:5537-42 (2008)\n",
      "DOI:10.1073/pnas.0801196105\n",
      "REFERENCE   PMID:10557362\n",
      "AUTHORS   Omata T, Price GD, Badger MR, Okamura M, Gohta S, Ogawa T\n",
      "TITLE     Identification of an ATP-binding cassette transporter involved in bicarbonate uptake in the cyanobacterium Synechococcus sp. strain PCC 7942.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 96:13571-6 (1999)\n",
      "DOI:10.1073/pnas.96.23.13571\n",
      "REFERENCE   PMID:11731493\n",
      "AUTHORS   Quintero MJ, Montesinos ML, Herrero A, Flores E\n",
      "TITLE     Identification of genes encoding amino acid permeases by inactivation of selected ORFs from the Synechocystis genomic sequence.\n",
      "JOURNAL   Genome Res 11:2034-40 (2001)\n",
      "DOI:10.1101/gr.196301\n",
      "REFERENCE   PMID:16135226\n",
      "AUTHORS   Picossi S, Montesinos ML, Pernil R, Lichtle C, Herrero A, Flores E\n",
      "TITLE     ABC-type neutral amino acid permease N-I is required for optimal diazotrophic growth and is repressed in the heterocysts of Anabaena sp. strain PCC 7120.\n",
      "JOURNAL   Mol Microbiol 57:1582-92 (2005)\n",
      "DOI:10.1111/j.1365-2958.2005.04791.x\n",
      "REFERENCE   PMID:11929526\n",
      "AUTHORS   Valladares A, Montesinos ML, Herrero A, Flores E.\n",
      "TITLE     An ABC-type, high-affinity urea permease identified in cyanobacteria.\n",
      "JOURNAL   Mol Microbiol 43:703-15 (2002)\n",
      "DOI:10.1046/j.1365-2958.2002.02778.x\n",
      "REFERENCE   PMID:18326572\n",
      "AUTHORS   Richard-Fogal CL, Frawley ER, Kranz RG\n",
      "TITLE     Topology and function of CcmD in cytochrome c maturation.\n",
      "JOURNAL   J Bacteriol 190:3489-93 (2008)\n",
      "DOI:10.1128/JB.00146-08\n",
      "REFERENCE   PMID:16109926\n",
      "AUTHORS   Suzuki H, Koyanagi T, Izuka S, Onishi A, Kumagai H\n",
      "TITLE     The yliA, -B, -C, and -D genes of Escherichia coli K-12 encode a novel glutathione importer with an ATP-binding cassette.\n",
      "JOURNAL   J Bacteriol 187:5861-7 (2005)\n",
      "DOI:10.1128/JB.187.17.5861-5867.2005\n",
      "REFERENCE   PMID:17873039\n",
      "AUTHORS   Novikova M, Metlitskaya A, Datsenko K, Kazakov T, Kazakov A, Wanner B, Severinov K\n",
      "TITLE     The Escherichia coli Yej transporter is required for the uptake of translation inhibitor microcin C.\n",
      "JOURNAL   J Bacteriol 189:8361-5 (2007)\n",
      "DOI:10.1128/JB.01028-07\n",
      "REFERENCE   PMID:17214741\n",
      "AUTHORS   Tikhonova EB, Devroy VK, Lau SY, Zgurskaya HI\n",
      "TITLE     Reconstitution of the Escherichia coli macrolide transporter: the periplasmic membrane fusion protein MacA stimulates the ATPase activity of MacB.\n",
      "JOURNAL   Mol Microbiol 63:895-910 (2007)\n",
      "DOI:10.1111/j.1365-2958.2006.05549.x\n",
      "REFERENCE   PMID:18805970\n",
      "AUTHORS   Yamanaka H, Kobayashi H, Takahashi E, Okamoto K\n",
      "TITLE     MacAB is involved in the secretion of Escherichia coli heat-stable enterotoxin II.\n",
      "JOURNAL   J Bacteriol 190:7693-8 (2008)\n",
      "DOI:10.1128/JB.00853-08\n",
      "KO_PATHWAY  ko02010\n",
      "///\n",
      "ENTRY       ath03008                    Pathway\n",
      "NAME        Ribosome biogenesis in eukaryotes - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Ribosomes are the cellular factories responsible for making proteins. In eukaryotes, ribosome biogenesis involves the production and correct assembly of four rRNAs and about 80 ribosomal proteins. It requires hundreds of factors not present in the mature particle. In the absence of these proteins, ribosome biogenesis is stalled and cell growth is terminated even under optimal growth conditions. The primary pre-rRNA transcript is assembled into the 90S pre-ribosome, which contains both 40S and 60S assembly factors. Within this complex, the pre-rRNA is cleaved. pre-60S ribosomes are subjected to several sequential processing steps in the nucleoplasm involving numerous assembly intermediates before it is exported to the cytoplasm and matured into the 60S ribosomal subunit. The pre-40S ribosome is matured to the small ribosomal subunit in the cytoplasm by cleavage.\n",
      "CLASS       Genetic Information Processing; Translation\n",
      "PATHWAY_MAP ath03008  Ribosome biogenesis in eukaryotes\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G60250  CKB3; casein kinase II beta chain 3 [KO:K03115]\n",
      "AT4G17640  CKB2; casein kinase II beta chain 2 [KO:K03115]\n",
      "AT2G44680  CKB4; casein kinase II beta subunit 4 [KO:K03115]\n",
      "AT5G47080  CKB1; casein kinase II beta chain 1 [KO:K03115]\n",
      "AT2G23070  Protein kinase superfamily protein [KO:K03097] [EC:2.7.11.1]\n",
      "AT2G23080  CKA3; Protein kinase superfamily protein [KO:K03097] [EC:2.7.11.1]\n",
      "AT3G50000  CKA2; casein kinase II, alpha chain 2 [KO:K03097] [EC:2.7.11.1]\n",
      "AT5G67380  CKA1; casein kinase alpha 1 [KO:K03097] [EC:2.7.11.1]\n",
      "AT1G63810  nucleolar protein [KO:K14544]\n",
      "AT5G38720  ribosomal RNA-processing 7 protein [KO:K14545]\n",
      "AT1G15420  WD repeat protein [KO:K14546]\n",
      "AT5G11240  transducin family protein / WD-40 repeat family protein [KO:K14546]\n",
      "AT1G27470  transducin family protein / WD-40 repeat family protein [KO:K14548]\n",
      "AT4G07410  PCN; Transducin family protein / WD-40 repeat family protein [KO:K14548]\n",
      "AT2G47990  SWA1; transducin family protein / WD-40 repeat family protein [KO:K14549]\n",
      "AT3G06530  ARM repeat superfamily protein [KO:K14550]\n",
      "AT3G26480  Transducin family protein / WD-40 repeat family protein [KO:K14552]\n",
      "AT2G18900  Transducin/WD40 repeat-like superfamily protein [KO:K14552]\n",
      "AT5G14050  Transducin/WD40 repeat-like superfamily protein [KO:K14553]\n",
      "AT4G04940  transducin family protein / WD-40 repeat family protein [KO:K14554]\n",
      "AT5G16750  TOZ; Transducin family protein / WD-40 repeat family protein [KO:K14555]\n",
      "AT3G21540  transducin family protein / WD-40 repeat family protein [KO:K14556]\n",
      "AT4G28200  U3 small nucleolar RNA-associated-like protein [KO:K14557]\n",
      "AT1G15440  PWP2; periodic tryptophan protein 2 [KO:K14558]\n",
      "AT2G25420  transducin family protein / WD-40 repeat family protein [KO:K14558]\n",
      "AT5G66540  U3 small nucleolar ribonucleoprotein [KO:K14559]\n",
      "AT5G15750  Alpha-L RNA-binding motif/Ribosomal protein S4 family protein [KO:K14560]\n",
      "AT1G63780  IMP4; Ribosomal RNA processing Brix domain protein [KO:K14561]\n",
      "AT4G25630  FIB2; fibrillarin 2 [KO:K14563] [EC:2.1.1.-]\n",
      "AT5G52490  Fibrillarin family protein [KO:K14563] [EC:2.1.1.-]\n",
      "AT5G52470  FIB1; fibrillarin 1 [KO:K14563] [EC:2.1.1.-]\n",
      "AT1G56110  NOP56; NOP56-like pre RNA processing ribonucleoprotein [KO:K14564]\n",
      "AT3G12860  NOP56-like pre RNA processing ribonucleoprotein [KO:K14564]\n",
      "AT5G27120  NOP56-like pre RNA processing ribonucleoprotein [KO:K14565]\n",
      "AT3G05060  NOP56-like pre RNA processing ribonucleoprotein [KO:K14565]\n",
      "AT5G27140  NOP56-like pre RNA processing ribonucleoprotein [KO:K14565]\n",
      "AT4G22380  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K12845]\n",
      "AT4G12600  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K12845]\n",
      "AT5G20160  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K12845]\n",
      "AT3G57150  NAP57; homologue of NAP57 [KO:K11131] [EC:5.4.99.-]\n",
      "AT5G08180  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K11129]\n",
      "AT3G03920  H/ACA ribonucleoprotein complex, subunit Gar1/Naf1 protein [KO:K11128]\n",
      "AT5G18180  H/ACA ribonucleoprotein complex, subunit Gar1/Naf1 protein [KO:K11128]\n",
      "AT2G20490  NOP10; nucleolar RNA-binding Nop10p family protein [KO:K11130]\n",
      "AT2G46230  PIN domain-like family protein [KO:K14566]\n",
      "AT1G26530  PIN domain-like family protein [KO:K14566]\n",
      "AT4G02400  U3 ribonucleoprotein (Utp) family protein [KO:K14567]\n",
      "AT5G08600  U3 ribonucleoprotein (Utp) family protein [KO:K14567]\n",
      "AT3G57000  nucleolar essential protein-like protein [KO:K14568] [EC:2.1.1.260]\n",
      "AT1G06720  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K14569]\n",
      "AT3G57940  GNAT acetyltransferase (DUF699) [KO:K14521] [EC:2.3.1.-]\n",
      "AT1G10490  GNAT acetyltransferase (DUF699) [KO:K14521] [EC:2.3.1.-]\n",
      "AT5G22100  RNA cyclase family protein [KO:K11108]\n",
      "AT2G47300  POP1; ribonuclease Ps [KO:K01164] [EC:3.1.26.5]\n",
      "AT2G43190  POP4; ribonuclease P family protein [KO:K03538] [EC:3.1.26.5]\n",
      "AT1G04635  EMB1687; ribonuclease P family protein / Rpp14 family protein [KO:K03537] [EC:3.1.26.5]\n",
      "AT5G59980  GAF1; Polymerase/histidinol phosphatase-like protein [KO:K03539] [EC:3.1.26.5]\n",
      "AT5G25800  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K14570] [EC:3.1.-.-]\n",
      "AT5G67240  SDN3; small RNA degrading nuclease 3 [KO:K14570] [EC:3.1.-.-]\n",
      "AT3G50100  SDN1; small RNA degrading nuclease 1 [KO:K14570] [EC:3.1.-.-]\n",
      "AT3G50090  Exonuclease family protein [KO:K14570] [EC:3.1.-.-]\n",
      "AT5G05540  SDN2; small RNA degrading nuclease 2 [KO:K14570] [EC:3.1.-.-]\n",
      "AT2G26970  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K13288] [EC:3.1.-.-]\n",
      "AT5G42540  XRN2; exoribonuclease 2 [KO:K12619] [EC:3.1.13.-]\n",
      "AT1G75660  XRN3; 5'-3' exoribonuclease 3 [KO:K12619] [EC:3.1.13.-]\n",
      "AT1G50920  Nucleolar GTP-binding protein [KO:K06943]\n",
      "AT1G10300  Nucleolar GTP-binding protein [KO:K06943]\n",
      "AT1G80770  PDE318; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K06943]\n",
      "AT3G23860  GTP-binding protein-like protein [KO:K06943]\n",
      "AT1G52980  AtNug2; GTP-binding family protein [KO:K14537]\n",
      "AT3G07050  NSN1; GTP-binding family protein [KO:K14538]\n",
      "AT3G01610  CDC48C; cell division cycle 48C [KO:K14571]\n",
      "AT1G67120  midasin-like protein [KO:K14572]\n",
      "AT2G21440  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14573]\n",
      "AT5G41190  RNA-binding NOB1-like protein [KO:K11883]\n",
      "AT5G20010  RAN-1; RAS-related nuclear protein-1 [KO:K07936]\n",
      "AT5G20020  RAN2; RAS-related GTP-binding nuclear protein 2 [KO:K07936]\n",
      "AT5G55080  RAN4; ras-related nuclear protein 4 [KO:K07936]\n",
      "AT5G55190  RAN3; RAN GTPase 3 [KO:K07936]\n",
      "AT3G03110  XPO1B; exportin 1B [KO:K14290]\n",
      "AT5G17020  XPO1A; exportin 1A [KO:K14290]\n",
      "AT2G03820  NMD3; nonsense-mediated mRNA decay NMD3 family protein [KO:K07562]\n",
      "AT2G39820  eIF6B; Translation initiation factor IF6 [KO:K03264]\n",
      "AT3G55620  eIF6A; Translation initiation factor IF6 [KO:K03264]\n",
      "AT1G43860  sequence-specific DNA binding transcription factor [KO:K14574]\n",
      "AT3G22980  Ribosomal protein S5/Elongation factor G/III/V family protein [KO:K14536] [EC:3.6.5.-]\n",
      "AT2G27200  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K14539] [EC:3.6.1.-]\n",
      "AT1G08410  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K14539] [EC:3.6.1.-]\n",
      "AT5G60340  AAK6; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K18532] [EC:2.7.4.3]\n",
      "AT2G24990  Serine/threonine-protein kinase Rio1 [KO:K07178] [EC:2.7.11.1]\n",
      "AT5G37350  Serine/threonine-protein kinase Rio1 [KO:K07178] [EC:2.7.11.1]\n",
      "AT3G51270  protein serine/threonine kinase [KO:K07179] [EC:2.7.11.1]\n",
      "ArthCr086  rrn16S; 16S ribosomal RNA [KO:K01979]\n",
      "ArthCr088  16S ribosomal RNA [KO:K01979]\n",
      "AT3G41768  18S ribosomal RNA [KO:K01979]\n",
      "ArthCr087  rrn23S; 23S ribosomal RNA [KO:K01982]\n",
      "ArthCr089  23S ribosomal RNA [KO:K01982]\n",
      "ArthCr091  5S ribosomal RNA [KO:K01981]\n",
      "REFERENCE   PMID:20174677\n",
      "AUTHORS   Freed EF, Bleichert F, Dutca LM, Baserga SJ\n",
      "TITLE     When ribosomes go bad: diseases of ribosome biogenesis.\n",
      "JOURNAL   Mol Biosyst 6:481-93 (2010)\n",
      "DOI:10.1039/b919670f\n",
      "REFERENCE   PMID:17515605\n",
      "AUTHORS   Perez-Fernandez J, Roman A, De Las Rivas J, Bustelo XR, Dosil M\n",
      "TITLE     The 90S preribosome is a multimodular structure that is assembled through a hierarchical mechanism.\n",
      "JOURNAL   Mol Cell Biol 27:5414-29 (2007)\n",
      "DOI:10.1128/MCB.00380-07\n",
      "REFERENCE   PMID:19850913\n",
      "AUTHORS   Strunk BS, Karbstein K\n",
      "TITLE     Powering through ribosome assembly.\n",
      "JOURNAL   RNA 15:2083-104 (2009)\n",
      "DOI:10.1261/rna.1792109\n",
      "REFERENCE   PMID:15120992\n",
      "AUTHORS   Granneman S, Baserga SJ\n",
      "TITLE     Ribosome biogenesis: of knobs and RNA processing.\n",
      "JOURNAL   Exp Cell Res 296:43-50 (2004)\n",
      "DOI:10.1016/j.yexcr.2004.03.016\n",
      "REFERENCE   PMID:19879902\n",
      "AUTHORS   Kressler D, Hurt E, Bassler J\n",
      "TITLE     Driving ribosome assembly.\n",
      "JOURNAL   Biochim Biophys Acta 1803:673-83 (2010)\n",
      "DOI:10.1016/j.bbamcr.2009.10.009\n",
      "REL_PATHWAY ath03010  Ribosome\n",
      "KO_PATHWAY  ko03008\n",
      "///\n",
      "ENTRY       ath03010                    Pathway\n",
      "NAME        Ribosome - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Genetic Information Processing; Translation\n",
      "PATHWAY_MAP ath03010  Ribosome\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G71720  PDE338; Nucleic acid-binding proteins superfamily [KO:K02945]\n",
      "AT5G30510  RPS1; ribosomal protein S1 [KO:K02945]\n",
      "ArthCp011  rps2; ribosomal protein S2 [KO:K02967]\n",
      "AT3G03600  RPS2; ribosomal protein S2 [KO:K02967]\n",
      "ArthCp061  rps3; ribosomal protein S3 [KO:K02982]\n",
      "DA397_mgp40  rps3; ribosomal protein S3 [KO:K02982]\n",
      "ArthCp024  rps4; ribosomal protein S4 [KO:K02986]\n",
      "AT2G07734  Alpha-L RNA-binding motif/Ribosomal protein S4 family protein [KO:K02986]\n",
      "DA397_mgp29  rps4; ribosomal protein S4 [KO:K02986]\n",
      "AT2G33800  EMB3113; Ribosomal protein S5 family protein [KO:K02988]\n",
      "AT1G64880  Ribosomal protein S5 family protein [KO:K02988]\n",
      "AT3G18760  Translation elongation factor EF1B/ribosomal protein S6 family protein [KO:K02990]\n",
      "AT1G64510  Translation elongation factor EF1B/ribosomal protein S6 family protein [KO:K02990]\n",
      "ArthCp069  rps7; ribosomal protein S7 [KO:K02992]\n",
      "ArthCp088  rps7; ribosomal protein S7 [KO:K02992]\n",
      "AT2G07696  Ribosomal protein S7p/S5e family protein [KO:K02992]\n",
      "DA397_mgp32  rps7; ribosomal protein S7 [KO:K02992]\n",
      "ArthCp058  rps8; ribosomal protein S8 [KO:K02994]\n",
      "AT1G74970  RPS9; ribosomal protein S9 [KO:K02996]\n",
      "AT3G49080  Ribosomal protein S5 domain 2-like superfamily protein [KO:K02996]\n",
      "AT3G13120  Ribosomal protein S10p/S20e family protein [KO:K02946]\n",
      "AT3G22300  RPS10; ribosomal protein S10 [KO:K02946]\n",
      "ArthCp056  rps11; ribosomal protein S11 [KO:K02948]\n",
      "ArthCp001  rps12; ribosomal protein S12 [KO:K02950]\n",
      "ArthCp047  rps12; ribosomal protein S12 [KO:K02950]\n",
      "AT2G07675  Ribosomal protein S12/S23 family protein [KO:K02950]\n",
      "DA397_mgp13  rps12; ribosomal protein S12 [KO:K02950]\n",
      "AT1G77750  Ribosomal protein S13/S18 family [KO:K02952]\n",
      "AT5G14320  EMB3137; Ribosomal protein S13/S18 family [KO:K02952]\n",
      "AT2G34520  RPS14; mitochondrial ribosomal protein S14 [KO:K02954]\n",
      "ArthCp020  rps14; ribosomal protein S14 [KO:K02954]\n",
      "ArthCp081  rps15; ribosomal protein S15 [KO:K02956]\n",
      "ArthCp004  rps16; ribosomal protein S16 [KO:K02959]\n",
      "AT5G56940  Ribosomal protein S16 family protein [KO:K02959]\n",
      "AT4G34620  SSR16; small subunit ribosomal protein 16 [KO:K02959]\n",
      "AT1G49400  emb1129; Nucleic acid-binding, OB-fold-like protein [KO:K02961]\n",
      "AT1G79850  RPS17; ribosomal protein S17 [KO:K02961]\n",
      "AT3G18880  Nucleic acid-binding, OB-fold-like protein [KO:K02961]\n",
      "ArthCp044  rps18; ribosomal protein S18 [KO:K02963]\n",
      "AT5G47320  RPS19; ribosomal protein S19 [KO:K02965]\n",
      "ArthCp063  rps19; ribosomal protein S19 [KO:K02965]\n",
      "AT3G15190  PRPS20; chloroplast 30S ribosomal protein S20 [KO:K02968]\n",
      "AT3G26360  Ribosomal protein S21 family protein [KO:K02970]\n",
      "AT1G58380  XW6; Ribosomal protein S5 family protein [KO:K02981]\n",
      "AT1G59359  Ribosomal protein S5 family protein [KO:K02981]\n",
      "AT3G57490  Ribosomal protein S5 family protein [KO:K02981]\n",
      "AT2G41840  Ribosomal protein S5 family protein [KO:K02981]\n",
      "AT1G58684  Ribosomal protein S5 family protein [KO:K02981]\n",
      "AT1G58983  Ribosomal protein S5 family protein [KO:K02981]\n",
      "AT2G31610  Ribosomal protein S3 family protein [KO:K02985]\n",
      "AT5G35530  Ribosomal protein S3 family protein [KO:K02985]\n",
      "AT3G53870  Ribosomal protein S3 family protein [KO:K02985]\n",
      "AT3G04840  Ribosomal protein S3Ae [KO:K02984]\n",
      "AT4G34670  Ribosomal protein S3Ae [KO:K02984]\n",
      "AT5G58420  Ribosomal protein S4 (RPS4A) family protein [KO:K02987]\n",
      "AT2G17360  Ribosomal protein S4 (RPS4A) family protein [KO:K02987]\n",
      "AT5G07090  Ribosomal protein S4 (RPS4A) family protein [KO:K02987]\n",
      "AT2G37270  RPS5B; ribosomal protein 5B [KO:K02989]\n",
      "AT3G11940  RPS5A; ribosomal protein 5A [KO:K02989]\n",
      "AT5G10360  EMB3010; Ribosomal protein S6e [KO:K02991]\n",
      "AT4G31700  RPS6; ribosomal protein S6 [KO:K02991]\n",
      "AT5G16130  Ribosomal protein S7e family protein [KO:K02993]\n",
      "AT3G02560  Ribosomal protein S7e family protein [KO:K02993]\n",
      "AT1G48830  Ribosomal protein S7e family protein [KO:K02993]\n",
      "AT5G59240  Ribosomal protein S8e family protein [KO:K02995]\n",
      "AT5G20290  Ribosomal protein S8e family protein [KO:K02995]\n",
      "AT5G15200  Ribosomal protein S4 [KO:K02997]\n",
      "AT5G39850  Ribosomal protein S4 [KO:K02997]\n",
      "AT5G41520  RPS10B; RNA binding Plectin/S10 domain-containing protein [KO:K02947]\n",
      "AT5G52650  RNA binding Plectin/S10 domain-containing protein [KO:K02947]\n",
      "AT4G25740  RNA binding Plectin/S10 domain-containing protein [KO:K02947]\n",
      "AT3G48930  EMB1080; Nucleic acid-binding, OB-fold-like protein [KO:K02949]\n",
      "AT5G23740  RPS11-BETA; ribosomal protein S11-beta [KO:K02949]\n",
      "AT4G30800  Nucleic acid-binding, OB-fold-like protein [KO:K02949]\n",
      "AT2G32060  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K02951]\n",
      "AT1G15930  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K02951]\n",
      "AT3G60770  Ribosomal protein S13/S15 [KO:K02953]\n",
      "AT4G00100  RPS13A; ribosomal protein S13A [KO:K02953]\n",
      "AT3G11510  Ribosomal protein S11 family protein [KO:K02955]\n",
      "AT3G52580  Ribosomal protein S11 family protein [KO:K02955]\n",
      "AT2G36160  Ribosomal protein S11 family protein [KO:K02955]\n",
      "AT5G63070  Ribosomal protein S19 family protein [KO:K02958]\n",
      "AT5G09490  Ribosomal protein S19 family protein [KO:K02958]\n",
      "AT5G43640  Ribosomal protein S19 family protein [KO:K02958]\n",
      "AT5G09510  Ribosomal protein S19 family protein [KO:K02958]\n",
      "AT5G09500  Ribosomal protein S19 family protein [KO:K02958]\n",
      "AT1G04270  RPS15; cytosolic ribosomal protein S15 [KO:K02958]\n",
      "AT1G33850  Ribosomal protein S19 family protein [KO:K02958]\n",
      "AT2G39590  40S ribosomal protein S15a [KO:K02957]\n",
      "AT3G46040  RPS15AD; ribosomal protein S15A D [KO:K02957]\n",
      "AT2G19720  rps15ab; ribosomal protein S15A B [KO:K02957]\n",
      "AT1G07770  RPS15A; ribosomal protein S15A [KO:K02957]\n",
      "AT4G29430  rps15ae; ribosomal protein S15A E [KO:K02957]\n",
      "AT5G59850  Ribosomal protein S8 family protein [KO:K02957]\n",
      "AT3G04230  Ribosomal protein S5 domain 2-like superfamily protein [KO:K02960]\n",
      "AT5G18380  Ribosomal protein S5 domain 2-like superfamily protein [KO:K02960]\n",
      "AT2G09990  Ribosomal protein S5 domain 2-like superfamily protein [KO:K02960]\n",
      "AT2G04390  Ribosomal S17 family protein [KO:K02962]\n",
      "AT3G10610  Ribosomal S17 family protein [KO:K02962]\n",
      "AT2G05220  Ribosomal S17 family protein [KO:K02962]\n",
      "AT5G04800  Ribosomal S17 family protein [KO:K02962]\n",
      "AT1G22780  PFL; Ribosomal protein S13/S18 family [KO:K02964]\n",
      "AT1G34030  Ribosomal protein S13/S18 family [KO:K02964]\n",
      "AT4G09800  RPS18C; S18 ribosomal protein [KO:K02964]\n",
      "AT5G15520  Ribosomal protein S19e family protein [KO:K02966]\n",
      "AT5G61170  Ribosomal protein S19e family protein [KO:K02966]\n",
      "AT3G02080  Ribosomal protein S19e family protein [KO:K02966]\n",
      "AT5G62300  Ribosomal protein S10p/S20e family protein [KO:K02969]\n",
      "AT3G45030  Ribosomal protein S10p/S20e family protein [KO:K02969]\n",
      "AT3G47370  Ribosomal protein S10p/S20e family protein [KO:K02969]\n",
      "AT5G27700  Ribosomal protein S21e [KO:K02971]\n",
      "AT3G53890  Ribosomal protein S21e [KO:K02971]\n",
      "AT3G09680  Ribosomal protein S12/S23 family protein [KO:K02973]\n",
      "AT5G02960  Ribosomal protein S12/S23 family protein [KO:K02973]\n",
      "AT5G28060  Ribosomal protein S24e family protein [KO:K02974]\n",
      "AT3G04920  Ribosomal protein S24e family protein [KO:K02974]\n",
      "AT4G34555  Ribosomal protein S25 family protein [KO:K02975]\n",
      "AT2G16360  40S ribosomal protein S25 [KO:K02975]\n",
      "AT4G39200  Ribosomal protein S25 family protein [KO:K02975]\n",
      "AT2G21580  Ribosomal protein S25 family protein [KO:K02975]\n",
      "AT2G40510  Ribosomal protein S26e family protein [KO:K02976]\n",
      "AT2G40590  Ribosomal protein S26e family protein [KO:K02976]\n",
      "AT3G56340  Ribosomal protein S26e family protein [KO:K02976]\n",
      "AT2G45710  Zinc-binding ribosomal protein family protein [KO:K02978]\n",
      "AT3G61110  RS27A; ribosomal protein S27 [KO:K02978]\n",
      "AT5G47930  Zinc-binding ribosomal protein family protein [KO:K02978]\n",
      "AT3G61111  Zinc-binding ribosomal protein family protein [KO:K02978]\n",
      "AT1G23410  Ribosomal protein S27a / Ubiquitin family protein [KO:K02977]\n",
      "AT3G62250  UBQ5; ubiquitin 5 [KO:K02977]\n",
      "AT2G47110  UBQ6; ubiquitin 6 [KO:K02977]\n",
      "AT5G03850  Nucleic acid-binding, OB-fold-like protein [KO:K02979]\n",
      "AT5G64140  RPS28; ribosomal protein S28 [KO:K02979]\n",
      "AT3G10090  Nucleic acid-binding, OB-fold-like protein [KO:K02979]\n",
      "AT3G43980  Ribosomal protein S14p/S29e family protein [KO:K02980]\n",
      "AT4G33865  Ribosomal protein S14p/S29e family protein [KO:K02980]\n",
      "AT3G44010  Ribosomal protein S14p/S29e family protein [KO:K02980]\n",
      "AT5G56670  Ribosomal protein S30 family protein [KO:K02983]\n",
      "AT2G19750  Ribosomal protein S30 family protein [KO:K02983]\n",
      "AT4G29390  Ribosomal protein S30 family protein [KO:K02983]\n",
      "AT1G72370  P40; 40s ribosomal protein SA [KO:K02998]\n",
      "AT3G04770  RPSAb; 40s ribosomal protein SA B [KO:K02998]\n",
      "AT3G63490  EMB3126; Ribosomal protein L1p/L10e family [KO:K02863]\n",
      "ArthCp064  rpl2; ribosomal protein L2 [KO:K02886]\n",
      "ArthCp085  rpl2; ribosomal protein L2 [KO:K02886]\n",
      "AT2G43030  Ribosomal protein L3 family protein [KO:K02906]\n",
      "AT3G17465  RPL3P; ribosomal protein L3 plastid [KO:K02906]\n",
      "AT2G20060  Ribosomal protein L4/L1 family [KO:K02926]\n",
      "AT1G07320  RPL4; ribosomal protein L4 [KO:K02926]\n",
      "AT4G01310  Ribosomal L5P family protein [KO:K02931]\n",
      "AT2G18400  ribosomal protein L6 family protein [KO:K02933]\n",
      "AT1G05190  emb2394; Ribosomal protein L6 family [KO:K02933]\n",
      "AT3G27830  RPL12-A; ribosomal protein L12-A [KO:K02935]\n",
      "AT3G27840  RPL12-B; ribosomal protein L12-B [KO:K02935]\n",
      "AT3G27850  RPL12-C; ribosomal protein L12-C [KO:K02935]\n",
      "AT1G70190  ribosomal protein L7/L12 domain-containing protein [KO:K02935]\n",
      "AT4G36420  Ribosomal protein L12 family protein [KO:K02935]\n",
      "AT3G06040  Ribosomal protein L12/ ATP-dependent Clp protease adaptor protein ClpS family protein [KO:K02935]\n",
      "AT4G37660  Ribosomal protein L12/ ATP-dependent Clp protease adaptor protein ClpS family protein [KO:K02935]\n",
      "AT3G44890  RPL9; ribosomal protein L9 [KO:K02939]\n",
      "AT5G53070  Ribosomal protein L9/RNase H1 [KO:K02939]\n",
      "AT3G12370  Ribosomal protein L10 family protein [KO:K02864]\n",
      "AT5G13510  EMB3136; Ribosomal protein L10 family protein [KO:K02864]\n",
      "AT1G32990  PRPL11; plastid ribosomal protein l11 [KO:K02867]\n",
      "AT5G51610  Ribosomal protein L11 family protein [KO:K02867]\n",
      "AT4G35490  MRPL11; mitochondrial ribosomal protein L11 [KO:K02867]\n",
      "AT1G78630  emb1473; Ribosomal protein L13 family protein [KO:K02871]\n",
      "AT3G01790  Ribosomal protein L13 family protein [KO:K02871]\n",
      "AT1G17560  HLL; Ribosomal protein L14p/L23e family protein [KO:K02874]\n",
      "AT5G46160  Ribosomal protein L14p/L23e family protein [KO:K02874]\n",
      "ArthCp059  rpl14; ribosomal protein L14 [KO:K02874]\n",
      "AT3G25920  RPL15; ribosomal protein L15 [KO:K02876]\n",
      "AT5G64670  Ribosomal protein L18e/L15 superfamily protein [KO:K02876]\n",
      "ArthCp060  rpl16; ribosomal protein L16 [KO:K02878]\n",
      "DA397_mgp39  rpl16; ribosomal protein L16 [KO:K02878]\n",
      "AT3G54210  Ribosomal protein L17 family protein [KO:K02879]\n",
      "AT5G64650  Ribosomal protein L17 family protein [KO:K02879]\n",
      "AT5G09770  Ribosomal protein L17 family protein [KO:K02879]\n",
      "AT1G48350  EMB3105; Ribosomal L18p/L5e family protein [KO:K02881]\n",
      "AT3G22450  Ribosomal L18p/L5e family protein [KO:K02881]\n",
      "AT3G20230  Ribosomal L18p/L5e family protein [KO:K02881]\n",
      "AT4G17560  Ribosomal protein L19 family protein [KO:K02884]\n",
      "AT5G47190  Ribosomal protein L19 family protein [KO:K02884]\n",
      "ArthCp045  rpl20; ribosomal protein L20 [KO:K02887]\n",
      "AT1G16740  Ribosomal protein L20 [KO:K02887]\n",
      "AT1G35680  RPL21C; Ribosomal protein L21 [KO:K02888]\n",
      "AT4G30930  NFD1; Ribosomal protein L21 [KO:K02888]\n",
      "ArthCp062  rpl22; ribosomal protein L22 [KO:K02890]\n",
      "AT4G28360  Ribosomal protein L22p/L17e family protein [KO:K02890]\n",
      "AT1G52370  Ribosomal protein L22p/L17e family protein [KO:K02890]\n",
      "ArthCp065  rpl23; ribosomal protein L23 [KO:K02892]\n",
      "ArthCp084  rpl23; ribosomal protein L23 [KO:K02892]\n",
      "AT4G39880  Ribosomal protein L23/L15e family protein [KO:K02892]\n",
      "AT5G54600  RPL24; Translation protein SH3-like family protein [KO:K02895]\n",
      "AT5G23535  KOW domain-containing protein [KO:K02895]\n",
      "AT4G23620  Ribosomal protein L25/Gln-tRNA synthetase, anti-codon-binding domain-containing protein [KO:K02897]\n",
      "AT5G66860  Ribosomal protein L25/Gln-tRNA synthetase, anti-codon-binding domain-containing protein [KO:K02897]\n",
      "AT5G40950  RPL27; ribosomal protein large subunit 27 [KO:K02899]\n",
      "AT5G15220  Ribosomal protein L27 family protein [KO:K02899]\n",
      "AT2G16930  Ribosomal protein L27 family protein [KO:K02899]\n",
      "AT2G33450  PRPL28; Ribosomal L28 family [KO:K02902]\n",
      "AT4G31460  Ribosomal L28 family [KO:K02902]\n",
      "AT5G65220  Ribosomal L29 family protein [KO:K02904]\n",
      "AT5G55140  ribosomal protein L30 family protein [KO:K02907]\n",
      "AT1G75350  emb2184; Ribosomal protein L31 [KO:K02909]\n",
      "ArthCp072  rpl32; ribosomal protein L32 [KO:K02911]\n",
      "AT1G26740  Ribosomal L32p protein family [KO:K02911]\n",
      "AT1G69485  Ribosomal L32p protein family [KO:K02911]\n",
      "ArthCp043  rpl33; ribosomal protein L33 [KO:K02913]\n",
      "AT5G18790  Ribosomal protein L33 family protein [KO:K02913]\n",
      "AT3G06320  Ribosomal protein L33 family protein [KO:K02913]\n",
      "AT3G13882  Ribosomal protein L34 [KO:K02914]\n",
      "AT1G29070  Ribosomal protein L34 [KO:K02914]\n",
      "AT2G24090  PRPL35; Ribosomal protein L35 [KO:K02916]\n",
      "AT5G20180  Ribosomal protein L36 [KO:K02919]\n",
      "ArthCp057  rpl36; ribosomal protein L36 [KO:K02919]\n",
      "AT1G61580  RPL3B; R-protein L3 B [KO:K02925]\n",
      "AT1G43170  RP1; ribosomal protein 1 [KO:K02925]\n",
      "AT5G02870  Ribosomal protein L4/L1 family [KO:K02930]\n",
      "AT3G09630  Ribosomal protein L4/L1 family [KO:K02930]\n",
      "AT3G25520  ATL5; ribosomal protein L5 [KO:K02932]\n",
      "AT5G39740  RPL5B; ribosomal protein L5 B [KO:K02932]\n",
      "AT1G74060  Ribosomal protein L6 family protein [KO:K02934]\n",
      "AT1G18540  Ribosomal protein L6 family protein [KO:K02934]\n",
      "AT1G74050  Ribosomal protein L6 family protein [KO:K02934]\n",
      "AT1G80750  Ribosomal protein L30/L7 family protein [KO:K02937]\n",
      "AT2G01250  Ribosomal protein L30/L7 family protein [KO:K02937]\n",
      "AT3G13580  Ribosomal protein L30/L7 family protein [KO:K02937]\n",
      "AT2G44120  Ribosomal protein L30/L7 family protein [KO:K02937]\n",
      "AT2G47610  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K02936]\n",
      "AT3G62870  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K02936]\n",
      "AT3G51190  Ribosomal protein L2 family [KO:K02938]\n",
      "AT2G18020  EMB2296; Ribosomal protein L2 family [KO:K02938]\n",
      "AT4G36130  Ribosomal protein L2 family [KO:K02938]\n",
      "AT4G10450  Ribosomal protein L6 family [KO:K02940]\n",
      "AT1G33140  PGY2; Ribosomal protein L6 family [KO:K02940]\n",
      "AT1G33120  Ribosomal protein L6 family [KO:K02940]\n",
      "AT1G66580  SAG24; senescence associated gene 24 [KO:K02866]\n",
      "AT1G26910  RPL10B; Ribosomal protein L16p/L10e family protein [KO:K02866]\n",
      "AT1G14320  SAC52; Ribosomal protein L16p/L10e family protein [KO:K02866]\n",
      "AT5G22440  Ribosomal protein L1p/L10e family [KO:K02865]\n",
      "AT1G08360  Ribosomal protein L1p/L10e family [KO:K02865]\n",
      "AT2G27530  PGY1; Ribosomal protein L1p/L10e family [KO:K02865]\n",
      "AT3G58700  Ribosomal L5P family protein [KO:K02868]\n",
      "AT4G18730  RPL16B; ribosomal protein L16B [KO:K02868]\n",
      "AT5G45775  Ribosomal L5P family protein [KO:K02868]\n",
      "AT2G42740  RPL16A; ribosomal protein large subunit 16A [KO:K02868]\n",
      "AT2G37190  Ribosomal protein L11 family protein [KO:K02870]\n",
      "AT5G60670  Ribosomal protein L11 family protein [KO:K02870]\n",
      "AT3G53430  Ribosomal protein L11 family protein [KO:K02870]\n",
      "AT3G48960  Ribosomal protein L13e family protein [KO:K02873]\n",
      "AT5G23900  Ribosomal protein L13e family protein [KO:K02873]\n",
      "AT3G49010  BBC1; breast basic conserved 1 [KO:K02873]\n",
      "AT4G13170  Ribosomal protein L13 family protein [KO:K02872]\n",
      "AT3G24830  Ribosomal protein L13 family protein [KO:K02872]\n",
      "AT5G48760  Ribosomal protein L13 family protein [KO:K02872]\n",
      "AT3G07110  Ribosomal protein L13 family protein [KO:K02872]\n",
      "AT2G20450  Ribosomal protein L14 [KO:K02875]\n",
      "AT4G27090  Ribosomal protein L14 [KO:K02875]\n",
      "AT4G17390  Ribosomal protein L23/L15e family protein [KO:K02877]\n",
      "AT4G16720  Ribosomal protein L23/L15e family protein [KO:K02877]\n",
      "AT1G27400  Ribosomal protein L22p/L17e family protein [KO:K02880]\n",
      "AT1G67430  Ribosomal protein L22p/L17e family protein [KO:K02880]\n",
      "AT2G47570  60S ribosomal protein L18 [KO:K02883]\n",
      "AT5G27850  Ribosomal protein L18e/L15 superfamily protein [KO:K02883]\n",
      "AT3G05590  RPL18; ribosomal protein L18 [KO:K02883]\n",
      "AT3G14600  Ribosomal protein L18ae/LX family protein [KO:K02882]\n",
      "AT2G34480  Ribosomal protein L18ae/LX family protein [KO:K02882]\n",
      "AT1G29965  Ribosomal protein L18ae/LX family protein [KO:K02882]\n",
      "AT1G29970  RPL18AA; 60S ribosomal protein L18A-1 [KO:K02882]\n",
      "AT4G02230  Ribosomal protein L19e family protein [KO:K02885]\n",
      "AT3G16780  Ribosomal protein L19e family protein [KO:K02885]\n",
      "AT1G02780  emb2386; Ribosomal protein L19e family protein [KO:K02885]\n",
      "AT4G16030  Ribosomal protein L19e family protein [KO:K02885]\n",
      "AT1G57860  Translation protein SH3-like family protein [KO:K02889]\n",
      "AT1G09690  Translation protein SH3-like family protein [KO:K02889]\n",
      "AT1G57660  Translation protein SH3-like family protein [KO:K02889]\n",
      "AT1G09590  Translation protein SH3-like family protein [KO:K02889]\n",
      "AT1G02830  Ribosomal L22e protein family [KO:K02891]\n",
      "AT5G27770  Ribosomal L22e protein family [KO:K02891]\n",
      "AT3G05560  Ribosomal L22e protein family [KO:K02891]\n",
      "AT3G04400  emb2171; Ribosomal protein L14p/L23e family protein [KO:K02894]\n",
      "AT1G04480  Ribosomal protein L14p/L23e family protein [KO:K02894]\n",
      "AT2G33370  Ribosomal protein L14p/L23e family protein [KO:K02894]\n",
      "AT3G55280  RPL23AB; ribosomal protein L23AB [KO:K02893]\n",
      "AT2G39460  RPL23AA; ribosomal protein L23AA [KO:K02893]\n",
      "AT2G36620  RPL24A; ribosomal protein L24 [KO:K02896]\n",
      "AT2G44860  Ribosomal protein L24e family protein [KO:K02896]\n",
      "AT3G53020  STV1; Ribosomal protein L24e family protein [KO:K02896]\n",
      "AT5G67510  Translation protein SH3-like family protein [KO:K02898]\n",
      "AT3G49910  Translation protein SH3-like family protein [KO:K02898]\n",
      "AT4G15000  Ribosomal L27e protein family [KO:K02901]\n",
      "AT3G22230  Ribosomal L27e protein family [KO:K02901]\n",
      "AT2G32220  Ribosomal L27e protein family [KO:K02901]\n",
      "AT1G23290  RPL27AB; Ribosomal protein L18e/L15 superfamily protein [KO:K02900]\n",
      "AT1G70600  Ribosomal protein L18e/L15 superfamily protein [KO:K02900]\n",
      "AT1G12960  Ribosomal protein L18e/L15 superfamily protein [KO:K02900]\n",
      "AT4G29410  Ribosomal L28e protein family [KO:K02903]\n",
      "AT2G19730  Ribosomal L28e protein family [KO:K02903]\n",
      "AT3G06680  Ribosomal L29e protein family [KO:K02905]\n",
      "AT3G06700  Ribosomal L29e protein family [KO:K02905]\n",
      "AT1G77940  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K02908]\n",
      "AT1G36240  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K02908]\n",
      "AT3G18740  RLK902; Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K02908]\n",
      "AT2G19740  Ribosomal protein L31e family protein [KO:K02910]\n",
      "AT5G56710  Ribosomal protein L31e family protein [KO:K02910]\n",
      "AT4G26230  Ribosomal protein L31e family protein [KO:K02910]\n",
      "AT4G18100  Ribosomal protein L32e [KO:K02912]\n",
      "AT5G46430  Ribosomal protein L32e [KO:K02912]\n",
      "AT3G28900  Ribosomal protein L34e superfamily protein [KO:K02915]\n",
      "AT1G69620  RPL34; ribosomal protein L34 [KO:K02915]\n",
      "AT1G26880  Ribosomal protein L34e superfamily protein [KO:K02915]\n",
      "AT3G09500  Ribosomal L29 family protein [KO:K02918]\n",
      "AT5G02610  Ribosomal L29 family protein [KO:K02918]\n",
      "AT2G39390  Ribosomal L29 family protein [KO:K02918]\n",
      "AT3G55170  Ribosomal L29 family protein [KO:K02918]\n",
      "AT3G55750  Ribosomal protein L35Ae family protein [KO:K02917]\n",
      "AT1G41880  Ribosomal protein L35Ae family protein [KO:K02917]\n",
      "AT1G07070  Ribosomal protein L35Ae family protein [KO:K02917]\n",
      "AT1G74270  Ribosomal protein L35Ae family protein [KO:K02917]\n",
      "AT5G02450  Ribosomal protein L36e family protein [KO:K02920]\n",
      "AT2G37600  Ribosomal protein L36e family protein [KO:K02920]\n",
      "AT3G53740  Ribosomal protein L36e family protein [KO:K02920]\n",
      "AT3G16080  Zinc-binding ribosomal protein family protein [KO:K02922]\n",
      "AT1G15250  Zinc-binding ribosomal protein family protein [KO:K02922]\n",
      "AT1G52300  Zinc-binding ribosomal protein family protein [KO:K02922]\n",
      "AT3G10950  Zinc-binding ribosomal protein family protein [KO:K02921]\n",
      "AT3G60245  Zinc-binding ribosomal protein family protein [KO:K02921]\n",
      "AT3G59540  Ribosomal L38e protein family [KO:K02923]\n",
      "AT2G43460  Ribosomal L38e protein family [KO:K02923]\n",
      "AT3G02190  Ribosomal protein L39 family protein [KO:K02924]\n",
      "AT4G31985  Ribosomal protein L39 family protein [KO:K02924]\n",
      "AT2G25210  Ribosomal protein L39 family protein [KO:K02924]\n",
      "AT2G36170  60S ribosomal protein L40-1 [KO:K02927]\n",
      "AT3G52590  UBQ1; ubiquitin extension protein 1 [KO:K02927]\n",
      "AT2G40205  Ribosomal protein L41 family [KO:K02928]\n",
      "AT3G08520  Ribosomal protein L41 family [KO:K02928]\n",
      "AT3G11120  Ribosomal protein L41 family [KO:K02928]\n",
      "AT3G56020  Ribosomal protein L41 family [KO:K02928]\n",
      "AT1G56045  Ribosomal protein L41 family [KO:K02928]\n",
      "AT4G14320  Zinc-binding ribosomal protein family protein [KO:K02929]\n",
      "AT3G23390  Zinc-binding ribosomal protein family protein [KO:K02929]\n",
      "AT2G40010  Ribosomal protein L10 family protein [KO:K02941]\n",
      "AT3G09200  Ribosomal protein L10 family protein [KO:K02941]\n",
      "AT3G11250  Ribosomal protein L10 family protein [KO:K02941]\n",
      "AT5G24510  60S acidic ribosomal protein family [KO:K02942]\n",
      "AT5G47700  60S acidic ribosomal protein family [KO:K02942]\n",
      "AT4G00810  60S acidic ribosomal protein family [KO:K02942]\n",
      "AT4G25890  60S acidic ribosomal protein family [KO:K02942]\n",
      "AT5G57290  60S acidic ribosomal protein family [KO:K02942]\n",
      "AT1G01100  60S acidic ribosomal protein family [KO:K02942]\n",
      "AT2G27720  60S acidic ribosomal protein family [KO:K02943]\n",
      "AT5G40040  60S acidic ribosomal protein family [KO:K02943]\n",
      "AT3G28500  60S acidic ribosomal protein family [KO:K02943]\n",
      "AT3G44590  60S acidic ribosomal protein family [KO:K02943]\n",
      "AT2G27710  60S acidic ribosomal protein family [KO:K02943]\n",
      "ArthCr086  rrn16S; 16S ribosomal RNA [KO:K01979]\n",
      "ArthCr088  16S ribosomal RNA [KO:K01979]\n",
      "AT3G41768  18S ribosomal RNA [KO:K01979]\n",
      "ArthCr087  rrn23S; 23S ribosomal RNA [KO:K01982]\n",
      "ArthCr089  23S ribosomal RNA [KO:K01982]\n",
      "ArthCr091  5S ribosomal RNA [KO:K01981]\n",
      "REFERENCE   PMID:10476961\n",
      "AUTHORS   Ban N, Nissen P, Hansen J, Capel M, Moore PB, Steitz TA.\n",
      "TITLE     Placement of protein and RNA structures into a 5 A-resolution map of the 50S ribosomal subunit.\n",
      "JOURNAL   Nature 400:841-847 (1999)\n",
      "DOI:10.1038/23641\n",
      "REFERENCE   PMID:10476960\n",
      "AUTHORS   Clemons WM Jr, May JL, Wimberly BT, McCutcheon JP, Capel MS, Ramakrishnan V.\n",
      "TITLE     Structure of a bacterial 30S ribosomal subunit at 5.5 A resolution.\n",
      "JOURNAL   Nature 400:833-840 (1999)\n",
      "DOI:10.1038/23631\n",
      "REFERENCE   PMID:10497122\n",
      "AUTHORS   Cate JH, Yusupov MM, Yusupova GZ, Earnest TN, Noller HF.\n",
      "TITLE     X-ray crystal structures of 70S ribosome functional complexes.\n",
      "JOURNAL   Science 285:2095-2104 (1999)\n",
      "DOI:10.1126/science.285.5436.2095\n",
      "REFERENCE   PMID:10937989\n",
      "AUTHORS   Ban N, Nissen P, Hansen J, Moore PB, Steitz TA.\n",
      "TITLE     The complete atomic structure of the large ribosomal subunit at 2.4 A resolution.\n",
      "JOURNAL   Science 289:905-920 (2000)\n",
      "DOI:10.1126/science.289.5481.905\n",
      "REFERENCE   PMID:10937990\n",
      "AUTHORS   Nissen P, Hansen J, Ban N, Moore PB, Steitz TA.\n",
      "TITLE     The structural basis of ribosome activity in peptide bond synthesis.\n",
      "JOURNAL   Science 289:920-930 (2000)\n",
      "DOI:10.1126/science.289.5481.920\n",
      "REFERENCE   PMID:11014182\n",
      "AUTHORS   Wimberly BT, Brodersen DE, Clemons WM Jr, Morgan-Warren RJ, Carter AP, Vonrhein C, Hartsch T, Ramakrishnan V.\n",
      "TITLE     Structure of the 30S ribosomal subunit.\n",
      "JOURNAL   Nature 407:327-339 (2000)\n",
      "DOI:10.1038/35030006\n",
      "REFERENCE   PMID:11014183\n",
      "AUTHORS   Carter AP, Clemons WM, Brodersen DE, Morgan-Warren RJ, Wimberly BT, Ramakrishnan V.\n",
      "TITLE     Functional insights from the structure of the 30S ribosomal subunit and its interactions with antibiotics.\n",
      "JOURNAL   Nature 407:340-348 (2000)\n",
      "DOI:10.1038/35030019\n",
      "KO_PATHWAY  ko03010\n",
      "///\n",
      "ENTRY       ath03013                    Pathway\n",
      "NAME        Nucleocytoplasmic transport - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The exchange of molecules between the nucleus and cytoplasm is mediated through nuclear pore complexes (NPCs) embedded in the nuclear envelope. The NPC is composed of highly conserved, distinct structural modules: the inner pore ring, the cytoplasmic and nuclear rings as well as the cytoplasmic filaments and the nuclear basket. Nuclear transport receptors (e.g. importins and exportins) mediate translocation of cargos through the NPC. Cargo proteins harbouring a nuclear export signal (NES) bind to exportins and Ran-GTP to form an export complex in the nucleoplasm. Nuclear export of bulk mRNA is provided by the non-karyopherin transport proteins TAP-p15 in a Ran-GTP-independent manner. Several EJC factors are known to function in mRNA export.\n",
      "CLASS       Genetic Information Processing; Translation\n",
      "PATHWAY_MAP ath03013  Nucleocytoplasmic transport\n",
      "DBLINKS     GO: 0050658\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G56900  Transducin/WD40 repeat-like superfamily protein [KO:K14320]\n",
      "AT1G75340  Zinc finger C-x8-C-x5-C-x3-H type family protein [KO:K14321]\n",
      "AT1G13120  GLE1; nucleoporin GLE1-like protein [KO:K18723]\n",
      "AT3G53110  LOS4; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K18655] [EC:3.6.4.13]\n",
      "AT1G80670  RAE1; Transducin/WD40 repeat-like superfamily protein [KO:K14298]\n",
      "AT1G15850  Transducin/WD40 repeat-like superfamily protein [KO:K14298]\n",
      "AT1G10390  Nucleoporin autopeptidase [KO:K14297]\n",
      "AT1G59660  Nucleoporin autopeptidase [KO:K14297]\n",
      "AT1G80680  SAR3; SUPPRESSOR OF AUXIN RESISTANCE 3 [KO:K14297]\n",
      "AT1G55540  LNO1; Nuclear pore complex protein [KO:K14317]\n",
      "AT5G05680  MOS7; nuclear pore complex protein-like protein [KO:K14318]\n",
      "AT3G63130  RANGAP1; RAN GTPase activating protein 1 [KO:K14319]\n",
      "AT5G19320  RANGAP2; RAN GTPase activating protein 2 [KO:K14319]\n",
      "AT3G06000  RNI-like superfamily protein [KO:K14319]\n",
      "AT3G57870  SCE1; sumo conjugation enzyme 1 [KO:K10577]\n",
      "AT1G33410  SAR1; SUPPRESSOR OF AUXIN RESISTANCE1 [KO:K14303]\n",
      "AT4G32910  nuclear pore complex Nup85-like protein [KO:K14304]\n",
      "AT2G30050  transducin family protein / WD-40 repeat family protein [KO:K14004]\n",
      "AT3G01340  Transducin/WD40 repeat-like superfamily protein [KO:K14004]\n",
      "AT3G14120  nuclear pore complex protein [KO:K14301]\n",
      "AT2G05120  Nucleoporin, Nup133/Nup155-like protein [KO:K14300]\n",
      "AT1G64350  SEH1H; Transducin/WD40 repeat-like superfamily protein [KO:K14299]\n",
      "AT4G30840  Transducin/WD40 repeat-like superfamily protein [KO:K14305]\n",
      "AT2G45000  EMB2766; structural constituent of nuclear pore [KO:K14306]\n",
      "AT4G37130  hydroxyproline-rich glycoprotein family protein [KO:K14307]\n",
      "AT1G24310  nuclear pore complex protein [KO:K14308]\n",
      "AT5G51200  EMB3142; nuclear pore complex protein (DUF3414) [KO:K14310]\n",
      "AT4G38760  nucleoporin (DUF3414) [KO:K14311]\n",
      "AT1G14850  NUP155; nucleoporin 155 [KO:K14312]\n",
      "AT2G41620  Nucleoporin interacting component (Nup93/Nic96-like) family protein [KO:K14309]\n",
      "AT3G57350  Nucleoporin interacting component (Nup93/Nic96-like) family protein [KO:K14309]\n",
      "AT3G16310  mitotic phosphoprotein N' end (MPPN) family protein [KO:K14313]\n",
      "AT1G73240  nucleoporin protein Ndc1-Nup protein [KO:K14315]\n",
      "AT5G40480  EMB3012; embryo defective 3012 [KO:K14314]\n",
      "AT3G02420  dihydroflavonol 4-reductase/flavanone protein [KO:K20724]\n",
      "AT1G79280  NUA; nuclear pore anchor [KO:K09291]\n",
      "AT1G02690  IMPA-6; importin alpha isoform 6 [KO:K15042]\n",
      "AT5G49310  IMPA-5; importin alpha isoform 5 [KO:K15042]\n",
      "AT5G03070  IMPA-9; importin alpha isoform 9 [KO:K15042]\n",
      "AT3G06720  IMPA-1; importin alpha isoform 1 [KO:K15042]\n",
      "AT1G09270  IMPA-4; importin alpha isoform 4 [KO:K15042]\n",
      "AT3G05720  IMPA-7; importin alpha isoform 7 [KO:K15042]\n",
      "AT4G02150  MOS6; ARM repeat superfamily protein [KO:K15042]\n",
      "AT3G08943  ARM repeat superfamily protein [KO:K14293]\n",
      "AT5G53480  ARM repeat superfamily protein [KO:K14293]\n",
      "AT3G08947  ARM repeat superfamily protein [KO:K14293]\n",
      "AT2G16950  TRN1; transportin 1 [KO:K18752]\n",
      "AT5G62600  MOS14; ARM repeat superfamily protein [KO:K15436]\n",
      "AT1G12930  ARM repeat superfamily protein [KO:K15436]\n",
      "AT4G27640  ARM repeat superfamily protein [KO:K20221]\n",
      "AT5G19820  emb2734; ARM repeat superfamily protein [KO:K20222]\n",
      "AT2G31660  SAD2; ARM repeat superfamily protein [KO:K20223]\n",
      "AT3G59020  ARM repeat superfamily protein [KO:K20223]\n",
      "AT1G26170  ARM repeat superfamily protein [KO:K20224]\n",
      "AT3G08960  ARM repeat superfamily protein [KO:K25201]\n",
      "AT4G24880  snurportin-1 protein [KO:K13151]\n",
      "AT3G03110  XPO1B; exportin 1B [KO:K14290]\n",
      "AT5G17020  XPO1A; exportin 1A [KO:K14290]\n",
      "AT2G46520  cellular apoptosis susceptibility protein, putative / importin-alpha re-exporter [KO:K18423]\n",
      "AT1G72560  PSD; ARM repeat superfamily protein [KO:K14288]\n",
      "AT3G04490  exportin-4 protein [KO:K25203]\n",
      "AT3G05040  HST; ARM repeat superfamily protein [KO:K14289]\n",
      "AT5G06120  ARM repeat superfamily protein [KO:K18460]\n",
      "AT5G20010  RAN-1; RAS-related nuclear protein-1 [KO:K07936]\n",
      "AT5G20020  RAN2; RAS-related GTP-binding nuclear protein 2 [KO:K07936]\n",
      "AT5G55080  RAN4; ras-related nuclear protein 4 [KO:K07936]\n",
      "AT5G55190  RAN3; RAN GTPase 3 [KO:K07936]\n",
      "AT1G07940  GTP binding Elongation factor Tu family protein [KO:K03231]\n",
      "AT5G60390  GTP binding Elongation factor Tu family protein [KO:K03231]\n",
      "AT1G07920  GTP binding Elongation factor Tu family protein [KO:K03231]\n",
      "AT1G07930  GTP binding Elongation factor Tu family protein [KO:K03231]\n",
      "AT1G35550  elongation factor Tu C-terminal domain-containing protein [KO:K03231]\n",
      "AT3G20430  phosphorylated adapter RNA export-like protein [KO:K14291]\n",
      "AT2G13540  ABH1; ARM repeat superfamily protein [KO:K12882]\n",
      "AT5G44200  CBP20; CAP-binding protein 20 [KO:K12883]\n",
      "AT2G03820  NMD3; nonsense-mediated mRNA decay NMD3 family protein [KO:K07562]\n",
      "AT1G51510  Y14; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12876]\n",
      "AT1G02140  MAGO; mago nashi family protein [KO:K12877]\n",
      "AT3G19760  EIF4A-III; eukaryotic initiation factor 4A-III [KO:K13025] [EC:3.6.4.13]\n",
      "AT1G51380  DEA(D/H)-box RNA helicase family protein [KO:K13025] [EC:3.6.4.13]\n",
      "AT4G39680  SAP domain-containing protein [KO:K12875]\n",
      "AT2G45640  SAP18; SIN3 associated polypeptide P18 [KO:K14324]\n",
      "AT1G16610  SR45; arginine/serine-rich 45 [KO:K14325]\n",
      "AT1G15200  protein-protein interaction regulator family protein [KO:K13114]\n",
      "AT5G02530  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT5G37720  ALY4; ALWAYS EARLY 4 [KO:K12881]\n",
      "AT1G66260  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT5G59950  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT5G47010  LBA1; RNA helicase [KO:K14326] [EC:3.6.4.-]\n",
      "AT2G39260  regulator of nonsense transcripts UPF2 [KO:K14327]\n",
      "AT1G33980  UPF3; Smg-4/UPF3 family protein [KO:K14328]\n",
      "AT5G11170  UAP56a; DEAD/DEAH box RNA helicase family protein [KO:K12812] [EC:3.6.4.13]\n",
      "AT5G11200  UAP56b; DEAD/DEAH box RNA helicase family protein [KO:K12812] [EC:3.6.4.13]\n",
      "AT2G29210  splicing factor PWI domain-containing protein [KO:K13171]\n",
      "AT1G11400  PYM; partner of Y14-MAGO [KO:K14294]\n",
      "AT5G09860  THO1; nuclear matrix protein-like protein [KO:K12878]\n",
      "AT1G24706  THO2; THO2 [KO:K12879]\n",
      "AT1G45233  THO5; THO complex, subunit 5 [KO:K13174]\n",
      "AT5G42920  THO5; THO complex, subunit 5 [KO:K13174]\n",
      "AT2G19430  DWA1; DWD (DDB1-binding WD40 protein) hypersensitive to ABA 1 [KO:K13175]\n",
      "AT5G16790  THO7; Tho complex subunit 7/Mft1p [KO:K13176]\n",
      "AT3G02950  THO7; Tho complex subunit 7/Mft1p [KO:K13176]\n",
      "AT5G56130  TEX1; Transducin/WD40 repeat-like superfamily protein [KO:K12880]\n",
      "REFERENCE   PMID:16777596\n",
      "AUTHORS   Tran EJ, Wente SR\n",
      "TITLE     Dynamic nuclear pore complexes: life on the edge.\n",
      "JOURNAL   Cell 125:1041-53 (2006)\n",
      "DOI:10.1016/j.cell.2006.05.027\n",
      "REFERENCE   PMID:15519698\n",
      "AUTHORS   Rodriguez MS, Dargemont C, Stutz F\n",
      "TITLE     Nuclear export of RNA.\n",
      "JOURNAL   Biol Cell 96:639-55 (2004)\n",
      "DOI:10.1016/j.biolcel.2004.04.014\n",
      "REFERENCE   PMID:18313922\n",
      "AUTHORS   Palancade B, Doye V\n",
      "TITLE     Sumoylating and desumoylating enzymes at nuclear pores: underpinning their unexpected duties?\n",
      "JOURNAL   Trends Cell Biol 18:174-83 (2008)\n",
      "DOI:10.1016/j.tcb.2008.02.001\n",
      "REFERENCE   PMID:15901496\n",
      "AUTHORS   Reed R, Cheng H\n",
      "TITLE     TREX, SR proteins and export of mRNA.\n",
      "JOURNAL   Curr Opin Cell Biol 17:269-73 (2005)\n",
      "DOI:10.1016/j.ceb.2005.04.011\n",
      "REFERENCE   PMID:16314458\n",
      "AUTHORS   Tange TO, Shibuya T, Jurica MS, Moore MJ\n",
      "TITLE     Biochemical analysis of the EJC reveals two new factors and a stable tetrameric protein core.\n",
      "JOURNAL   RNA 11:1869-83 (2005)\n",
      "DOI:10.1261/rna.2155905\n",
      "REFERENCE   PMID:28673770\n",
      "AUTHORS   Beck M, Schirmacher P, Singer S\n",
      "TITLE     Alterations of the nuclear transport system in hepatocellular carcinoma - New basis for therapeutic strategies.\n",
      "JOURNAL   J Hepatol 67:1051-1061 (2017)\n",
      "DOI:10.1016/j.jhep.2017.06.021\n",
      "REL_PATHWAY ath03015  mRNA surveillance pathway\n",
      "KO_PATHWAY  ko03013\n",
      "///\n",
      "ENTRY       ath03015                    Pathway\n",
      "NAME        mRNA surveillance pathway - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The mRNA surveillance pathway is a quality control mechanism that detects and degrades abnormal mRNAs. These pathways include nonsense-mediated mRNA decay (NMD), nonstop mRNA decay (NSD), and no-go decay (NGD). NMD is a mechanism that eliminates mRNAs containing premature translation-termination codons (PTCs). In vertebrates, PTCs trigger efficient NMD when located upstream of an exon junction complex (EJC). Upf3, together with Upf1 and Upf2, may signal the presence of the PTC to the 5'end of the transcript, resulting in decapping and rapid exonucleolytic digestion of the mRNA. In the NSD pathway, which targets mRNAs lacking termination codons, the ribosome is believed to translate through the 3' untranslated region and stall at the end of the poly(A) tail. NSD involves an eRF3-like protein, Ski7p, which is hypothesized to bind the empty A site of the ribosome and recruit the exosome to degrade the mRNA from the 3' end. NGD targets mRNAs with stalls in translation elongation for endonucleolytic cleavage in a process involving the Dom34 and Hbs1 proteins.\n",
      "CLASS       Genetic Information Processing; Translation\n",
      "PATHWAY_MAP ath03015  mRNA surveillance pathway\n",
      "DBLINKS     GO: 0071028\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G13540  ABH1; ARM repeat superfamily protein [KO:K12882]\n",
      "AT5G44200  CBP20; CAP-binding protein 20 [KO:K12883]\n",
      "AT1G33980  UPF3; Smg-4/UPF3 family protein [KO:K14328]\n",
      "AT1G51510  Y14; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12876]\n",
      "AT1G02140  MAGO; mago nashi family protein [KO:K12877]\n",
      "AT3G19760  EIF4A-III; eukaryotic initiation factor 4A-III [KO:K13025] [EC:3.6.4.13]\n",
      "AT1G51380  DEA(D/H)-box RNA helicase family protein [KO:K13025] [EC:3.6.4.13]\n",
      "AT2G45640  SAP18; SIN3 associated polypeptide P18 [KO:K14324]\n",
      "AT1G15200  protein-protein interaction regulator family protein [KO:K13114]\n",
      "AT4G39680  SAP domain-containing protein [KO:K12875]\n",
      "AT1G16610  SR45; arginine/serine-rich 45 [KO:K14325]\n",
      "AT5G02530  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT5G37720  ALY4; ALWAYS EARLY 4 [KO:K12881]\n",
      "AT1G66260  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT5G59950  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT3G53110  LOS4; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K18655] [EC:3.6.4.13]\n",
      "AT1G13120  GLE1; nucleoporin GLE1-like protein [KO:K18723]\n",
      "AT5G11170  UAP56a; DEAD/DEAH box RNA helicase family protein [KO:K12812] [EC:3.6.4.13]\n",
      "AT5G11200  UAP56b; DEAD/DEAH box RNA helicase family protein [KO:K12812] [EC:3.6.4.13]\n",
      "AT1G11400  PYM; partner of Y14-MAGO [KO:K14294]\n",
      "AT2G29210  splicing factor PWI domain-containing protein [KO:K13171]\n",
      "AT5G28210  mRNA capping enzyme family protein [KO:K13917] [EC:2.7.7.50 3.6.1.74]\n",
      "AT3G09100  mRNA capping enzyme family protein [KO:K13917] [EC:2.7.7.50 3.6.1.74]\n",
      "AT5G01290  mRNA capping enzyme family protein [KO:K13917] [EC:2.7.7.50 3.6.1.74]\n",
      "AT3G20650  mRNA capping enzyme family protein [KO:K00565] [EC:2.1.1.56]\n",
      "AT3G52210  S-adenosyl-L-methionine-dependent methyltransferases superfamily protein [KO:K00565] [EC:2.1.1.56]\n",
      "AT5G58470  TAF15b; TBP-associated factor 15B [KO:K13098]\n",
      "AT5G51120  PABN1; polyadenylate-binding protein 1 [KO:K14396]\n",
      "AT5G10350  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14396]\n",
      "AT5G65260  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14396]\n",
      "AT4G25550  Cleavage/polyadenylation specificity factor, 25kDa subunit [KO:K14397]\n",
      "AT4G29820  CFIM-25; homolog of CFIM-25 [KO:K14397]\n",
      "AT1G13190  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14398]\n",
      "AT5G55670  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14398]\n",
      "AT1G17980  PAPS1; poly(A) polymerase 1 [KO:K14376] [EC:2.7.7.19]\n",
      "AT4G32850  nPAP; nuclear poly(a) polymerase [KO:K14376] [EC:2.7.7.19]\n",
      "AT2G25850  PAPS2; poly(A) polymerase 2 [KO:K14376] [EC:2.7.7.19]\n",
      "AT3G06560  PAPS3; poly(A) polymerase 3 [KO:K14376] [EC:2.7.7.19]\n",
      "AT3G04680  CLPS3; CLP-similar protein 3 [KO:K14399] [EC:2.7.1.78]\n",
      "AT5G39930  CLPS5; CLP1-similar protein 5 [KO:K14399] [EC:2.7.1.78]\n",
      "AT2G36480  pre-mRNA cleavage complex 2 Pcf11-like protein [KO:K14400]\n",
      "AT4G04885  PCFS4; PCF11P-similar protein 4 [KO:K14400]\n",
      "AT1G66500  Pre-mRNA cleavage complex II [KO:K14400]\n",
      "AT5G43620  Pre-mRNA cleavage complex II [KO:K14400]\n",
      "AT5G51660  CPSF160; cleavage and polyadenylation specificity factor 160 [KO:K14401]\n",
      "AT5G23880  CPSF100; cleavage and polyadenylation specificity factor 100 [KO:K14402]\n",
      "AT1G61010  CPSF73-I; cleavage and polyadenylation specificity factor 73-I [KO:K14403] [EC:3.1.27.-]\n",
      "AT1G30460  CPSF30; cleavage and polyadenylation specificity factor 30 [KO:K14404]\n",
      "AT5G18440  NUFIP; GATA zinc finger protein [KO:K14404]\n",
      "AT5G13480  FY; Transducin/WD40 repeat-like superfamily protein [KO:K15542]\n",
      "AT5G14530  Transducin/WD40 repeat-like superfamily protein [KO:K14962]\n",
      "AT5G66240  Transducin/WD40 repeat-like superfamily protein [KO:K14962]\n",
      "AT2G39840  TOPP4; type one serine/threonine protein phosphatase 4 [KO:K06269] [EC:3.1.3.16]\n",
      "AT5G59160  TOPP2; type one serine/threonine protein phosphatase 2 [KO:K06269] [EC:3.1.3.16]\n",
      "AT3G46820  TOPP5; type one serine/threonine protein phosphatase 5 [KO:K06269] [EC:3.1.3.16]\n",
      "AT2G29400  TOPP1; type one protein phosphatase 1 [KO:K06269] [EC:3.1.3.16]\n",
      "AT4G11240  TOPP7; Calcineurin-like metallo-phosphoesterase superfamily protein [KO:K06269] [EC:3.1.3.16]\n",
      "AT1G64040  TOPP3; type one serine/threonine protein phosphatase 3 [KO:K06269] [EC:3.1.3.16]\n",
      "AT3G05580  TOPP9; Calcineurin-like metallo-phosphoesterase superfamily protein [KO:K06269] [EC:3.1.3.16]\n",
      "AT5G27840  TOPP8; Calcineurin-like metallo-phosphoesterase superfamily protein [KO:K06269] [EC:3.1.3.16]\n",
      "AT5G43380  TOPP6; type one serine/threonine protein phosphatase 6 [KO:K06269] [EC:3.1.3.16]\n",
      "AT1G73820  Ssu72-like family protein [KO:K15544] [EC:3.1.3.16]\n",
      "AT5G60940  Transducin/WD40 repeat-like superfamily protein [KO:K14406]\n",
      "AT1G71800  CSTF64; cleavage stimulating factor 64 [KO:K14407]\n",
      "AT1G17760  CSTF77; Tetratricopeptide repeat (TPR)-like superfamily protein [KO:K14408]\n",
      "AT1G27595  symplekin [KO:K06100]\n",
      "AT5G01400  ESP4; HEAT repeat-containing protein [KO:K06100]\n",
      "AT5G47620  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14411]\n",
      "AT2G33410  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14411]\n",
      "AT4G14300  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14411]\n",
      "AT3G07810  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14411]\n",
      "AT4G26650  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14411]\n",
      "AT5G55550  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14411]\n",
      "AT1G58470  RBP1; RNA-binding protein 1 [KO:K14411]\n",
      "AT4G36960  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K14411]\n",
      "AT1G49760  PAB8; poly(A) binding protein 8 [KO:K13126]\n",
      "AT4G34110  PAB2; poly(A) binding protein 2 [KO:K13126]\n",
      "AT3G52150  PSRP2; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K13126]\n",
      "AT1G22760  PAB3; poly(A) binding protein 3 [KO:K13126]\n",
      "AT1G71770  PAB5; poly(A)-binding protein 5 [KO:K13126]\n",
      "AT2G23350  PAB4; poly(A) binding protein 4 [KO:K13126]\n",
      "AT1G34140  PAB1; poly(A) binding protein 1 [KO:K13126]\n",
      "AT2G36660  PAB7; poly(A) binding protein 7 [KO:K13126]\n",
      "AT3G16380  PAB6; poly(A) binding protein 6 [KO:K13126]\n",
      "AT5G47880  ERF1-1; eukaryotic release factor 1-1 [KO:K03265]\n",
      "AT1G12920  ERF1-2; eukaryotic release factor 1-2 [KO:K03265]\n",
      "AT3G26618  ERF1-3; eukaryotic release factor 1-3 [KO:K03265]\n",
      "AT1G18070  Translation elongation factor EF1A/initiation factor IF2gamma family protein [KO:K03267]\n",
      "AT5G47010  LBA1; RNA helicase [KO:K14326] [EC:3.6.4.-]\n",
      "AT2G39260  regulator of nonsense transcripts UPF2 [KO:K14327]\n",
      "AT5G19400  SMG7; Telomerase activating protein Est1 [KO:K14409]\n",
      "AT1G10430  PP2A-2; protein phosphatase 2A-2 [KO:K04382] [EC:3.1.3.16]\n",
      "AT1G59830  PP2A-1; protein phosphatase 2A-2 [KO:K04382] [EC:3.1.3.16]\n",
      "AT1G69960  PP2A; serine/threonine protein phosphatase 2A [KO:K04382] [EC:3.1.3.16]\n",
      "AT2G42500  PP2A-3; protein phosphatase 2A-3 [KO:K04382] [EC:3.1.3.16]\n",
      "AT3G58500  PP2A-4; protein phosphatase 2A-4 [KO:K04382] [EC:3.1.3.16]\n",
      "AT1G13320  PP2AA3; protein phosphatase 2A subunit A3 [KO:K03456]\n",
      "AT3G25800  PP2AA2; protein phosphatase 2A subunit A2 [KO:K03456]\n",
      "AT1G25490  RCN1; ARM repeat superfamily protein [KO:K03456]\n",
      "AT1G51690  B_ALPHA; protein phosphatase 2A 55 kDa regulatory subunit B alpha isoform [KO:K04354]\n",
      "AT1G17720  ATB_BETA; Protein phosphatase 2A, regulatory subunit PR55 [KO:K04354]\n",
      "AT1G03960  Calcium-binding EF hand family protein [KO:K11583]\n",
      "AT1G54450  Calcium-binding EF-hand family protein [KO:K11583]\n",
      "AT5G28850  Calcium-binding EF-hand family protein [KO:K11583]\n",
      "AT5G28900  Calcium-binding EF-hand family protein [KO:K11583]\n",
      "AT5G18580  FASS; tonneau 2 (TON2) [KO:K11583]\n",
      "AT5G44090  Calcium-binding EF-hand family protein [KO:K11583]\n",
      "AT3G09880  ATB'_BETA; Protein phosphatase 2A regulatory B subunit family protein [KO:K11584]\n",
      "AT1G13460  Protein phosphatase 2A regulatory B subunit family protein [KO:K11584]\n",
      "AT5G03470  ATB'_ALPHA; Protein phosphatase 2A regulatory B subunit family protein [KO:K11584]\n",
      "AT4G15415  ATB'_GAMMA; Protein phosphatase 2A regulatory B subunit family protein [KO:K11584]\n",
      "AT3G26020  Protein phosphatase 2A regulatory B subunit family protein [KO:K11584]\n",
      "AT3G21650  Protein phosphatase 2A regulatory B subunit family protein [KO:K11584]\n",
      "AT5G25510  Protein phosphatase 2A regulatory B subunit family protein [KO:K11584]\n",
      "AT3G26030  ATB'_DELTA; serine/threonine protein phosphatase 2A 55 kDa regulatory subunit B prime delta [KO:K11584]\n",
      "AT3G54930  Protein phosphatase 2A regulatory B subunit family protein [KO:K11584]\n",
      "AT5G10630  Translation elongation factor EF1A/initiation factor IF2gamma family protein [KO:K14416]\n",
      "AT4G27650  PEL1; Eukaryotic release factor 1 (eRF1) family protein [KO:K06965]\n",
      "AT3G58390  Eukaryotic release factor 1 (eRF1) family protein [KO:K06965]\n",
      "REFERENCE   PMID:15040442\n",
      "AUTHORS   Maquat LE\n",
      "TITLE     Nonsense-mediated mRNA decay: splicing, translation and mRNP dynamics.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 5:89-99 (2004)\n",
      "DOI:10.1038/nrm1310\n",
      "REFERENCE   PMID:16723977\n",
      "AUTHORS   Amrani N, Sachs MS, Jacobson A\n",
      "TITLE     Early nonsense: mRNA decay solves a translational problem.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 7:415-25 (2006)\n",
      "DOI:10.1038/nrm1942\n",
      "REFERENCE   PMID:17183357\n",
      "AUTHORS   Eulalio A, Behm-Ansmant I, Izaurralde E\n",
      "TITLE     P bodies: at the crossroads of post-transcriptional pathways.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 8:9-22 (2007)\n",
      "DOI:10.1038/nrm2080\n",
      "REFERENCE   PMID:18158581\n",
      "AUTHORS   Mandel CR, Bai Y, Tong L\n",
      "TITLE     Protein factors in pre-mRNA 3'-end processing.\n",
      "JOURNAL   Cell Mol Life Sci 65:1099-122 (2008)\n",
      "DOI:10.1007/s00018-007-7474-3\n",
      "REFERENCE   PMID:16715045\n",
      "AUTHORS   Clement SL, Lykke-Andersen J\n",
      "TITLE     No mercy for messages that mess with the ribosome.\n",
      "JOURNAL   Nat Struct Mol Biol 13:299-301 (2006)\n",
      "DOI:10.1038/nsmb0406-299\n",
      "REL_PATHWAY ath03013  Nucleocytoplasmic transport\n",
      "ath03018  RNA degradation\n",
      "ath03040  Spliceosome\n",
      "KO_PATHWAY  ko03015\n",
      "///\n",
      "ENTRY       ath03018                    Pathway\n",
      "NAME        RNA degradation - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The correct processing, quality control and turnover of cellular RNA molecules are critical to many aspects in the expression of genetic information. In eukaryotes, two major pathways of mRNA decay exist and both pathways are initiated by poly(A) shortening of the mRNA. In the 5' to 3' pathway, this is followed by decapping which then permits the 5' to 3' exonucleolytic degradation of transcripts. In the 3' to 5' pathway, the exosome, a large multisubunit complex, plays a key role. The exosome exists in archaeal cells, too. In bacteria, endoribonuclease E, a key enzyme involved in RNA decay and processing, organizes a protein complex called degradosome. RNase E or R interacts with the phosphate-dependent exoribonuclease polynucleotide phosphorylase, DEAD-box helicases, and additional factors in the RNA-degrading complex.\n",
      "CLASS       Genetic Information Processing; Folding, sorting and degradation\n",
      "PATHWAY_MAP ath03018  RNA degradation\n",
      "DBLINKS     GO: 0006401\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G38890  Nucleic acid-binding, OB-fold-like protein [KO:K07573]\n",
      "AT1G03360  RRP4; ribosomal RNA processing 4 [KO:K03679]\n",
      "AT2G25355  PNAS-3-like protein [KO:K03681]\n",
      "AT4G32175  PNAS-3-like protein [KO:K03681]\n",
      "AT1G60080  3'-5'-exoribonuclease family protein [KO:K12586]\n",
      "AT4G27490  RRP41L; 3'-5'-exoribonuclease family protein [KO:K12587]\n",
      "AT3G07750  3'-5'-exoribonuclease family protein [KO:K12589]\n",
      "AT3G61620  RRP41; 3'-5'-exoribonuclease family protein [KO:K11600]\n",
      "AT3G46210  Ribosomal protein S5 domain 2-like superfamily protein [KO:K12590]\n",
      "AT3G60500  CER7; 3'-5'-exoribonuclease family protein [KO:K03678]\n",
      "AT3G12990  RRP45a; ribonuclease PH45A [KO:K03678]\n",
      "AT2G17510  EMB2763; ribonuclease II family protein [KO:K12585] [EC:3.1.13.-]\n",
      "AT5G35910  Polynucleotidyl transferase, ribonuclease H fold protein with HRDC domain-containing protein [KO:K12591] [EC:3.1.13.-]\n",
      "AT1G54440  Polynucleotidyl transferase, ribonuclease H fold protein with HRDC domain-containing protein [KO:K12591] [EC:3.1.13.-]\n",
      "AT5G25080  Sas10/Utp3/C1D family [KO:K12592]\n",
      "AT5G59460  scarecrow-like transcription factor 11 (SCL11) [KO:K12593]\n",
      "AT5G53770  Nucleotidyltransferase family protein [KO:K03514] [EC:2.7.7.19]\n",
      "AT1G59760  MTR4; RNA helicase, ATP-dependent, SK12/DOB1 protein [KO:K12598] [EC:3.6.4.13]\n",
      "AT2G06990  HEN2; RNA helicase, ATP-dependent, SK12/DOB1 protein [KO:K12598] [EC:3.6.4.13]\n",
      "AT3G46960  RNA helicase, ATP-dependent, SK12/DOB1 protein [KO:K12599] [EC:3.6.4.-]\n",
      "AT1G76630  Tetratricopeptide repeat (TPR)-like superfamily protein [KO:K12600]\n",
      "AT4G29830  VIP3; Transducin/WD40 repeat-like superfamily protein [KO:K12602]\n",
      "AT3G58560  DNAse I-like superfamily protein [KO:K12603] [EC:3.1.13.4]\n",
      "AT3G58580  DNAse I-like superfamily protein [KO:K12603] [EC:3.1.13.4]\n",
      "AT1G31500  DNAse I-like superfamily protein [KO:K12603] [EC:3.1.13.4]\n",
      "AT1G31530  DNAse I-like superfamily protein [KO:K12603] [EC:3.1.13.4]\n",
      "AT1G02080  transcription regulator [KO:K12604]\n",
      "AT1G07705  NOT2 / NOT3 / NOT5 family [KO:K12605]\n",
      "AT5G59710  VIP2; VIRE2 interacting protein 2 [KO:K12605]\n",
      "AT5G18230  transcription regulator NOT2/NOT3/NOT5 family protein [KO:K12580]\n",
      "AT3G45630  RNA binding (RRM/RBD/RNP motifs) family protein [KO:K10643] [EC:2.3.2.27]\n",
      "AT5G60170  RNA binding (RRM/RBD/RNP motifs) family protein [KO:K10643] [EC:2.3.2.27]\n",
      "AT2G28540  RNA binding (RRM/RBD/RNP motifs) family protein [KO:K10643] [EC:2.3.2.27]\n",
      "AT1G80780  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT5G10960  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT1G06450  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT1G27890  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT3G44240  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT1G27820  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT1G61470  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT2G32070  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT1G15920  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT3G44260  CAF1a; Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT5G22250  CAF1b; Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K12581]\n",
      "AT3G20800  Cell differentiation, Rcd1-like protein [KO:K12606]\n",
      "AT5G12980  Cell differentiation, Rcd1-like protein [KO:K12606]\n",
      "AT2G32550  Cell differentiation, Rcd1-like protein [KO:K12606]\n",
      "AT5G35430  Tetratricopeptide repeat (TPR)-like superfamily protein [KO:K12607]\n",
      "AT1G03905  ABCI19; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12608]\n",
      "AT5G02270  ABCI20; non-intrinsic ABC protein 9 [KO:K12608]\n",
      "AT5G44110  ABCI21; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12608]\n",
      "AT2G35920  RNA helicase family protein [KO:K14442] [EC:3.6.4.13]\n",
      "AT1G48650  DEA(D/H)-box RNA helicase family protein [KO:K14442] [EC:3.6.4.13]\n",
      "AT2G01130  DEA(D/H)-box RNA helicase family protein [KO:K14442] [EC:3.6.4.13]\n",
      "AT5G04895  ABO6; DEA(D/H)-box RNA helicase family protein [KO:K14442] [EC:3.6.4.13]\n",
      "AT1G06670  NIH; nuclear DEIH-boxhelicase [KO:K14442] [EC:3.6.4.13]\n",
      "AT2G30800  HVT1; helicase in vascular tissue and tapetum [KO:K14442] [EC:3.6.4.13]\n",
      "AT1G55870  AHG2; Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K01148] [EC:3.1.13.4]\n",
      "AT3G25430  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K01148] [EC:3.1.13.4]\n",
      "AT3G25440  LOH1; RNA-binding CRS1 / YhbY (CRM) domain protein [KO:K01148] [EC:3.1.13.4]\n",
      "AT1G49760  PAB8; poly(A) binding protein 8 [KO:K13126]\n",
      "AT4G34110  PAB2; poly(A) binding protein 2 [KO:K13126]\n",
      "AT3G52150  PSRP2; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K13126]\n",
      "AT1G22760  PAB3; poly(A) binding protein 3 [KO:K13126]\n",
      "AT1G71770  PAB5; poly(A)-binding protein 5 [KO:K13126]\n",
      "AT2G23350  PAB4; poly(A) binding protein 4 [KO:K13126]\n",
      "AT1G34140  PAB1; poly(A) binding protein 1 [KO:K13126]\n",
      "AT2G36660  PAB7; poly(A) binding protein 7 [KO:K13126]\n",
      "AT3G16380  PAB6; poly(A) binding protein 6 [KO:K13126]\n",
      "AT1G08370  DCP1; decapping 1 [KO:K12611] [EC:3.-.-.-]\n",
      "AT5G13570  DCP2; decapping 2 [KO:K12613] [EC:3.6.1.62]\n",
      "AT3G61240  DEA(D/H)-box RNA helicase family protein [KO:K12614] [EC:3.6.4.13]\n",
      "AT2G45810  DEA(D/H)-box RNA helicase family protein [KO:K12614] [EC:3.6.4.13]\n",
      "AT4G00660  RH8; RNAhelicase-like 8 [KO:K12614] [EC:3.6.4.13]\n",
      "AT3G13300  VCS; Transducin/WD40 repeat-like superfamily protein [KO:K12616]\n",
      "AT3G13290  VCR; varicose-like protein [KO:K12616]\n",
      "AT1G79090  topoisomerase II-associated protein [KO:K12617]\n",
      "AT4G14990  Topoisomerase II-associated protein PAT1 [KO:K12617]\n",
      "AT3G22270  Topoisomerase II-associated protein PAT1 [KO:K12617]\n",
      "AT5G42540  XRN2; exoribonuclease 2 [KO:K12619] [EC:3.1.13.-]\n",
      "AT1G75660  XRN3; 5'-3' exoribonuclease 3 [KO:K12619] [EC:3.1.13.-]\n",
      "AT3G14080  LSM1B; Small nuclear ribonucleoprotein family protein [KO:K12620]\n",
      "AT1G19120  LSM1A; Small nuclear ribonucleoprotein family protein [KO:K12620]\n",
      "AT1G03330  LSM2; Small nuclear ribonucleoprotein family protein [KO:K12621]\n",
      "AT1G76860  LSM3B; Small nuclear ribonucleoprotein family protein [KO:K12622]\n",
      "AT1G21190  LSM3A; Small nuclear ribonucleoprotein family protein [KO:K12622]\n",
      "AT5G27720  emb1644; Small nuclear ribonucleoprotein family protein [KO:K12623]\n",
      "AT5G48870  SAD1; Small nuclear ribonucleoprotein family protein [KO:K12624]\n",
      "AT3G59810  LSM6A; Small nuclear ribonucleoprotein family protein [KO:K12625]\n",
      "AT2G43810  LSM6B; Small nuclear ribonucleoprotein family protein [KO:K12625]\n",
      "AT2G03870  EMB2816; Small nuclear ribonucleoprotein family protein [KO:K12626]\n",
      "AT1G65700  LSM8; Small nuclear ribonucleoprotein family protein [KO:K12627]\n",
      "AT1G74030  ENO1; enolase 1 [KO:K01689] [EC:4.2.1.11]\n",
      "AT2G36530  LOS2; Enolase [KO:K01689] [EC:4.2.1.11]\n",
      "AT2G29560  ENOC; cytosolic enolase [KO:K01689] [EC:4.2.1.11]\n",
      "AT5G14580  polyribonucleotide nucleotidyltransferase [KO:K00962] [EC:2.7.7.8]\n",
      "AT3G03710  RIF10; polyribonucleotide nucleotidyltransferase [KO:K00962] [EC:2.7.7.8]\n",
      "AT4G32840  PFK6; phosphofructokinase 6 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G56630  PFK7; phosphofructokinase 7 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G61580  PFK4; phosphofructokinase 4 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G29220  PFK1; phosphofructokinase 1 [KO:K00850] [EC:2.7.1.11]\n",
      "AT2G22480  PFK5; phosphofructokinase 5 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G26270  PFK3; phosphofructokinase 3 [KO:K00850] [EC:2.7.1.11]\n",
      "AT5G47810  PFK2; phosphofructokinase 2 [KO:K00850] [EC:2.7.1.11]\n",
      "AT4G37910  mtHsc70-1; mitochondrial heat shock protein 70-1 [KO:K04043]\n",
      "AT5G09590  MTHSC70-2; mitochondrial HSO70 2 [KO:K04043]\n",
      "AT2G33210  HSP60-2; heat shock protein 60-2 [KO:K04077]\n",
      "AT3G23990  HSP60; heat shock protein 60 [KO:K04077]\n",
      "AT3G13860  HSP60-3A; heat shock protein 60-3A [KO:K04077]\n",
      "AT3G13470  Cpn60beta2; TCP-1/cpn60 chaperonin family protein [KO:K04077]\n",
      "AT1G26230  Cpn60beta4; TCP-1/cpn60 chaperonin family protein [KO:K04077]\n",
      "AT1G55490  CPN60B; chaperonin 60 beta [KO:K04077]\n",
      "AT5G56500  Cpn60beta3; TCP-1/cpn60 chaperonin family protein [KO:K04077]\n",
      "REFERENCE   PMID:19239894\n",
      "AUTHORS   Houseley J, Tollervey D\n",
      "TITLE     The many pathways of RNA degradation.\n",
      "JOURNAL   Cell 136:763-76 (2009)\n",
      "DOI:10.1016/j.cell.2009.01.019\n",
      "REFERENCE   PMID:16968219\n",
      "AUTHORS   Buttner K, Wenig K, Hopfner KP\n",
      "TITLE     The exosome: a macromolecular cage for controlled RNA degradation.\n",
      "JOURNAL   Mol Microbiol 61:1372-9 (2006)\n",
      "DOI:10.1111/j.1365-2958.2006.05331.x\n",
      "REFERENCE   PMID:16939780\n",
      "AUTHORS   Schilders G, van Dijk E, Raijmakers R, Pruijn GJ\n",
      "TITLE     Cell and molecular biology of the exosome: how to make or break an RNA.\n",
      "JOURNAL   Int Rev Cytol 251:159-208 (2006)\n",
      "DOI:10.1016/S0074-7696(06)51005-8\n",
      "REFERENCE   PMID:17471261\n",
      "AUTHORS   Hartung S, Hopfner KP\n",
      "TITLE     The exosome, plugged.\n",
      "JOURNAL   EMBO Rep 8:456-7 (2007)\n",
      "DOI:10.1038/sj.embor.7400961\n",
      "REFERENCE   PMID:17447862\n",
      "AUTHORS   Carpousis AJ\n",
      "TITLE     The RNA degradosome of Escherichia coli: an mRNA-degrading machine assembled on RNase E.\n",
      "JOURNAL   Annu Rev Microbiol 61:71-87 (2007)\n",
      "DOI:10.1146/annurev.micro.61.080706.093440\n",
      "REFERENCE   PMID:16766188\n",
      "AUTHORS   Marcaida MJ, DePristo MA, Chandran V, Carpousis AJ, Luisi BF\n",
      "TITLE     The RNA degradosome: life in the fast lane of adaptive molecular evolution.\n",
      "JOURNAL   Trends Biochem Sci 31:359-65 (2006)\n",
      "DOI:10.1016/j.tibs.2006.05.005\n",
      "REFERENCE   PMID:17560162\n",
      "AUTHORS   Condon C\n",
      "TITLE     Maturation and degradation of RNA in bacteria.\n",
      "JOURNAL   Curr Opin Microbiol 10:271-8 (2007)\n",
      "DOI:10.1016/j.mib.2007.05.008\n",
      "REL_PATHWAY ath03015  mRNA surveillance pathway\n",
      "KO_PATHWAY  ko03018\n",
      "///\n",
      "ENTRY       ath03020                    Pathway\n",
      "NAME        RNA polymerase - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Genetic Information Processing; Transcription\n",
      "PATHWAY_MAP ath03020  RNA polymerase\n",
      "DBLINKS     GO: 0000428\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        ArthCp014  rpoB; RNA polymerase beta subunit [KO:K03043] [EC:2.7.7.6]\n",
      "ArthCp013  rpoC1; RNA polymerase beta' subunit [KO:K03046] [EC:2.7.7.6]\n",
      "ArthCp012  rpoC2; RNA polymerase beta'' subunit [KO:K03046] [EC:2.7.7.6]\n",
      "ArthCp055  rpoA; RNA polymerase alpha subunit [KO:K03040] [EC:2.7.7.6]\n",
      "AT4G21710  NRPB2; DNA-directed RNA polymerase family protein [KO:K03010] [EC:2.7.7.6]\n",
      "AT4G35800  NRPB1; RNA polymerase II large subunit [KO:K03006] [EC:2.7.7.6]\n",
      "AT2G15430  NRPB3; DNA-directed RNA polymerase family protein [KO:K03011]\n",
      "AT2G15400  NRPE3B; DNA-directed RNA polymerase family protein [KO:K03011]\n",
      "AT3G52090  NRPB11; DNA-directed RNA polymerase, RBP11-like protein [KO:K03008]\n",
      "AT5G09920  NRPB4; RNA polymerase II, Rpb4, core protein [KO:K03012]\n",
      "AT5G59180  NRPB7; DNA-directed RNA polymerase II [KO:K03015]\n",
      "AT3G16980  NRPB9A; RNA polymerases M/15 Kd subunit [KO:K03017]\n",
      "AT4G16265  NRPB9B; RNA polymerases M/15 Kd subunit [KO:K03017]\n",
      "AT5G45140  NRPC2; nuclear RNA polymerase C2 [KO:K03021] [EC:2.7.7.6]\n",
      "AT5G60040  NRPC1; nuclear RNA polymerase C1 [KO:K03018] [EC:2.7.7.6]\n",
      "AT2G29540  RPC14; RNApolymerase 14 kDa subunit [KO:K03020]\n",
      "AT1G60620  RPAC43; RNA polymerase I subunit 43 [KO:K03027]\n",
      "AT1G60850  ATRPAC42; DNA-directed RNA polymerase family protein [KO:K03027]\n",
      "AT3G49000  RNA polymerase III subunit RPC82 family protein [KO:K03023]\n",
      "AT4G25180  RNA polymerase III RPC4 [KO:K03026]\n",
      "AT5G09380  RNA polymerase III RPC4 [KO:K03026]\n",
      "AT5G49530  SIN-like family protein [KO:K14721]\n",
      "AT5G23710  DNA binding / DNA-directed RNA polymerase [KO:K03025]\n",
      "AT4G01590  DNA-directed RNA polymerase III subunit [KO:K03024]\n",
      "AT1G06790  RNA polymerase Rpb7 N-terminal domain-containing protein [KO:K03022]\n",
      "AT3G28956  RNA polymerase II, Rpb4, core protein [KO:K25304]\n",
      "AT1G01210  DNA-directed RNA polymerase, subunit M, archaeal [KO:K03019]\n",
      "AT4G07950  DNA-directed RNA polymerase, subunit M, archaeal [KO:K03019]\n",
      "AT1G29940  NRPA2; nuclear RNA polymerase A2 [KO:K03002] [EC:2.7.7.6]\n",
      "AT3G57660  NRPA1; nuclear RNA polymerase A1 [KO:K02999] [EC:2.7.7.6]\n",
      "AT3G25940  TFIIB zinc-binding protein [KO:K03000]\n",
      "AT3G13940  DNA binding / DNA-directed RNA polymerase [KO:K03005]\n",
      "AT3G22320  NRPB5; Eukaryotic rpb5 RNA polymerase subunit family protein [KO:K03013]\n",
      "AT2G41340  RPB5D; RNA polymerase II fifth largest subunit, D [KO:K03013]\n",
      "AT3G54490  RPB5E; RNA polymerase II fifth largest subunit, E [KO:K03013]\n",
      "AT3G57080  NRPE5; Eukaryotic rpb5 RNA polymerase subunit family protein [KO:K03013]\n",
      "AT5G57980  RPB5C; RNA polymerase II fifth largest subunit, C [KO:K03013]\n",
      "AT5G51940  NRPB6A; RNA polymerase Rpb6 [KO:K03014]\n",
      "AT2G04630  NRPB6B; RNA polymerase Rpb6 [KO:K03014]\n",
      "AT3G59600  NRPB8B; RNA polymerase Rpb8 [KO:K03016]\n",
      "AT1G54250  NRPB8A; RNA polymerase Rpb8 [KO:K03016]\n",
      "AT5G41010  NRPB12; DNA directed RNA polymerase, 7 kDa subunit [KO:K03009]\n",
      "AT1G53690  DNA directed RNA polymerase, 7 kDa subunit [KO:K03009]\n",
      "AT1G61700  RNA polymerases N / 8 kDa subunit [KO:K03007]\n",
      "AT1G11475  NRPB10; RNA polymerases N / 8 kDa subunit [KO:K03007]\n",
      "AT1G63020  NRPD1A; nuclear RNA polymerase D1A [KO:K16250] [EC:2.7.7.6]\n",
      "AT2G40030  NRPD1B; nuclear RNA polymerase D1B [KO:K16251] [EC:2.7.7.6]\n",
      "AT3G23780  NRPD2A; nuclear RNA polymerase D2A [KO:K16252] [EC:2.7.7.6]\n",
      "AT3G18090  NRPD2B; nuclear RNA polymerase D2B [KO:K16252] [EC:2.7.7.6]\n",
      "AT3G22900  NRPD7; RNA polymerase Rpb7-like, N-terminal domain-containing protein [KO:K16253]\n",
      "AT4G14520  DNA-directed RNA polymerase II-like protein [KO:K16253]\n",
      "AT4G14660  NRPE7; RNA polymerase Rpb7-like, N-terminal domain-containing protein [KO:K16253]\n",
      "REFERENCE   PMID:10499798\n",
      "AUTHORS   Zhang G, Campbell EA, Minakhin L, Richter C, Severinov K, Darst SA.\n",
      "TITLE     Crystal structure of Thermus aquaticus core RNA polymerase at 3.3 A resolution.\n",
      "JOURNAL   Cell 98:811-824 (1999)\n",
      "DOI:10.1016/S0092-8674(00)81515-9\n",
      "REFERENCE   PMID:10784442\n",
      "AUTHORS   Cramer P, Bushnell DA, Fu J, Gnatt AL, Maier-Davis B, Thompson NE, Burgess RR, Edwards AM, David PR, Kornberg RD.\n",
      "TITLE     Architecture of RNA polymerase II and implications for the transcription mechanism.\n",
      "JOURNAL   Science 288:640-649 (2000)\n",
      "DOI:10.1126/science.288.5466.640\n",
      "REFERENCE   PMID:19880312\n",
      "AUTHORS   Hirata A, Murakami KS\n",
      "TITLE     Archaeal RNA polymerase.\n",
      "JOURNAL   Curr Opin Struct Biol 19:724-31 (2009)\n",
      "DOI:10.1016/j.sbi.2009.10.006\n",
      "REFERENCE   PMID:18573085\n",
      "AUTHORS   Cramer P, Armache KJ, Baumli S, Benkert S, Brueckner F, Buchen C, Damsma GE, Dengl S, Geiger SR, Jasiak AJ, Jawhari A, Jennebach S, Kamenski T, Kettenberger H, Kuhn CD, Lehmann E, Leike K, Sydow JF, Vannini A\n",
      "TITLE     Structure of eukaryotic RNA polymerases.\n",
      "JOURNAL   Annu Rev Biophys 37:337-52 (2008)\n",
      "DOI:10.1146/annurev.biophys.37.032807.130008\n",
      "KO_PATHWAY  ko03020\n",
      "///\n",
      "ENTRY       ath03022                    Pathway\n",
      "NAME        Basal transcription factors - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Genetic Information Processing; Transcription\n",
      "PATHWAY_MAP ath03022  Basal transcription factors\n",
      "DBLINKS     GO: 0005668 0005669 0005672 0005673 0005674 0005675\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G07470  Transcription factor IIA, alpha/beta subunit [KO:K03122]\n",
      "AT5G59230  transcription factor-like protein [KO:K03122]\n",
      "AT1G07480  Transcription factor IIA, alpha/beta subunit [KO:K03122]\n",
      "AT4G24440  transcription initiation factor IIA gamma chain / TFIIA-gamma (TFIIA-S) [KO:K03123]\n",
      "AT4G10680  transcription factor IIB (TFIIB) family protein [KO:K03124]\n",
      "AT4G36650  PBRP; plant-specific TFIIB-related protein [KO:K03124]\n",
      "AT2G41630  TFIIB; transcription factor IIB [KO:K03124]\n",
      "AT3G10330  Cyclin-like family protein [KO:K03124]\n",
      "AT3G29380  pBRP2; Cyclin-like family protein [KO:K03124]\n",
      "AT3G57370  Cyclin family protein [KO:K03124]\n",
      "AT1G55520  TBP2; TATA binding protein 2 [KO:K03120]\n",
      "AT3G13445  TBP1; TATA binding protein 1 [KO:K03120]\n",
      "AT1G32750  HAF01; HAC13 protein (HAC13) [KO:K03125] [EC:2.3.1.48 2.7.11.1]\n",
      "AT3G19040  HAF2; histone acetyltransferase of the TAFII250 family 2 [KO:K03125] [EC:2.3.1.48 2.7.11.1]\n",
      "AT1G73960  TAF2; TBP-associated factor 2 [KO:K03128]\n",
      "AT1G55300  TAF7; TBP-associated factor 7 [KO:K03132]\n",
      "AT1G31240  Bromodomain transcription factor [KO:K14649]\n",
      "AT5G15570  Bromodomain transcription factor [KO:K14649]\n",
      "AT3G02160  Bromodomain transcription factor [KO:K14649]\n",
      "AT4G34340  TAF8; TBP-associated factor 8 [KO:K14649]\n",
      "AT4G31720  TAFII15; TBP-associated factor II 15 [KO:K03134]\n",
      "AT5G25150  TAF5; TBP-associated factor 5 [KO:K03130]\n",
      "AT5G43130  TAF4; TBP-associated factor 4 [KO:K03129]\n",
      "AT1G27720  TAF4B; TBP-associated factor 4B [KO:K03129]\n",
      "AT3G10070  TAF12; TBP-associated factor 12 [KO:K03126]\n",
      "AT1G17440  EER4; Transcription initiation factor TFIID subunit A [KO:K03126]\n",
      "AT1G04950  TAFII59; TATA BOX ASSOCIATED FACTOR II 59 [KO:K03131]\n",
      "AT1G54360  TAF6B; TBP-ASSOCIATED FACTOR 6B [KO:K03131]\n",
      "AT1G54140  TAFII21; TATA binding protein associated factor 21kDa subunit [KO:K03133]\n",
      "AT4G20280  TAF11; TBP-associated factor 11 [KO:K03135]\n",
      "AT1G20000  TAF11b; TBP-associated factor 11B [KO:K03135]\n",
      "AT1G02680  TAF13; TBP-associated factor 13 [KO:K03127]\n",
      "AT1G03280  Transcription factor TFIIE, alpha subunit [KO:K03136]\n",
      "AT4G20810  transcription initiation factor IIE (TFIIE) alpha subunit family protein / general transcription factor TFIIE family protein [KO:K03136]\n",
      "AT4G20340  Transcription factor TFIIE, alpha subunit [KO:K03136]\n",
      "AT4G20330  Transcription initiation factor TFIIE, beta subunit [KO:K03137]\n",
      "AT4G21010  Transcription initiation factor TFIIE, beta subunit [KO:K03137]\n",
      "AT4G12610  RAP74; transcription initiation factor IIF subunit alpha RAP74 [KO:K03138]\n",
      "AT1G75510  Transcription initiation factor IIF, beta subunit [KO:K03139] [EC:3.6.4.12]\n",
      "AT3G52270  Transcription initiation factor IIF, beta subunit [KO:K03139] [EC:3.6.4.12]\n",
      "AT3G61420  BSD domain (BTF2-like transcription factors, Synapse-associated proteins and DOS2-like proteins) [KO:K03141]\n",
      "AT1G55750  BSD domain (BTF2-like transcription factors, Synapse-associated proteins and DOS2-like proteins) [KO:K03141]\n",
      "AT1G05055  GTF2H2; general transcription factor II H2 [KO:K03142]\n",
      "AT1G18340  basal transcription factor complex subunit-like protein [KO:K03143]\n",
      "AT4G17020  transcription factor-like protein [KO:K03144]\n",
      "AT5G41360  XPB2; homolog of Xeroderma pigmentosum complementation group B 2 [KO:K10843] [EC:3.6.4.12]\n",
      "AT5G41370  XPB1; DNA repair helicase XPB1-like protein [KO:K10843] [EC:3.6.4.12]\n",
      "AT1G03190  UVH6; RAD3-like DNA-binding helicase protein [KO:K10844] [EC:3.6.4.12]\n",
      "AT1G12400  Nucleotide excision repair, TFIIH, subunit TTDA [KO:K10845]\n",
      "AT1G62886  Nucleotide excision repair, TFIIH, subunit TTDA [KO:K10845]\n",
      "AT1G18040  CDKD1;3; cyclin-dependent kinase D1;3 [KO:K02202] [EC:2.7.11.22 2.7.11.23]\n",
      "AT1G66750  CAK4; CDK-activating kinase 4 [KO:K02202] [EC:2.7.11.22 2.7.11.23]\n",
      "AT1G73690  CDKD1;1; cyclin-dependent kinase D1;1 [KO:K02202] [EC:2.7.11.22 2.7.11.23]\n",
      "AT4G30820  cyclin-dependent kinase-activating kinase assembly factor-related / CDK-activating kinase assembly factor-like protein [KO:K10842]\n",
      "AT5G27620  CYCH;1; cyclin H;1 [KO:K06634]\n",
      "REFERENCE   PMID:16858867\n",
      "AUTHORS   Thomas MC, Chiang CM\n",
      "TITLE     The general transcription machinery and general cofactors.\n",
      "JOURNAL   Crit Rev Biochem Mol Biol 41:105-78 (2006)\n",
      "DOI:10.1080/10409230600648736\n",
      "REFERENCE   PMID:12516863\n",
      "AUTHORS   Martinez E\n",
      "TITLE     Multi-protein complexes in eukaryotic gene transcription.\n",
      "JOURNAL   Plant Mol Biol 50:925-47 (2002)\n",
      "DOI:10.1023/A:1021258713850\n",
      "REFERENCE   PMID:19308322\n",
      "AUTHORS   Cler E, Papai G, Schultz P, Davidson I\n",
      "TITLE     Recent advances in understanding the structure and function of general transcription factor TFIID.\n",
      "JOURNAL   Cell Mol Life Sci 66:2123-34 (2009)\n",
      "DOI:10.1007/s00018-009-0009-3\n",
      "REFERENCE   PMID:10838567\n",
      "AUTHORS   Malik S, Roeder RG.\n",
      "TITLE     Transcriptional regulation through Mediator-like coactivators in yeast and metazoan cells.\n",
      "JOURNAL   Trends Biochem Sci 25:277-283 (2000)\n",
      "DOI:10.1016/S0968-0004(00)01596-6\n",
      "REL_PATHWAY ath03020  RNA polymerase\n",
      "ath03420  Nucleotide excision repair\n",
      "KO_PATHWAY  ko03022\n",
      "///\n",
      "ENTRY       ath03030                    Pathway\n",
      "NAME        DNA replication - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION A complex network of interacting proteins and enzymes is required for DNA replication. Generally, DNA replication follows a multistep enzymatic pathway. At the DNA replication fork, a DNA helicase (DnaB or MCM complex) precedes the DNA synthetic machinery and unwinds the duplex parental DNA in cooperation with the SSB or RPA. On the leading strand, replication occurs continuously in a 5 to 3 direction, whereas on the lagging strand, DNA replication occurs discontinuously by synthesis and joining of short Okazaki fragments. In prokaryotes, the leading strand replication apparatus consists of a DNA polymerase (pol III core), a sliding clamp (beta), and a clamp loader (gamma delta complex). The DNA primase (DnaG) is needed to form RNA primers. Normally, during replication of the lagging-strand DNA template, an RNA primer is removed either by an RNase H or by the 5 to 3 exonuclease activity of DNA pol I, and the DNA ligase joins the Okazaki fragments. In eukaryotes, three DNA polymerases (alpha, delta, and epsilon) have been identified. DNA primase forms a permanent complex with DNA polymerase alpha. PCNA and RFC function as a clamp and a clamp loader. FEN 1 and RNase H1 remove the RNA from the Okazaki fragments and DNA ligase I joins the DNA.\n",
      "CLASS       Genetic Information Processing; Replication and repair\n",
      "PATHWAY_MAP ath03030  DNA replication\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G18580  Nucleic acid-binding, OB-fold-like protein [KO:K03111]\n",
      "AT1G50840  POLGAMMA2; polymerase gamma 2 [KO:K02335] [EC:2.7.7.7]\n",
      "AT3G20540  POLGAMMA1; polymerase gamma 1 [KO:K02335] [EC:2.7.7.7]\n",
      "AT2G06510  RPA1A; replication protein A 1A [KO:K07466]\n",
      "AT1G10590  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT5G45400  RPA70C; Replication factor-A protein 1-like protein [KO:K07466]\n",
      "AT1G23750  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT2G33845  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT4G28440  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT1G03810  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT5G61000  RPA70D; Replication factor-A protein 1-like protein [KO:K07466]\n",
      "AT5G08020  RPA70B; RPA70-kDa subunit B [KO:K07466]\n",
      "AT2G29570  PCNA2; proliferating cell nuclear antigen 2 [KO:K04802]\n",
      "AT1G07370  PCNA1; proliferating cellular nuclear antigen 1 [KO:K04802]\n",
      "AT1G08840  emb2411; DNA replication helicase [KO:K10742] [EC:3.6.4.12]\n",
      "AT5G26680  5'-3' exonuclease family protein [KO:K04799] [EC:3.-.-.-]\n",
      "AT1G08130  LIG1; DNA ligase 1 [KO:K10747] [EC:6.5.1.1 6.5.1.6 6.5.1.7]\n",
      "AT1G49250  ATP-dependent DNA ligase [KO:K10747] [EC:6.5.1.1 6.5.1.6 6.5.1.7]\n",
      "AT5G67100  ICU2; DNA-directed DNA polymerase [KO:K02320] [EC:2.7.7.7]\n",
      "AT1G67630  POLA2; DNA polymerase alpha 2 [KO:K02321]\n",
      "AT5G41880  POLA3; DNA primase POLA3 [KO:K02684] [EC:2.7.7.102]\n",
      "AT1G67320  EMB2813; DNA primase, large subunit family [KO:K02685]\n",
      "AT5G63960  DNA polymerase delta subunit 1 [KO:K02327] [EC:2.7.7.7]\n",
      "AT2G42120  POLD2; DNA polymerase delta small subunit [KO:K02328]\n",
      "AT1G78650  POLD3; DNA-directed DNA polymerase [KO:K03504]\n",
      "AT1G09815  POLD4; polymerase delta 4 [KO:K03505]\n",
      "AT1G08260  TIL1; DNA polymerase epsilon catalytic subunit [KO:K02324] [EC:2.7.7.7]\n",
      "AT2G27120  TIL2; DNA polymerase epsilon catalytic subunit [KO:K02324] [EC:2.7.7.7]\n",
      "AT5G22110  DPB2; DNA polymerase epsilon subunit B2 [KO:K02325] [EC:2.7.7.7]\n",
      "AT2G27470  NF-YB11; nuclear factor Y, subunit B11 [KO:K02326] [EC:2.7.7.7]\n",
      "AT1G44900  MCM2; minichromosome maintenance (MCM2/3/5) family protein [KO:K02540] [EC:3.6.4.12]\n",
      "AT5G46280  MCM3; Minichromosome maintenance (MCM2/3/5) family protein [KO:K02541] [EC:3.6.4.12]\n",
      "AT2G16440  MCM4; Minichromosome maintenance (MCM2/3/5) family protein [KO:K02212] [EC:3.6.4.12]\n",
      "AT2G07690  MCM5; Minichromosome maintenance (MCM2/3/5) family protein [KO:K02209] [EC:3.6.4.12]\n",
      "AT5G44635  MCM6; minichromosome maintenance (MCM2/3/5) family protein [KO:K02542] [EC:3.6.4.12]\n",
      "AT4G02060  PRL; Minichromosome maintenance (MCM2/3/5) family protein [KO:K02210] [EC:3.6.4.12]\n",
      "AT2G24490  RPA2; replicon protein A2 [KO:K10739]\n",
      "AT3G02920  RPA32B; Replication protein A, subunit RPA32 [KO:K10739]\n",
      "AT3G52630  Nucleic acid-binding, OB-fold-like protein [KO:K10740]\n",
      "AT4G18590  Nucleic acid-binding, OB-fold-like protein [KO:K10740]\n",
      "AT5G22010  RFC1; replication factor C1 [KO:K10754]\n",
      "AT1G21690  EMB1968; ATPase family associated with various cellular activities (AAA) [KO:K10755]\n",
      "AT1G63160  RFC2; replication factor C 2 [KO:K10755]\n",
      "AT1G77470  RFC3; replication factor C subunit 3 [KO:K10756]\n",
      "AT5G27740  EMB2775; ATPase family associated with various cellular activities (AAA) [KO:K10756]\n",
      "AT2G25100  Polynucleotidyl transferase, ribonuclease H-like superfamily protein [KO:K10743] [EC:3.1.26.4]\n",
      "AT4G20325  ribonuclease H2 subunit B [KO:K10744]\n",
      "AT2G39440  ribonuclease H2 subunit C-like protein [KO:K10745]\n",
      "REFERENCE   PMID:8087839\n",
      "AUTHORS   Stillman B.\n",
      "TITLE     Smart machines at the DNA replication fork.\n",
      "JOURNAL   Cell 78:725-8 (1994)\n",
      "DOI:10.1016/S0092-8674(94)90362-X\n",
      "REFERENCE   PMID:9759502\n",
      "AUTHORS   Waga S, Stillman B.\n",
      "TITLE     The DNA replication fork in eukaryotic cells.\n",
      "JOURNAL   Annu Rev Biochem 67:721-51 (1998)\n",
      "DOI:10.1146/annurev.biochem.67.1.721\n",
      "KO_PATHWAY  ko03030\n",
      "///\n",
      "ENTRY       ath03040                    Pathway\n",
      "NAME        Spliceosome - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION After transcription, eukaryotic mRNA precursors contain protein-coding exons and noncoding introns. In the following splicing, introns are excised and exons are joined by a macromolecular complex, the spliceosome. The standard spliceosome is made up of five small nuclear ribonucleoproteins (snRNPs), U1, U2, U4, U5, and U6 snRNPs, and several spliceosome-associated proteins (SAPs). Spliceosomes are not a simple stable complex, but a dynamic family of particles that assemble on the mRNA precursor and help fold it into a conformation that allows transesterification to proceed. Various spliceosome forms (e.g. A-, B- and C-complexes) have been identified.\n",
      "CLASS       Genetic Information Processing; Transcription\n",
      "PATHWAY_MAP ath03040  Spliceosome\n",
      "DBLINKS     GO: 0005681\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G20920  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12811] [EC:3.6.4.13]\n",
      "AT3G09620  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12811] [EC:3.6.4.13]\n",
      "AT5G11170  UAP56a; DEAD/DEAH box RNA helicase family protein [KO:K12812] [EC:3.6.4.13]\n",
      "AT5G11200  UAP56b; DEAD/DEAH box RNA helicase family protein [KO:K12812] [EC:3.6.4.13]\n",
      "AT1G32490  ESP3; RNA helicase family protein [KO:K12813] [EC:3.6.4.13]\n",
      "AT4G16680  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12813] [EC:3.6.4.13]\n",
      "AT2G35340  MEE29; helicase domain-containing protein [KO:K12813] [EC:3.6.4.13]\n",
      "AT5G13010  EMB3011; RNA helicase family protein [KO:K12815] [EC:3.6.4.13]\n",
      "AT1G10580  Transducin/WD40 repeat-like superfamily protein [KO:K12816]\n",
      "AT1G03140  splicing factor Prp18 family protein [KO:K12817]\n",
      "AT1G54590  pre-mRNA-splicing factor-like protein [KO:K12817]\n",
      "AT3G26560  ATP-dependent RNA helicase [KO:K12818] [EC:3.6.4.13]\n",
      "AT4G01020  helicase domain-containing protein / IBR domain-containing protein / zinc finger protein-like protein [KO:K12818] [EC:3.6.4.13]\n",
      "AT5G10370  helicase domain-containing protein / IBR domain-containing protein / zinc finger protein-like protein [KO:K12818] [EC:3.6.4.13]\n",
      "AT1G27900  RNA helicase family protein [KO:K12818] [EC:3.6.4.13]\n",
      "AT1G26370  RNA helicase family protein [KO:K12818] [EC:3.6.4.13]\n",
      "AT1G65660  SMP1; Pre-mRNA splicing Prp18-interacting factor [KO:K12819]\n",
      "AT3G45950  Pre-mRNA splicing Prp18-interacting factor [KO:K12819]\n",
      "AT4G37120  SMP2; Pre-mRNA splicing Prp18-interacting factor [KO:K12819]\n",
      "AT3G62310  RNA helicase family protein [KO:K12820] [EC:3.6.4.13]\n",
      "AT2G47250  RNA helicase family protein [KO:K12820] [EC:3.6.4.13]\n",
      "AT5G14900  helicase associated (HA2) domain-containing protein [KO:K12820] [EC:3.6.4.13]\n",
      "AT4G20440  smB; small nuclear ribonucleoprotein associated protein B [KO:K11086]\n",
      "AT5G44500  Small nuclear ribonucleoprotein family protein [KO:K11086]\n",
      "AT3G07590  Small nuclear ribonucleoprotein family protein [KO:K11087]\n",
      "AT4G02840  Small nuclear ribonucleoprotein family protein [KO:K11087]\n",
      "AT2G47640  Small nuclear ribonucleoprotein family protein [KO:K11096]\n",
      "AT3G62840  Small nuclear ribonucleoprotein family protein [KO:K11096]\n",
      "AT1G76300  SmD3; snRNP core protein SMD3 [KO:K11088]\n",
      "AT1G20580  Small nuclear ribonucleoprotein family protein [KO:K11088]\n",
      "AT4G30330  Small nuclear ribonucleoprotein family protein [KO:K11097]\n",
      "AT2G18740  Small nuclear ribonucleoprotein family protein [KO:K11097]\n",
      "AT4G30220  RUXF; small nuclear ribonucleoprotein F [KO:K11098]\n",
      "AT2G14285  Small nuclear ribonucleoprotein family protein [KO:K11098]\n",
      "AT3G11500  Small nuclear ribonucleoprotein family protein [KO:K11099]\n",
      "AT2G23930  SNRNP-G; Putative small nuclear ribonucleoprotein G [KO:K11099]\n",
      "AT3G50670  U1-70K; U1 small nuclear ribonucleoprotein-70K [KO:K11093]\n",
      "AT2G47580  U1A; spliceosomal protein U1A [KO:K11091]\n",
      "AT4G03120  C2H2 and C2HC zinc fingers superfamily protein [KO:K11095]\n",
      "AT1G44910  PRP40A; pre-mRNA-processing protein 40A [KO:K12821]\n",
      "AT3G19670  PRP40B; pre-mRNA-processing protein 40B [KO:K12821]\n",
      "AT1G60200  splicing factor PWI domain-containing protein / RNA recognition motif (RRM)-containing protein [KO:K12822]\n",
      "AT1G55150  RH20; DEA(D/H)-box RNA helicase family protein [KO:K12823] [EC:3.6.4.13]\n",
      "AT3G01540  DRH1; DEAD box RNA helicase 1 [KO:K12823] [EC:3.6.4.13]\n",
      "AT3G06480  DEAD box RNA helicase family protein [KO:K12823] [EC:3.6.4.13]\n",
      "AT5G14610  DEAD box RNA helicase family protein [KO:K12823] [EC:3.6.4.13]\n",
      "AT5G63120  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12823] [EC:3.6.4.13]\n",
      "AT3G19840  PRP40C; pre-mRNA-processing protein 40C [KO:K12824]\n",
      "AT5G58470  TAF15b; TBP-associated factor 15B [KO:K13098]\n",
      "AT1G09760  U2A'; U2 small nuclear ribonucleoprotein A [KO:K11092]\n",
      "AT2G30260  U2B''; U2 small nuclear ribonucleoprotein B [KO:K11094]\n",
      "AT1G06960  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K11094]\n",
      "AT1G14650  SWAP (Suppressor-of-White-APricot)/surp domain-containing protein / ubiquitin family protein [KO:K12825]\n",
      "AT1G14640  SWAP (Suppressor-of-White-APricot)/surp domain-containing protein [KO:K12825]\n",
      "AT4G16200  SWAP (Suppressor-of-White-APricot)/surp domain-containing protein [KO:K12825]\n",
      "AT5G06520  SWAP (Suppressor-of-White-APricot)/surp domain-containing protein [KO:K12825]\n",
      "AT2G43960  SWAP (Suppressor-of-White-APricot)/surp domain-containing protein [KO:K12825]\n",
      "AT3G27600  SWAP (Suppressor-of-White-APricot)/surp RNA-binding domain-containing protein [KO:K12825]\n",
      "AT2G32600  hydroxyproline-rich glycoprotein family protein [KO:K12826]\n",
      "AT5G06160  ATO; splicing factor-like protein [KO:K12827]\n",
      "AT5G64270  splicing factor [KO:K12828]\n",
      "AT4G21660  proline-rich spliceosome-associated (PSP) family protein [KO:K12829]\n",
      "AT3G55200  SAP130a; Cleavage and polyadenylation specificity factor (CPSF) A subunit protein [KO:K12830]\n",
      "AT3G55220  SAP130b; Cleavage and polyadenylation specificity factor (CPSF) A subunit protein [KO:K12830]\n",
      "AT3G11960  Cleavage and polyadenylation specificity factor (CPSF) A subunit protein [KO:K12830]\n",
      "AT2G18510  emb2444; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12831]\n",
      "AT4G10110  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12831]\n",
      "AT4G14342  Splicing factor 3B subunit 5/RDS3 complex subunit 10 [KO:K12832]\n",
      "AT3G23325  Splicing factor 3B subunit 5/RDS3 complex subunit 10 [KO:K12832]\n",
      "AT5G12190  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12833]\n",
      "AT2G14870  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12833]\n",
      "AT1G07170  PHF5-like protein [KO:K12834]\n",
      "AT2G30000  PHF5-like protein [KO:K12834]\n",
      "AT2G47330  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12835] [EC:3.6.4.13]\n",
      "AT5G42820  U2AF35B; Zinc finger C-x8-C-x5-C-x3-H type family protein [KO:K12836]\n",
      "AT3G44785  Zinc finger C-x8-C-x5-C-x3-H type family protein [KO:K12836]\n",
      "AT1G27650  ATU2AF35A; U2 snRNP auxiliary factor small subunit [KO:K12836]\n",
      "AT1G60900  U2 snRNP auxilliary factor, large subunit, splicing factor [KO:K12837]\n",
      "AT4G36690  ATU2AF65A; U2 snRNP auxilliary factor, large subunit, splicing factor [KO:K12837]\n",
      "AT1G60830  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12837]\n",
      "AT2G02570  nucleic acid binding/RNA binding protein [KO:K12839]\n",
      "AT1G30480  DRT111; D111/G-patch domain-containing protein [KO:K12840]\n",
      "AT5G25060  RRC1; RNA recognition motif (RRM)-containing protein [KO:K12842]\n",
      "AT5G10800  RNA recognition motif (RRM)-containing protein [KO:K12842]\n",
      "AT1G03330  LSM2; Small nuclear ribonucleoprotein family protein [KO:K12621]\n",
      "AT1G76860  LSM3B; Small nuclear ribonucleoprotein family protein [KO:K12622]\n",
      "AT1G21190  LSM3A; Small nuclear ribonucleoprotein family protein [KO:K12622]\n",
      "AT5G27720  emb1644; Small nuclear ribonucleoprotein family protein [KO:K12623]\n",
      "AT5G48870  SAD1; Small nuclear ribonucleoprotein family protein [KO:K12624]\n",
      "AT3G59810  LSM6A; Small nuclear ribonucleoprotein family protein [KO:K12625]\n",
      "AT2G43810  LSM6B; Small nuclear ribonucleoprotein family protein [KO:K12625]\n",
      "AT2G03870  EMB2816; Small nuclear ribonucleoprotein family protein [KO:K12626]\n",
      "AT1G65700  LSM8; Small nuclear ribonucleoprotein family protein [KO:K12627]\n",
      "AT1G28060  Pre-mRNA-splicing factor 3 [KO:K12843]\n",
      "AT3G55930  pre-mRNA-splicing factor [KO:K12843]\n",
      "AT2G41500  LIS; WD-40 repeat family protein / small nuclear ribonucleoprotein Prp4p-like protein [KO:K12662]\n",
      "AT2G38730  Cyclophilin-like peptidyl-prolyl cis-trans isomerase family protein [KO:K09567] [EC:5.2.1.8]\n",
      "AT1G60170  emb1220; pre-mRNA processing ribonucleoprotein binding region-containing protein [KO:K12844]\n",
      "AT4G22380  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K12845]\n",
      "AT4G12600  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K12845]\n",
      "AT5G20160  Ribosomal protein L7Ae/L30e/S12e/Gadd45 family protein [KO:K12845]\n",
      "AT5G57370  U4/U6.U5 small nuclear ribonucleoprotein [KO:K12846]\n",
      "AT4G22350  Ubiquitin C-terminal hydrolases superfamily protein [KO:K12847]\n",
      "AT4G22285  Ubiquitin C-terminal hydrolases superfamily protein [KO:K12847]\n",
      "AT4G22410  U4/U6.U5 tri-snRNP-associated-like protein [KO:K12847]\n",
      "AT5G16780  DOT2; SART-1 family [KO:K11984]\n",
      "AT3G14700  SART-1 family [KO:K11984]\n",
      "AT3G05760  C2H2 and C2HC zinc fingers superfamily protein [KO:K12848]\n",
      "AT2G40650  PRP38 family protein [KO:K12849]\n",
      "AT5G37370  ATSRL1; PRP38 family protein [KO:K12850]\n",
      "AT1G06220  MEE5; Ribosomal protein S5/Elongation factor G/III/V family protein [KO:K12852]\n",
      "AT5G25230  Ribosomal protein S5/Elongation factor G/III/V family protein [KO:K12852]\n",
      "AT1G20960  emb1507; U5 small nuclear ribonucleoprotein helicase [KO:K12854] [EC:3.6.4.13]\n",
      "AT2G42270  U5 small nuclear ribonucleoprotein helicase [KO:K12854] [EC:3.6.4.13]\n",
      "AT4G03430  EMB2770; pre-mRNA splicing factor-like protein [KO:K12855]\n",
      "AT1G80070  SUS2; Pre-mRNA-processing-splicing factor [KO:K12856]\n",
      "AT4G38780  pre-mRNA-processing-splicing factor-like protein [KO:K12856]\n",
      "AT2G43770  Transducin/WD40 repeat-like superfamily protein [KO:K12857]\n",
      "AT2G33730  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12858] [EC:3.6.4.13]\n",
      "AT1G28180  DEAD-box ATP-dependent RNA helicase-like protein [KO:K12858] [EC:3.6.4.13]\n",
      "AT5G08290  YLS8; mRNA splicing factor, thioredoxin-like U5 snRNP [KO:K12859]\n",
      "AT1G04510  MAC3A; MOS4-associated complex 3A [KO:K10599] [EC:2.3.2.27]\n",
      "AT2G33340  MAC3B; MOS4-associated complex 3B [KO:K10599] [EC:2.3.2.27]\n",
      "AT1G09770  CDC5; cell division cycle 5 [KO:K12860]\n",
      "AT3G18165  MOS4; modifier of snc1,4 [KO:K12861]\n",
      "AT4G15900  PRL1; pleiotropic regulatory locus 1 [KO:K12862]\n",
      "AT3G16650  Transducin/WD40 repeat-like superfamily protein [KO:K12862]\n",
      "AT3G13200  EMB2769; Cwf15 / Cwc15 cell cycle control family protein [KO:K12863]\n",
      "AT3G02710  ARM repeat superfamily protein [KO:K12864]\n",
      "AT3G12580  HSP70; heat shock protein 70 [KO:K03283]\n",
      "AT3G09440  Heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT4G24280  cpHsc70-1; chloroplast heat shock protein 70-1 [KO:K03283]\n",
      "AT5G49910  cpHsc70-2; chloroplast heat shock protein 70-2 [KO:K03283]\n",
      "AT5G02490  Hsp70-2; Heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT5G02500  HSC70-1; heat shock cognate protein 70-1 [KO:K03283]\n",
      "AT1G56410  ERD2; heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT1G16030  Hsp70b; heat shock protein 70B [KO:K03283]\n",
      "AT2G41020  WW domain-containing protein [KO:K12865]\n",
      "AT1G77180  SKIP; chromatin protein family [KO:K06063]\n",
      "AT5G28740  Tetratricopeptide repeat (TPR)-like superfamily protein [KO:K12867]\n",
      "AT2G16860  GCIP-interacting family protein [KO:K12868]\n",
      "AT5G41770  crooked neck protein, putative / cell cycle protein [KO:K12869]\n",
      "AT3G13210  crooked neck protein, putative / cell cycle protein [KO:K12869]\n",
      "AT5G45990  crooked neck protein, putative / cell cycle protein [KO:K12869]\n",
      "AT3G18790  pre-mRNA-splicing factor ISY1-like protein [KO:K12870]\n",
      "AT2G36130  Cyclophilin-like peptidyl-prolyl cis-trans isomerase family protein [KO:K12733] [EC:5.2.1.8]\n",
      "AT3G05070  Cwf18 pre-mRNA splicing factor [KO:K12871]\n",
      "AT1G07360  MAC5A; CCCH-type zinc fingerfamily protein with RNA-binding domain-containing protein [KO:K12872]\n",
      "AT2G29580  MAC5B; CCCH-type zinc fingerfamily protein with RNA-binding domain-containing protein [KO:K12872]\n",
      "AT4G21110  G10 family protein [KO:K12873]\n",
      "AT2G38770  EMB2765; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12874]\n",
      "AT4G39680  SAP domain-containing protein [KO:K12875]\n",
      "AT3G19760  EIF4A-III; eukaryotic initiation factor 4A-III [KO:K13025] [EC:3.6.4.13]\n",
      "AT1G51380  DEA(D/H)-box RNA helicase family protein [KO:K13025] [EC:3.6.4.13]\n",
      "AT1G51510  Y14; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12876]\n",
      "AT1G02140  MAGO; mago nashi family protein [KO:K12877]\n",
      "AT5G09860  THO1; nuclear matrix protein-like protein [KO:K12878]\n",
      "AT1G24706  THO2; THO2 [KO:K12879]\n",
      "AT5G56130  TEX1; Transducin/WD40 repeat-like superfamily protein [KO:K12880]\n",
      "AT5G02530  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT5G37720  ALY4; ALWAYS EARLY 4 [KO:K12881]\n",
      "AT1G66260  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT5G59950  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12881]\n",
      "AT2G13540  ABH1; ARM repeat superfamily protein [KO:K12882]\n",
      "AT5G44200  CBP20; CAP-binding protein 20 [KO:K12883]\n",
      "AT5G40490  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12741]\n",
      "AT5G61030  GR-RBP3; glycine-rich RNA-binding protein 3 [KO:K12741]\n",
      "AT1G74230  GR-RBP5; glycine-rich RNA-binding protein 5 [KO:K12741]\n",
      "AT3G15010  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12741]\n",
      "AT3G56860  UBA2A; UBP1-associated protein 2A [KO:K12741]\n",
      "AT2G41060  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12741]\n",
      "AT1G17640  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12741]\n",
      "AT3G13224  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12741]\n",
      "AT1G02840  SR34; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12890]\n",
      "AT1G09140  SR30; SERINE-ARGININE PROTEIN 30 [KO:K12890]\n",
      "AT3G49430  SR34a; SER/ARG-rich protein 34A [KO:K12890]\n",
      "AT4G02430  SR34b; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12890]\n",
      "AT5G64200  SC35; serine/arginine-rich splicing factor-like protein, putative [KO:K12891]\n",
      "AT3G61860  RS31; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12893]\n",
      "AT2G46610  RS31a; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12893]\n",
      "AT4G25500  RS40; arginine/serine-rich splicing factor 35 [KO:K12893]\n",
      "AT5G52040  RS41; RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12893]\n",
      "AT3G53500  RS2Z32; RNA-binding (RRM/RBD/RNP motifs) family protein with retrovirus zinc finger-like domain-containing protein [KO:K12896]\n",
      "AT2G37340  RS2Z33; arginine/serine-rich zinc knuckle-containing protein 33 [KO:K12896]\n",
      "AT4G31580  RSZ22; serine/arginine-rich 22 [KO:K12896]\n",
      "AT2G24590  RSZ22a; RNA recognition motif and CCHC-type zinc finger domains containing protein [KO:K12896]\n",
      "AT1G23860  RSZ21; RS-containing zinc finger protein 21 [KO:K12896]\n",
      "AT4G35785  RNA-binding (RRM/RBD/RNP motifs) family protein [KO:K12897]\n",
      "AT5G18810  SCL28; SC35-like splicing factor 28 [KO:K12900]\n",
      "AT3G13570  SCL30A; SC35-like splicing factor 30A [KO:K12900]\n",
      "AT1G55310  SCL33; SC35-like splicing factor 33 [KO:K12900]\n",
      "AT3G55460  SCL30; SC35-like splicing factor 30 [KO:K12900]\n",
      "REFERENCE   PMID:19239890\n",
      "AUTHORS   Wahl MC, Will CL, Luhrmann R\n",
      "TITLE     The spliceosome: design principles of a dynamic RNP machine.\n",
      "JOURNAL   Cell 136:701-18 (2009)\n",
      "DOI:10.1016/j.cell.2009.02.009\n",
      "REFERENCE   PMID:10978320\n",
      "AUTHORS   Morris DP, Greenleaf AL\n",
      "TITLE     The splicing factor, Prp40, binds the phosphorylated carboxyl-terminal domain of RNA polymerase II.\n",
      "JOURNAL   J Biol Chem 275:39935-43 (2000)\n",
      "DOI:10.1074/jbc.M004118200\n",
      "REFERENCE   PMID:17537823\n",
      "AUTHORS   Chen YI, Moore RE, Ge HY, Young MK, Lee TD, Stevens SW\n",
      "TITLE     Proteomic analysis of in vivo-assembled pre-mRNA splicing complexes expands the catalog of participating factors.\n",
      "JOURNAL   Nucleic Acids Res 35:3928-44 (2007)\n",
      "DOI:10.1093/nar/gkm347\n",
      "REFERENCE   PMID:19733268\n",
      "AUTHORS   Ritchie DB, Schellenberg MJ, MacMillan AM\n",
      "TITLE     Spliceosome structure: piece by piece.\n",
      "JOURNAL   Biochim Biophys Acta 1789:624-33 (2009)\n",
      "DOI:10.1016/j.bbagrm.2009.08.010\n",
      "KO_PATHWAY  ko03040\n",
      "///\n",
      "ENTRY       ath03050                    Pathway\n",
      "NAME        Proteasome - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The proteasome is a protein-destroying apparatus involved in many essential cellular functions, such as regulation of cell cycle, cell differentiation, signal transduction pathways, antigen processing for appropriate immune responses, stress signaling, inflammatory responses, and apoptosis. It is capable of degrading a variety of cellular proteins in a rapid and timely fashion and most substrate proteins are modified by ubiquitin before their degradation by the proteasome. The proteasome is a large protein complex consisting of a proteolytic core called the 20S particle and ancillary factors that regulate its activity in various ways. The most common form is the 26S proteasome containing one 20S core particle and two 19S regulatory particles that enable the proteasome to degrade ubiquitinated proteins by an ATP-dependent mechanism. Another form is the immunoproteasome containing two 11S regulatory particles, PA28 alpha and PA28 beta, which are induced by interferon gamma under the conditions of intensified immune response. Other regulatory particles include PA28 gamma and PA200. Although PA28 gamma also belongs to a family of activators of the 20S proteasome, it is localized within the nucleus and forms a homoheptamer. PA28 gamma has been implicated in the regulation of cell cycle progression and apoptosis. PA200 has been identified as a large nuclear protein that stimulates proteasomal hydrolysis of peptides.\n",
      "CLASS       Genetic Information Processing; Folding, sorting and degradation\n",
      "PATHWAY_MAP ath03050  Proteasome\n",
      "DBLINKS     GO: 0000502\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G20200  EMB2719; PAM domain (PCI/PINT associated module) protein [KO:K03033]\n",
      "AT1G75990  PAM domain (PCI/PINT associated module) protein [KO:K03033]\n",
      "AT5G57950  26S proteasome regulatory subunit [KO:K06693]\n",
      "AT5G09900  EMB2107; 26S proteasome regulatory subunit, putative (RPN5) [KO:K03035]\n",
      "AT5G64760  RPN5B; regulatory particle non-ATPase subunit 5B [KO:K03035]\n",
      "AT1G29150  ATS9; non-ATPase subunit 9 [KO:K03036]\n",
      "AT4G24820  26S proteasome regulatory subunit Rpn7 [KO:K03037]\n",
      "AT5G05780  RPN8A; RP non-ATPase subunit 8A [KO:K03038]\n",
      "AT3G11270  MEE34; Mov34/MPN/PAD-1 family protein [KO:K03038]\n",
      "AT5G45620  Proteasome component (PCI) domain protein [KO:K03039]\n",
      "AT4G19006  Proteasome component (PCI) domain protein [KO:K03039]\n",
      "AT5G23540  Mov34/MPN/PAD-1 family protein [KO:K03030]\n",
      "AT1G64520  RPN12a; regulatory particle non-ATPase 12A [KO:K03031]\n",
      "AT5G42040  RPN12b; regulatory particle non-ATPase 12B [KO:K03031]\n",
      "AT1G64750  DSS1(I); deletion of SUV3 suppressor 1(I) [KO:K10881]\n",
      "AT5G45010  DSS1(V); DSS1 homolog on chromosome V [KO:K10881]\n",
      "AT4G38630  RPN10; regulatory particle non-ATPase 10 [KO:K03029]\n",
      "AT2G20580  RPN1A; 26S proteasome regulatory subunit S2 1A [KO:K03028]\n",
      "AT4G28470  RPN1B; 26S proteasome regulatory subunit S2 1B [KO:K03028]\n",
      "AT2G32730  26S proteasome regulatory complex, non-ATPase subcomplex, Rpn2/Psmd1 subunit [KO:K03032]\n",
      "AT1G04810  26S proteasome regulatory complex, non-ATPase subcomplex, Rpn2/Psmd1 subunit [KO:K03032]\n",
      "AT2G26590  RPN13; regulatory particle non-ATPase 13 [KO:K06691]\n",
      "AT1G53750  RPT1A; regulatory particle triple-A 1A [KO:K03061]\n",
      "AT1G53780  26S proteasome regulatory complex ATPase [KO:K03061]\n",
      "AT4G29040  RPT2a; regulatory particle AAA-ATPase 2A [KO:K03062]\n",
      "AT2G20140  RPT2b; AAA-type ATPase family protein [KO:K03062]\n",
      "AT5G19990  RPT6A; regulatory particle triple-A ATPase 6A [KO:K03066]\n",
      "AT5G20000  AAA-type ATPase family protein [KO:K03066]\n",
      "AT1G45000  AAA-type ATPase family protein [KO:K03064]\n",
      "AT5G43010  RPT4A; regulatory particle triple-A ATPase 4A [KO:K03064]\n",
      "AT3G05530  RPT5A; regulatory particle triple-A ATPase 5A [KO:K03065]\n",
      "AT1G09100  RPT5B; 26S proteasome AAA-ATPase subunit RPT5B [KO:K03065]\n",
      "AT5G58290  RPT3; regulatory particle triple-A ATPase 3 [KO:K03063]\n",
      "AT3G13330  PA200; proteasome activating protein 200 [KO:K06699]\n",
      "AT5G35590  PAA1; proteasome alpha subunit A1 [KO:K02730] [EC:3.4.25.1]\n",
      "AT2G05840  PAA2; 20S proteasome subunit PAA2 [KO:K02730] [EC:3.4.25.1]\n",
      "AT1G16470  PAB1; proteasome subunit PAB1 [KO:K02726] [EC:3.4.25.1]\n",
      "AT1G79210  N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K02726] [EC:3.4.25.1]\n",
      "AT3G22110  PAC1; 20S proteasome alpha subunit C1 [KO:K02728] [EC:3.4.25.1]\n",
      "AT4G15165  N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K02728] [EC:3.4.25.1]\n",
      "AT5G66140  PAD2; proteasome alpha subunit D2 [KO:K02731] [EC:3.4.25.1]\n",
      "AT3G51260  PAD1; 20S proteasome alpha subunit PAD1 [KO:K02731] [EC:3.4.25.1]\n",
      "AT1G53850  PAE1; 20S proteasome alpha subunit E1 [KO:K02729] [EC:3.4.25.1]\n",
      "AT3G14290  PAE2; 20S proteasome alpha subunit E2 [KO:K02729] [EC:3.4.25.1]\n",
      "AT1G47250  PAF2; 20S proteasome alpha subunit F2 [KO:K02725] [EC:3.4.25.1]\n",
      "AT5G42790  PAF1; proteasome alpha subunit F1 [KO:K02725] [EC:3.4.25.1]\n",
      "AT2G27020  PAG1; 20S proteasome alpha subunit G1 [KO:K02727] [EC:3.4.25.1]\n",
      "AT4G31300  PBA1; N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K02738] [EC:3.4.25.1]\n",
      "AT3G27430  PBB1; N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K02739] [EC:3.4.25.1]\n",
      "AT5G40580  PBB2; 20S proteasome beta subunit PBB2 [KO:K02739] [EC:3.4.25.1]\n",
      "AT1G21720  PBC1; proteasome beta subunit C1 [KO:K02735] [EC:3.4.25.1]\n",
      "AT1G77440  PBC2; 20S proteasome beta subunit C2 [KO:K02735] [EC:3.4.25.1]\n",
      "AT4G14800  PBD2; 20S proteasome beta subunit D2 [KO:K02734] [EC:3.4.25.1]\n",
      "AT3G22630  PBD1; 20S proteasome beta subunit D1 [KO:K02734] [EC:3.4.25.1]\n",
      "AT1G13060  PBE1; 20S proteasome beta subunit E1 [KO:K02737] [EC:3.4.25.1]\n",
      "AT3G26340  N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K02737] [EC:3.4.25.1]\n",
      "AT3G60820  PBF1; N-terminal nucleophile aminohydrolases (Ntn hydrolases) superfamily protein [KO:K02732] [EC:3.4.25.1]\n",
      "AT1G56450  PBG1; 20S proteasome beta subunit G1 [KO:K02736] [EC:3.4.25.1]\n",
      "AT3G53970  proteasome inhibitor-like protein [KO:K06700]\n",
      "AT5G38650  Proteasome maturation factor UMP1 [KO:K11599]\n",
      "AT1G67250  Proteasome maturation factor UMP1 [KO:K11599]\n",
      "REFERENCE   PMID:16338359\n",
      "AUTHORS   Hirano Y, Murata S, Tanaka K\n",
      "TITLE     Large- and small-scale purification of mammalian 26S proteasomes.\n",
      "JOURNAL   Methods Enzymol 399:227-40 (2005)\n",
      "DOI:10.1016/S0076-6879(05)99015-0\n",
      "REFERENCE   PMID:18497808\n",
      "AUTHORS   Saeki Y, Tanaka K\n",
      "TITLE     Cell biology: two hands for degradation.\n",
      "JOURNAL   Nature 453:460-1 (2008)\n",
      "DOI:10.1038/453460a\n",
      "REFERENCE   PMID:16919475\n",
      "AUTHORS   Smith DM, Benaroudj N, Goldberg A\n",
      "TITLE     Proteasomes and their associated ATPases: a destructive combination.\n",
      "JOURNAL   J Struct Biol 156:72-83 (2006)\n",
      "DOI:10.1016/j.jsb.2006.04.012\n",
      "REFERENCE   PMID:16181324\n",
      "AUTHORS   Strehl B, Seifert U, Kruger E, Heink S, Kuckelkorn U, Kloetzel PM\n",
      "TITLE     Interferon-gamma, the functional plasticity of the ubiquitin-proteasome system, and MHC class I antigen processing.\n",
      "JOURNAL   Immunol Rev 207:19-30 (2005)\n",
      "DOI:10.1111/j.0105-2896.2005.00308.x\n",
      "REFERENCE   PMID:19483713\n",
      "AUTHORS   Darwin KH\n",
      "TITLE     Prokaryotic ubiquitin-like protein (Pup), proteasomes and pathogenesis.\n",
      "JOURNAL   Nat Rev Microbiol 7:485-91 (2009)\n",
      "DOI:10.1038/nrmicro2148\n",
      "REFERENCE   PMID:23495936\n",
      "AUTHORS   Tomko RJ Jr, Hochstrasser M\n",
      "TITLE     Molecular architecture and assembly of the eukaryotic proteasome.\n",
      "JOURNAL   Annu Rev Biochem 82:415-45 (2013)\n",
      "DOI:10.1146/annurev-biochem-060410-150257\n",
      "REFERENCE   PMID:28583440\n",
      "AUTHORS   Budenholzer L, Cheng CL, Li Y, Hochstrasser M\n",
      "TITLE     Proteasome Structure and Assembly.\n",
      "JOURNAL   J Mol Biol 429:3500-3524 (2017)\n",
      "DOI:10.1016/j.jmb.2017.05.027\n",
      "REFERENCE   PMID:29071527\n",
      "AUTHORS   Cloos J, Roeten MS, Franke NE, van Meerloo J, Zweegman S, Kaspers GJ, Jansen G\n",
      "TITLE     (Immuno)proteasomes as therapeutic target in acute leukemia.\n",
      "JOURNAL   Cancer Metastasis Rev 36:599-615 (2017)\n",
      "DOI:10.1007/s10555-017-9699-4\n",
      "REFERENCE   PMID:22215586\n",
      "AUTHORS   Sakata E, Bohn S, Mihalache O, Kiss P, Beck F, Nagy I, Nickell S, Tanaka K, Saeki Y, Forster F, Baumeister W\n",
      "TITLE     Localization of the proteasomal ubiquitin receptors Rpn10 and Rpn13 by electron cryomicroscopy.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 109:1479-84 (2012)\n",
      "DOI:10.1073/pnas.1119394109\n",
      "REL_PATHWAY ath04120  Ubiquitin mediated proteolysis\n",
      "KO_PATHWAY  ko03050\n",
      "///\n",
      "ENTRY       ath03060                    Pathway\n",
      "NAME        Protein export - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The protein export is the active transport of proteins from the cytoplasm to the exterior of the cell, or to the periplasmic compartment in Gram-negative bacteria. The sec dependent pathway is the general protein export system that transports newly synthesized proteins into or across the cell membrane. The translocation channel is formed from a conserved trimeric membrane protein complex, called the Sec61/SecY complex. The twin-arginine translocation (Tat) pathway is another protein transport system that transports folded proteins in bacteria, archaea, and chloroplasts. Many Tat systems comprise three functionally different membrane proteins, TatA, TatB, and TatC, but TatA and TatE seem to have overlapping functions, with TatA having by far the more important role.\n",
      "CLASS       Genetic Information Processing; Folding, sorting and degradation\n",
      "PATHWAY_MAP ath03060  Protein export\n",
      "DBLINKS     GO: 0043952 0043953\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G24490  ALB4; OxaA/YidC-like membrane insertion protein [KO:K03217]\n",
      "AT5G62050  OXA1; inner membrane OXA1-like protein [KO:K03217]\n",
      "AT2G46470  OXA1L; inner membrane protein OXA1-like protein [KO:K03217]\n",
      "AT2G28800  ALB3; 63 kDa inner membrane family protein [KO:K03217]\n",
      "AT4G01800  AGY1; Albino or Glassy Yellow 1 [KO:K03070] [EC:7.4.2.8]\n",
      "AT1G21650  SECA2; Preprotein translocase SecA family protein [KO:K03070] [EC:7.4.2.8]\n",
      "AT5G66970  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K03106] [EC:3.6.5.4]\n",
      "AT5G03940  CPSRP54; chloroplast signal recognition particle 54 kDa subunit [KO:K03106] [EC:3.6.5.4]\n",
      "AT1G15310  ATHSRP54A; signal recognition particle 54 kDa subunit [KO:K03106] [EC:3.6.5.4]\n",
      "AT5G49500  Signal recognition particle, SRP54 subunit protein [KO:K03106] [EC:3.6.5.4]\n",
      "AT1G48900  Signal recognition particle, SRP54 subunit protein [KO:K03106] [EC:3.6.5.4]\n",
      "AT2G45770  CPFTSY; signal recognition particle receptor protein, chloroplast (FTSY) [KO:K03110]\n",
      "AT1G78720  SecY protein transport family protein [KO:K10956]\n",
      "AT1G29310  SecY protein transport family protein [KO:K10956]\n",
      "AT2G18710  SCY1; SECY homolog 1 [KO:K10956]\n",
      "AT2G34250  SecY protein transport family protein [KO:K10956]\n",
      "AT5G60460  Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT2G45070  SEC61_BETA; Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT3G60540  Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT4G24920  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT3G48570  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT5G50460  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT3G20920  translocation protein-like protein [KO:K12275]\n",
      "AT1G79940  ATERDJ2A; DnaJ / Sec63 Brl domains-containing protein [KO:K09540]\n",
      "AT4G21180  ATERDJ2B; DnaJ / Sec63 Brl domains-containing protein [KO:K09540]\n",
      "AT5G42020  BIP2; Heat shock protein 70 (Hsp 70) family protein [KO:K09490] [EC:3.6.4.10]\n",
      "AT5G28540  BIP1; heat shock protein 70 (Hsp 70) family protein [KO:K09490] [EC:3.6.4.10]\n",
      "AT1G09080  BIP3; Heat shock protein 70 (Hsp 70) family protein [KO:K09490] [EC:3.6.4.10]\n",
      "AT3G49100  Signal recognition particle, SRP9/SRP14 subunit [KO:K03109]\n",
      "AT2G43640  Signal recognition particle, SRP9/SRP14 subunit [KO:K03104]\n",
      "AT1G67650  SRP72 RNA-binding domain-containing protein [KO:K03108]\n",
      "AT1G67680  SRP72 RNA-binding domain-containing protein [KO:K03108]\n",
      "AT5G61970  signal recognition particle-related / SRP-like protein [KO:K03107]\n",
      "AT1G48160  signal recognition particle 19 kDa protein, putative / SRP19 [KO:K03105]\n",
      "AT4G30600  signal recognition particle receptor alpha subunit family protein [KO:K13431]\n",
      "AT5G05670  signal recognition particle binding protein [KO:K12272]\n",
      "AT2G18770  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K12272]\n",
      "AT5G28750  Bacterial sec-independent translocation protein mttA/Hcf106 [KO:K03116]\n",
      "AT2G01110  APG2; Sec-independent periplasmic protein translocase [KO:K03118]\n",
      "AT3G24590  PLSP1; plastidic type i signal peptidase 1 [KO:K03100] [EC:3.4.21.89]\n",
      "AT2G30440  TPP; thylakoid processing peptide [KO:K03100] [EC:3.4.21.89]\n",
      "AT1G06870  Plsp2A; Peptidase S24/S26A/S26B/S26C family protein [KO:K03100] [EC:3.4.21.89]\n",
      "AT2G22425  Microsomal signal peptidase 12 kDa subunit (SPC12) [KO:K12946] [EC:3.4.-.-]\n",
      "AT4G40042  Microsomal signal peptidase 12 kDa subunit (SPC12) [KO:K12946] [EC:3.4.-.-]\n",
      "AT2G39960  Microsomal signal peptidase 25 kDa subunit (SPC25) [KO:K12947] [EC:3.4.-.-]\n",
      "AT4G04200  Microsomal signal peptidase 25 kDa subunit (SPC25) [KO:K12947] [EC:3.4.-.-]\n",
      "AT5G27430  Signal peptidase subunit [KO:K12948] [EC:3.4.-.-]\n",
      "AT3G05230  Signal peptidase subunit [KO:K12948] [EC:3.4.-.-]\n",
      "AT1G52600  Peptidase S24/S26A/S26B/S26C family protein [KO:K13280] [EC:3.4.21.89]\n",
      "AT3G15710  Peptidase S24/S26A/S26B/S26C family protein [KO:K13280] [EC:3.4.21.89]\n",
      "AT1G23465  Peptidase S24/S26A/S26B/S26C family protein [KO:K09647] [EC:3.4.21.-]\n",
      "AT1G29960  Peptidase S24/S26A/S26B/S26C family protein [KO:K09647] [EC:3.4.21.-]\n",
      "AT1G53530  Peptidase S24/S26A/S26B/S26C family protein [KO:K09647] [EC:3.4.21.-]\n",
      "AT3G08980  Peptidase S24/S26A/S26B/S26C family protein [KO:K09648] [EC:3.4.21.-]\n",
      "REFERENCE   PMID:10087917\n",
      "AUTHORS   Dalbey RE, Robinson C.\n",
      "TITLE     Protein translocation into and across the bacterial plasma membrane and the plant thylakoid membrane.\n",
      "JOURNAL   Trends Biochem Sci 24:17-22 (1999)\n",
      "DOI:10.1016/S0968-0004(98)01333-4\n",
      "REFERENCE   PMID:10652088\n",
      "AUTHORS   Berks BC, Sargent F, Palmer T.\n",
      "TITLE     The Tat protein export pathway.\n",
      "JOURNAL   Mol Microbiol 35:260-74 (2000)\n",
      "DOI:10.1046/j.1365-2958.2000.01719.x\n",
      "REFERENCE   PMID:10747959\n",
      "AUTHORS   Wexler M, Sargent F, Jack RL, Stanley NR, Bogsch EG, Robinson C, Berks BC, Palmer T.\n",
      "TITLE     TatD is a cytoplasmic protein with DNase activity. No requirement for TatD family proteins in sec-independent protein export.\n",
      "JOURNAL   J Biol Chem 275:16717-22 (2000)\n",
      "DOI:10.1074/jbc.M000800200\n",
      "REFERENCE   PMID:11007775\n",
      "AUTHORS   Jongbloed JD, Martin U, Antelmann H, Hecker M, Tjalsma H, Venema G, Bron S, van Dijl JM, Muller J.\n",
      "TITLE     TatC is a specificity determinant for protein secretion via the twin-arginine translocation pathway.\n",
      "JOURNAL   J Biol Chem 275:41350-7 (2000)\n",
      "DOI:10.1074/jbc.M004887200\n",
      "REFERENCE   PMID:18046402\n",
      "AUTHORS   Rapoport TA\n",
      "TITLE     Protein translocation across the eukaryotic endoplasmic reticulum and bacterial plasma membranes.\n",
      "JOURNAL   Nature 450:663-9 (2007)\n",
      "DOI:10.1038/nature06384\n",
      "REFERENCE   PMID:17368028\n",
      "AUTHORS   Flower AM\n",
      "TITLE     The SecY translocation complex: convergence of genetics and structure.\n",
      "JOURNAL   Trends Microbiol 15:203-10 (2007)\n",
      "DOI:10.1016/j.tim.2007.03.001\n",
      "REFERENCE   PMID:16212506\n",
      "AUTHORS   Osborne AR, Rapoport TA, van den Berg B\n",
      "TITLE     Protein translocation by the Sec61/SecY channel.\n",
      "JOURNAL   Annu Rev Cell Dev Biol 21:529-50 (2005)\n",
      "DOI:10.1146/annurev.cellbio.21.012704.133214\n",
      "REFERENCE   PMID:12475201\n",
      "AUTHORS   Paetzel M, Karla A, Strynadka NC, Dalbey RE\n",
      "TITLE     Signal peptidases.\n",
      "JOURNAL   Chem Rev 102:4549-80 (2002)\n",
      "DOI:10.1021/cr010166y\n",
      "REFERENCE   PMID:10924740\n",
      "AUTHORS   Paetzel M, Dalbey RE, Strynadka NC\n",
      "TITLE     The structure and mechanism of bacterial type I signal peptidases. A novel antibiotic target.\n",
      "JOURNAL   Pharmacol Ther 87:27-49 (2000)\n",
      "DOI:10.1016/S0163-7258(00)00064-4\n",
      "REFERENCE   PMID:17259602\n",
      "AUTHORS   Ng SY, Chaban B, VanDyke DJ, Jarrell KF\n",
      "TITLE     Archaeal signal peptidases.\n",
      "JOURNAL   Microbiology 153:305-14 (2007)\n",
      "DOI:10.1099/mic.0.2006/003087-0\n",
      "REFERENCE   PMID:19299134\n",
      "AUTHORS   Desvaux M, Hebraud M, Talon R, Henderson IR\n",
      "TITLE     Secretion and subcellular localizations of bacterial proteins: a semantic awareness issue.\n",
      "JOURNAL   Trends Microbiol 17:139-45 (2009)\n",
      "DOI:10.1016/j.tim.2009.01.004\n",
      "REFERENCE   PMID:15063851\n",
      "AUTHORS   Nakatogawa H, Murakami A, Ito K\n",
      "TITLE     Control of SecA and SecM translation by protein secretion.\n",
      "JOURNAL   Curr Opin Microbiol 7:145-50 (2004)\n",
      "DOI:10.1016/j.mib.2004.01.001\n",
      "KO_PATHWAY  ko03060\n",
      "///\n",
      "ENTRY       ath03410                    Pathway\n",
      "NAME        Base excision repair - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Base excision repair (BER) is the predominant DNA damage repair pathway for the processing of small base lesions, derived from oxidation and alkylation damages. BER is normally defined as DNA repair initiated by lesion-specific DNA glycosylases and completed by either of the two sub-pathways: short-patch BER where only one nucleotide is replaced and long-patch BER where 2-13 nucleotides are replaced. Each sub-pathway of BER relies on the formation of protein complexes that assemble at the site of the DNA lesion and facilitate repair in a coordinated fashion. This process of complex formation appears to provide an increase in specificity and efficiency to the BER pathway, thereby facilitating the maintenance of genome integrity by preventing the accumulation of highly toxic repair intermediates.\n",
      "CLASS       Genetic Information Processing; Replication and repair\n",
      "PATHWAY_MAP ath03410  Base excision repair\n",
      "DBLINKS     GO: 0006284 0006285 0006286 0006287 0006288\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G52500  MMH-1; MUTM homolog-1 [KO:K10563] [EC:3.2.2.23 4.2.99.18]\n",
      "AT1G21710  OGG1; 8-oxoguanine-DNA glycosylase 1 [KO:K03660] [EC:3.2.2.- 4.2.99.18]\n",
      "AT2G31450  ATNTH1; DNA glycosylase superfamily protein [KO:K10773] [EC:4.2.99.18]\n",
      "AT1G05900  NTH2; endonuclease III 2 [KO:K10773] [EC:4.2.99.18]\n",
      "AT3G47830  DNA glycosylase superfamily protein [KO:K10773] [EC:4.2.99.18]\n",
      "AT5G13920  GRF zinc finger / Zinc knuckle protein [KO:K10569]\n",
      "AT1G75230  DNA glycosylase superfamily protein [KO:K01247] [EC:3.2.2.21]\n",
      "AT3G50880  DNA glycosylase superfamily protein [KO:K01247] [EC:3.2.2.21]\n",
      "AT1G19480  DNA glycosylase superfamily protein [KO:K01247] [EC:3.2.2.21]\n",
      "AT1G75090  DNA glycosylase superfamily protein [KO:K01246] [EC:3.2.2.20]\n",
      "AT5G44680  DNA glycosylase superfamily protein [KO:K01246] [EC:3.2.2.20]\n",
      "AT1G15970  DNA glycosylase superfamily protein [KO:K01246] [EC:3.2.2.20]\n",
      "AT3G12710  DNA glycosylase superfamily protein [KO:K01246] [EC:3.2.2.20]\n",
      "AT5G57970  DNA glycosylase superfamily protein [KO:K01246] [EC:3.2.2.20]\n",
      "AT1G13635  DNA glycosylase superfamily protein [KO:K01246] [EC:3.2.2.20]\n",
      "AT1G80850  DNA glycosylase superfamily protein [KO:K01246] [EC:3.2.2.20]\n",
      "AT3G18630  UNG; uracil dna glycosylase [KO:K03648] [EC:3.2.2.27]\n",
      "AT4G12740  HhH-GPD base excision DNA repair family protein [KO:K03575] [EC:3.2.2.31]\n",
      "AT3G12040  DNA-3-methyladenine glycosylase (MAG) [KO:K03652] [EC:3.2.2.21]\n",
      "AT3G07930  DNA glycosylase superfamily protein [KO:K10801] [EC:3.2.2.-]\n",
      "AT2G41460  ARP; apurinic endonuclease-redox protein [KO:K01142] [EC:3.1.11.2]\n",
      "AT4G36050  endonuclease/exonuclease/phosphatase family protein [KO:K10772] [EC:4.2.99.18]\n",
      "AT1G50840  POLGAMMA2; polymerase gamma 2 [KO:K02335] [EC:2.7.7.7]\n",
      "AT3G20540  POLGAMMA1; polymerase gamma 1 [KO:K02335] [EC:2.7.7.7]\n",
      "AT1G10520  Pol{lambda}; DNA polymerase lambda (POLL) [KO:K03512] [EC:2.7.7.7 4.2.99.-]\n",
      "AT1G80420  ATXRCC1; BRCT domain-containing DNA repair protein [KO:K10803]\n",
      "AT2G29570  PCNA2; proliferating cell nuclear antigen 2 [KO:K04802]\n",
      "AT1G07370  PCNA1; proliferating cellular nuclear antigen 1 [KO:K04802]\n",
      "AT5G63960  DNA polymerase delta subunit 1 [KO:K02327] [EC:2.7.7.7]\n",
      "AT2G42120  POLD2; DNA polymerase delta small subunit [KO:K02328]\n",
      "AT1G78650  POLD3; DNA-directed DNA polymerase [KO:K03504]\n",
      "AT1G09815  POLD4; polymerase delta 4 [KO:K03505]\n",
      "AT1G08260  TIL1; DNA polymerase epsilon catalytic subunit [KO:K02324] [EC:2.7.7.7]\n",
      "AT2G27120  TIL2; DNA polymerase epsilon catalytic subunit [KO:K02324] [EC:2.7.7.7]\n",
      "AT5G22110  DPB2; DNA polymerase epsilon subunit B2 [KO:K02325] [EC:2.7.7.7]\n",
      "AT2G27470  NF-YB11; nuclear factor Y, subunit B11 [KO:K02326] [EC:2.7.7.7]\n",
      "AT1G08130  LIG1; DNA ligase 1 [KO:K10747] [EC:6.5.1.1 6.5.1.6 6.5.1.7]\n",
      "AT1G49250  ATP-dependent DNA ligase [KO:K10747] [EC:6.5.1.1 6.5.1.6 6.5.1.7]\n",
      "AT2G31320  PARP1; poly(ADP-ribose) polymerase 2 [KO:K24070] [EC:2.4.2.30]\n",
      "AT4G02390  PARP2; poly(ADP-ribose) polymerase [KO:K10798] [EC:2.4.2.30]\n",
      "AT5G22470  poly ADP-ribose polymerase 3 [KO:K10798] [EC:2.4.2.30]\n",
      "AT5G26680  5'-3' exonuclease family protein [KO:K04799] [EC:3.-.-.-]\n",
      "REFERENCE   PMID:17893748\n",
      "AUTHORS   Krwawicz J, Arczewska KD, Speina E, Maciejewska A, Grzesiuk E.\n",
      "TITLE     Bacterial DNA repair genes and their eukaryotic homologues: 1. Mutations in genes involved in base excision repair (BER) and DNA-end processors and their implication in mutagenesis and human disease.\n",
      "JOURNAL   Acta Biochim Pol 54:413-34 (2007)\n",
      "REFERENCE   PMID:17337257\n",
      "AUTHORS   Almeida KH, Sobol RW.\n",
      "TITLE     A unified view of base excision repair: lesion-dependent protein complexes regulated by post-translational modification.\n",
      "JOURNAL   DNA Repair (Amst) 6:695-711 (2007)\n",
      "DOI:10.1016/j.dnarep.2007.01.009\n",
      "REFERENCE   PMID:21665970\n",
      "AUTHORS   Moen MN, Knaevelsrud I, Haugland GT, Grosvik K, Birkeland NK, Klungland A, Bjelland S\n",
      "TITLE     Uracil-DNA glycosylase of Thermoplasma acidophilum directs long-patch base excision repair, which is promoted by deoxynucleoside triphosphates and ATP/ADP, into short-patch repair.\n",
      "JOURNAL   J Bacteriol 193:4495-508 (2011)\n",
      "DOI:10.1128/JB.00233-11\n",
      "REFERENCE   PMID:11436317\n",
      "AUTHORS   Ikeda S, Seki S.\n",
      "TITLE     [Base excision repair: DNA glycosylase and AP endonuclease] Japanese\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 46:916-23 (2001)\n",
      "KO_PATHWAY  ko03410\n",
      "///\n",
      "ENTRY       ath03420                    Pathway\n",
      "NAME        Nucleotide excision repair - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Nucleotide excision repair (NER) is a mechanism to recognize and repair bulky DNA damage caused by compounds, environmental carcinogens, and exposure to UV-light. In humans hereditary defects in the NER pathway are linked to at least three diseases: xeroderma pigmentosum (XP), Cockayne syndrome (CS), and trichothiodystrophy (TTD). The repair of damaged DNA involves at least 30 polypeptides within two different sub-pathways of NER known as transcription-coupled repair (TCR-NER) and global genome repair (GGR-NER). TCR refers to the expedited repair of lesions located in the actively transcribed strand of genes by RNA polymerase II (RNAP II). In GGR-NER the first step of damage recognition involves XPC-hHR23B complex together with XPE complex (in prokaryotes, uvrAB complex). The following steps of GGR-NER and TCR-NER are similar.\n",
      "CLASS       Genetic Information Processing; Replication and repair\n",
      "PATHWAY_MAP ath03420  Nucleotide excision repair\n",
      "DBLINKS     GO: 0006289\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G50840  POLGAMMA2; polymerase gamma 2 [KO:K02335] [EC:2.7.7.7]\n",
      "AT3G20540  POLGAMMA1; polymerase gamma 1 [KO:K02335] [EC:2.7.7.7]\n",
      "AT3G42830  RING/U-box superfamily protein [KO:K03868] [EC:2.3.2.32]\n",
      "AT5G20570  RBX1; RING-box 1 [KO:K03868] [EC:2.3.2.32]\n",
      "AT5G46210  CUL4; cullin4 [KO:K10609]\n",
      "AT3G46910  Cullin family protein [KO:K10609]\n",
      "AT4G12100  Cullin family protein [KO:K10609]\n",
      "AT4G05420  DDB1A; damaged DNA binding protein 1A [KO:K10610]\n",
      "AT4G21100  DDB1B; damaged DNA binding protein 1B [KO:K10610]\n",
      "AT5G58760  DDB2; damaged DNA binding 2 [KO:K10140]\n",
      "AT5G16630  RAD4; DNA repair protein Rad4 family [KO:K10838]\n",
      "AT1G16190  RAD23A; Rad23 UV excision repair protein family [KO:K10839]\n",
      "AT1G79650  RAD23B; Rad23 UV excision repair protein family [KO:K10839]\n",
      "AT5G38470  RAD23D; Rad23 UV excision repair protein family [KO:K10839]\n",
      "AT3G02540  RAD23C; Rad23 UV excision repair protein family [KO:K10839]\n",
      "AT5G16090  RAD23 UV excision repair family protein [KO:K10839]\n",
      "AT1G19750  Transducin/WD40 repeat-like superfamily protein [KO:K10570]\n",
      "AT1G27840  ATCSA-1; Transducin/WD40 repeat-like superfamily protein [KO:K10570]\n",
      "AT2G18760  CHR8; chromatin remodeling 8 [KO:K10841]\n",
      "AT1G18040  CDKD1;3; cyclin-dependent kinase D1;3 [KO:K02202] [EC:2.7.11.22 2.7.11.23]\n",
      "AT1G66750  CAK4; CDK-activating kinase 4 [KO:K02202] [EC:2.7.11.22 2.7.11.23]\n",
      "AT1G73690  CDKD1;1; cyclin-dependent kinase D1;1 [KO:K02202] [EC:2.7.11.22 2.7.11.23]\n",
      "AT4G30820  cyclin-dependent kinase-activating kinase assembly factor-related / CDK-activating kinase assembly factor-like protein [KO:K10842]\n",
      "AT5G27620  CYCH;1; cyclin H;1 [KO:K06634]\n",
      "AT5G41360  XPB2; homolog of Xeroderma pigmentosum complementation group B 2 [KO:K10843] [EC:3.6.4.12]\n",
      "AT5G41370  XPB1; DNA repair helicase XPB1-like protein [KO:K10843] [EC:3.6.4.12]\n",
      "AT1G03190  UVH6; RAD3-like DNA-binding helicase protein [KO:K10844] [EC:3.6.4.12]\n",
      "AT1G12400  Nucleotide excision repair, TFIIH, subunit TTDA [KO:K10845]\n",
      "AT1G62886  Nucleotide excision repair, TFIIH, subunit TTDA [KO:K10845]\n",
      "AT3G61420  BSD domain (BTF2-like transcription factors, Synapse-associated proteins and DOS2-like proteins) [KO:K03141]\n",
      "AT1G55750  BSD domain (BTF2-like transcription factors, Synapse-associated proteins and DOS2-like proteins) [KO:K03141]\n",
      "AT1G05055  GTF2H2; general transcription factor II H2 [KO:K03142]\n",
      "AT1G18340  basal transcription factor complex subunit-like protein [KO:K03143]\n",
      "AT4G17020  transcription factor-like protein [KO:K03144]\n",
      "AT3G28030  UVH3; 5'-3' exonuclease family protein [KO:K10846]\n",
      "AT2G06510  RPA1A; replication protein A 1A [KO:K07466]\n",
      "AT1G10590  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT5G45400  RPA70C; Replication factor-A protein 1-like protein [KO:K07466]\n",
      "AT1G23750  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT2G33845  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT4G28440  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT1G03810  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT5G61000  RPA70D; Replication factor-A protein 1-like protein [KO:K07466]\n",
      "AT5G08020  RPA70B; RPA70-kDa subunit B [KO:K07466]\n",
      "AT2G24490  RPA2; replicon protein A2 [KO:K10739]\n",
      "AT3G02920  RPA32B; Replication protein A, subunit RPA32 [KO:K10739]\n",
      "AT3G52630  Nucleic acid-binding, OB-fold-like protein [KO:K10740]\n",
      "AT4G18590  Nucleic acid-binding, OB-fold-like protein [KO:K10740]\n",
      "AT5G41150  UVH1; Restriction endonuclease, type II-like superfamily protein [KO:K10848] [EC:3.1.-.-]\n",
      "AT3G05210  ERCC1; nucleotide repair protein [KO:K10849]\n",
      "AT5G63960  DNA polymerase delta subunit 1 [KO:K02327] [EC:2.7.7.7]\n",
      "AT2G42120  POLD2; DNA polymerase delta small subunit [KO:K02328]\n",
      "AT1G78650  POLD3; DNA-directed DNA polymerase [KO:K03504]\n",
      "AT1G09815  POLD4; polymerase delta 4 [KO:K03505]\n",
      "AT1G08260  TIL1; DNA polymerase epsilon catalytic subunit [KO:K02324] [EC:2.7.7.7]\n",
      "AT2G27120  TIL2; DNA polymerase epsilon catalytic subunit [KO:K02324] [EC:2.7.7.7]\n",
      "AT5G22110  DPB2; DNA polymerase epsilon subunit B2 [KO:K02325] [EC:2.7.7.7]\n",
      "AT2G27470  NF-YB11; nuclear factor Y, subunit B11 [KO:K02326] [EC:2.7.7.7]\n",
      "AT2G29570  PCNA2; proliferating cell nuclear antigen 2 [KO:K04802]\n",
      "AT1G07370  PCNA1; proliferating cellular nuclear antigen 1 [KO:K04802]\n",
      "AT5G22010  RFC1; replication factor C1 [KO:K10754]\n",
      "AT1G21690  EMB1968; ATPase family associated with various cellular activities (AAA) [KO:K10755]\n",
      "AT1G63160  RFC2; replication factor C 2 [KO:K10755]\n",
      "AT1G77470  RFC3; replication factor C subunit 3 [KO:K10756]\n",
      "AT5G27740  EMB2775; ATPase family associated with various cellular activities (AAA) [KO:K10756]\n",
      "AT1G08130  LIG1; DNA ligase 1 [KO:K10747] [EC:6.5.1.1 6.5.1.6 6.5.1.7]\n",
      "AT1G49250  ATP-dependent DNA ligase [KO:K10747] [EC:6.5.1.1 6.5.1.6 6.5.1.7]\n",
      "REFERENCE   PMID:11436314\n",
      "AUTHORS   Sugasawa K.\n",
      "TITLE     [Molecular mechanism of mammalian nucleotide excision repair] Japanese\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 46:893-901 (2001)\n",
      "REFERENCE   PMID:11436316\n",
      "AUTHORS   Nakatsu Y.\n",
      "TITLE     [Transcription-coupled repair and Cockayne syndrome] Japanese\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 46:908-15 (2001)\n",
      "REFERENCE   PMID:11436323\n",
      "AUTHORS   Nakagawa N, Masui R, Kuramitsu S.\n",
      "TITLE     [Structure and function of DNA repair enzyme UvrB from Thermus thermophilus HB8] Japanese\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 46:968-75 (2001)\n",
      "REFERENCE   PMID:17276014\n",
      "AUTHORS   Kraemer KH, Patronas NJ, Schiffmann R, Brooks BP, Tamura D, DiGiovanna JJ.\n",
      "TITLE     Xeroderma pigmentosum, trichothiodystrophy and Cockayne syndrome: a complex genotype-phenotype relationship.\n",
      "JOURNAL   Neuroscience 145:1388-96 (2007)\n",
      "DOI:10.1016/j.neuroscience.2006.12.020\n",
      "REFERENCE   PMID:17884153\n",
      "AUTHORS   Saldivar JS, Wu X, Follen M, Gershenson D.\n",
      "TITLE     Nucleotide excision repair pathway review I: implications in ovarian cancer and platinum sensitivity.\n",
      "JOURNAL   Gynecol Oncol 107:S56-71 (2007)\n",
      "DOI:10.1016/j.ygyno.2007.07.043\n",
      "REFERENCE   PMID:16922398\n",
      "AUTHORS   Sugasawa K.\n",
      "TITLE     [DNA repair pathways involving Cul4A ubiquitin ligases] Japanese\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 51:1339-44 (2006)\n",
      "REFERENCE   PMID:18018633\n",
      "AUTHORS   Ito S, Ando D, Tanaka K.\n",
      "TITLE     [Cross-talk between DNA repair and transcription: molecular mechanism and disorders] Japanese\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 52:1823-31 (2007)\n",
      "REL_PATHWAY ath03020  RNA polymerase\n",
      "ath04120  Ubiquitin mediated proteolysis\n",
      "KO_PATHWAY  ko03420\n",
      "///\n",
      "ENTRY       ath03430                    Pathway\n",
      "NAME        Mismatch repair - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION DNA mismatch repair (MMR) is a highly conserved biological pathway that plays a key role in maintaining genomic stability. MMR corrects DNA mismatches generated during DNA replication, thereby preventing mutations from becoming permanent in dividing cells. MMR also suppresses homologous recombination and was recently shown to play a role in DNA damage signaling. Defects in MMR are associated with genome-wide instability, predisposition to certain types of cancer including HNPCC, resistance to certain chemotherapeutic agents, and abnormalities in meiosis and sterility in mammalian systems.\n",
      "The Escherichia coli MMR pathway has been extensively studied and is well characterized. In E. coli, the mismatch-activated MutS-MutL-ATP complex licenses MutH to incise the nearest unmethylated GATC sequence. UvrD and an exonuclease generate a gap. This gap is filled by pol III and DNA ligase. The GATC sites are then methylated by Dam. Several human MMR proteins have been identified based on their homology to E. coli MMR proteins. These include human homologs of MutS and MutL. Although E. coli MutS and MutL proteins are homodimers, human MutS and MutL homologs are heterodimers. The role of hemimethylated dGATC sites as a signal for strand discrimination is not conserved from E. coli to human. Human MMR is presumed to be nick-directed in vivo, and is thought to discriminate daughter and template strands using a strand-specific nick.\n",
      "CLASS       Genetic Information Processing; Replication and repair\n",
      "PATHWAY_MAP ath03430  Mismatch repair\n",
      "DBLINKS     GO: 0006298\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G65070  DNA mismatch repair protein MutS, type 2 [KO:K07456]\n",
      "AT3G18580  Nucleic acid-binding, OB-fold-like protein [KO:K03111]\n",
      "AT4G02460  PMS1; DNA mismatch repair protein [KO:K10858]\n",
      "AT4G09140  MLH1; MUTL-homologue 1 [KO:K08734]\n",
      "AT4G02070  MSH6; MUTS homolog 6 [KO:K08737]\n",
      "AT3G24495  MSH7; MUTS homolog 7 [KO:K08737]\n",
      "AT3G18524  MSH2; MUTS homolog 2 [KO:K08735]\n",
      "AT4G25540  MSH3; homolog of DNA mismatch repair protein MSH3 [KO:K08736]\n",
      "AT4G35520  MLH3; MUTL protein homolog 3 [KO:K08739]\n",
      "AT5G22010  RFC1; replication factor C1 [KO:K10754]\n",
      "AT1G21690  EMB1968; ATPase family associated with various cellular activities (AAA) [KO:K10755]\n",
      "AT1G63160  RFC2; replication factor C 2 [KO:K10755]\n",
      "AT1G77470  RFC3; replication factor C subunit 3 [KO:K10756]\n",
      "AT5G27740  EMB2775; ATPase family associated with various cellular activities (AAA) [KO:K10756]\n",
      "AT2G29570  PCNA2; proliferating cell nuclear antigen 2 [KO:K04802]\n",
      "AT1G07370  PCNA1; proliferating cellular nuclear antigen 1 [KO:K04802]\n",
      "AT1G29630  5'-3' exonuclease family protein [KO:K10746] [EC:3.1.-.-]\n",
      "AT1G18090  5'-3' exonuclease family protein [KO:K10746] [EC:3.1.-.-]\n",
      "AT2G06510  RPA1A; replication protein A 1A [KO:K07466]\n",
      "AT1G10590  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT5G45400  RPA70C; Replication factor-A protein 1-like protein [KO:K07466]\n",
      "AT1G23750  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT2G33845  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT4G28440  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT1G03810  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT5G61000  RPA70D; Replication factor-A protein 1-like protein [KO:K07466]\n",
      "AT5G08020  RPA70B; RPA70-kDa subunit B [KO:K07466]\n",
      "AT2G24490  RPA2; replicon protein A2 [KO:K10739]\n",
      "AT3G02920  RPA32B; Replication protein A, subunit RPA32 [KO:K10739]\n",
      "AT3G52630  Nucleic acid-binding, OB-fold-like protein [KO:K10740]\n",
      "AT4G18590  Nucleic acid-binding, OB-fold-like protein [KO:K10740]\n",
      "AT5G63960  DNA polymerase delta subunit 1 [KO:K02327] [EC:2.7.7.7]\n",
      "AT2G42120  POLD2; DNA polymerase delta small subunit [KO:K02328]\n",
      "AT1G78650  POLD3; DNA-directed DNA polymerase [KO:K03504]\n",
      "AT1G09815  POLD4; polymerase delta 4 [KO:K03505]\n",
      "AT1G08130  LIG1; DNA ligase 1 [KO:K10747] [EC:6.5.1.1 6.5.1.6 6.5.1.7]\n",
      "AT1G49250  ATP-dependent DNA ligase [KO:K10747] [EC:6.5.1.1 6.5.1.6 6.5.1.7]\n",
      "REFERENCE   PMID:16612326\n",
      "AUTHORS   Jiricny J.\n",
      "TITLE     The multifaceted mismatch-repair system.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 7:335-46 (2006)\n",
      "DOI:10.1038/nrm1907\n",
      "REFERENCE   PMID:18157157\n",
      "AUTHORS   Li GM.\n",
      "TITLE     Mechanisms and functions of DNA mismatch repair.\n",
      "JOURNAL   Cell Res 18:85-98 (2008)\n",
      "DOI:10.1038/cr.2007.115\n",
      "REFERENCE   PMID:11920679\n",
      "AUTHORS   Marti TM, Kunz C, Fleck O.\n",
      "TITLE     DNA mismatch repair and mutation avoidance pathways.\n",
      "JOURNAL   J Cell Physiol 191:28-41 (2002)\n",
      "DOI:10.1002/jcp.10077\n",
      "REFERENCE   PMID:11436301\n",
      "AUTHORS   Ikejima M, Shimada T.\n",
      "TITLE     [Molecular mechanism of mismatch repair] Japanese\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 46:1124-9 (2001)\n",
      "KO_PATHWAY  ko03430\n",
      "///\n",
      "ENTRY       ath03440                    Pathway\n",
      "NAME        Homologous recombination - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Homologous recombination (HR) is essential for the accurate repair of DNA double-strand breaks (DSBs), potentially lethal lesions. HR takes place in the late S-G2 phase of the cell cycle and involves the generation of a single-stranded region of DNA, followed by strand invasion, formation of a Holliday junction, DNA synthesis using the intact strand as a template, branch migration and resolution. It is investigated that RecA/Rad51 family proteins play a central role. The breast cancer susceptibility protein Brca2 and the RecQ helicase BLM (Bloom syndrome mutated) are tumor suppressors that maintain genome integrity, at least in part, through HR.\n",
      "CLASS       Genetic Information Processing; Replication and repair\n",
      "PATHWAY_MAP ath03440  Homologous recombination\n",
      "DBLINKS     GO: 0000724\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G18580  Nucleic acid-binding, OB-fold-like protein [KO:K03111]\n",
      "AT2G19490  RECA2; recA DNA recombination family protein [KO:K03553]\n",
      "AT1G79050  RECA1; recA DNA recombination family protein [KO:K03553]\n",
      "AT3G32920  P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K03553]\n",
      "AT1G50840  POLGAMMA2; polymerase gamma 2 [KO:K02335] [EC:2.7.7.7]\n",
      "AT3G20540  POLGAMMA1; polymerase gamma 1 [KO:K02335] [EC:2.7.7.7]\n",
      "AT2G01440  DEAD/DEAH box RNA helicase family protein [KO:K03655] [EC:3.6.4.12]\n",
      "AT2G31970  RAD50; DNA repair-recombination protein (RAD50) [KO:K10866] [EC:3.6.-.-]\n",
      "AT5G54260  MRE11; DNA repair and meiosis protein (Mre11) [KO:K10865]\n",
      "AT3G02680  NBS1; nijmegen breakage syndrome 1 [KO:K10867]\n",
      "AT3G48190  ATM; Serine/Threonine-kinase ATM-like protein [KO:K04728] [EC:2.7.11.1]\n",
      "AT1G04020  BARD1; breast cancer associated RING 1 [KO:K10683]\n",
      "AT4G21070  BRCA1; breast cancer susceptibility1 [KO:K10683]\n",
      "AT1G20720  RAD3-like DNA-binding helicase protein [KO:K15362] [EC:3.6.4.12]\n",
      "AT1G20750  RAD3-like DNA-binding helicase protein [KO:K15362] [EC:3.6.4.12]\n",
      "AT1G77320  MEI1; transcription coactivator [KO:K10728]\n",
      "AT4G02110  transcription coactivator [KO:K10728]\n",
      "AT4G32960  BRISC/BRCA1-A complex protein [KO:K20776]\n",
      "AT5G42470  BRCA1-A complex subunit BRE-like protein [KO:K12173]\n",
      "AT1G80210  BRCC36A; Mov34/MPN/PAD-1 family protein [KO:K11864] [EC:3.4.19.-]\n",
      "AT3G06820  BRCC36B; Mov34/MPN/PAD-1 family protein [KO:K11864] [EC:3.4.19.-]\n",
      "AT5G01630  BRCA2B; BRCA2-like B [KO:K08775]\n",
      "AT4G00020  BRCA2(IV); BREAST CANCER 2 like 2A [KO:K08775]\n",
      "AT1G64750  DSS1(I); deletion of SUV3 suppressor 1(I) [KO:K10881]\n",
      "AT5G45010  DSS1(V); DSS1 homolog on chromosome V [KO:K10881]\n",
      "AT2G06510  RPA1A; replication protein A 1A [KO:K07466]\n",
      "AT1G10590  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT5G45400  RPA70C; Replication factor-A protein 1-like protein [KO:K07466]\n",
      "AT1G23750  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT2G33845  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT4G28440  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT1G03810  Nucleic acid-binding, OB-fold-like protein [KO:K07466]\n",
      "AT5G61000  RPA70D; Replication factor-A protein 1-like protein [KO:K07466]\n",
      "AT5G08020  RPA70B; RPA70-kDa subunit B [KO:K07466]\n",
      "AT2G24490  RPA2; replicon protein A2 [KO:K10739]\n",
      "AT3G02920  RPA32B; Replication protein A, subunit RPA32 [KO:K10739]\n",
      "AT3G52630  Nucleic acid-binding, OB-fold-like protein [KO:K10740]\n",
      "AT4G18590  Nucleic acid-binding, OB-fold-like protein [KO:K10740]\n",
      "AT5G20850  RAD51; RAS associated with diabetes protein 51 [KO:K04482]\n",
      "AT2G28560  RAD51B; DNA repair (Rad51) family protein [KO:K10869]\n",
      "AT2G45280  RAD51C; RAS associated with diabetes protein 51C [KO:K10870]\n",
      "AT1G07745  RAD51D; DNA repair RAD51-like protein [KO:K10871]\n",
      "AT5G64520  XRCC2; homolog of X-ray repair cross complementing 2 (XRCC2) [KO:K10879]\n",
      "AT5G57450  XRCC3; homolog of X-ray repair cross complementing 3 (XRCC3) [KO:K10880]\n",
      "AT3G19210  RAD54; DNA repair/recombination protein [KO:K10875] [EC:3.6.4.-]\n",
      "AT3G24340  CHR40; chromatin remodeling 40 [KO:K10875] [EC:3.6.4.-]\n",
      "AT1G05490  chr31; chromatin remodeling 31 [KO:K10875] [EC:3.6.4.-]\n",
      "AT2G16390  DRD1; SNF2 domain-containing protein / helicase domain-containing protein [KO:K10875] [EC:3.6.4.-]\n",
      "AT2G21450  CHR34; chromatin remodeling 34 [KO:K10875] [EC:3.6.4.-]\n",
      "AT3G42670  CHR38; chromatin remodeling 38 [KO:K10875] [EC:3.6.4.-]\n",
      "AT5G20420  CHR42; chromatin remodeling 42 [KO:K10875] [EC:3.6.4.-]\n",
      "AT5G63960  DNA polymerase delta subunit 1 [KO:K02327] [EC:2.7.7.7]\n",
      "AT2G42120  POLD2; DNA polymerase delta small subunit [KO:K02328]\n",
      "AT1G78650  POLD3; DNA-directed DNA polymerase [KO:K03504]\n",
      "AT1G09815  POLD4; polymerase delta 4 [KO:K03505]\n",
      "AT1G10930  RECQ4A; DNA helicase (RECQl4A) [KO:K10901] [EC:3.6.4.12]\n",
      "AT1G60930  RECQ4B; RECQ helicase L4B [KO:K10901] [EC:3.6.4.12]\n",
      "AT4G35740  RecQl3; DEAD/DEAH box RNA helicase family protein [KO:K10901] [EC:3.6.4.12]\n",
      "AT5G63920  TOP3A; topoisomerase 3alpha [KO:K03165] [EC:5.6.2.1]\n",
      "AT2G32000  DNA topoisomerase, type IA, core [KO:K03165] [EC:5.6.2.1]\n",
      "AT4G30870  MUS81; Restriction endonuclease, type II-like superfamily protein [KO:K08991] [EC:3.1.22.-]\n",
      "AT5G39770  Restriction endonuclease, type II-like superfamily protein [KO:K08991] [EC:3.1.22.-]\n",
      "AT2G22140  EME1B; essential meiotic endonuclease 1B [KO:K10882] [EC:3.1.22.-]\n",
      "AT2G21800  EME1A; essential meiotic endonuclease 1A [KO:K10882] [EC:3.1.22.-]\n",
      "REFERENCE   PMID:18086882\n",
      "AUTHORS   Maloisel L, Fabre F, Gangloff S.\n",
      "TITLE     DNA polymerase delta is preferentially recruited during homologous recombination to promote heteroduplex DNA extension.\n",
      "JOURNAL   Mol Cell Biol 28:1373-82 (2008)\n",
      "DOI:10.1128/MCB.01651-07\n",
      "REFERENCE   PMID:17571595\n",
      "AUTHORS   Haruta-Takahashi N, Iwasaki H.\n",
      "TITLE     [Recombination mediators] Japanese\n",
      "JOURNAL   Seikagaku 79:449-53 (2007)\n",
      "REFERENCE   PMID:18430459\n",
      "AUTHORS   Ouyang KJ, Woo LL, Ellis NA.\n",
      "TITLE     Homologous recombination and maintenance of genome integrity: Cancer and aging through the prism of human RecQ helicases.\n",
      "JOURNAL   Mech Ageing Dev 129:425-40 (2008)\n",
      "DOI:10.1016/j.mad.2008.03.003\n",
      "REFERENCE   PMID:15902993\n",
      "AUTHORS   Kawabata M, Kawabata T, Nishibori M.\n",
      "TITLE     Role of recA/RAD51 family proteins in mammals.\n",
      "JOURNAL   Acta Med Okayama 59:1-9 (2005)\n",
      "DOI:10.18926/AMO/31987\n",
      "REFERENCE   PMID:12589470\n",
      "AUTHORS   Prado F, Cortes-Ledesma F, Huertas P, Aguilera A.\n",
      "TITLE     Mitotic recombination in Saccharomyces cerevisiae.\n",
      "JOURNAL   Curr Genet 42:185-98 (2003)\n",
      "DOI:10.1007/s00294-002-0346-3\n",
      "REFERENCE   PMID:16935872\n",
      "AUTHORS   Heyer WD, Li X, Rolfsmeier M, Zhang XP.\n",
      "TITLE     Rad54: the Swiss Army knife of homologous recombination?\n",
      "JOURNAL   Nucleic Acids Res 34:4115-25 (2006)\n",
      "DOI:10.1093/nar/gkl481\n",
      "REFERENCE   PMID:17395553\n",
      "AUTHORS   Michel B, Boubakri H, Baharoglu Z, LeMasson M, Lestini R.\n",
      "TITLE     Recombination proteins and rescue of arrested replication forks.\n",
      "JOURNAL   DNA Repair (Amst) 6:967-80 (2007)\n",
      "DOI:10.1016/j.dnarep.2007.02.016\n",
      "KO_PATHWAY  ko03440\n",
      "///\n",
      "ENTRY       ath03450                    Pathway\n",
      "NAME        Non-homologous end-joining - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Nonhomologous end joining (NHEJ) eliminates DNA double-strand breaks (DSBs) by direct ligation. NHEJ involves binding of the KU heterodimer to double-stranded DNA ends, recruitment of DNA-PKcs (MRX complex in yeast), processing of ends, and recruitment of the DNA ligase IV (LIG4)-XRCC4 complex, which brings about ligation. A recent study shows that bacteria accomplish NHEJ using just two proteins (Ku and DNA ligase), whereas eukaryotes require many factors. NHEJ repairs DSBs at all stages of the cell cycle, bringing about the ligation of two DNA DSBs without the need for sequence homology, and so is error-prone.\n",
      "CLASS       Genetic Information Processing; Replication and repair\n",
      "PATHWAY_MAP ath03450  Non-homologous end-joining\n",
      "DBLINKS     GO: 0006303\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G16970  KU70; ATP-dependent DNA helicase 2 subunit Ku70-like protein [KO:K10884]\n",
      "AT1G48050  KU80; Ku80 family protein [KO:K10885]\n",
      "AT1G10520  Pol{lambda}; DNA polymerase lambda (POLL) [KO:K03512] [EC:2.7.7.7 4.2.99.-]\n",
      "AT5G57160  ATLIG4; DNA ligase IV [KO:K10777] [EC:6.5.1.1]\n",
      "AT3G23100  XRCC4; DNA ligase IV-binding protein [KO:K10886]\n",
      "AT2G31970  RAD50; DNA repair-recombination protein (RAD50) [KO:K10866] [EC:3.6.-.-]\n",
      "AT5G54260  MRE11; DNA repair and meiosis protein (Mre11) [KO:K10865]\n",
      "AT5G26680  5'-3' exonuclease family protein [KO:K04799] [EC:3.-.-.-]\n",
      "REFERENCE   PMID:16971555\n",
      "AUTHORS   Gennery AR.\n",
      "TITLE     Primary immunodeficiency syndromes associated with defective DNA double-strand break repair.\n",
      "JOURNAL   Br Med Bull 77-78:71-85 (2006)\n",
      "DOI:10.1093/bmb/ldl006\n",
      "REFERENCE   PMID:17589524\n",
      "AUTHORS   Zhang Y, Hefferin ML, Chen L, Shim EY, Tseng HM, Kwon Y, Sung P, Lee SE, Tomkinson AE.\n",
      "TITLE     Role of Dnl4-Lif1 in nonhomologous end-joining repair complex assembly and suppression of homologous recombination.\n",
      "JOURNAL   Nat Struct Mol Biol 14:639-46 (2007)\n",
      "DOI:10.1038/nsmb1261\n",
      "REFERENCE   PMID:16895466\n",
      "AUTHORS   Wyman C, Kanaar R.\n",
      "TITLE     DNA double-strand break repair: all's well that ends well.\n",
      "JOURNAL   Annu Rev Genet 40:363-83 (2006)\n",
      "DOI:10.1146/annurev.genet.40.110405.090451\n",
      "REFERENCE   PMID:15578053\n",
      "AUTHORS   Weller GR, Brandt VL, Roth DB.\n",
      "TITLE     Doing more with less in bacterial DNA repair.\n",
      "JOURNAL   Nat Struct Mol Biol 11:1158-9 (2004)\n",
      "DOI:10.1038/nsmb1204-1158\n",
      "REFERENCE   PMID:12215643\n",
      "AUTHORS   Weller GR, Kysela B, Roy R, Tonkin LM, Scanlan E, Della M, Devine SK, Day JP, Wilkinson A, d'Adda di Fagagna F, Devine KM, Bowater RP, Jeggo PA, Jackson SP, Doherty AJ.\n",
      "TITLE     Identification of a DNA nonhomologous end-joining complex in bacteria.\n",
      "JOURNAL   Science 297:1686-9 (2002)\n",
      "DOI:10.1126/science.1074584\n",
      "REFERENCE   PMID:17506672\n",
      "AUTHORS   Pitcher RS, Brissett NC, Doherty AJ.\n",
      "TITLE     Nonhomologous end-joining in bacteria: a microbial perspective.\n",
      "JOURNAL   Annu Rev Microbiol 61:259-82 (2007)\n",
      "DOI:10.1146/annurev.micro.61.080706.093354\n",
      "KO_PATHWAY  ko03450\n",
      "///\n",
      "ENTRY       ath04016                    Pathway\n",
      "NAME        MAPK signaling pathway - plant - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Mitogen-activated protein kinase (MAPK) cascades, the evolutionarily conserved signaling modules in all eukaryotes, serve in transducing extracellular signals to the nucleus or cytoplasm for appropriate cellular responses, including cell division, differentiation, programmed cell death, and adaption to various stresses. In plant, several cascades are triggered by different biotic and abiotic stress stimuli such as pathogen infection, wounding, low temperature, drought, osmotic shock, high salinity, and reactive oxygen species. MAPK pathways also play pivotal roles in processes such as hormonal and developmental signaling. Multiple functional pathways initiated from distinct receptors often share the same kinase components while maintaining their signaling specificity. In contrast to those of animals and fungi, all the plant MAPK genes that have been reported belong to extracellular signal-regulated kinase (ERK) subfamily.\n",
      "CLASS       Environmental Information Processing; Signal transduction\n",
      "PATHWAY_MAP ath04016  MAPK signaling pathway - plant\n",
      "DBLINKS     GO: 0000165\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G46330  FLS2; Leucine-rich receptor-like protein kinase family protein [KO:K13420] [EC:2.7.11.1]\n",
      "AT4G33430  BAK1; BRI1-associated receptor kinase [KO:K13416] [EC:2.7.10.1 2.7.11.1]\n",
      "AT4G08500  MEKK1; MAPK/ERK kinase kinase 1 [KO:K13414] [EC:2.7.11.25]\n",
      "AT4G26070  MEK1; MAP kinase/ ERK kinase 1 [KO:K20602] [EC:2.7.12.2]\n",
      "AT4G29810  MKK2; MAP kinase kinase 2 [KO:K20603] [EC:2.7.12.2]\n",
      "AT4G01370  MPK4; MAP kinase 4 [KO:K20600] [EC:2.7.11.24]\n",
      "AT4G08480  MAPKKK9; mitogen-activated protein kinase kinase kinase 9 [KO:K20605] [EC:2.7.11.25]\n",
      "AT1G12280  SUMM2; LRR and NB-ARC domains-containing disease resistance protein [KO:K20599]\n",
      "AT3G18690  MKS1; MAP kinase substrate 1 [KO:K20725]\n",
      "AT2G38470  WRKY33; WRKY DNA-binding protein 33 [KO:K13424]\n",
      "AT3G26830  PAD3; Cytochrome P450 superfamily protein [KO:K20559] [EC:1.14.19.52]\n",
      "AT1G51660  MKK4; mitogen-activated protein kinase kinase 4 [KO:K13413] [EC:2.7.12.2]\n",
      "AT3G21220  MKK5; MAP kinase kinase 5 [KO:K13413] [EC:2.7.12.2]\n",
      "AT3G45640  MPK3; mitogen-activated protein kinase 3 [KO:K20536] [EC:2.7.11.24]\n",
      "AT2G43790  MPK6; MAP kinase 6 [KO:K14512] [EC:2.7.11.24]\n",
      "AT3G59790  MPK10; MAP kinase 10 [KO:K14512] [EC:2.7.11.24]\n",
      "AT1G43700  VIP1; VIRE2-interacting protein 1 [KO:K20557]\n",
      "AT2G14610  PR1; pathogenesis-related protein 1 [KO:K13449]\n",
      "AT4G33720  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT1G50060  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT2G14580  PRB1; basic pathogenesis-related protein 1 [KO:K13449]\n",
      "AT3G19690  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT2G19990  PR-1-LIKE; pathogenesis-related protein-1-like protein [KO:K13449]\n",
      "AT4G33730  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT4G33710  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT4G01250  WRKY22; WRKY family transcription factor [KO:K13425]\n",
      "AT4G23550  WRKY29; WRKY family transcription factor [KO:K13426]\n",
      "AT2G19190  FRK1; FLG22-induced receptor-like kinase 1 [KO:K16224]\n",
      "AT4G11280  ACS6; 1-aminocyclopropane-1-carboxylic acid (acc) synthase 6 [KO:K20772] [EC:4.4.1.14]\n",
      "AT1G01480  ACS2; 1-amino-cyclopropane-1-carboxylate synthase 2 [KO:K20772] [EC:4.4.1.14]\n",
      "AT3G61510  ACS1; ACC synthase 1 [KO:K20772] [EC:4.4.1.14]\n",
      "AT3G25250  AGC2-1; AGC (cAMP-dependent, cGMP-dependent and protein kinase C) kinase family protein [KO:K20714] [EC:2.7.11.1]\n",
      "AT4G13000  AGC (cAMP-dependent, cGMP-dependent and protein kinase C) kinase family protein [KO:K20714] [EC:2.7.11.1]\n",
      "AT4G23900  Nucleoside diphosphate kinase family protein [KO:K00940] [EC:2.7.4.6]\n",
      "AT5G63310  NDPK2; nucleoside diphosphate kinase 2 [KO:K00940] [EC:2.7.4.6]\n",
      "AT4G11010  NDPK3; nucleoside diphosphate kinase 3 [KO:K00940] [EC:2.7.4.6]\n",
      "AT4G09320  NDPK1; nucleoside diphosphate kinase [KO:K00940] [EC:2.7.4.6]\n",
      "AT1G09000  NP1; NPK1-related protein kinase 1 [KO:K20606] [EC:2.7.11.25]\n",
      "AT1G54960  NP2; NPK1-related protein kinase 2 [KO:K20606] [EC:2.7.11.25]\n",
      "AT2G30250  WRKY25; WRKY DNA-binding protein 25 [KO:K13423]\n",
      "AT5G40440  MKK3; mitogen-activated protein kinase kinase 3 [KO:K20607] [EC:2.7.12.2]\n",
      "AT1G10210  MPK1; mitogen-activated protein kinase 1 [KO:K20535] [EC:2.7.11.24]\n",
      "AT1G59580  MPK2; mitogen-activated protein kinase homolog 2 [KO:K20535] [EC:2.7.11.24]\n",
      "AT4G36450  MPK14; mitogen-activated protein kinase 14 [KO:K20537] [EC:2.7.11.24]\n",
      "AT2G18170  MPK7; MAP kinase 7 [KO:K20537] [EC:2.7.11.24]\n",
      "AT1G04310  ERS2; ethylene response sensor 2 [KO:K14509] [EC:2.7.13.-]\n",
      "AT1G66340  ETR1; Signal transduction histidine kinase, hybrid-type, ethylene sensor [KO:K14509] [EC:2.7.13.-]\n",
      "AT2G40940  ERS1; ethylene response sensor 1 [KO:K14509] [EC:2.7.13.-]\n",
      "AT3G04580  EIN4; Signal transduction histidine kinase, hybrid-type, ethylene sensor [KO:K14509] [EC:2.7.13.-]\n",
      "AT3G23150  ETR2; Signal transduction histidine kinase, hybrid-type, ethylene sensor [KO:K14509] [EC:2.7.13.-]\n",
      "AT5G03730  CTR1; Protein kinase superfamily protein [KO:K14510] [EC:2.7.11.1]\n",
      "AT2G26070  RTE1; protein REVERSION-TO-ETHYLENE SENSITIVITY-like protein (DUF778) [KO:K20726]\n",
      "AT3G51040  RTH; RTE1-homolog [KO:K20726]\n",
      "AT5G44790  RAN1; copper-exporting ATPase / responsive-to-antagonist 1 / copper-transporting ATPase (RAN1) [KO:K17686] [EC:7.2.2.8]\n",
      "AT1G63440  HMA5; heavy metal atpase 5 [KO:K17686] [EC:7.2.2.8]\n",
      "AT5G21930  PAA2; P-type ATPase of Arabidopsis 2 [KO:K17686] [EC:7.2.2.8]\n",
      "AT4G33520  PAA1; P-type ATP-ase 1 [KO:K17686] [EC:7.2.2.8]\n",
      "AT1G73500  MKK9; MAP kinase kinase 9 [KO:K20604] [EC:2.7.12.2]\n",
      "AT3G20770  EIN3; Ethylene insensitive 3 family protein [KO:K14514]\n",
      "AT2G27050  EIL1; ETHYLENE-INSENSITIVE3-like 1 [KO:K14514]\n",
      "AT1G73730  EIL3; ETHYLENE-INSENSITIVE3-like 3 [KO:K14514]\n",
      "AT5G10120  Ethylene insensitive 3 family protein [KO:K14514]\n",
      "AT5G21120  EIL2; ETHYLENE-INSENSITIVE3-like 2 [KO:K14514]\n",
      "AT5G65100  Ethylene insensitive 3 family protein [KO:K14514]\n",
      "AT5G03280  EIN2; NRAMP metal ion transporter family protein [KO:K14513]\n",
      "AT2G25490  EBF1; EIN3-binding F box protein 1 [KO:K14515]\n",
      "AT5G25350  EBF2; EIN3-binding F box protein 2 [KO:K14515]\n",
      "AT1G54490  XRN4; exoribonuclease 4 [KO:K20553] [EC:3.1.13.-]\n",
      "AT3G23240  ERF1; ethylene response factor 1 [KO:K14516]\n",
      "AT2G26020  PDF1.2b; plant defensin 1.2b [KO:K20727]\n",
      "AT5G44430  PDF1.2c; plant defensin 1.2C [KO:K20727]\n",
      "AT5G44420  PDF1.2; plant defensin 1.2 [KO:K20727]\n",
      "AT3G12500  HCHIB; basic chitinase [KO:K20547] [EC:3.2.1.14]\n",
      "AT1G32640  MYC2; Basic helix-loop-helix (bHLH) DNA-binding family protein [KO:K13422]\n",
      "AT5G24770  VSP2; vegetative storage protein 2 [KO:K20728]\n",
      "AT2G38310  PYL4; PYR1-like 4 [KO:K14496]\n",
      "AT2G40330  PYL6; PYR1-like 6 [KO:K14496]\n",
      "AT4G01026  PYL7; PYR1-like 7 [KO:K14496]\n",
      "AT4G17870  PYR1; Polyketide cyclase/dehydrase and lipid transport superfamily protein [KO:K14496]\n",
      "AT4G18620  PYL13; PYR1-like 13 [KO:K14496]\n",
      "AT4G27920  PYL10; PYR1-like 10 [KO:K14496]\n",
      "AT5G05440  PYL5; Polyketide cyclase/dehydrase and lipid transport superfamily protein [KO:K14496]\n",
      "AT5G45860  PYL11; PYR1-like 11 [KO:K14496]\n",
      "AT5G45870  PYL12; PYR1-like 12 [KO:K14496]\n",
      "AT5G46790  PYL1; PYR1-like 1 [KO:K14496]\n",
      "AT5G53160  RCAR3; regulatory components of ABA receptor 3 [KO:K14496]\n",
      "AT1G01360  RCAR1; regulatory component of ABA receptor 1 [KO:K14496]\n",
      "AT1G73000  PYL3; PYR1-like 3 [KO:K14496]\n",
      "AT2G26040  PYL2; PYR1-like 2 [KO:K14496]\n",
      "AT1G07430  HAI2; highly ABA-induced PP2C protein 2 [KO:K14497] [EC:3.1.3.16]\n",
      "AT1G17550  HAB2; homology to ABI2 [KO:K14497] [EC:3.1.3.16]\n",
      "AT2G29380  HAI3; highly ABA-induced PP2C protein 3 [KO:K14497] [EC:3.1.3.16]\n",
      "AT3G11410  PP2CA; protein phosphatase 2CA [KO:K14497] [EC:3.1.3.16]\n",
      "AT4G26080  ABI1; Protein phosphatase 2C family protein [KO:K14497] [EC:3.1.3.16]\n",
      "AT5G51760  AHG1; Protein phosphatase 2C family protein [KO:K14497] [EC:3.1.3.16]\n",
      "AT5G57050  ABI2; Protein phosphatase 2C family protein [KO:K14497] [EC:3.1.3.16]\n",
      "AT5G59220  HAI1; PP2C protein (Clade A protein phosphatases type 2C) [KO:K14497] [EC:3.1.3.16]\n",
      "AT1G72770  HAB1; HYPERSENSITIVE TO ABA1 [KO:K14497] [EC:3.1.3.16]\n",
      "AT1G10940  SNRK2.4; Protein kinase superfamily protein [KO:K14498] [EC:2.7.11.1]\n",
      "AT1G60940  SNRK2.10; SNF1-related protein kinase 2.10 [KO:K14498] [EC:2.7.11.1]\n",
      "AT1G78290  SNRK2-8; Protein kinase superfamily protein [KO:K14498] [EC:2.7.11.1]\n",
      "AT2G23030  SNRK2.9; SNF1-related protein kinase 2.9 [KO:K14498] [EC:2.7.11.1]\n",
      "AT3G50500  SNRK2.2; SNF1-related protein kinase 2.2 [KO:K14498] [EC:2.7.11.1]\n",
      "AT4G33950  OST1; Protein kinase superfamily protein [KO:K14498] [EC:2.7.11.1]\n",
      "AT4G40010  SNRK2.7; SNF1-related protein kinase 2.7 [KO:K14498] [EC:2.7.11.1]\n",
      "AT5G08590  SNRK2.1; SNF1-related protein kinase 2.1 [KO:K14498] [EC:2.7.11.1]\n",
      "AT5G63650  SNRK2.5; SNF1-related protein kinase 2.5 [KO:K14498] [EC:2.7.11.1]\n",
      "AT5G66880  SNRK2.3; sucrose nonfermenting 1(SNF1)-related protein kinase 2.3 [KO:K14498] [EC:2.7.11.1]\n",
      "AT2G32510  MAPKKK17; mitogen-activated protein kinase kinase kinase 17 [KO:K20716]\n",
      "AT1G05100  MAPKKK18; mitogen-activated protein kinase kinase kinase 18 [KO:K20716]\n",
      "AT1G20630  CAT1; catalase 1 [KO:K03781] [EC:1.11.1.6]\n",
      "AT4G35090  CAT2; catalase 2 [KO:K03781] [EC:1.11.1.6]\n",
      "AT1G20620  CAT3; catalase 3 [KO:K03781] [EC:1.11.1.6]\n",
      "AT3G06110  MKP2; MAPK phosphatase 2 [KO:K20551] [EC:3.1.3.16 3.1.3.48]\n",
      "AT1G18150  ATMPK8; Protein kinase superfamily protein [KO:K20538] [EC:2.7.11.24]\n",
      "AT5G37780  CAM1; calmodulin 1 [KO:K02183]\n",
      "AT3G43810  CAM7; calmodulin 7 [KO:K02183]\n",
      "AT3G56800  CAM3; calmodulin 3 [KO:K02183]\n",
      "AT1G66410  CAM4; calmodulin 4 [KO:K02183]\n",
      "AT2G41110  CAM2; calmodulin 2 [KO:K02183]\n",
      "AT2G27030  CAM5; calmodulin 5 [KO:K02183]\n",
      "AT5G21274  CAM6; calmodulin 6 [KO:K02183]\n",
      "AT2G41090  Calcium-binding EF-hand family protein [KO:K02183]\n",
      "AT5G49480  CP1; Ca2+-binding protein 1 [KO:K02183]\n",
      "AT1G64060  RBOH_F; respiratory burst oxidase protein F [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT5G07390  RBOHA; respiratory burst oxidase homolog A [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT5G47910  RBOHD; respiratory burst oxidase homologue D [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT5G51060  RHD2; NADPH/respiratory burst oxidase protein D [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT4G25090  Riboflavin synthase-like superfamily protein [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT4G11230  Riboflavin synthase-like superfamily protein [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT1G19230  Riboflavin synthase-like superfamily protein [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT1G09090  RBOHB; respiratory burst oxidase-like protein [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT1G34245  EPF2; Putative membrane lipoprotein [KO:K20729]\n",
      "AT2G20875  EPF1; epidermal patterning factor 1 [KO:K20729]\n",
      "AT5G62230  ERL1; ERECTA-like 1 [KO:K20718] [EC:2.7.11.1]\n",
      "AT2G26330  ER; Leucine-rich receptor-like protein kinase family protein [KO:K20718] [EC:2.7.11.1]\n",
      "AT5G07180  ERL2; ERECTA-like 2 [KO:K20718] [EC:2.7.11.1]\n",
      "AT1G63700  YDA; Protein kinase superfamily protein [KO:K20717] [EC:2.7.11.25]\n",
      "AT5G53210  SPCH; basic helix-loop-helix (bHLH) DNA-binding superfamily protein [KO:K20558]\n",
      "REFERENCE   PMID:11875555\n",
      "AUTHORS   Asai T, Tena G, Plotnikova J, Willmann MR, Chiu WL, Gomez-Gomez L, Boller T, Ausubel FM, Sheen J\n",
      "TITLE     MAP kinase signalling cascade in Arabidopsis innate immunity.\n",
      "JOURNAL   Nature 415:977-83 (2002)\n",
      "DOI:10.1038/415977a\n",
      "REFERENCE   PMID:12119167\n",
      "AUTHORS\n",
      "TITLE     Mitogen-activated protein kinase cascades in plants: a new nomenclature.\n",
      "JOURNAL   Trends Plant Sci 7:301-8 (2002)\n",
      "DOI:10.1016/S1360-1385(02)02302-6\n",
      "REFERENCE   PMID:18570633\n",
      "AUTHORS   Colcombet J, Hirt H\n",
      "TITLE     Arabidopsis MAPKs: a complex signalling network involved in multiple biological processes.\n",
      "JOURNAL   Biochem J 413:217-26 (2008)\n",
      "DOI:10.1042/BJ20080625\n",
      "REFERENCE   PMID:18692429\n",
      "AUTHORS   Kendrick MD, Chang C\n",
      "TITLE     Ethylene signaling: new levels of complexity and regulation.\n",
      "JOURNAL   Curr Opin Plant Biol 11:479-85 (2008)\n",
      "DOI:10.1016/j.pbi.2008.06.011\n",
      "REFERENCE   PMID:20441529\n",
      "AUTHORS   Rodriguez MC, Petersen M, Mundy J\n",
      "TITLE     Mitogen-activated protein kinase signaling in plants.\n",
      "JOURNAL   Annu Rev Plant Biol 61:621-49 (2010)\n",
      "DOI:10.1146/annurev-arplant-042809-112252\n",
      "REFERENCE   PMID:20980831\n",
      "AUTHORS   Taj G, Agarwal P, Grant M, Kumar A\n",
      "TITLE     MAPK machinery in plants: recognition and response to different stresses through multiple signal transduction pathways.\n",
      "JOURNAL   Plant Signal Behav 5:1370-8 (2010)\n",
      "DOI:10.4161/psb.5.11.13020\n",
      "REFERENCE   PMID:23263767\n",
      "AUTHORS   Meng X, Wang H, He Y, Liu Y, Walker JC, Torii KU, Zhang S\n",
      "TITLE     A MAPK cascade downstream of ERECTA receptor-like protein kinase regulates Arabidopsis inflorescence architecture by promoting localized cell proliferation.\n",
      "JOURNAL   Plant Cell 24:4948-60 (2012)\n",
      "DOI:10.1105/tpc.112.104695\n",
      "REFERENCE   PMID:25720833\n",
      "AUTHORS   Danquah A, de Zelicourt A, Boudsocq M, Neubauer J, Frei Dit Frey N, Leonhardt N, Pateyron S, Gwinner F, Tamby JP, Ortiz-Masia D, Marcote MJ, Hirt H, Colcombet J\n",
      "TITLE     Identification and characterization of an ABA-activated MAP kinase cascade in Arabidopsis thaliana.\n",
      "JOURNAL   Plant J 82:232-44 (2015)\n",
      "DOI:10.1111/tpj.12808\n",
      "KO_PATHWAY  ko04016\n",
      "///\n",
      "ENTRY       ath04070                    Pathway\n",
      "NAME        Phosphatidylinositol signaling system - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Environmental Information Processing; Signal transduction\n",
      "PATHWAY_MAP ath04070  Phosphatidylinositol signaling system\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G49340  ATPI4K_ALPHA; Phosphatidylinositol 3- and 4-kinase family protein [KO:K00888] [EC:2.7.1.67]\n",
      "AT1G51040  Protein kinase superfamily protein [KO:K00888] [EC:2.7.1.67]\n",
      "AT5G09350  PI-4KBETA2; phosphatidylinositol 4-OH kinase beta2 [KO:K19801] [EC:2.7.1.67]\n",
      "AT5G64070  PI-4KBETA1; phosphatidylinositol 4-OH kinase beta1 [KO:K19801] [EC:2.7.1.67]\n",
      "AT3G51460  RHD4; Phosphoinositide phosphatase family protein [KO:K21797] [EC:3.1.3.-]\n",
      "AT5G66020  ATSAC1B; Phosphoinositide phosphatase family protein [KO:K21797] [EC:3.1.3.-]\n",
      "AT3G51830  SAC8; SAC domain-containing protein 8 [KO:K21797] [EC:3.1.3.-]\n",
      "AT2G26420  PIP5K3; 1-phosphatidylinositol-4-phosphate 5-kinase 3 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G10900  Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G21980  PIP5K1; phosphatidylinositol-4-phosphate 5-kinase 1 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G60890  Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G77740  PIP5K2; phosphatidylinositol-4-phosphate 5-kinase 2 [KO:K00889] [EC:2.7.1.68]\n",
      "AT2G41210  PIP5K5; phosphatidylinositol- 4-phosphate 5-kinase 5 [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G07960  PIP5K6; Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G56960  PIP5K4; phosphatidyl inositol monophosphate 5 kinase 4 [KO:K00889] [EC:2.7.1.68]\n",
      "AT4G01190  PIPK10; phosphatidylinositol phosphate kinase 10 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G01460  PIPK11; Phosphatidylinositol-4-phosphate 5-kinase, core [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G09920  PIP5K9; phosphatidyl inositol monophosphate 5 kinase [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G19420  PEN2; PTEN 2 [KO:K01110] [EC:3.1.3.16 3.1.3.48 3.1.3.67]\n",
      "AT3G50110  PEN3; PTEN 3 [KO:K01110] [EC:3.1.3.16 3.1.3.48 3.1.3.67]\n",
      "AT5G39400  PTEN1; Calcium/lipid-binding (CaLB) phosphatase [KO:K01110] [EC:3.1.3.16 3.1.3.48 3.1.3.67]\n",
      "AT4G33240  FAB1A; 1-phosphatidylinositol-3-phosphate 5-kinase FAB1A [KO:K00921] [EC:2.7.1.150]\n",
      "AT3G14270  FAB1B; phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00921] [EC:2.7.1.150]\n",
      "AT1G34260  FAB1D; FORMS APLOID AND BINUCLEATE CELLS 1A [KO:K00921] [EC:2.7.1.150]\n",
      "AT1G71010  FAB1C; FORMS APLOID AND BINUCLEATE CELLS 1C [KO:K00921] [EC:2.7.1.150]\n",
      "AT5G04540  MTM2; Myotubularin-like phosphatases II superfamily [KO:K18081] [EC:3.1.3.64 3.1.3.95]\n",
      "AT3G10550  MTM1; Myotubularin-like phosphatases II superfamily [KO:K18081] [EC:3.1.3.64 3.1.3.95]\n",
      "AT1G60490  VPS34; vacuolar protein sorting 34 [KO:K00914] [EC:2.7.1.137]\n",
      "AT2G40116  Phosphoinositide-specific phospholipase C family protein [KO:K05857] [EC:3.1.4.11]\n",
      "AT3G08510  PLC2; phospholipase C 2 [KO:K05857] [EC:3.1.4.11]\n",
      "AT5G58690  PLC5; phosphatidylinositol-speciwc phospholipase C5 [KO:K05857] [EC:3.1.4.11]\n",
      "AT3G55940  Phosphoinositide-specific phospholipase C family protein [KO:K05857] [EC:3.1.4.11]\n",
      "AT5G58700  PLC4; phosphatidylinositol-speciwc phospholipase C4 [KO:K05857] [EC:3.1.4.11]\n",
      "AT5G58670  PLC1; phospholipase C1 [KO:K05857] [EC:3.1.4.11]\n",
      "AT4G38530  PLC1; phospholipase C1 [KO:K05857] [EC:3.1.4.11]\n",
      "AT5G61760  IPK2BETA; inositol polyphosphate kinase 2 beta [KO:K00915] [EC:2.7.1.140 2.7.1.151]\n",
      "AT5G07370  IPK2a; inositol polyphosphate kinase 2 alpha [KO:K00915] [EC:2.7.1.140 2.7.1.151]\n",
      "AT4G08170  Inositol 1,3,4-trisphosphate 5/6-kinase family protein [KO:K00913] [EC:2.7.1.159 2.7.1.134]\n",
      "AT5G16760  ITPK1; Inositol 1,3,4-trisphosphate 5/6-kinase family protein [KO:K00913] [EC:2.7.1.159 2.7.1.134]\n",
      "AT4G33770  Inositol 1,3,4-trisphosphate 5/6-kinase family protein [KO:K00913] [EC:2.7.1.159 2.7.1.134]\n",
      "AT5G42810  IPK1; inositol-pentakisphosphate 2-kinase 1 [KO:K10572] [EC:2.7.1.158]\n",
      "AT1G22100  Inositol-pentakisphosphate 2-kinase family protein [KO:K10572] [EC:2.7.1.158]\n",
      "AT1G59312  Inositol-pentakisphosphate 2-kinase family protein [KO:K10572] [EC:2.7.1.158]\n",
      "AT1G58643  Inositol-pentakisphosphate 2-kinase family protein [KO:K10572] [EC:2.7.1.158]\n",
      "AT1G58936  Inositol-pentakisphosphate 2-kinase family protein [KO:K10572] [EC:2.7.1.158]\n",
      "AT3G01310  Phosphoglycerate mutase-like family protein [KO:K13024] [EC:2.7.4.24]\n",
      "AT5G15070  Phosphoglycerate mutase-like family protein [KO:K13024] [EC:2.7.4.24]\n",
      "AT5G09290  Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G63980  SAL1; SAL1 phosphatase-like protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G64000  SAL2; Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT5G63990  Inositol monophosphatase family protein [KO:K15422] [EC:3.1.3.7 3.1.3.57]\n",
      "AT1G31190  IMPL1; myo-inositol monophosphatase like 1 [KO:K01092] [EC:3.1.3.25]\n",
      "AT3G02870  VTC4; Inositol monophosphatase family protein [KO:K10047] [EC:3.1.3.25 3.1.3.93]\n",
      "AT4G39120  IMPL2; inositol monophosphatase family protein [KO:K18649] [EC:3.1.3.25 3.1.3.93 3.1.3.15]\n",
      "AT5G63770  DGK2; diacylglycerol kinase 2 [KO:K00901] [EC:2.7.1.107]\n",
      "AT5G07920  DGK1; diacylglycerol kinase1 [KO:K00901] [EC:2.7.1.107]\n",
      "AT5G57690  DGK4; diacylglycerol kinase 4 [KO:K00901] [EC:2.7.1.107]\n",
      "AT2G18730  DGK3; diacylglycerol kinase 3 [KO:K00901] [EC:2.7.1.107]\n",
      "AT4G30340  DGK7; diacylglycerol kinase 7 [KO:K00901] [EC:2.7.1.107]\n",
      "AT4G28130  DGK6; diacylglycerol kinase 6 [KO:K00901] [EC:2.7.1.107]\n",
      "AT2G20900  DGK5; diacylglycerol kinase 5 [KO:K00901] [EC:2.7.1.107]\n",
      "AT4G26770  Phosphatidate cytidylyltransferase family protein [KO:K00981] [EC:2.7.7.41]\n",
      "AT1G62430  CDS1; CDP-diacylglycerol synthase 1 [KO:K00981] [EC:2.7.7.41]\n",
      "AT4G22340  CDS2; cytidinediphosphate diacylglycerol synthase 2 [KO:K00981] [EC:2.7.7.41]\n",
      "AT2G45150  CDS4; cytidinediphosphate diacylglycerol synthase 4 [KO:K00981] [EC:2.7.7.41]\n",
      "AT3G60620  CDS5; cytidinediphosphate diacylglycerol synthase 5 [KO:K00981] [EC:2.7.7.41]\n",
      "AT4G38570  PIS2; Putative CDP-diacylglycerol-inositol 3-phosphatidyltransferase 2 [KO:K00999] [EC:2.7.8.11]\n",
      "AT1G68000  PIS1; phosphatidylinositol synthase 1 [KO:K00999] [EC:2.7.8.11]\n",
      "AT5G37780  CAM1; calmodulin 1 [KO:K02183]\n",
      "AT3G43810  CAM7; calmodulin 7 [KO:K02183]\n",
      "AT3G56800  CAM3; calmodulin 3 [KO:K02183]\n",
      "AT1G66410  CAM4; calmodulin 4 [KO:K02183]\n",
      "AT2G41110  CAM2; calmodulin 2 [KO:K02183]\n",
      "AT2G27030  CAM5; calmodulin 5 [KO:K02183]\n",
      "AT5G21274  CAM6; calmodulin 6 [KO:K02183]\n",
      "AT2G41090  Calcium-binding EF-hand family protein [KO:K02183]\n",
      "AT5G49480  CP1; Ca2+-binding protein 1 [KO:K02183]\n",
      "COMPOUND    C00076  Calcium cation\n",
      "C00137  myo-Inositol\n",
      "C00269  CDP-diacylglycerol\n",
      "C00416  Phosphatidate\n",
      "C00641  1,2-Diacyl-sn-glycerol\n",
      "C01177  Inositol 1-phosphate\n",
      "C01194  1-Phosphatidyl-D-myo-inositol\n",
      "C01204  Phytic acid\n",
      "C01220  1D-myo-Inositol 1,4-bisphosphate\n",
      "C01243  1D-myo-Inositol 1,3,4-trisphosphate\n",
      "C01245  D-myo-Inositol 1,4,5-trisphosphate\n",
      "C01272  1D-myo-Inositol 1,3,4,5-tetrakisphosphate\n",
      "C01277  1-Phosphatidyl-1D-myo-inositol 4-phosphate\n",
      "C01284  1D-myo-Inositol 1,3,4,5,6-pentakisphosphate\n",
      "C03546  myo-Inositol 4-phosphate\n",
      "C04006  1D-myo-Inositol 3-phosphate\n",
      "C04062  D-myo-Inositol 1,3-bisphosphate\n",
      "C04063  D-myo-Inositol 3,4-bisphosphate\n",
      "C04477  1D-myo-Inositol 1,3,4,6-tetrakisphosphate\n",
      "C04549  1-Phosphatidyl-1D-myo-inositol 3-phosphate\n",
      "C04637  1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\n",
      "C05981  Phosphatidylinositol-3,4,5-trisphosphate\n",
      "C11174  1-Diphosinositol pentakisphosphate\n",
      "C11526  5-PP-InsP5\n",
      "C11554  1-Phosphatidyl-1D-myo-inositol 3,4-bisphosphate\n",
      "C11555  1D-myo-Inositol 1,4,5,6-tetrakisphosphate\n",
      "C11556  1-Phosphatidyl-1D-myo-inositol 3,5-bisphosphate\n",
      "C11557  1-Phosphatidyl-1D-myo-inositol 5-phosphate\n",
      "C20690  1D-myo-Inositol 1,5-bis(diphosphate) 2,3,4,6-tetrakisphosphate\n",
      "REFERENCE   PMID:19580826\n",
      "AUTHORS   Sasaki T, Takasuga S, Sasaki J, Kofuji S, Eguchi S, Yamazaki M, Suzuki A\n",
      "TITLE     Mammalian phosphoinositide kinases and phosphatases.\n",
      "JOURNAL   Prog Lipid Res 48:307-43 (2009)\n",
      "DOI:10.1016/j.plipres.2009.06.001\n",
      "REFERENCE   PMID:19439500\n",
      "AUTHORS   Shears SB\n",
      "TITLE     Diphosphoinositol polyphosphates: metabolic messengers?\n",
      "JOURNAL   Mol Pharmacol 76:236-52 (2009)\n",
      "DOI:10.1124/mol.109.055897\n",
      "REL_PATHWAY ath04144  Endocytosis\n",
      "KO_PATHWAY  ko04070\n",
      "///\n",
      "ENTRY       ath04075                    Pathway\n",
      "NAME        Plant hormone signal transduction - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Environmental Information Processing; Signal transduction\n",
      "PATHWAY_MAP ath04075  Plant hormone signal transduction\n",
      "DBLINKS     GO: 0009734 0009736 0009738 0009742 0009867 0009873 0010476\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G38120  AUX1; Transmembrane amino acid transporter family protein [KO:K13946]\n",
      "AT1G77690  LAX3; like AUX1 3 [KO:K13946]\n",
      "AT2G21050  LAX2; like AUXIN RESISTANT 2 [KO:K13946]\n",
      "AT5G01240  LAX1; like AUXIN RESISTANT 1 [KO:K13946]\n",
      "AT1G04240  SHY2; AUX/IAA transcriptional regulator family protein [KO:K14484]\n",
      "AT1G04250  AXR3; AUX/IAA transcriptional regulator family protein [KO:K14484]\n",
      "AT1G04550  IAA12; AUX/IAA transcriptional regulator family protein [KO:K14484]\n",
      "AT1G51950  IAA18; indole-3-acetic acid inducible 18 [KO:K14484]\n",
      "AT1G52830  IAA6; indole-3-acetic acid 6 [KO:K14484]\n",
      "AT3G15540  IAA19; indole-3-acetic acid inducible 19 [KO:K14484]\n",
      "AT3G23050  IAA7; indole-3-acetic acid 7 [KO:K14484]\n",
      "AT5G25890  IAA28; indole-3-acetic acid inducible 28 [KO:K14484]\n",
      "AT3G04730  IAA16; indoleacetic acid-induced protein 16 [KO:K14484]\n",
      "AT5G65670  IAA9; indole-3-acetic acid inducible 9 [KO:K14484]\n",
      "AT4G14550  IAA14; indole-3-acetic acid inducible 14 [KO:K14484]\n",
      "AT5G43700  ATAUX2-11; AUX/IAA transcriptional regulator family protein [KO:K14484]\n",
      "AT3G16500  PAP1; phytochrome-associated protein 1 [KO:K14484]\n",
      "AT4G29080  PAP2; phytochrome-associated protein 2 [KO:K14484]\n",
      "AT2G22670  IAA8; indoleacetic acid-induced protein 8 [KO:K14484]\n",
      "AT1G80390  IAA15; indole-3-acetic acid inducible 15 [KO:K14484]\n",
      "AT1G15580  IAA5; indole-3-acetic acid inducible 5 [KO:K14484]\n",
      "AT3G23030  IAA2; indole-3-acetic acid inducible 2 [KO:K14484]\n",
      "AT4G14560  IAA1; indole-3-acetic acid inducible [KO:K14484]\n",
      "AT2G46990  IAA20; indole-3-acetic acid inducible 20 [KO:K14484]\n",
      "AT4G28640  IAA11; indole-3-acetic acid inducible 11 [KO:K14484]\n",
      "AT2G33310  IAA13; auxin-induced protein 13 [KO:K14484]\n",
      "AT1G04100  IAA10; indoleacetic acid-induced protein 10 [KO:K14484]\n",
      "AT3G62100  IAA30; indole-3-acetic acid inducible 30 [KO:K14484]\n",
      "AT3G17600  IAA31; indole-3-acetic acid inducible 31 [KO:K14484]\n",
      "AT1G15050  IAA34; indole-3-acetic acid inducible 34 [KO:K14484]\n",
      "AT2G01200  IAA32; indole-3-acetic acid inducible 32 [KO:K14484]\n",
      "AT4G32280  IAA29; indole-3-acetic acid inducible 29 [KO:K14484]\n",
      "AT3G62980  TIR1; F-box/RNI-like superfamily protein [KO:K14485]\n",
      "AT1G19850  MP; Transcriptional factor B3 family protein / auxin-responsive factor AUX/IAA-like protein [KO:K14486]\n",
      "AT1G59750  ARF1; auxin response factor 1 [KO:K14486]\n",
      "AT2G33860  ETT; Transcriptional factor B3 family protein / auxin-responsive factor AUX/IAA-like protein [KO:K14486]\n",
      "AT5G20730  NPH4; Transcriptional factor B3 family protein / auxin-responsive factor AUX/IAA-like protein [KO:K14486]\n",
      "AT4G23980  ARF9; auxin response factor 9 [KO:K14486]\n",
      "AT3G61830  ARF18; auxin response factor 18 [KO:K14486]\n",
      "AT1G34310  ARF12; auxin response factor 12 [KO:K14486]\n",
      "AT1G34390  ARF22; auxin response factor 22 [KO:K14486]\n",
      "AT1G35520  ARF15; auxin response factor 15 [KO:K14486]\n",
      "AT1G35540  ARF14; auxin response factor 14 [KO:K14486]\n",
      "AT1G23160  Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT1G28130  GH3.17; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT1G48660  Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT1G48670  auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT1G59500  GH3.4; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT2G14960  GH3.1; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT2G23170  GH3.3; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT2G47750  GH3.9; putative indole-3-acetic acid-amido synthetase GH3.9 [KO:K14487]\n",
      "AT4G03400  DFL2; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT4G27260  WES1; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT4G37390  BRU6; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT5G13320  PBS3; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT5G13350  Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT5G13360  Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT5G13370  Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT5G13380  Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT5G51470  Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT5G54510  DFL1; Auxin-responsive GH3 family protein [KO:K14487]\n",
      "AT1G19830  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT1G19840  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT1G56150  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT1G75580  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT1G75590  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT1G16510  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT1G79130  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G16580  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G18010  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G21200  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G21210  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G21220  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G24400  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G28085  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G36210  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G37030  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G45210  SAUR36; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G46690  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G03820  SAUR29; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G03830  SAUR28; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G03840  SAUR27; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G03850  SAUR26; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G09870  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G12830  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G20220  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G43120  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G51200  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G53250  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G60690  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT3G61900  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G00880  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G09530  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G12410  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G13790  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G22620  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G31320  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G34750  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G34760  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G34770  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G34780  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G34790  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G34800  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G34810  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G36110  SAUR9; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G38825  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G38840  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G38850  SAUR15; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT4G38860  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G10990  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G18010  SAUR19; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G18020  SAUR20; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G18030  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G18050  SAUR22; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G18060  SAUR23; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G18080  SAUR24; SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G20810  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G50760  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G53590  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT5G66260  SAUR-like auxin-responsive protein family [KO:K14488]\n",
      "AT2G01830  WOL; CHASE domain containing histidine kinase protein [KO:K14489] [EC:2.7.13.3]\n",
      "AT1G27320  HK3; histidine kinase 3 [KO:K14489] [EC:2.7.13.3]\n",
      "AT5G35750  HK2; histidine kinase 2 [KO:K14489] [EC:2.7.13.3]\n",
      "AT1G03430  AHP5; histidine-containing phosphotransfer factor 5 [KO:K14490]\n",
      "AT1G80100  HP6; histidine phosphotransfer protein 6 [KO:K14490]\n",
      "AT3G16360  AHP4; HPT phosphotransmitter 4 [KO:K14490]\n",
      "AT3G21510  AHP1; histidine-containing phosphotransmitter 1 [KO:K14490]\n",
      "AT3G29350  AHP2; histidine-containing phosphotransmitter 2 [KO:K14490]\n",
      "AT4G04402  two-component phosphorelay mediator [KO:K14490]\n",
      "AT5G39340  AHP3; histidine-containing phosphotransmitter 3 [KO:K14490]\n",
      "AT1G49190  RR19; response regulator 19 [KO:K14491]\n",
      "AT1G67710  ARR11; response regulator 11 [KO:K14491]\n",
      "AT1G68210  APRR6; pseudo-response regulator 6 [KO:K14491]\n",
      "AT2G01760  RR14; response regulator 14 [KO:K14491]\n",
      "AT2G25180  RR12; response regulator 12 [KO:K14491]\n",
      "AT2G27070  RR13; response regulator 13 [KO:K14491]\n",
      "AT3G16857  RR1; response regulator 1 [KO:K14491]\n",
      "AT3G62670  RR20; two-component response regulator [KO:K14491]\n",
      "AT4G16110  RR2; response regulator 2 [KO:K14491]\n",
      "AT4G31920  RR10; response regulator 10 [KO:K14491]\n",
      "AT5G07210  RR21; response regulator 21 [KO:K14491]\n",
      "AT5G49240  APRR4; pseudo-response regulator 4 [KO:K14491]\n",
      "AT5G58080  RR18; response regulator 18 [KO:K14491]\n",
      "AT1G10470  ARR4; response regulator 4 [KO:K14492]\n",
      "AT1G19050  ARR7; response regulator 7 [KO:K14492]\n",
      "AT1G59940  ARR3; response regulator 3 [KO:K14492]\n",
      "AT1G74890  ARR15; response regulator 15 [KO:K14492]\n",
      "AT2G40670  RR16; response regulator 16 [KO:K14492]\n",
      "AT2G41310  RR3; response regulator 3 [KO:K14492]\n",
      "AT3G48100  RR5; response regulator 5 [KO:K14492]\n",
      "AT3G56380  RR17; response regulator 17 [KO:K14492]\n",
      "AT3G57040  ARR9; response regulator 9 [KO:K14492]\n",
      "AT5G62920  ARR6; response regulator 6 [KO:K14492]\n",
      "AT5G27320  GID1C; alpha/beta-Hydrolases superfamily protein [KO:K14493] [EC:3.-.-.-]\n",
      "AT3G05120  GID1A; alpha/beta-Hydrolases superfamily protein [KO:K14493] [EC:3.-.-.-]\n",
      "AT3G63010  GID1B; alpha/beta-Hydrolases superfamily protein [KO:K14493] [EC:3.-.-.-]\n",
      "AT4G24210  SLY1; F-box family protein [KO:K14495]\n",
      "AT1G66350  RGL1; RGA-like 1 [KO:K14494]\n",
      "AT3G03450  RGL2; RGA-like 2 [KO:K14494]\n",
      "AT5G17490  RGL3; RGA-like protein 3 [KO:K14494]\n",
      "AT2G01570  RGA1; GRAS family transcription factor family protein [KO:K14494]\n",
      "AT1G14920  GAI; GRAS family transcription factor family protein [KO:K14494]\n",
      "AT1G09530  PIF3; phytochrome interacting factor 3 [KO:K12126]\n",
      "AT2G43010  PIF4; phytochrome interacting factor 4 [KO:K16189]\n",
      "AT3G59060  PIL6; phytochrome interacting factor 3-like 6 [KO:K16189]\n",
      "AT2G38310  PYL4; PYR1-like 4 [KO:K14496]\n",
      "AT2G40330  PYL6; PYR1-like 6 [KO:K14496]\n",
      "AT4G01026  PYL7; PYR1-like 7 [KO:K14496]\n",
      "AT4G17870  PYR1; Polyketide cyclase/dehydrase and lipid transport superfamily protein [KO:K14496]\n",
      "AT4G18620  PYL13; PYR1-like 13 [KO:K14496]\n",
      "AT4G27920  PYL10; PYR1-like 10 [KO:K14496]\n",
      "AT5G05440  PYL5; Polyketide cyclase/dehydrase and lipid transport superfamily protein [KO:K14496]\n",
      "AT5G45860  PYL11; PYR1-like 11 [KO:K14496]\n",
      "AT5G45870  PYL12; PYR1-like 12 [KO:K14496]\n",
      "AT5G46790  PYL1; PYR1-like 1 [KO:K14496]\n",
      "AT5G53160  RCAR3; regulatory components of ABA receptor 3 [KO:K14496]\n",
      "AT1G01360  RCAR1; regulatory component of ABA receptor 1 [KO:K14496]\n",
      "AT1G73000  PYL3; PYR1-like 3 [KO:K14496]\n",
      "AT2G26040  PYL2; PYR1-like 2 [KO:K14496]\n",
      "AT1G07430  HAI2; highly ABA-induced PP2C protein 2 [KO:K14497] [EC:3.1.3.16]\n",
      "AT1G17550  HAB2; homology to ABI2 [KO:K14497] [EC:3.1.3.16]\n",
      "AT2G29380  HAI3; highly ABA-induced PP2C protein 3 [KO:K14497] [EC:3.1.3.16]\n",
      "AT3G11410  PP2CA; protein phosphatase 2CA [KO:K14497] [EC:3.1.3.16]\n",
      "AT4G26080  ABI1; Protein phosphatase 2C family protein [KO:K14497] [EC:3.1.3.16]\n",
      "AT5G51760  AHG1; Protein phosphatase 2C family protein [KO:K14497] [EC:3.1.3.16]\n",
      "AT5G57050  ABI2; Protein phosphatase 2C family protein [KO:K14497] [EC:3.1.3.16]\n",
      "AT5G59220  HAI1; PP2C protein (Clade A protein phosphatases type 2C) [KO:K14497] [EC:3.1.3.16]\n",
      "AT1G72770  HAB1; HYPERSENSITIVE TO ABA1 [KO:K14497] [EC:3.1.3.16]\n",
      "AT1G10940  SNRK2.4; Protein kinase superfamily protein [KO:K14498] [EC:2.7.11.1]\n",
      "AT1G60940  SNRK2.10; SNF1-related protein kinase 2.10 [KO:K14498] [EC:2.7.11.1]\n",
      "AT1G78290  SNRK2-8; Protein kinase superfamily protein [KO:K14498] [EC:2.7.11.1]\n",
      "AT2G23030  SNRK2.9; SNF1-related protein kinase 2.9 [KO:K14498] [EC:2.7.11.1]\n",
      "AT3G50500  SNRK2.2; SNF1-related protein kinase 2.2 [KO:K14498] [EC:2.7.11.1]\n",
      "AT4G33950  OST1; Protein kinase superfamily protein [KO:K14498] [EC:2.7.11.1]\n",
      "AT4G40010  SNRK2.7; SNF1-related protein kinase 2.7 [KO:K14498] [EC:2.7.11.1]\n",
      "AT5G08590  SNRK2.1; SNF1-related protein kinase 2.1 [KO:K14498] [EC:2.7.11.1]\n",
      "AT5G63650  SNRK2.5; SNF1-related protein kinase 2.5 [KO:K14498] [EC:2.7.11.1]\n",
      "AT5G66880  SNRK2.3; sucrose nonfermenting 1(SNF1)-related protein kinase 2.3 [KO:K14498] [EC:2.7.11.1]\n",
      "AT1G49720  ABF1; abscisic acid responsive element-binding factor 1 [KO:K14432]\n",
      "AT1G45249  ABF2; abscisic acid responsive elements-binding factor 2 [KO:K14432]\n",
      "AT2G36270  ABI5; Basic-leucine zipper (bZIP) transcription factor family protein [KO:K14432]\n",
      "AT2G41070  EEL; Basic-leucine zipper (bZIP) transcription factor family protein [KO:K14432]\n",
      "AT3G44460  DPBF2; Basic-leucine zipper (bZIP) transcription factor family protein [KO:K14432]\n",
      "AT3G56850  AREB3; ABA-responsive element binding protein 3 [KO:K14432]\n",
      "AT5G42910  Basic-leucine zipper (bZIP) transcription factor family protein [KO:K14432]\n",
      "AT5G44080  Basic-leucine zipper (bZIP) transcription factor family protein [KO:K14432]\n",
      "AT1G03970  GBF4; G-box binding factor 4 [KO:K14432]\n",
      "AT4G34000  ABF3; abscisic acid responsive elements-binding factor 3 [KO:K14432]\n",
      "AT3G19290  ABF4; ABRE binding factor 4 [KO:K14432]\n",
      "AT1G04310  ERS2; ethylene response sensor 2 [KO:K14509] [EC:2.7.13.-]\n",
      "AT1G66340  ETR1; Signal transduction histidine kinase, hybrid-type, ethylene sensor [KO:K14509] [EC:2.7.13.-]\n",
      "AT2G40940  ERS1; ethylene response sensor 1 [KO:K14509] [EC:2.7.13.-]\n",
      "AT3G04580  EIN4; Signal transduction histidine kinase, hybrid-type, ethylene sensor [KO:K14509] [EC:2.7.13.-]\n",
      "AT3G23150  ETR2; Signal transduction histidine kinase, hybrid-type, ethylene sensor [KO:K14509] [EC:2.7.13.-]\n",
      "AT5G03730  CTR1; Protein kinase superfamily protein [KO:K14510] [EC:2.7.11.1]\n",
      "AT1G51660  MKK4; mitogen-activated protein kinase kinase 4 [KO:K13413] [EC:2.7.12.2]\n",
      "AT3G21220  MKK5; MAP kinase kinase 5 [KO:K13413] [EC:2.7.12.2]\n",
      "AT2G43790  MPK6; MAP kinase 6 [KO:K14512] [EC:2.7.11.24]\n",
      "AT3G59790  MPK10; MAP kinase 10 [KO:K14512] [EC:2.7.11.24]\n",
      "AT5G03280  EIN2; NRAMP metal ion transporter family protein [KO:K14513]\n",
      "AT3G20770  EIN3; Ethylene insensitive 3 family protein [KO:K14514]\n",
      "AT2G27050  EIL1; ETHYLENE-INSENSITIVE3-like 1 [KO:K14514]\n",
      "AT1G73730  EIL3; ETHYLENE-INSENSITIVE3-like 3 [KO:K14514]\n",
      "AT5G10120  Ethylene insensitive 3 family protein [KO:K14514]\n",
      "AT5G21120  EIL2; ETHYLENE-INSENSITIVE3-like 2 [KO:K14514]\n",
      "AT5G65100  Ethylene insensitive 3 family protein [KO:K14514]\n",
      "AT2G25490  EBF1; EIN3-binding F box protein 1 [KO:K14515]\n",
      "AT5G25350  EBF2; EIN3-binding F box protein 2 [KO:K14515]\n",
      "AT3G23240  ERF1; ethylene response factor 1 [KO:K14516]\n",
      "AT5G47220  ERF2; ethylene responsive element binding factor 2 [KO:K14517]\n",
      "AT4G33430  BAK1; BRI1-associated receptor kinase [KO:K13416] [EC:2.7.10.1 2.7.11.1]\n",
      "AT4G39400  BRI1; Leucine-rich receptor-like protein kinase family protein [KO:K13415] [EC:2.7.10.1 2.7.11.1]\n",
      "AT5G42750  BKI1; BRI1 kinase inhibitor 1 [KO:K14499]\n",
      "AT4G00710  BSK3; BR-signaling kinase 3 [KO:K14500] [EC:2.7.11.1]\n",
      "AT5G46570  BSK2; BR-signaling kinase 2 [KO:K14500] [EC:2.7.11.1]\n",
      "AT4G35230  BSK1; BR-signaling kinase 1 [KO:K14500] [EC:2.7.11.1]\n",
      "AT1G01740  BSK4; kinase with tetratricopeptide repeat domain-containing protein [KO:K14500] [EC:2.7.11.1]\n",
      "AT5G41260  BSK8; kinase with tetratricopeptide repeat domain-containing protein [KO:K14500] [EC:2.7.11.1]\n",
      "AT1G63500  BSK7; kinase with tetratricopeptide repeat domain-containing protein [KO:K14500] [EC:2.7.11.1]\n",
      "AT1G50990  BSK11; kinase with tetratricopeptide repeat domain-containing protein [KO:K14500] [EC:2.7.11.1]\n",
      "AT2G17090  SSP; kinase with tetratricopeptide repeat domain-containing protein [KO:K14500] [EC:2.7.11.1]\n",
      "AT1G03445  BSU1; Serine/threonine protein phosphatase family protein [KO:K14501] [EC:3.1.3.16]\n",
      "AT4G18710  BIN2; Protein kinase superfamily protein [KO:K14502] [EC:2.7.11.1]\n",
      "AT1G06390  GSK1; GSK3/SHAGGY-like protein kinase 1 [KO:K14502] [EC:2.7.11.1]\n",
      "AT1G75080  BZR1; Brassinosteroid signaling positive regulator (BZR1) family protein [KO:K14503]\n",
      "AT1G19350  BES1; Brassinosteroid signaling positive regulator (BZR1) family protein [KO:K14503]\n",
      "AT4G36780  BEH2; BES1/BZR1 homolog 2 [KO:K14503]\n",
      "AT3G50750  BEH1; BES1/BZR1 homolog 1 [KO:K14503]\n",
      "AT5G57560  TCH4; Xyloglucan endotransglucosylase/hydrolase family protein [KO:K14504] [EC:2.4.1.207]\n",
      "AT5G67260  CYCD3;2; CYCLIN D3;2 [KO:K14505]\n",
      "AT4G34160  CYCD3;1; CYCLIN D3;1 [KO:K14505]\n",
      "AT3G50070  CYCD3;3; CYCLIN D3;3 [KO:K14505]\n",
      "AT2G46370  JAR1; Auxin-responsive GH3 family protein [KO:K14506] [EC:6.3.2.52]\n",
      "AT2G39940  COI1; RNI-like superfamily protein [KO:K13463]\n",
      "AT3G17860  JAZ3; jasmonate-zim-domain protein 3 [KO:K13464]\n",
      "AT1G19180  JAZ1; jasmonate-zim-domain protein 1 [KO:K13464]\n",
      "AT1G48500  JAZ4; jasmonate-zim-domain protein 4 [KO:K13464]\n",
      "AT1G17380  JAZ5; jasmonate-zim-domain protein 5 [KO:K13464]\n",
      "AT1G70700  TIFY7; TIFY domain/Divergent CCT motif family protein [KO:K13464]\n",
      "AT1G72450  JAZ6; jasmonate-zim-domain protein 6 [KO:K13464]\n",
      "AT1G74950  TIFY10B; TIFY domain/Divergent CCT motif family protein [KO:K13464]\n",
      "AT3G43440  JAZ11; jasmonate-zim-domain protein 11 [KO:K13464]\n",
      "AT5G13220  JAZ10; jasmonate-zim-domain protein 10 [KO:K13464]\n",
      "AT5G20900  JAZ12; jasmonate-zim-domain protein 12 [KO:K13464]\n",
      "AT1G32640  MYC2; Basic helix-loop-helix (bHLH) DNA-binding family protein [KO:K13422]\n",
      "AT1G64280  NPR1; regulatory protein (NPR1) [KO:K14508]\n",
      "AT2G41370  BOP2; Ankyrin repeat family protein / BTB/POZ domain-containing protein [KO:K14508]\n",
      "AT4G26120  Ankyrin repeat family protein / BTB/POZ domain-containing protein [KO:K14508]\n",
      "AT5G45110  NPR3; NPR1-like protein 3 [KO:K14508]\n",
      "AT4G19660  NPR4; NPR1-like protein 4 [KO:K14508]\n",
      "AT3G57130  BOP1; Ankyrin repeat family protein / BTB/POZ domain-containing protein [KO:K14508]\n",
      "AT5G65210  TGA1; bZIP transcription factor family protein [KO:K14431]\n",
      "AT1G22070  TGA3; transcription factor TGA3 [KO:K14431]\n",
      "AT5G06950  AHBP-1B; bZIP transcription factor family protein [KO:K14431]\n",
      "AT5G10030  TGA4; TGACG motif-binding factor 4 [KO:K14431]\n",
      "AT1G77920  TGA7; bZIP transcription factor family protein [KO:K14431]\n",
      "AT3G12250  TGA6; TGACG motif-binding factor 6 [KO:K14431]\n",
      "AT1G08320  TGA9; bZIP transcription factor family protein [KO:K14431]\n",
      "AT1G68640  PAN; bZIP transcription factor family protein [KO:K14431]\n",
      "AT5G06839  TGA10; bZIP transcription factor family protein [KO:K14431]\n",
      "AT5G06960  OBF5; OCS-element binding factor 5 [KO:K14431]\n",
      "AT2G14610  PR1; pathogenesis-related protein 1 [KO:K13449]\n",
      "AT4G33720  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT1G50060  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT2G14580  PRB1; basic pathogenesis-related protein 1 [KO:K13449]\n",
      "AT3G19690  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT2G19990  PR-1-LIKE; pathogenesis-related protein-1-like protein [KO:K13449]\n",
      "AT4G33730  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT4G33710  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "COMPOUND    C00371  Zeatin\n",
      "C00805  Salicylate\n",
      "C00859  Gibberellin A1\n",
      "C00954  Indole-3-acetate\n",
      "C02029  Dihydrozeatin\n",
      "C04083  N6-(Delta2-Isopentenyl)-adenine\n",
      "C06082  Abscisate\n",
      "C06547  Ethylene\n",
      "C08491  (-)-Jasmonic acid\n",
      "C08814  Brassinolide\n",
      "C11864  Gibberellin A4\n",
      "C18699  (-)-Jasmonoyl-L-isoleucine\n",
      "REFERENCE   PMID:19686081\n",
      "AUTHORS   Chapman EJ, Estelle M\n",
      "TITLE     Mechanism of auxin-regulated gene expression in plants.\n",
      "JOURNAL   Annu Rev Genet 43:265-85 (2009)\n",
      "DOI:10.1146/annurev-genet-102108-134148\n",
      "REFERENCE   PMID:16212609\n",
      "AUTHORS   Brenner WG, Romanov GA, Kollmer I, Burkle L, Schmulling T\n",
      "TITLE     Immediate-early and delayed cytokinin response genes of Arabidopsis thaliana identified by genome-wide expression profiling reveal novel cytokinin-sensitive processes and suggest cytokinin action through transcriptional cascades.\n",
      "JOURNAL   Plant J 44:314-33 (2005)\n",
      "DOI:10.1111/j.1365-313X.2005.02530.x\n",
      "REFERENCE   PMID:16227453\n",
      "AUTHORS   Mason MG, Mathews DE, Argyros DA, Maxwell BB, Kieber JJ, Alonso JM, Ecker JR, Schaller GE\n",
      "TITLE     Multiple type-B response regulators mediate cytokinin signal transduction in Arabidopsis.\n",
      "JOURNAL   Plant Cell 17:3007-18 (2005)\n",
      "DOI:10.1105/tpc.105.035451\n",
      "REFERENCE   PMID:19470587\n",
      "AUTHORS   Harberd NP, Belfield E, Yasumura Y\n",
      "TITLE     The angiosperm gibberellin-GID1-DELLA growth regulatory mechanism: how an \"inhibitor of an inhibitor\" enables flexible response to fluctuating environments.\n",
      "JOURNAL   Plant Cell 21:1328-39 (2009)\n",
      "DOI:10.1105/tpc.109.066969\n",
      "REFERENCE   PMID:17965273\n",
      "AUTHORS   Hirano K, Nakajima M, Asano K, Nishiyama T, Sakakibara H, Kojima M, Katoh E, Xiang H, Tanahashi T, Hasebe M, Banks JA, Ashikari M, Kitano H, Ueguchi-Tanaka M, Matsuoka M\n",
      "TITLE     The GID1-mediated gibberellin perception mechanism is conserved in the Lycophyte Selaginella moellendorffii but not in the Bryophyte Physcomitrella patens.\n",
      "JOURNAL   Plant Cell 19:3058-79 (2007)\n",
      "DOI:10.1105/tpc.107.051524\n",
      "REFERENCE   PMID:19805022\n",
      "AUTHORS   Umezawa T, Sugiyama N, Mizoguchi M, Hayashi S, Myouga F, Yamaguchi-Shinozaki K, Ishihama Y, Hirayama T, Shinozaki K\n",
      "TITLE     Type 2C protein phosphatases directly regulate abscisic acid-activated protein kinases in Arabidopsis.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 106:17588-93 (2009)\n",
      "DOI:10.1073/pnas.0907095106\n",
      "REFERENCE   PMID:15753119\n",
      "AUTHORS   Chen YF, Etheridge N, Schaller GE\n",
      "TITLE     Ethylene signal transduction.\n",
      "JOURNAL   Ann Bot 95:901-15 (2005)\n",
      "DOI:10.1093/aob/mci100\n",
      "REFERENCE   PMID:12628921\n",
      "AUTHORS   Ouaked F, Rozhon W, Lecourieux D, Hirt H\n",
      "TITLE     A MAPK pathway mediates ethylene signaling in plants.\n",
      "JOURNAL   EMBO J 22:1282-8 (2003)\n",
      "DOI:10.1093/emboj/cdg131\n",
      "REFERENCE   PMID:20192752\n",
      "AUTHORS   Kim TW, Wang ZY\n",
      "TITLE     Brassinosteroid signal transduction from receptor kinases to transcription factors.\n",
      "JOURNAL   Annu Rev Plant Biol 61:681-704 (2010)\n",
      "DOI:10.1146/annurev.arplant.043008.092057\n",
      "REFERENCE   PMID:19734888\n",
      "AUTHORS   Kim TW, Guan S, Sun Y, Deng Z, Tang W, Shang JX, Sun Y, Burlingame AL, Wang ZY\n",
      "TITLE     Brassinosteroid signal transduction from cell-surface receptor kinases to nuclear transcription factors.\n",
      "JOURNAL   Nat Cell Biol 11:1254-60 (2009)\n",
      "DOI:10.1038/ncb1970\n",
      "REFERENCE   PMID:20360743\n",
      "AUTHORS   Pauwels L, Barbero GF, Geerinck J, Tilleman S, Grunewald W, Perez AC, Chico JM, Bossche RV, Sewell J, Gil E, Garcia-Casado G, Witters E, Inze D, Long JA, De Jaeger G, Solano R, Goossens A\n",
      "TITLE     NINJA connects the co-repressor TOPLESS to jasmonate signalling.\n",
      "JOURNAL   Nature 464:788-91 (2010)\n",
      "DOI:10.1038/nature08854\n",
      "REFERENCE   PMID:18583180\n",
      "AUTHORS   Katsir L, Chung HS, Koo AJ, Howe GA\n",
      "TITLE     Jasmonate signaling: a conserved mechanism of hormone sensing.\n",
      "JOURNAL   Curr Opin Plant Biol 11:428-35 (2008)\n",
      "DOI:10.1016/j.pbi.2008.05.004\n",
      "REFERENCE   PMID:12897257\n",
      "AUTHORS   Johnson C, Boden E, Arias J\n",
      "TITLE     Salicylic acid and NPR1 induce the recruitment of trans-activating TGA factors to a defense gene promoter in Arabidopsis.\n",
      "JOURNAL   Plant Cell 15:1846-58 (2003)\n",
      "DOI:10.1105/tpc.012211\n",
      "REL_PATHWAY ath00270  Cysteine and methionine metabolism\n",
      "ath00360  Phenylalanine metabolism\n",
      "ath00380  Tryptophan metabolism\n",
      "ath00592  alpha-Linolenic acid metabolism\n",
      "ath00901  Indole alkaloid biosynthesis\n",
      "ath00902  Monoterpenoid biosynthesis\n",
      "ath00904  Diterpenoid biosynthesis\n",
      "ath00905  Brassinosteroid biosynthesis\n",
      "ath00906  Carotenoid biosynthesis\n",
      "ath00908  Zeatin biosynthesis\n",
      "ath04120  Ubiquitin mediated proteolysis\n",
      "KO_PATHWAY  ko04075\n",
      "///\n",
      "ENTRY       ath04120                    Pathway\n",
      "NAME        Ubiquitin mediated proteolysis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Protein ubiquitination plays an important role in eukaryotic cellular processes. It mainly functions as a signal for 26S proteasome dependent protein degradation. The addition of ubiquitin to proteins being degraded is performed by a reaction cascade consisting of three enzymes, named E1 (ubiquitin activating enzyme), E2 (ubiquitin conjugating enzyme), and E3 (ubiquitin ligase). Each E3 has specificity to its substrate, or proteins to be targeted by ubiquitination. Many E3s are discovered in eukaryotes and they are classified into four types: HECT type, U-box type, single RING-finger type, and multi-subunit RING-finger type. Multi-subunit RING-finger E3s are exemplified by cullin-Rbx E3s and APC/C. They consist of a RING-finger-containing subunit (RBX1 or RBX2) that functions to bind E2s, a scaffold-like cullin molecule, adaptor proteins, and a target recognizing subunit that binds substrates.\n",
      "CLASS       Genetic Information Processing; Folding, sorting and degradation\n",
      "PATHWAY_MAP ath04120  Ubiquitin mediated proteolysis\n",
      "DBLINKS     GO: 0016567\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G09790  UBQ8; ubiquitin 8 [KO:K08770]\n",
      "AT4G05320  UBQ10; polyubiquitin 10 [KO:K08770]\n",
      "AT5G20620  UBQ4; ubiquitin 4 [KO:K08770]\n",
      "AT4G02890  UBQ14; Ubiquitin family protein [KO:K08770]\n",
      "AT4G05050  UBQ11; ubiquitin 11 [KO:K08770]\n",
      "AT5G03240  UBQ3; polyubiquitin 3 [KO:K08770]\n",
      "AT5G37640  UBQ9; ubiquitin 9 [KO:K08770]\n",
      "AT1G53930  Ubiquitin-like superfamily protein [KO:K08770]\n",
      "AT1G65350  UBQ13; ubiquitin 13 [KO:K08770]\n",
      "AT2G36170  60S ribosomal protein L40-1 [KO:K02927]\n",
      "AT3G52590  UBQ1; ubiquitin extension protein 1 [KO:K02927]\n",
      "AT1G23410  Ribosomal protein S27a / Ubiquitin family protein [KO:K02977]\n",
      "AT3G62250  UBQ5; ubiquitin 5 [KO:K02977]\n",
      "AT2G47110  UBQ6; ubiquitin 6 [KO:K02977]\n",
      "AT2G30110  UBA1; ubiquitin-activating enzyme 1 [KO:K03178] [EC:6.2.1.45]\n",
      "AT5G06460  UBA_2; ubiquitin activating enzyme 2 [KO:K03178] [EC:6.2.1.45]\n",
      "AT5G50580  SAE1B; SUMO-activating enzyme 1B [KO:K10684] [EC:6.2.1.45]\n",
      "AT4G24940  SAE1A; SUMO-activating enzyme 1A [KO:K10684] [EC:6.2.1.45]\n",
      "AT5G50680  SAE1B; SUMO activating enzyme 1B [KO:K10684] [EC:6.2.1.45]\n",
      "AT2G21470  SAE2; SUMO-activating enzyme 2 [KO:K10685] [EC:6.2.1.45]\n",
      "AT5G19180  ECR1; E1 C-terminal related 1 [KO:K10686] [EC:6.2.1.64]\n",
      "AT1G14400  UBC1; ubiquitin carrier protein 1 [KO:K10573] [EC:2.3.2.23]\n",
      "AT2G02760  UBC2; ubiquiting-conjugating enzyme 2 [KO:K10573] [EC:2.3.2.23]\n",
      "AT5G62540  UBC3; ubiquitin-conjugating enzyme 3 [KO:K10573] [EC:2.3.2.23]\n",
      "AT3G20060  UBC19; ubiquitin-conjugating enzyme19 [KO:K06688] [EC:2.3.2.23]\n",
      "AT1G50490  UBC20; ubiquitin-conjugating enzyme 20 [KO:K06688] [EC:2.3.2.23]\n",
      "AT3G08700  UBC12; ubiquitin-conjugating enzyme 12 [KO:K06689] [EC:2.3.2.23]\n",
      "AT2G16740  UBC29; ubiquitin-conjugating enzyme 29 [KO:K06689] [EC:2.3.2.23]\n",
      "AT5G56150  UBC30; ubiquitin-conjugating enzyme 30 [KO:K06689] [EC:2.3.2.23]\n",
      "AT1G64230  UBC28; ubiquitin-conjugating enzyme 28 [KO:K06689] [EC:2.3.2.23]\n",
      "AT3G08690  UBC11; ubiquitin-conjugating enzyme 11 [KO:K06689] [EC:2.3.2.23]\n",
      "AT3G13550  FUS9; Ubiquitin-conjugating enzyme family protein [KO:K20217] [EC:2.3.2.23]\n",
      "AT3G46460  UBC13; ubiquitin-conjugating enzyme 13 [KO:K10575] [EC:2.3.2.23]\n",
      "AT3G55380  UBC14; ubiquitin-conjugating enzyme 14 [KO:K10575] [EC:2.3.2.23]\n",
      "AT5G59300  UBC7; ubiquitin-conjugating enzyme E2 [KO:K10575] [EC:2.3.2.23]\n",
      "AT1G63800  UBC5; ubiquitin-conjugating enzyme 5 [KO:K10576] [EC:2.3.2.23]\n",
      "AT2G46030  UBC6; ubiquitin-conjugating enzyme 6 [KO:K10576] [EC:2.3.2.23]\n",
      "AT5G41340  UBC4; ubiquitin conjugating enzyme 4 [KO:K10576] [EC:2.3.2.23]\n",
      "AT3G57870  SCE1; sumo conjugation enzyme 1 [KO:K10577]\n",
      "AT3G17000  UBC32; ubiquitin-conjugating enzyme 32 [KO:K10578] [EC:2.3.2.23]\n",
      "AT1G17280  UBC34; ubiquitin-conjugating enzyme 34 [KO:K04554] [EC:2.3.2.23]\n",
      "AT5G50430  UBC33; ubiquitin-conjugating enzyme 33 [KO:K04554] [EC:2.3.2.23]\n",
      "AT2G18600  Ubiquitin-conjugating enzyme family protein [KO:K10579] [EC:2.3.2.34]\n",
      "AT4G36800  RCE1; RUB1 conjugating enzyme 1 [KO:K10579] [EC:2.3.2.34]\n",
      "AT1G16890  UBC36; ubiquitin-conjugating enzyme 36 [KO:K10580] [EC:2.3.2.23]\n",
      "AT1G78870  UBC35; ubiquitin-conjugating enzyme 35 [KO:K10580] [EC:2.3.2.23]\n",
      "AT2G16920  UBC23; ubiquitin-conjugating enzyme 23 [KO:K10581] [EC:2.3.2.24]\n",
      "AT1G53025  Ubiquitin-conjugating enzyme family protein [KO:K10581] [EC:2.3.2.24]\n",
      "AT1G53023  Ubiquitin-conjugating enzyme family protein [KO:K10581] [EC:2.3.2.24]\n",
      "AT3G15355  UBC25; ubiquitin-conjugating enzyme 25 [KO:K10581] [EC:2.3.2.24]\n",
      "AT2G33770  PHO2; phosphate 2 [KO:K10581] [EC:2.3.2.24]\n",
      "AT5G05080  UBC22; ubiquitin-conjugating enzyme 22 [KO:K10583] [EC:2.3.2.23]\n",
      "AT1G75440  UBC16; ubiquitin-conjugating enzyme 16 [KO:K10688] [EC:2.3.2.25]\n",
      "AT1G45050  ATUBC2-1; Ubiquitin-conjugating enzyme family protein [KO:K10688] [EC:2.3.2.25]\n",
      "AT5G42990  UBC18; ubiquitin-conjugating enzyme 18 [KO:K10688] [EC:2.3.2.25]\n",
      "AT4G36410  UBC17; ubiquitin-conjugating enzyme 17 [KO:K10688] [EC:2.3.2.25]\n",
      "AT5G50870  UBC27; ubiquitin-conjugating enzyme 27 [KO:K04649] [EC:2.3.2.23]\n",
      "AT3G53090  UPL7; ubiquitin-protein ligase 7 [KO:K10588] [EC:2.3.2.26]\n",
      "AT3G17205  UPL6; ubiquitin protein ligase 6 [KO:K10589] [EC:2.3.2.26]\n",
      "AT4G38600  KAK; HECT ubiquitin protein ligase family protein KAK [KO:K10590] [EC:2.3.2.26]\n",
      "AT5G02880  UPL4; ubiquitin-protein ligase 4 [KO:K10590] [EC:2.3.2.26]\n",
      "AT5G15400  U-box domain-containing protein [KO:K10597] [EC:2.3.2.27]\n",
      "AT3G07370  CHIP; carboxyl terminus of HSC70-interacting protein [KO:K09561] [EC:2.3.2.27]\n",
      "AT5G67530  PUB49; plant U-box 49 [KO:K10598] [EC:5.2.1.8]\n",
      "AT1G04510  MAC3A; MOS4-associated complex 3A [KO:K10599] [EC:2.3.2.27]\n",
      "AT2G33340  MAC3B; MOS4-associated complex 3B [KO:K10599] [EC:2.3.2.27]\n",
      "AT3G58040  SINAT2; seven in absentia of Arabidopsis 2 [KO:K04506] [EC:2.3.2.27]\n",
      "AT2G41980  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT4G27880  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT3G61790  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT5G53360  TRAF-like superfamily protein [KO:K04506] [EC:2.3.2.27]\n",
      "AT1G66620  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT1G66630  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT1G66650  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT1G66660  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT5G37890  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT5G37900  TRAF-like superfamily protein [KO:K04506] [EC:2.3.2.27]\n",
      "AT5G37910  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT5G37930  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT5G62800  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT5G37870  Protein with RING/U-box and TRAF-like domain [KO:K04506] [EC:2.3.2.27]\n",
      "AT3G13672  TRAF-like superfamily protein [KO:K04506] [EC:2.3.2.27]\n",
      "AT2G32950  COP1; Transducin/WD40 repeat-like superfamily protein [KO:K10143] [EC:2.3.2.27]\n",
      "AT5G52250  RUP1; Transducin/WD40 repeat-like superfamily protein [KO:K10143] [EC:2.3.2.27]\n",
      "AT5G23730  RUP2; Transducin/WD40 repeat-like superfamily protein [KO:K10143] [EC:2.3.2.27]\n",
      "AT3G62970  zinc finger (C3HC4-type RING finger) family protein [KO:K10144] [EC:2.3.2.27]\n",
      "AT5G18650  CHY-type/CTCHY-type/RING-type Zinc finger protein [KO:K10144] [EC:2.3.2.27]\n",
      "AT5G22920  CHY-type/CTCHY-type/RING-type Zinc finger protein [KO:K10144] [EC:2.3.2.27]\n",
      "AT5G25560  CHY-type/CTCHY-type/RING-type Zinc finger protein [KO:K10144] [EC:2.3.2.27]\n",
      "AT5G41580  RING/U-box superfamily protein [KO:K04706] [EC:2.3.2.-]\n",
      "AT1G08910  zinc ion binding protein [KO:K04706] [EC:2.3.2.-]\n",
      "AT3G16090  Hrd1A; RING/U-box superfamily protein [KO:K10601] [EC:2.3.2.27]\n",
      "AT1G65040  Hrd1B; RING/U-box superfamily protein [KO:K10601] [EC:2.3.2.27]\n",
      "AT5G65740  zinc ion binding protein [KO:K10606] [EC:2.3.2.27]\n",
      "AT3G42830  RING/U-box superfamily protein [KO:K03868] [EC:2.3.2.32]\n",
      "AT5G20570  RBX1; RING-box 1 [KO:K03868] [EC:2.3.2.32]\n",
      "AT4G02570  CUL1; cullin 1 [KO:K03347]\n",
      "AT1G02980  CUL2; cullin 2 [KO:K03347]\n",
      "AT1G59790  Cullin family protein [KO:K03347]\n",
      "AT1G59800  Cullin family protein [KO:K03347]\n",
      "AT1G43140  Cullin family protein [KO:K03347]\n",
      "AT1G75950  SKP1; S phase kinase-associated protein 1 [KO:K03094]\n",
      "AT5G42190  E3 ubiquitin ligase SCF complex subunit SKP1/ASK1 family protein [KO:K03094]\n",
      "AT1G20140  SK4; SKP1-like 4 [KO:K03094]\n",
      "AT2G25700  SK3; SKP1-like 3 [KO:K03094]\n",
      "AT2G03170  SK14; SKP1-like 14 [KO:K03094]\n",
      "AT3G21830  SK8; SKP1-like 8 [KO:K03094]\n",
      "AT3G21850  SK9; SKP1-like 9 [KO:K03094]\n",
      "AT3G21860  SK10; SKP1-like 10 [KO:K03094]\n",
      "AT3G60010  SK13; SKP1-like 13 [KO:K03094]\n",
      "AT4G34210  SK11; SKP1-like 11 [KO:K03094]\n",
      "AT4G34470  SK12; SKP1-like 12 [KO:K03094]\n",
      "AT1G10230  SK18; E3 ubiquitin ligase SCF complex subunit SKP1/ASK1 family protein [KO:K03094]\n",
      "AT2G03160  SK19; SKP1-like 19 [KO:K03094]\n",
      "AT2G03190  SK16; SKP1-like 16 [KO:K03094]\n",
      "AT2G20160  MEO; E3 ubiquitin ligase SCF complex subunit SKP1/ASK1 family protein [KO:K03094]\n",
      "AT3G25650  SK15; SKP1-like 15 [KO:K03094]\n",
      "AT3G60020  SK5; SKP1-like 5 [KO:K03094]\n",
      "AT3G21840  SK7; SKP1-like 7 [KO:K03094]\n",
      "AT3G53060  SK6; SKP1-like 6 [KO:K03094]\n",
      "AT1G77000  SKP2B; RNI-like superfamily protein [KO:K03875]\n",
      "AT1G21410  SKP2A; F-box/RNI-like superfamily protein [KO:K03875]\n",
      "AT5G59140  BTB/POZ domain-containing protein [KO:K03872]\n",
      "AT1G26830  CUL3; cullin 3 [KO:K03869]\n",
      "AT1G69670  CUL3B; cullin 3B [KO:K03869]\n",
      "AT5G46210  CUL4; cullin4 [KO:K10609]\n",
      "AT3G46910  Cullin family protein [KO:K10609]\n",
      "AT4G12100  Cullin family protein [KO:K10609]\n",
      "AT4G05420  DDB1A; damaged DNA binding protein 1A [KO:K10610]\n",
      "AT4G21100  DDB1B; damaged DNA binding protein 1B [KO:K10610]\n",
      "AT5G58760  DDB2; damaged DNA binding 2 [KO:K10140]\n",
      "AT1G19750  Transducin/WD40 repeat-like superfamily protein [KO:K10570]\n",
      "AT1G27840  ATCSA-1; Transducin/WD40 repeat-like superfamily protein [KO:K10570]\n",
      "AT4G10180  DET1; light-mediated development protein 1 / deetiolated1 (DET1) [KO:K10571]\n",
      "AT3G05870  APC11; anaphase-promoting complex/cyclosome 11 [KO:K03358]\n",
      "AT2G04660  APC2; anaphase-promoting complex/cyclosome 2 [KO:K03349]\n",
      "AT4G33270  CDC20.1; Transducin family protein / WD-40 repeat family protein [KO:K03363]\n",
      "AT5G27945  Transducin/WD40 repeat-like superfamily protein [KO:K03363]\n",
      "AT5G26900  CDC20.4; Transducin family protein / WD-40 repeat family protein [KO:K03363]\n",
      "AT5G27080  CDC20.3; cell division cycle 20.2, cofactor of APC complex-like protein [KO:K03363]\n",
      "AT5G27570  CDC20.5; cell division cycle 20.2, cofactor of APC complex-like protein [KO:K03363]\n",
      "AT4G33260  CDC20.2; Transducin family protein / WD-40 repeat family protein [KO:K03363]\n",
      "AT4G22910  FZR2; FIZZY-related 2 [KO:K03364]\n",
      "AT4G11920  CCS52A2; cell cycle switch protein 52 A2 [KO:K03364]\n",
      "AT5G13840  FZR3; FIZZY-related 3 [KO:K03364]\n",
      "AT5G05560  EMB2771; E3 ubiquitin ligase [KO:K03348]\n",
      "AT2G20000  HBT; CDC27 family protein [KO:K03350]\n",
      "AT3G16320  CDC27a; Tetratricopeptide repeat (TPR)-like superfamily protein [KO:K03350]\n",
      "AT4G21530  APC4; Transducin/WD40 repeat-like superfamily protein [KO:K03351]\n",
      "AT1G06590  anaphase-promoting complex subunit [KO:K03352]\n",
      "AT1G78770  APC6; anaphase promoting complex 6 [KO:K03353]\n",
      "AT2G39090  APC7; tetratricopeptide repeat (TPR)-containing protein [KO:K03354]\n",
      "AT3G48150  APC8; anaphase-promoting complex subunit 8 [KO:K03355]\n",
      "AT2G18290  APC10; anaphase promoting complex 10 [KO:K03357]\n",
      "AT1G73177  BNS; bonsai [KO:K12456]\n",
      "REFERENCE   PMID:16922370 (Japanese)\n",
      "AUTHORS   Kamura T.\n",
      "TITLE     [Cullin-based E3 family] Japanese\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 51:1167-72 (2006)\n",
      "REFERENCE   PMID:16922408 (Japanese)\n",
      "AUTHORS   Matsuda N.\n",
      "TITLE     [Overview: hereditary disorders caused by dysfunction of ubiquitylation]\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 51:1402-7 (2006)\n",
      "REFERENCE   PMID:16922367 (Japanese)\n",
      "AUTHORS   Tokunaga F.\n",
      "TITLE     [E2 ubiquitin-conjugating enzymes: structures and functions]\n",
      "JOURNAL   Tanpakushitsu Kakusan Koso 51:1150-6 (2006)\n",
      "REFERENCE   PMID:16816840\n",
      "AUTHORS   Nalepa G, Rolfe M, Harper JW.\n",
      "TITLE     Drug discovery in the ubiquitin-proteasome system.\n",
      "JOURNAL   Nat Rev Drug Discov 5:596-613 (2006)\n",
      "DOI:10.1038/nrd2056\n",
      "REFERENCE   PMID:16405856\n",
      "AUTHORS   Hsu YJ, Goodman SR.\n",
      "TITLE     Spectrin and ubiquitination: a review.\n",
      "JOURNAL   Cell Mol Biol (Noisy-le-grand) Suppl 51:OL801-7 (2005)\n",
      "REFERENCE   PMID:12944364\n",
      "AUTHORS   Hatakeyama S, Nakayama KI.\n",
      "TITLE     Ubiquitylation as a quality control system for intracellular proteins.\n",
      "JOURNAL   J Biochem 134:1-8 (2003)\n",
      "DOI:10.1093/jb/mvg106\n",
      "REFERENCE   PMID:17588513\n",
      "AUTHORS   Lee J, Zhou P.\n",
      "TITLE     DCAFs, the missing link of the CUL4-DDB1 ubiquitin ligase.\n",
      "JOURNAL   Mol Cell 26:775-80 (2007)\n",
      "DOI:10.1016/j.molcel.2007.06.001\n",
      "REL_PATHWAY ath03050  Proteasome\n",
      "KO_PATHWAY  ko04120\n",
      "///\n",
      "ENTRY       ath04122                    Pathway\n",
      "NAME        Sulfur relay system - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Ubiquitin and ubiquitin-like proteins (Ubls) are signalling messengers that control many cellular functions, such as cell proliferation, apoptosis, and DNA repair. It is suggested that Ub-protein modification evolved from prokaryotic sulfurtransfer systems. Molybdenum cofactor (Moco) and thiamin are sulfur-containing cofactors whose biosynthesis includes a key sulfur transfer step that uses unique sulfur carrier proteins, MoaD and ThiS. Ubiquitin, MoaD, and ThiS are all structurally related proteins whose C-termini are activated through adenylation by homologous E1-like enzymes. s2T biosynthesis may share similar chemistry with Moco and thiamin synthesis. In Saccharomyces cerevisiae, Urm1 and Uba4 function as part of a ubl protein conjugation system, though they have sequence homology to bacterial sulfur-transfer enzymes and the ability to function in sulfur transfer.\n",
      "CLASS       Genetic Information Processing; Folding, sorting and degradation\n",
      "PATHWAY_MAP ath04122  Sulfur relay system\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT5G65720  NFS1; nitrogen fixation S (NIFS)-like 1 [KO:K04487] [EC:2.8.1.7]\n",
      "AT1G79230  MST1; mercaptopyruvate sulfurtransferase 1 [KO:K01011] [EC:2.8.1.1 2.8.1.2]\n",
      "AT1G16460  RDH2; rhodanese homologue 2 [KO:K01011] [EC:2.8.1.1 2.8.1.2]\n",
      "AT2G45695  Ubiquitin related modifier 1 [KO:K12161]\n",
      "AT3G61113  Ubiquitin related modifier 1 [KO:K12161]\n",
      "AT5G55130  CNX5; co-factor for nitrate, reductase and xanthine dehydrogenase 5 [KO:K11996] [EC:2.7.7.80 2.8.1.11]\n",
      "AT1G76170  2-thiocytidine tRNA biosynthesis protein, TtcA [KO:K14168] [EC:2.7.7.-]\n",
      "AT2G44270  ROL5; repressor of lrx1 [KO:K14168] [EC:2.7.7.-]\n",
      "AT4G35910  Adenine nucleotide alpha hydrolases-like superfamily protein [KO:K14169]\n",
      "AT4G10100  CNX7; co-factor for nitrate, reductase and xanthine dehydrogenase 7 [KO:K21232]\n",
      "AT2G43760  molybdopterin biosynthesis MoaE family protein [KO:K03635] [EC:2.8.1.12]\n",
      "AT2G31955  CNX2; cofactor of nitrate reductase and xanthine dehydrogenase 2 [KO:K03639] [EC:4.1.99.22]\n",
      "AT1G01290  CNX3; cofactor of nitrate reductase and xanthine dehydrogenase 3 [KO:K03637] [EC:4.6.1.17]\n",
      "COMPOUND    C00019  S-Adenosyl-L-methionine\n",
      "C00041  L-Alanine\n",
      "C00044  GTP\n",
      "C00097  L-Cysteine\n",
      "C00378  Thiamine\n",
      "C00868  tRNA uridine\n",
      "C00979  O-Acetyl-L-serine\n",
      "C05924  Molybdopterin\n",
      "C17322  tRNA containing 2-thiouridine\n",
      "C18237  Molybdoenzyme molybdenum cofactor\n",
      "C18239  Precursor Z\n",
      "REFERENCE   PMID:19151091\n",
      "AUTHORS   Noma A, Sakaguchi Y, Suzuki T\n",
      "TITLE     Mechanistic characterization of the sulfur-relay system for eukaryotic 2-thiouridine biogenesis at tRNA wobble positions.\n",
      "JOURNAL   Nucleic Acids Res 37:1335-52 (2009)\n",
      "DOI:10.1093/nar/gkn1023\n",
      "REFERENCE   PMID:16859499\n",
      "AUTHORS   Iyer LM, Burroughs AM, Aravind L\n",
      "TITLE     The prokaryotic antecedents of the ubiquitin-signaling system and the early evolution of ubiquitin-like beta-grasp domains.\n",
      "JOURNAL   Genome Biol 7:R60 (2006)\n",
      "DOI:10.1186/gb-2006-7-7-r60\n",
      "REFERENCE   PMID:19675644\n",
      "AUTHORS   Schwarz G, Mendel RR, Ribbe MW\n",
      "TITLE     Molybdenum cofactors, enzymes and pathways.\n",
      "JOURNAL   Nature 460:839-47 (2009)\n",
      "DOI:10.1038/nature08302\n",
      "REL_PATHWAY ath00270  Cysteine and methionine metabolism\n",
      "ath00730  Thiamine metabolism\n",
      "ath00790  Folate biosynthesis\n",
      "ath04120  Ubiquitin mediated proteolysis\n",
      "KO_PATHWAY  ko04122\n",
      "///\n",
      "ENTRY       ath04130                    Pathway\n",
      "NAME        SNARE interactions in vesicular transport - Arabidopsis thaliana (thale cress)\n",
      "CLASS       Genetic Information Processing; Folding, sorting and degradation\n",
      "PATHWAY_MAP ath04130  SNARE interactions in vesicular transport\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G08560  SYP111; syntaxin of plants 111 [KO:K08486]\n",
      "AT1G11250  SYP125; syntaxin of plants 125 [KO:K08486]\n",
      "AT1G61290  SYP124; syntaxin of plants 124 [KO:K08486]\n",
      "AT2G18260  SYP112; syntaxin of plants 112 [KO:K08486]\n",
      "AT3G03800  SYP131; syntaxin of plants 131 [KO:K08486]\n",
      "AT3G11820  SYP121; syntaxin of plants 121 [KO:K08486]\n",
      "AT3G52400  SYP122; syntaxin of plants 122 [KO:K08486]\n",
      "AT4G03330  SYP123; syntaxin of plants 123 [KO:K08486]\n",
      "AT5G08080  SYP132; syntaxin of plants 132 [KO:K08486]\n",
      "AT3G05710  SYP43; syntaxin of plants 43 [KO:K08489]\n",
      "AT4G02195  SYP42; syntaxin of plants 42 [KO:K08489]\n",
      "AT5G26980  SYP41; syntaxin of plants 41 [KO:K08489]\n",
      "AT3G24350  SYP32; syntaxin of plants 32 [KO:K08490]\n",
      "AT5G05760  SYP31; syntaxin of plants 31 [KO:K08490]\n",
      "AT1G51740  SYP81; syntaxin of plants 81 [KO:K08492]\n",
      "AT1G26670  VTI1B; Vesicle transport v-SNARE family protein [KO:K08493]\n",
      "AT3G29100  VTI13; vesicle transport V-snare 13 [KO:K08493]\n",
      "AT5G39510  SGR4; Vesicle transport v-SNARE family protein [KO:K08493]\n",
      "AT5G39630  Vesicle transport v-SNARE family protein [KO:K08493]\n",
      "AT1G15880  GOS11; golgi snare 11 [KO:K08495]\n",
      "AT2G45200  GOS12; golgi snare 12 [KO:K08495]\n",
      "AT2G36900  MEMB11; membrin 11 [KO:K08496]\n",
      "AT5G50440  MEMB12; membrin 12 [KO:K08496]\n",
      "AT3G24315  AtSec20; Sec20 family protein [KO:K08497]\n",
      "AT1G28490  SYP61; syntaxin of plants 61 [KO:K08500]\n",
      "AT1G16230  Target SNARE coiled-coil domain protein [KO:K08503]\n",
      "AT1G16240  SYP51; syntaxin of plants 51 [KO:K08503]\n",
      "AT1G79590  SYP52; syntaxin of plants 52 [KO:K08503]\n",
      "AT1G16225  Target SNARE coiled-coil domain protein [KO:K08503]\n",
      "AT3G58170  BS14A; BET1P/SFT1P-like protein 14A [KO:K08504]\n",
      "AT4G14455  ATBET12; Target SNARE coiled-coil domain protein [KO:K08504]\n",
      "AT1G29060  Target SNARE coiled-coil domain protein [KO:K08505]\n",
      "AT4G14600  Target SNARE coiled-coil domain protein [KO:K08505]\n",
      "AT3G09740  SYP71; syntaxin of plants 71 [KO:K08506]\n",
      "AT3G45280  SYP72; syntaxin of plants 72 [KO:K08506]\n",
      "AT3G61450  SYP73; syntaxin of plants 73 [KO:K08506]\n",
      "AT1G54110  Membrane fusion protein Use1 [KO:K08507]\n",
      "AT3G55600  Membrane fusion protein Use1 [KO:K08507]\n",
      "AT5G07880  SNAP29; synaptosomal-associated protein SNAP25-like 29 [KO:K08509]\n",
      "AT2G25340  VAMP712; vesicle-associated membrane protein 712 [KO:K08515]\n",
      "AT4G32150  VAMP711; vesicle-associated membrane protein 711 [KO:K08515]\n",
      "AT5G11150  VAMP713; vesicle-associated membrane protein 713 [KO:K08515]\n",
      "AT5G22360  VAMP714; vesicle-associated membrane protein 714 [KO:K08515]\n",
      "AT5G58060  YKT61; SNARE-like superfamily protein [KO:K08516]\n",
      "AT5G58180  ATYKT62; Synaptobrevin family protein [KO:K08516]\n",
      "AT1G11890  SEC22; Synaptobrevin family protein [KO:K08517]\n",
      "AT5G52270  SNARE-like superfamily protein [KO:K08517]\n",
      "REFERENCE   PMID:16882042\n",
      "AUTHORS   Yoshizawa AC, Kawashima S, Okuda S, Fujita M, Itoh M, Moriya Y, Hattori M, Kanehisa M\n",
      "TITLE     Extracting sequence motifs and the phylogenetic features of SNARE-dependent membrane traffic.\n",
      "JOURNAL   Traffic 7:1104-18 (2006)\n",
      "DOI:10.1111/j.1600-0854.2006.00451.x\n",
      "REFERENCE   PMID:10839363\n",
      "AUTHORS   Scales SJ, Chen YA, Yoo BY, Patel SM, Doung YC, Scheller RH.\n",
      "TITLE     SNAREs contribute to the specificity of membrane fusion.\n",
      "JOURNAL   Neuron 26:457-64 (2000)\n",
      "DOI:10.1016/S0896-6273(00)81177-0\n",
      "REFERENCE   PMID:11737951\n",
      "AUTHORS   Teng FY, Wang Y, Tang BL.\n",
      "TITLE     The syntaxins.\n",
      "JOURNAL   Genome Biol 2:REVIEWS3012 (2001)\n",
      "DOI:10.1186/gb-2001-2-11-reviews3012\n",
      "REFERENCE   PMID:11115874\n",
      "AUTHORS   Sanderfoot AA, Assaad FF, Raikhel NV.\n",
      "TITLE     The Arabidopsis genome. An abundance of soluble N-ethylmaleimide-sensitive factor adaptor protein receptors.\n",
      "JOURNAL   Plant Physiol 124:1558-69 (2000)\n",
      "DOI:10.1104/pp.124.4.1558\n",
      "REFERENCE   PMID:15342965\n",
      "AUTHORS   Uemura T, Ueda T, Ohniwa RL, Nakano A, Takeyasu K, Sato MH.\n",
      "TITLE     Systematic analysis of SNARE molecules in Arabidopsis: dissection of the post-Golgi network in plant cells.\n",
      "JOURNAL   Cell Struct Funct 29:49-65 (2004)\n",
      "DOI:10.1247/csf.29.49\n",
      "REFERENCE   PMID:10872468\n",
      "AUTHORS   Jahn R, Sudhof TC.\n",
      "TITLE     Membrane fusion and exocytosis.\n",
      "JOURNAL   Annu Rev Biochem 68:863-911 (1999)\n",
      "DOI:10.1146/annurev.biochem.68.1.863\n",
      "REFERENCE   PMID:11306253\n",
      "AUTHORS   Pelham HR.\n",
      "TITLE     SNAREs and the specificity of membrane fusion.\n",
      "JOURNAL   Trends Cell Biol 11:99-101 (2001)\n",
      "DOI:10.1016/S0962-8924(01)01929-8\n",
      "REFERENCE   PMID:11104523\n",
      "AUTHORS   Armstrong J\n",
      "TITLE     Membrane traffic between genomes.\n",
      "JOURNAL   Genome Biol 1:REVIEWS104 (2000)\n",
      "DOI:10.1186/gb-2000-1-1-reviews104\n",
      "REFERENCE   PMID:11237004\n",
      "AUTHORS   Bock JB, Matern HT, Peden AA, Scheller RH.\n",
      "TITLE     A genomic perspective on membrane compartment organization.\n",
      "JOURNAL   Nature 409:839-41 (2001)\n",
      "DOI:10.1038/35057024\n",
      "REFERENCE   PMID:11252968\n",
      "AUTHORS   Chen YA, Scheller RH.\n",
      "TITLE     SNARE-mediated membrane fusion.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 2:98-106 (2001)\n",
      "DOI:10.1038/35052017\n",
      "REFERENCE   PMID:9763449\n",
      "AUTHORS   Seron K, Tieaho V, Prescianotto-Baschong C, Aust T, Blondel MO, Guillaud P, Devilliers G, Rossanese OW, Glick BS, Riezman H, Keranen S, Haguenauer-Tsapis R.\n",
      "TITLE     A yeast t-SNARE involved in endocytosis.\n",
      "JOURNAL   Mol Biol Cell 9:2873-89 (1998)\n",
      "DOI:10.1091/mbc.9.10.2873\n",
      "REFERENCE   PMID:10213783\n",
      "AUTHORS   Sanderfoot AA, Raikhel NV.\n",
      "TITLE     The specificity of vesicle trafficking: coat proteins and SNAREs.\n",
      "JOURNAL   Plant Cell 11:629-42 (1999)\n",
      "DOI:10.1105/tpc.11.4.629\n",
      "REFERENCE   PMID:9565594\n",
      "AUTHORS   Abeliovich H, Grote E, Novick P, Ferro-Novick S.\n",
      "TITLE     Tlg2p, a yeast syntaxin homolog that resides on the Golgi and endocytic structures.\n",
      "JOURNAL   J Biol Chem 273:11719-27 (1998)\n",
      "DOI:10.1074/jbc.273.19.11719\n",
      "REFERENCE   PMID:9553086\n",
      "AUTHORS   Advani RJ, Bae HR, Bock JB, Chao DS, Doung YC, Prekeris R, Yoo JS, Scheller RH.\n",
      "TITLE     Seven novel mammalian SNARE proteins localize to distinct membrane compartments.\n",
      "JOURNAL   J Biol Chem 273:10317-24 (1998)\n",
      "DOI:10.1074/jbc.273.17.10317\n",
      "REFERENCE   PMID:10459012\n",
      "AUTHORS   Advani RJ, Yang B, Prekeris R, Lee KC, Klumperman J, Scheller RH.\n",
      "TITLE     VAMP-7 mediates vesicular transport from endosomes to lysosomes.\n",
      "JOURNAL   J Cell Biol 146:765-76 (1999)\n",
      "DOI:10.1083/jcb.146.4.765\n",
      "REFERENCE   PMID:12893879\n",
      "AUTHORS   Burri L, Varlamov O, Doege CA, Hofmann K, Beilharz T, Rothman JE, Sollner TH, Lithgow T.\n",
      "TITLE     A SNARE required for retrograde transport to the endoplasmic reticulum.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 100:9873-7 (2003)\n",
      "DOI:10.1073/pnas.1734000100\n",
      "REFERENCE   PMID:10085287\n",
      "AUTHORS   Chao DS, Hay JC, Winnick S, Prekeris R, Klumperman J, Scheller RH.\n",
      "TITLE     SNARE membrane trafficking dynamics in vivo.\n",
      "JOURNAL   J Cell Biol 144:869-81 (1999)\n",
      "DOI:10.1083/jcb.144.5.869\n",
      "REFERENCE   PMID:10397773\n",
      "AUTHORS   Coe JG, Lim AC, Xu J, Hong W.\n",
      "TITLE     A role for Tlg1p in the transport of proteins within the Golgi apparatus of Saccharomyces cerevisiae.\n",
      "JOURNAL   Mol Biol Cell 10:2407-23 (1999)\n",
      "DOI:10.1091/mbc.10.7.2407\n",
      "REFERENCE   PMID:11445562\n",
      "AUTHORS   Dilcher M, Kohler B, von Mollard GF.\n",
      "TITLE     Genetic interactions with the yeast Q-SNARE VTI1 reveal novel functions for the R-SNARE YKT6.\n",
      "JOURNAL   J Biol Chem 276:34537-44 (2001)\n",
      "DOI:10.1074/jbc.M101551200\n",
      "REFERENCE   PMID:12853481\n",
      "AUTHORS   Dilcher M, Veith B, Chidambaram S, Hartmann E, Schmitt HD, Fischer von Mollard G.\n",
      "TITLE     Use1p is a yeast SNARE protein required for retrograde traffic to the ER.\n",
      "JOURNAL   EMBO J 22:3664-74 (2003)\n",
      "DOI:10.1093/emboj/cdg339\n",
      "REFERENCE   PMID:9446565\n",
      "AUTHORS   Fischer von Mollard G, Stevens TH.\n",
      "TITLE     A human homolog can functionally replace the yeast vesicle-associated SNARE Vti1p in two vesicle transport pathways.\n",
      "JOURNAL   J Biol Chem 273:2624-30 (1998)\n",
      "DOI:10.1074/jbc.273.5.2624\n",
      "REFERENCE   PMID:10359592\n",
      "AUTHORS   Fischer von Mollard G, Stevens TH.\n",
      "TITLE     The Saccharomyces cerevisiae v-SNARE Vti1p is required for multiple membrane transport pathways to the vacuole.\n",
      "JOURNAL   Mol Biol Cell 10:1719-32 (1999)\n",
      "DOI:10.1091/mbc.10.6.1719\n",
      "REFERENCE   PMID:9614185\n",
      "AUTHORS   Galli T, Zahraoui A, Vaidyanathan VV, Raposo G, Tian JM, Karin M, Niemann H, Louvard D.\n",
      "TITLE     A novel tetanus neurotoxin-insensitive vesicle-associated membrane protein in SNARE complexes of the apical plasma membrane of epithelial cells.\n",
      "JOURNAL   Mol Biol Cell 9:1437-48 (1998)\n",
      "DOI:10.1091/mbc.9.6.1437\n",
      "REFERENCE   PMID:11208059\n",
      "AUTHORS   Gerrard SR, Mecklem AB, Stevens TH.\n",
      "TITLE     The yeast endosomal t-SNARE, Pep12p, functions in the absence of its transmembrane domain.\n",
      "JOURNAL   Traffic 1:45-55 (2000)\n",
      "DOI:10.1034/j.1600-0854.2000.010108.x\n",
      "REFERENCE   PMID:10379359\n",
      "AUTHORS   Gerst JE.\n",
      "TITLE     SNAREs and SNARE regulators in membrane fusion and exocytosis.\n",
      "JOURNAL   Cell Mol Life Sci 55:707-34 (1999)\n",
      "DOI:10.1007/s000180050328\n",
      "REFERENCE   PMID:11029060\n",
      "AUTHORS   Gurunathan S, Chapman-Shimshoni D, Trajkovic S, Gerst JE.\n",
      "TITLE     Yeast exocytic v-SNAREs confer endocytosis.\n",
      "JOURNAL   Mol Biol Cell 11:3629-43 (2000)\n",
      "DOI:10.1091/mbc.11.10.3629\n",
      "REFERENCE   PMID:10788491\n",
      "AUTHORS   Hatsuzawa K, Hirose H, Tani K, Yamamoto A, Scheller RH, Tagaya M.\n",
      "TITLE     Syntaxin 18, a SNAP receptor that functions in the endoplasmic reticulum, intermediate compartment, and cis-Golgi vesicle trafficking.\n",
      "JOURNAL   J Biol Chem 275:13713-20 (2000)\n",
      "DOI:10.1074/jbc.275.18.13713\n",
      "REFERENCE   PMID:8621431\n",
      "AUTHORS   Hay JC, Hirling H, Scheller RH.\n",
      "TITLE     Mammalian vesicle trafficking proteins of the endoplasmic reticulum and Golgi apparatus.\n",
      "JOURNAL   J Biol Chem 271:5671-9 (1996)\n",
      "DOI:10.1074/jbc.271.10.5671\n",
      "REFERENCE   PMID:9647643\n",
      "AUTHORS   Hay JC, Klumperman J, Oorschot V, Steegmaier M, Kuo CS, Scheller RH.\n",
      "TITLE     Localization, dynamics, and protein interactions reveal distinct roles for ER and Golgi SNAREs.\n",
      "JOURNAL   J Cell Biol 141:1489-502 (1998)\n",
      "DOI:10.1083/jcb.141.7.1489\n",
      "REFERENCE   PMID:9427746\n",
      "AUTHORS   Holthuis JC, Nichols BJ, Dhruvakumar S, Pelham HR.\n",
      "TITLE     Two syntaxin homologues in the TGN/endosomal system of yeast.\n",
      "JOURNAL   EMBO J 17:113-26 (1998)\n",
      "DOI:10.1093/emboj/17.1.113\n",
      "REFERENCE   PMID:9843576\n",
      "AUTHORS   Holthuis JC, Nichols BJ, Pelham HR.\n",
      "TITLE     The syntaxin Tlg1p mediates trafficking of chitin synthase III to polarized growth sites in yeast.\n",
      "JOURNAL   Mol Biol Cell 9:3383-97 (1998)\n",
      "DOI:10.1091/mbc.9.12.3383\n",
      "REFERENCE   PMID:12600315\n",
      "AUTHORS   Jahn R, Lang T, Sudhof TC.\n",
      "TITLE     Membrane fusion.\n",
      "JOURNAL   Cell 112:519-33 (2003)\n",
      "DOI:10.1016/S0092-8674(03)00112-0\n",
      "REFERENCE   PMID:10097106\n",
      "AUTHORS   Lafont F, Verkade P, Galli T, Wimmer C, Louvard D, Simons K.\n",
      "TITLE     Raft association of SNAP receptors acting in apical trafficking in Madin-Darby canine kidney cells.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 96:3734-8 (1999)\n",
      "DOI:10.1073/pnas.96.7.3734\n",
      "REFERENCE   PMID:9214619\n",
      "AUTHORS   Lewis MJ, Rayner JC, Pelham HR.\n",
      "TITLE     A novel SNARE complex implicated in vesicle fusion with the endoplasmic reticulum.\n",
      "JOURNAL   EMBO J 16:3017-24 (1997)\n",
      "DOI:10.1093/emboj/16.11.3017\n",
      "REFERENCE   PMID:9647644\n",
      "AUTHORS   Low SH, Chapin SJ, Wimmer C, Whiteheart SW, Komuves LG, Mostov KE, Weimbs T.\n",
      "TITLE     The SNARE machinery is involved in apical plasma membrane trafficking in MDCK cells.\n",
      "JOURNAL   J Cell Biol 141:1503-13 (1998)\n",
      "DOI:10.1083/jcb.141.7.1503\n",
      "REFERENCE   PMID:9452464\n",
      "AUTHORS   Low SH, Roche PA, Anderson HA, van Ijzendoorn SC, Zhang M, Mostov KE, Weimbs T.\n",
      "TITLE     Targeting of SNAP-23 and SNAP-25 in polarized epithelial cells.\n",
      "JOURNAL   J Biol Chem 273:3422-30 (1998)\n",
      "DOI:10.1074/jbc.273.6.3422\n",
      "REFERENCE   PMID:12669082\n",
      "AUTHORS   Mostov K, Su T, ter Beest M.\n",
      "TITLE     Polarized epithelial membrane traffic: conservation and plasticity.\n",
      "JOURNAL   Nat Cell Biol 5:287-93 (2003)\n",
      "DOI:10.1038/ncb0403-287\n",
      "REFERENCE   PMID:10873817\n",
      "AUTHORS   Mostov KE, Verges M, Altschuler Y.\n",
      "TITLE     Membrane traffic in polarized epithelial cells.\n",
      "JOURNAL   Curr Opin Cell Biol 12:483-90 (2000)\n",
      "DOI:10.1016/S0955-0674(00)00120-4\n",
      "REFERENCE   PMID:10982406\n",
      "AUTHORS   Mullock BM, Smith CW, Ihrke G, Bright NA, Lindsay M, Parkinson EJ, Brooks DA, Parton RG, James DE, Luzio JP, Piper RC.\n",
      "TITLE     Syntaxin 7 is localized to late endosome compartments, associates with Vamp 8, and Is required for late endosome-lysosome fusion.\n",
      "JOURNAL   Mol Biol Cell 11:3137-53 (2000)\n",
      "DOI:10.1091/mbc.11.9.3137\n",
      "REFERENCE   PMID:9817754\n",
      "AUTHORS   Prekeris R, Klumperman J, Chen YA, Scheller RH.\n",
      "TITLE     Syntaxin 13 mediates cycling of plasma membrane proteins via tubulovesicular recycling endosomes.\n",
      "JOURNAL   J Cell Biol 143:957-71 (1998)\n",
      "DOI:10.1083/jcb.143.4.957\n",
      "REFERENCE   PMID:10564279\n",
      "AUTHORS   Prekeris R, Yang B, Oorschot V, Klumperman J, Scheller RH.\n",
      "TITLE     Differential roles of syntaxin 7 and syntaxin 8 in endosomal trafficking.\n",
      "JOURNAL   Mol Biol Cell 10:3891-908 (1999)\n",
      "DOI:10.1091/mbc.10.11.3891\n",
      "REFERENCE   PMID:9707576\n",
      "AUTHORS   Sanderfoot AA, Ahmed SU, Marty-Mazars D, Rapoport I, Kirchhausen T, Marty F, Raikhel NV.\n",
      "TITLE     A putative vacuolar cargo receptor partially colocalizes with AtPEP12p on a prevacuolar compartment in Arabidopsis roots.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 95:9920-5 (1998)\n",
      "DOI:10.1073/pnas.95.17.9920\n",
      "REFERENCE   PMID:11739776\n",
      "AUTHORS   Sanderfoot AA, Kovaleva V, Bassham DC, Raikhel NV.\n",
      "TITLE     Interactions between syntaxins identify at least five SNARE complexes within the Golgi/prevacuolar system of the Arabidopsis cell.\n",
      "JOURNAL   Mol Biol Cell 12:3733-43 (2001)\n",
      "DOI:10.1091/mbc.12.12.3733\n",
      "REFERENCE   PMID:10557242\n",
      "AUTHORS   Sanderfoot AA, Kovaleva V, Zheng H, Raikhel NV.\n",
      "TITLE     The t-SNARE AtVAM3p resides on the prevacuolar compartment in Arabidopsis root cells.\n",
      "JOURNAL   Plant Physiol 121:929-38 (1999)\n",
      "DOI:10.1104/pp.121.3.929\n",
      "REFERENCE   PMID:11251103\n",
      "AUTHORS   Sanderfoot AA, Pilgrim M, Adam L, Raikhel NV.\n",
      "TITLE     Disruption of individual members of Arabidopsis syntaxin gene families indicates each has essential functions.\n",
      "JOURNAL   Plant Cell 13:659-66 (2001)\n",
      "DOI:10.1105/tpc.13.3.659\n",
      "REFERENCE   PMID:9305917\n",
      "AUTHORS   Sato MH, Nakamura N, Ohsumi Y, Kouchi H, Kondo M, Hara-Nishimura I, Nishimura M, Wada Y.\n",
      "TITLE     The AtVAM3 encodes a syntaxin-related molecule implicated in the vacuolar assembly in Arabidopsis thaliana.\n",
      "JOURNAL   J Biol Chem 272:24530-5 (1997)\n",
      "DOI:10.1074/jbc.272.39.24530\n",
      "REFERENCE   PMID:9710615\n",
      "AUTHORS   Sato TK, Darsow T, Emr SD.\n",
      "TITLE     Vam7p, a SNAP-25-like molecule, and Vam3p, a syntaxin homolog, function together in yeast vacuolar protein trafficking.\n",
      "JOURNAL   Mol Cell Biol 18:5308-19 (1998)\n",
      "DOI:10.1128/MCB.18.9.5308\n",
      "REFERENCE   PMID:9813082\n",
      "AUTHORS   Spang A, Schekman R.\n",
      "TITLE     Reconstitution of retrograde transport from the Golgi to the ER in vitro.\n",
      "JOURNAL   J Cell Biol 143:589-99 (1998)\n",
      "DOI:10.1083/jcb.143.3.589\n",
      "REFERENCE   PMID:10359608\n",
      "AUTHORS   Steegmaier M, Klumperman J, Foletti DL, Yoo JS, Scheller RH.\n",
      "TITLE     Vesicle-associated membrane protein 4 is implicated in trans-Golgi network vesicle trafficking.\n",
      "JOURNAL   Mol Biol Cell 10:1957-72 (1999)\n",
      "DOI:10.1091/mbc.10.6.1957\n",
      "REFERENCE   PMID:11208143\n",
      "AUTHORS   Steegmaier M, Lee KC, Prekeris R, Scheller RH.\n",
      "TITLE     SNARE protein trafficking in polarized MDCK cells.\n",
      "JOURNAL   Traffic 1:553-60 (2000)\n",
      "DOI:10.1034/j.1600-0854.2000.010705.x\n",
      "REFERENCE   PMID:9852078\n",
      "AUTHORS   Steegmaier M, Yang B, Yoo JS, Huang B, Shen M, Yu S, Luo Y, Scheller RH.\n",
      "TITLE     Three novel proteins of the syntaxin/SNAP-25 family.\n",
      "JOURNAL   J Biol Chem 273:34171-9 (1998)\n",
      "DOI:10.1074/jbc.273.51.34171\n",
      "REFERENCE   PMID:10683148\n",
      "AUTHORS   Subramaniam VN, Loh E, Horstmann H, Habermann A, Xu Y, Coe J, Griffiths G, Hong W.\n",
      "TITLE     Preferential association of syntaxin 8 with the early endosome.\n",
      "JOURNAL   J Cell Sci 113 ( Pt 6):997-1008 (2000)\n",
      "REFERENCE   PMID:9507000\n",
      "AUTHORS   Tang BL, Tan AE, Lim LK, Lee SS, Low DY, Hong W.\n",
      "TITLE     Syntaxin 12, a member of the syntaxin family localized to the endosome.\n",
      "JOURNAL   J Biol Chem 273:6944-50 (1998)\n",
      "DOI:10.1074/jbc.273.12.6944\n",
      "REFERENCE   PMID:9628864\n",
      "AUTHORS   Ungermann C, Wickner W.\n",
      "TITLE     Vam7p, a vacuolar SNAP-25 homolog, is required for SNARE complex integrity and vacuole docking and fusion.\n",
      "JOURNAL   EMBO J 17:3269-76 (1998)\n",
      "DOI:10.1093/emboj/17.12.3269\n",
      "REFERENCE   PMID:10036234\n",
      "AUTHORS   Valdez AC, Cabaniols JP, Brown MJ, Roche PA.\n",
      "TITLE     Syntaxin 11 is associated with SNAP-23 on late endosomes and the trans-Golgi network.\n",
      "JOURNAL   J Cell Sci 112 ( Pt 6):845-54 (1999)\n",
      "REFERENCE   PMID:9199167\n",
      "AUTHORS   von Mollard GF, Nothwehr SF, Stevens TH.\n",
      "TITLE     The yeast v-SNARE Vti1p mediates two vesicle transport pathways through interactions with the t-SNAREs Sed5p and Pep12p.\n",
      "JOURNAL   J Cell Biol 137:1511-24 (1997)\n",
      "DOI:10.1083/jcb.137.7.1511\n",
      "REFERENCE   PMID:11278762\n",
      "AUTHORS   Wade N, Bryant NJ, Connolly LM, Simpson RJ, Luzio JP, Piper RC, James DE.\n",
      "TITLE     Syntaxin 7 complexes with mouse Vps10p tail interactor 1b, syntaxin 6, vesicle-associated membrane protein (VAMP)8, and VAMP7 in b16 melanoma cells.\n",
      "JOURNAL   J Biol Chem 276:19820-7 (2001)\n",
      "DOI:10.1074/jbc.M010838200\n",
      "REFERENCE   PMID:11555414\n",
      "AUTHORS   Wendler F, Tooze S.\n",
      "TITLE     Syntaxin 6: the promiscuous behaviour of a SNARE protein.\n",
      "JOURNAL   Traffic 2:606-11 (2001)\n",
      "DOI:10.1034/j.1600-0854.2001.20903.x\n",
      "REFERENCE   PMID:9614193\n",
      "AUTHORS   Wong SH, Zhang T, Xu Y, Subramaniam VN, Griffiths G, Hong W.\n",
      "TITLE     Endobrevin, a novel synaptobrevin/VAMP-like protein preferentially associated with the early endosome.\n",
      "JOURNAL   Mol Biol Cell 9:1549-63 (1998)\n",
      "DOI:10.1091/mbc.9.6.1549\n",
      "REFERENCE   PMID:11035026\n",
      "AUTHORS   Xu D, Joglekar AP, Williams AL, Hay JC.\n",
      "TITLE     Subunit structure of a mammalian ER/Golgi SNARE complex.\n",
      "JOURNAL   J Biol Chem 275:39631-9 (2000)\n",
      "DOI:10.1074/jbc.M007684200\n",
      "REFERENCE   PMID:11323436\n",
      "AUTHORS   Zhang T, Hong W.\n",
      "TITLE     Ykt6 forms a SNARE complex with syntaxin 5, GS28, and Bet1 and participates in a late stage in endoplasmic reticulum-Golgi transport.\n",
      "JOURNAL   J Biol Chem 276:27480-7 (2001)\n",
      "DOI:10.1074/jbc.M102786200\n",
      "REFERENCE   PMID:12068098\n",
      "AUTHORS   Zheng H, Bednarek SY, Sanderfoot AA, Alonso J, Ecker JR, Raikhel NV.\n",
      "TITLE     NPSN11 is a cell plate-associated SNARE protein that interacts with the syntaxin KNOLLE.\n",
      "JOURNAL   Plant Physiol 129:530-9 (2002)\n",
      "DOI:10.1104/pp.003970\n",
      "REFERENCE   PMID:10397763\n",
      "AUTHORS   Zheng H, von Mollard GF, Kovaleva V, Stevens TH, Raikhel NV.\n",
      "TITLE     The plant vesicle-associated SNARE AtVTI1a likely mediates vesicle transport from the trans-Golgi network to the prevacuolar compartment.\n",
      "JOURNAL   Mol Biol Cell 10:2251-64 (1999)\n",
      "DOI:10.1091/mbc.10.7.2251\n",
      "KO_PATHWAY  ko04130\n",
      "///\n",
      "ENTRY       ath04136                    Pathway\n",
      "NAME        Autophagy - other - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Autophagy is a degradative pathway for the removal of cytoplasmic materials in eukaryotic cells, and is characterized by the formation of a double-membrane structure called the autophagosome, either in a housekeeping capacity or during stress and senescence. The process of autophagy could be divided into several stages: induction, vesicle nucleation, elongation and closure, and fusion and digestion. Most essential autophagic machineries are conserved throughout eukaryotes (see map04140 for animals and map04138 for fungi). This map is for other eukaryotes including plants and protists, where autophagy related genes (ATGs) play similar roles in the life cycle. However, autophagy has been relatively less studied in lower eukaryotes.\n",
      "CLASS       Cellular Processes; Transport and catabolism\n",
      "PATHWAY_MAP ath04136  Autophagy - other\n",
      "DBLINKS     GO: 0006914\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G08850  RAPTOR1; Regulatory-associated protein of TOR 1 [KO:K07204]\n",
      "AT5G01770  RAPTOR2; regulatory-associated protein of TOR 2 (RAPTOR2) [KO:K07204]\n",
      "AT1G50030  TOR; target of rapamycin [KO:K07203] [EC:2.7.11.1]\n",
      "AT2G22040  LST8-2; Transducin/WD40 repeat-like superfamily protein [KO:K08266]\n",
      "AT3G18140  LST8-1; Transducin/WD40 repeat-like superfamily protein [KO:K08266]\n",
      "AT2G37840  Protein kinase superfamily protein [KO:K08269] [EC:2.7.11.1]\n",
      "AT3G53930  Protein kinase superfamily protein [KO:K08269] [EC:2.7.11.1]\n",
      "AT3G18770  Autophagy-related protein 13 [KO:K08331]\n",
      "AT3G49590  ATG13; Autophagy-related protein 13 [KO:K08331]\n",
      "AT5G66930  meiotically up-regulated protein [KO:K19730]\n",
      "AT4G30790  autophagy-like protein [KO:K08330]\n",
      "AT5G53000  TAP46; 2A phosphatase associated protein of 46 kD [KO:K17606]\n",
      "AT1G10430  PP2A-2; protein phosphatase 2A-2 [KO:K04382] [EC:3.1.3.16]\n",
      "AT1G59830  PP2A-1; protein phosphatase 2A-2 [KO:K04382] [EC:3.1.3.16]\n",
      "AT1G69960  PP2A; serine/threonine protein phosphatase 2A [KO:K04382] [EC:3.1.3.16]\n",
      "AT2G42500  PP2A-3; protein phosphatase 2A-3 [KO:K04382] [EC:3.1.3.16]\n",
      "AT3G58500  PP2A-4; protein phosphatase 2A-4 [KO:K04382] [EC:3.1.3.16]\n",
      "AT2G31260  APG9; autophagy 9 (APG9) [KO:K17907]\n",
      "AT3G19190  ATG2; autophagy 2 [KO:K17906]\n",
      "AT4G30510  ATG18B; autophagy 18 B-like protein [KO:K17908]\n",
      "AT3G61710  ATG6; AUTOPHAGY 6 [KO:K08334]\n",
      "AT1G60490  VPS34; vacuolar protein sorting 34 [KO:K00914] [EC:2.7.1.137]\n",
      "AT4G29380  VPS15; protein kinase family protein / WD-40 repeat family protein [KO:K08333] [EC:2.7.11.1]\n",
      "AT3G13970  APG12B; Ubiquitin-like superfamily protein [KO:K08336]\n",
      "AT1G54210  ATG12A; Ubiquitin-like superfamily protein [KO:K08336]\n",
      "AT5G17290  APG5; autophagy protein Apg5 family [KO:K08339]\n",
      "AT5G50230  Transducin/WD40 repeat-like superfamily protein [KO:K17890]\n",
      "AT5G45900  APG7; ThiF family protein [KO:K08337]\n",
      "AT3G07525  ATG10; autophagocytosis-associated family protein [KO:K17888]\n",
      "AT5G61500  ATG3; autophagy 3 (APG3) [KO:K08343]\n",
      "AT2G05630  ATG8D; Ubiquitin-like superfamily protein [KO:K08341]\n",
      "AT4G16520  ATG8F; Ubiquitin-like superfamily protein [KO:K08341]\n",
      "AT4G04620  ATG8B; Ubiquitin-like superfamily protein [KO:K08341]\n",
      "AT4G21980  APG8A; Ubiquitin-like superfamily protein [KO:K08341]\n",
      "AT1G62040  ATG8C; Ubiquitin-like superfamily protein [KO:K08341]\n",
      "AT2G45170  ATG8E; AUTOPHAGY 8E [KO:K08341]\n",
      "AT3G60640  ATG8G; Ubiquitin-like superfamily protein [KO:K08341]\n",
      "AT3G06420  ATG8H; Ubiquitin-like superfamily protein [KO:K08341]\n",
      "AT3G15580  APG8H; Ubiquitin-like superfamily protein [KO:K08341]\n",
      "AT2G44140  Peptidase family C54 protein [KO:K08342] [EC:3.4.22.-]\n",
      "AT3G59950  Peptidase family C54 protein [KO:K08342] [EC:3.4.22.-]\n",
      "COMPOUND    C00350  Phosphatidylethanolamine\n",
      "C01194  1-Phosphatidyl-D-myo-inositol\n",
      "C04549  1-Phosphatidyl-1D-myo-inositol 3-phosphate\n",
      "REFERENCE   PMID:20962583\n",
      "AUTHORS   Duszenko M, Ginger ML, Brennand A, Gualdron-Lopez M, Colombo MI, Coombs GH, Coppens I, Jayabalasingham B, Langsley G, de Castro SL, Menna-Barreto R, Mottram JC, Navarro M, Rigden DJ, Romano PS, Stoka V, Turk B, Michels PA\n",
      "TITLE     Autophagy in protists.\n",
      "JOURNAL   Autophagy 7:127-58 (2011)\n",
      "DOI:10.4161/auto.7.2.13310\n",
      "REFERENCE   PMID:21660427\n",
      "AUTHORS   Avin-Wittenberg T, Honig A, Galili G\n",
      "TITLE     Variations on a theme: plant autophagy in comparison to yeast and mammals.\n",
      "JOURNAL   Protoplasma 249:285-99 (2012)\n",
      "DOI:10.1007/s00709-011-0296-z\n",
      "REFERENCE   PMID:22242963\n",
      "AUTHORS   Liu Y, Bassham DC\n",
      "TITLE     Autophagy: pathways for self-eating in plant cells.\n",
      "JOURNAL   Annu Rev Plant Biol 63:215-37 (2012)\n",
      "DOI:10.1146/annurev-arplant-042811-105441\n",
      "REFERENCE   PMID:22764279\n",
      "AUTHORS   Yoshimoto K\n",
      "TITLE     Beginning to understand autophagy, an intracellular self-degradation system in plants.\n",
      "JOURNAL   Plant Cell Physiol 53:1355-65 (2012)\n",
      "DOI:10.1093/pcp/pcs099\n",
      "REFERENCE   PMID:24563201\n",
      "AUTHORS   Li F, Chung T, Vierstra RD\n",
      "TITLE     AUTOPHAGY-RELATED11 plays a critical role in general autophagy- and senescence-induced mitophagy in Arabidopsis.\n",
      "JOURNAL   Plant Cell 26:788-807 (2014)\n",
      "DOI:10.1105/tpc.113.120014\n",
      "REFERENCE   PMID:26598298\n",
      "AUTHORS   Michaeli S, Galili G, Genschik P, Fernie AR, Avin-Wittenberg T\n",
      "TITLE     Autophagy in Plants--What's New on the Menu?\n",
      "JOURNAL   Trends Plant Sci 21:134-44 (2016)\n",
      "DOI:10.1016/j.tplants.2015.10.008\n",
      "REFERENCE   PMID:26614870\n",
      "AUTHORS   Yang X, Bassham DC\n",
      "TITLE     New Insight into the Mechanism and Function of Autophagy in Plant Cells.\n",
      "JOURNAL   Int Rev Cell Mol Biol 320:1-40 (2015)\n",
      "DOI:10.1016/bs.ircmb.2015.07.005\n",
      "KO_PATHWAY  ko04136\n",
      "///\n",
      "ENTRY       ath04141                    Pathway\n",
      "NAME        Protein processing in endoplasmic reticulum - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The endoplasmic reticulum (ER) is a subcellular organelle where proteins are folded with the help of lumenal chaperones. Newly synthesized peptides enter the ER via the sec61 pore and are glycosylated. Correctly folded proteins are packaged into transport vesicles that shuttle them to the Golgi complex. Misfolded proteins are retained within the ER lumen in complex with molecular chaperones. Proteins that are terminally misfolded bind to BiP and are directed toward degradation through the proteasome in a process called ER-associated degradation (ERAD). Accumulation of misfolded proteins in the ER causes ER stress and activates a signaling pathway called the unfolded protein response (UPR). In certain severe situations, however, the protective mechanisms activated by the UPR are not sufficient to restore normal ER function and cells die by apoptosis.\n",
      "CLASS       Genetic Information Processing; Folding, sorting and degradation\n",
      "PATHWAY_MAP ath04141  Protein processing in endoplasmic reticulum\n",
      "DBLINKS     GO: 0030433 0034976\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G78720  SecY protein transport family protein [KO:K10956]\n",
      "AT1G29310  SecY protein transport family protein [KO:K10956]\n",
      "AT2G18710  SCY1; SECY homolog 1 [KO:K10956]\n",
      "AT2G34250  SecY protein transport family protein [KO:K10956]\n",
      "AT5G60460  Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT2G45070  SEC61_BETA; Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT3G60540  Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT4G24920  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT3G48570  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT5G50460  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT3G20920  translocation protein-like protein [KO:K12275]\n",
      "AT1G79940  ATERDJ2A; DnaJ / Sec63 Brl domains-containing protein [KO:K09540]\n",
      "AT4G21180  ATERDJ2B; DnaJ / Sec63 Brl domains-containing protein [KO:K09540]\n",
      "AT1G76400  Ribophorin I [KO:K12666]\n",
      "AT2G01720  Ribophorin I [KO:K12666]\n",
      "AT4G21150  HAP6; ribophorin II (RPN2) family protein [KO:K12667]\n",
      "AT1G32210  ATDAD1; Defender against death (DAD family) protein [KO:K12668]\n",
      "AT2G35520  DAD2; Defender against death (DAD family) protein [KO:K12668]\n",
      "AT1G61790  Oligosaccharyltransferase complex/magnesium transporter family protein [KO:K12669]\n",
      "AT1G11560  Oligosaccharyltransferase complex/magnesium transporter family protein [KO:K12669]\n",
      "AT5G66680  DGL1; dolichyl-diphosphooligosaccharide-protein glycosyltransferase 48kDa subunit family protein [KO:K12670]\n",
      "AT1G34130  STT3B; staurosporin and temperature sensitive 3-like b [KO:K07151] [EC:2.4.99.18]\n",
      "AT5G19690  STT3A; staurosporin and temperature sensitive 3-like A [KO:K07151] [EC:2.4.99.18]\n",
      "AT1G24320  Six-hairpin glycosidases superfamily protein [KO:K01228] [EC:3.2.1.106]\n",
      "AT1G67490  GCS1; glucosidase 1 [KO:K01228] [EC:3.2.1.106]\n",
      "AT4G16660  heat shock protein 70 (Hsp 70) family protein [KO:K09486]\n",
      "AT5G42020  BIP2; Heat shock protein 70 (Hsp 70) family protein [KO:K09490] [EC:3.6.4.10]\n",
      "AT5G28540  BIP1; heat shock protein 70 (Hsp 70) family protein [KO:K09490] [EC:3.6.4.10]\n",
      "AT1G09080  BIP3; Heat shock protein 70 (Hsp 70) family protein [KO:K09490] [EC:3.6.4.10]\n",
      "AT3G62600  ATERDJ3B; DNAJ heat shock family protein [KO:K09517]\n",
      "AT5G03160  P58IPK; DnaJ P58IPK-like protein [KO:K09523]\n",
      "AT3G07770  Hsp89.1; HEAT SHOCK PROTEIN 89.1 [KO:K09487]\n",
      "AT2G04030  CR88; Chaperone protein htpG family protein [KO:K09487]\n",
      "AT4G24190  SHD; Chaperone protein htpG family protein [KO:K09487]\n",
      "AT5G63840  RSW3; Glycosyl hydrolases family 31 protein [KO:K05546] [EC:3.2.1.207]\n",
      "AT5G56360  PSL4; calmodulin-binding protein [KO:K08288]\n",
      "AT5G61790  CNX1; calnexin 1 [KO:K08054]\n",
      "AT5G07340  Calreticulin family protein [KO:K08054]\n",
      "AT1G09210  CRT1b; calreticulin 1b [KO:K08057]\n",
      "AT1G56340  CRT1a; calreticulin 1a [KO:K08057]\n",
      "AT1G08450  CRT3; calreticulin 3 [KO:K08057]\n",
      "AT3G21160  MNS2; alpha-mannosidase 2 [KO:K01230] [EC:3.2.1.113]\n",
      "AT1G51590  MNS1; alpha-mannosidase 1 [KO:K01230] [EC:3.2.1.113]\n",
      "AT1G30000  MNS3; alpha-mannosidase 3 [KO:K23741] [EC:3.2.1.209]\n",
      "AT2G01470  STL2P; SEC12P-like 2 protein [KO:K14003]\n",
      "AT5G50650  Transducin/WD40 repeat-like superfamily protein [KO:K14003]\n",
      "AT5G50550  Transducin/WD40 repeat-like superfamily protein [KO:K14003]\n",
      "AT3G52190  PHF1; phosphate transporter traffic facilitator1 [KO:K14003]\n",
      "AT3G62560  Ras-related small GTP-binding family protein [KO:K07953] [EC:3.6.5.-]\n",
      "AT1G56330  SAR1B; secretion-associated RAS 1B [KO:K07953] [EC:3.6.5.-]\n",
      "AT1G02620  Ras-related small GTP-binding family protein [KO:K07953] [EC:3.6.5.-]\n",
      "AT4G02080  SAR2; secretion-associated RAS super family 2 [KO:K07953] [EC:3.6.5.-]\n",
      "AT1G09180  SARA1A; secretion-associated RAS super family 1 [KO:K07953] [EC:3.6.5.-]\n",
      "AT2G30050  transducin family protein / WD-40 repeat family protein [KO:K14004]\n",
      "AT3G01340  Transducin/WD40 repeat-like superfamily protein [KO:K14004]\n",
      "AT3G63460  transducin family protein / WD-40 repeat family protein [KO:K14005]\n",
      "AT1G18830  Transducin/WD40 repeat-like superfamily protein [KO:K14005]\n",
      "AT3G23660  Sec23/Sec24 protein transport family protein [KO:K14006]\n",
      "AT4G14160  Sec23/Sec24 protein transport family protein [KO:K14006]\n",
      "AT1G05520  Sec23/Sec24 protein transport family protein [KO:K14006]\n",
      "AT2G21630  Sec23/Sec24 protein transport family protein [KO:K14006]\n",
      "AT5G43670  Sec23/Sec24 protein transport family protein [KO:K14006]\n",
      "AT3G07100  ERMO2; Sec23/Sec24 protein transport family protein [KO:K14007]\n",
      "AT4G32640  Sec23/Sec24 protein transport family protein [KO:K14007]\n",
      "AT3G44340  CEF; hypothetical protein [KO:K14007]\n",
      "AT1G71220  EBS1; EMS-MUTAGENIZED BRI1 SUPPRESSOR 1 [KO:K11718] [EC:2.4.1.-]\n",
      "AT1G27520  Glycosyl hydrolase family 47 protein [KO:K10084]\n",
      "AT5G43710  Glycosyl hydrolase family 47 protein [KO:K10085]\n",
      "AT5G60640  PDIL1-4; PDI-like 1-4 [KO:K09580] [EC:5.3.4.1]\n",
      "AT1G77510  PDIL1-2; PDI-like 1-2 [KO:K09580] [EC:5.3.4.1]\n",
      "AT1G21750  PDIL1-1; PDI-like 1-1 [KO:K09580] [EC:5.3.4.1]\n",
      "AT3G16110  PDIL1-6; PDI-like 1-6 [KO:K09580] [EC:5.3.4.1]\n",
      "AT1G52260  PDIL1-5; PDI-like 1-5 [KO:K09580] [EC:5.3.4.1]\n",
      "AT3G54960  PDIL1-3; PDI-like 1-3 [KO:K09580] [EC:5.3.4.1]\n",
      "AT1G35620  PDIL5-2; PDI-like 5-2 [KO:K09580] [EC:5.3.4.1]\n",
      "AT2G47470  UNE5; thioredoxin family protein [KO:K09584] [EC:5.3.4.1]\n",
      "AT1G04980  PDIL2-2; PDI-like 2-2 [KO:K09584] [EC:5.3.4.1]\n",
      "AT2G32920  PDIL2-3; PDI-like 2-3 [KO:K09584] [EC:5.3.4.1]\n",
      "AT2G38960  ERO2; endoplasmic reticulum oxidoreductins 2 [KO:K10950] [EC:1.8.4.-]\n",
      "AT1G72280  ERO1; endoplasmic reticulum oxidoreductins 1 [KO:K10950] [EC:1.8.4.-]\n",
      "AT5G35080  OS9; ER lectin-like protein [KO:K10088]\n",
      "AT2G21160  Translocon-associated protein (TRAP), alpha subunit [KO:K13249]\n",
      "AT2G16595  Translocon-associated protein (TRAP), alpha subunit [KO:K13249]\n",
      "AT5G14030  translocon-associated protein beta (TRAPB) family protein [KO:K13250]\n",
      "AT5G42570  B-cell receptor-associated 31-like protein [KO:K14009]\n",
      "AT1G11905  B-cell receptor-associated protein 31-like protein [KO:K14009]\n",
      "AT5G48660  B-cell receptor-associated protein 31-like protein [KO:K14009]\n",
      "AT3G07190  B-cell receptor-associated protein 31-like protein [KO:K14009]\n",
      "AT3G20450  B-cell receptor-associated protein 31-like protein [KO:K14009]\n",
      "AT4G04860  DER2.2; DERLIN-2.2 [KO:K13989]\n",
      "AT4G29330  DER1; DERLIN-1 [KO:K13989]\n",
      "AT4G21810  DER2.1; DERLIN-2.1 [KO:K13989]\n",
      "AT2G43210  Ubiquitin-like superfamily protein [KO:K24348]\n",
      "AT1G04850  ubiquitin-associated (UBA)/TS-N domain-containing protein [KO:K24348]\n",
      "AT2G01650  PUX2; plant UBX domain-containing protein 2 [KO:K14011]\n",
      "AT4G15410  PUX5; serine/threonine protein phosphatase 2A 55 kDa regulatory subunit B prime gamma [KO:K14012]\n",
      "AT3G21660  UBX domain-containing protein [KO:K14012]\n",
      "AT4G22150  PUX3; UBA/UBX domain protein [KO:K14012]\n",
      "AT4G04210  PUX4; plant UBX domain containing protein 4 [KO:K14012]\n",
      "AT3G53230  AtCDC48B; ATPase, AAA-type, CDC48 protein [KO:K13525]\n",
      "AT3G09840  CDC48; cell division cycle 48 [KO:K13525]\n",
      "AT5G03340  AtCDC48C; ATPase, AAA-type, CDC48 protein [KO:K13525]\n",
      "AT3G63000  NPL41; NPL4-like protein 1 [KO:K14015]\n",
      "AT2G47970  Nuclear pore localization protein NPL4 [KO:K14015]\n",
      "AT2G21270  UFD1; ubiquitin fusion degradation 1 [KO:K14016]\n",
      "AT4G38930  Ubiquitin fusion degradation UFD1 family protein [KO:K14016]\n",
      "AT2G29070  Ubiquitin fusion degradation UFD1 family protein [KO:K14016]\n",
      "AT3G12580  HSP70; heat shock protein 70 [KO:K03283]\n",
      "AT3G09440  Heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT4G24280  cpHsc70-1; chloroplast heat shock protein 70-1 [KO:K03283]\n",
      "AT5G49910  cpHsc70-2; chloroplast heat shock protein 70-2 [KO:K03283]\n",
      "AT5G02490  Hsp70-2; Heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT5G02500  HSC70-1; heat shock cognate protein 70-1 [KO:K03283]\n",
      "AT1G56410  ERD2; heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT1G16030  Hsp70b; heat shock protein 70B [KO:K03283]\n",
      "AT3G44110  J3; DNAJ homologue 3 [KO:K09503]\n",
      "AT5G22060  J2; DNAJ homologue 2 [KO:K09503]\n",
      "AT3G57340  DnaJ heat shock amino-terminal domain protein (DUF1977) [KO:K09518]\n",
      "AT5G05750  DNAJ heat shock N-terminal domain-containing protein [KO:K09518]\n",
      "AT5G49060  DnaJ heat shock amino-terminal domain protein (DUF1977) [KO:K09518]\n",
      "AT5G52640  HSP90.1; heat shock-like protein [KO:K04079]\n",
      "AT5G56030  HSP81-2; heat shock protein 81-2 [KO:K04079]\n",
      "AT5G56000  Hsp81.4; HEAT SHOCK PROTEIN 81.4 [KO:K04079]\n",
      "AT5G56010  HSP81-3; heat shock protein 81-3 [KO:K04079]\n",
      "AT3G53800  Fes1B; Fes1B [KO:K09562]\n",
      "AT3G09350  Fes1A; Fes1A [KO:K09562]\n",
      "AT5G02150  Fes1C; Fes1C [KO:K09562]\n",
      "AT3G51980  ARM repeat superfamily protein [KO:K09562]\n",
      "AT1G07400  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT1G53540  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT1G59860  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT2G29500  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT5G20970  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT4G10250  ATHSP22.0; HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT1G52560  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT1G54050  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT4G21870  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT4G27670  HSP21; heat shock protein 21 [KO:K13993]\n",
      "AT4G25200  HSP23.6-MITO; mitochondrion-localized small heat shock protein 23.6 [KO:K13993]\n",
      "AT5G51440  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT5G59720  HSP18.2; heat shock protein 18.2 [KO:K13993]\n",
      "AT3G46230  HSP17.4; heat shock protein 17.4 [KO:K13993]\n",
      "AT5G12020  HSP17.6II; 17.6 kDa class II heat shock protein [KO:K13993]\n",
      "AT5G12030  HSP17.6A; heat shock protein 17.6A [KO:K13993]\n",
      "AT5G37670  HSP20-like chaperones superfamily protein [KO:K13993]\n",
      "AT1G50670  OTU-like cysteine protease family protein [KO:K13719] [EC:3.1.2.-]\n",
      "AT3G18860  transducin family protein / WD-40 repeat family protein [KO:K14018]\n",
      "AT1G16190  RAD23A; Rad23 UV excision repair protein family [KO:K10839]\n",
      "AT1G79650  RAD23B; Rad23 UV excision repair protein family [KO:K10839]\n",
      "AT5G38470  RAD23D; Rad23 UV excision repair protein family [KO:K10839]\n",
      "AT3G02540  RAD23C; Rad23 UV excision repair protein family [KO:K10839]\n",
      "AT5G16090  RAD23 UV excision repair family protein [KO:K10839]\n",
      "AT2G17190  ubiquitin family protein [KO:K04523]\n",
      "AT2G17200  DSK2; ubiquitin family protein [KO:K04523]\n",
      "AT5G49570  PNG1; peptide-N-glycanase 1 [KO:K01456] [EC:3.5.1.52]\n",
      "AT3G54130  Josephin family protein [KO:K11863] [EC:3.4.22.-]\n",
      "AT5G15400  U-box domain-containing protein [KO:K10597] [EC:2.3.2.27]\n",
      "AT3G59410  GCN2; protein kinase family protein [KO:K16196] [EC:2.7.11.1]\n",
      "AT5G05470  EIF2_ALPHA; eukaryotic translation initiation factor 2 alpha subunit [KO:K03237]\n",
      "AT2G40290  Eukaryotic translation initiation factor 2 subunit 1 [KO:K03237]\n",
      "AT5G19660  S1P; SITE-1 protease [KO:K08653] [EC:3.4.21.112]\n",
      "AT4G20310  Peptidase M50 family protein [KO:K07765] [EC:3.4.24.85]\n",
      "AT2G17520  IRE1A; Endoribonuclease/protein kinase IRE1-like protein [KO:K08852] [EC:2.7.11.1 3.1.26.-]\n",
      "AT4G34100  CER9; RING/U-box superfamily protein [KO:K10661] [EC:2.3.2.27]\n",
      "AT4G32670  RING/FYVE/PHD zinc finger superfamily protein [KO:K10661] [EC:2.3.2.27]\n",
      "AT3G17000  UBC32; ubiquitin-conjugating enzyme 32 [KO:K10578] [EC:2.3.2.23]\n",
      "AT1G17280  UBC34; ubiquitin-conjugating enzyme 34 [KO:K04554] [EC:2.3.2.23]\n",
      "AT5G50430  UBC33; ubiquitin-conjugating enzyme 33 [KO:K04554] [EC:2.3.2.23]\n",
      "AT3G46460  UBC13; ubiquitin-conjugating enzyme 13 [KO:K10575] [EC:2.3.2.23]\n",
      "AT3G55380  UBC14; ubiquitin-conjugating enzyme 14 [KO:K10575] [EC:2.3.2.23]\n",
      "AT5G59300  UBC7; ubiquitin-conjugating enzyme E2 [KO:K10575] [EC:2.3.2.23]\n",
      "AT3G16090  Hrd1A; RING/U-box superfamily protein [KO:K10601] [EC:2.3.2.27]\n",
      "AT1G65040  Hrd1B; RING/U-box superfamily protein [KO:K10601] [EC:2.3.2.27]\n",
      "AT2G23780  RING/U-box superfamily protein [KO:K10666] [EC:2.3.2.27]\n",
      "AT1G19310  RING/U-box superfamily protein [KO:K10666] [EC:2.3.2.27]\n",
      "AT3G58030  RING/U-box superfamily protein [KO:K10666] [EC:2.3.2.27]\n",
      "AT4G27470  RMA3; RING membrane-anchor 3 [KO:K10666] [EC:2.3.2.27]\n",
      "AT4G03510  RMA1; RING membrane-anchor 1 [KO:K10666] [EC:2.3.2.27]\n",
      "AT4G28270  RMA2; RING membrane-anchor 2 [KO:K10666] [EC:2.3.2.27]\n",
      "AT1G18260  EBS5; HCP-like superfamily protein [KO:K14026]\n",
      "AT1G73570  HCP-like superfamily protein [KO:K14026]\n",
      "AT4G25230  RIN2; RPM1 interacting protein 2 [KO:K10636] [EC:2.3.2.36]\n",
      "AT5G51450  RIN3; RPM1 interacting protein 3 [KO:K10636] [EC:2.3.2.36]\n",
      "AT3G07370  CHIP; carboxyl terminus of HSC70-interacting protein [KO:K09561] [EC:2.3.2.27]\n",
      "AT3G08700  UBC12; ubiquitin-conjugating enzyme 12 [KO:K06689] [EC:2.3.2.23]\n",
      "AT2G16740  UBC29; ubiquitin-conjugating enzyme 29 [KO:K06689] [EC:2.3.2.23]\n",
      "AT5G56150  UBC30; ubiquitin-conjugating enzyme 30 [KO:K06689] [EC:2.3.2.23]\n",
      "AT1G64230  UBC28; ubiquitin-conjugating enzyme 28 [KO:K06689] [EC:2.3.2.23]\n",
      "AT3G08690  UBC11; ubiquitin-conjugating enzyme 11 [KO:K06689] [EC:2.3.2.23]\n",
      "AT3G42830  RING/U-box superfamily protein [KO:K03868] [EC:2.3.2.32]\n",
      "AT5G20570  RBX1; RING-box 1 [KO:K03868] [EC:2.3.2.32]\n",
      "AT4G02570  CUL1; cullin 1 [KO:K03347]\n",
      "AT1G02980  CUL2; cullin 2 [KO:K03347]\n",
      "AT1G59790  Cullin family protein [KO:K03347]\n",
      "AT1G59800  Cullin family protein [KO:K03347]\n",
      "AT1G43140  Cullin family protein [KO:K03347]\n",
      "AT1G75950  SKP1; S phase kinase-associated protein 1 [KO:K03094]\n",
      "AT5G42190  E3 ubiquitin ligase SCF complex subunit SKP1/ASK1 family protein [KO:K03094]\n",
      "AT1G20140  SK4; SKP1-like 4 [KO:K03094]\n",
      "AT2G25700  SK3; SKP1-like 3 [KO:K03094]\n",
      "AT2G03170  SK14; SKP1-like 14 [KO:K03094]\n",
      "AT3G21830  SK8; SKP1-like 8 [KO:K03094]\n",
      "AT3G21850  SK9; SKP1-like 9 [KO:K03094]\n",
      "AT3G21860  SK10; SKP1-like 10 [KO:K03094]\n",
      "AT3G60010  SK13; SKP1-like 13 [KO:K03094]\n",
      "AT4G34210  SK11; SKP1-like 11 [KO:K03094]\n",
      "AT4G34470  SK12; SKP1-like 12 [KO:K03094]\n",
      "AT1G10230  SK18; E3 ubiquitin ligase SCF complex subunit SKP1/ASK1 family protein [KO:K03094]\n",
      "AT2G03160  SK19; SKP1-like 19 [KO:K03094]\n",
      "AT2G03190  SK16; SKP1-like 16 [KO:K03094]\n",
      "AT2G20160  MEO; E3 ubiquitin ligase SCF complex subunit SKP1/ASK1 family protein [KO:K03094]\n",
      "AT3G25650  SK15; SKP1-like 15 [KO:K03094]\n",
      "AT3G60020  SK5; SKP1-like 5 [KO:K03094]\n",
      "AT3G21840  SK7; SKP1-like 7 [KO:K03094]\n",
      "AT3G53060  SK6; SKP1-like 6 [KO:K03094]\n",
      "COMPOUND    C00076  Calcium cation\n",
      "G00009\n",
      "G00010\n",
      "G00011\n",
      "G00012\n",
      "G10694\n",
      "REFERENCE   PMID:19491040\n",
      "AUTHORS   Naidoo N\n",
      "TITLE     ER and aging-Protein folding and the ER stress response.\n",
      "JOURNAL   Ageing Res Rev 8:150-9 (2009)\n",
      "DOI:10.1016/j.arr.2009.03.001\n",
      "REFERENCE   PMID:18023214\n",
      "AUTHORS   Malhotra JD, Kaufman RJ\n",
      "TITLE     The endoplasmic reticulum and the unfolded protein response.\n",
      "JOURNAL   Semin Cell Dev Biol 18:716-31 (2007)\n",
      "DOI:10.1016/j.semcdb.2007.09.003\n",
      "REFERENCE   PMID:20347046\n",
      "AUTHORS   Maattanen P, Gehring K, Bergeron JJ, Thomas DY\n",
      "TITLE     Protein quality control in the ER: the recognition of misfolded proteins.\n",
      "JOURNAL   Semin Cell Dev Biol 21:500-11 (2010)\n",
      "DOI:10.1016/j.semcdb.2010.03.006\n",
      "REFERENCE   PMID:20219571\n",
      "AUTHORS   Stolz A, Wolf DH\n",
      "TITLE     Endoplasmic reticulum associated protein degradation: a chaperone assisted journey to hell.\n",
      "JOURNAL   Biochim Biophys Acta 1803:694-705 (2010)\n",
      "DOI:10.1016/j.bbamcr.2010.02.005\n",
      "REFERENCE   PMID:20112977\n",
      "AUTHORS   Dejgaard K, Theberge JF, Heath-Engel H, Chevet E, Tremblay ML, Thomas DY\n",
      "TITLE     Organization of the Sec61 translocon, studied by high resolution native electrophoresis.\n",
      "JOURNAL   J Proteome Res 9:1763-71 (2010)\n",
      "DOI:10.1021/pr900900x\n",
      "REL_PATHWAY ath00510  N-Glycan biosynthesis\n",
      "ath03050  Proteasome\n",
      "ath03060  Protein export\n",
      "KO_PATHWAY  ko04141\n",
      "///\n",
      "ENTRY       ath04144                    Pathway\n",
      "NAME        Endocytosis - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Endocytosis is a mechanism for cells to remove ligands, nutrients, and plasma membrane (PM) proteins, and lipids from the cell surface, bringing them into the cell interior. Transmembrane proteins entering through clathrin-dependent endocytosis (CDE) have sequences in their cytoplasmic domains that bind to the APs (adaptor-related protein complexes) and enable their rapid removal from the PM. In addition to APs and clathrin, there are numerous accessory proteins including dynamin. Depending on the various proteins that enter the endosome membrane, these cargoes are sorted to distinct destinations. Some cargoes, such as nutrient receptors, are recycled back to the PM. Ubiquitylated membrane proteins, such as activated growth-factor receptors, are sorted into intraluminal vesicles and eventually end up in the lysosome lumen via multivesicular endosomes (MVEs). There are distinct mechanisms of clathrin-independent endocytosis (CIE) depending upon the cargo and the cell type.\n",
      "CLASS       Cellular Processes; Transport and catabolism\n",
      "PATHWAY_MAP ath04144  Endocytosis\n",
      "DBLINKS     GO: 0006897\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G26420  PIP5K3; 1-phosphatidylinositol-4-phosphate 5-kinase 3 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G10900  Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G21980  PIP5K1; phosphatidylinositol-4-phosphate 5-kinase 1 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G60890  Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G77740  PIP5K2; phosphatidylinositol-4-phosphate 5-kinase 2 [KO:K00889] [EC:2.7.1.68]\n",
      "AT2G41210  PIP5K5; phosphatidylinositol- 4-phosphate 5-kinase 5 [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G07960  PIP5K6; Phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G56960  PIP5K4; phosphatidyl inositol monophosphate 5 kinase 4 [KO:K00889] [EC:2.7.1.68]\n",
      "AT4G01190  PIPK10; phosphatidylinositol phosphate kinase 10 [KO:K00889] [EC:2.7.1.68]\n",
      "AT1G01460  PIPK11; Phosphatidylinositol-4-phosphate 5-kinase, core [KO:K00889] [EC:2.7.1.68]\n",
      "AT3G09920  PIP5K9; phosphatidyl inositol monophosphate 5 kinase [KO:K00889] [EC:2.7.1.68]\n",
      "AT2G42010  PLDBETA1; phospholipase D beta 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT5G25370  PLDALPHA3; phospholipase D alpha 3 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11840  PLDGAMMA3; phospholipase D gamma 3 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11850  PLDGAMMA1; phospholipase D gamma 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT1G55180  PLDEPSILON; phospholipase D alpha 4 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G05630  PLDP2; phospholipase D P2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G15730  PLDALPHA1; phospholipase D alpha 1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT3G16785  PLDP1; phospholipase D P1 [KO:K01115] [EC:3.1.4.4]\n",
      "AT1G52570  PLDALPHA2; phospholipase D alpha 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G35790  PLDDELTA; phospholipase D delta [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G00240  PLDBETA2; phospholipase D beta 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT4G11830  PLDGAMMA2; phospholipase D gamma 2 [KO:K01115] [EC:3.1.4.4]\n",
      "AT1G59610  DL3; dynamin-like 3 [KO:K01528] [EC:3.6.5.5]\n",
      "AT1G10290  ADL6; dynamin-like protein 6 [KO:K01528] [EC:3.6.5.5]\n",
      "AT3G08530  Clathrin, heavy chain [KO:K04646]\n",
      "AT3G11130  Clathrin, heavy chain [KO:K04646]\n",
      "AT5G22770  alpha-ADR; alpha-adaptin [KO:K11824]\n",
      "AT5G22780  Adaptor protein complex AP-2, alpha subunit [KO:K11824]\n",
      "AT5G46630  Clathrin adaptor complexes medium subunit family protein [KO:K11826]\n",
      "AT1G47830  SNARE-like superfamily protein [KO:K11827]\n",
      "AT3G59290  ENTH/VHS family protein [KO:K12471]\n",
      "AT5G11710  ENTH/VHS family protein [KO:K12471]\n",
      "AT2G43160  ENTH/VHS family protein [KO:K12471]\n",
      "AT3G46540  ENTH/VHS family protein [KO:K12471]\n",
      "AT1G21630  Calcium-binding EF hand family protein [KO:K12472]\n",
      "AT2G17840  ERD7; Senescence/dehydration-associated protein-like protein [KO:K19366]\n",
      "AT3G51250  Senescence/dehydration-associated protein-like protein [KO:K19366]\n",
      "AT4G35985  Senescence/dehydration-associated protein-like protein [KO:K19366]\n",
      "AT3G27000  ARP2; actin related protein 2 [KO:K17260]\n",
      "AT1G13180  DIS1; Actin-like ATPase superfamily protein [KO:K18584]\n",
      "AT2G31300  ARPC1B; actin-related protein C1B [KO:K05757]\n",
      "AT2G30910  ARPC1A; actin-related protein C1A [KO:K05757]\n",
      "AT1G30825  DIS2; Arp2/3 complex, 34 kD subunit p34-Arc [KO:K05758]\n",
      "AT2G33385  arpc2b; actin-related protein C2B [KO:K05758]\n",
      "AT1G60430  ARPC3; actin-related protein C3 [KO:K05756]\n",
      "AT4G14147  ARPC4; binding protein [KO:K05755]\n",
      "AT4G01710  CRK; ARP2/3 complex 16 kDa subunit (p16-Arc) [KO:K05754]\n",
      "AT5G65274  ARP2/3 complex 16 kDa subunit (p16-Arc) [KO:K05754]\n",
      "AT3G12580  HSP70; heat shock protein 70 [KO:K03283]\n",
      "AT3G09440  Heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT4G24280  cpHsc70-1; chloroplast heat shock protein 70-1 [KO:K03283]\n",
      "AT5G49910  cpHsc70-2; chloroplast heat shock protein 70-2 [KO:K03283]\n",
      "AT5G02490  Hsp70-2; Heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT5G02500  HSC70-1; heat shock cognate protein 70-1 [KO:K03283]\n",
      "AT1G56410  ERD2; heat shock protein 70 (Hsp 70) family protein [KO:K03283]\n",
      "AT1G16030  Hsp70b; heat shock protein 70B [KO:K03283]\n",
      "AT3G05520  CPA; Subunits of heterodimeric actin filament capping protein Capz superfamily [KO:K10364]\n",
      "AT1G71790  CPB; Subunits of heterodimeric actin filament capping protein Capz superfamily [KO:K10365]\n",
      "AT3G47810  MAG1; Calcineurin-like metallo-phosphoesterase superfamily protein [KO:K18467]\n",
      "AT5G53530  VPS26A; vacuolar protein sorting 26A [KO:K18466]\n",
      "AT4G27690  VPS26B; vacuolar protein sorting 26B [KO:K18466]\n",
      "AT2G17790  VPS35A; vacuolar protein sorting-associated 35A-like protein [KO:K18468]\n",
      "AT3G51310  VPS35C; VPS35 homolog C [KO:K18468]\n",
      "AT1G75850  VPS35B; VPS35 homolog B [KO:K18468]\n",
      "AT4G16144  AMSH3; associated molecule with the SH3 domain of STAM 3 [KO:K11866] [EC:3.4.19.12]\n",
      "AT1G48790  AMSH1; associated molecule with the SH3 domain of STAM 1 [KO:K11866] [EC:3.4.19.12]\n",
      "AT1G10600  AMSH2; associated molecule with the SH3 domain of STAM 2 [KO:K11866] [EC:3.4.19.12]\n",
      "AT3G18820  RAB7B; RAB GTPase homolog G3F [KO:K07897]\n",
      "AT1G52280  RABG3d; RAB GTPase homolog G3D [KO:K07897]\n",
      "AT3G16100  RABG3c; RAB GTPase homolog G3C [KO:K07897]\n",
      "AT1G22740  RABG3B; RAB GTPase homolog G3B [KO:K07897]\n",
      "AT4G09720  RABG3A; RAB GTPase homolog G3A [KO:K07897]\n",
      "AT2G21880  RAB7A; RAB GTPase homolog 7A [KO:K07897]\n",
      "AT5G13860  ELC-Like; ELCH-like protein [KO:K12183]\n",
      "AT3G12400  ELC; Ubiquitin-conjugating enzyme/RWD-like protein [KO:K12183]\n",
      "AT2G38830  Ubiquitin-conjugating enzyme/RWD-like protein [KO:K12183]\n",
      "AT4G21560  VPS28-1; vacuolar protein sorting-associated protein-like protein [KO:K12184]\n",
      "AT4G05000  VPS28-2; Vacuolar protein sorting-associated protein VPS28 family protein [KO:K12184]\n",
      "AT3G53120  VPS37-1; Modifier of rudimentary (Mod(r)) protein [KO:K12185]\n",
      "AT2G36680  Modifier of rudimentary (Mod(r)) protein [KO:K12185]\n",
      "AT4G27040  VPS22; EAP30/Vps36 family protein [KO:K12188]\n",
      "AT5G04920  EAP30/Vps36 family protein [KO:K12190]\n",
      "AT4G19003  VPS25; E2F/DP family winged-helix DNA-binding domain-containing protein [KO:K12189]\n",
      "AT5G63880  VPS20.1; SNF7 family protein [KO:K12195]\n",
      "AT5G09260  VPS20.2; vacuolar protein sorting-associated protein 20.2 [KO:K12195]\n",
      "AT2G19830  SNF7.2; SNF7 family protein [KO:K12194]\n",
      "AT4G29160  SNF7.1; SNF7 family protein [KO:K12194]\n",
      "AT3G45000  VPS24.2; SNF7 family protein [KO:K12193]\n",
      "AT5G22950  VPS24.1; SNF7 family protein [KO:K12193]\n",
      "AT2G06530  VPS2.1; SNF7 family protein [KO:K12191]\n",
      "AT5G44560  VPS2.2; SNF7 family protein [KO:K12191]\n",
      "AT1G03950  VPS2.3; vacuolar protein sorting-associated protein 2.3 [KO:K12192]\n",
      "AT3G62080  SNF7 family protein [KO:K15053]\n",
      "AT2G27600  SKD1; AAA-type ATPase family protein [KO:K12196]\n",
      "AT1G08270  vacuolar protein sorting-associated protein [KO:K12196]\n",
      "AT4G26750  EXT-like; hydroxyproline-rich glycoprotein family protein [KO:K12199]\n",
      "AT1G17730  VPS46.1; vacuolar protein sorting 46.1 [KO:K12197]\n",
      "AT1G73030  VPS46.2; SNF7 family protein [KO:K12197]\n",
      "AT3G10640  VPS60.1; SNF7 family protein [KO:K12198]\n",
      "AT5G04850  VPS60.2; SNF7 family protein [KO:K12198]\n",
      "AT1G15130  Endosomal targeting BRO1-like domain-containing protein [KO:K12200]\n",
      "AT4G12230  alpha/beta-Hydrolases superfamily protein [KO:K19367]\n",
      "AT5G06140  SNX1; sorting nexin 1 [KO:K17917]\n",
      "AT3G20290  EHD1; EPS15 homology domain 1 [KO:K12483]\n",
      "AT4G05520  EHD2; EPS15 homology domain 2 [KO:K12483]\n",
      "AT1G77140  VPS45; vacuolar protein sorting 45 [KO:K12479]\n",
      "AT5G59840  Ras-related small GTP-binding family protein [KO:K07901]\n",
      "AT3G46060  RABE1c; RAB GTPase homolog 8A [KO:K07901]\n",
      "AT3G53610  RAB8; RAB GTPase homolog 8 [KO:K07901]\n",
      "AT5G03520  RAB8C; RAB GTPase homolog 8C [KO:K07901]\n",
      "AT3G09900  RABE1e; RAB GTPase homolog E1E [KO:K07901]\n",
      "AT4G18800  RABA1d; RAB GTPase homolog A1D [KO:K07904]\n",
      "AT1G09630  RAB11c; RAB GTPase 11C [KO:K07904]\n",
      "AT1G16920  RABA1b; RAB GTPase homolog A1B [KO:K07904]\n",
      "AT5G47960  RABA4C; RAB GTPase homolog A4C [KO:K07904]\n",
      "AT2G31680  RABA5d; RAB GTPase homolog A5D [KO:K07904]\n",
      "AT3G12160  RABA4D; RAB GTPase homolog A4D [KO:K07904]\n",
      "AT5G65270  RABA4a; RAB GTPase homolog A4A [KO:K07904]\n",
      "AT5G47520  RABA5a; RAB GTPase homolog A5A [KO:K07904]\n",
      "AT3G46830  RABA2c; RAB GTPase homolog A2C [KO:K07904]\n",
      "AT4G39990  RABA4B; RAB GTPase homolog A4B [KO:K07904]\n",
      "AT5G45750  RABA1c; RAB GTPase homolog A1C [KO:K07904]\n",
      "AT2G43130  ARA4; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K07904]\n",
      "AT1G01200  RABA3; RAB GTPase homolog A3 [KO:K07904]\n",
      "AT5G60860  RABA1f; RAB GTPase homolog A1F [KO:K07904]\n",
      "AT5G59150  RABA2D; RAB GTPase homolog A2D [KO:K07904]\n",
      "AT5G54310  AGD5; ARF-GAP domain 5 [KO:K12486]\n",
      "AT3G17660  AGD15; ARF-GAP domain 15 [KO:K12486]\n",
      "AT3G07940  Calcium-dependent ARF-type GTPase activating protein family [KO:K12486]\n",
      "AT4G21160  ZAC; Calcium-dependent ARF-type GTPase activating protein family [KO:K12486]\n",
      "AT4G05330  AGD13; ARF-GAP domain 13 [KO:K12486]\n",
      "AT1G60860  AGD2; ARF-GAP domain 2 [KO:K12489]\n",
      "AT1G10870  AGD4; ARF-GAP domain 4 [KO:K12489]\n",
      "AT5G13300  SFC; ARF GTPase-activating protein [KO:K12489]\n",
      "AT5G61980  AGD1; ARF-GAP domain 1 [KO:K12489]\n",
      "AT3G53710  AGD6; ARF-GAP domain 6 [KO:K12492]\n",
      "AT2G37550  AGD7; ARF-GAP domain 7 [KO:K12492]\n",
      "AT5G46750  AGD9; ARF-GAP domain 9 [KO:K12493]\n",
      "AT4G17890  AGD8; ARF-GAP domain 8 [KO:K12493]\n",
      "AT2G35210  RPA; root and pollen arfgap [KO:K12493]\n",
      "AT1G01960  EDA10; SEC7-like guanine nucleotide exchange family protein [KO:K18442]\n",
      "AT4G35380  SEC7-like guanine nucleotide exchange family protein [KO:K18442]\n",
      "AT3G60860  SEC7-like guanine nucleotide exchange family protein [KO:K18442]\n",
      "AT4G38200  SEC7-like guanine nucleotide exchange family protein [KO:K18442]\n",
      "AT5G19610  GNL2; GNOM-like 2 [KO:K18443]\n",
      "AT5G39500  GNL1; GNOM-like 1 [KO:K18443]\n",
      "AT1G13980  GN; sec7 domain-containing protein [KO:K18443]\n",
      "AT1G10630  ARFA1F; ADP-ribosylation factor A1F [KO:K07937]\n",
      "AT1G23490  ARF1; ADP-ribosylation factor 1 [KO:K07937]\n",
      "AT1G70490  ARFA1D; Ras-related small GTP-binding family protein [KO:K07937]\n",
      "AT5G14670  ARFA1B; ADP-ribosylation factor A1B [KO:K07937]\n",
      "AT2G47170  ARF1A1C; Ras-related small GTP-binding family protein [KO:K07937]\n",
      "AT3G62290  ARFA1E; ADP-ribosylation factor A1E [KO:K07937]\n",
      "AT5G17060  ARFB1B; ADP-ribosylation factor B1B [KO:K07937]\n",
      "AT2G15310  ARFB1A; ADP-ribosylation factor B1A [KO:K07937]\n",
      "AT3G03120  ARFB1C; ADP-ribosylation factor B1C [KO:K07937]\n",
      "AT1G02430  ARFD1B; ADP-ribosylation factor D1B [KO:K07937]\n",
      "COMPOUND    C00035  GDP\n",
      "C00044  GTP\n",
      "C04549  1-Phosphatidyl-1D-myo-inositol 3-phosphate\n",
      "C04637  1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate\n",
      "C05981  Phosphatidylinositol-3,4,5-trisphosphate\n",
      "C11556  1-Phosphatidyl-1D-myo-inositol 3,5-bisphosphate\n",
      "REFERENCE   PMID:19007773\n",
      "AUTHORS   Acconcia F, Sigismund S, Polo S\n",
      "TITLE     Ubiquitin in trafficking: the network at work.\n",
      "JOURNAL   Exp Cell Res 315:1610-8 (2009)\n",
      "DOI:10.1016/j.yexcr.2008.10.014\n",
      "REFERENCE   PMID:18647649\n",
      "AUTHORS   Donaldson JG, Porat-Shliom N, Cohen LA\n",
      "TITLE     Clathrin-independent endocytosis: a unique platform for cell signaling and PM remodeling.\n",
      "JOURNAL   Cell Signal 21:1-6 (2009)\n",
      "DOI:10.1016/j.cellsig.2008.06.020\n",
      "REFERENCE   PMID:15261671\n",
      "AUTHORS   Sorkin A\n",
      "TITLE     Cargo recognition during clathrin-mediated endocytosis: a team effort.\n",
      "JOURNAL   Curr Opin Cell Biol 16:392-9 (2004)\n",
      "DOI:10.1016/j.ceb.2004.06.001\n",
      "REFERENCE   PMID:19325624\n",
      "AUTHORS   Raiborg C, Stenmark H\n",
      "TITLE     The ESCRT machinery in endosomal sorting of ubiquitylated membrane proteins.\n",
      "JOURNAL   Nature 458:445-52 (2009)\n",
      "DOI:10.1038/nature07961\n",
      "REFERENCE   PMID:20110990\n",
      "AUTHORS   Scita G, Di Fiore PP\n",
      "TITLE     The endocytic matrix.\n",
      "JOURNAL   Nature 463:464-73 (2010)\n",
      "DOI:10.1038/nature08910\n",
      "REFERENCE   PMID:15687999\n",
      "AUTHORS   Le Roy C, Wrana JL\n",
      "TITLE     Clathrin- and non-clathrin-mediated endocytic regulation of cell signalling.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 6:112-26 (2005)\n",
      "DOI:10.1038/nrm1571\n",
      "REFERENCE   PMID:19924646\n",
      "AUTHORS   Jovic M, Sharma M, Rahajeng J, Caplan S\n",
      "TITLE     The early endosome: a busy sorting station for proteins at the crossroads.\n",
      "JOURNAL   Histol Histopathol 25:99-112 (2010)\n",
      "DOI:10.14670/HH-25.99\n",
      "REFERENCE   PMID:16203612\n",
      "AUTHORS   Pietiainen VM, Marjomaki V, Heino J, Hyypia T\n",
      "TITLE     Viral entry, lipid rafts and caveosomes.\n",
      "JOURNAL   Ann Med 37:394-403 (2005)\n",
      "DOI:10.1080/07853890510011976\n",
      "REFERENCE   PMID:19696797\n",
      "AUTHORS   Grant BD, Donaldson JG\n",
      "TITLE     Pathways and mechanisms of endocytic recycling.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 10:597-608 (2009)\n",
      "DOI:10.1038/nrm2755\n",
      "REFERENCE   PMID:22193161\n",
      "AUTHORS   Cullen PJ, Korswagen HC\n",
      "TITLE     Sorting nexins provide diversity for retromer-dependent trafficking events.\n",
      "JOURNAL   Nat Cell Biol 14:29-37 (2012)\n",
      "DOI:10.1038/ncb2374\n",
      "REL_PATHWAY ath04070  Phosphatidylinositol signaling system\n",
      "ath04130  SNARE interactions in vesicular transport\n",
      "KO_PATHWAY  ko04144\n",
      "///\n",
      "ENTRY       ath04145                    Pathway\n",
      "NAME        Phagosome - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Phagocytosis is the process of taking in relatively large particles by a cell, and is a central mechanism in the tissue remodeling, inflammation, and defense against infectious agents. A phagosome is formed when the specific receptors on the phagocyte surface recognize ligands on the particle surface. After formation, nascent phagosomes progressively acquire digestive characteristics. This maturation of phagosomes involves regulated interaction with the other membrane organelles, including recycling endosomes, late endosomes and lysosomes. The fusion of phagosomes and lysosomes releases toxic products that kill most bacteria and degrade them into fragments. However, some bacteria have strategies to escape the bactericidal mechanisms associated with phagocytosis and survive within host phagocytes.\n",
      "CLASS       Cellular Processes; Transport and catabolism\n",
      "PATHWAY_MAP ath04145  Phagosome\n",
      "DBLINKS     GO: 0045335\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT1G51740  SYP81; syntaxin of plants 81 [KO:K08492]\n",
      "AT1G11890  SEC22; Synaptobrevin family protein [KO:K08517]\n",
      "AT5G52270  SNARE-like superfamily protein [KO:K08517]\n",
      "AT1G60490  VPS34; vacuolar protein sorting 34 [KO:K00914] [EC:2.7.1.137]\n",
      "AT1G78900  VHA-A; vacuolar ATP synthase subunit A [KO:K02145] [EC:7.1.2.2]\n",
      "AT1G76030  VAB1; ATPase, V1 complex, subunit B protein [KO:K02147]\n",
      "AT1G20260  VAB3; ATPase, V1 complex, subunit B protein [KO:K02147]\n",
      "AT4G38510  VAB2; ATPase, V1 complex, subunit B protein [KO:K02147]\n",
      "AT1G12840  DET3; vacuolar ATP synthase subunit C (VATC) / V-ATPase C subunit / vacuolar proton pump C subunit (DET3) [KO:K02148]\n",
      "AT3G58730  vacuolar ATP synthase subunit D (VATD) / V-ATPase D subunit / vacuolar proton pump D subunit (VATPD) [KO:K02149]\n",
      "AT1G64200  VHA-E3; vacuolar H+-ATPase subunit E isoform 3 [KO:K02150]\n",
      "AT3G08560  VHA-E2; vacuolar H+-ATPase subunit E isoform 2 [KO:K02150]\n",
      "AT4G11150  TUF; vacuolar ATP synthase subunit E1 [KO:K02150]\n",
      "AT4G02620  vacuolar ATPase subunit F family protein [KO:K02151]\n",
      "AT4G23710  VAG2; vacuolar ATP synthase subunit G2 [KO:K02152]\n",
      "AT4G25950  VATG3; vacuolar ATP synthase G3 [KO:K02152]\n",
      "AT3G01390  VMA10; vacuolar membrane ATPase 10 [KO:K02152]\n",
      "AT5G55290  ATPase, V0 complex, subunit E [KO:K02153]\n",
      "AT4G26710  ATPase, V0 complex, subunit E [KO:K02153]\n",
      "AT2G21410  VHA-A2; vacuolar proton ATPase A2 [KO:K02154]\n",
      "AT2G28520  VHA-A1; vacuolar proton ATPase A1 [KO:K02154]\n",
      "AT4G39080  VHA-A3; vacuolar proton ATPase A3 [KO:K02154]\n",
      "AT3G28710  ATPase, V0/A0 complex, subunit C/D [KO:K02146]\n",
      "AT3G28715  ATPase, V0/A0 complex, subunit C/D [KO:K02146]\n",
      "AT3G42050  vacuolar ATP synthase subunit H family protein [KO:K02144]\n",
      "AT1G19910  AVA-P2; ATPase, F0/V0 complex, subunit C protein [KO:K02155]\n",
      "AT1G75630  AVA-P4; vacuolar H+-pumping ATPase 16 kDa proteolipid subunit 4 [KO:K02155]\n",
      "AT2G16510  ATPase, F0/V0 complex, subunit C protein [KO:K02155]\n",
      "AT4G34720  AVA-P1; ATPase, F0/V0 complex, subunit C protein [KO:K02155]\n",
      "AT4G38920  VHA-C3; vacuolar-type H[+]-ATPase C3 [KO:K02155]\n",
      "AT2G25610  ATPase, F0/V0 complex, subunit C protein [KO:K03661]\n",
      "AT4G32530  ATPase, F0/V0 complex, subunit C protein [KO:K03661]\n",
      "AT3G18820  RAB7B; RAB GTPase homolog G3F [KO:K07897]\n",
      "AT1G52280  RABG3d; RAB GTPase homolog G3D [KO:K07897]\n",
      "AT3G16100  RABG3c; RAB GTPase homolog G3C [KO:K07897]\n",
      "AT1G22740  RABG3B; RAB GTPase homolog G3B [KO:K07897]\n",
      "AT4G09720  RABG3A; RAB GTPase homolog G3A [KO:K07897]\n",
      "AT2G21880  RAB7A; RAB GTPase homolog 7A [KO:K07897]\n",
      "AT4G14960  TUA6; Tubulin/FtsZ family protein [KO:K07374]\n",
      "AT5G19770  TUA3; tubulin alpha-3 [KO:K07374]\n",
      "AT1G04820  TUA4; tubulin alpha-4 chain [KO:K07374]\n",
      "AT1G50010  TUA2; tubulin alpha-2 chain [KO:K07374]\n",
      "AT5G19780  TUA5; tubulin alpha-5 [KO:K07374]\n",
      "AT1G64740  TUA1; alpha-1 tubulin [KO:K07374]\n",
      "AT2G29550  TUB7; tubulin beta-7 chain [KO:K07375]\n",
      "AT5G12250  TUB6; beta-6 tubulin [KO:K07375]\n",
      "AT1G20010  TUB5; tubulin beta-5 chain [KO:K07375]\n",
      "AT1G75780  TUB1; tubulin beta-1 chain [KO:K07375]\n",
      "AT4G20890  TUB9; tubulin beta-9 chain [KO:K07375]\n",
      "AT5G23860  TUB8; tubulin beta 8 [KO:K07375]\n",
      "AT5G44340  TUB4; tubulin beta chain 4 [KO:K07375]\n",
      "AT5G62690  TUB2; tubulin beta chain 2 [KO:K07375]\n",
      "AT5G62700  TUB3; tubulin beta chain 3 [KO:K07375]\n",
      "AT4G33240  FAB1A; 1-phosphatidylinositol-3-phosphate 5-kinase FAB1A [KO:K00921] [EC:2.7.1.150]\n",
      "AT3G14270  FAB1B; phosphatidylinositol-4-phosphate 5-kinase family protein [KO:K00921] [EC:2.7.1.150]\n",
      "AT1G34260  FAB1D; FORMS APLOID AND BINUCLEATE CELLS 1A [KO:K00921] [EC:2.7.1.150]\n",
      "AT1G71010  FAB1C; FORMS APLOID AND BINUCLEATE CELLS 1C [KO:K00921] [EC:2.7.1.150]\n",
      "AT1G78720  SecY protein transport family protein [KO:K10956]\n",
      "AT1G29310  SecY protein transport family protein [KO:K10956]\n",
      "AT2G18710  SCY1; SECY homolog 1 [KO:K10956]\n",
      "AT2G34250  SecY protein transport family protein [KO:K10956]\n",
      "AT5G60460  Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT2G45070  SEC61_BETA; Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT3G60540  Preprotein translocase Sec, Sec61-beta subunit protein [KO:K09481]\n",
      "AT4G24920  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT3G48570  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT5G50460  secE/sec61-gamma protein transport protein [KO:K07342]\n",
      "AT1G09210  CRT1b; calreticulin 1b [KO:K08057]\n",
      "AT1G56340  CRT1a; calreticulin 1a [KO:K08057]\n",
      "AT1G08450  CRT3; calreticulin 3 [KO:K08057]\n",
      "AT5G61790  CNX1; calnexin 1 [KO:K08054]\n",
      "AT5G07340  Calreticulin family protein [KO:K08054]\n",
      "AT4G35020  RAC3; RAC-like 3 [KO:K04392]\n",
      "AT4G35950  RAC6; RAC-like 6 [KO:K04392]\n",
      "AT5G45970  RAC2; RAC-like 2 [KO:K04392]\n",
      "AT1G75840  ARAC5; RAC-like GTP binding protein 5 [KO:K04392]\n",
      "AT2G44690  ARAC9; RAC-like 9 [KO:K04392]\n",
      "AT3G48040  ROP10; RHO-related protein from plants 10 [KO:K04392]\n",
      "AT3G51300  ROP1; RHO-related protein from plants 1 [KO:K04392]\n",
      "AT4G28950  ROP9; RHO-related protein from plants 9 [KO:K04392]\n",
      "AT5G62880  RAC10; RAC-like 10 [KO:K04392]\n",
      "AT1G20090  ROP2; RHO-related protein from plants 2 [KO:K04392]\n",
      "AT2G17800  ARAC1; RAC-like 1 [KO:K04392]\n",
      "COMPOUND    C04549  1-Phosphatidyl-1D-myo-inositol 3-phosphate\n",
      "REFERENCE   PMID:15894272\n",
      "AUTHORS   Stuart LM, Ezekowitz RA\n",
      "TITLE     Phagocytosis: elegant complexity.\n",
      "JOURNAL   Immunity 22:539-50 (2005)\n",
      "DOI:10.1016/j.immuni.2005.05.002\n",
      "REFERENCE   PMID:10684585\n",
      "AUTHORS   Tjelle TE, Lovdal T, Berg T\n",
      "TITLE     Phagosome dynamics and function.\n",
      "JOURNAL   Bioessays 22:255-63 (2000)\n",
      "DOI:10.1002/(SICI)1521-1878(200003)22:3<255::AID-BIES7>3.0.CO;2-R\n",
      "REFERENCE   PMID:18492791\n",
      "AUTHORS   Dupuy AG, Caron E\n",
      "TITLE     Integrin-dependent phagocytosis: spreading from microadhesion to new concepts.\n",
      "JOURNAL   J Cell Sci 121:1773-83 (2008)\n",
      "DOI:10.1242/jcs.018036\n",
      "REFERENCE   PMID:15036145\n",
      "AUTHORS   Brumell JH, Grinstein S\n",
      "TITLE     Salmonella redirects phagosomal maturation.\n",
      "JOURNAL   Curr Opin Microbiol 7:78-84 (2004)\n",
      "DOI:10.1016/j.mib.2003.12.005\n",
      "REFERENCE   PMID:17245426\n",
      "AUTHORS   Huynh KK, Eskelinen EL, Scott CC, Malevanets A, Saftig P, Grinstein S\n",
      "TITLE     LAMP proteins are required for fusion of lysosomes with phagosomes.\n",
      "JOURNAL   EMBO J 26:313-24 (2007)\n",
      "DOI:10.1038/sj.emboj.7601511\n",
      "REFERENCE   PMID:17637737\n",
      "AUTHORS   Luzio JP, Pryor PR, Bright NA\n",
      "TITLE     Lysosomes: fusion and function.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 8:622-32 (2007)\n",
      "DOI:10.1038/nrm2217\n",
      "REFERENCE   PMID:18513324\n",
      "AUTHORS   Sumimoto H\n",
      "TITLE     Structure, regulation and evolution of Nox-family NADPH oxidases that produce reactive oxygen species.\n",
      "JOURNAL   FEBS J 275:3249-77 (2008)\n",
      "DOI:10.1111/j.1742-4658.2008.06488.x\n",
      "REL_PATHWAY ath04144  Endocytosis\n",
      "KO_PATHWAY  ko04145\n",
      "///\n",
      "ENTRY       ath04146                    Pathway\n",
      "NAME        Peroxisome - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Peroxisomes are essential organelles that play a key role in redox signalling and lipid homeostasis. They contribute to many crucial metabolic processes such as fatty acid oxidation, biosynthesis of ether lipids and free radical detoxification. The biogenesis of peroxisomes starts with the early peroxins PEX3, PEX16 and PEX19 and proceeds via several steps. The import of membrane proteins into peroxisomes needs PEX19 for recognition, targeting and insertion via docking at PEX3. Matrix proteins in the cytosol are recognized by peroxisomal targeting signals (PTS) and transported to the docking complex at the peroxisomal membrane. Peroxisomes' deficiencies lead to severe and often fatal inherited peroxisomal disorders (PD). PDs are usually classified in two groups. The first group is disorders of peroxisome biogenesis which include Zellweger syndrome, and the second group is single peroxisomal enzyme deficiencies.\n",
      "CLASS       Cellular Processes; Transport and catabolism\n",
      "PATHWAY_MAP ath04146  Peroxisome\n",
      "DBLINKS     GO: 0005777\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G45690  SSE1; shrunken seed protein (SSE1) [KO:K13335]\n",
      "AT3G18160  PEX3-1; peroxin 3-1 [KO:K13336]\n",
      "AT1G48635  PEX3; peroxin 3 [KO:K13336]\n",
      "AT5G17550  PEX19-2; peroxin 19-2 [KO:K13337]\n",
      "AT3G03490  PEX19-1; peroxin 19-1 [KO:K13337]\n",
      "AT5G08470  PEX1; peroxisome 1 [KO:K13338]\n",
      "AT1G03000  PEX6; peroxin 6 [KO:K13339]\n",
      "AT1G29260  PEX7; peroxin 7 [KO:K13341]\n",
      "AT5G56290  PEX5; peroxin 5 [KO:K13342]\n",
      "AT5G62810  PEX14; peroxin 14 [KO:K13343]\n",
      "AT3G07560  PEX13; peroxin 13 [KO:K13344]\n",
      "AT3G04460  PEX12; peroxin-12 [KO:K13345]\n",
      "AT2G26350  PEX10; peroxin 10 [KO:K13346]\n",
      "AT1G79810  TED3; Pex2/Pex12 N-terminal domain-containing protein / zinc finger (C3HC4-type RING finger) family protein [KO:K06664]\n",
      "AT4G04470  PMP22; Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13347]\n",
      "AT4G14305  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13347]\n",
      "AT2G14860  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13348]\n",
      "AT5G19750  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13348]\n",
      "AT2G42770  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13348]\n",
      "AT4G03410  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13348]\n",
      "AT3G24570  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13348]\n",
      "AT4G33905  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13348]\n",
      "AT5G43140  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13348]\n",
      "AT1G52870  Peroxisomal membrane 22 kDa (Mpv17/PMP22) family protein [KO:K13348]\n",
      "AT2G39970  PXN; Mitochondrial substrate carrier family protein [KO:K13354]\n",
      "AT5G17380  Thiamine pyrophosphate dependent pyruvate decarboxylase family protein [KO:K12261] [EC:4.1.2.63]\n",
      "AT2G01490  PAHX; phytanoyl-CoA dioxygenase (PhyH) family protein [KO:K00477] [EC:1.14.11.18]\n",
      "AT2G35690  ACX5; acyl-CoA oxidase 5 [KO:K00232] [EC:1.3.3.6]\n",
      "AT4G16760  ACX1; acyl-CoA oxidase 1 [KO:K00232] [EC:1.3.3.6]\n",
      "AT5G65110  ACX2; acyl-CoA oxidase 2 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G51840  ACX4; acyl-CoA oxidase 4 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06290  ACX3; acyl-CoA oxidase 3 [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G06310  ACX6; acyl-CoA oxidase 6 [KO:K00232] [EC:1.3.3.6]\n",
      "AT3G06690  putative acyl-coenzyme A oxidase [KO:K00232] [EC:1.3.3.6]\n",
      "AT1G04710  PKT4; peroxisomal 3-ketoacyl-CoA thiolase 4 [KO:K07513] [EC:2.3.1.16]\n",
      "AT2G33150  PKT3; peroxisomal 3-ketoacyl-CoA thiolase 3 [KO:K07513] [EC:2.3.1.16]\n",
      "AT5G48880  KAT5; peroxisomal 3-keto-acyl-CoA thiolase 2 [KO:K07513] [EC:2.3.1.16]\n",
      "AT3G12800  SDRB; short-chain dehydrogenase-reductase B [KO:K13237] [EC:1.3.1.124]\n",
      "AT2G07640  NAD(P)-binding Rossmann-fold superfamily protein [KO:K13237] [EC:1.3.1.124]\n",
      "AT5G43280  DCI1; delta(3,5),delta(2,4)-dienoyl-CoA isomerase 1 [KO:K12663] [EC:5.3.3.21]\n",
      "AT4G14070  AAE15; acyl-activating enzyme 15 [KO:K01897] [EC:6.2.1.3]\n",
      "AT5G27600  LACS7; long-chain acyl-CoA synthetase 7 [KO:K01897] [EC:6.2.1.3]\n",
      "AT3G05970  LACS6; long-chain acyl-CoA synthetase 6 [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G64400  LACS3; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G49430  LACS2; long-chain acyl-CoA synthetase 2 [KO:K01897] [EC:6.2.1.3]\n",
      "AT1G77590  LACS9; long chain acyl-CoA synthetase 9 [KO:K01897] [EC:6.2.1.3]\n",
      "AT4G11030  AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT2G04350  LACS8; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT3G23790  AAE16; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT2G47240  LACS1; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT4G23850  LACS4; AMP-dependent synthetase and ligase family protein [KO:K01897] [EC:6.2.1.3]\n",
      "AT5G20070  NUDX19; nudix hydrolase homolog 19 [KO:K03426] [EC:3.6.1.22]\n",
      "AT4G04320  malonyl-CoA decarboxylase family protein [KO:K01578] [EC:4.1.1.9]\n",
      "AT4G33790  CER4; Jojoba acyl CoA reductase-related male sterility protein [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G11980  MS2; Jojoba acyl CoA reductase-related male sterility protein [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G44540  FAR4; fatty acid reductase 4 [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G44550  FAR5; fatty acid reductase 5 [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G44560  FAR8; fatty acid reductase 8 [KO:K13356] [EC:1.2.1.84]\n",
      "AT3G56700  FAR6; fatty acid reductase 6 [KO:K13356] [EC:1.2.1.84]\n",
      "AT5G22500  FAR1; fatty acid reductase 1 [KO:K13356] [EC:1.2.1.84]\n",
      "AT5G22420  FAR7; fatty acid reductase 7 [KO:K13356] [EC:1.2.1.84]\n",
      "AT5G27450  MK; mevalonate kinase [KO:K00869] [EC:2.7.1.36]\n",
      "AT2G13360  AGT; alanine:glyoxylate aminotransferase [KO:K00830] [EC:2.6.1.44 2.6.1.45 2.6.1.51]\n",
      "AT1G65930  cICDH; cytosolic NADP+-dependent isocitrate dehydrogenase [KO:K00031] [EC:1.1.1.42]\n",
      "AT1G54340  ICDH; isocitrate dehydrogenase [KO:K00031] [EC:1.1.1.42]\n",
      "AT5G14590  Isocitrate/isopropylmalate dehydrogenase family protein [KO:K00031] [EC:1.1.1.42]\n",
      "AT2G24580  FAD-dependent oxidoreductase family protein [KO:K00306] [EC:1.5.3.1 1.5.3.7]\n",
      "AT2G26800  Aldolase superfamily protein [KO:K01640] [EC:4.1.3.4]\n",
      "AT3G14415  GOX2; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT4G18360  GOX3; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT3G14150  HAOX2; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT3G14130  HAOX1; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT3G14420  GOX1; Aldolase-type TIM barrel family protein [KO:K11517] [EC:1.1.3.15]\n",
      "AT1G20630  CAT1; catalase 1 [KO:K03781] [EC:1.11.1.6]\n",
      "AT4G35090  CAT2; catalase 2 [KO:K03781] [EC:1.11.1.6]\n",
      "AT1G20620  CAT3; catalase 3 [KO:K03781] [EC:1.11.1.6]\n",
      "AT1G08830  CSD1; copper/zinc superoxide dismutase 1 [KO:K04565] [EC:1.15.1.1]\n",
      "AT2G28190  CSD2; copper/zinc superoxide dismutase 2 [KO:K04565] [EC:1.15.1.1]\n",
      "AT5G18100  CSD3; copper/zinc superoxide dismutase 3 [KO:K04565] [EC:1.15.1.1]\n",
      "AT3G56350  Iron/manganese superoxide dismutase family protein [KO:K04564] [EC:1.15.1.1]\n",
      "AT3G10920  MSD1; manganese superoxide dismutase 1 [KO:K04564] [EC:1.15.1.1]\n",
      "AT5G23310  FSD3; Fe superoxide dismutase 3 [KO:K04564] [EC:1.15.1.1]\n",
      "AT4G25100  FSD1; Fe superoxide dismutase 1 [KO:K04564] [EC:1.15.1.1]\n",
      "AT5G51100  FSD2; Fe superoxide dismutase 2 [KO:K04564] [EC:1.15.1.1]\n",
      "AT4G34890  XDH1; xanthine dehydrogenase 1 [KO:K00106] [EC:1.17.1.4 1.17.3.2]\n",
      "AT4G34900  XDH2; xanthine dehydrogenase 2 [KO:K00106] [EC:1.17.1.4 1.17.3.2]\n",
      "AT4G05530  IBR1; indole-3-butyric acid response 1 [KO:K11147] [EC:1.1.-.-]\n",
      "REFERENCE   PMID:17055078\n",
      "AUTHORS   Wanders RJ, Waterham HR\n",
      "TITLE     Peroxisomal disorders: the single peroxisomal enzyme deficiencies.\n",
      "JOURNAL   Biochim Biophys Acta 1763:1707-20 (2006)\n",
      "DOI:10.1016/j.bbamcr.2006.08.010\n",
      "REFERENCE   PMID:17055079\n",
      "AUTHORS   Steinberg SJ, Dodt G, Raymond GV, Braverman NE, Moser AB, Moser HW\n",
      "TITLE     Peroxisome biogenesis disorders.\n",
      "JOURNAL   Biochim Biophys Acta 1763:1733-48 (2006)\n",
      "DOI:10.1016/j.bbamcr.2006.09.010\n",
      "REFERENCE   PMID:17010456\n",
      "AUTHORS   Rottensteiner H, Theodoulou FL\n",
      "TITLE     The ins and outs of peroxisomes: co-ordination of membrane transport and peroxisomal metabolism.\n",
      "JOURNAL   Biochim Biophys Acta 1763:1527-40 (2006)\n",
      "DOI:10.1016/j.bbamcr.2006.08.012\n",
      "REFERENCE   PMID:16631601\n",
      "AUTHORS   Iida R, Yasuda T, Tsubota E, Takatsuka H, Matsuki T, Kishi K\n",
      "TITLE     Human Mpv17-like protein is localized in peroxisomes and regulates expression of antioxidant enzymes.\n",
      "JOURNAL   Biochem Biophys Res Commun 344:948-54 (2006)\n",
      "DOI:10.1016/j.bbrc.2006.04.008\n",
      "REL_PATHWAY ath00071  Fatty acid degradation\n",
      "ath00230  Purine metabolism\n",
      "ath00564  Glycerophospholipid metabolism\n",
      "ath00565  Ether lipid metabolism\n",
      "ath00900  Terpenoid backbone biosynthesis\n",
      "KO_PATHWAY  ko04146\n",
      "///\n",
      "ENTRY       ath04626                    Pathway\n",
      "NAME        Plant-pathogen interaction - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION Plants lack animal-like adaptive immunity mechanisms, and therefore have evolved a specific system with multiple layers against invading pathogens. The primary response includes the perception of pathogens by cell-surface pattern-recognition receptors (PRRs) and is referred to as PAMP-triggered immunity (PTI). Activation of FLS2 and EFR triggers MAPK signaling pathway that activates defense genes for antimictobial compounds. The increase in the cytosolic Ca2+ concentration is also a regulator for production of reactive oxygen species and localized programmed cell death/hypersensitive response. The secondary response is called effector-triggered immunity (ETI). Pathogens can acquire the ability to suppress PTI by directly injecting effector proteins into the plant cell through secretion systems. In addition, pathogens can manipulate plant hormone signaling pathways to evade host immune responses using coronatine toxin. Some plants possess specific intracellular surveillance proteins (R proteins) to monitor the presence of pathogen virulence proteins. This ETI occurs with localized programmed cell death to arrest pathogen growth, resulting in cultivar-specific disease resistance.\n",
      "CLASS       Organismal Systems; Environmental adaptation\n",
      "PATHWAY_MAP ath04626  Plant-pathogen interaction\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT3G10660  CPK2; calmodulin-domain protein kinase cdpk isoform 2 [KO:K13412] [EC:2.7.11.1]\n",
      "AT2G17290  CPK6; Calcium-dependent protein kinase family protein [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G04700  CPK27; calcium-dependent protein kinase 27 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G04695  CPK31; calcium-dependent protein kinase 31 [KO:K13412] [EC:2.7.11.1]\n",
      "AT5G19450  CDPK19; calcium-dependent protein kinase 19 [KO:K13412] [EC:2.7.11.1]\n",
      "AT2G31500  CPK24; calcium-dependent protein kinase 24 [KO:K13412] [EC:2.7.11.1]\n",
      "AT2G41860  CPK14; calcium-dependent protein kinase 14 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G09570  CPK4; calcium-dependent protein kinase 4 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G04720  CPK21; calcium-dependent protein kinase 21 [KO:K13412] [EC:2.7.11.1]\n",
      "AT5G12480  CPK7; calmodulin-domain protein kinase 7 [KO:K13412] [EC:2.7.11.1]\n",
      "AT1G74740  CPK30; calcium-dependent protein kinase 30 [KO:K13412] [EC:2.7.11.1]\n",
      "AT5G23580  CDPK9; calmodulin-like domain protein kinase 9 [KO:K13412] [EC:2.7.11.1]\n",
      "AT1G18890  CDPK1; calcium-dependent protein kinase 1 [KO:K13412] [EC:2.7.11.1]\n",
      "AT1G35670  CDPK2; calcium-dependent protein kinase 2 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G04740  CPK23; calcium-dependent protein kinase 23 [KO:K13412] [EC:2.7.11.1]\n",
      "AT2G38910  CPK20; calcium-dependent protein kinase 20 [KO:K13412] [EC:2.7.11.1]\n",
      "AT5G04870  CPK1; calcium dependent protein kinase 1 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G35310  CPK5; calmodulin-domain protein kinase 5 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G04710  CPK22; calcium-dependent kinase-like protein [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G21940  CPK15; calcium-dependent protein kinase 15 [KO:K13412] [EC:2.7.11.1]\n",
      "AT3G20410  CPK9; calmodulin-domain protein kinase 9 [KO:K13412] [EC:2.7.11.1]\n",
      "AT3G51850  CPK13; calcium-dependent protein kinase 13 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G38230  CPK26; calcium-dependent protein kinase 26 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G23650  CDPK6; calcium-dependent protein kinase 6 [KO:K13412] [EC:2.7.11.1]\n",
      "AT1G50700  CPK33; calcium-dependent protein kinase 33 [KO:K13412] [EC:2.7.11.1]\n",
      "AT3G57530  CPK32; calcium-dependent protein kinase 32 [KO:K13412] [EC:2.7.11.1]\n",
      "AT5G12180  CPK17; calcium-dependent protein kinase 17 [KO:K13412] [EC:2.7.11.1]\n",
      "AT5G19360  CPK34; calcium-dependent protein kinase 34 [KO:K13412] [EC:2.7.11.1]\n",
      "AT5G66210  CPK28; calcium-dependent protein kinase 28 [KO:K13412] [EC:2.7.11.1]\n",
      "AT2G17890  CPK16; calcium-dependent protein kinase 16 [KO:K13412] [EC:2.7.11.1]\n",
      "AT1G76040  CPK29; calcium-dependent protein kinase 29 [KO:K13412] [EC:2.7.11.1]\n",
      "AT2G35890  CPK25; calcium-dependent protein kinase 25 [KO:K13412] [EC:2.7.11.1]\n",
      "AT4G36070  CPK18; calcium-dependent protein kinase 18 [KO:K13412] [EC:2.7.11.1]\n",
      "AT1G61950  CPK19; calcium-dependent protein kinase 19 [KO:K13412] [EC:2.7.11.1]\n",
      "AT1G64060  RBOH_F; respiratory burst oxidase protein F [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT5G07390  RBOHA; respiratory burst oxidase homolog A [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT5G47910  RBOHD; respiratory burst oxidase homologue D [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT5G51060  RHD2; NADPH/respiratory burst oxidase protein D [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT4G25090  Riboflavin synthase-like superfamily protein [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT4G11230  Riboflavin synthase-like superfamily protein [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT1G19230  Riboflavin synthase-like superfamily protein [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT1G09090  RBOHB; respiratory burst oxidase-like protein [KO:K13447] [EC:1.6.3.- 1.11.1.-]\n",
      "AT1G01340  CNGC10; cyclic nucleotide gated channel 10 [KO:K05391]\n",
      "AT1G15990  CNGC7; cyclic nucleotide gated channel 7 [KO:K05391]\n",
      "AT1G19780  CNGC8; cyclic nucleotide gated channel 8 [KO:K05391]\n",
      "AT2G23980  CNGC6; cyclic nucleotide-gated channel 6 [KO:K05391]\n",
      "AT2G24610  CNGC14; cyclic nucleotide-gated channel 14 [KO:K05391]\n",
      "AT2G28260  CNGC15; cyclic nucleotide-gated channel 15 [KO:K05391]\n",
      "AT2G46430  CNGC3; cyclic nucleotide gated channel 3 [KO:K05391]\n",
      "AT2G46450  CNGC12; cyclic nucleotide-gated channel 12 [KO:K05391]\n",
      "AT3G17690  CNGC19; cyclic nucleotide gated channel 19 [KO:K05391]\n",
      "AT3G17700  CNBT1; cyclic nucleotide-binding transporter 1 [KO:K05391]\n",
      "AT3G48010  CNGC16; cyclic nucleotide-gated channel 16 [KO:K05391]\n",
      "AT4G01010  CNGC13; cyclic nucleotide-gated channel 13 [KO:K05391]\n",
      "AT4G30360  CNGC17; cyclic nucleotide-gated channel 17 [KO:K05391]\n",
      "AT4G30560  CNGC9; cyclic nucleotide gated channel 9 [KO:K05391]\n",
      "AT5G14870  CNGC18; cyclic nucleotide-gated channel 18 [KO:K05391]\n",
      "AT5G15410  DND1; Cyclic nucleotide-regulated ion channel family protein [KO:K05391]\n",
      "AT5G53130  CNGC1; cyclic nucleotide gated channel 1 [KO:K05391]\n",
      "AT5G57940  CNGC5; cyclic nucleotide gated channel 5 [KO:K05391]\n",
      "AT2G46440  CNGC11; cyclic nucleotide-gated channels [KO:K05391]\n",
      "AT5G54250  CNGC4; cyclic nucleotide-gated cation channel 4 [KO:K05391]\n",
      "AT5G37780  CAM1; calmodulin 1 [KO:K02183]\n",
      "AT3G43810  CAM7; calmodulin 7 [KO:K02183]\n",
      "AT3G56800  CAM3; calmodulin 3 [KO:K02183]\n",
      "AT1G66410  CAM4; calmodulin 4 [KO:K02183]\n",
      "AT2G41110  CAM2; calmodulin 2 [KO:K02183]\n",
      "AT2G27030  CAM5; calmodulin 5 [KO:K02183]\n",
      "AT5G21274  CAM6; calmodulin 6 [KO:K02183]\n",
      "AT2G41090  Calcium-binding EF-hand family protein [KO:K02183]\n",
      "AT5G49480  CP1; Ca2+-binding protein 1 [KO:K02183]\n",
      "AT3G03400  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT1G73630  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT3G50770  CML41; calmodulin-like 41 [KO:K13448]\n",
      "AT1G05990  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT4G37010  CEN2; centrin 2 [KO:K13448]\n",
      "AT3G25600  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT3G03410  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT1G24620  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT4G12860  UNE14; EF hand calcium-binding protein family [KO:K13448]\n",
      "AT3G10300  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT2G41410  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT5G37770  TCH2; EF hand calcium-binding protein family [KO:K13448]\n",
      "AT1G18210  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT2G36180  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT5G04170  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT3G59440  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT4G14640  CAM8; calmodulin 8 [KO:K13448]\n",
      "AT3G07490  AGD11; ARF-GAP domain 11 [KO:K13448]\n",
      "AT3G03000  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT2G43290  MSS3; Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT3G50360  CEN2; centrin2 [KO:K13448]\n",
      "AT1G12310  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT3G10190  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT5G17470  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT5G17480  PC1; pollen calcium-binding protein 1 [KO:K13448]\n",
      "AT3G22930  CML11; calmodulin-like 11 [KO:K13448]\n",
      "AT4G26470  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT1G76640  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT5G42380  CML37; calmodulin like 37 [KO:K13448]\n",
      "AT2G27480  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT5G44460  CML43; calmodulin like 43 [KO:K13448]\n",
      "AT1G76650  CML38; calmodulin-like 38 [KO:K13448]\n",
      "AT3G01830  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT3G51920  CAM9; calmodulin 9 [KO:K13448]\n",
      "AT3G47480  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT3G03430  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT1G18530  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT3G24110  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT2G41100  TCH3; Calcium-binding EF hand family protein [KO:K13448]\n",
      "AT1G21550  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT4G20780  CML42; calmodulin like 42 [KO:K13448]\n",
      "AT1G66400  CML23; calmodulin like 23 [KO:K13448]\n",
      "AT1G62820  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT4G03290  EF hand calcium-binding protein family [KO:K13448]\n",
      "AT1G32250  Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT2G15680  CML30; Calcium-binding EF-hand family protein [KO:K13448]\n",
      "AT3G47450  NOA1; P-loop containing nucleoside triphosphate hydrolases superfamily protein [KO:K13427] [EC:1.14.13.39]\n",
      "AT4G01370  MPK4; MAP kinase 4 [KO:K20600] [EC:2.7.11.24]\n",
      "AT5G46330  FLS2; Leucine-rich receptor-like protein kinase family protein [KO:K13420] [EC:2.7.11.1]\n",
      "AT4G33430  BAK1; BRI1-associated receptor kinase [KO:K13416] [EC:2.7.10.1 2.7.11.1]\n",
      "AT2G13790  SERK4; somatic embryogenesis receptor-like kinase 4 [KO:K13417] [EC:2.7.10.1 2.7.11.1]\n",
      "AT4G20360  RABE1b; RAB GTPase homolog E1B [KO:K02358]\n",
      "AT4G02930  GTP binding Elongation factor Tu family protein [KO:K02358]\n",
      "AT5G20480  EFR; EF-TU receptor [KO:K13428] [EC:2.7.11.1]\n",
      "AT4G08500  MEKK1; MAPK/ERK kinase kinase 1 [KO:K13414] [EC:2.7.11.25]\n",
      "AT5G56580  MKK6; MAP kinase kinase 6 [KO:K04368] [EC:2.7.12.2]\n",
      "AT1G51660  MKK4; mitogen-activated protein kinase kinase 4 [KO:K13413] [EC:2.7.12.2]\n",
      "AT3G21220  MKK5; MAP kinase kinase 5 [KO:K13413] [EC:2.7.12.2]\n",
      "AT3G45640  MPK3; mitogen-activated protein kinase 3 [KO:K20536] [EC:2.7.11.24]\n",
      "AT2G43790  MPK6; MAP kinase 6 [KO:K14512] [EC:2.7.11.24]\n",
      "AT3G59790  MPK10; MAP kinase 10 [KO:K14512] [EC:2.7.11.24]\n",
      "AT2G30250  WRKY25; WRKY DNA-binding protein 25 [KO:K13423]\n",
      "AT2G38470  WRKY33; WRKY DNA-binding protein 33 [KO:K13424]\n",
      "AT4G01250  WRKY22; WRKY family transcription factor [KO:K13425]\n",
      "AT4G23550  WRKY29; WRKY family transcription factor [KO:K13426]\n",
      "AT2G19190  FRK1; FLG22-induced receptor-like kinase 1 [KO:K16224]\n",
      "AT1G80460  NHO1; Actin-like ATPase superfamily protein [KO:K00864] [EC:2.7.1.30]\n",
      "AT2G14610  PR1; pathogenesis-related protein 1 [KO:K13449]\n",
      "AT4G33720  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT1G50060  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT2G14580  PRB1; basic pathogenesis-related protein 1 [KO:K13449]\n",
      "AT3G19690  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT2G19990  PR-1-LIKE; pathogenesis-related protein-1-like protein [KO:K13449]\n",
      "AT4G33730  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT4G33710  CAP (Cysteine-rich secretory proteins, Antigen 5, and Pathogenesis-related 1 protein) superfamily protein [KO:K13449]\n",
      "AT1G06700  Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT3G62220  Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT3G59350  Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT2G30730  Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT2G30740  Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT2G47060  PTI1-4; Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT3G17410  Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT1G48210  Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT2G43230  Protein kinase superfamily protein [KO:K13436] [EC:2.7.11.1]\n",
      "AT3G25070  RIN4; RPM1 interacting protein 4 [KO:K13456]\n",
      "AT3G07040  RPM1; NB-ARC domain-containing disease resistance protein [KO:K13457]\n",
      "AT5G51700  PBS2; cysteine and histidine-rich domain-containing protein RAR1 [KO:K13458]\n",
      "AT4G11260  SGT1B; phosphatase-like protein [KO:K12795]\n",
      "AT4G23570  SGT1A; phosphatase-like protein [KO:K12795]\n",
      "AT3G07770  Hsp89.1; HEAT SHOCK PROTEIN 89.1 [KO:K09487]\n",
      "AT2G04030  CR88; Chaperone protein htpG family protein [KO:K09487]\n",
      "AT4G24190  SHD; Chaperone protein htpG family protein [KO:K09487]\n",
      "AT5G52640  HSP90.1; heat shock-like protein [KO:K04079]\n",
      "AT5G56030  HSP81-2; heat shock protein 81-2 [KO:K04079]\n",
      "AT5G56000  Hsp81.4; HEAT SHOCK PROTEIN 81.4 [KO:K04079]\n",
      "AT5G56010  HSP81-3; heat shock protein 81-3 [KO:K04079]\n",
      "AT4G26090  RPS2; NB-ARC domain-containing disease resistance protein [KO:K13459]\n",
      "AT5G13160  PBS1; Protein kinase superfamily protein [KO:K13430] [EC:2.7.11.1]\n",
      "AT1G12220  RPS5; Disease resistance protein (CC-NBS-LRR class) family [KO:K13460]\n",
      "AT5G45250  RPS4; Disease resistance protein (TIR-NBS-LRR class) family [KO:K16226]\n",
      "AT3G48080  alpha/beta-Hydrolases superfamily protein [KO:K18875]\n",
      "AT3G48090  EDS1; alpha/beta-Hydrolases superfamily protein [KO:K18875]\n",
      "AT4G39090  RD19; Papain family cysteine protease [KO:K01373] [EC:3.4.22.41]\n",
      "AT2G21430  Papain family cysteine protease [KO:K01373] [EC:3.4.22.41]\n",
      "AT4G16190  Papain family cysteine protease [KO:K01373] [EC:3.4.22.41]\n",
      "AT3G54940  Papain family cysteine protease [KO:K01373] [EC:3.4.22.41]\n",
      "AT5G45260  RRS1; Disease resistance protein (TIR-NBS-LRR class) [KO:K16225]\n",
      "AT3G28910  MYB30; myb domain protein 30 [KO:K23865]\n",
      "AT2G46720  KCS13; 3-ketoacyl-CoA synthase 13 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G01120  KCS1; 3-ketoacyl-CoA synthase 1 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G26640  KCS11; 3-ketoacyl-CoA synthase 11 [KO:K15397] [EC:2.3.1.199]\n",
      "AT3G10280  KCS14; 3-ketoacyl-CoA synthase 14 [KO:K15397] [EC:2.3.1.199]\n",
      "AT4G34510  KCS17; 3-ketoacyl-CoA synthase 17 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G04220  KCS2; 3-ketoacyl-CoA synthase 2 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G19440  KCS4; 3-ketoacyl-CoA synthase 4 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G25450  KCS5; 3-ketoacyl-CoA synthase 5 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G68530  KCS6; 3-ketoacyl-CoA synthase 6 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G71160  KCS7; 3-ketoacyl-CoA synthase 7 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G15090  KCS8; 3-ketoacyl-CoA synthase 8 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G16280  KCS9; 3-ketoacyl-CoA synthase 9 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G26250  KCS10; 3-ketoacyl-CoA synthase 10 [KO:K15397] [EC:2.3.1.199]\n",
      "AT4G34250  KCS16; 3-ketoacyl-CoA synthase 16 [KO:K15397] [EC:2.3.1.199]\n",
      "AT4G34520  KCS18; 3-ketoacyl-CoA synthase 18 [KO:K15397] [EC:2.3.1.199]\n",
      "AT5G43760  KCS20; 3-ketoacyl-CoA synthase 20 [KO:K15397] [EC:2.3.1.199]\n",
      "AT3G52160  KCS15; 3-ketoacyl-CoA synthase 15 [KO:K15397] [EC:2.3.1.199]\n",
      "AT2G28630  KCS12; 3-ketoacyl-CoA synthase 12 [KO:K15397] [EC:2.3.1.199]\n",
      "AT5G04530  KCS19; 3-ketoacyl-CoA synthase 19 [KO:K15397] [EC:2.3.1.199]\n",
      "AT1G07720  KCS3; 3-ketoacyl-CoA synthase 3 [KO:K15397] [EC:2.3.1.199]\n",
      "AT5G49070  KCS21; 3-ketoacyl-CoA synthase 21 [KO:K15397] [EC:2.3.1.199]\n",
      "AT4G14440  HCD1; 3-hydroxyacyl-CoA dehydratase 1 [KO:K18880] [EC:5.3.3.8]\n",
      "AT2G04880  ZAP1; zinc-dependent activator protein-1 [KO:K18834]\n",
      "AT5G56270  WRKY2; WRKY DNA-binding protein 2 [KO:K18835]\n",
      "AT3G21630  CERK1; chitin elicitor receptor kinase 1 [KO:K13429]\n",
      "COMPOUND    C00027  Hydrogen peroxide\n",
      "C00076  Calcium cation\n",
      "C00533  Nitric oxide\n",
      "C00704  Superoxide\n",
      "C16844  Hydroxyl radical\n",
      "REFERENCE   PMID:18206360\n",
      "AUTHORS   Zipfel C\n",
      "TITLE     Pattern-recognition receptors in plant innate immunity.\n",
      "JOURNAL   Curr Opin Immunol 20:10-6 (2008)\n",
      "DOI:10.1016/j.coi.2007.11.003\n",
      "REFERENCE   PMID:19748302\n",
      "AUTHORS   Chinchilla D, Shan L, He P, de Vries S, Kemmerling B\n",
      "TITLE     One for all: the receptor-associated kinase BAK1.\n",
      "JOURNAL   Trends Plant Sci 14:535-41 (2009)\n",
      "DOI:10.1016/j.tplants.2009.08.002\n",
      "REFERENCE   PMID:19423812\n",
      "AUTHORS   Boller T, He SY\n",
      "TITLE     Innate immunity in plants: an arms race between pattern recognition receptors in plants and effectors in microbial pathogens.\n",
      "JOURNAL   Science 324:742-4 (2009)\n",
      "DOI:10.1126/science.1171647\n",
      "REFERENCE   PMID:19794842\n",
      "AUTHORS   Ma W, Smigel A, Verma R, Berkowitz GA\n",
      "TITLE     Cyclic nucleotide gated channels and related signaling components in plant innate immunity.\n",
      "JOURNAL   Plant Signal Behav 4:277-82 (2009)\n",
      "DOI:10.4161/psb.4.4.8103\n",
      "REFERENCE   PMID:11875555\n",
      "AUTHORS   Asai T, Tena G, Plotnikova J, Willmann MR, Chiu WL, Gomez-Gomez L, Boller T, Ausubel FM, Sheen J\n",
      "TITLE     MAP kinase signalling cascade in Arabidopsis innate immunity.\n",
      "JOURNAL   Nature 415:977-83 (2002)\n",
      "DOI:10.1038/415977a\n",
      "REFERENCE   PMID:15922649\n",
      "AUTHORS   Zipfel C, Felix G\n",
      "TITLE     Plants and animals: a different taste for microbes?\n",
      "JOURNAL   Curr Opin Plant Biol 8:353-60 (2005)\n",
      "DOI:10.1016/j.pbi.2005.05.004\n",
      "REFERENCE   PMID:17451411\n",
      "AUTHORS   He P, Shan L, Sheen J\n",
      "TITLE     Elicitation and suppression of microbe-associated molecular pattern-triggered immunity in plant-microbe interactions.\n",
      "JOURNAL   Cell Microbiol 9:1385-96 (2007)\n",
      "DOI:10.1111/j.1462-5822.2007.00944.x\n",
      "REFERENCE   PMID:17876517\n",
      "AUTHORS   de Wit PJ\n",
      "TITLE     How plants recognize pathogens and defend themselves.\n",
      "JOURNAL   Cell Mol Life Sci 64:2726-32 (2007)\n",
      "DOI:10.1007/s00018-007-7284-7\n",
      "REFERENCE   PMID:16497589\n",
      "AUTHORS   Chisholm ST, Coaker G, Day B, Staskawicz BJ\n",
      "TITLE     Host-microbe interactions: shaping the evolution of the plant immune response.\n",
      "JOURNAL   Cell 124:803-14 (2006)\n",
      "DOI:10.1016/j.cell.2006.02.008\n",
      "REFERENCE   PMID:16677430\n",
      "AUTHORS   McHale L, Tan X, Koehl P, Michelmore RW\n",
      "TITLE     Plant NBS-LRR proteins: adaptable guards.\n",
      "JOURNAL   Genome Biol 7:212 (2006)\n",
      "DOI:10.1186/gb-2006-7-4-212\n",
      "REFERENCE   PMID:11459065\n",
      "AUTHORS   Dangl JL, Jones JD\n",
      "TITLE     Plant pathogens and integrated defence responses to infection.\n",
      "JOURNAL   Nature 411:826-33 (2001)\n",
      "DOI:10.1038/35081161\n",
      "REFERENCE   PMID:17108957\n",
      "AUTHORS   Jones JD, Dangl JL\n",
      "TITLE     The plant immune system.\n",
      "JOURNAL   Nature 444:323-9 (2006)\n",
      "DOI:10.1038/nature05286\n",
      "REFERENCE   PMID:16697674\n",
      "AUTHORS   da Cunha L, McFall AJ, Mackey D\n",
      "TITLE     Innate immunity in plants: a continuum of layered defenses.\n",
      "JOURNAL   Microbes Infect 8:1372-81 (2006)\n",
      "DOI:10.1016/j.micinf.2005.12.018\n",
      "REFERENCE   PMID:15862106\n",
      "AUTHORS   Mudgett MB\n",
      "TITLE     New insights to the function of phytopathogenic bacterial type III effectors in plants.\n",
      "JOURNAL   Annu Rev Plant Biol 56:509-31 (2005)\n",
      "DOI:10.1146/annurev.arplant.56.032604.144218\n",
      "REFERENCE   PMID:17714518\n",
      "AUTHORS   Ma W, Berkowitz GA\n",
      "TITLE     The grateful dead: calcium and cell death in plant innate immunity.\n",
      "JOURNAL   Cell Microbiol 9:2571-85 (2007)\n",
      "DOI:10.1111/j.1462-5822.2007.01031.x\n",
      "REFERENCE   PMID:14711430\n",
      "AUTHORS   Schulze-Lefert P\n",
      "TITLE     Plant immunity: the origami of receptor activation.\n",
      "JOURNAL   Curr Biol 14:R22-4 (2004)\n",
      "DOI:10.1016/j.cub.2003.12.017\n",
      "REFERENCE   PMID:9868361\n",
      "AUTHORS   Van der Biezen EA, Jones JD\n",
      "TITLE     Plant disease-resistance proteins and the gene-for-gene concept.\n",
      "JOURNAL   Trends Biochem Sci 23:454-6 (1998)\n",
      "DOI:10.1016/S0968-0004(98)01311-5\n",
      "REFERENCE   PMID:16936700\n",
      "AUTHORS   Abramovitch RB, Anderson JC, Martin GB\n",
      "TITLE     Bacterial elicitation and evasion of plant innate immunity.\n",
      "JOURNAL   Nat Rev Mol Cell Biol 7:601-11 (2006)\n",
      "DOI:10.1038/nrm1984\n",
      "REL_PATHWAY ath04144  Endocytosis\n",
      "KO_PATHWAY  ko04626\n",
      "///\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[12:41:41]  pathway :: INFO : Performing KEGG enrichment analysis on pySeqRNA_results/diff_genes/C1-D6.txt\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ENTRY       ath04712                    Pathway\n",
      "NAME        Circadian rhythm - plant - Arabidopsis thaliana (thale cress)\n",
      "DESCRIPTION The circadian system is an endogenous rhythm with the period of about 24 hours to provide temporal organization of biological activity. In Arabidopsis the circadian oscillator at the core of this system is composed of the interlocked feedback loop formed by the major transcriptional factors CCA1 (circadian clock associated), LHY (late elongated hypocotyl) and TOC1 (timing of cab), a pseudo response regulator (PRR). The morning-expressed CCA1/LHY Myb transcription factors suppress TOC1 expression by binding to its promoter. Moreover, CCA1/LHY activates the expression of PRR7/9 in the morning and then PRR7/9 repress the transcription of CCA1/LHY during the rest of the day. By contrast, the evening-expressed TOC1 activates the expression of CCA1/LHY.\n",
      "CLASS       Organismal Systems; Environmental adaptation\n",
      "PATHWAY_MAP ath04712  Circadian rhythm - plant\n",
      "DBLINKS     GO: 0007623\n",
      "ORGANISM    Arabidopsis thaliana (thale cress) [GN:ath]\n",
      "GENE        AT2G18790  PHYB; phytochrome B [KO:K12121]\n",
      "AT1G09570  PHYA; phytochrome A [KO:K12120]\n",
      "AT1G09530  PIF3; phytochrome interacting factor 3 [KO:K12126]\n",
      "AT2G25930  ELF3; hydroxyproline-rich glycoprotein family protein [KO:K12125]\n",
      "AT2G46790  PRR9; pseudo-response regulator 9 [KO:K12128]\n",
      "AT5G02810  PRR7; pseudo-response regulator 7 [KO:K12129]\n",
      "AT5G24470  PRR5; two-component response regulator-like protein [KO:K12130]\n",
      "AT1G01060  LHY; Homeodomain-like superfamily protein [KO:K12133]\n",
      "AT2G46830  CCA1; circadian clock associated 1 [KO:K12134]\n",
      "AT2G23070  Protein kinase superfamily protein [KO:K03097] [EC:2.7.11.1]\n",
      "AT2G23080  CKA3; Protein kinase superfamily protein [KO:K03097] [EC:2.7.11.1]\n",
      "AT3G50000  CKA2; casein kinase II, alpha chain 2 [KO:K03097] [EC:2.7.11.1]\n",
      "AT5G67380  CKA1; casein kinase alpha 1 [KO:K03097] [EC:2.7.11.1]\n",
      "AT3G60250  CKB3; casein kinase II beta chain 3 [KO:K03115]\n",
      "AT4G17640  CKB2; casein kinase II beta chain 2 [KO:K03115]\n",
      "AT2G44680  CKB4; casein kinase II beta subunit 4 [KO:K03115]\n",
      "AT5G47080  CKB1; casein kinase II beta chain 1 [KO:K03115]\n",
      "AT5G08330  TCP11; TCP family transcription factor [KO:K16221]\n",
      "AT5G23280  TCP family transcription factor [KO:K16221]\n",
      "AT5G61380  TOC1; CCT motif -containing response regulator protein [KO:K12127]\n",
      "AT5G57360  ZTL; Galactose oxidase/kelch repeat superfamily protein [KO:K12115]\n",
      "AT1G22770  GI; gigantea protein (GI) [KO:K12124]\n",
      "AT5G60100  PRR3; pseudo-response regulator 3 [KO:K12131]\n",
      "AT1G56650  PAP1; production of anthocyanin pigment 1 [KO:K16166]\n",
      "AT5G13930  TT4; Chalcone and stilbene synthase family protein [KO:K00660] [EC:2.3.1.74]\n",
      "AT4G08920  CRY1; cryptochrome 1 [KO:K12118]\n",
      "AT1G04400  CRY2; cryptochrome 2 [KO:K12119]\n",
      "AT2G32950  COP1; Transducin/WD40 repeat-like superfamily protein [KO:K10143] [EC:2.3.2.27]\n",
      "AT5G52250  RUP1; Transducin/WD40 repeat-like superfamily protein [KO:K10143] [EC:2.3.2.27]\n",
      "AT5G23730  RUP2; Transducin/WD40 repeat-like superfamily protein [KO:K10143] [EC:2.3.2.27]\n",
      "AT1G68050  FKF1; flavin-binding, kelch repeat, f box 1 [KO:K12116]\n",
      "AT5G62430  CDF1; cycling DOF factor 1 [KO:K16222]\n",
      "AT5G15840  CO; B-box type zinc finger protein with CCT domain-containing protein [KO:K12135]\n",
      "AT1G65480  FT; PEBP (phosphatidylethanolamine-binding protein) family protein [KO:K16223]\n",
      "AT4G20370  TSF; PEBP (phosphatidylethanolamine-binding protein) family protein [KO:K16223]\n",
      "AT2G46340  SPA1; SPA (suppressor of phyA-105) protein family [KO:K16240]\n",
      "AT4G11110  SPA2; SPA1-related 2 [KO:K16240]\n",
      "AT5G11260  HY5; Basic-leucine zipper (bZIP) transcription factor family protein [KO:K16241]\n",
      "AT3G17609  HYH; HY5-homolog [KO:K16241]\n",
      "REFERENCE   PMID:11710980\n",
      "AUTHORS   McWatters HG, Roden LC, Staiger D.\n",
      "TITLE     Picking out parallels: plant circadian clocks in context.\n",
      "JOURNAL   Philos Trans R Soc Lond B Biol Sci 356:1735-43 (2001)\n",
      "DOI:10.1098/rstb.2001.0936\n",
      "REFERENCE   PMID:11597501\n",
      "AUTHORS   Yanovsky MJ, Kay SA.\n",
      "TITLE     Signaling networks in the plant circadian system.\n",
      "JOURNAL   Curr Opin Plant Biol 4:429-35 (2001)\n",
      "DOI:10.1016/S1369-5266(00)00196-5\n",
      "REFERENCE   PMID:10792828\n",
      "AUTHORS   Kim L, Kircher S, Toth R, Adam E, Schafer E, Nagy F.\n",
      "TITLE     Light-induced nuclear import of phytochrome-A:GFP fusion proteins is differentially regulated in transgenic tobacco and Arabidopsis.\n",
      "JOURNAL   Plant J 22:125-33 (2000)\n",
      "DOI:10.1046/j.1365-313x.2000.00729.x\n",
      "REFERENCE   PMID:8953247\n",
      "AUTHORS   Sakamoto K, Nagatani A.\n",
      "TITLE     Nuclear localization activity of phytochrome B.\n",
      "JOURNAL   Plant J 10:859-68 (1996)\n",
      "DOI:10.1046/j.1365-313X.1996.10050859.x\n",
      "REFERENCE   PMID:11402161\n",
      "AUTHORS   Liu XL, Covington MF, Fankhauser C, Chory J, Wagner DR.\n",
      "TITLE     ELF3 encodes a circadian clock-regulated nuclear protein that functions in an Arabidopsis PHYB signal transduction pathway.\n",
      "JOURNAL   Plant Cell 13:1293-304 (2001)\n",
      "DOI:10.1105/TPC.000475\n",
      "REFERENCE   PMID:11069292\n",
      "AUTHORS   Zhu Y, Tepperman JM, Fairchild CD, Quail PH.\n",
      "TITLE     Phytochrome B binds with greater apparent affinity than phytochrome A to the basic helix-loop-helix factor PIF3 in a reaction requiring the PAS domain of PIF3.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 97:13419-24 (2000)\n",
      "DOI:10.1073/pnas.230433797\n",
      "REFERENCE   PMID:9845368\n",
      "AUTHORS   Ni M, Tepperman JM, Quail PH.\n",
      "TITLE     PIF3, a phytochrome-interacting factor necessary for normal photoinduced signal transduction, is a novel basic helix-loop-helix protein.\n",
      "JOURNAL   Cell 95:657-67 (1998)\n",
      "DOI:10.1016/S0092-8674(00)81636-0\n",
      "REFERENCE   PMID:11743105\n",
      "AUTHORS   Toth R, Kevei E, Hall A, Millar AJ, Nagy F, Kozma-Bognar L.\n",
      "TITLE     Circadian clock-regulated expression of phytochrome and cryptochrome genes in Arabidopsis.\n",
      "JOURNAL   Plant Physiol 127:1607-16 (2001)\n",
      "DOI:10.1104/pp.900010\n",
      "REFERENCE   PMID:11752379\n",
      "AUTHORS   Schultz TF, Kiyosue T, Yanovsky M, Wada M, Kay SA.\n",
      "TITLE     A role for LKP2 in the circadian clock of Arabidopsis.\n",
      "JOURNAL   Plant Cell 13:2659-70 (2001)\n",
      "DOI:10.1105/tpc.010332\n",
      "REFERENCE   PMID:11260718\n",
      "AUTHORS   Jarillo JA, Capel J, Tang RH, Yang HQ, Alonso JM, Ecker JR, Cashmore AR.\n",
      "TITLE     An Arabidopsis circadian clock component interacts with both CRY1 and phyB.\n",
      "JOURNAL   Nature 410:487-90 (2001)\n",
      "DOI:10.1038/35068589\n",
      "REFERENCE   PMID:11828029\n",
      "AUTHORS   Matsushika A, Makino S, Kojima M, Yamashino T, Mizuno T.\n",
      "TITLE     The APRR1/TOC1 quintet implicated in circadian rhythms of Arabidopsis thaliana: II. Characterization with CCA1-overexpressing plants.\n",
      "JOURNAL   Plant Cell Physiol 43:118-22 (2002)\n",
      "DOI:10.1093/pcp/pcf006\n",
      "REFERENCE   PMID:10535927\n",
      "AUTHORS   Sugano S, Andronis C, Ong MS, Green RM, Tobin EM.\n",
      "TITLE     The protein kinase CK2 is involved in regulation of circadian rhythms in Arabidopsis.\n",
      "JOURNAL   Proc Natl Acad Sci U S A 96:12362-6 (1999)\n",
      "DOI:10.1073/pnas.96.22.12362\n",
      "REFERENCE   PMID:11089975\n",
      "AUTHORS   Mas P, Devlin PF, Panda S, Kay SA.\n",
      "TITLE     Functional interaction of phytochrome B and cryptochrome 2.\n",
      "JOURNAL   Nature 408:207-11 (2000)\n",
      "DOI:10.1038/35041583\n",
      "REFERENCE   PMID:9651577\n",
      "AUTHORS   Ahmad M, Jarillo JA, Smirnova O, Cashmore AR.\n",
      "TITLE     The CRY1 blue light photoreceptor of Arabidopsis interacts with phytochrome A in vitro.\n",
      "JOURNAL   Mol Cell 1:939-48 (1998)\n",
      "DOI:10.1016/S1097-2765(00)80094-5\n",
      "REFERENCE   PMID:11486091\n",
      "AUTHORS   Alabadi D, Oyama T, Yanovsky MJ, Harmon FG, Mas P, Kay SA.\n",
      "TITLE     Reciprocal regulation between TOC1 and LHY/CCA1 within the Arabidopsis circadian clock.\n",
      "JOURNAL   Science 293:880-3 (2001)\n",
      "DOI:10.1126/science.1061320\n",
      "REFERENCE   PMID:11148285\n",
      "AUTHORS   Borevitz JO, Xia Y, Blount J, Dixon RA, Lamb C.\n",
      "TITLE     Activation tagging identifies a conserved MYB regulator of phenylpropanoid biosynthesis.\n",
      "JOURNAL   Plant Cell 12:2383-2394 (2000)\n",
      "DOI:10.1105/tpc.12.12.2383\n",
      "REFERENCE   PMID:12091722\n",
      "AUTHORS   Murakami-Kojima M, Nakamichi N, Yamashino T, Mizuno T.\n",
      "TITLE     The APRR3 component of the clock-associated APRR1/TOC1 quintet is phosphorylated by a novel protein kinase belonging to the WNK family, the gene for which is also transcribed rhythmically in Arabidopsis thaliana.\n",
      "JOURNAL   Plant Cell Physiol 43:675-83 (2002)\n",
      "DOI:10.1093/pcp/pcf084\n",
      "REFERENCE   PMID:19386812\n",
      "AUTHORS   Pan Y, Michael TP, Hudson ME, Kay SA, Chory J, Schuler MA\n",
      "TITLE     Cytochrome P450 monooxygenases as reporters for circadian-regulated pathways.\n",
      "JOURNAL   Plant Physiol 150:858-78 (2009)\n",
      "DOI:10.1104/pp.108.130757\n",
      "REFERENCE   PMID:11118138\n",
      "AUTHORS   Harmer SL, Hogenesch JB, Straume M, Chang HS, Han B, Zhu T, Wang X, Kreps JA, Kay SA\n",
      "TITLE     Orchestrated transcription of key pathways in Arabidopsis by the circadian clock.\n",
      "JOURNAL   Science 290:2110-3 (2000)\n",
      "DOI:10.1126/science.290.5499.2110\n",
      "REFERENCE   PMID:12045273\n",
      "AUTHORS   Mouradov A, Cremer F, Coupland G\n",
      "TITLE     Control of flowering time: interacting pathways as a basis for diversity.\n",
      "JOURNAL   Plant Cell 14 Suppl:S111-30 (2002)\n",
      "DOI:10.1105/tpc.001362\n",
      "REFERENCE   PMID:15568973\n",
      "AUTHORS   Chen M, Chory J, Fankhauser C\n",
      "TITLE     Light signal transduction in higher plants.\n",
      "JOURNAL   Annu Rev Genet 38:87-117 (2004)\n",
      "DOI:10.1146/annurev.genet.38.072902.092259\n",
      "REFERENCE   PMID:16096959\n",
      "AUTHORS   Mas P\n",
      "TITLE     Circadian clock signaling in Arabidopsis thaliana: from gene expression to physiology and development.\n",
      "JOURNAL   Int J Dev Biol 49:491-500 (2005)\n",
      "DOI:10.1387/ijdb.041968pm\n",
      "REFERENCE   PMID:17304247\n",
      "AUTHORS   Jiao Y, Lau OS, Deng XW\n",
      "TITLE     Light-regulated transcriptional networks in higher plants.\n",
      "JOURNAL   Nat Rev Genet 8:217-30 (2007)\n",
      "DOI:10.1038/nrg2049\n",
      "REFERENCE   PMID:18444908\n",
      "AUTHORS   Turck F, Fornara F, Coupland G\n",
      "TITLE     Regulation and identity of florigen: FLOWERING LOCUS T moves center stage.\n",
      "JOURNAL   Annu Rev Plant Biol 59:573-94 (2008)\n",
      "DOI:10.1146/annurev.arplant.59.032607.092755\n",
      "REFERENCE   PMID:19836294\n",
      "AUTHORS   Imaizumi T\n",
      "TITLE     Arabidopsis circadian clock and photoperiodism: time to think about location.\n",
      "JOURNAL   Curr Opin Plant Biol 13:83-9 (2010)\n",
      "DOI:10.1016/j.pbi.2009.09.007\n",
      "REFERENCE   PMID:20459860\n",
      "AUTHORS   McClung CR\n",
      "TITLE     A modern circadian clock in the common angiosperm ancestor of monocots and eudicots.\n",
      "JOURNAL   BMC Biol 8:55 (2010)\n",
      "DOI:10.1186/1741-7007-8-55\n",
      "REFERENCE   PMID:20809800\n",
      "AUTHORS   Doherty CJ, Kay SA\n",
      "TITLE     Circadian control of global gene expression patterns.\n",
      "JOURNAL   Annu Rev Genet 44:419-44 (2010)\n",
      "DOI:10.1146/annurev-genet-102209-163432\n",
      "REFERENCE   PMID:20889330\n",
      "AUTHORS   McClung CR, Gutierrez RA\n",
      "TITLE     Network news: prime time for systems biology of the plant circadian clock.\n",
      "JOURNAL   Curr Opin Genet Dev 20:588-98 (2010)\n",
      "DOI:10.1016/j.gde.2010.08.010\n",
      "REFERENCE   PMID:22190341\n",
      "AUTHORS   Lu SX, Webb CJ, Knowles SM, Kim SH, Wang Z, Tobin EM\n",
      "TITLE     CCA1 and ELF3 Interact in the control of hypocotyl length and flowering time in Arabidopsis.\n",
      "JOURNAL   Plant Physiol 158:1079-88 (2012)\n",
      "DOI:10.1104/pp.111.189670\n",
      "REFERENCE   PMID:22705257\n",
      "AUTHORS   Lau OS, Deng XW\n",
      "TITLE     The photomorphogenic repressors COP1 and DET1: 20 years later.\n",
      "JOURNAL   Trends Plant Sci 17:584-93 (2012)\n",
      "DOI:10.1016/j.tplants.2012.05.004\n",
      "REL_PATHWAY ath00196  Photosynthesis - antenna proteins\n",
      "KO_PATHWAY  ko04712\n",
      "///\n"
     ]
    }
   ],
   "source": [
    "dk = pathway.enrichKEGG(file=\"pySeqRNA_results/diff_genes/C1-D6.txt\", species='ath')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "dk"
   ]
  }
 ],
 "metadata": {
  "interpreter": {
   "hash": "bae6830252d665342b2c3fbf83e080a2a19635753e7236c13ce78b53db4459af"
  },
  "kernelspec": {
   "display_name": "Python 3.7.11 64-bit ('pyseqrna': conda)",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.11"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
